{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "# Medical transcript classification [sklearn]\n",
    "* Multiclass classification of medical transcript.\n",
    "* Reference notebook: <https://www.kaggle.com/code/leekahwin/text-classification-using-n-gram-0-8-f1/notebook>\n",
    "* Dataset: <https://www.kaggle.com/code/leekahwin/text-classification-using-n-gram-0-8-f1/input>\n",
    "\n",
    "By running this notebook, you’ll create a whole test suite in a few lines of code. The model used here is a Random Forest classification model with the medical transcript dataset. Feel free to use your own model (tabular, text, or LLM).\n",
    "\n",
    "You’ll learn how to:\n",
    "* Detect vulnerabilities by scanning the model\n",
    "* Generate a test suite with domain-specific tests\n",
    "* Customize your test suite by loading a test from the Giskard catalog\n",
    "* Upload your model to the Giskard server to:\n",
    "    * Compare models to decide which one to promote\n",
    "    * Debug your tests to diagnose issues\n",
    "    * Share your results and collect business feedback from your team"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Install Giskard"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "pip install \"giskard>=2.0.0b\" -U"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Install necessary dependencies"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!pip install nltk"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Import libraries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:27.771693Z",
     "start_time": "2023-06-02T15:56:22.764215Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import os\n",
    "import string\n",
    "from pathlib import Path\n",
    "from typing import Iterable\n",
    "from urllib.request import urlretrieve\n",
    "\n",
    "import nltk\n",
    "import pandas as pd\n",
    "from nltk.corpus import stopwords\n",
    "from nltk.stem.snowball import SnowballStemmer\n",
    "from sklearn.ensemble import RandomForestClassifier\n",
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "from sklearn.metrics import classification_report\n",
    "from sklearn.model_selection import train_test_split\n",
    "from sklearn.pipeline import Pipeline\n",
    "from sklearn.preprocessing import FunctionTransformer\n",
    "\n",
    "import giskard\n",
    "from giskard import Dataset, Model, GiskardClient, testing"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Download NLTK stopwords corpus"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# Download list of english stopwords.\n",
    "nltk.download('stopwords')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Define constants"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:31.307739Z",
     "start_time": "2023-06-02T15:56:31.293380Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# Constants.\n",
    "LABELS_LIST = [\n",
    "    'Neurosurgery',\n",
    "    'ENT - Otolaryngology',\n",
    "    'Discharge Summary',\n",
    "    'General Medicine',\n",
    "    'Gastroenterology',\n",
    "    'Neurology',\n",
    "    'SOAP / Chart / Progress Notes',\n",
    "    'Obstetrics / Gynecology',\n",
    "    'Urology'\n",
    "]\n",
    "\n",
    "TEXT_COLUMN_NAME = \"transcription\"\n",
    "TARGET_COLUMN_NAME = \"medical_specialty\"\n",
    "\n",
    "RANDOM_SEED = 8888\n",
    "\n",
    "# Data.\n",
    "DATA_URL = os.path.join(\"ftp://sys.giskard.ai\", \"pub\", \"unit_test_resources\",\n",
    "                        \"medical_transcript_classification_dataset\", \"mtsamples.csv\")\n",
    "DATA_PATH = Path.home() / \".giskard\" / \"medical_transcript_classification_dataset\" / \"mtsamples.csv\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Dataset preparation"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Load data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:32.244129Z",
     "start_time": "2023-06-02T15:56:32.233725Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def fetch_from_ftp(url: str, file: Path) -> None:\n",
    "    \"\"\"Helper to fetch data from the FTP server.\"\"\"\n",
    "    if not file.parent.exists():\n",
    "        file.parent.mkdir(parents=True, exist_ok=True)\n",
    "\n",
    "    if not file.exists():\n",
    "        print(f\"Downloading data from {url}\")\n",
    "        urlretrieve(url, file)\n",
    "\n",
    "    print(f\"Data was loaded!\")\n",
    "\n",
    "\n",
    "def load_data() -> pd.DataFrame:\n",
    "    \"\"\"Load and initially preprocess data.\"\"\"\n",
    "    fetch_from_ftp(DATA_URL, DATA_PATH)\n",
    "\n",
    "    df = pd.read_csv(DATA_PATH)\n",
    "\n",
    "    # Drop useless columns.\n",
    "    df = df.drop(columns=['Unnamed: 0', \"description\", \"sample_name\", \"keywords\"])\n",
    "\n",
    "    # Trim text.\n",
    "    df = df.apply(lambda x: x.str.strip())\n",
    "\n",
    "    # Filter samples by label.\n",
    "    df = df[df[TARGET_COLUMN_NAME].isin(LABELS_LIST)]\n",
    "\n",
    "    # Drop rows with no transcript.\n",
    "    df = df[df[TEXT_COLUMN_NAME].notna()]\n",
    "\n",
    "    return df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:33.138579Z",
     "start_time": "2023-06-02T15:56:32.909490Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Data was loaded!\n"
     ]
    }
   ],
   "source": [
    "transcript_df = load_data()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Train-test split"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:33.407547Z",
     "start_time": "2023-06-02T15:56:33.370987Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "X_train, X_test, y_train, y_test = train_test_split(transcript_df[[TEXT_COLUMN_NAME]],\n",
    "                                                    transcript_df[TARGET_COLUMN_NAME],\n",
    "                                                    random_state=RANDOM_SEED)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Wrap dataset with Giskard"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:34.102804Z",
     "start_time": "2023-06-02T15:56:34.071403Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "raw_data = pd.concat([X_test, y_test], axis=1)\n",
    "wrapped_data = Dataset(raw_data,\n",
    "                       name=\"medical_transcript_dataset\",\n",
    "                       target=TARGET_COLUMN_NAME,\n",
    "                       column_types={TEXT_COLUMN_NAME: \"text\"})"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Model training"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Define preprocessing steps"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:35.135316Z",
     "start_time": "2023-06-02T15:56:35.106935Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "stemmer = SnowballStemmer(\"english\")\n",
    "stop_words = stopwords.words(\"english\")\n",
    "\n",
    "\n",
    "def preprocess_text(df: pd.DataFrame) -> pd.DataFrame:\n",
    "    \"\"\"Preprocess text.\"\"\"\n",
    "    # Lower.\n",
    "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(lambda x: x.lower())\n",
    "\n",
    "    # Remove punctuation.\n",
    "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(lambda x: x.translate(str.maketrans('', '', string.punctuation)))\n",
    "\n",
    "    # Tokenize.\n",
    "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(lambda x: x.split())\n",
    "\n",
    "    # Stem.\n",
    "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(lambda x: [stemmer.stem(word) for word in x])\n",
    "\n",
    "    # Remove stop-words.\n",
    "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(\n",
    "        lambda x: ' '.join([word for word in x if word not in stop_words]))\n",
    "\n",
    "    return df\n",
    "\n",
    "\n",
    "def adapt_vectorizer_input(df: pd.DataFrame) -> Iterable:\n",
    "    \"\"\"Adapt input for the vectorizers.\n",
    "\n",
    "    The problem is that vectorizers accept iterable, not DataFrame, but Series. Thus, we need to ravel dataframe with text have input single dimension.\n",
    "    Issue reference: https://stackoverflow.com/questions/50665240/valueerror-found-input-variables-with-inconsistent-numbers-of-samples-1-3185\"\"\"\n",
    "\n",
    "    df = df.iloc[:, 0]\n",
    "    return df\n",
    "\n",
    "\n",
    "text_preprocessor = FunctionTransformer(preprocess_text)\n",
    "vectorizer_input_adapter = FunctionTransformer(adapt_vectorizer_input)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Build estimator"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:54.806438Z",
     "start_time": "2023-06-02T15:56:35.867094Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                               precision    recall  f1-score   support\n",
      "\n",
      "            Discharge Summary       0.43      0.38      0.41        34\n",
      "         ENT - Otolaryngology       1.00      0.59      0.74        27\n",
      "             Gastroenterology       0.57      0.81      0.67        43\n",
      "             General Medicine       0.46      0.61      0.52        69\n",
      "                    Neurology       0.74      0.66      0.70        53\n",
      "                 Neurosurgery       0.73      0.79      0.76        24\n",
      "      Obstetrics / Gynecology       0.85      0.58      0.69        50\n",
      "SOAP / Chart / Progress Notes       0.40      0.36      0.38        33\n",
      "                      Urology       0.74      0.68      0.71        38\n",
      "\n",
      "                     accuracy                           0.61       371\n",
      "                    macro avg       0.66      0.61      0.62       371\n",
      "                 weighted avg       0.64      0.61      0.62       371\n",
      "\n"
     ]
    }
   ],
   "source": [
    "pipeline = Pipeline(steps=[\n",
    "    (\"text_preprocessor\", text_preprocessor),\n",
    "    (\"vectorizer_input_adapter\", vectorizer_input_adapter),\n",
    "    (\"vectorizer\", CountVectorizer(ngram_range=(1, 1))),\n",
    "    (\"estimator\", RandomForestClassifier(random_state=RANDOM_SEED))\n",
    "])\n",
    "\n",
    "pipeline.fit(X_train, y_train)\n",
    "y_pred = pipeline.predict(X_test)\n",
    "\n",
    "print(classification_report(y_test, y_pred))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Wrap model with Giskard"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:57.005694Z",
     "start_time": "2023-06-02T15:56:54.814070Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                               precision    recall  f1-score   support\n",
      "\n",
      "            Discharge Summary       0.43      0.38      0.41        34\n",
      "         ENT - Otolaryngology       1.00      0.59      0.74        27\n",
      "             Gastroenterology       0.57      0.81      0.67        43\n",
      "             General Medicine       0.46      0.61      0.52        69\n",
      "                    Neurology       0.74      0.66      0.70        53\n",
      "                 Neurosurgery       0.73      0.79      0.76        24\n",
      "      Obstetrics / Gynecology       0.85      0.58      0.69        50\n",
      "SOAP / Chart / Progress Notes       0.40      0.36      0.38        33\n",
      "                      Urology       0.74      0.68      0.71        38\n",
      "\n",
      "                     accuracy                           0.61       371\n",
      "                    macro avg       0.66      0.61      0.62       371\n",
      "                 weighted avg       0.64      0.61      0.62       371\n",
      "\n"
     ]
    }
   ],
   "source": [
    "wrapped_model = Model(pipeline.predict_proba,\n",
    "                      model_type=\"classification\",\n",
    "                      name=\"medical_transcript_classification\",\n",
    "                      feature_names=[TEXT_COLUMN_NAME],\n",
    "                      classification_labels=pipeline.classes_)\n",
    "\n",
    "# Validate wrapped model and data.\n",
    "print(classification_report(y_test, pipeline.classes_[wrapped_model.predict(wrapped_data).raw_prediction]))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Scan your model to find vulnerabilities\n",
    "With the Giskard scan feature, you can detect vulnerabilities in your model, including performance biases, unrobustness, data leakage, stochasticity, underconfidence, ethical issues, and more. For detailed information about the scan feature, please refer to our scan documentation."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T16:02:40.215531Z",
     "start_time": "2023-06-02T15:57:00.800368Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Your model is successfully validated.\n",
      "Running scan…\n",
      "2023-06-02 17:57:04,714 pid:10102 MainThread giskard.scanner.logger INFO     Running detectors: ['PerformanceBiasDetector', 'TextPerturbationDetector', 'EthicalBiasDetector', 'DataLeakageDetector', 'StochasticityDetector', 'OverconfidenceDetector', 'UnderconfidenceDetector']\n",
      "Running detector PerformanceBiasDetector…2023-06-02 17:57:04,715 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Running\n",
      "2023-06-02 17:57:04,715 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Calculating loss\n",
      "2023-06-02 17:57:05,038 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Loss calculated (took 0:00:00.321848)\n",
      "2023-06-02 17:57:05,038 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Finding data slices\n",
      "2023-06-02 17:58:00,022 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: 321 slices found (took 0:00:54.982054)\n",
      "2023-06-02 17:58:00,023 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Analyzing issues\n",
      "2023-06-02 17:58:00,035 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing 321 slices for performance issues.\n",
      "2023-06-02 17:58:05,414 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `language(transcription)` == \"en\"\tF1 Score = 0.614 (global 0.612) Δm = 0.003\tis_issue = False\n",
      "2023-06-02 17:58:05,438 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"section\"\tF1 Score = 0.692 (global 0.612) Δm = 0.131\tis_issue = False\n",
      "2023-06-02 17:58:05,461 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"contrast\"\tF1 Score = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:05,488 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spine\"\tF1 Score = 0.703 (global 0.612) Δm = 0.148\tis_issue = False\n",
      "2023-06-02 17:58:05,516 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inserted\"\tF1 Score = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:05,540 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"length\"\tF1 Score = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:05,567 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adequate\"\tF1 Score = 0.816 (global 0.612) Δm = 0.333\tis_issue = False\n",
      "2023-06-02 17:58:05,594 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"identified\"\tF1 Score = 0.785 (global 0.612) Δm = 0.282\tis_issue = False\n",
      "2023-06-02 17:58:05,621 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scissors\"\tF1 Score = 0.852 (global 0.612) Δm = 0.392\tis_issue = False\n",
      "2023-06-02 17:58:05,645 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colon\"\tF1 Score = 0.690 (global 0.612) Δm = 0.127\tis_issue = False\n",
      "2023-06-02 17:58:05,671 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distress\"\tF1 Score = 0.486 (global 0.612) Δm = -0.206\tis_issue = True\n",
      "2023-06-02 17:58:05,695 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"motor\"\tF1 Score = 0.795 (global 0.612) Δm = 0.300\tis_issue = False\n",
      "2023-06-02 17:58:05,717 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pain\"\tF1 Score = 0.571 (global 0.612) Δm = -0.066\tis_issue = True\n",
      "2023-06-02 17:58:05,750 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"chromic\"\tF1 Score = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:05,778 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"transverse\"\tF1 Score = 0.812 (global 0.612) Δm = 0.328\tis_issue = False\n",
      "2023-06-02 17:58:05,807 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"revealed\"\tF1 Score = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:05,838 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tumor\"\tF1 Score = 0.583 (global 0.612) Δm = -0.047\tis_issue = False\n",
      "2023-06-02 17:58:05,865 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"areas\"\tF1 Score = 0.607 (global 0.612) Δm = -0.008\tis_issue = False\n",
      "2023-06-02 17:58:05,893 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"unremarkable\"\tF1 Score = 0.632 (global 0.612) Δm = 0.033\tis_issue = False\n",
      "2023-06-02 17:58:05,917 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilateral\"\tF1 Score = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:05,941 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wound\"\tF1 Score = 0.638 (global 0.612) Δm = 0.043\tis_issue = False\n",
      "2023-06-02 17:58:05,966 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"followup\"\tF1 Score = 0.396 (global 0.612) Δm = -0.353\tis_issue = True\n",
      "2023-06-02 17:58:05,987 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"20\"\tF1 Score = 0.598 (global 0.612) Δm = -0.023\tis_issue = False\n",
      "2023-06-02 17:58:06,011 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"membranes\"\tF1 Score = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:06,037 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"created\"\tF1 Score = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:06,061 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"portion\"\tF1 Score = 0.964 (global 0.612) Δm = 0.576\tis_issue = False\n",
      "2023-06-02 17:58:06,089 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"check\"\tF1 Score = 0.560 (global 0.612) Δm = -0.085\tis_issue = True\n",
      "2023-06-02 17:58:06,111 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"days\"\tF1 Score = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:06,137 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"symptoms\"\tF1 Score = 0.551 (global 0.612) Δm = -0.099\tis_issue = True\n",
      "2023-06-02 17:58:06,161 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cord\"\tF1 Score = 0.939 (global 0.612) Δm = 0.535\tis_issue = False\n",
      "2023-06-02 17:58:06,186 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"skin\"\tF1 Score = 0.723 (global 0.612) Δm = 0.182\tis_issue = False\n",
      "2023-06-02 17:58:06,210 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"description\"\tF1 Score = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:06,236 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualization\"\tF1 Score = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:06,260 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rule\"\tF1 Score = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:06,284 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"blade\"\tF1 Score = 0.900 (global 0.612) Δm = 0.471\tis_issue = False\n",
      "2023-06-02 17:58:06,309 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cm\"\tF1 Score = 0.716 (global 0.612) Δm = 0.171\tis_issue = False\n",
      "2023-06-02 17:58:06,338 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"metzenbaum\"\tF1 Score = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:06,367 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"place\"\tF1 Score = 0.828 (global 0.612) Δm = 0.353\tis_issue = False\n",
      "2023-06-02 17:58:06,388 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"draped\"\tF1 Score = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:06,411 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"use\"\tF1 Score = 0.681 (global 0.612) Δm = 0.112\tis_issue = False\n",
      "2023-06-02 17:58:06,436 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extremity\"\tF1 Score = 0.619 (global 0.612) Δm = 0.012\tis_issue = False\n",
      "2023-06-02 17:58:06,459 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"08\"\tF1 Score = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:06,486 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"iv\"\tF1 Score = 0.758 (global 0.612) Δm = 0.239\tis_issue = False\n",
      "2023-06-02 17:58:06,511 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oxygen\"\tF1 Score = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:06,533 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscle\"\tF1 Score = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:06,560 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"difficulty\"\tF1 Score = 0.734 (global 0.612) Δm = 0.200\tis_issue = False\n",
      "2023-06-02 17:58:06,584 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"xyz\"\tF1 Score = 0.318 (global 0.612) Δm = -0.480\tis_issue = True\n",
      "2023-06-02 17:58:06,608 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"evidence\"\tF1 Score = 0.691 (global 0.612) Δm = 0.130\tis_issue = False\n",
      "2023-06-02 17:58:06,634 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suite\"\tF1 Score = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:06,660 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"informed\"\tF1 Score = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:06,687 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspirin\"\tF1 Score = 0.435 (global 0.612) Δm = -0.289\tis_issue = True\n",
      "2023-06-02 17:58:06,711 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mild\"\tF1 Score = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:06,738 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scope\"\tF1 Score = 0.864 (global 0.612) Δm = 0.411\tis_issue = False\n",
      "2023-06-02 17:58:06,775 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"resection\"\tF1 Score = 0.700 (global 0.612) Δm = 0.144\tis_issue = False\n",
      "2023-06-02 17:58:06,818 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearance\"\tF1 Score = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:06,841 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"obtained\"\tF1 Score = 0.737 (global 0.612) Δm = 0.205\tis_issue = False\n",
      "2023-06-02 17:58:06,871 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nose\"\tF1 Score = 0.794 (global 0.612) Δm = 0.298\tis_issue = False\n",
      "2023-06-02 17:58:06,900 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colonoscopy\"\tF1 Score = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:06,928 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extended\"\tF1 Score = 0.929 (global 0.612) Δm = 0.518\tis_issue = False\n",
      "2023-06-02 17:58:06,958 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"similar\"\tF1 Score = 0.724 (global 0.612) Δm = 0.184\tis_issue = False\n",
      "2023-06-02 17:58:06,981 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"examination\"\tF1 Score = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:07,009 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denied\"\tF1 Score = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:07,035 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"found\"\tF1 Score = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:07,061 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"region\"\tF1 Score = 0.724 (global 0.612) Δm = 0.184\tis_issue = False\n",
      "2023-06-02 17:58:07,088 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ml\"\tF1 Score = 0.717 (global 0.612) Δm = 0.172\tis_issue = False\n",
      "2023-06-02 17:58:07,112 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"large\"\tF1 Score = 0.647 (global 0.612) Δm = 0.058\tis_issue = False\n",
      "2023-06-02 17:58:07,136 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cc\"\tF1 Score = 0.787 (global 0.612) Δm = 0.287\tis_issue = False\n",
      "2023-06-02 17:58:07,156 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"preoperative\"\tF1 Score = 0.761 (global 0.612) Δm = 0.243\tis_issue = False\n",
      "2023-06-02 17:58:07,183 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"supine\"\tF1 Score = 0.764 (global 0.612) Δm = 0.248\tis_issue = False\n",
      "2023-06-02 17:58:07,209 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knee\"\tF1 Score = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:07,236 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"urinary\"\tF1 Score = 0.486 (global 0.612) Δm = -0.206\tis_issue = True\n",
      "2023-06-02 17:58:07,261 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligament\"\tF1 Score = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:07,290 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stomach\"\tF1 Score = 0.850 (global 0.612) Δm = 0.389\tis_issue = False\n",
      "2023-06-02 17:58:07,316 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mri\"\tF1 Score = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:07,344 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"small\"\tF1 Score = 0.770 (global 0.612) Δm = 0.259\tis_issue = False\n",
      "2023-06-02 17:58:07,372 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"compression\"\tF1 Score = 0.636 (global 0.612) Δm = 0.040\tis_issue = False\n",
      "2023-06-02 17:58:07,394 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"day\"\tF1 Score = 0.481 (global 0.612) Δm = -0.214\tis_issue = True\n",
      "2023-06-02 17:58:07,422 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualized\"\tF1 Score = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:07,449 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"entered\"\tF1 Score = 0.875 (global 0.612) Δm = 0.430\tis_issue = False\n",
      "2023-06-02 17:58:07,475 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"external\"\tF1 Score = 0.606 (global 0.612) Δm = -0.009\tis_issue = False\n",
      "2023-06-02 17:58:07,506 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bladder\"\tF1 Score = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:07,536 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"running\"\tF1 Score = 0.825 (global 0.612) Δm = 0.348\tis_issue = False\n",
      "2023-06-02 17:58:07,559 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"position\"\tF1 Score = 0.794 (global 0.612) Δm = 0.298\tis_issue = False\n",
      "2023-06-02 17:58:07,584 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"continue\"\tF1 Score = 0.442 (global 0.612) Δm = -0.277\tis_issue = True\n",
      "2023-06-02 17:58:07,608 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"laterally\"\tF1 Score = 0.846 (global 0.612) Δm = 0.383\tis_issue = False\n",
      "2023-06-02 17:58:07,633 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ear\"\tF1 Score = 0.619 (global 0.612) Δm = 0.012\tis_issue = False\n",
      "2023-06-02 17:58:07,659 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"easily\"\tF1 Score = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:07,685 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"status\"\tF1 Score = 0.515 (global 0.612) Δm = -0.158\tis_issue = True\n",
      "2023-06-02 17:58:07,709 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"home\"\tF1 Score = 0.519 (global 0.612) Δm = -0.153\tis_issue = True\n",
      "2023-06-02 17:58:07,734 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nerve\"\tF1 Score = 0.850 (global 0.612) Δm = 0.389\tis_issue = False\n",
      "2023-06-02 17:58:07,756 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"foley\"\tF1 Score = 0.730 (global 0.612) Δm = 0.193\tis_issue = False\n",
      "2023-06-02 17:58:07,782 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cardiac\"\tF1 Score = 0.529 (global 0.612) Δm = -0.135\tis_issue = True\n",
      "2023-06-02 17:58:07,811 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pupils\"\tF1 Score = 0.706 (global 0.612) Δm = 0.154\tis_issue = False\n",
      "2023-06-02 17:58:07,842 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extending\"\tF1 Score = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:07,874 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"counts\"\tF1 Score = 0.892 (global 0.612) Δm = 0.458\tis_issue = False\n",
      "2023-06-02 17:58:07,900 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"estimated\"\tF1 Score = 0.823 (global 0.612) Δm = 0.344\tis_issue = False\n",
      "2023-06-02 17:58:07,925 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gross\"\tF1 Score = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:07,957 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spinal\"\tF1 Score = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:07,980 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterus\"\tF1 Score = 0.647 (global 0.612) Δm = 0.058\tis_issue = False\n",
      "2023-06-02 17:58:08,005 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appeared\"\tF1 Score = 0.765 (global 0.612) Δm = 0.250\tis_issue = False\n",
      "2023-06-02 17:58:08,029 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removal\"\tF1 Score = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:08,057 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sterile\"\tF1 Score = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:08,081 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"able\"\tF1 Score = 0.633 (global 0.612) Δm = 0.034\tis_issue = False\n",
      "2023-06-02 17:58:08,107 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abc\"\tF1 Score = 0.417 (global 0.612) Δm = -0.319\tis_issue = True\n",
      "2023-06-02 17:58:08,132 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"exam\"\tF1 Score = 0.663 (global 0.612) Δm = 0.084\tis_issue = False\n",
      "2023-06-02 17:58:08,158 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"renal\"\tF1 Score = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:08,181 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hemostasis\"\tF1 Score = 0.787 (global 0.612) Δm = 0.286\tis_issue = False\n",
      "2023-06-02 17:58:08,207 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"proximal\"\tF1 Score = 0.889 (global 0.612) Δm = 0.453\tis_issue = False\n",
      "2023-06-02 17:58:08,231 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectal\"\tF1 Score = 0.643 (global 0.612) Δm = 0.051\tis_issue = False\n",
      "2023-06-02 17:58:08,256 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"12\"\tF1 Score = 0.541 (global 0.612) Δm = -0.117\tis_issue = True\n",
      "2023-06-02 17:58:08,281 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"attention\"\tF1 Score = 0.786 (global 0.612) Δm = 0.284\tis_issue = False\n",
      "2023-06-02 17:58:08,308 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"activity\"\tF1 Score = 0.514 (global 0.612) Δm = -0.161\tis_issue = True\n",
      "2023-06-02 17:58:08,334 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"father\"\tF1 Score = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:08,359 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissection\"\tF1 Score = 0.762 (global 0.612) Δm = 0.245\tis_issue = False\n",
      "2023-06-02 17:58:08,384 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brain\"\tF1 Score = 0.645 (global 0.612) Δm = 0.054\tis_issue = False\n",
      "2023-06-02 17:58:08,412 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tubes\"\tF1 Score = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:08,438 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bleeding\"\tF1 Score = 0.727 (global 0.612) Δm = 0.189\tis_issue = False\n",
      "2023-06-02 17:58:08,462 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mouth\"\tF1 Score = 0.700 (global 0.612) Δm = 0.144\tis_issue = False\n",
      "2023-06-02 17:58:08,487 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incision\"\tF1 Score = 0.747 (global 0.612) Δm = 0.221\tis_issue = False\n",
      "2023-06-02 17:58:08,511 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wall\"\tF1 Score = 0.763 (global 0.612) Δm = 0.247\tis_issue = False\n",
      "2023-06-02 17:58:08,539 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"initially\"\tF1 Score = 0.517 (global 0.612) Δm = -0.155\tis_issue = True\n",
      "2023-06-02 17:58:08,563 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"family\"\tF1 Score = 0.573 (global 0.612) Δm = -0.063\tis_issue = True\n",
      "2023-06-02 17:58:08,586 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removed\"\tF1 Score = 0.790 (global 0.612) Δm = 0.291\tis_issue = False\n",
      "2023-06-02 17:58:08,610 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"treatment\"\tF1 Score = 0.518 (global 0.612) Δm = -0.154\tis_issue = True\n",
      "2023-06-02 17:58:08,634 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nasal\"\tF1 Score = 0.594 (global 0.612) Δm = -0.030\tis_issue = False\n",
      "2023-06-02 17:58:08,659 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fever\"\tF1 Score = 0.558 (global 0.612) Δm = -0.089\tis_issue = True\n",
      "2023-06-02 17:58:08,687 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reports\"\tF1 Score = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:08,710 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anterior\"\tF1 Score = 0.756 (global 0.612) Δm = 0.236\tis_issue = False\n",
      "2023-06-02 17:58:08,734 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"leg\"\tF1 Score = 0.567 (global 0.612) Δm = -0.074\tis_issue = True\n",
      "2023-06-02 17:58:08,761 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"changes\"\tF1 Score = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:08,788 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"strength\"\tF1 Score = 0.738 (global 0.612) Δm = 0.206\tis_issue = False\n",
      "2023-06-02 17:58:08,811 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fashion\"\tF1 Score = 0.807 (global 0.612) Δm = 0.319\tis_issue = False\n",
      "2023-06-02 17:58:08,836 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fascia\"\tF1 Score = 0.824 (global 0.612) Δm = 0.346\tis_issue = False\n",
      "2023-06-02 17:58:08,861 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lumbar\"\tF1 Score = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:08,888 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"enlarged\"\tF1 Score = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:08,912 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denies\"\tF1 Score = 0.459 (global 0.612) Δm = -0.250\tis_issue = True\n",
      "2023-06-02 17:58:08,941 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"office\"\tF1 Score = 0.476 (global 0.612) Δm = -0.222\tis_issue = True\n",
      "2023-06-02 17:58:08,966 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adenopathy\"\tF1 Score = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:08,992 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abnormalities\"\tF1 Score = 0.656 (global 0.612) Δm = 0.073\tis_issue = False\n",
      "2023-06-02 17:58:09,013 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"placed\"\tF1 Score = 0.780 (global 0.612) Δm = 0.274\tis_issue = False\n",
      "2023-06-02 17:58:09,041 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"improved\"\tF1 Score = 0.419 (global 0.612) Δm = -0.315\tis_issue = True\n",
      "2023-06-02 17:58:09,067 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mucosa\"\tF1 Score = 0.697 (global 0.612) Δm = 0.139\tis_issue = False\n",
      "2023-06-02 17:58:09,094 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"return\"\tF1 Score = 0.514 (global 0.612) Δm = -0.159\tis_issue = True\n",
      "2023-06-02 17:58:09,118 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"post\"\tF1 Score = 0.552 (global 0.612) Δm = -0.098\tis_issue = True\n",
      "2023-06-02 17:58:09,141 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"noted\"\tF1 Score = 0.651 (global 0.612) Δm = 0.064\tis_issue = False\n",
      "2023-06-02 17:58:09,167 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"closed\"\tF1 Score = 0.806 (global 0.612) Δm = 0.317\tis_issue = False\n",
      "2023-06-02 17:58:09,192 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distal\"\tF1 Score = 0.795 (global 0.612) Δm = 0.299\tis_issue = False\n",
      "2023-06-02 17:58:09,222 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"specimens\"\tF1 Score = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:09,250 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sponge\"\tF1 Score = 0.886 (global 0.612) Δm = 0.448\tis_issue = False\n",
      "2023-06-02 17:58:09,276 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clear\"\tF1 Score = 0.515 (global 0.612) Δm = -0.158\tis_issue = True\n",
      "2023-06-02 17:58:09,298 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"room\"\tF1 Score = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:09,328 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"weakness\"\tF1 Score = 0.732 (global 0.612) Δm = 0.196\tis_issue = False\n",
      "2023-06-02 17:58:09,362 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"superior\"\tF1 Score = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:09,388 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"neck\"\tF1 Score = 0.553 (global 0.612) Δm = -0.096\tis_issue = True\n",
      "2023-06-02 17:58:09,421 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"infection\"\tF1 Score = 0.607 (global 0.612) Δm = -0.009\tis_issue = False\n",
      "2023-06-02 17:58:09,449 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drainage\"\tF1 Score = 0.630 (global 0.612) Δm = 0.029\tis_issue = False\n",
      "2023-06-02 17:58:09,476 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ct\"\tF1 Score = 0.600 (global 0.612) Δm = -0.019\tis_issue = False\n",
      "2023-06-02 17:58:09,507 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"swelling\"\tF1 Score = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:09,532 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mass\"\tF1 Score = 0.596 (global 0.612) Δm = -0.026\tis_issue = False\n",
      "2023-06-02 17:58:09,563 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinic\"\tF1 Score = 0.542 (global 0.612) Δm = -0.115\tis_issue = True\n",
      "2023-06-02 17:58:09,589 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pelvis\"\tF1 Score = 0.733 (global 0.612) Δm = 0.199\tis_issue = False\n",
      "2023-06-02 17:58:09,615 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"taken\"\tF1 Score = 0.713 (global 0.612) Δm = 0.165\tis_issue = False\n",
      "2023-06-02 17:58:09,641 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervix\"\tF1 Score = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:09,669 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bovie\"\tF1 Score = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:09,697 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinical\"\tF1 Score = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:09,722 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"upper\"\tF1 Score = 0.718 (global 0.612) Δm = 0.173\tis_issue = False\n",
      "2023-06-02 17:58:09,748 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"arm\"\tF1 Score = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:09,777 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesions\"\tF1 Score = 0.681 (global 0.612) Δm = 0.113\tis_issue = False\n",
      "2023-06-02 17:58:09,804 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tone\"\tF1 Score = 0.759 (global 0.612) Δm = 0.240\tis_issue = False\n",
      "2023-06-02 17:58:09,832 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"single\"\tF1 Score = 0.696 (global 0.612) Δm = 0.137\tis_issue = False\n",
      "2023-06-02 17:58:09,859 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drains\"\tF1 Score = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:09,879 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"right\"\tF1 Score = 0.659 (global 0.612) Δm = 0.077\tis_issue = False\n",
      "2023-06-02 17:58:09,909 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearing\"\tF1 Score = 0.680 (global 0.612) Δm = 0.111\tis_issue = False\n",
      "2023-06-02 17:58:09,934 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"head\"\tF1 Score = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:09,965 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reflexes\"\tF1 Score = 0.775 (global 0.612) Δm = 0.267\tis_issue = False\n",
      "2023-06-02 17:58:09,988 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abdominal\"\tF1 Score = 0.615 (global 0.612) Δm = 0.006\tis_issue = False\n",
      "2023-06-02 17:58:10,015 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"needle\"\tF1 Score = 0.821 (global 0.612) Δm = 0.343\tis_issue = False\n",
      "2023-06-02 17:58:10,039 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"daily\"\tF1 Score = 0.385 (global 0.612) Δm = -0.371\tis_issue = True\n",
      "2023-06-02 17:58:10,074 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthetic\"\tF1 Score = 0.913 (global 0.612) Δm = 0.492\tis_issue = False\n",
      "2023-06-02 17:58:10,105 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesion\"\tF1 Score = 0.692 (global 0.612) Δm = 0.131\tis_issue = False\n",
      "2023-06-02 17:58:10,131 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complications\"\tF1 Score = 0.697 (global 0.612) Δm = 0.140\tis_issue = False\n",
      "2023-06-02 17:58:10,155 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"postoperative\"\tF1 Score = 0.729 (global 0.612) Δm = 0.191\tis_issue = False\n",
      "2023-06-02 17:58:10,185 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"follow\"\tF1 Score = 0.550 (global 0.612) Δm = -0.101\tis_issue = True\n",
      "2023-06-02 17:58:10,215 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visual\"\tF1 Score = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:10,242 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"end\"\tF1 Score = 0.757 (global 0.612) Δm = 0.237\tis_issue = False\n",
      "2023-06-02 17:58:10,273 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluids\"\tF1 Score = 0.786 (global 0.612) Δm = 0.284\tis_issue = False\n",
      "2023-06-02 17:58:10,301 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"count\"\tF1 Score = 0.400 (global 0.612) Δm = -0.346\tis_issue = True\n",
      "2023-06-02 17:58:10,332 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspect\"\tF1 Score = 0.787 (global 0.612) Δm = 0.287\tis_issue = False\n",
      "2023-06-02 17:58:10,360 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"biopsy\"\tF1 Score = 0.625 (global 0.612) Δm = 0.021\tis_issue = False\n",
      "2023-06-02 17:58:10,382 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"performed\"\tF1 Score = 0.683 (global 0.612) Δm = 0.116\tis_issue = False\n",
      "2023-06-02 17:58:10,413 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mg\"\tF1 Score = 0.473 (global 0.612) Δm = -0.227\tis_issue = True\n",
      "2023-06-02 17:58:10,451 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"open\"\tF1 Score = 0.769 (global 0.612) Δm = 0.257\tis_issue = False\n",
      "2023-06-02 17:58:10,483 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"discharge\"\tF1 Score = 0.424 (global 0.612) Δm = -0.308\tis_issue = True\n",
      "2023-06-02 17:58:10,514 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stool\"\tF1 Score = 0.516 (global 0.612) Δm = -0.156\tis_issue = True\n",
      "2023-06-02 17:58:10,543 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carefully\"\tF1 Score = 0.826 (global 0.612) Δm = 0.350\tis_issue = False\n",
      "2023-06-02 17:58:10,579 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"age\"\tF1 Score = 0.639 (global 0.612) Δm = 0.044\tis_issue = False\n",
      "2023-06-02 17:58:10,610 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headaches\"\tF1 Score = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:10,637 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"high\"\tF1 Score = 0.488 (global 0.612) Δm = -0.203\tis_issue = True\n",
      "2023-06-02 17:58:10,665 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"erythema\"\tF1 Score = 0.655 (global 0.612) Δm = 0.071\tis_issue = False\n",
      "2023-06-02 17:58:10,689 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthesia\"\tF1 Score = 0.764 (global 0.612) Δm = 0.249\tis_issue = False\n",
      "2023-06-02 17:58:10,712 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"midline\"\tF1 Score = 0.836 (global 0.612) Δm = 0.367\tis_issue = False\n",
      "2023-06-02 17:58:10,737 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"equal\"\tF1 Score = 0.655 (global 0.612) Δm = 0.071\tis_issue = False\n",
      "2023-06-02 17:58:10,763 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"numbness\"\tF1 Score = 0.815 (global 0.612) Δm = 0.332\tis_issue = False\n",
      "2023-06-02 17:58:10,792 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suction\"\tF1 Score = 0.826 (global 0.612) Δm = 0.350\tis_issue = False\n",
      "2023-06-02 17:58:10,817 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pulmonary\"\tF1 Score = 0.533 (global 0.612) Δm = -0.128\tis_issue = True\n",
      "2023-06-02 17:58:10,848 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"underlying\"\tF1 Score = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:10,874 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cancer\"\tF1 Score = 0.622 (global 0.612) Δm = 0.016\tis_issue = False\n",
      "2023-06-02 17:58:10,901 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scan\"\tF1 Score = 0.568 (global 0.612) Δm = -0.071\tis_issue = True\n",
      "2023-06-02 17:58:10,913 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"patient\"\tF1 Score = 0.597 (global 0.612) Δm = -0.023\tis_issue = False\n",
      "2023-06-02 17:58:10,942 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"passed\"\tF1 Score = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:10,976 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"saline\"\tF1 Score = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:11,004 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cautery\"\tF1 Score = 0.741 (global 0.612) Δm = 0.211\tis_issue = False\n",
      "2023-06-02 17:58:11,029 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"body\"\tF1 Score = 0.816 (global 0.612) Δm = 0.333\tis_issue = False\n",
      "2023-06-02 17:58:11,057 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervical\"\tF1 Score = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:11,081 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prostate\"\tF1 Score = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:11,111 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sleep\"\tF1 Score = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:11,138 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"epinephrine\"\tF1 Score = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:11,166 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"shoulder\"\tF1 Score = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:11,192 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tenderness\"\tF1 Score = 0.543 (global 0.612) Δm = -0.112\tis_issue = True\n",
      "2023-06-02 17:58:11,223 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stitch\"\tF1 Score = 0.909 (global 0.612) Δm = 0.486\tis_issue = False\n",
      "2023-06-02 17:58:11,251 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sinus\"\tF1 Score = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:11,276 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"point\"\tF1 Score = 0.615 (global 0.612) Δm = 0.006\tis_issue = False\n",
      "2023-06-02 17:58:11,304 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"second\"\tF1 Score = 0.679 (global 0.612) Δm = 0.110\tis_issue = False\n",
      "2023-06-02 17:58:11,338 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterine\"\tF1 Score = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:11,373 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"breast\"\tF1 Score = 0.444 (global 0.612) Δm = -0.274\tis_issue = True\n",
      "2023-06-02 17:58:11,399 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"having\"\tF1 Score = 0.523 (global 0.612) Δm = -0.145\tis_issue = True\n",
      "2023-06-02 17:58:11,433 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"local\"\tF1 Score = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:11,462 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"normal\"\tF1 Score = 0.641 (global 0.612) Δm = 0.048\tis_issue = False\n",
      "2023-06-02 17:58:11,501 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stable\"\tF1 Score = 0.627 (global 0.612) Δm = 0.025\tis_issue = False\n",
      "2023-06-02 17:58:11,543 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sutures\"\tF1 Score = 0.636 (global 0.612) Δm = 0.040\tis_issue = False\n",
      "2023-06-02 17:58:11,577 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"admission\"\tF1 Score = 0.375 (global 0.612) Δm = -0.387\tis_issue = True\n",
      "2023-06-02 17:58:11,607 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"round\"\tF1 Score = 0.774 (global 0.612) Δm = 0.265\tis_issue = False\n",
      "2023-06-02 17:58:11,638 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"artery\"\tF1 Score = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:11,668 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"problems\"\tF1 Score = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:11,699 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recurrent\"\tF1 Score = 0.654 (global 0.612) Δm = 0.069\tis_issue = False\n",
      "2023-06-02 17:58:11,731 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medication\"\tF1 Score = 0.541 (global 0.612) Δm = -0.117\tis_issue = True\n",
      "2023-06-02 17:58:11,765 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sac\"\tF1 Score = 0.913 (global 0.612) Δm = 0.492\tis_issue = False\n",
      "2023-06-02 17:58:11,801 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"s1\"\tF1 Score = 0.556 (global 0.612) Δm = -0.092\tis_issue = True\n",
      "2023-06-02 17:58:11,834 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consistent\"\tF1 Score = 0.828 (global 0.612) Δm = 0.353\tis_issue = False\n",
      "2023-06-02 17:58:11,872 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"diarrhea\"\tF1 Score = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:11,905 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tolerated\"\tF1 Score = 0.698 (global 0.612) Δm = 0.140\tis_issue = False\n",
      "2023-06-02 17:58:11,934 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headache\"\tF1 Score = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:11,972 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scalpel\"\tF1 Score = 0.957 (global 0.612) Δm = 0.563\tis_issue = False\n",
      "2023-06-02 17:58:12,006 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lymph\"\tF1 Score = 0.769 (global 0.612) Δm = 0.257\tis_issue = False\n",
      "2023-06-02 17:58:12,033 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pregnancy\"\tF1 Score = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:12,061 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complex\"\tF1 Score = 0.522 (global 0.612) Δm = -0.147\tis_issue = True\n",
      "2023-06-02 17:58:12,094 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"care\"\tF1 Score = 0.537 (global 0.612) Δm = -0.122\tis_issue = True\n",
      "2023-06-02 17:58:12,124 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"units\"\tF1 Score = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:12,155 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hx\"\tF1 Score = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:12,190 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscles\"\tF1 Score = 0.815 (global 0.612) Δm = 0.332\tis_issue = False\n",
      "2023-06-02 17:58:12,223 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"irrigated\"\tF1 Score = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:12,255 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consent\"\tF1 Score = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:12,289 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reveals\"\tF1 Score = 0.652 (global 0.612) Δm = 0.066\tis_issue = False\n",
      "2023-06-02 17:58:12,327 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reviewed\"\tF1 Score = 0.529 (global 0.612) Δm = -0.135\tis_issue = True\n",
      "2023-06-02 17:58:12,364 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subsequently\"\tF1 Score = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:12,399 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oropharynx\"\tF1 Score = 0.686 (global 0.612) Δm = 0.121\tis_issue = False\n",
      "2023-06-02 17:58:12,418 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"history\"\tF1 Score = 0.564 (global 0.612) Δm = -0.078\tis_issue = True\n",
      "2023-06-02 17:58:12,449 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectus\"\tF1 Score = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:12,478 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prepped\"\tF1 Score = 0.767 (global 0.612) Δm = 0.253\tis_issue = False\n",
      "2023-06-02 17:58:12,509 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suture\"\tF1 Score = 0.804 (global 0.612) Δm = 0.315\tis_issue = False\n",
      "2023-06-02 17:58:12,543 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clamped\"\tF1 Score = 0.960 (global 0.612) Δm = 0.569\tis_issue = False\n",
      "2023-06-02 17:58:12,574 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"function\"\tF1 Score = 0.414 (global 0.612) Δm = -0.324\tis_issue = True\n",
      "2023-06-02 17:58:12,608 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"coronary\"\tF1 Score = 0.391 (global 0.612) Δm = -0.360\tis_issue = True\n",
      "2023-06-02 17:58:12,640 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"speech\"\tF1 Score = 0.704 (global 0.612) Δm = 0.150\tis_issue = False\n",
      "2023-06-02 17:58:12,675 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vaginal\"\tF1 Score = 0.586 (global 0.612) Δm = -0.042\tis_issue = False\n",
      "2023-06-02 17:58:12,707 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subcuticular\"\tF1 Score = 0.792 (global 0.612) Δm = 0.294\tis_issue = False\n",
      "2023-06-02 17:58:12,745 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cut\"\tF1 Score = 0.917 (global 0.612) Δm = 0.498\tis_issue = False\n",
      "2023-06-02 17:58:12,776 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"correct\"\tF1 Score = 0.881 (global 0.612) Δm = 0.440\tis_issue = False\n",
      "2023-06-02 17:58:12,814 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dorsal\"\tF1 Score = 0.929 (global 0.612) Δm = 0.518\tis_issue = False\n",
      "2023-06-02 17:58:12,849 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"indications\"\tF1 Score = 0.889 (global 0.612) Δm = 0.453\tis_issue = False\n",
      "2023-06-02 17:58:12,883 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"technique\"\tF1 Score = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:12,914 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"findings\"\tF1 Score = 0.691 (global 0.612) Δm = 0.129\tis_issue = False\n",
      "2023-06-02 17:58:12,946 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"greater\"\tF1 Score = 0.435 (global 0.612) Δm = -0.289\tis_issue = True\n",
      "2023-06-02 17:58:12,984 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"presentation\"\tF1 Score = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:13,025 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"internal\"\tF1 Score = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:13,054 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligated\"\tF1 Score = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:13,084 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissected\"\tF1 Score = 0.881 (global 0.612) Δm = 0.440\tis_issue = False\n",
      "2023-06-02 17:58:13,124 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suctioned\"\tF1 Score = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:13,157 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lung\"\tF1 Score = 0.400 (global 0.612) Δm = -0.346\tis_issue = True\n",
      "2023-06-02 17:58:13,185 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"posterior\"\tF1 Score = 0.768 (global 0.612) Δm = 0.255\tis_issue = False\n",
      "2023-06-02 17:58:13,221 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hours\"\tF1 Score = 0.558 (global 0.612) Δm = -0.089\tis_issue = True\n",
      "2023-06-02 17:58:13,258 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vicryl\"\tF1 Score = 0.791 (global 0.612) Δm = 0.293\tis_issue = False\n",
      "2023-06-02 17:58:13,340 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knife\"\tF1 Score = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:13,385 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilaterally\"\tF1 Score = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:13,422 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"administered\"\tF1 Score = 0.767 (global 0.612) Δm = 0.253\tis_issue = False\n",
      "2023-06-02 17:58:13,457 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"grasped\"\tF1 Score = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:13,492 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subjective\"\tF1 Score = 0.367 (global 0.612) Δm = -0.401\tis_issue = True\n",
      "2023-06-02 17:58:13,522 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carotid\"\tF1 Score = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:13,564 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"impression\"\tF1 Score = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:13,599 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medial\"\tF1 Score = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:13,620 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"left\"\tF1 Score = 0.693 (global 0.612) Δm = 0.132\tis_issue = False\n",
      "2023-06-02 17:58:13,650 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mm\"\tF1 Score = 0.827 (global 0.612) Δm = 0.351\tis_issue = False\n",
      "2023-06-02 17:58:13,686 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"condition\"\tF1 Score = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:13,722 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gait\"\tF1 Score = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:13,754 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inferior\"\tF1 Score = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:13,791 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"masses\"\tF1 Score = 0.605 (global 0.612) Δm = -0.011\tis_issue = False\n",
      "2023-06-02 17:58:13,831 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectum\"\tF1 Score = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:13,863 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brought\"\tF1 Score = 0.747 (global 0.612) Δm = 0.221\tis_issue = False\n",
      "2023-06-02 17:58:13,893 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"delivered\"\tF1 Score = 0.920 (global 0.612) Δm = 0.504\tis_issue = False\n",
      "2023-06-02 17:58:13,937 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gentleman\"\tF1 Score = 0.696 (global 0.612) Δm = 0.137\tis_issue = False\n",
      "2023-06-02 17:58:13,966 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluid\"\tF1 Score = 0.759 (global 0.612) Δm = 0.241\tis_issue = False\n",
      "2023-06-02 17:58:14,003 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"change\"\tF1 Score = 0.725 (global 0.612) Δm = 0.185\tis_issue = False\n",
      "2023-06-02 17:58:14,039 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"light\"\tF1 Score = 0.674 (global 0.612) Δm = 0.102\tis_issue = False\n",
      "2023-06-02 17:58:14,073 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tube\"\tF1 Score = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:14,097 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"loss\"\tF1 Score = 0.781 (global 0.612) Δm = 0.276\tis_issue = False\n",
      "2023-06-02 17:58:14,134 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lateral\"\tF1 Score = 0.797 (global 0.612) Δm = 0.303\tis_issue = False\n",
      "2023-06-02 17:58:14,175 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sensory\"\tF1 Score = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:14,212 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vision\"\tF1 Score = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:14,251 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incised\"\tF1 Score = 0.900 (global 0.612) Δm = 0.471\tis_issue = False\n",
      "2023-06-02 17:58:14,288 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visit\"\tF1 Score = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:14,325 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tract\"\tF1 Score = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:14,354 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"procedure\"\tF1 Score = 0.726 (global 0.612) Δm = 0.187\tis_issue = False\n",
      "2023-06-02 17:58:14,401 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"advanced\"\tF1 Score = 0.759 (global 0.612) Δm = 0.240\tis_issue = False\n",
      "2023-06-02 17:58:14,434 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cavity\"\tF1 Score = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:14,472 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"flap\"\tF1 Score = 0.727 (global 0.612) Δm = 0.189\tis_issue = False\n",
      "2023-06-02 17:58:14,510 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"negative\"\tF1 Score = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:14,542 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"direct\"\tF1 Score = 0.731 (global 0.612) Δm = 0.194\tis_issue = False\n",
      "2023-06-02 17:58:14,574 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"months\"\tF1 Score = 0.562 (global 0.612) Δm = -0.081\tis_issue = True\n",
      "2023-06-02 17:58:14,614 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"states\"\tF1 Score = 0.535 (global 0.612) Δm = -0.126\tis_issue = True\n",
      "2023-06-02 17:58:14,648 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"following\"\tF1 Score = 0.609 (global 0.612) Δm = -0.004\tis_issue = False\n",
      "2023-06-02 17:58:14,685 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recovery\"\tF1 Score = 0.843 (global 0.612) Δm = 0.378\tis_issue = False\n",
      "2023-06-02 17:58:20,078 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `language(transcription)` == \"en\"\tPrecision = 0.614 (global 0.612) Δm = 0.003\tis_issue = False\n",
      "2023-06-02 17:58:20,103 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"section\"\tPrecision = 0.692 (global 0.612) Δm = 0.131\tis_issue = False\n",
      "2023-06-02 17:58:20,127 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"contrast\"\tPrecision = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:20,156 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spine\"\tPrecision = 0.703 (global 0.612) Δm = 0.148\tis_issue = False\n",
      "2023-06-02 17:58:20,187 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inserted\"\tPrecision = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:20,220 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"length\"\tPrecision = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:20,246 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adequate\"\tPrecision = 0.816 (global 0.612) Δm = 0.333\tis_issue = False\n",
      "2023-06-02 17:58:20,277 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"identified\"\tPrecision = 0.785 (global 0.612) Δm = 0.282\tis_issue = False\n",
      "2023-06-02 17:58:20,305 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scissors\"\tPrecision = 0.852 (global 0.612) Δm = 0.392\tis_issue = False\n",
      "2023-06-02 17:58:20,334 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colon\"\tPrecision = 0.690 (global 0.612) Δm = 0.127\tis_issue = False\n",
      "2023-06-02 17:58:20,359 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distress\"\tPrecision = 0.486 (global 0.612) Δm = -0.206\tis_issue = True\n",
      "2023-06-02 17:58:20,384 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"motor\"\tPrecision = 0.795 (global 0.612) Δm = 0.300\tis_issue = False\n",
      "2023-06-02 17:58:20,408 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pain\"\tPrecision = 0.571 (global 0.612) Δm = -0.066\tis_issue = True\n",
      "2023-06-02 17:58:20,436 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"chromic\"\tPrecision = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:20,461 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"transverse\"\tPrecision = 0.812 (global 0.612) Δm = 0.328\tis_issue = False\n",
      "2023-06-02 17:58:20,495 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"revealed\"\tPrecision = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:20,530 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tumor\"\tPrecision = 0.583 (global 0.612) Δm = -0.047\tis_issue = False\n",
      "2023-06-02 17:58:20,561 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"areas\"\tPrecision = 0.607 (global 0.612) Δm = -0.008\tis_issue = False\n",
      "2023-06-02 17:58:20,589 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"unremarkable\"\tPrecision = 0.632 (global 0.612) Δm = 0.033\tis_issue = False\n",
      "2023-06-02 17:58:20,610 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilateral\"\tPrecision = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:20,640 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wound\"\tPrecision = 0.638 (global 0.612) Δm = 0.043\tis_issue = False\n",
      "2023-06-02 17:58:20,665 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"followup\"\tPrecision = 0.396 (global 0.612) Δm = -0.353\tis_issue = True\n",
      "2023-06-02 17:58:20,691 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"20\"\tPrecision = 0.598 (global 0.612) Δm = -0.023\tis_issue = False\n",
      "2023-06-02 17:58:20,718 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"membranes\"\tPrecision = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:20,747 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"created\"\tPrecision = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:20,775 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"portion\"\tPrecision = 0.964 (global 0.612) Δm = 0.576\tis_issue = False\n",
      "2023-06-02 17:58:20,803 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"check\"\tPrecision = 0.560 (global 0.612) Δm = -0.085\tis_issue = True\n",
      "2023-06-02 17:58:20,830 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"days\"\tPrecision = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:20,858 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"symptoms\"\tPrecision = 0.551 (global 0.612) Δm = -0.099\tis_issue = True\n",
      "2023-06-02 17:58:20,891 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cord\"\tPrecision = 0.939 (global 0.612) Δm = 0.535\tis_issue = False\n",
      "2023-06-02 17:58:20,922 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"skin\"\tPrecision = 0.723 (global 0.612) Δm = 0.182\tis_issue = False\n",
      "2023-06-02 17:58:20,952 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"description\"\tPrecision = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:20,977 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualization\"\tPrecision = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:21,009 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rule\"\tPrecision = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:21,034 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"blade\"\tPrecision = 0.900 (global 0.612) Δm = 0.471\tis_issue = False\n",
      "2023-06-02 17:58:21,066 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cm\"\tPrecision = 0.716 (global 0.612) Δm = 0.171\tis_issue = False\n",
      "2023-06-02 17:58:21,095 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"metzenbaum\"\tPrecision = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:21,131 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"place\"\tPrecision = 0.828 (global 0.612) Δm = 0.353\tis_issue = False\n",
      "2023-06-02 17:58:21,155 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"draped\"\tPrecision = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:21,186 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"use\"\tPrecision = 0.681 (global 0.612) Δm = 0.112\tis_issue = False\n",
      "2023-06-02 17:58:21,209 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extremity\"\tPrecision = 0.619 (global 0.612) Δm = 0.012\tis_issue = False\n",
      "2023-06-02 17:58:21,236 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"08\"\tPrecision = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:21,268 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"iv\"\tPrecision = 0.758 (global 0.612) Δm = 0.239\tis_issue = False\n",
      "2023-06-02 17:58:21,298 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oxygen\"\tPrecision = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:21,327 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscle\"\tPrecision = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:21,356 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"difficulty\"\tPrecision = 0.734 (global 0.612) Δm = 0.200\tis_issue = False\n",
      "2023-06-02 17:58:21,383 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"xyz\"\tPrecision = 0.318 (global 0.612) Δm = -0.480\tis_issue = True\n",
      "2023-06-02 17:58:21,413 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"evidence\"\tPrecision = 0.691 (global 0.612) Δm = 0.130\tis_issue = False\n",
      "2023-06-02 17:58:21,443 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suite\"\tPrecision = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:21,474 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"informed\"\tPrecision = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:21,507 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspirin\"\tPrecision = 0.435 (global 0.612) Δm = -0.289\tis_issue = True\n",
      "2023-06-02 17:58:21,534 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mild\"\tPrecision = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:21,568 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scope\"\tPrecision = 0.864 (global 0.612) Δm = 0.411\tis_issue = False\n",
      "2023-06-02 17:58:21,598 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"resection\"\tPrecision = 0.700 (global 0.612) Δm = 0.144\tis_issue = False\n",
      "2023-06-02 17:58:21,625 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearance\"\tPrecision = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:21,651 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"obtained\"\tPrecision = 0.737 (global 0.612) Δm = 0.205\tis_issue = False\n",
      "2023-06-02 17:58:21,677 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nose\"\tPrecision = 0.794 (global 0.612) Δm = 0.298\tis_issue = False\n",
      "2023-06-02 17:58:21,721 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colonoscopy\"\tPrecision = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:21,751 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extended\"\tPrecision = 0.929 (global 0.612) Δm = 0.518\tis_issue = False\n",
      "2023-06-02 17:58:21,782 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"similar\"\tPrecision = 0.724 (global 0.612) Δm = 0.184\tis_issue = False\n",
      "2023-06-02 17:58:21,804 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"examination\"\tPrecision = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:21,834 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denied\"\tPrecision = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:21,858 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"found\"\tPrecision = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:21,887 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"region\"\tPrecision = 0.724 (global 0.612) Δm = 0.184\tis_issue = False\n",
      "2023-06-02 17:58:21,910 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ml\"\tPrecision = 0.717 (global 0.612) Δm = 0.172\tis_issue = False\n",
      "2023-06-02 17:58:21,942 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"large\"\tPrecision = 0.647 (global 0.612) Δm = 0.058\tis_issue = False\n",
      "2023-06-02 17:58:21,967 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cc\"\tPrecision = 0.787 (global 0.612) Δm = 0.287\tis_issue = False\n",
      "2023-06-02 17:58:21,988 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"preoperative\"\tPrecision = 0.761 (global 0.612) Δm = 0.243\tis_issue = False\n",
      "2023-06-02 17:58:22,018 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"supine\"\tPrecision = 0.764 (global 0.612) Δm = 0.248\tis_issue = False\n",
      "2023-06-02 17:58:22,043 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knee\"\tPrecision = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:22,075 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"urinary\"\tPrecision = 0.486 (global 0.612) Δm = -0.206\tis_issue = True\n",
      "2023-06-02 17:58:22,104 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligament\"\tPrecision = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:22,133 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stomach\"\tPrecision = 0.850 (global 0.612) Δm = 0.389\tis_issue = False\n",
      "2023-06-02 17:58:22,163 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mri\"\tPrecision = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:22,192 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"small\"\tPrecision = 0.770 (global 0.612) Δm = 0.259\tis_issue = False\n",
      "2023-06-02 17:58:22,224 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"compression\"\tPrecision = 0.636 (global 0.612) Δm = 0.040\tis_issue = False\n",
      "2023-06-02 17:58:22,246 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"day\"\tPrecision = 0.481 (global 0.612) Δm = -0.214\tis_issue = True\n",
      "2023-06-02 17:58:22,278 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualized\"\tPrecision = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:22,304 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"entered\"\tPrecision = 0.875 (global 0.612) Δm = 0.430\tis_issue = False\n",
      "2023-06-02 17:58:22,360 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"external\"\tPrecision = 0.606 (global 0.612) Δm = -0.009\tis_issue = False\n",
      "2023-06-02 17:58:22,393 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bladder\"\tPrecision = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:22,423 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"running\"\tPrecision = 0.825 (global 0.612) Δm = 0.348\tis_issue = False\n",
      "2023-06-02 17:58:22,445 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"position\"\tPrecision = 0.794 (global 0.612) Δm = 0.298\tis_issue = False\n",
      "2023-06-02 17:58:22,477 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"continue\"\tPrecision = 0.442 (global 0.612) Δm = -0.277\tis_issue = True\n",
      "2023-06-02 17:58:22,505 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"laterally\"\tPrecision = 0.846 (global 0.612) Δm = 0.383\tis_issue = False\n",
      "2023-06-02 17:58:22,529 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ear\"\tPrecision = 0.619 (global 0.612) Δm = 0.012\tis_issue = False\n",
      "2023-06-02 17:58:22,562 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"easily\"\tPrecision = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:22,591 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"status\"\tPrecision = 0.515 (global 0.612) Δm = -0.158\tis_issue = True\n",
      "2023-06-02 17:58:22,618 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"home\"\tPrecision = 0.519 (global 0.612) Δm = -0.153\tis_issue = True\n",
      "2023-06-02 17:58:22,642 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nerve\"\tPrecision = 0.850 (global 0.612) Δm = 0.389\tis_issue = False\n",
      "2023-06-02 17:58:22,672 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"foley\"\tPrecision = 0.730 (global 0.612) Δm = 0.193\tis_issue = False\n",
      "2023-06-02 17:58:22,697 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cardiac\"\tPrecision = 0.529 (global 0.612) Δm = -0.135\tis_issue = True\n",
      "2023-06-02 17:58:22,728 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pupils\"\tPrecision = 0.706 (global 0.612) Δm = 0.154\tis_issue = False\n",
      "2023-06-02 17:58:22,756 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extending\"\tPrecision = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:22,785 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"counts\"\tPrecision = 0.892 (global 0.612) Δm = 0.458\tis_issue = False\n",
      "2023-06-02 17:58:22,807 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"estimated\"\tPrecision = 0.823 (global 0.612) Δm = 0.344\tis_issue = False\n",
      "2023-06-02 17:58:22,837 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gross\"\tPrecision = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:22,866 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spinal\"\tPrecision = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:22,893 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterus\"\tPrecision = 0.647 (global 0.612) Δm = 0.058\tis_issue = False\n",
      "2023-06-02 17:58:22,919 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appeared\"\tPrecision = 0.765 (global 0.612) Δm = 0.250\tis_issue = False\n",
      "2023-06-02 17:58:22,950 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removal\"\tPrecision = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:22,978 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sterile\"\tPrecision = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:23,003 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"able\"\tPrecision = 0.633 (global 0.612) Δm = 0.034\tis_issue = False\n",
      "2023-06-02 17:58:23,030 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abc\"\tPrecision = 0.417 (global 0.612) Δm = -0.319\tis_issue = True\n",
      "2023-06-02 17:58:23,053 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"exam\"\tPrecision = 0.663 (global 0.612) Δm = 0.084\tis_issue = False\n",
      "2023-06-02 17:58:23,082 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"renal\"\tPrecision = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:23,108 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hemostasis\"\tPrecision = 0.787 (global 0.612) Δm = 0.286\tis_issue = False\n",
      "2023-06-02 17:58:23,134 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"proximal\"\tPrecision = 0.889 (global 0.612) Δm = 0.453\tis_issue = False\n",
      "2023-06-02 17:58:23,171 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectal\"\tPrecision = 0.643 (global 0.612) Δm = 0.051\tis_issue = False\n",
      "2023-06-02 17:58:23,201 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"12\"\tPrecision = 0.541 (global 0.612) Δm = -0.117\tis_issue = True\n",
      "2023-06-02 17:58:23,232 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"attention\"\tPrecision = 0.786 (global 0.612) Δm = 0.284\tis_issue = False\n",
      "2023-06-02 17:58:23,268 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"activity\"\tPrecision = 0.514 (global 0.612) Δm = -0.161\tis_issue = True\n",
      "2023-06-02 17:58:23,293 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"father\"\tPrecision = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:23,320 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissection\"\tPrecision = 0.762 (global 0.612) Δm = 0.245\tis_issue = False\n",
      "2023-06-02 17:58:23,353 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brain\"\tPrecision = 0.645 (global 0.612) Δm = 0.054\tis_issue = False\n",
      "2023-06-02 17:58:23,378 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tubes\"\tPrecision = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:23,408 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bleeding\"\tPrecision = 0.727 (global 0.612) Δm = 0.189\tis_issue = False\n",
      "2023-06-02 17:58:23,434 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mouth\"\tPrecision = 0.700 (global 0.612) Δm = 0.144\tis_issue = False\n",
      "2023-06-02 17:58:23,461 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incision\"\tPrecision = 0.747 (global 0.612) Δm = 0.221\tis_issue = False\n",
      "2023-06-02 17:58:23,491 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wall\"\tPrecision = 0.763 (global 0.612) Δm = 0.247\tis_issue = False\n",
      "2023-06-02 17:58:23,523 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"initially\"\tPrecision = 0.517 (global 0.612) Δm = -0.155\tis_issue = True\n",
      "2023-06-02 17:58:23,548 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"family\"\tPrecision = 0.573 (global 0.612) Δm = -0.063\tis_issue = True\n",
      "2023-06-02 17:58:23,570 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removed\"\tPrecision = 0.790 (global 0.612) Δm = 0.291\tis_issue = False\n",
      "2023-06-02 17:58:23,603 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"treatment\"\tPrecision = 0.518 (global 0.612) Δm = -0.154\tis_issue = True\n",
      "2023-06-02 17:58:23,626 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nasal\"\tPrecision = 0.594 (global 0.612) Δm = -0.030\tis_issue = False\n",
      "2023-06-02 17:58:23,657 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fever\"\tPrecision = 0.558 (global 0.612) Δm = -0.089\tis_issue = True\n",
      "2023-06-02 17:58:23,680 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reports\"\tPrecision = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:23,710 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anterior\"\tPrecision = 0.756 (global 0.612) Δm = 0.236\tis_issue = False\n",
      "2023-06-02 17:58:23,734 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"leg\"\tPrecision = 0.567 (global 0.612) Δm = -0.074\tis_issue = True\n",
      "2023-06-02 17:58:23,767 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"changes\"\tPrecision = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:23,794 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"strength\"\tPrecision = 0.738 (global 0.612) Δm = 0.206\tis_issue = False\n",
      "2023-06-02 17:58:23,822 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fashion\"\tPrecision = 0.807 (global 0.612) Δm = 0.319\tis_issue = False\n",
      "2023-06-02 17:58:23,847 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fascia\"\tPrecision = 0.824 (global 0.612) Δm = 0.346\tis_issue = False\n",
      "2023-06-02 17:58:23,875 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lumbar\"\tPrecision = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:23,901 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"enlarged\"\tPrecision = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:23,926 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denies\"\tPrecision = 0.459 (global 0.612) Δm = -0.250\tis_issue = True\n",
      "2023-06-02 17:58:23,958 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"office\"\tPrecision = 0.476 (global 0.612) Δm = -0.222\tis_issue = True\n",
      "2023-06-02 17:58:23,984 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adenopathy\"\tPrecision = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:24,012 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abnormalities\"\tPrecision = 0.656 (global 0.612) Δm = 0.073\tis_issue = False\n",
      "2023-06-02 17:58:24,033 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"placed\"\tPrecision = 0.780 (global 0.612) Δm = 0.274\tis_issue = False\n",
      "2023-06-02 17:58:24,064 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"improved\"\tPrecision = 0.419 (global 0.612) Δm = -0.315\tis_issue = True\n",
      "2023-06-02 17:58:24,093 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mucosa\"\tPrecision = 0.697 (global 0.612) Δm = 0.139\tis_issue = False\n",
      "2023-06-02 17:58:24,124 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"return\"\tPrecision = 0.514 (global 0.612) Δm = -0.159\tis_issue = True\n",
      "2023-06-02 17:58:24,149 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"post\"\tPrecision = 0.552 (global 0.612) Δm = -0.098\tis_issue = True\n",
      "2023-06-02 17:58:24,178 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"noted\"\tPrecision = 0.651 (global 0.612) Δm = 0.064\tis_issue = False\n",
      "2023-06-02 17:58:24,206 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"closed\"\tPrecision = 0.806 (global 0.612) Δm = 0.317\tis_issue = False\n",
      "2023-06-02 17:58:24,236 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distal\"\tPrecision = 0.795 (global 0.612) Δm = 0.299\tis_issue = False\n",
      "2023-06-02 17:58:24,262 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"specimens\"\tPrecision = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:24,295 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sponge\"\tPrecision = 0.886 (global 0.612) Δm = 0.448\tis_issue = False\n",
      "2023-06-02 17:58:24,317 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clear\"\tPrecision = 0.515 (global 0.612) Δm = -0.158\tis_issue = True\n",
      "2023-06-02 17:58:24,345 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"room\"\tPrecision = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:24,371 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"weakness\"\tPrecision = 0.732 (global 0.612) Δm = 0.196\tis_issue = False\n",
      "2023-06-02 17:58:24,408 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"superior\"\tPrecision = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:24,432 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"neck\"\tPrecision = 0.553 (global 0.612) Δm = -0.096\tis_issue = True\n",
      "2023-06-02 17:58:24,462 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"infection\"\tPrecision = 0.607 (global 0.612) Δm = -0.009\tis_issue = False\n",
      "2023-06-02 17:58:24,493 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drainage\"\tPrecision = 0.630 (global 0.612) Δm = 0.029\tis_issue = False\n",
      "2023-06-02 17:58:24,522 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ct\"\tPrecision = 0.600 (global 0.612) Δm = -0.019\tis_issue = False\n",
      "2023-06-02 17:58:24,557 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"swelling\"\tPrecision = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:24,590 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mass\"\tPrecision = 0.596 (global 0.612) Δm = -0.026\tis_issue = False\n",
      "2023-06-02 17:58:24,628 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinic\"\tPrecision = 0.542 (global 0.612) Δm = -0.115\tis_issue = True\n",
      "2023-06-02 17:58:24,667 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pelvis\"\tPrecision = 0.733 (global 0.612) Δm = 0.199\tis_issue = False\n",
      "2023-06-02 17:58:24,702 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"taken\"\tPrecision = 0.713 (global 0.612) Δm = 0.165\tis_issue = False\n",
      "2023-06-02 17:58:24,744 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervix\"\tPrecision = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:24,779 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bovie\"\tPrecision = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:24,817 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinical\"\tPrecision = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:24,853 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"upper\"\tPrecision = 0.718 (global 0.612) Δm = 0.173\tis_issue = False\n",
      "2023-06-02 17:58:24,892 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"arm\"\tPrecision = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:24,927 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesions\"\tPrecision = 0.681 (global 0.612) Δm = 0.113\tis_issue = False\n",
      "2023-06-02 17:58:24,969 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tone\"\tPrecision = 0.759 (global 0.612) Δm = 0.240\tis_issue = False\n",
      "2023-06-02 17:58:25,010 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"single\"\tPrecision = 0.696 (global 0.612) Δm = 0.137\tis_issue = False\n",
      "2023-06-02 17:58:25,041 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drains\"\tPrecision = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:25,066 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"right\"\tPrecision = 0.659 (global 0.612) Δm = 0.077\tis_issue = False\n",
      "2023-06-02 17:58:25,105 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearing\"\tPrecision = 0.680 (global 0.612) Δm = 0.111\tis_issue = False\n",
      "2023-06-02 17:58:25,139 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"head\"\tPrecision = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:25,176 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reflexes\"\tPrecision = 0.775 (global 0.612) Δm = 0.267\tis_issue = False\n",
      "2023-06-02 17:58:25,206 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abdominal\"\tPrecision = 0.615 (global 0.612) Δm = 0.006\tis_issue = False\n",
      "2023-06-02 17:58:25,245 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"needle\"\tPrecision = 0.821 (global 0.612) Δm = 0.343\tis_issue = False\n",
      "2023-06-02 17:58:25,281 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"daily\"\tPrecision = 0.385 (global 0.612) Δm = -0.371\tis_issue = True\n",
      "2023-06-02 17:58:25,318 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthetic\"\tPrecision = 0.913 (global 0.612) Δm = 0.492\tis_issue = False\n",
      "2023-06-02 17:58:25,355 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesion\"\tPrecision = 0.692 (global 0.612) Δm = 0.131\tis_issue = False\n",
      "2023-06-02 17:58:25,389 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complications\"\tPrecision = 0.697 (global 0.612) Δm = 0.140\tis_issue = False\n",
      "2023-06-02 17:58:25,411 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"postoperative\"\tPrecision = 0.729 (global 0.612) Δm = 0.191\tis_issue = False\n",
      "2023-06-02 17:58:25,452 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"follow\"\tPrecision = 0.550 (global 0.612) Δm = -0.101\tis_issue = True\n",
      "2023-06-02 17:58:25,488 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visual\"\tPrecision = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:25,518 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"end\"\tPrecision = 0.757 (global 0.612) Δm = 0.237\tis_issue = False\n",
      "2023-06-02 17:58:25,555 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluids\"\tPrecision = 0.786 (global 0.612) Δm = 0.284\tis_issue = False\n",
      "2023-06-02 17:58:25,589 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"count\"\tPrecision = 0.400 (global 0.612) Δm = -0.346\tis_issue = True\n",
      "2023-06-02 17:58:25,617 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspect\"\tPrecision = 0.787 (global 0.612) Δm = 0.287\tis_issue = False\n",
      "2023-06-02 17:58:25,647 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"biopsy\"\tPrecision = 0.625 (global 0.612) Δm = 0.021\tis_issue = False\n",
      "2023-06-02 17:58:25,678 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"performed\"\tPrecision = 0.683 (global 0.612) Δm = 0.116\tis_issue = False\n",
      "2023-06-02 17:58:25,710 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mg\"\tPrecision = 0.473 (global 0.612) Δm = -0.227\tis_issue = True\n",
      "2023-06-02 17:58:25,750 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"open\"\tPrecision = 0.769 (global 0.612) Δm = 0.257\tis_issue = False\n",
      "2023-06-02 17:58:25,790 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"discharge\"\tPrecision = 0.424 (global 0.612) Δm = -0.308\tis_issue = True\n",
      "2023-06-02 17:58:25,825 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stool\"\tPrecision = 0.516 (global 0.612) Δm = -0.156\tis_issue = True\n",
      "2023-06-02 17:58:25,864 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carefully\"\tPrecision = 0.826 (global 0.612) Δm = 0.350\tis_issue = False\n",
      "2023-06-02 17:58:25,903 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"age\"\tPrecision = 0.639 (global 0.612) Δm = 0.044\tis_issue = False\n",
      "2023-06-02 17:58:25,935 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headaches\"\tPrecision = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:25,971 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"high\"\tPrecision = 0.488 (global 0.612) Δm = -0.203\tis_issue = True\n",
      "2023-06-02 17:58:26,009 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"erythema\"\tPrecision = 0.655 (global 0.612) Δm = 0.071\tis_issue = False\n",
      "2023-06-02 17:58:26,042 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthesia\"\tPrecision = 0.764 (global 0.612) Δm = 0.249\tis_issue = False\n",
      "2023-06-02 17:58:26,074 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"midline\"\tPrecision = 0.836 (global 0.612) Δm = 0.367\tis_issue = False\n",
      "2023-06-02 17:58:26,107 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"equal\"\tPrecision = 0.655 (global 0.612) Δm = 0.071\tis_issue = False\n",
      "2023-06-02 17:58:26,145 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"numbness\"\tPrecision = 0.815 (global 0.612) Δm = 0.332\tis_issue = False\n",
      "2023-06-02 17:58:26,188 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suction\"\tPrecision = 0.826 (global 0.612) Δm = 0.350\tis_issue = False\n",
      "2023-06-02 17:58:26,228 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pulmonary\"\tPrecision = 0.533 (global 0.612) Δm = -0.128\tis_issue = True\n",
      "2023-06-02 17:58:26,262 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"underlying\"\tPrecision = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:26,302 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cancer\"\tPrecision = 0.622 (global 0.612) Δm = 0.016\tis_issue = False\n",
      "2023-06-02 17:58:26,338 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scan\"\tPrecision = 0.568 (global 0.612) Δm = -0.071\tis_issue = True\n",
      "2023-06-02 17:58:26,357 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"patient\"\tPrecision = 0.597 (global 0.612) Δm = -0.023\tis_issue = False\n",
      "2023-06-02 17:58:26,391 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"passed\"\tPrecision = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:26,437 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"saline\"\tPrecision = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:26,471 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cautery\"\tPrecision = 0.741 (global 0.612) Δm = 0.211\tis_issue = False\n",
      "2023-06-02 17:58:26,502 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"body\"\tPrecision = 0.816 (global 0.612) Δm = 0.333\tis_issue = False\n",
      "2023-06-02 17:58:26,552 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervical\"\tPrecision = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:26,605 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prostate\"\tPrecision = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:26,657 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sleep\"\tPrecision = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:26,686 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"epinephrine\"\tPrecision = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:26,792 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"shoulder\"\tPrecision = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:26,822 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tenderness\"\tPrecision = 0.543 (global 0.612) Δm = -0.112\tis_issue = True\n",
      "2023-06-02 17:58:26,864 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stitch\"\tPrecision = 0.909 (global 0.612) Δm = 0.486\tis_issue = False\n",
      "2023-06-02 17:58:26,907 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sinus\"\tPrecision = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:26,937 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"point\"\tPrecision = 0.615 (global 0.612) Δm = 0.006\tis_issue = False\n",
      "2023-06-02 17:58:26,985 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"second\"\tPrecision = 0.679 (global 0.612) Δm = 0.110\tis_issue = False\n",
      "2023-06-02 17:58:27,019 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterine\"\tPrecision = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:27,057 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"breast\"\tPrecision = 0.444 (global 0.612) Δm = -0.274\tis_issue = True\n",
      "2023-06-02 17:58:27,089 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"having\"\tPrecision = 0.523 (global 0.612) Δm = -0.145\tis_issue = True\n",
      "2023-06-02 17:58:27,123 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"local\"\tPrecision = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:27,151 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"normal\"\tPrecision = 0.641 (global 0.612) Δm = 0.048\tis_issue = False\n",
      "2023-06-02 17:58:27,190 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stable\"\tPrecision = 0.627 (global 0.612) Δm = 0.025\tis_issue = False\n",
      "2023-06-02 17:58:27,231 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sutures\"\tPrecision = 0.636 (global 0.612) Δm = 0.040\tis_issue = False\n",
      "2023-06-02 17:58:27,269 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"admission\"\tPrecision = 0.375 (global 0.612) Δm = -0.387\tis_issue = True\n",
      "2023-06-02 17:58:27,306 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"round\"\tPrecision = 0.774 (global 0.612) Δm = 0.265\tis_issue = False\n",
      "2023-06-02 17:58:27,348 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"artery\"\tPrecision = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:27,381 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"problems\"\tPrecision = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:27,422 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recurrent\"\tPrecision = 0.654 (global 0.612) Δm = 0.069\tis_issue = False\n",
      "2023-06-02 17:58:27,453 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medication\"\tPrecision = 0.541 (global 0.612) Δm = -0.117\tis_issue = True\n",
      "2023-06-02 17:58:27,487 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sac\"\tPrecision = 0.913 (global 0.612) Δm = 0.492\tis_issue = False\n",
      "2023-06-02 17:58:27,533 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"s1\"\tPrecision = 0.556 (global 0.612) Δm = -0.092\tis_issue = True\n",
      "2023-06-02 17:58:27,573 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consistent\"\tPrecision = 0.828 (global 0.612) Δm = 0.353\tis_issue = False\n",
      "2023-06-02 17:58:27,606 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"diarrhea\"\tPrecision = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:27,641 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tolerated\"\tPrecision = 0.698 (global 0.612) Δm = 0.140\tis_issue = False\n",
      "2023-06-02 17:58:27,681 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headache\"\tPrecision = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:27,720 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scalpel\"\tPrecision = 0.957 (global 0.612) Δm = 0.563\tis_issue = False\n",
      "2023-06-02 17:58:27,759 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lymph\"\tPrecision = 0.769 (global 0.612) Δm = 0.257\tis_issue = False\n",
      "2023-06-02 17:58:27,795 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pregnancy\"\tPrecision = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:27,828 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complex\"\tPrecision = 0.522 (global 0.612) Δm = -0.147\tis_issue = True\n",
      "2023-06-02 17:58:27,867 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"care\"\tPrecision = 0.537 (global 0.612) Δm = -0.122\tis_issue = True\n",
      "2023-06-02 17:58:27,907 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"units\"\tPrecision = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:27,939 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hx\"\tPrecision = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:27,976 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscles\"\tPrecision = 0.815 (global 0.612) Δm = 0.332\tis_issue = False\n",
      "2023-06-02 17:58:28,021 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"irrigated\"\tPrecision = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:28,052 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consent\"\tPrecision = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:28,121 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reveals\"\tPrecision = 0.652 (global 0.612) Δm = 0.066\tis_issue = False\n",
      "2023-06-02 17:58:28,155 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reviewed\"\tPrecision = 0.529 (global 0.612) Δm = -0.135\tis_issue = True\n",
      "2023-06-02 17:58:28,187 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subsequently\"\tPrecision = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:28,212 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oropharynx\"\tPrecision = 0.686 (global 0.612) Δm = 0.121\tis_issue = False\n",
      "2023-06-02 17:58:28,227 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"history\"\tPrecision = 0.564 (global 0.612) Δm = -0.078\tis_issue = True\n",
      "2023-06-02 17:58:28,260 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectus\"\tPrecision = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:28,284 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prepped\"\tPrecision = 0.767 (global 0.612) Δm = 0.253\tis_issue = False\n",
      "2023-06-02 17:58:28,317 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suture\"\tPrecision = 0.804 (global 0.612) Δm = 0.315\tis_issue = False\n",
      "2023-06-02 17:58:28,343 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clamped\"\tPrecision = 0.960 (global 0.612) Δm = 0.569\tis_issue = False\n",
      "2023-06-02 17:58:28,379 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"function\"\tPrecision = 0.414 (global 0.612) Δm = -0.324\tis_issue = True\n",
      "2023-06-02 17:58:28,405 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"coronary\"\tPrecision = 0.391 (global 0.612) Δm = -0.360\tis_issue = True\n",
      "2023-06-02 17:58:28,437 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"speech\"\tPrecision = 0.704 (global 0.612) Δm = 0.150\tis_issue = False\n",
      "2023-06-02 17:58:28,467 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vaginal\"\tPrecision = 0.586 (global 0.612) Δm = -0.042\tis_issue = False\n",
      "2023-06-02 17:58:28,498 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subcuticular\"\tPrecision = 0.792 (global 0.612) Δm = 0.294\tis_issue = False\n",
      "2023-06-02 17:58:28,534 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cut\"\tPrecision = 0.917 (global 0.612) Δm = 0.498\tis_issue = False\n",
      "2023-06-02 17:58:28,561 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"correct\"\tPrecision = 0.881 (global 0.612) Δm = 0.440\tis_issue = False\n",
      "2023-06-02 17:58:28,589 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dorsal\"\tPrecision = 0.929 (global 0.612) Δm = 0.518\tis_issue = False\n",
      "2023-06-02 17:58:28,619 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"indications\"\tPrecision = 0.889 (global 0.612) Δm = 0.453\tis_issue = False\n",
      "2023-06-02 17:58:28,647 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"technique\"\tPrecision = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:28,674 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"findings\"\tPrecision = 0.691 (global 0.612) Δm = 0.129\tis_issue = False\n",
      "2023-06-02 17:58:28,699 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"greater\"\tPrecision = 0.435 (global 0.612) Δm = -0.289\tis_issue = True\n",
      "2023-06-02 17:58:28,728 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"presentation\"\tPrecision = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:28,756 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"internal\"\tPrecision = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:28,789 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligated\"\tPrecision = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:28,816 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissected\"\tPrecision = 0.881 (global 0.612) Δm = 0.440\tis_issue = False\n",
      "2023-06-02 17:58:28,850 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suctioned\"\tPrecision = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:28,874 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lung\"\tPrecision = 0.400 (global 0.612) Δm = -0.346\tis_issue = True\n",
      "2023-06-02 17:58:28,906 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"posterior\"\tPrecision = 0.768 (global 0.612) Δm = 0.255\tis_issue = False\n",
      "2023-06-02 17:58:28,935 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hours\"\tPrecision = 0.558 (global 0.612) Δm = -0.089\tis_issue = True\n",
      "2023-06-02 17:58:28,960 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vicryl\"\tPrecision = 0.791 (global 0.612) Δm = 0.293\tis_issue = False\n",
      "2023-06-02 17:58:28,990 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knife\"\tPrecision = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:29,016 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilaterally\"\tPrecision = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:29,043 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"administered\"\tPrecision = 0.767 (global 0.612) Δm = 0.253\tis_issue = False\n",
      "2023-06-02 17:58:29,077 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"grasped\"\tPrecision = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:29,104 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subjective\"\tPrecision = 0.367 (global 0.612) Δm = -0.401\tis_issue = True\n",
      "2023-06-02 17:58:29,136 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carotid\"\tPrecision = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:29,163 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"impression\"\tPrecision = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:29,195 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medial\"\tPrecision = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:29,215 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"left\"\tPrecision = 0.693 (global 0.612) Δm = 0.132\tis_issue = False\n",
      "2023-06-02 17:58:29,247 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mm\"\tPrecision = 0.827 (global 0.612) Δm = 0.351\tis_issue = False\n",
      "2023-06-02 17:58:29,273 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"condition\"\tPrecision = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:29,303 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gait\"\tPrecision = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:29,333 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inferior\"\tPrecision = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:29,367 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"masses\"\tPrecision = 0.605 (global 0.612) Δm = -0.011\tis_issue = False\n",
      "2023-06-02 17:58:29,391 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectum\"\tPrecision = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:29,423 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brought\"\tPrecision = 0.747 (global 0.612) Δm = 0.221\tis_issue = False\n",
      "2023-06-02 17:58:29,455 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"delivered\"\tPrecision = 0.920 (global 0.612) Δm = 0.504\tis_issue = False\n",
      "2023-06-02 17:58:29,478 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gentleman\"\tPrecision = 0.696 (global 0.612) Δm = 0.137\tis_issue = False\n",
      "2023-06-02 17:58:29,510 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluid\"\tPrecision = 0.759 (global 0.612) Δm = 0.241\tis_issue = False\n",
      "2023-06-02 17:58:29,538 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"change\"\tPrecision = 0.725 (global 0.612) Δm = 0.185\tis_issue = False\n",
      "2023-06-02 17:58:29,569 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"light\"\tPrecision = 0.674 (global 0.612) Δm = 0.102\tis_issue = False\n",
      "2023-06-02 17:58:29,599 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tube\"\tPrecision = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:29,619 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"loss\"\tPrecision = 0.781 (global 0.612) Δm = 0.276\tis_issue = False\n",
      "2023-06-02 17:58:29,649 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lateral\"\tPrecision = 0.797 (global 0.612) Δm = 0.303\tis_issue = False\n",
      "2023-06-02 17:58:29,680 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sensory\"\tPrecision = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:29,710 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vision\"\tPrecision = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:29,738 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incised\"\tPrecision = 0.900 (global 0.612) Δm = 0.471\tis_issue = False\n",
      "2023-06-02 17:58:29,769 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visit\"\tPrecision = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:29,798 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tract\"\tPrecision = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:29,818 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"procedure\"\tPrecision = 0.726 (global 0.612) Δm = 0.187\tis_issue = False\n",
      "2023-06-02 17:58:29,849 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"advanced\"\tPrecision = 0.759 (global 0.612) Δm = 0.240\tis_issue = False\n",
      "2023-06-02 17:58:29,875 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cavity\"\tPrecision = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:29,908 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"flap\"\tPrecision = 0.727 (global 0.612) Δm = 0.189\tis_issue = False\n",
      "2023-06-02 17:58:29,934 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"negative\"\tPrecision = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:29,964 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"direct\"\tPrecision = 0.731 (global 0.612) Δm = 0.194\tis_issue = False\n",
      "2023-06-02 17:58:29,994 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"months\"\tPrecision = 0.562 (global 0.612) Δm = -0.081\tis_issue = True\n",
      "2023-06-02 17:58:30,020 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"states\"\tPrecision = 0.535 (global 0.612) Δm = -0.126\tis_issue = True\n",
      "2023-06-02 17:58:30,044 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"following\"\tPrecision = 0.609 (global 0.612) Δm = -0.004\tis_issue = False\n",
      "2023-06-02 17:58:30,077 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recovery\"\tPrecision = 0.843 (global 0.612) Δm = 0.378\tis_issue = False\n",
      "2023-06-02 17:58:35,285 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `language(transcription)` == \"en\"\tRecall = 0.614 (global 0.612) Δm = 0.003\tis_issue = False\n",
      "2023-06-02 17:58:35,309 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"section\"\tRecall = 0.692 (global 0.612) Δm = 0.131\tis_issue = False\n",
      "2023-06-02 17:58:35,334 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"contrast\"\tRecall = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:35,359 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spine\"\tRecall = 0.703 (global 0.612) Δm = 0.148\tis_issue = False\n",
      "2023-06-02 17:58:35,401 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inserted\"\tRecall = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:35,429 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"length\"\tRecall = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:35,462 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adequate\"\tRecall = 0.816 (global 0.612) Δm = 0.333\tis_issue = False\n",
      "2023-06-02 17:58:35,494 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"identified\"\tRecall = 0.785 (global 0.612) Δm = 0.282\tis_issue = False\n",
      "2023-06-02 17:58:35,522 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scissors\"\tRecall = 0.852 (global 0.612) Δm = 0.392\tis_issue = False\n",
      "2023-06-02 17:58:35,554 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colon\"\tRecall = 0.690 (global 0.612) Δm = 0.127\tis_issue = False\n",
      "2023-06-02 17:58:35,583 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distress\"\tRecall = 0.486 (global 0.612) Δm = -0.206\tis_issue = True\n",
      "2023-06-02 17:58:35,609 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"motor\"\tRecall = 0.795 (global 0.612) Δm = 0.300\tis_issue = False\n",
      "2023-06-02 17:58:35,636 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pain\"\tRecall = 0.571 (global 0.612) Δm = -0.066\tis_issue = True\n",
      "2023-06-02 17:58:35,662 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"chromic\"\tRecall = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:35,694 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"transverse\"\tRecall = 0.812 (global 0.612) Δm = 0.328\tis_issue = False\n",
      "2023-06-02 17:58:35,726 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"revealed\"\tRecall = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:35,762 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tumor\"\tRecall = 0.583 (global 0.612) Δm = -0.047\tis_issue = False\n",
      "2023-06-02 17:58:35,800 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"areas\"\tRecall = 0.607 (global 0.612) Δm = -0.008\tis_issue = False\n",
      "2023-06-02 17:58:35,823 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"unremarkable\"\tRecall = 0.632 (global 0.612) Δm = 0.033\tis_issue = False\n",
      "2023-06-02 17:58:35,854 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilateral\"\tRecall = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:35,878 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wound\"\tRecall = 0.638 (global 0.612) Δm = 0.043\tis_issue = False\n",
      "2023-06-02 17:58:35,911 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"followup\"\tRecall = 0.396 (global 0.612) Δm = -0.353\tis_issue = True\n",
      "2023-06-02 17:58:35,937 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"20\"\tRecall = 0.598 (global 0.612) Δm = -0.023\tis_issue = False\n",
      "2023-06-02 17:58:35,964 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"membranes\"\tRecall = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:35,998 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"created\"\tRecall = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:36,027 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"portion\"\tRecall = 0.964 (global 0.612) Δm = 0.576\tis_issue = False\n",
      "2023-06-02 17:58:36,051 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"check\"\tRecall = 0.560 (global 0.612) Δm = -0.085\tis_issue = True\n",
      "2023-06-02 17:58:36,077 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"days\"\tRecall = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:36,104 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"symptoms\"\tRecall = 0.551 (global 0.612) Δm = -0.099\tis_issue = True\n",
      "2023-06-02 17:58:36,129 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cord\"\tRecall = 0.939 (global 0.612) Δm = 0.535\tis_issue = False\n",
      "2023-06-02 17:58:36,165 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"skin\"\tRecall = 0.723 (global 0.612) Δm = 0.182\tis_issue = False\n",
      "2023-06-02 17:58:36,190 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"description\"\tRecall = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:36,226 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualization\"\tRecall = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:36,251 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rule\"\tRecall = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:36,285 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"blade\"\tRecall = 0.900 (global 0.612) Δm = 0.471\tis_issue = False\n",
      "2023-06-02 17:58:36,308 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cm\"\tRecall = 0.716 (global 0.612) Δm = 0.171\tis_issue = False\n",
      "2023-06-02 17:58:36,336 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"metzenbaum\"\tRecall = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:36,364 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"place\"\tRecall = 0.828 (global 0.612) Δm = 0.353\tis_issue = False\n",
      "2023-06-02 17:58:36,386 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"draped\"\tRecall = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:36,419 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"use\"\tRecall = 0.681 (global 0.612) Δm = 0.112\tis_issue = False\n",
      "2023-06-02 17:58:36,444 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extremity\"\tRecall = 0.619 (global 0.612) Δm = 0.012\tis_issue = False\n",
      "2023-06-02 17:58:36,472 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"08\"\tRecall = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:36,503 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"iv\"\tRecall = 0.758 (global 0.612) Δm = 0.239\tis_issue = False\n",
      "2023-06-02 17:58:36,531 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oxygen\"\tRecall = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:36,563 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscle\"\tRecall = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:36,596 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"difficulty\"\tRecall = 0.734 (global 0.612) Δm = 0.200\tis_issue = False\n",
      "2023-06-02 17:58:36,620 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"xyz\"\tRecall = 0.318 (global 0.612) Δm = -0.480\tis_issue = True\n",
      "2023-06-02 17:58:36,645 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"evidence\"\tRecall = 0.691 (global 0.612) Δm = 0.130\tis_issue = False\n",
      "2023-06-02 17:58:36,676 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suite\"\tRecall = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:36,703 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"informed\"\tRecall = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:36,735 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspirin\"\tRecall = 0.435 (global 0.612) Δm = -0.289\tis_issue = True\n",
      "2023-06-02 17:58:36,758 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mild\"\tRecall = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:36,820 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scope\"\tRecall = 0.864 (global 0.612) Δm = 0.411\tis_issue = False\n",
      "2023-06-02 17:58:36,850 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"resection\"\tRecall = 0.700 (global 0.612) Δm = 0.144\tis_issue = False\n",
      "2023-06-02 17:58:36,876 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearance\"\tRecall = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:36,908 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"obtained\"\tRecall = 0.737 (global 0.612) Δm = 0.205\tis_issue = False\n",
      "2023-06-02 17:58:36,936 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nose\"\tRecall = 0.794 (global 0.612) Δm = 0.298\tis_issue = False\n",
      "2023-06-02 17:58:36,965 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colonoscopy\"\tRecall = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:36,994 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extended\"\tRecall = 0.929 (global 0.612) Δm = 0.518\tis_issue = False\n",
      "2023-06-02 17:58:37,023 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"similar\"\tRecall = 0.724 (global 0.612) Δm = 0.184\tis_issue = False\n",
      "2023-06-02 17:58:37,051 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"examination\"\tRecall = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:37,078 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denied\"\tRecall = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:37,105 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"found\"\tRecall = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:37,138 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"region\"\tRecall = 0.724 (global 0.612) Δm = 0.184\tis_issue = False\n",
      "2023-06-02 17:58:37,165 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ml\"\tRecall = 0.717 (global 0.612) Δm = 0.172\tis_issue = False\n",
      "2023-06-02 17:58:37,190 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"large\"\tRecall = 0.647 (global 0.612) Δm = 0.058\tis_issue = False\n",
      "2023-06-02 17:58:37,224 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cc\"\tRecall = 0.787 (global 0.612) Δm = 0.287\tis_issue = False\n",
      "2023-06-02 17:58:37,250 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"preoperative\"\tRecall = 0.761 (global 0.612) Δm = 0.243\tis_issue = False\n",
      "2023-06-02 17:58:37,275 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"supine\"\tRecall = 0.764 (global 0.612) Δm = 0.248\tis_issue = False\n",
      "2023-06-02 17:58:37,308 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knee\"\tRecall = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:37,333 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"urinary\"\tRecall = 0.486 (global 0.612) Δm = -0.206\tis_issue = True\n",
      "2023-06-02 17:58:37,362 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligament\"\tRecall = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:37,392 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stomach\"\tRecall = 0.850 (global 0.612) Δm = 0.389\tis_issue = False\n",
      "2023-06-02 17:58:37,420 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mri\"\tRecall = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:37,454 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"small\"\tRecall = 0.770 (global 0.612) Δm = 0.259\tis_issue = False\n",
      "2023-06-02 17:58:37,479 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"compression\"\tRecall = 0.636 (global 0.612) Δm = 0.040\tis_issue = False\n",
      "2023-06-02 17:58:37,501 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"day\"\tRecall = 0.481 (global 0.612) Δm = -0.214\tis_issue = True\n",
      "2023-06-02 17:58:37,535 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualized\"\tRecall = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:37,561 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"entered\"\tRecall = 0.875 (global 0.612) Δm = 0.430\tis_issue = False\n",
      "2023-06-02 17:58:37,589 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"external\"\tRecall = 0.606 (global 0.612) Δm = -0.009\tis_issue = False\n",
      "2023-06-02 17:58:37,616 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bladder\"\tRecall = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:37,642 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"running\"\tRecall = 0.825 (global 0.612) Δm = 0.348\tis_issue = False\n",
      "2023-06-02 17:58:37,667 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"position\"\tRecall = 0.794 (global 0.612) Δm = 0.298\tis_issue = False\n",
      "2023-06-02 17:58:37,697 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"continue\"\tRecall = 0.442 (global 0.612) Δm = -0.277\tis_issue = True\n",
      "2023-06-02 17:58:37,722 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"laterally\"\tRecall = 0.846 (global 0.612) Δm = 0.383\tis_issue = False\n",
      "2023-06-02 17:58:37,751 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ear\"\tRecall = 0.619 (global 0.612) Δm = 0.012\tis_issue = False\n",
      "2023-06-02 17:58:37,780 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"easily\"\tRecall = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:37,809 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"status\"\tRecall = 0.515 (global 0.612) Δm = -0.158\tis_issue = True\n",
      "2023-06-02 17:58:37,846 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"home\"\tRecall = 0.519 (global 0.612) Δm = -0.153\tis_issue = True\n",
      "2023-06-02 17:58:37,871 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nerve\"\tRecall = 0.850 (global 0.612) Δm = 0.389\tis_issue = False\n",
      "2023-06-02 17:58:37,901 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"foley\"\tRecall = 0.730 (global 0.612) Δm = 0.193\tis_issue = False\n",
      "2023-06-02 17:58:37,931 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cardiac\"\tRecall = 0.529 (global 0.612) Δm = -0.135\tis_issue = True\n",
      "2023-06-02 17:58:37,957 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pupils\"\tRecall = 0.706 (global 0.612) Δm = 0.154\tis_issue = False\n",
      "2023-06-02 17:58:37,988 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extending\"\tRecall = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:38,016 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"counts\"\tRecall = 0.892 (global 0.612) Δm = 0.458\tis_issue = False\n",
      "2023-06-02 17:58:38,042 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"estimated\"\tRecall = 0.823 (global 0.612) Δm = 0.344\tis_issue = False\n",
      "2023-06-02 17:58:38,065 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gross\"\tRecall = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:38,098 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spinal\"\tRecall = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:38,125 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterus\"\tRecall = 0.647 (global 0.612) Δm = 0.058\tis_issue = False\n",
      "2023-06-02 17:58:38,154 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appeared\"\tRecall = 0.765 (global 0.612) Δm = 0.250\tis_issue = False\n",
      "2023-06-02 17:58:38,183 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removal\"\tRecall = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:38,211 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sterile\"\tRecall = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:38,239 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"able\"\tRecall = 0.633 (global 0.612) Δm = 0.034\tis_issue = False\n",
      "2023-06-02 17:58:38,268 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abc\"\tRecall = 0.417 (global 0.612) Δm = -0.319\tis_issue = True\n",
      "2023-06-02 17:58:38,291 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"exam\"\tRecall = 0.663 (global 0.612) Δm = 0.084\tis_issue = False\n",
      "2023-06-02 17:58:38,323 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"renal\"\tRecall = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:38,350 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hemostasis\"\tRecall = 0.787 (global 0.612) Δm = 0.286\tis_issue = False\n",
      "2023-06-02 17:58:38,377 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"proximal\"\tRecall = 0.889 (global 0.612) Δm = 0.453\tis_issue = False\n",
      "2023-06-02 17:58:38,406 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectal\"\tRecall = 0.643 (global 0.612) Δm = 0.051\tis_issue = False\n",
      "2023-06-02 17:58:38,428 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"12\"\tRecall = 0.541 (global 0.612) Δm = -0.117\tis_issue = True\n",
      "2023-06-02 17:58:38,451 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"attention\"\tRecall = 0.786 (global 0.612) Δm = 0.284\tis_issue = False\n",
      "2023-06-02 17:58:38,482 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"activity\"\tRecall = 0.514 (global 0.612) Δm = -0.161\tis_issue = True\n",
      "2023-06-02 17:58:38,510 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"father\"\tRecall = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:38,539 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissection\"\tRecall = 0.762 (global 0.612) Δm = 0.245\tis_issue = False\n",
      "2023-06-02 17:58:38,566 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brain\"\tRecall = 0.645 (global 0.612) Δm = 0.054\tis_issue = False\n",
      "2023-06-02 17:58:38,597 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tubes\"\tRecall = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:38,628 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bleeding\"\tRecall = 0.727 (global 0.612) Δm = 0.189\tis_issue = False\n",
      "2023-06-02 17:58:38,655 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mouth\"\tRecall = 0.700 (global 0.612) Δm = 0.144\tis_issue = False\n",
      "2023-06-02 17:58:38,684 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incision\"\tRecall = 0.747 (global 0.612) Δm = 0.221\tis_issue = False\n",
      "2023-06-02 17:58:38,710 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wall\"\tRecall = 0.763 (global 0.612) Δm = 0.247\tis_issue = False\n",
      "2023-06-02 17:58:38,739 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"initially\"\tRecall = 0.517 (global 0.612) Δm = -0.155\tis_issue = True\n",
      "2023-06-02 17:58:38,764 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"family\"\tRecall = 0.573 (global 0.612) Δm = -0.063\tis_issue = True\n",
      "2023-06-02 17:58:38,793 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removed\"\tRecall = 0.790 (global 0.612) Δm = 0.291\tis_issue = False\n",
      "2023-06-02 17:58:38,819 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"treatment\"\tRecall = 0.518 (global 0.612) Δm = -0.154\tis_issue = True\n",
      "2023-06-02 17:58:38,850 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nasal\"\tRecall = 0.594 (global 0.612) Δm = -0.030\tis_issue = False\n",
      "2023-06-02 17:58:38,878 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fever\"\tRecall = 0.558 (global 0.612) Δm = -0.089\tis_issue = True\n",
      "2023-06-02 17:58:38,902 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reports\"\tRecall = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:38,932 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anterior\"\tRecall = 0.756 (global 0.612) Δm = 0.236\tis_issue = False\n",
      "2023-06-02 17:58:38,959 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"leg\"\tRecall = 0.567 (global 0.612) Δm = -0.074\tis_issue = True\n",
      "2023-06-02 17:58:38,985 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"changes\"\tRecall = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:39,020 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"strength\"\tRecall = 0.738 (global 0.612) Δm = 0.206\tis_issue = False\n",
      "2023-06-02 17:58:39,045 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fashion\"\tRecall = 0.807 (global 0.612) Δm = 0.319\tis_issue = False\n",
      "2023-06-02 17:58:39,072 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fascia\"\tRecall = 0.824 (global 0.612) Δm = 0.346\tis_issue = False\n",
      "2023-06-02 17:58:39,098 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lumbar\"\tRecall = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:39,125 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"enlarged\"\tRecall = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:39,151 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denies\"\tRecall = 0.459 (global 0.612) Δm = -0.250\tis_issue = True\n",
      "2023-06-02 17:58:39,177 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"office\"\tRecall = 0.476 (global 0.612) Δm = -0.222\tis_issue = True\n",
      "2023-06-02 17:58:39,210 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adenopathy\"\tRecall = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:39,238 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abnormalities\"\tRecall = 0.656 (global 0.612) Δm = 0.073\tis_issue = False\n",
      "2023-06-02 17:58:39,259 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"placed\"\tRecall = 0.780 (global 0.612) Δm = 0.274\tis_issue = False\n",
      "2023-06-02 17:58:39,292 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"improved\"\tRecall = 0.419 (global 0.612) Δm = -0.315\tis_issue = True\n",
      "2023-06-02 17:58:39,315 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mucosa\"\tRecall = 0.697 (global 0.612) Δm = 0.139\tis_issue = False\n",
      "2023-06-02 17:58:39,346 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"return\"\tRecall = 0.514 (global 0.612) Δm = -0.159\tis_issue = True\n",
      "2023-06-02 17:58:39,373 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"post\"\tRecall = 0.552 (global 0.612) Δm = -0.098\tis_issue = True\n",
      "2023-06-02 17:58:39,394 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"noted\"\tRecall = 0.651 (global 0.612) Δm = 0.064\tis_issue = False\n",
      "2023-06-02 17:58:39,423 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"closed\"\tRecall = 0.806 (global 0.612) Δm = 0.317\tis_issue = False\n",
      "2023-06-02 17:58:39,451 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distal\"\tRecall = 0.795 (global 0.612) Δm = 0.299\tis_issue = False\n",
      "2023-06-02 17:58:39,479 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"specimens\"\tRecall = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:39,514 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sponge\"\tRecall = 0.886 (global 0.612) Δm = 0.448\tis_issue = False\n",
      "2023-06-02 17:58:39,538 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clear\"\tRecall = 0.515 (global 0.612) Δm = -0.158\tis_issue = True\n",
      "2023-06-02 17:58:39,559 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"room\"\tRecall = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:39,582 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"weakness\"\tRecall = 0.732 (global 0.612) Δm = 0.196\tis_issue = False\n",
      "2023-06-02 17:58:39,620 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"superior\"\tRecall = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:39,643 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"neck\"\tRecall = 0.553 (global 0.612) Δm = -0.096\tis_issue = True\n",
      "2023-06-02 17:58:39,679 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"infection\"\tRecall = 0.607 (global 0.612) Δm = -0.009\tis_issue = False\n",
      "2023-06-02 17:58:39,716 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drainage\"\tRecall = 0.630 (global 0.612) Δm = 0.029\tis_issue = False\n",
      "2023-06-02 17:58:39,751 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ct\"\tRecall = 0.600 (global 0.612) Δm = -0.019\tis_issue = False\n",
      "2023-06-02 17:58:39,792 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"swelling\"\tRecall = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:39,829 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mass\"\tRecall = 0.596 (global 0.612) Δm = -0.026\tis_issue = False\n",
      "2023-06-02 17:58:39,860 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinic\"\tRecall = 0.542 (global 0.612) Δm = -0.115\tis_issue = True\n",
      "2023-06-02 17:58:39,897 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pelvis\"\tRecall = 0.733 (global 0.612) Δm = 0.199\tis_issue = False\n",
      "2023-06-02 17:58:39,928 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"taken\"\tRecall = 0.713 (global 0.612) Δm = 0.165\tis_issue = False\n",
      "2023-06-02 17:58:39,961 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervix\"\tRecall = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:39,998 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bovie\"\tRecall = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:40,036 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinical\"\tRecall = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:40,069 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"upper\"\tRecall = 0.718 (global 0.612) Δm = 0.173\tis_issue = False\n",
      "2023-06-02 17:58:40,103 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"arm\"\tRecall = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:40,142 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesions\"\tRecall = 0.681 (global 0.612) Δm = 0.113\tis_issue = False\n",
      "2023-06-02 17:58:40,183 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tone\"\tRecall = 0.759 (global 0.612) Δm = 0.240\tis_issue = False\n",
      "2023-06-02 17:58:40,220 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"single\"\tRecall = 0.696 (global 0.612) Δm = 0.137\tis_issue = False\n",
      "2023-06-02 17:58:40,255 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drains\"\tRecall = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:40,277 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"right\"\tRecall = 0.659 (global 0.612) Δm = 0.077\tis_issue = False\n",
      "2023-06-02 17:58:40,311 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearing\"\tRecall = 0.680 (global 0.612) Δm = 0.111\tis_issue = False\n",
      "2023-06-02 17:58:40,349 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"head\"\tRecall = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:40,381 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reflexes\"\tRecall = 0.775 (global 0.612) Δm = 0.267\tis_issue = False\n",
      "2023-06-02 17:58:40,417 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abdominal\"\tRecall = 0.615 (global 0.612) Δm = 0.006\tis_issue = False\n",
      "2023-06-02 17:58:40,454 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"needle\"\tRecall = 0.821 (global 0.612) Δm = 0.343\tis_issue = False\n",
      "2023-06-02 17:58:40,487 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"daily\"\tRecall = 0.385 (global 0.612) Δm = -0.371\tis_issue = True\n",
      "2023-06-02 17:58:40,526 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthetic\"\tRecall = 0.913 (global 0.612) Δm = 0.492\tis_issue = False\n",
      "2023-06-02 17:58:40,590 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesion\"\tRecall = 0.692 (global 0.612) Δm = 0.131\tis_issue = False\n",
      "2023-06-02 17:58:40,625 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complications\"\tRecall = 0.697 (global 0.612) Δm = 0.140\tis_issue = False\n",
      "2023-06-02 17:58:40,653 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"postoperative\"\tRecall = 0.729 (global 0.612) Δm = 0.191\tis_issue = False\n",
      "2023-06-02 17:58:40,686 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"follow\"\tRecall = 0.550 (global 0.612) Δm = -0.101\tis_issue = True\n",
      "2023-06-02 17:58:40,714 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visual\"\tRecall = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:40,754 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"end\"\tRecall = 0.757 (global 0.612) Δm = 0.237\tis_issue = False\n",
      "2023-06-02 17:58:40,794 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluids\"\tRecall = 0.786 (global 0.612) Δm = 0.284\tis_issue = False\n",
      "2023-06-02 17:58:40,834 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"count\"\tRecall = 0.400 (global 0.612) Δm = -0.346\tis_issue = True\n",
      "2023-06-02 17:58:40,869 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspect\"\tRecall = 0.787 (global 0.612) Δm = 0.287\tis_issue = False\n",
      "2023-06-02 17:58:40,903 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"biopsy\"\tRecall = 0.625 (global 0.612) Δm = 0.021\tis_issue = False\n",
      "2023-06-02 17:58:40,934 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"performed\"\tRecall = 0.683 (global 0.612) Δm = 0.116\tis_issue = False\n",
      "2023-06-02 17:58:40,970 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mg\"\tRecall = 0.473 (global 0.612) Δm = -0.227\tis_issue = True\n",
      "2023-06-02 17:58:41,011 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"open\"\tRecall = 0.769 (global 0.612) Δm = 0.257\tis_issue = False\n",
      "2023-06-02 17:58:41,048 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"discharge\"\tRecall = 0.424 (global 0.612) Δm = -0.308\tis_issue = True\n",
      "2023-06-02 17:58:41,094 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stool\"\tRecall = 0.516 (global 0.612) Δm = -0.156\tis_issue = True\n",
      "2023-06-02 17:58:41,122 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carefully\"\tRecall = 0.826 (global 0.612) Δm = 0.350\tis_issue = False\n",
      "2023-06-02 17:58:41,149 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"age\"\tRecall = 0.639 (global 0.612) Δm = 0.044\tis_issue = False\n",
      "2023-06-02 17:58:41,187 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headaches\"\tRecall = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:41,221 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"high\"\tRecall = 0.488 (global 0.612) Δm = -0.203\tis_issue = True\n",
      "2023-06-02 17:58:41,257 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"erythema\"\tRecall = 0.655 (global 0.612) Δm = 0.071\tis_issue = False\n",
      "2023-06-02 17:58:41,289 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthesia\"\tRecall = 0.764 (global 0.612) Δm = 0.249\tis_issue = False\n",
      "2023-06-02 17:58:41,318 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"midline\"\tRecall = 0.836 (global 0.612) Δm = 0.367\tis_issue = False\n",
      "2023-06-02 17:58:41,358 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"equal\"\tRecall = 0.655 (global 0.612) Δm = 0.071\tis_issue = False\n",
      "2023-06-02 17:58:41,392 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"numbness\"\tRecall = 0.815 (global 0.612) Δm = 0.332\tis_issue = False\n",
      "2023-06-02 17:58:41,426 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suction\"\tRecall = 0.826 (global 0.612) Δm = 0.350\tis_issue = False\n",
      "2023-06-02 17:58:41,471 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pulmonary\"\tRecall = 0.533 (global 0.612) Δm = -0.128\tis_issue = True\n",
      "2023-06-02 17:58:41,507 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"underlying\"\tRecall = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:41,541 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cancer\"\tRecall = 0.622 (global 0.612) Δm = 0.016\tis_issue = False\n",
      "2023-06-02 17:58:41,584 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scan\"\tRecall = 0.568 (global 0.612) Δm = -0.071\tis_issue = True\n",
      "2023-06-02 17:58:41,603 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"patient\"\tRecall = 0.597 (global 0.612) Δm = -0.023\tis_issue = False\n",
      "2023-06-02 17:58:41,635 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"passed\"\tRecall = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:41,680 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"saline\"\tRecall = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:41,710 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cautery\"\tRecall = 0.741 (global 0.612) Δm = 0.211\tis_issue = False\n",
      "2023-06-02 17:58:41,745 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"body\"\tRecall = 0.816 (global 0.612) Δm = 0.333\tis_issue = False\n",
      "2023-06-02 17:58:41,780 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervical\"\tRecall = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:41,868 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prostate\"\tRecall = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:41,911 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sleep\"\tRecall = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:41,955 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"epinephrine\"\tRecall = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:41,993 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"shoulder\"\tRecall = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:42,032 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tenderness\"\tRecall = 0.543 (global 0.612) Δm = -0.112\tis_issue = True\n",
      "2023-06-02 17:58:42,073 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stitch\"\tRecall = 0.909 (global 0.612) Δm = 0.486\tis_issue = False\n",
      "2023-06-02 17:58:42,107 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sinus\"\tRecall = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:42,150 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"point\"\tRecall = 0.615 (global 0.612) Δm = 0.006\tis_issue = False\n",
      "2023-06-02 17:58:42,185 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"second\"\tRecall = 0.679 (global 0.612) Δm = 0.110\tis_issue = False\n",
      "2023-06-02 17:58:42,224 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterine\"\tRecall = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:42,259 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"breast\"\tRecall = 0.444 (global 0.612) Δm = -0.274\tis_issue = True\n",
      "2023-06-02 17:58:42,291 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"having\"\tRecall = 0.523 (global 0.612) Δm = -0.145\tis_issue = True\n",
      "2023-06-02 17:58:42,334 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"local\"\tRecall = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:42,360 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"normal\"\tRecall = 0.641 (global 0.612) Δm = 0.048\tis_issue = False\n",
      "2023-06-02 17:58:42,396 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stable\"\tRecall = 0.627 (global 0.612) Δm = 0.025\tis_issue = False\n",
      "2023-06-02 17:58:42,440 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sutures\"\tRecall = 0.636 (global 0.612) Δm = 0.040\tis_issue = False\n",
      "2023-06-02 17:58:42,471 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"admission\"\tRecall = 0.375 (global 0.612) Δm = -0.387\tis_issue = True\n",
      "2023-06-02 17:58:42,507 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"round\"\tRecall = 0.774 (global 0.612) Δm = 0.265\tis_issue = False\n",
      "2023-06-02 17:58:42,544 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"artery\"\tRecall = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:42,576 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"problems\"\tRecall = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:42,612 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recurrent\"\tRecall = 0.654 (global 0.612) Δm = 0.069\tis_issue = False\n",
      "2023-06-02 17:58:42,653 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medication\"\tRecall = 0.541 (global 0.612) Δm = -0.117\tis_issue = True\n",
      "2023-06-02 17:58:42,686 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sac\"\tRecall = 0.913 (global 0.612) Δm = 0.492\tis_issue = False\n",
      "2023-06-02 17:58:42,725 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"s1\"\tRecall = 0.556 (global 0.612) Δm = -0.092\tis_issue = True\n",
      "2023-06-02 17:58:42,761 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consistent\"\tRecall = 0.828 (global 0.612) Δm = 0.353\tis_issue = False\n",
      "2023-06-02 17:58:42,793 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"diarrhea\"\tRecall = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:42,834 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tolerated\"\tRecall = 0.698 (global 0.612) Δm = 0.140\tis_issue = False\n",
      "2023-06-02 17:58:42,869 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headache\"\tRecall = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:42,913 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scalpel\"\tRecall = 0.957 (global 0.612) Δm = 0.563\tis_issue = False\n",
      "2023-06-02 17:58:42,949 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lymph\"\tRecall = 0.769 (global 0.612) Δm = 0.257\tis_issue = False\n",
      "2023-06-02 17:58:42,982 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pregnancy\"\tRecall = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:43,023 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complex\"\tRecall = 0.522 (global 0.612) Δm = -0.147\tis_issue = True\n",
      "2023-06-02 17:58:43,060 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"care\"\tRecall = 0.537 (global 0.612) Δm = -0.122\tis_issue = True\n",
      "2023-06-02 17:58:43,095 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"units\"\tRecall = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:43,125 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hx\"\tRecall = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:43,164 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscles\"\tRecall = 0.815 (global 0.612) Δm = 0.332\tis_issue = False\n",
      "2023-06-02 17:58:43,218 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"irrigated\"\tRecall = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:43,257 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consent\"\tRecall = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:43,303 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reveals\"\tRecall = 0.652 (global 0.612) Δm = 0.066\tis_issue = False\n",
      "2023-06-02 17:58:43,332 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reviewed\"\tRecall = 0.529 (global 0.612) Δm = -0.135\tis_issue = True\n",
      "2023-06-02 17:58:43,358 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subsequently\"\tRecall = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:43,394 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oropharynx\"\tRecall = 0.686 (global 0.612) Δm = 0.121\tis_issue = False\n",
      "2023-06-02 17:58:43,409 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"history\"\tRecall = 0.564 (global 0.612) Δm = -0.078\tis_issue = True\n",
      "2023-06-02 17:58:43,443 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectus\"\tRecall = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:43,477 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prepped\"\tRecall = 0.767 (global 0.612) Δm = 0.253\tis_issue = False\n",
      "2023-06-02 17:58:43,510 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suture\"\tRecall = 0.804 (global 0.612) Δm = 0.315\tis_issue = False\n",
      "2023-06-02 17:58:43,536 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clamped\"\tRecall = 0.960 (global 0.612) Δm = 0.569\tis_issue = False\n",
      "2023-06-02 17:58:43,563 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"function\"\tRecall = 0.414 (global 0.612) Δm = -0.324\tis_issue = True\n",
      "2023-06-02 17:58:43,600 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"coronary\"\tRecall = 0.391 (global 0.612) Δm = -0.360\tis_issue = True\n",
      "2023-06-02 17:58:43,629 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"speech\"\tRecall = 0.704 (global 0.612) Δm = 0.150\tis_issue = False\n",
      "2023-06-02 17:58:43,660 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vaginal\"\tRecall = 0.586 (global 0.612) Δm = -0.042\tis_issue = False\n",
      "2023-06-02 17:58:43,702 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subcuticular\"\tRecall = 0.792 (global 0.612) Δm = 0.294\tis_issue = False\n",
      "2023-06-02 17:58:43,728 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cut\"\tRecall = 0.917 (global 0.612) Δm = 0.498\tis_issue = False\n",
      "2023-06-02 17:58:43,766 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"correct\"\tRecall = 0.881 (global 0.612) Δm = 0.440\tis_issue = False\n",
      "2023-06-02 17:58:43,793 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dorsal\"\tRecall = 0.929 (global 0.612) Δm = 0.518\tis_issue = False\n",
      "2023-06-02 17:58:43,821 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"indications\"\tRecall = 0.889 (global 0.612) Δm = 0.453\tis_issue = False\n",
      "2023-06-02 17:58:43,854 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"technique\"\tRecall = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:43,877 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"findings\"\tRecall = 0.691 (global 0.612) Δm = 0.129\tis_issue = False\n",
      "2023-06-02 17:58:43,904 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"greater\"\tRecall = 0.435 (global 0.612) Δm = -0.289\tis_issue = True\n",
      "2023-06-02 17:58:43,937 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"presentation\"\tRecall = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:43,966 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"internal\"\tRecall = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:43,993 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligated\"\tRecall = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:44,025 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissected\"\tRecall = 0.881 (global 0.612) Δm = 0.440\tis_issue = False\n",
      "2023-06-02 17:58:44,059 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suctioned\"\tRecall = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:44,085 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lung\"\tRecall = 0.400 (global 0.612) Δm = -0.346\tis_issue = True\n",
      "2023-06-02 17:58:44,120 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"posterior\"\tRecall = 0.768 (global 0.612) Δm = 0.255\tis_issue = False\n",
      "2023-06-02 17:58:44,143 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hours\"\tRecall = 0.558 (global 0.612) Δm = -0.089\tis_issue = True\n",
      "2023-06-02 17:58:44,181 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vicryl\"\tRecall = 0.791 (global 0.612) Δm = 0.293\tis_issue = False\n",
      "2023-06-02 17:58:44,216 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knife\"\tRecall = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:44,241 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilaterally\"\tRecall = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:44,274 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"administered\"\tRecall = 0.767 (global 0.612) Δm = 0.253\tis_issue = False\n",
      "2023-06-02 17:58:44,306 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"grasped\"\tRecall = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:44,335 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subjective\"\tRecall = 0.367 (global 0.612) Δm = -0.401\tis_issue = True\n",
      "2023-06-02 17:58:44,371 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carotid\"\tRecall = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:44,403 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"impression\"\tRecall = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:44,430 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medial\"\tRecall = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:44,458 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"left\"\tRecall = 0.693 (global 0.612) Δm = 0.132\tis_issue = False\n",
      "2023-06-02 17:58:44,487 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mm\"\tRecall = 0.827 (global 0.612) Δm = 0.351\tis_issue = False\n",
      "2023-06-02 17:58:44,518 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"condition\"\tRecall = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:44,545 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gait\"\tRecall = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:44,583 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inferior\"\tRecall = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:44,612 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"masses\"\tRecall = 0.605 (global 0.612) Δm = -0.011\tis_issue = False\n",
      "2023-06-02 17:58:44,647 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectum\"\tRecall = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:44,674 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brought\"\tRecall = 0.747 (global 0.612) Δm = 0.221\tis_issue = False\n",
      "2023-06-02 17:58:44,700 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"delivered\"\tRecall = 0.920 (global 0.612) Δm = 0.504\tis_issue = False\n",
      "2023-06-02 17:58:44,732 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gentleman\"\tRecall = 0.696 (global 0.612) Δm = 0.137\tis_issue = False\n",
      "2023-06-02 17:58:44,755 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluid\"\tRecall = 0.759 (global 0.612) Δm = 0.241\tis_issue = False\n",
      "2023-06-02 17:58:44,784 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"change\"\tRecall = 0.725 (global 0.612) Δm = 0.185\tis_issue = False\n",
      "2023-06-02 17:58:44,815 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"light\"\tRecall = 0.674 (global 0.612) Δm = 0.102\tis_issue = False\n",
      "2023-06-02 17:58:44,838 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tube\"\tRecall = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:44,866 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"loss\"\tRecall = 0.781 (global 0.612) Δm = 0.276\tis_issue = False\n",
      "2023-06-02 17:58:44,892 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lateral\"\tRecall = 0.797 (global 0.612) Δm = 0.303\tis_issue = False\n",
      "2023-06-02 17:58:44,920 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sensory\"\tRecall = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:44,955 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vision\"\tRecall = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:44,984 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incised\"\tRecall = 0.900 (global 0.612) Δm = 0.471\tis_issue = False\n",
      "2023-06-02 17:58:45,008 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visit\"\tRecall = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:45,042 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tract\"\tRecall = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:45,067 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"procedure\"\tRecall = 0.726 (global 0.612) Δm = 0.187\tis_issue = False\n",
      "2023-06-02 17:58:45,092 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"advanced\"\tRecall = 0.759 (global 0.612) Δm = 0.240\tis_issue = False\n",
      "2023-06-02 17:58:45,123 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cavity\"\tRecall = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:45,150 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"flap\"\tRecall = 0.727 (global 0.612) Δm = 0.189\tis_issue = False\n",
      "2023-06-02 17:58:45,173 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"negative\"\tRecall = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:45,205 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"direct\"\tRecall = 0.731 (global 0.612) Δm = 0.194\tis_issue = False\n",
      "2023-06-02 17:58:45,235 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"months\"\tRecall = 0.562 (global 0.612) Δm = -0.081\tis_issue = True\n",
      "2023-06-02 17:58:45,260 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"states\"\tRecall = 0.535 (global 0.612) Δm = -0.126\tis_issue = True\n",
      "2023-06-02 17:58:45,291 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"following\"\tRecall = 0.609 (global 0.612) Δm = -0.004\tis_issue = False\n",
      "2023-06-02 17:58:45,319 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recovery\"\tRecall = 0.843 (global 0.612) Δm = 0.378\tis_issue = False\n",
      "2023-06-02 17:58:50,505 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `language(transcription)` == \"en\"\tAccuracy = 0.614 (global 0.612) Δm = 0.003\tis_issue = False\n",
      "2023-06-02 17:58:50,529 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"section\"\tAccuracy = 0.692 (global 0.612) Δm = 0.131\tis_issue = False\n",
      "2023-06-02 17:58:50,555 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"contrast\"\tAccuracy = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:50,579 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spine\"\tAccuracy = 0.703 (global 0.612) Δm = 0.148\tis_issue = False\n",
      "2023-06-02 17:58:50,611 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inserted\"\tAccuracy = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:50,650 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"length\"\tAccuracy = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:50,680 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adequate\"\tAccuracy = 0.816 (global 0.612) Δm = 0.333\tis_issue = False\n",
      "2023-06-02 17:58:50,707 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"identified\"\tAccuracy = 0.785 (global 0.612) Δm = 0.282\tis_issue = False\n",
      "2023-06-02 17:58:50,737 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scissors\"\tAccuracy = 0.852 (global 0.612) Δm = 0.392\tis_issue = False\n",
      "2023-06-02 17:58:50,760 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colon\"\tAccuracy = 0.690 (global 0.612) Δm = 0.127\tis_issue = False\n",
      "2023-06-02 17:58:50,788 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distress\"\tAccuracy = 0.486 (global 0.612) Δm = -0.206\tis_issue = True\n",
      "2023-06-02 17:58:50,816 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"motor\"\tAccuracy = 0.795 (global 0.612) Δm = 0.300\tis_issue = False\n",
      "2023-06-02 17:58:50,834 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pain\"\tAccuracy = 0.571 (global 0.612) Δm = -0.066\tis_issue = True\n",
      "2023-06-02 17:58:50,862 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"chromic\"\tAccuracy = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:50,891 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"transverse\"\tAccuracy = 0.812 (global 0.612) Δm = 0.328\tis_issue = False\n",
      "2023-06-02 17:58:50,912 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"revealed\"\tAccuracy = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:50,948 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tumor\"\tAccuracy = 0.583 (global 0.612) Δm = -0.047\tis_issue = False\n",
      "2023-06-02 17:58:50,976 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"areas\"\tAccuracy = 0.607 (global 0.612) Δm = -0.008\tis_issue = False\n",
      "2023-06-02 17:58:50,999 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"unremarkable\"\tAccuracy = 0.632 (global 0.612) Δm = 0.033\tis_issue = False\n",
      "2023-06-02 17:58:51,028 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilateral\"\tAccuracy = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:51,059 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wound\"\tAccuracy = 0.638 (global 0.612) Δm = 0.043\tis_issue = False\n",
      "2023-06-02 17:58:51,083 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"followup\"\tAccuracy = 0.396 (global 0.612) Δm = -0.353\tis_issue = True\n",
      "2023-06-02 17:58:51,104 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"20\"\tAccuracy = 0.598 (global 0.612) Δm = -0.023\tis_issue = False\n",
      "2023-06-02 17:58:51,137 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"membranes\"\tAccuracy = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:51,169 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"created\"\tAccuracy = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:51,192 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"portion\"\tAccuracy = 0.964 (global 0.612) Δm = 0.576\tis_issue = False\n",
      "2023-06-02 17:58:51,237 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"check\"\tAccuracy = 0.560 (global 0.612) Δm = -0.085\tis_issue = True\n",
      "2023-06-02 17:58:51,270 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"days\"\tAccuracy = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:51,292 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"symptoms\"\tAccuracy = 0.551 (global 0.612) Δm = -0.099\tis_issue = True\n",
      "2023-06-02 17:58:51,323 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cord\"\tAccuracy = 0.939 (global 0.612) Δm = 0.535\tis_issue = False\n",
      "2023-06-02 17:58:51,352 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"skin\"\tAccuracy = 0.723 (global 0.612) Δm = 0.182\tis_issue = False\n",
      "2023-06-02 17:58:51,378 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"description\"\tAccuracy = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:51,405 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualization\"\tAccuracy = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:51,434 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rule\"\tAccuracy = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:51,464 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"blade\"\tAccuracy = 0.900 (global 0.612) Δm = 0.471\tis_issue = False\n",
      "2023-06-02 17:58:51,489 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cm\"\tAccuracy = 0.716 (global 0.612) Δm = 0.171\tis_issue = False\n",
      "2023-06-02 17:58:51,525 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"metzenbaum\"\tAccuracy = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:51,555 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"place\"\tAccuracy = 0.828 (global 0.612) Δm = 0.353\tis_issue = False\n",
      "2023-06-02 17:58:51,579 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"draped\"\tAccuracy = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:51,606 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"use\"\tAccuracy = 0.681 (global 0.612) Δm = 0.112\tis_issue = False\n",
      "2023-06-02 17:58:51,638 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extremity\"\tAccuracy = 0.619 (global 0.612) Δm = 0.012\tis_issue = False\n",
      "2023-06-02 17:58:51,662 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"08\"\tAccuracy = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:51,689 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"iv\"\tAccuracy = 0.758 (global 0.612) Δm = 0.239\tis_issue = False\n",
      "2023-06-02 17:58:51,726 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oxygen\"\tAccuracy = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:51,767 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscle\"\tAccuracy = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:51,798 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"difficulty\"\tAccuracy = 0.734 (global 0.612) Δm = 0.200\tis_issue = False\n",
      "2023-06-02 17:58:51,819 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"xyz\"\tAccuracy = 0.318 (global 0.612) Δm = -0.480\tis_issue = True\n",
      "2023-06-02 17:58:51,843 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"evidence\"\tAccuracy = 0.691 (global 0.612) Δm = 0.130\tis_issue = False\n",
      "2023-06-02 17:58:51,873 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suite\"\tAccuracy = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:51,899 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"informed\"\tAccuracy = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:51,927 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspirin\"\tAccuracy = 0.435 (global 0.612) Δm = -0.289\tis_issue = True\n",
      "2023-06-02 17:58:51,958 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mild\"\tAccuracy = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:51,986 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scope\"\tAccuracy = 0.864 (global 0.612) Δm = 0.411\tis_issue = False\n",
      "2023-06-02 17:58:52,011 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"resection\"\tAccuracy = 0.700 (global 0.612) Δm = 0.144\tis_issue = False\n",
      "2023-06-02 17:58:52,044 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearance\"\tAccuracy = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:52,073 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"obtained\"\tAccuracy = 0.737 (global 0.612) Δm = 0.205\tis_issue = False\n",
      "2023-06-02 17:58:52,095 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nose\"\tAccuracy = 0.794 (global 0.612) Δm = 0.298\tis_issue = False\n",
      "2023-06-02 17:58:52,126 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colonoscopy\"\tAccuracy = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:52,156 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extended\"\tAccuracy = 0.929 (global 0.612) Δm = 0.518\tis_issue = False\n",
      "2023-06-02 17:58:52,181 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"similar\"\tAccuracy = 0.724 (global 0.612) Δm = 0.184\tis_issue = False\n",
      "2023-06-02 17:58:52,208 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"examination\"\tAccuracy = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:52,236 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denied\"\tAccuracy = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:52,257 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"found\"\tAccuracy = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:52,287 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"region\"\tAccuracy = 0.724 (global 0.612) Δm = 0.184\tis_issue = False\n",
      "2023-06-02 17:58:52,315 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ml\"\tAccuracy = 0.717 (global 0.612) Δm = 0.172\tis_issue = False\n",
      "2023-06-02 17:58:52,338 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"large\"\tAccuracy = 0.647 (global 0.612) Δm = 0.058\tis_issue = False\n",
      "2023-06-02 17:58:52,364 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cc\"\tAccuracy = 0.787 (global 0.612) Δm = 0.287\tis_issue = False\n",
      "2023-06-02 17:58:52,389 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"preoperative\"\tAccuracy = 0.761 (global 0.612) Δm = 0.243\tis_issue = False\n",
      "2023-06-02 17:58:52,412 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"supine\"\tAccuracy = 0.764 (global 0.612) Δm = 0.248\tis_issue = False\n",
      "2023-06-02 17:58:52,441 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knee\"\tAccuracy = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:52,472 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"urinary\"\tAccuracy = 0.486 (global 0.612) Δm = -0.206\tis_issue = True\n",
      "2023-06-02 17:58:52,496 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligament\"\tAccuracy = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:52,530 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stomach\"\tAccuracy = 0.850 (global 0.612) Δm = 0.389\tis_issue = False\n",
      "2023-06-02 17:58:52,560 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mri\"\tAccuracy = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:52,590 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"small\"\tAccuracy = 0.770 (global 0.612) Δm = 0.259\tis_issue = False\n",
      "2023-06-02 17:58:52,613 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"compression\"\tAccuracy = 0.636 (global 0.612) Δm = 0.040\tis_issue = False\n",
      "2023-06-02 17:58:52,642 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"day\"\tAccuracy = 0.481 (global 0.612) Δm = -0.214\tis_issue = True\n",
      "2023-06-02 17:58:52,674 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualized\"\tAccuracy = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:52,697 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"entered\"\tAccuracy = 0.875 (global 0.612) Δm = 0.430\tis_issue = False\n",
      "2023-06-02 17:58:52,727 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"external\"\tAccuracy = 0.606 (global 0.612) Δm = -0.009\tis_issue = False\n",
      "2023-06-02 17:58:52,748 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bladder\"\tAccuracy = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:52,775 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"running\"\tAccuracy = 0.825 (global 0.612) Δm = 0.348\tis_issue = False\n",
      "2023-06-02 17:58:52,794 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"position\"\tAccuracy = 0.794 (global 0.612) Δm = 0.298\tis_issue = False\n",
      "2023-06-02 17:58:52,824 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"continue\"\tAccuracy = 0.442 (global 0.612) Δm = -0.277\tis_issue = True\n",
      "2023-06-02 17:58:52,851 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"laterally\"\tAccuracy = 0.846 (global 0.612) Δm = 0.383\tis_issue = False\n",
      "2023-06-02 17:58:52,873 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ear\"\tAccuracy = 0.619 (global 0.612) Δm = 0.012\tis_issue = False\n",
      "2023-06-02 17:58:52,906 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"easily\"\tAccuracy = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:52,931 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"status\"\tAccuracy = 0.515 (global 0.612) Δm = -0.158\tis_issue = True\n",
      "2023-06-02 17:58:52,954 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"home\"\tAccuracy = 0.519 (global 0.612) Δm = -0.153\tis_issue = True\n",
      "2023-06-02 17:58:52,983 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nerve\"\tAccuracy = 0.850 (global 0.612) Δm = 0.389\tis_issue = False\n",
      "2023-06-02 17:58:53,009 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"foley\"\tAccuracy = 0.730 (global 0.612) Δm = 0.193\tis_issue = False\n",
      "2023-06-02 17:58:53,034 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cardiac\"\tAccuracy = 0.529 (global 0.612) Δm = -0.135\tis_issue = True\n",
      "2023-06-02 17:58:53,057 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pupils\"\tAccuracy = 0.706 (global 0.612) Δm = 0.154\tis_issue = False\n",
      "2023-06-02 17:58:53,084 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extending\"\tAccuracy = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:53,116 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"counts\"\tAccuracy = 0.892 (global 0.612) Δm = 0.458\tis_issue = False\n",
      "2023-06-02 17:58:53,136 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"estimated\"\tAccuracy = 0.823 (global 0.612) Δm = 0.344\tis_issue = False\n",
      "2023-06-02 17:58:53,163 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gross\"\tAccuracy = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:53,192 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spinal\"\tAccuracy = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:53,217 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterus\"\tAccuracy = 0.647 (global 0.612) Δm = 0.058\tis_issue = False\n",
      "2023-06-02 17:58:53,239 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appeared\"\tAccuracy = 0.765 (global 0.612) Δm = 0.250\tis_issue = False\n",
      "2023-06-02 17:58:53,271 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removal\"\tAccuracy = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:53,301 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sterile\"\tAccuracy = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:53,324 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"able\"\tAccuracy = 0.633 (global 0.612) Δm = 0.034\tis_issue = False\n",
      "2023-06-02 17:58:53,358 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abc\"\tAccuracy = 0.417 (global 0.612) Δm = -0.319\tis_issue = True\n",
      "2023-06-02 17:58:53,381 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"exam\"\tAccuracy = 0.663 (global 0.612) Δm = 0.084\tis_issue = False\n",
      "2023-06-02 17:58:53,403 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"renal\"\tAccuracy = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:53,439 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hemostasis\"\tAccuracy = 0.787 (global 0.612) Δm = 0.286\tis_issue = False\n",
      "2023-06-02 17:58:53,466 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"proximal\"\tAccuracy = 0.889 (global 0.612) Δm = 0.453\tis_issue = False\n",
      "2023-06-02 17:58:53,490 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectal\"\tAccuracy = 0.643 (global 0.612) Δm = 0.051\tis_issue = False\n",
      "2023-06-02 17:58:53,522 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"12\"\tAccuracy = 0.541 (global 0.612) Δm = -0.117\tis_issue = True\n",
      "2023-06-02 17:58:53,556 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"attention\"\tAccuracy = 0.786 (global 0.612) Δm = 0.284\tis_issue = False\n",
      "2023-06-02 17:58:53,581 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"activity\"\tAccuracy = 0.514 (global 0.612) Δm = -0.161\tis_issue = True\n",
      "2023-06-02 17:58:53,606 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"father\"\tAccuracy = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:53,638 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissection\"\tAccuracy = 0.762 (global 0.612) Δm = 0.245\tis_issue = False\n",
      "2023-06-02 17:58:53,662 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brain\"\tAccuracy = 0.645 (global 0.612) Δm = 0.054\tis_issue = False\n",
      "2023-06-02 17:58:53,687 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tubes\"\tAccuracy = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:53,713 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bleeding\"\tAccuracy = 0.727 (global 0.612) Δm = 0.189\tis_issue = False\n",
      "2023-06-02 17:58:53,743 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mouth\"\tAccuracy = 0.700 (global 0.612) Δm = 0.144\tis_issue = False\n",
      "2023-06-02 17:58:53,764 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incision\"\tAccuracy = 0.747 (global 0.612) Δm = 0.221\tis_issue = False\n",
      "2023-06-02 17:58:53,795 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wall\"\tAccuracy = 0.763 (global 0.612) Δm = 0.247\tis_issue = False\n",
      "2023-06-02 17:58:53,825 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"initially\"\tAccuracy = 0.517 (global 0.612) Δm = -0.155\tis_issue = True\n",
      "2023-06-02 17:58:53,845 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"family\"\tAccuracy = 0.573 (global 0.612) Δm = -0.063\tis_issue = True\n",
      "2023-06-02 17:58:53,872 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removed\"\tAccuracy = 0.790 (global 0.612) Δm = 0.291\tis_issue = False\n",
      "2023-06-02 17:58:53,900 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"treatment\"\tAccuracy = 0.518 (global 0.612) Δm = -0.154\tis_issue = True\n",
      "2023-06-02 17:58:53,923 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nasal\"\tAccuracy = 0.594 (global 0.612) Δm = -0.030\tis_issue = False\n",
      "2023-06-02 17:58:53,948 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fever\"\tAccuracy = 0.558 (global 0.612) Δm = -0.089\tis_issue = True\n",
      "2023-06-02 17:58:53,977 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reports\"\tAccuracy = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:54,003 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anterior\"\tAccuracy = 0.756 (global 0.612) Δm = 0.236\tis_issue = False\n",
      "2023-06-02 17:58:54,026 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"leg\"\tAccuracy = 0.567 (global 0.612) Δm = -0.074\tis_issue = True\n",
      "2023-06-02 17:58:54,057 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"changes\"\tAccuracy = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:54,090 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"strength\"\tAccuracy = 0.738 (global 0.612) Δm = 0.206\tis_issue = False\n",
      "2023-06-02 17:58:54,114 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fashion\"\tAccuracy = 0.807 (global 0.612) Δm = 0.319\tis_issue = False\n",
      "2023-06-02 17:58:54,137 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fascia\"\tAccuracy = 0.824 (global 0.612) Δm = 0.346\tis_issue = False\n",
      "2023-06-02 17:58:54,168 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lumbar\"\tAccuracy = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:54,193 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"enlarged\"\tAccuracy = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:54,213 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denies\"\tAccuracy = 0.459 (global 0.612) Δm = -0.250\tis_issue = True\n",
      "2023-06-02 17:58:54,251 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"office\"\tAccuracy = 0.476 (global 0.612) Δm = -0.222\tis_issue = True\n",
      "2023-06-02 17:58:54,278 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adenopathy\"\tAccuracy = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:54,303 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abnormalities\"\tAccuracy = 0.656 (global 0.612) Δm = 0.073\tis_issue = False\n",
      "2023-06-02 17:58:54,326 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"placed\"\tAccuracy = 0.780 (global 0.612) Δm = 0.274\tis_issue = False\n",
      "2023-06-02 17:58:54,356 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"improved\"\tAccuracy = 0.419 (global 0.612) Δm = -0.315\tis_issue = True\n",
      "2023-06-02 17:58:54,378 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mucosa\"\tAccuracy = 0.697 (global 0.612) Δm = 0.139\tis_issue = False\n",
      "2023-06-02 17:58:54,407 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"return\"\tAccuracy = 0.514 (global 0.612) Δm = -0.159\tis_issue = True\n",
      "2023-06-02 17:58:54,433 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"post\"\tAccuracy = 0.552 (global 0.612) Δm = -0.098\tis_issue = True\n",
      "2023-06-02 17:58:54,452 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"noted\"\tAccuracy = 0.651 (global 0.612) Δm = 0.064\tis_issue = False\n",
      "2023-06-02 17:58:54,488 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"closed\"\tAccuracy = 0.806 (global 0.612) Δm = 0.317\tis_issue = False\n",
      "2023-06-02 17:58:54,526 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distal\"\tAccuracy = 0.795 (global 0.612) Δm = 0.299\tis_issue = False\n",
      "2023-06-02 17:58:54,572 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"specimens\"\tAccuracy = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:54,603 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sponge\"\tAccuracy = 0.886 (global 0.612) Δm = 0.448\tis_issue = False\n",
      "2023-06-02 17:58:54,633 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clear\"\tAccuracy = 0.515 (global 0.612) Δm = -0.158\tis_issue = True\n",
      "2023-06-02 17:58:54,661 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"room\"\tAccuracy = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:54,695 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"weakness\"\tAccuracy = 0.732 (global 0.612) Δm = 0.196\tis_issue = False\n",
      "2023-06-02 17:58:54,730 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"superior\"\tAccuracy = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:54,760 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"neck\"\tAccuracy = 0.553 (global 0.612) Δm = -0.096\tis_issue = True\n",
      "2023-06-02 17:58:54,794 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"infection\"\tAccuracy = 0.607 (global 0.612) Δm = -0.009\tis_issue = False\n",
      "2023-06-02 17:58:54,828 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drainage\"\tAccuracy = 0.630 (global 0.612) Δm = 0.029\tis_issue = False\n",
      "2023-06-02 17:58:54,859 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ct\"\tAccuracy = 0.600 (global 0.612) Δm = -0.019\tis_issue = False\n",
      "2023-06-02 17:58:54,897 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"swelling\"\tAccuracy = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:54,930 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mass\"\tAccuracy = 0.596 (global 0.612) Δm = -0.026\tis_issue = False\n",
      "2023-06-02 17:58:54,961 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinic\"\tAccuracy = 0.542 (global 0.612) Δm = -0.115\tis_issue = True\n",
      "2023-06-02 17:58:55,001 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pelvis\"\tAccuracy = 0.733 (global 0.612) Δm = 0.199\tis_issue = False\n",
      "2023-06-02 17:58:55,035 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"taken\"\tAccuracy = 0.713 (global 0.612) Δm = 0.165\tis_issue = False\n",
      "2023-06-02 17:58:55,061 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervix\"\tAccuracy = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:55,098 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bovie\"\tAccuracy = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:55,127 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinical\"\tAccuracy = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:55,156 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"upper\"\tAccuracy = 0.718 (global 0.612) Δm = 0.173\tis_issue = False\n",
      "2023-06-02 17:58:55,188 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"arm\"\tAccuracy = 0.591 (global 0.612) Δm = -0.034\tis_issue = False\n",
      "2023-06-02 17:58:55,223 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesions\"\tAccuracy = 0.681 (global 0.612) Δm = 0.113\tis_issue = False\n",
      "2023-06-02 17:58:55,257 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tone\"\tAccuracy = 0.759 (global 0.612) Δm = 0.240\tis_issue = False\n",
      "2023-06-02 17:58:55,289 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"single\"\tAccuracy = 0.696 (global 0.612) Δm = 0.137\tis_issue = False\n",
      "2023-06-02 17:58:55,323 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drains\"\tAccuracy = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:55,351 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"right\"\tAccuracy = 0.659 (global 0.612) Δm = 0.077\tis_issue = False\n",
      "2023-06-02 17:58:55,378 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearing\"\tAccuracy = 0.680 (global 0.612) Δm = 0.111\tis_issue = False\n",
      "2023-06-02 17:58:55,410 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"head\"\tAccuracy = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:55,443 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reflexes\"\tAccuracy = 0.775 (global 0.612) Δm = 0.267\tis_issue = False\n",
      "2023-06-02 17:58:55,475 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abdominal\"\tAccuracy = 0.615 (global 0.612) Δm = 0.006\tis_issue = False\n",
      "2023-06-02 17:58:55,510 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"needle\"\tAccuracy = 0.821 (global 0.612) Δm = 0.343\tis_issue = False\n",
      "2023-06-02 17:58:55,539 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"daily\"\tAccuracy = 0.385 (global 0.612) Δm = -0.371\tis_issue = True\n",
      "2023-06-02 17:58:55,583 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthetic\"\tAccuracy = 0.913 (global 0.612) Δm = 0.492\tis_issue = False\n",
      "2023-06-02 17:58:55,620 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesion\"\tAccuracy = 0.692 (global 0.612) Δm = 0.131\tis_issue = False\n",
      "2023-06-02 17:58:55,658 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complications\"\tAccuracy = 0.697 (global 0.612) Δm = 0.140\tis_issue = False\n",
      "2023-06-02 17:58:55,679 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"postoperative\"\tAccuracy = 0.729 (global 0.612) Δm = 0.191\tis_issue = False\n",
      "2023-06-02 17:58:55,712 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"follow\"\tAccuracy = 0.550 (global 0.612) Δm = -0.101\tis_issue = True\n",
      "2023-06-02 17:58:55,750 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visual\"\tAccuracy = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:55,782 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"end\"\tAccuracy = 0.757 (global 0.612) Δm = 0.237\tis_issue = False\n",
      "2023-06-02 17:58:55,821 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluids\"\tAccuracy = 0.786 (global 0.612) Δm = 0.284\tis_issue = False\n",
      "2023-06-02 17:58:55,858 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"count\"\tAccuracy = 0.400 (global 0.612) Δm = -0.346\tis_issue = True\n",
      "2023-06-02 17:58:55,887 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspect\"\tAccuracy = 0.787 (global 0.612) Δm = 0.287\tis_issue = False\n",
      "2023-06-02 17:58:55,920 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"biopsy\"\tAccuracy = 0.625 (global 0.612) Δm = 0.021\tis_issue = False\n",
      "2023-06-02 17:58:55,948 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"performed\"\tAccuracy = 0.683 (global 0.612) Δm = 0.116\tis_issue = False\n",
      "2023-06-02 17:58:55,984 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mg\"\tAccuracy = 0.473 (global 0.612) Δm = -0.227\tis_issue = True\n",
      "2023-06-02 17:58:56,015 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"open\"\tAccuracy = 0.769 (global 0.612) Δm = 0.257\tis_issue = False\n",
      "2023-06-02 17:58:56,047 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"discharge\"\tAccuracy = 0.424 (global 0.612) Δm = -0.308\tis_issue = True\n",
      "2023-06-02 17:58:56,087 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stool\"\tAccuracy = 0.516 (global 0.612) Δm = -0.156\tis_issue = True\n",
      "2023-06-02 17:58:56,125 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carefully\"\tAccuracy = 0.826 (global 0.612) Δm = 0.350\tis_issue = False\n",
      "2023-06-02 17:58:56,154 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"age\"\tAccuracy = 0.639 (global 0.612) Δm = 0.044\tis_issue = False\n",
      "2023-06-02 17:58:56,191 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headaches\"\tAccuracy = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:56,228 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"high\"\tAccuracy = 0.488 (global 0.612) Δm = -0.203\tis_issue = True\n",
      "2023-06-02 17:58:56,256 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"erythema\"\tAccuracy = 0.655 (global 0.612) Δm = 0.071\tis_issue = False\n",
      "2023-06-02 17:58:56,283 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthesia\"\tAccuracy = 0.764 (global 0.612) Δm = 0.249\tis_issue = False\n",
      "2023-06-02 17:58:56,323 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"midline\"\tAccuracy = 0.836 (global 0.612) Δm = 0.367\tis_issue = False\n",
      "2023-06-02 17:58:56,354 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"equal\"\tAccuracy = 0.655 (global 0.612) Δm = 0.071\tis_issue = False\n",
      "2023-06-02 17:58:56,383 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"numbness\"\tAccuracy = 0.815 (global 0.612) Δm = 0.332\tis_issue = False\n",
      "2023-06-02 17:58:56,423 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suction\"\tAccuracy = 0.826 (global 0.612) Δm = 0.350\tis_issue = False\n",
      "2023-06-02 17:58:56,461 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pulmonary\"\tAccuracy = 0.533 (global 0.612) Δm = -0.128\tis_issue = True\n",
      "2023-06-02 17:58:56,499 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"underlying\"\tAccuracy = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:56,593 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cancer\"\tAccuracy = 0.622 (global 0.612) Δm = 0.016\tis_issue = False\n",
      "2023-06-02 17:58:56,627 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scan\"\tAccuracy = 0.568 (global 0.612) Δm = -0.071\tis_issue = True\n",
      "2023-06-02 17:58:56,642 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"patient\"\tAccuracy = 0.597 (global 0.612) Δm = -0.023\tis_issue = False\n",
      "2023-06-02 17:58:56,681 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"passed\"\tAccuracy = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:56,717 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"saline\"\tAccuracy = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:56,754 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cautery\"\tAccuracy = 0.741 (global 0.612) Δm = 0.211\tis_issue = False\n",
      "2023-06-02 17:58:56,798 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"body\"\tAccuracy = 0.816 (global 0.612) Δm = 0.333\tis_issue = False\n",
      "2023-06-02 17:58:56,826 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervical\"\tAccuracy = 0.708 (global 0.612) Δm = 0.158\tis_issue = False\n",
      "2023-06-02 17:58:56,861 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prostate\"\tAccuracy = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:56,899 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sleep\"\tAccuracy = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:56,932 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"epinephrine\"\tAccuracy = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:56,971 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"shoulder\"\tAccuracy = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:57,007 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tenderness\"\tAccuracy = 0.543 (global 0.612) Δm = -0.112\tis_issue = True\n",
      "2023-06-02 17:58:57,048 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stitch\"\tAccuracy = 0.909 (global 0.612) Δm = 0.486\tis_issue = False\n",
      "2023-06-02 17:58:57,083 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sinus\"\tAccuracy = 0.714 (global 0.612) Δm = 0.167\tis_issue = False\n",
      "2023-06-02 17:58:57,119 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"point\"\tAccuracy = 0.615 (global 0.612) Δm = 0.006\tis_issue = False\n",
      "2023-06-02 17:58:57,159 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"second\"\tAccuracy = 0.679 (global 0.612) Δm = 0.110\tis_issue = False\n",
      "2023-06-02 17:58:57,192 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterine\"\tAccuracy = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:57,233 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"breast\"\tAccuracy = 0.444 (global 0.612) Δm = -0.274\tis_issue = True\n",
      "2023-06-02 17:58:57,266 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"having\"\tAccuracy = 0.523 (global 0.612) Δm = -0.145\tis_issue = True\n",
      "2023-06-02 17:58:57,301 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"local\"\tAccuracy = 0.800 (global 0.612) Δm = 0.307\tis_issue = False\n",
      "2023-06-02 17:58:57,329 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"normal\"\tAccuracy = 0.641 (global 0.612) Δm = 0.048\tis_issue = False\n",
      "2023-06-02 17:58:57,361 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stable\"\tAccuracy = 0.627 (global 0.612) Δm = 0.025\tis_issue = False\n",
      "2023-06-02 17:58:57,403 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sutures\"\tAccuracy = 0.636 (global 0.612) Δm = 0.040\tis_issue = False\n",
      "2023-06-02 17:58:57,431 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"admission\"\tAccuracy = 0.375 (global 0.612) Δm = -0.387\tis_issue = True\n",
      "2023-06-02 17:58:57,460 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"round\"\tAccuracy = 0.774 (global 0.612) Δm = 0.265\tis_issue = False\n",
      "2023-06-02 17:58:57,494 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"artery\"\tAccuracy = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:57,530 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"problems\"\tAccuracy = 0.565 (global 0.612) Δm = -0.076\tis_issue = True\n",
      "2023-06-02 17:58:57,562 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recurrent\"\tAccuracy = 0.654 (global 0.612) Δm = 0.069\tis_issue = False\n",
      "2023-06-02 17:58:57,595 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medication\"\tAccuracy = 0.541 (global 0.612) Δm = -0.117\tis_issue = True\n",
      "2023-06-02 17:58:57,636 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sac\"\tAccuracy = 0.913 (global 0.612) Δm = 0.492\tis_issue = False\n",
      "2023-06-02 17:58:57,679 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"s1\"\tAccuracy = 0.556 (global 0.612) Δm = -0.092\tis_issue = True\n",
      "2023-06-02 17:58:57,713 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consistent\"\tAccuracy = 0.828 (global 0.612) Δm = 0.353\tis_issue = False\n",
      "2023-06-02 17:58:57,749 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"diarrhea\"\tAccuracy = 0.462 (global 0.612) Δm = -0.246\tis_issue = True\n",
      "2023-06-02 17:58:57,788 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tolerated\"\tAccuracy = 0.698 (global 0.612) Δm = 0.140\tis_issue = False\n",
      "2023-06-02 17:58:57,817 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headache\"\tAccuracy = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:57,851 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scalpel\"\tAccuracy = 0.957 (global 0.612) Δm = 0.563\tis_issue = False\n",
      "2023-06-02 17:58:57,883 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lymph\"\tAccuracy = 0.769 (global 0.612) Δm = 0.257\tis_issue = False\n",
      "2023-06-02 17:58:57,906 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pregnancy\"\tAccuracy = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:57,931 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complex\"\tAccuracy = 0.522 (global 0.612) Δm = -0.147\tis_issue = True\n",
      "2023-06-02 17:58:57,964 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"care\"\tAccuracy = 0.537 (global 0.612) Δm = -0.122\tis_issue = True\n",
      "2023-06-02 17:58:57,993 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"units\"\tAccuracy = 0.640 (global 0.612) Δm = 0.046\tis_issue = False\n",
      "2023-06-02 17:58:58,018 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hx\"\tAccuracy = 0.810 (global 0.612) Δm = 0.323\tis_issue = False\n",
      "2023-06-02 17:58:58,043 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscles\"\tAccuracy = 0.815 (global 0.612) Δm = 0.332\tis_issue = False\n",
      "2023-06-02 17:58:58,096 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"irrigated\"\tAccuracy = 0.854 (global 0.612) Δm = 0.395\tis_issue = False\n",
      "2023-06-02 17:58:58,123 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consent\"\tAccuracy = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:58,146 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reveals\"\tAccuracy = 0.652 (global 0.612) Δm = 0.066\tis_issue = False\n",
      "2023-06-02 17:58:58,173 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reviewed\"\tAccuracy = 0.529 (global 0.612) Δm = -0.135\tis_issue = True\n",
      "2023-06-02 17:58:58,203 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subsequently\"\tAccuracy = 0.750 (global 0.612) Δm = 0.226\tis_issue = False\n",
      "2023-06-02 17:58:58,228 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oropharynx\"\tAccuracy = 0.686 (global 0.612) Δm = 0.121\tis_issue = False\n",
      "2023-06-02 17:58:58,242 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"history\"\tAccuracy = 0.564 (global 0.612) Δm = -0.078\tis_issue = True\n",
      "2023-06-02 17:58:58,265 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectus\"\tAccuracy = 0.950 (global 0.612) Δm = 0.553\tis_issue = False\n",
      "2023-06-02 17:58:58,300 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prepped\"\tAccuracy = 0.767 (global 0.612) Δm = 0.253\tis_issue = False\n",
      "2023-06-02 17:58:58,328 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suture\"\tAccuracy = 0.804 (global 0.612) Δm = 0.315\tis_issue = False\n",
      "2023-06-02 17:58:58,352 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clamped\"\tAccuracy = 0.960 (global 0.612) Δm = 0.569\tis_issue = False\n",
      "2023-06-02 17:58:58,386 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"function\"\tAccuracy = 0.414 (global 0.612) Δm = -0.324\tis_issue = True\n",
      "2023-06-02 17:58:58,409 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"coronary\"\tAccuracy = 0.391 (global 0.612) Δm = -0.360\tis_issue = True\n",
      "2023-06-02 17:58:58,433 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"speech\"\tAccuracy = 0.704 (global 0.612) Δm = 0.150\tis_issue = False\n",
      "2023-06-02 17:58:58,463 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vaginal\"\tAccuracy = 0.586 (global 0.612) Δm = -0.042\tis_issue = False\n",
      "2023-06-02 17:58:58,499 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subcuticular\"\tAccuracy = 0.792 (global 0.612) Δm = 0.294\tis_issue = False\n",
      "2023-06-02 17:58:58,522 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cut\"\tAccuracy = 0.917 (global 0.612) Δm = 0.498\tis_issue = False\n",
      "2023-06-02 17:58:58,552 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"correct\"\tAccuracy = 0.881 (global 0.612) Δm = 0.440\tis_issue = False\n",
      "2023-06-02 17:58:58,584 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dorsal\"\tAccuracy = 0.929 (global 0.612) Δm = 0.518\tis_issue = False\n",
      "2023-06-02 17:58:58,614 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"indications\"\tAccuracy = 0.889 (global 0.612) Δm = 0.453\tis_issue = False\n",
      "2023-06-02 17:58:58,637 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"technique\"\tAccuracy = 0.783 (global 0.612) Δm = 0.279\tis_issue = False\n",
      "2023-06-02 17:58:58,666 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"findings\"\tAccuracy = 0.691 (global 0.612) Δm = 0.129\tis_issue = False\n",
      "2023-06-02 17:58:58,703 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"greater\"\tAccuracy = 0.435 (global 0.612) Δm = -0.289\tis_issue = True\n",
      "2023-06-02 17:58:58,730 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"presentation\"\tAccuracy = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:58,761 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"internal\"\tAccuracy = 0.857 (global 0.612) Δm = 0.401\tis_issue = False\n",
      "2023-06-02 17:58:58,795 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligated\"\tAccuracy = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:58,820 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissected\"\tAccuracy = 0.881 (global 0.612) Δm = 0.440\tis_issue = False\n",
      "2023-06-02 17:58:58,845 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suctioned\"\tAccuracy = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:58,876 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lung\"\tAccuracy = 0.400 (global 0.612) Δm = -0.346\tis_issue = True\n",
      "2023-06-02 17:58:58,901 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"posterior\"\tAccuracy = 0.768 (global 0.612) Δm = 0.255\tis_issue = False\n",
      "2023-06-02 17:58:58,922 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hours\"\tAccuracy = 0.558 (global 0.612) Δm = -0.089\tis_issue = True\n",
      "2023-06-02 17:58:58,953 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vicryl\"\tAccuracy = 0.791 (global 0.612) Δm = 0.293\tis_issue = False\n",
      "2023-06-02 17:58:58,985 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knife\"\tAccuracy = 0.952 (global 0.612) Δm = 0.557\tis_issue = False\n",
      "2023-06-02 17:58:59,011 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilaterally\"\tAccuracy = 0.646 (global 0.612) Δm = 0.056\tis_issue = False\n",
      "2023-06-02 17:58:59,034 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"administered\"\tAccuracy = 0.767 (global 0.612) Δm = 0.253\tis_issue = False\n",
      "2023-06-02 17:58:59,065 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"grasped\"\tAccuracy = 0.840 (global 0.612) Δm = 0.373\tis_issue = False\n",
      "2023-06-02 17:58:59,099 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subjective\"\tAccuracy = 0.367 (global 0.612) Δm = -0.401\tis_issue = True\n",
      "2023-06-02 17:58:59,122 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carotid\"\tAccuracy = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:59,152 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"impression\"\tAccuracy = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:59,188 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medial\"\tAccuracy = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:59,205 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"left\"\tAccuracy = 0.693 (global 0.612) Δm = 0.132\tis_issue = False\n",
      "2023-06-02 17:58:59,226 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mm\"\tAccuracy = 0.827 (global 0.612) Δm = 0.351\tis_issue = False\n",
      "2023-06-02 17:58:59,254 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"condition\"\tAccuracy = 0.667 (global 0.612) Δm = 0.090\tis_issue = False\n",
      "2023-06-02 17:58:59,278 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gait\"\tAccuracy = 0.742 (global 0.612) Δm = 0.213\tis_issue = False\n",
      "2023-06-02 17:58:59,307 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inferior\"\tAccuracy = 0.818 (global 0.612) Δm = 0.337\tis_issue = False\n",
      "2023-06-02 17:58:59,329 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"masses\"\tAccuracy = 0.605 (global 0.612) Δm = -0.011\tis_issue = False\n",
      "2023-06-02 17:58:59,368 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectum\"\tAccuracy = 0.650 (global 0.612) Δm = 0.062\tis_issue = False\n",
      "2023-06-02 17:58:59,405 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brought\"\tAccuracy = 0.747 (global 0.612) Δm = 0.221\tis_issue = False\n",
      "2023-06-02 17:58:59,451 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"delivered\"\tAccuracy = 0.920 (global 0.612) Δm = 0.504\tis_issue = False\n",
      "2023-06-02 17:58:59,478 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gentleman\"\tAccuracy = 0.696 (global 0.612) Δm = 0.137\tis_issue = False\n",
      "2023-06-02 17:58:59,500 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluid\"\tAccuracy = 0.759 (global 0.612) Δm = 0.241\tis_issue = False\n",
      "2023-06-02 17:58:59,527 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"change\"\tAccuracy = 0.725 (global 0.612) Δm = 0.185\tis_issue = False\n",
      "2023-06-02 17:58:59,556 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"light\"\tAccuracy = 0.674 (global 0.612) Δm = 0.102\tis_issue = False\n",
      "2023-06-02 17:58:59,592 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tube\"\tAccuracy = 0.773 (global 0.612) Δm = 0.263\tis_issue = False\n",
      "2023-06-02 17:58:59,615 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"loss\"\tAccuracy = 0.781 (global 0.612) Δm = 0.276\tis_issue = False\n",
      "2023-06-02 17:58:59,636 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lateral\"\tAccuracy = 0.797 (global 0.612) Δm = 0.303\tis_issue = False\n",
      "2023-06-02 17:58:59,668 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sensory\"\tAccuracy = 0.833 (global 0.612) Δm = 0.362\tis_issue = False\n",
      "2023-06-02 17:58:59,704 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vision\"\tAccuracy = 0.778 (global 0.612) Δm = 0.271\tis_issue = False\n",
      "2023-06-02 17:58:59,728 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incised\"\tAccuracy = 0.900 (global 0.612) Δm = 0.471\tis_issue = False\n",
      "2023-06-02 17:58:59,755 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visit\"\tAccuracy = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:59,779 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tract\"\tAccuracy = 0.450 (global 0.612) Δm = -0.265\tis_issue = True\n",
      "2023-06-02 17:58:59,801 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"procedure\"\tAccuracy = 0.726 (global 0.612) Δm = 0.187\tis_issue = False\n",
      "2023-06-02 17:58:59,825 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"advanced\"\tAccuracy = 0.759 (global 0.612) Δm = 0.240\tis_issue = False\n",
      "2023-06-02 17:58:59,850 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cavity\"\tAccuracy = 0.688 (global 0.612) Δm = 0.124\tis_issue = False\n",
      "2023-06-02 17:58:59,880 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"flap\"\tAccuracy = 0.727 (global 0.612) Δm = 0.189\tis_issue = False\n",
      "2023-06-02 17:58:59,903 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"negative\"\tAccuracy = 0.500 (global 0.612) Δm = -0.183\tis_issue = True\n",
      "2023-06-02 17:58:59,925 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"direct\"\tAccuracy = 0.731 (global 0.612) Δm = 0.194\tis_issue = False\n",
      "2023-06-02 17:58:59,952 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"months\"\tAccuracy = 0.562 (global 0.612) Δm = -0.081\tis_issue = True\n",
      "2023-06-02 17:58:59,985 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"states\"\tAccuracy = 0.535 (global 0.612) Δm = -0.126\tis_issue = True\n",
      "2023-06-02 17:59:00,010 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"following\"\tAccuracy = 0.609 (global 0.612) Δm = -0.004\tis_issue = False\n",
      "2023-06-02 17:59:00,034 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recovery\"\tAccuracy = 0.843 (global 0.612) Δm = 0.378\tis_issue = False\n",
      "2023-06-02 17:59:00,037 pid:10102 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: 70 issues found (took 0:01:00.011648)\n",
      " 15 issues detected. (Took 0:01:55.320886)\n",
      "Running detector TextPerturbationDetector…2023-06-02 17:59:00,039 pid:10102 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Running with transformations=['Transform to uppercase', 'Transform to lowercase', 'Transform to title case', 'Add typos', 'Punctuation Removal'] threshold=0.05 output_sensitivity=0.05 num_samples=1000\n",
      "2023-06-02 17:59:02,396 pid:10102 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Testing `transcription` for perturbation `Transform to uppercase`\tFail rate: 0.000\n",
      "2023-06-02 17:59:04,667 pid:10102 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Testing `transcription` for perturbation `Transform to lowercase`\tFail rate: 0.000\n",
      "2023-06-02 17:59:06,925 pid:10102 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Testing `transcription` for perturbation `Transform to title case`\tFail rate: 0.000\n",
      "2023-06-02 17:59:09,329 pid:10102 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Testing `transcription` for perturbation `Add typos`\tFail rate: 0.068\n",
      "2023-06-02 17:59:20,600 pid:10102 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Testing `transcription` for perturbation `Punctuation Removal`\tFail rate: 0.000\n",
      " 1 issues detected. (Took 0:00:20.561983)\n",
      "Running detector EthicalBiasDetector…2023-06-02 17:59:20,603 pid:10102 MainThread giskard.scanner.logger INFO     EthicalBiasDetector: Running with transformations=['Switch Gender', 'Switch Religion', 'Switch countries from high- to low-income and vice versa'] threshold=0.05 output_sensitivity=0.05 num_samples=1000\n",
      "2023-06-02 17:59:28,220 pid:10102 MainThread giskard.scanner.logger INFO     EthicalBiasDetector: Testing `transcription` for perturbation `Switch Gender`\tFail rate: 0.000\n",
      "2023-06-02 17:59:34,109 pid:10102 MainThread giskard.scanner.logger INFO     EthicalBiasDetector: Testing `transcription` for perturbation `Switch Religion`\tFail rate: 0.000\n",
      "2023-06-02 17:59:48,731 pid:10102 MainThread giskard.scanner.logger INFO     EthicalBiasDetector: Testing `transcription` for perturbation `Switch countries from high- to low-income and vice versa`\tFail rate: 0.000\n",
      " 0 issues detected. (Took 0:00:28.128792)\n",
      "Running detector DataLeakageDetector…2023-06-02 17:59:48,732 pid:10102 MainThread giskard.scanner.logger INFO     DataLeakageDetector: Running\n",
      " 0 issues detected. (Took 0:00:02.464612)\n",
      "Running detector StochasticityDetector…2023-06-02 17:59:51,198 pid:10102 MainThread giskard.scanner.logger INFO     StochasticityDetector: Running\n",
      " 0 issues detected. (Took 0:00:01.269315)\n",
      "Running detector OverconfidenceDetector…2023-06-02 17:59:52,468 pid:10102 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Running\n",
      "2023-06-02 17:59:52,469 pid:10102 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Calculating loss\n",
      "2023-06-02 17:59:52,480 pid:10102 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Loss calculated (took 0:00:00.008603)\n",
      "2023-06-02 17:59:52,481 pid:10102 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Finding data slices\n",
      "2023-06-02 18:00:17,174 pid:10102 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: 103 slices found (took 0:00:24.691918)\n",
      "2023-06-02 18:00:17,175 pid:10102 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Analyzing issues\n",
      "2023-06-02 18:00:17,185 pid:10102 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Using overconfidence threshold = 0.2468526289804987\n",
      "2023-06-02 18:01:05,501 pid:10102 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: 30 issues found (took 0:00:48.325031)\n",
      " 15 issues detected. (Took 0:01:13.032466)\n",
      "Running detector UnderconfidenceDetector…2023-06-02 18:01:05,503 pid:10102 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: Running\n",
      "2023-06-02 18:01:05,504 pid:10102 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: Calculating loss\n",
      "2023-06-02 18:01:05,517 pid:10102 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: Loss calculated (took 0:00:00.011595)\n",
      "2023-06-02 18:01:05,518 pid:10102 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: Finding data slices\n",
      "2023-06-02 18:01:59,644 pid:10102 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: 286 slices found (took 0:00:54.125191)\n",
      "2023-06-02 18:01:59,645 pid:10102 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: Analyzing issues\n",
      "2023-06-02 18:02:40,202 pid:10102 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: 0 issues found (took 0:00:40.555785)\n",
      " 0 issues detected. (Took 0:01:34.698440)\n",
      "Scan completed: 31 issues found. (Took 0:05:35.482660)\n"
     ]
    }
   ],
   "source": [
    "results = giskard.scan(wrapped_model, wrapped_data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T16:02:41.204208Z",
     "start_time": "2023-06-02T16:02:40.220504Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<iframe id=\"scan-4947914384\" srcdoc=\"\n",
       "&lt;style&gt;pre code.hljs{display:block;overflow-x:auto;padding:1em}code.hljs{padding:3px 5px}\n",
       "/*!\n",
       "  Theme: GitHub Dark\n",
       "  Description: Dark theme as seen on github.com\n",
       "  Author: github.com\n",
       "  Maintainer: @Hirse\n",
       "  Updated: 2021-05-15\n",
       "\n",
       "  Outdated base version: https://github.com/primer/github-syntax-dark\n",
       "  Current colors taken from GitHub&#x27;s CSS\n",
       "*/.hljs{background:#0d1117;color:#c9d1d9}.hljs-doctag,.hljs-keyword,.hljs-meta .hljs-keyword,.hljs-template-tag,.hljs-template-variable,.hljs-type,.hljs-variable.language_{color:#ff7b72}.hljs-title,.hljs-title.class_,.hljs-title.class_.inherited__,.hljs-title.function_{color:#d2a8ff}.hljs-attr,.hljs-attribute,.hljs-literal,.hljs-meta,.hljs-number,.hljs-operator,.hljs-selector-attr,.hljs-selector-class,.hljs-selector-id,.hljs-variable{color:#79c0ff}.hljs-meta .hljs-string,.hljs-regexp,.hljs-string{color:#a5d6ff}.hljs-built_in,.hljs-symbol{color:#ffa657}.hljs-code,.hljs-comment,.hljs-formula{color:#8b949e}.hljs-name,.hljs-quote,.hljs-selector-pseudo,.hljs-selector-tag{color:#7ee787}.hljs-subst{color:#c9d1d9}.hljs-section{color:#1f6feb;font-weight:700}.hljs-bullet{color:#f2cc60}.hljs-emphasis{color:#c9d1d9;font-style:italic}.hljs-strong{color:#c9d1d9;font-weight:700}.hljs-addition{background-color:#033a16;color:#aff5b4}.hljs-deletion{background-color:#67060c;color:#ffdcd7}.hljs-copy-wrapper{overflow:hidden;position:relative}.hljs-copy-button:focus,.hljs-copy-wrapper:hover .hljs-copy-button{transform:translateX(0)}.hljs-copy-button{background-color:#2d2b57;background-color:var(--hljs-theme-background);background-image:url(&#x27;data:image/svg+xml;charset=utf-8,&lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; width=&quot;16&quot; height=&quot;16&quot; fill=&quot;none&quot; viewBox=&quot;0 0 24 24&quot;&gt;&lt;path fill=&quot;%23fff&quot; fill-rule=&quot;evenodd&quot; d=&quot;M6 5a1 1 0 0 0-1 1v14a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1V6a1 1 0 0 0-1-1h-2a1 1 0 1 1 0-2h2a3 3 0 0 1 3 3v14a3 3 0 0 1-3 3H6a3 3 0 0 1-3-3V6a3 3 0 0 1 3-3h2a1 1 0 0 1 0 2H6Z&quot; clip-rule=&quot;evenodd&quot;/&gt;&lt;path fill=&quot;%23fff&quot; fill-rule=&quot;evenodd&quot; d=&quot;M7 3a2 2 0 0 1 2-2h6a2 2 0 0 1 2 2v2a2 2 0 0 1-2 2H9a2 2 0 0 1-2-2V3Zm8 0H9v2h6V3Z&quot; clip-rule=&quot;evenodd&quot;/&gt;&lt;/svg&gt;&#x27;);background-position:50%;background-repeat:no-repeat;border:1px solid #ffffff22;border-radius:.25rem;color:#fff;height:2rem;position:absolute;right:1em;text-indent:-9999px;top:1em;transition:background-color .2s ease,transform .2s ease-out;width:2rem}.hljs-copy-button:hover{border-color:#ffffff44}.hljs-copy-button:active{border-color:#ffffff66}.hljs-copy-button[data-copied=true]{background-image:none;text-indent:0;width:auto}@media (prefers-reduced-motion){.hljs-copy-button{transition:none}}.hljs-copy-alert{clip:rect(0 0 0 0);-webkit-clip-path:inset(50%);clip-path:inset(50%);height:1px;overflow:hidden;position:absolute;white-space:nowrap;width:1px}\n",
       "/*! tailwindcss v3.3.2 | MIT License | https://tailwindcss.com*/*,:after,:before{border:0 solid #e5e7eb;box-sizing:border-box}:after,:before{--tw-content:&quot;&quot;}html{-webkit-text-size-adjust:100%;font-feature-settings:normal;font-family:ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji;font-variation-settings:normal;line-height:1.5;-moz-tab-size:4;-o-tab-size:4;tab-size:4}body{line-height:inherit;margin:0}hr{border-top-width:1px;color:inherit;height:0}abbr:where([title]){-webkit-text-decoration:underline dotted;text-decoration:underline dotted}h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight:inherit}a{color:inherit;text-decoration:inherit}b,strong{font-weight:bolder}code,kbd,pre,samp{font-family:ui-monospace,SFMono-Regular,Menlo,Monaco,Consolas,Liberation Mono,Courier New,monospace;font-size:1em}small{font-size:80%}sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}sub{bottom:-.25em}sup{top:-.5em}table{border-collapse:collapse;border-color:inherit;text-indent:0}button,input,optgroup,select,textarea{color:inherit;font-family:inherit;font-size:100%;font-weight:inherit;line-height:inherit;margin:0;padding:0}button,select{text-transform:none}[type=button],[type=reset],[type=submit],button{-webkit-appearance:button;background-color:transparent;background-image:none}:-moz-focusring{outline:auto}:-moz-ui-invalid{box-shadow:none}progress{vertical-align:baseline}::-webkit-inner-spin-button,::-webkit-outer-spin-button{height:auto}[type=search]{-webkit-appearance:textfield;outline-offset:-2px}::-webkit-search-decoration{-webkit-appearance:none}::-webkit-file-upload-button{-webkit-appearance:button;font:inherit}summary{display:list-item}blockquote,dd,dl,figure,h1,h2,h3,h4,h5,h6,hr,p,pre{margin:0}fieldset{margin:0}fieldset,legend{padding:0}menu,ol,ul{list-style:none;margin:0;padding:0}textarea{resize:vertical}input::-moz-placeholder,textarea::-moz-placeholder{color:#9ca3af;opacity:1}input::placeholder,textarea::placeholder{color:#9ca3af;opacity:1}[role=button],button{cursor:pointer}:disabled{cursor:default}audio,canvas,embed,iframe,img,object,svg,video{display:block;vertical-align:middle}img,video{height:auto;max-width:100%}[hidden]{display:none}*,:after,:before{--tw-border-spacing-x:0;--tw-border-spacing-y:0;--tw-translate-x:0;--tw-translate-y:0;--tw-rotate:0;--tw-skew-x:0;--tw-skew-y:0;--tw-scale-x:1;--tw-scale-y:1;--tw-pan-x: ;--tw-pan-y: ;--tw-pinch-zoom: ;--tw-scroll-snap-strictness:proximity;--tw-gradient-from-position: ;--tw-gradient-via-position: ;--tw-gradient-to-position: ;--tw-ordinal: ;--tw-slashed-zero: ;--tw-numeric-figure: ;--tw-numeric-spacing: ;--tw-numeric-fraction: ;--tw-ring-inset: ;--tw-ring-offset-width:0px;--tw-ring-offset-color:#fff;--tw-ring-color:rgba(59,130,246,.5);--tw-ring-offset-shadow:0 0 #0000;--tw-ring-shadow:0 0 #0000;--tw-shadow:0 0 #0000;--tw-shadow-colored:0 0 #0000;--tw-blur: ;--tw-brightness: ;--tw-contrast: ;--tw-grayscale: ;--tw-hue-rotate: ;--tw-invert: ;--tw-saturate: ;--tw-sepia: ;--tw-drop-shadow: ;--tw-backdrop-blur: ;--tw-backdrop-brightness: ;--tw-backdrop-contrast: ;--tw-backdrop-grayscale: ;--tw-backdrop-hue-rotate: ;--tw-backdrop-invert: ;--tw-backdrop-opacity: ;--tw-backdrop-saturate: ;--tw-backdrop-sepia: }::backdrop{--tw-border-spacing-x:0;--tw-border-spacing-y:0;--tw-translate-x:0;--tw-translate-y:0;--tw-rotate:0;--tw-skew-x:0;--tw-skew-y:0;--tw-scale-x:1;--tw-scale-y:1;--tw-pan-x: ;--tw-pan-y: ;--tw-pinch-zoom: ;--tw-scroll-snap-strictness:proximity;--tw-gradient-from-position: ;--tw-gradient-via-position: ;--tw-gradient-to-position: ;--tw-ordinal: ;--tw-slashed-zero: ;--tw-numeric-figure: ;--tw-numeric-spacing: ;--tw-numeric-fraction: ;--tw-ring-inset: ;--tw-ring-offset-width:0px;--tw-ring-offset-color:#fff;--tw-ring-color:rgba(59,130,246,.5);--tw-ring-offset-shadow:0 0 #0000;--tw-ring-shadow:0 0 #0000;--tw-shadow:0 0 #0000;--tw-shadow-colored:0 0 #0000;--tw-blur: ;--tw-brightness: ;--tw-contrast: ;--tw-grayscale: ;--tw-hue-rotate: ;--tw-invert: ;--tw-saturate: ;--tw-sepia: ;--tw-drop-shadow: ;--tw-backdrop-blur: ;--tw-backdrop-brightness: ;--tw-backdrop-contrast: ;--tw-backdrop-grayscale: ;--tw-backdrop-hue-rotate: ;--tw-backdrop-invert: ;--tw-backdrop-opacity: ;--tw-backdrop-saturate: ;--tw-backdrop-sepia: }.collapse{visibility:collapse}.m-4{margin:1rem}.my-1{margin-bottom:.25rem;margin-top:.25rem}.my-2{margin-bottom:.5rem;margin-top:.5rem}.my-4{margin-top:1rem}.mb-4,.my-4{margin-bottom:1rem}.ml-1{margin-left:.25rem}.ml-2{margin-left:.5rem}.ml-4{margin-left:1rem}.mr-2{margin-right:.5rem}.mt-1{margin-top:.25rem}.mt-1\\.5{margin-top:.375rem}.mt-4{margin-top:1rem}.block{display:block}.inline-block{display:inline-block}.flex{display:flex}.table{display:table}.hidden{display:none}.h-11{height:2.75rem}.h-6{height:1.5rem}.w-6{width:1.5rem}.w-full{width:100%}.max-w-xl{max-width:36rem}.flex-grow{flex-grow:1}.table-auto{table-layout:auto}.cursor-pointer{cursor:pointer}.resize{resize:both}.list-disc{list-style-type:disc}.items-center{align-items:center}.space-x-1&gt;:not([hidden])~:not([hidden]){--tw-space-x-reverse:0;margin-left:calc(.25rem*(1 - var(--tw-space-x-reverse)));margin-right:calc(.25rem*var(--tw-space-x-reverse))}.overflow-scroll{overflow:scroll}.rounded{border-radius:.25rem}.rounded-full{border-radius:9999px}.rounded-sm{border-radius:.125rem}.rounded-t{border-top-left-radius:.25rem;border-top-right-radius:.25rem}.border{border-width:1px}.border-b{border-bottom-width:1px}.border-l{border-left-width:1px}.border-r{border-right-width:1px}.border-t{border-top-width:1px}.border-amber-200{--tw-border-opacity:1;border-color:rgb(253 230 138/var(--tw-border-opacity))}.border-gray-500{--tw-border-opacity:1;border-color:rgb(107 114 128/var(--tw-border-opacity))}.border-gray-600{--tw-border-opacity:1;border-color:rgb(75 85 99/var(--tw-border-opacity))}.border-red-400{--tw-border-opacity:1;border-color:rgb(248 113 113/var(--tw-border-opacity))}.border-zinc-100\\/50{border-color:hsla(240,5%,96%,.5)}.border-zinc-500{--tw-border-opacity:1;border-color:rgb(113 113 122/var(--tw-border-opacity))}.border-b-gray-500{--tw-border-opacity:1;border-bottom-color:rgb(107 114 128/var(--tw-border-opacity))}.bg-amber-100\\/40{background-color:hsla(48,96%,89%,.4)}.bg-amber-200{--tw-bg-opacity:1;background-color:rgb(253 230 138/var(--tw-bg-opacity))}.bg-green-100\\/40{background-color:rgba(220,252,231,.4)}.bg-red-400{--tw-bg-opacity:1;background-color:rgb(248 113 113/var(--tw-bg-opacity))}.bg-zinc-500{--tw-bg-opacity:1;background-color:rgb(113 113 122/var(--tw-bg-opacity))}.bg-zinc-700{--tw-bg-opacity:1;background-color:rgb(63 63 70/var(--tw-bg-opacity))}.p-3{padding:.75rem}.p-4{padding:1rem}.px-1{padding-left:.25rem;padding-right:.25rem}.px-2{padding-left:.5rem;padding-right:.5rem}.px-3{padding-left:.75rem;padding-right:.75rem}.px-4{padding-left:1rem;padding-right:1rem}.py-0{padding-bottom:0;padding-top:0}.py-0\\.5{padding-bottom:.125rem;padding-top:.125rem}.py-2{padding-bottom:.5rem;padding-top:.5rem}.pt-1{padding-top:.25rem}.text-left{text-align:left}.text-center{text-align:center}.text-right{text-align:right}.align-middle{vertical-align:middle}.text-sm{font-size:.875rem;line-height:1.25rem}.text-xs{font-size:.75rem;line-height:1rem}.font-bold{font-weight:700}.font-medium{font-weight:500}.uppercase{text-transform:uppercase}.lowercase{text-transform:lowercase}.text-amber-100{--tw-text-opacity:1;color:rgb(254 243 199/var(--tw-text-opacity))}.text-amber-200{--tw-text-opacity:1;color:rgb(253 230 138/var(--tw-text-opacity))}.text-amber-900{--tw-text-opacity:1;color:rgb(120 53 15/var(--tw-text-opacity))}.text-blue-300{--tw-text-opacity:1;color:rgb(147 197 253/var(--tw-text-opacity))}.text-gray-400{--tw-text-opacity:1;color:rgb(156 163 175/var(--tw-text-opacity))}.text-green-50{--tw-text-opacity:1;color:rgb(240 253 244/var(--tw-text-opacity))}.text-red-400{--tw-text-opacity:1;color:rgb(248 113 113/var(--tw-text-opacity))}.text-white{--tw-text-opacity:1;color:rgb(255 255 255/var(--tw-text-opacity))}.text-zinc-100\\/90{color:hsla(240,5%,96%,.9)}.filter{filter:var(--tw-blur) var(--tw-brightness) var(--tw-contrast) var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate) var(--tw-sepia) var(--tw-drop-shadow)}table.dataframe{max-width:100%;overflow:auto;width:100%}.dataframe tr{border-bottom:1px solid #555}.dataframe td,.dataframe th{padding:.5rem}.dataframe th{text-align:left!important}.first\\:border-t:first-child{border-top-width:1px}.hover\\:border-zinc-500:hover{--tw-border-opacity:1;border-color:rgb(113 113 122/var(--tw-border-opacity))}.hover\\:bg-zinc-400:hover{--tw-bg-opacity:1;background-color:rgb(161 161 170/var(--tw-bg-opacity))}.hover\\:bg-zinc-500:hover{--tw-bg-opacity:1;background-color:rgb(113 113 122/var(--tw-bg-opacity))}.hover\\:bg-zinc-700:hover{--tw-bg-opacity:1;background-color:rgb(63 63 70/var(--tw-bg-opacity))}.hover\\:text-white:hover{--tw-text-opacity:1;color:rgb(255 255 255/var(--tw-text-opacity))}.group.open .group-\\[\\.open\\]\\:inline-block{display:inline-block}.group.open .group-\\[\\.open\\]\\:hidden{display:none}.group.active .group-\\[\\.active\\]\\:border-gray-500{--tw-border-opacity:1;border-color:rgb(107 114 128/var(--tw-border-opacity))}.group.active .group-\\[\\.active\\]\\:border-b-zinc-800{--tw-border-opacity:1;border-bottom-color:rgb(39 39 42/var(--tw-border-opacity))}.group.active .group-\\[\\.active\\]\\:bg-zinc-800{--tw-bg-opacity:1;background-color:rgb(39 39 42/var(--tw-bg-opacity))}.peer.open~.peer-\\[\\.open\\]\\:visible{visibility:visible}:is(.dark .dark\\:bg-zinc-800){--tw-bg-opacity:1;background-color:rgb(39 39 42/var(--tw-bg-opacity))}:is(.dark .dark\\:bg-zinc-900){--tw-bg-opacity:1;background-color:rgb(24 24 27/var(--tw-bg-opacity))}:is(.dark .dark\\:fill-white){fill:#fff}:is(.dark .dark\\:text-white){--tw-text-opacity:1;color:rgb(255 255 255/var(--tw-text-opacity))}&lt;/style&gt;\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "&lt;div class=&quot;dark&quot;&gt;\n",
       "    &lt;div id=&quot;gsk-scan&quot; class=&quot;dark:text-white dark:bg-zinc-800 rounded border border-gray-500&quot;&gt;\n",
       "        &lt;!-- TAB HEADER --&gt;\n",
       "&lt;div class=&quot;flex items-center h-11 pt-1 dark:bg-zinc-900 rounded-t&quot;&gt;\n",
       "    \n",
       "    &lt;div class=&quot;flex items-center px-4 dark:fill-white border-b border-gray-500 h-11&quot;&gt;\n",
       "        &lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; xml:space=&quot;preserve&quot; viewBox=&quot;0 0 410 213&quot; height=&quot;15&quot;&gt;\n",
       "            &lt;path\n",
       "                d=&quot;M318.78 21.94a59.439 59.439 0 0 1 36.45-12.56v.03c7.09-.1 14.13 1.2 20.71 3.84 6.58 2.64 12.57 6.56 17.62 11.54s9.06 10.91 11.8 17.45a53.44 53.44 0 0 1 0 41.3 53.588 53.588 0 0 1-11.8 17.45c-5.05 4.98-11.04 8.9-17.62 11.54s-13.62 3.95-20.71 3.84l-67.92.16-35.87 49.11-135.24 46.84 40.81-47.24a139.229 139.229 0 0 1-66.06-23.8H45.11L0 101.46h55.73C60.38 44.84 115.97 0 183.59 0c31.71 0 63.58 9.86 87.47 27.04a105.149 105.149 0 0 1 26.65 27.18 59.4 59.4 0 0 1 21.07-32.28zM99.85 46.03l55.4 55.45h21.31l-64.81-64.87c-4.18 2.86-8.15 6.01-11.9 9.42zm57.78-28.45 83.9 83.9h21.29l-86.21-86.22c-6.37.32-12.71 1.1-18.98 2.32zm-92.84 98.94H39.72l11.1 9.82h22.44c-3-3.12-5.83-6.4-8.47-9.82zm134.27 0H84.8l-.03.01c20.8 20.91 51.79 33.67 84.66 34.3l29.63-34.31zm-34.81 63.35 77.79-26.94 26.6-36.39h-49.68l-54.71 63.33zm146.77-78.65h46.2l.05.01c9.98-.55 19.36-4.97 26.14-12.32a38.409 38.409 0 0 0 10.16-27.05 38.44 38.44 0 0 0-11.56-26.48 38.405 38.405 0 0 0-26.74-10.94A44.287 44.287 0 0 0 324 37.41a44.268 44.268 0 0 0-12.97 31.28v32.53zm51.82-52.83c5.27.52 9.12 5.22 8.6 10.49-.52 5.27-5.22 9.12-10.49 8.6s-9.12-5.22-8.6-10.49c.53-5.27 5.22-9.12 10.49-8.6z&quot;\n",
       "                style=&quot;fill-rule:evenodd;clip-rule:evenodd&quot; /&gt;\n",
       "        &lt;/svg&gt;\n",
       "        &lt;div class=&quot;ml-4 py-2&quot;&gt;\n",
       "            &lt;span class=&quot;uppercase text-sm&quot;&gt;31 issues detected&lt;/span&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "\n",
       "    \n",
       "    \n",
       "    &lt;div data-tab-target=&quot;Performance&quot;\n",
       "        class=&quot;cursor-pointer group active&quot;&gt;\n",
       "        &lt;div\n",
       "            class=&quot;group-[.active]:bg-zinc-800 group-[.active]:border-b-zinc-800 group-[.active]:border-gray-500 border-l px-3 py-2 border-r border-t border-b border-gray-600 border-b-gray-500 h-11&quot;&gt;\n",
       "            Performance\n",
       "            \n",
       "            \n",
       "            &lt;span class=&quot;ml-1 rounded-full text-xs min-w-4 min-h-4 px-1 py-0.5 inline-block text-center bg-red-400&quot;&gt;\n",
       "                \n",
       "                    15\n",
       "                &lt;/span&gt;\n",
       "                \n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "    \n",
       "    &lt;div data-tab-target=&quot;Robustness&quot;\n",
       "        class=&quot;cursor-pointer group&quot;&gt;\n",
       "        &lt;div\n",
       "            class=&quot;group-[.active]:bg-zinc-800 group-[.active]:border-b-zinc-800 group-[.active]:border-gray-500 border-l px-3 py-2 border-r border-t border-b border-gray-600 border-b-gray-500 h-11&quot;&gt;\n",
       "            Robustness\n",
       "            \n",
       "            \n",
       "                &lt;span\n",
       "                    class=&quot;ml-1 rounded-full text-xs min-w-4 min-h-4 px-1 py-0.5 inline-block text-center bg-amber-200 text-amber-900&quot;&gt;\n",
       "                    \n",
       "                    1\n",
       "                &lt;/span&gt;\n",
       "                \n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "    \n",
       "    &lt;div data-tab-target=&quot;Overconfidence&quot;\n",
       "        class=&quot;cursor-pointer group&quot;&gt;\n",
       "        &lt;div\n",
       "            class=&quot;group-[.active]:bg-zinc-800 group-[.active]:border-b-zinc-800 group-[.active]:border-gray-500 border-l px-3 py-2 border-r border-t border-b border-gray-600 border-b-gray-500 h-11&quot;&gt;\n",
       "            Overconfidence\n",
       "            \n",
       "            \n",
       "            &lt;span class=&quot;ml-1 rounded-full text-xs min-w-4 min-h-4 px-1 py-0.5 inline-block text-center bg-red-400&quot;&gt;\n",
       "                \n",
       "                    15\n",
       "                &lt;/span&gt;\n",
       "                \n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "    \n",
       "\n",
       "    \n",
       "    &lt;div class=&quot;flex-grow border-b border-gray-500 h-11&quot;&gt;&lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "&lt;!-- TAB HEADER END --&gt;\n",
       "        \n",
       "&lt;div id=&quot;Performance&quot; role=&quot;tabpanel&quot; class=&quot;m-4 mb-4&quot;&gt;\n",
       "    &lt;div class=&quot;p-3 bg-amber-100/40 rounded-sm w-full flex align-middle&quot;&gt;\n",
       "        &lt;div class=&quot;text-amber-100 mt-1.5&quot;&gt;\n",
       "            &lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; fill=&quot;none&quot; viewBox=&quot;0 0 24 24&quot; stroke-width=&quot;1.5&quot;\n",
       "                stroke=&quot;currentColor&quot; class=&quot;w-6 h-6&quot;&gt;\n",
       "                &lt;path stroke-linecap=&quot;round&quot; stroke-linejoin=&quot;round&quot;\n",
       "                    d=&quot;M12 9v3.75m-9.303 3.376c-.866 1.5.217 3.374 1.948 3.374h14.71c1.73 0 2.813-1.874 1.948-3.374L13.949 3.378c-.866-1.5-3.032-1.5-3.898 0L2.697 16.126zM12 15.75h.007v.008H12v-.008z&quot; /&gt;\n",
       "            &lt;/svg&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;div class=&quot;ml-2 text-amber-100 text-sm&quot;&gt;\n",
       "            \n",
       "            &lt;p&gt;\n",
       "                We found some data slices in your dataset on which your model performance is lower than average.\n",
       "                Performance bias may happen for different reasons:\n",
       "            &lt;ul class=&quot;list-disc ml-4 my-1&quot;&gt;\n",
       "                &lt;li&gt;Not enough examples in the low-performing data slice in the training set&lt;/li&gt;\n",
       "                &lt;li&gt;Wrong labels in the training set in the low-performing data slice&lt;/li&gt;\n",
       "                &lt;li&gt;Drift between your training set and test set&lt;/li&gt;\n",
       "            &lt;/ul&gt;\n",
       "            &lt;/p&gt;\n",
       "            \n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "\n",
       "    &lt;div class=&quot;flex items-center space-x-1&quot;&gt;\n",
       "        &lt;h2 class=&quot;uppercase my-4 mr-2 font-medium&quot;&gt;Issues&lt;/h2&gt;\n",
       "        \n",
       "        &lt;span class=&quot;text-xs border rounded px-1 uppercase text-red-400 border-red-400&quot;&gt;15\n",
       "            major&lt;/span&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/div&gt;\n",
       "\n",
       "    &lt;div&gt;\n",
       "    &lt;table class=&quot;table-auto w-full text-white&quot;&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;xyz&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.318\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -48.00% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            22 samples (5.93%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;xyz&amp;#34;,\n",
       "                the f1 score is 48.0%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.82)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3985&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Decreased ability to perfor...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.37)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;70&lt;/th&gt;\n",
       "      &lt;td&gt;XYZ, M.D. ,Suite 123, ABC Avenue ,City, STATE ...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (p = 0.34)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;subjective&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.367\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -40.07% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            30 samples (8.09%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;subjective&amp;#34;,\n",
       "                the f1 score is 40.1%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2524&lt;/th&gt;\n",
       "      &lt;td&gt;POSTOPERATIVE DAY #1, TOTAL ABDOMINAL HYSTEREC...&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.24)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1299&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , I am following the patient today...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.74)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3313&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 66-year-old fem...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.73)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;admission&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.375\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -38.71% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            64 samples (17.25%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;admission&amp;#34;,\n",
       "                the f1 score is 38.7%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2911&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSES:,1.  Bilateral lower extre...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.67)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3312&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , The patient was seen and examine...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.79)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3440&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:  ,Complex open wound righ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.76)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;daily&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.385\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -37.14% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            65 samples (17.52%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;daily&amp;#34;,\n",
       "                the f1 score is 37.1%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3414&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (p = 0.38)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3726&lt;/th&gt;\n",
       "      &lt;td&gt;ADMITTING DIAGNOSES:,1.  Fever.,2.  Otitis med...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.69)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3440&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:  ,Complex open wound righ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.76)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;coronary&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.391\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -36.05% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            23 samples (6.20%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;coronary&amp;#34;,\n",
       "                the f1 score is 36.0%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1441&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , The patient is not in acute dist...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.35)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1327&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS:,  The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.51)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3400&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:,1.  Epigastric pain.  Que...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.71)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;followup&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.396\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -35.31% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            48 samples (12.94%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;followup&amp;#34;,\n",
       "                the f1 score is 35.3%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3401&lt;/th&gt;\n",
       "      &lt;td&gt;ADMISSION DIAGNOSIS (ES):,1. Chronic obstructi...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.27)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1422&lt;/th&gt;\n",
       "      &lt;td&gt;PROBLEM LIST:,1.  Type 1 diabetes mellitus, in...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.41)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3317&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Followup on diabetes mellit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.85)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;count&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Precision = 0.400\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -34.63% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            50 samples (13.48%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;count&amp;#34;,\n",
       "                the precision is 34.6%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3461&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Abdominal pain.,HISTORY OF ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (p = 0.72)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3414&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (p = 0.38)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3312&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , The patient was seen and examine...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.79)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;lung&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Precision = 0.400\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -34.63% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            20 samples (5.39%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;lung&amp;#34;,\n",
       "                the precision is 34.6%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3723&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  This 3-year-old female pres...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.27)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3760&lt;/th&gt;\n",
       "      &lt;td&gt;TECHNIQUE: , Sequential axial CT images were o...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology (p = 0.36)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3402&lt;/th&gt;\n",
       "      &lt;td&gt;REASON FOR ADMISSION: , Fever of unknown origi...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.70)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;function&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.414\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -32.37% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            29 samples (7.82%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;function&amp;#34;,\n",
       "                the f1 score is 32.4%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3400&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:,1.  Epigastric pain.  Que...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.71)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3933&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:,1.  Respiratory failure i...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.34)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3461&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Abdominal pain.,HISTORY OF ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (p = 0.72)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;abc&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.417\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -31.90% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            24 samples (6.47%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;abc&amp;#34;,\n",
       "                the f1 score is 31.9%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1435&lt;/th&gt;\n",
       "      &lt;td&gt;REASON FOR VISIT:  ,Followup cervical spinal s...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology (p = 0.78)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3461&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Abdominal pain.,HISTORY OF ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (p = 0.72)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3451&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:  ,Blood in toilet.,HISTORY: ,...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (p = 0.69)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;improved&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.419\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -31.46% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            31 samples (8.36%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;improved&amp;#34;,\n",
       "                the f1 score is 31.5%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3933&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:,1.  Respiratory failure i...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.34)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3440&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:  ,Complex open wound righ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.76)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2910&lt;/th&gt;\n",
       "      &lt;td&gt;DIAGNOSES ON ADMISSION,1.  Cerebrovascular acc...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.76)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;discharge&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.424\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -30.78% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            85 samples (22.91%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;discharge&amp;#34;,\n",
       "                the f1 score is 30.8%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2647&lt;/th&gt;\n",
       "      &lt;td&gt;FINAL DIAGNOSES:,  Delivered pregnancy, choles...&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.67)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3395&lt;/th&gt;\n",
       "      &lt;td&gt;ADMITTING DIAGNOSIS:,  Aftercare of multiple t...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.70)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;130&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:, This 61-year-old male presen...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.36)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;aspirin&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.435\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -28.94% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            23 samples (6.20%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;aspirin&amp;#34;,\n",
       "                the f1 score is 28.9%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1330&lt;/th&gt;\n",
       "      &lt;td&gt;DIAGNOSIS:,  Polycythemia vera with secondary ...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.42)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.85)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3313&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 66-year-old fem...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.73)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;greater&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.435\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -28.94% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            23 samples (6.20%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;greater&amp;#34;,\n",
       "                the f1 score is 28.9%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3230&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , This is a 61-yea...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.78)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1422&lt;/th&gt;\n",
       "      &lt;td&gt;PROBLEM LIST:,1.  Type 1 diabetes mellitus, in...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.41)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3946&lt;/th&gt;\n",
       "      &lt;td&gt;ADMISSION DIAGNOSIS: , Symptomatic cholelithia...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (p = 0.70)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;continue&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        F1 Score = 0.442\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -27.71% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            52 samples (14.02%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;continue&amp;#34;,\n",
       "                the f1 score is 27.7%\n",
       "                lower than the global f1 score.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3317&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Followup on diabetes mellit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.85)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1369&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.81)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3763&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Septal irritation.,HISTORY ...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.81)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "    &lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "\n",
       "&lt;div id=&quot;Robustness&quot; role=&quot;tabpanel&quot; class=&quot;m-4 mb-4 hidden&quot;&gt;\n",
       "    &lt;div class=&quot;p-3 bg-amber-100/40 rounded-sm w-full flex align-middle&quot;&gt;\n",
       "        &lt;div class=&quot;text-amber-100 mt-1.5&quot;&gt;\n",
       "            &lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; fill=&quot;none&quot; viewBox=&quot;0 0 24 24&quot; stroke-width=&quot;1.5&quot;\n",
       "                stroke=&quot;currentColor&quot; class=&quot;w-6 h-6&quot;&gt;\n",
       "                &lt;path stroke-linecap=&quot;round&quot; stroke-linejoin=&quot;round&quot;\n",
       "                    d=&quot;M12 9v3.75m-9.303 3.376c-.866 1.5.217 3.374 1.948 3.374h14.71c1.73 0 2.813-1.874 1.948-3.374L13.949 3.378c-.866-1.5-3.032-1.5-3.898 0L2.697 16.126zM12 15.75h.007v.008H12v-.008z&quot; /&gt;\n",
       "            &lt;/svg&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;div class=&quot;ml-2 text-amber-100 text-sm&quot;&gt;\n",
       "            \n",
       "            &lt;p&gt;\n",
       "                Your model seems to be sensitive to small perturbations in the input data. These perturbations can\n",
       "                include adding typos, changing word order, or turning text into uppercase or lowercase. This happens\n",
       "                when:\n",
       "            &lt;ul class=&quot;list-disc ml-4 my-1&quot;&gt;\n",
       "                &lt;li&gt;There is not enough diversity in the training data&lt;/li&gt;\n",
       "                &lt;li&gt;Overreliance on spurious correlations like the presence of specific word&lt;/li&gt;\n",
       "                &lt;li&gt;Use of complex models with large number of parameters that tend to overfit the training data&lt;/li&gt;\n",
       "            &lt;/ul&gt;\n",
       "            &lt;/p&gt;\n",
       "            \n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "\n",
       "    &lt;div class=&quot;flex items-center space-x-1&quot;&gt;\n",
       "        &lt;h2 class=&quot;uppercase my-4 mr-2 font-medium&quot;&gt;Issues&lt;/h2&gt;\n",
       "        \n",
       "        \n",
       "        &lt;span class=&quot;text-xs border rounded px-1 uppercase text-amber-200 border-amber-200&quot;&gt;1\n",
       "            medium&lt;/span&gt;\n",
       "        \n",
       "    &lt;/div&gt;\n",
       "\n",
       "    &lt;div&gt;\n",
       "    &lt;table class=&quot;table-auto w-full text-white&quot;&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;\n",
       "        Feature `transcription`\n",
       "    &lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Add typos\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-amber-200&quot;&gt;\n",
       "            6.78% of samples changed prediction after perturbation\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            \n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                When we perturb the content of feature “transcription” with the transformation\n",
       "                “Add typos” (see examples below), your model changes its prediction in about\n",
       "                6.8% of the cases.\n",
       "                We expected the predictions not to be affected by this transformation.\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: right;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;Add typos(transcription)&lt;/th&gt;\n",
       "      &lt;th&gt;Original prediction&lt;/th&gt;\n",
       "      &lt;th&gt;Prediction after perturbation&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;29&lt;/th&gt;\n",
       "      &lt;td&gt;PREOPERATIVE DIAGNOSIS:,  Desire for sterility...&lt;/td&gt;\n",
       "      &lt;td&gt;PREOPERATIVE DIAGNOSIS:,  Deaire for sterility...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2655&lt;/th&gt;\n",
       "      &lt;td&gt;EXAM:,  Mammographic screening FFDM,HISTORY: ,...&lt;/td&gt;\n",
       "      &lt;td&gt;EXAM:,  Mammographic screening FFDM,HISTORY: ,...&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3909&lt;/th&gt;\n",
       "      &lt;td&gt;ADMITTING DIAGNOSIS: , Kawasaki disease.,DISCH...&lt;/td&gt;\n",
       "      &lt;td&gt;AVDMITTING DIAGNOSIS: , Kawasaki disease.,DISC...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "    &lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "\n",
       "&lt;div id=&quot;Overconfidence&quot; role=&quot;tabpanel&quot; class=&quot;m-4 mb-4 hidden&quot;&gt;\n",
       "    &lt;div class=&quot;p-3 bg-amber-100/40 rounded-sm w-full flex align-middle&quot;&gt;\n",
       "        &lt;div class=&quot;text-amber-100 mt-1.5&quot;&gt;\n",
       "            &lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; fill=&quot;none&quot; viewBox=&quot;0 0 24 24&quot; stroke-width=&quot;1.5&quot;\n",
       "                stroke=&quot;currentColor&quot; class=&quot;w-6 h-6&quot;&gt;\n",
       "                &lt;path stroke-linecap=&quot;round&quot; stroke-linejoin=&quot;round&quot;\n",
       "                    d=&quot;M12 9v3.75m-9.303 3.376c-.866 1.5.217 3.374 1.948 3.374h14.71c1.73 0 2.813-1.874 1.948-3.374L13.949 3.378c-.866-1.5-3.032-1.5-3.898 0L2.697 16.126zM12 15.75h.007v.008H12v-.008z&quot; /&gt;\n",
       "            &lt;/svg&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;div class=&quot;ml-2 text-amber-100 text-sm&quot;&gt;\n",
       "            \n",
       "            &lt;p&gt;\n",
       "                We found some data slices in your dataset containing significant number of overconfident predictions.\n",
       "                Overconfident predictions are rows that are incorrect but are predicted with high probabilities or\n",
       "                confidence scores. This happens when:\n",
       "            &lt;ul class=&quot;list-disc ml-4 my-1&quot;&gt;\n",
       "                &lt;li&gt;There are not enough examples in the overconfident data slice in the training set&lt;/li&gt;\n",
       "                &lt;li&gt;Wrongly labeled examples in the training set in the overconfident data slice&lt;/li&gt;\n",
       "                &lt;li&gt;For imbalanced datasets, the model may assign high probabilities to predictions of the majority\n",
       "                    class&lt;/li&gt;\n",
       "            &lt;/ul&gt;\n",
       "            &lt;/p&gt;\n",
       "            \n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "\n",
       "    &lt;div class=&quot;flex items-center space-x-1&quot;&gt;\n",
       "        &lt;h2 class=&quot;uppercase my-4 mr-2 font-medium&quot;&gt;Issues&lt;/h2&gt;\n",
       "        \n",
       "        &lt;span class=&quot;text-xs border rounded px-1 uppercase text-red-400 border-red-400&quot;&gt;11\n",
       "            major&lt;/span&gt;\n",
       "        \n",
       "        \n",
       "        &lt;span class=&quot;text-xs border rounded px-1 uppercase text-amber-200 border-amber-200&quot;&gt;4\n",
       "            medium&lt;/span&gt;\n",
       "        \n",
       "    &lt;/div&gt;\n",
       "\n",
       "    &lt;div&gt;\n",
       "    &lt;table class=&quot;table-auto w-full text-white&quot;&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;mass&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.762\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            27.57% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            16 out of 52 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;mass&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (16 samples, corresponding to\n",
       "                76.2% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;110&lt;/th&gt;\n",
       "      &lt;td&gt;1.  Pelvic tumor.,2.  Cystocele.,3.  Rectocele...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology (p = 0.83)&lt;br&gt;Gastroente...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1369&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.81)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;58&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient retu...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.72)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;ct&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.750\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            25.58% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            15 out of 50 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;ct&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (15 samples, corresponding to\n",
       "                75.0% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3230&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , This is a 61-yea...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.78)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2910&lt;/th&gt;\n",
       "      &lt;td&gt;DIAGNOSES ON ADMISSION,1.  Cerebrovascular acc...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.76)&lt;br&gt;Neurology (p = 0...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3940&lt;/th&gt;\n",
       "      &lt;td&gt;ADMITTING DIAGNOSIS: , Intractable migraine wi...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.71)&lt;br&gt;Neurology (p = 0.17)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;16&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.750\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            25.58% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            15 out of 38 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;16&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (15 samples, corresponding to\n",
       "                75.0% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1311&lt;/th&gt;\n",
       "      &lt;td&gt;PROBLEM LIST:,1.  Refractory hypertension, muc...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.81)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2530&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  The patient comes for three...&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.68)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;weight&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.742\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            24.23% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            23 out of 76 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;weight&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (23 samples, corresponding to\n",
       "                74.2% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1369&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.81)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;having&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.742\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            24.23% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            23 out of 65 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;having&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (23 samples, corresponding to\n",
       "                74.2% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2941&lt;/th&gt;\n",
       "      &lt;td&gt;PREOPERATIVE DIAGNOSIS: , Brain tumors, multip...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurosurgery (p = 0.77)&lt;br&gt;Urology (p = 0.06)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;diet&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.739\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            23.76% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            17 out of 38 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;diet&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (17 samples, corresponding to\n",
       "                73.9% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3929&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Chest pain and fever.,HISTO...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.74)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3412&lt;/th&gt;\n",
       "      &lt;td&gt;ADMISSION DIAGNOSES: , Hyperglycemia, cholelit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.76)&lt;br&gt;Gastroenterology...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;today&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.737\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            23.38% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            28 out of 71 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;today&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (28 samples, corresponding to\n",
       "                73.7% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2941&lt;/th&gt;\n",
       "      &lt;td&gt;PREOPERATIVE DIAGNOSIS: , Brain tumors, multip...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurosurgery (p = 0.77)&lt;br&gt;Urology (p = 0.06)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;temperature&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.733\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            22.79% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            22 out of 60 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;temperature&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (22 samples, corresponding to\n",
       "                73.3% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3763&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Septal irritation.,HISTORY ...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.81)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1445&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , The patient states that she feel...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.79)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;discussed&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.727\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            21.78% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            16 out of 55 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;discussed&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (16 samples, corresponding to\n",
       "                72.7% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1435&lt;/th&gt;\n",
       "      &lt;td&gt;REASON FOR VISIT:  ,Followup cervical spinal s...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology (p = 0.78)&lt;br&gt;SOAP / Chart / Progress ...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2733&lt;/th&gt;\n",
       "      &lt;td&gt;FINAL DIAGNOSES:,1.  Herniated nucleuses pulpo...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurosurgery&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.74)&lt;br&gt;Neurology (p = 0...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;dr&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.725\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            21.48% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            37 out of 97 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;dr&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (37 samples, corresponding to\n",
       "                72.5% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3317&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Followup on diabetes mellit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.85)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;follow&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.722\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            20.93% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            26 out of 80 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;follow&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (26 samples, corresponding to\n",
       "                72.2% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1311&lt;/th&gt;\n",
       "      &lt;td&gt;PROBLEM LIST:,1.  Refractory hypertension, muc...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.81)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3412&lt;/th&gt;\n",
       "      &lt;td&gt;ADMISSION DIAGNOSES: , Hyperglycemia, cholelit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (p = 0.76)&lt;br&gt;Gastroenterology...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3312&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , The patient was seen and examine...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.79)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;wound&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.714\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-amber-200&quot;&gt;\n",
       "            19.60% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            15 out of 58 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;wound&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (15 samples, corresponding to\n",
       "                71.4% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;110&lt;/th&gt;\n",
       "      &lt;td&gt;1.  Pelvic tumor.,2.  Cystocele.,3.  Rectocele...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology (p = 0.83)&lt;br&gt;Gastroente...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2941&lt;/th&gt;\n",
       "      &lt;td&gt;PREOPERATIVE DIAGNOSIS: , Brain tumors, multip...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurosurgery (p = 0.77)&lt;br&gt;Urology (p = 0.06)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2765&lt;/th&gt;\n",
       "      &lt;td&gt;PREOPERATIVE DIAGNOSIS:,  Rule out temporal ar...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurosurgery (p = 0.75)&lt;br&gt;Gastroenterology (p =...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;bladder&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.714\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-amber-200&quot;&gt;\n",
       "            19.60% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            15 out of 60 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;bladder&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (15 samples, corresponding to\n",
       "                71.4% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;110&lt;/th&gt;\n",
       "      &lt;td&gt;1.  Pelvic tumor.,2.  Cystocele.,3.  Rectocele...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology (p = 0.83)&lt;br&gt;Gastroente...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1435&lt;/th&gt;\n",
       "      &lt;td&gt;REASON FOR VISIT:  ,Followup cervical spinal s...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology (p = 0.78)&lt;br&gt;SOAP / Chart / Progress ...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1404&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS:,  The patient pres...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;Urology (p = 0.75)&lt;br&gt;SOAP / Chart / Progress No...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;times&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.714\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-amber-200&quot;&gt;\n",
       "            19.60% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            15 out of 44 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;times&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (15 samples, corresponding to\n",
       "                71.4% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3317&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Followup on diabetes mellit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.85)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3763&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Septal irritation.,HISTORY ...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (p = 0.81)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;bleeding&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rate = 0.708\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (Global = 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-amber-200&quot;&gt;\n",
       "            18.60% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            17 out of 88 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Show details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;Hide details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left collapse peer-[.open]:visible border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;bleeding&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (17 samples, corresponding to\n",
       "                70.8% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;110&lt;/th&gt;\n",
       "      &lt;td&gt;1.  Pelvic tumor.,2.  Cystocele.,3.  Rectocele...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology (p = 0.83)&lt;br&gt;Gastroente...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (p = 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1435&lt;/th&gt;\n",
       "      &lt;td&gt;REASON FOR VISIT:  ,Followup cervical spinal s...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology (p = 0.78)&lt;br&gt;SOAP / Chart / Progress ...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "    &lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "\n",
       "\n",
       "\n",
       "        \n",
       "        &lt;div class=&quot;p-4&quot;&gt;\n",
       "    &lt;h2 class=&quot;my-4 mr-2 font-medium&quot;&gt;Generate a Test Suite to debug the detected issues&lt;/h2&gt;\n",
       "\n",
       "    &lt;div class=&quot;text-sm&quot;&gt;\n",
       "        &lt;pre&gt;&lt;code class=&quot;language-python rounded&quot;&gt;from giskard import GiskardClient\n",
       "test_suite = results.generate_test_suite(&quot;My first test suite&quot;)\n",
       "\n",
       "# To start the Giskard server, run in your terminal within the Python\n",
       "# environment containing the dependencies of your model: \n",
       "giskard server start\n",
       "giskard worker start -u http://localhost:19000/\n",
       "\n",
       "# Then upload your test suite to the opened Giskard server\n",
       "client = GiskardClient(&quot;http://localhost:19000&quot;, &quot;GISKARD_API_KEY&quot;)\n",
       "client.create_project(&quot;my_project_id&quot;, &quot;my_project_name&quot;)\n",
       "test_suite.upload(client, &quot;my_project_id&quot;)\n",
       "&lt;/code&gt;&lt;/pre&gt;\n",
       "    &lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "        \n",
       "    &lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "&lt;script type=&quot;text/javascript&quot;&gt;\n",
       "    &quot;use strict&quot;;function _typeof(u){return(_typeof=&quot;function&quot;==typeof Symbol&amp;&amp;&quot;symbol&quot;==typeof Symbol.iterator?function(u){return typeof u}:function(u){return u&amp;&amp;&quot;function&quot;==typeof Symbol&amp;&amp;u.constructor===Symbol&amp;&amp;u!==Symbol.prototype?&quot;symbol&quot;:typeof u})(u)}function _wrapNativeSuper(u){var e=&quot;function&quot;==typeof Map?new Map:void 0;return(_wrapNativeSuper=function(u){if(null===u||!_isNativeFunction(u))return u;if(&quot;function&quot;!=typeof u)throw new TypeError(&quot;Super expression must either be null or a function&quot;);if(void 0!==e){if(e.has(u))return e.get(u);e.set(u,t)}function t(){return _construct(u,arguments,_getPrototypeOf(this).constructor)}return t.prototype=Object.create(u.prototype,{constructor:{value:t,enumerable:!1,writable:!0,configurable:!0}}),_setPrototypeOf(t,u)})(u)}function _construct(u,e,t){return(_construct=_isNativeReflectConstruct()?Reflect.construct.bind():function(u,e,t){var n=[null];n.push.apply(n,e);var D=new(Function.bind.apply(u,n));return t&amp;&amp;_setPrototypeOf(D,t.prototype),D}).apply(null,arguments)}function _isNativeFunction(u){return-1!==Function.toString.call(u).indexOf(&quot;[native code]&quot;)}function _slicedToArray(u,e){return _arrayWithHoles(u)||_iterableToArrayLimit(u,e)||_unsupportedIterableToArray(u,e)||_nonIterableRest()}function _nonIterableRest(){throw new TypeError(&quot;Invalid attempt to destructure non-iterable instance.\\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.&quot;)}function _iterableToArrayLimit(u,e){var t=null==u?null:&quot;undefined&quot;!=typeof Symbol&amp;&amp;u[Symbol.iterator]||u[&quot;@@iterator&quot;];if(null!=t){var n,D,r,o,i=[],a=!0,c=!1;try{if(r=(t=t.call(u)).next,0===e){if(Object(t)!==t)return;a=!1}else for(;!(a=(n=r.call(t)).done)&amp;&amp;(i.push(n.value),i.length!==e);a=!0);}catch(u){c=!0,D=u}finally{try{if(!a&amp;&amp;null!=t.return&amp;&amp;(o=t.return(),Object(o)!==o))return}finally{if(c)throw D}}return i}}function _arrayWithHoles(u){if(Array.isArray(u))return u}function _inherits(u,e){if(&quot;function&quot;!=typeof e&amp;&amp;null!==e)throw new TypeError(&quot;Super expression must either be null or a function&quot;);u.prototype=Object.create(e&amp;&amp;e.prototype,{constructor:{value:u,writable:!0,configurable:!0}}),Object.defineProperty(u,&quot;prototype&quot;,{writable:!1}),e&amp;&amp;_setPrototypeOf(u,e)}function _setPrototypeOf(u,e){return(_setPrototypeOf=Object.setPrototypeOf?Object.setPrototypeOf.bind():function(u,e){return u.__proto__=e,u})(u,e)}function _createSuper(u){var e=_isNativeReflectConstruct();return function(){var t,n=_getPrototypeOf(u);if(e){var D=_getPrototypeOf(this).constructor;t=Reflect.construct(n,arguments,D)}else t=n.apply(this,arguments);return _possibleConstructorReturn(this,t)}}function _possibleConstructorReturn(u,e){if(e&amp;&amp;(&quot;object&quot;===_typeof(e)||&quot;function&quot;==typeof e))return e;if(void 0!==e)throw new TypeError(&quot;Derived constructors may only return object or undefined&quot;);return _assertThisInitialized(u)}function _assertThisInitialized(u){if(void 0===u)throw new ReferenceError(&quot;this hasn&#x27;t been initialised - super() hasn&#x27;t been called&quot;);return u}function _isNativeReflectConstruct(){if(&quot;undefined&quot;==typeof Reflect||!Reflect.construct)return!1;if(Reflect.construct.sham)return!1;if(&quot;function&quot;==typeof Proxy)return!0;try{return Boolean.prototype.valueOf.call(Reflect.construct(Boolean,[],function(){})),!0}catch(u){return!1}}function _getPrototypeOf(u){return(_getPrototypeOf=Object.setPrototypeOf?Object.getPrototypeOf.bind():function(u){return u.__proto__||Object.getPrototypeOf(u)})(u)}function _toConsumableArray(u){return _arrayWithoutHoles(u)||_iterableToArray(u)||_unsupportedIterableToArray(u)||_nonIterableSpread()}function _nonIterableSpread(){throw new TypeError(&quot;Invalid attempt to spread non-iterable instance.\\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.&quot;)}function _unsupportedIterableToArray(u,e){if(u){if(&quot;string&quot;==typeof u)return _arrayLikeToArray(u,e);var t=Object.prototype.toString.call(u).slice(8,-1);return&quot;Object&quot;===t&amp;&amp;u.constructor&amp;&amp;(t=u.constructor.name),&quot;Map&quot;===t||&quot;Set&quot;===t?Array.from(u):&quot;Arguments&quot;===t||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(t)?_arrayLikeToArray(u,e):void 0}}function _iterableToArray(u){if(&quot;undefined&quot;!=typeof Symbol&amp;&amp;null!=u[Symbol.iterator]||null!=u[&quot;@@iterator&quot;])return Array.from(u)}function _arrayWithoutHoles(u){if(Array.isArray(u))return _arrayLikeToArray(u)}function _arrayLikeToArray(u,e){(null==e||e&gt;u.length)&amp;&amp;(e=u.length);for(var t=0,n=new Array(e);t&lt;e;t++)n[t]=u[t];return n}function _classCallCheck(u,e){if(!(u instanceof e))throw new TypeError(&quot;Cannot call a class as a function&quot;)}function _defineProperties(u,e){for(var t=0;t&lt;e.length;t++){var n=e[t];n.enumerable=n.enumerable||!1,n.configurable=!0,&quot;value&quot;in n&amp;&amp;(n.writable=!0),Object.defineProperty(u,_toPropertyKey(n.key),n)}}function _createClass(u,e,t){return e&amp;&amp;_defineProperties(u.prototype,e),t&amp;&amp;_defineProperties(u,t),Object.defineProperty(u,&quot;prototype&quot;,{writable:!1}),u}function _toPropertyKey(u){var e=_toPrimitive(u,&quot;string&quot;);return&quot;symbol&quot;===_typeof(e)?e:String(e)}function _toPrimitive(u,e){if(&quot;object&quot;!==_typeof(u)||null===u)return u;var t=u[Symbol.toPrimitive];if(void 0!==t){var n=t.call(u,e||&quot;default&quot;);if(&quot;object&quot;!==_typeof(n))return n;throw new TypeError(&quot;@@toPrimitive must return a primitive value.&quot;)}return(&quot;string&quot;===e?String:Number)(u)}function _typeof(u){return(_typeof=&quot;function&quot;==typeof Symbol&amp;&amp;&quot;symbol&quot;==typeof Symbol.iterator?function(u){return typeof u}:function(u){return u&amp;&amp;&quot;function&quot;==typeof Symbol&amp;&amp;u.constructor===Symbol&amp;&amp;u!==Symbol.prototype?&quot;symbol&quot;:typeof u})(u)}!function(u){if(&quot;undefined&quot;!=typeof window){var e=!0,t=&quot;&quot;,n=0,D=&quot;&quot;,r=null,o=&quot;&quot;,i=!1,a={resize:1,click:1},c=128,s=!0,l=1,F=&quot;bodyOffset&quot;,C=F,E=!0,d=&quot;&quot;,f={},A=32,g=null,h=!1,p=!1,B=&quot;[iFrameSizer]&quot;,m=B.length,y=&quot;&quot;,b={max:1,min:1,bodyScroll:1,documentElementScroll:1},v=&quot;child&quot;,w=!0,_=window.parent,S=&quot;*&quot;,x=0,k=!1,O=null,T=16,N=1,M=&quot;scroll&quot;,R=M,j=window,I=function(){Du(&quot;onMessage function not defined&quot;)},L=function(){},P=function(){},H={height:function(){return Du(&quot;Custom height calculation function not defined&quot;),document.documentElement.offsetHeight},width:function(){return Du(&quot;Custom width calculation function not defined&quot;),document.body.scrollWidth}},z={},U=!1;try{var K=Object.create({},{passive:{get:function(){U=!0}}});window.addEventListener(&quot;test&quot;,Q,K),window.removeEventListener(&quot;test&quot;,Q,K)}catch(Au){}var W,q,X,G,J,Z,$={bodyOffset:function(){return document.body.offsetHeight+du(&quot;marginTop&quot;)+du(&quot;marginBottom&quot;)},offset:function(){return $.bodyOffset()},bodyScroll:function(){return document.body.scrollHeight},custom:function(){return H.height()},documentElementOffset:function(){return document.documentElement.offsetHeight},documentElementScroll:function(){return document.documentElement.scrollHeight},max:function(){return Math.max.apply(null,Au($))},min:function(){return Math.min.apply(null,Au($))},grow:function(){return $.max()},lowestElement:function(){return Math.max($.bodyOffset()||$.documentElementOffset(),fu(&quot;bottom&quot;,hu()))},taggedElement:function(){return gu(&quot;bottom&quot;,&quot;data-iframe-height&quot;)}},V={bodyScroll:function(){return document.body.scrollWidth},bodyOffset:function(){return document.body.offsetWidth},custom:function(){return H.width()},documentElementScroll:function(){return document.documentElement.scrollWidth},documentElementOffset:function(){return document.documentElement.offsetWidth},scroll:function(){return Math.max(V.bodyScroll(),V.documentElementScroll())},max:function(){return Math.max.apply(null,Au(V))},min:function(){return Math.min.apply(null,Au(V))},rightMostElement:function(){return fu(&quot;right&quot;,hu())},taggedElement:function(){return gu(&quot;right&quot;,&quot;data-iframe-width&quot;)}},Y=(W=pu,J=null,Z=0,function(){var u=Date.now(),e=T-(u-(Z=Z||u));return q=this,X=arguments,e&lt;=0||T&lt;e?(J&amp;&amp;(clearTimeout(J),J=null),Z=u,G=W.apply(q,X),J||(q=X=null)):J=J||setTimeout(Bu,e),G});uu(window,&quot;message&quot;,function(e){var t={init:function(){d=e.data,_=e.source,ru(),s=!1,setTimeout(function(){E=!1},c)},reset:function(){E?nu(&quot;Page reset ignored by init&quot;):(nu(&quot;Page size reset by host page&quot;),bu(&quot;resetPage&quot;))},resize:function(){mu(&quot;resizeParent&quot;,&quot;Parent window requested size check&quot;)},moveToAnchor:function(){f.findTarget(D())},inPageLink:function(){this.moveToAnchor()},pageInfo:function(){var u=D();nu(&quot;PageInfoFromParent called from parent: &quot;+u),P(JSON.parse(u)),nu(&quot; --&quot;)},message:function(){var u=D();nu(&quot;onMessage called from parent: &quot;+u),I(JSON.parse(u)),nu(&quot; --&quot;)}};function n(){return e.data.split(&quot;]&quot;)[1].split(&quot;:&quot;)[0]}function D(){return e.data.slice(e.data.indexOf(&quot;:&quot;)+1)}function r(){return e.data.split(&quot;:&quot;)[2]in{true:1,false:1}}B===(&quot;&quot;+e.data).slice(0,m)&amp;&amp;(!1===s?function(){var D=n();D in t?t[D]():(&quot;undefined&quot;==typeof module||!module.exports)&amp;&amp;&quot;iFrameResize&quot;in window||window.jQuery!==u&amp;&amp;&quot;iFrameResize&quot;in window.jQuery.prototype||r()||Du(&quot;Unexpected message (&quot;+e.data+&quot;)&quot;)}():r()?t.init():nu(&#x27;Ignored message of type &quot;&#x27;+n()+&#x27;&quot;. Received before initialization.&#x27;))}),uu(window,&quot;readystatechange&quot;,_u),_u()}function Q(){}function uu(u,e,t,n){u.addEventListener(e,t,!!U&amp;&amp;(n||{}))}function eu(u){return u.charAt(0).toUpperCase()+u.slice(1)}function tu(u){return B+&quot;[&quot;+y+&quot;] &quot;+u}function nu(u){h&amp;&amp;&quot;object&quot;==_typeof(window.console)&amp;&amp;console.log(tu(u))}function Du(u){&quot;object&quot;==_typeof(window.console)&amp;&amp;console.warn(tu(u))}function ru(){function a(u){return&quot;true&quot;===u}function s(u,e){return&quot;function&quot;==typeof u&amp;&amp;(nu(&quot;Setup custom &quot;+e+&quot;CalcMethod&quot;),H[e]=u,u=&quot;custom&quot;),u}var l;function F(u){wu(0,0,u.type,u.screenY+&quot;:&quot;+u.screenX)}function E(u,e){nu(&quot;Add event listener: &quot;+e),uu(window.document,u,F)}l=d.slice(m).split(&quot;:&quot;),y=l[0],n=u===l[1]?n:Number(l[1]),i=u===l[2]?i:a(l[2]),h=u===l[3]?h:a(l[3]),A=u===l[4]?A:Number(l[4]),e=u===l[6]?e:a(l[6]),D=l[7],C=u===l[8]?C:l[8],t=l[9],o=l[10],x=u===l[11]?x:Number(l[11]),f.enable=u!==l[12]&amp;&amp;a(l[12]),v=u===l[13]?v:l[13],R=u===l[14]?R:l[14],p=u===l[15]?p:a(l[15]),nu(&quot;Initialising iFrame (&quot;+window.location.href+&quot;)&quot;),&quot;iFrameResizer&quot;in window&amp;&amp;Object===window.iFrameResizer.constructor&amp;&amp;(l=window.iFrameResizer,nu(&quot;Reading data from page: &quot;+JSON.stringify(l)),Object.keys(l).forEach(ou,l),I=&quot;onMessage&quot;in l?l.onMessage:I,L=&quot;onReady&quot;in l?l.onReady:L,S=&quot;targetOrigin&quot;in l?l.targetOrigin:S,C=&quot;heightCalculationMethod&quot;in l?l.heightCalculationMethod:C,R=&quot;widthCalculationMethod&quot;in l?l.widthCalculationMethod:R,C=s(C,&quot;height&quot;),R=s(R,&quot;width&quot;)),nu(&quot;TargetOrigin for parent set to: &quot;+S),iu(&quot;margin&quot;,function(u,e){return-1!==e.indexOf(&quot;-&quot;)&amp;&amp;(Du(&quot;Negative CSS value ignored for margin&quot;),e=&quot;&quot;),e}(0,D=u===D?n+&quot;px&quot;:D)),iu(&quot;background&quot;,t),iu(&quot;padding&quot;,o),(l=document.createElement(&quot;div&quot;)).style.clear=&quot;both&quot;,l.style.display=&quot;block&quot;,l.style.height=&quot;0&quot;,document.body.appendChild(l),lu(),Fu(),document.documentElement.style.height=&quot;&quot;,document.body.style.height=&quot;&quot;,nu(&#x27;HTML &amp; body height set to &quot;auto&quot;&#x27;),nu(&quot;Enable public methods&quot;),j.parentIFrame={autoResize:function(u){return!0===u&amp;&amp;!1===e?(e=!0,Cu()):!1===u&amp;&amp;!0===e&amp;&amp;(e=!1,cu(&quot;remove&quot;),null!==r&amp;&amp;r.disconnect(),clearInterval(g)),wu(0,0,&quot;autoResize&quot;,JSON.stringify(e)),e},close:function(){wu(0,0,&quot;close&quot;)},getId:function(){return y},getPageInfo:function(u){&quot;function&quot;==typeof u?(P=u,wu(0,0,&quot;pageInfo&quot;)):(P=function(){},wu(0,0,&quot;pageInfoStop&quot;))},moveToAnchor:function(u){f.findTarget(u)},reset:function(){vu(&quot;parentIFrame.reset&quot;)},scrollTo:function(u,e){wu(e,u,&quot;scrollTo&quot;)},scrollToOffset:function(u,e){wu(e,u,&quot;scrollToOffset&quot;)},sendMessage:function(u,e){wu(0,0,&quot;message&quot;,JSON.stringify(u),e)},setHeightCalculationMethod:function(u){C=u,lu()},setWidthCalculationMethod:function(u){R=u,Fu()},setTargetOrigin:function(u){nu(&quot;Set targetOrigin: &quot;+u),S=u},size:function(u,e){mu(&quot;size&quot;,&quot;parentIFrame.size(&quot;+(u||&quot;&quot;)+(e?&quot;,&quot;+e:&quot;&quot;)+&quot;)&quot;,u,e)}},!0===p&amp;&amp;(E(&quot;mouseenter&quot;,&quot;Mouse Enter&quot;),E(&quot;mouseleave&quot;,&quot;Mouse Leave&quot;)),Cu(),f=function(){function e(e){e=e.split(&quot;#&quot;)[1]||e;var t=decodeURIComponent(e);t=document.getElementById(t)||document.getElementsByName(t)[0];u===t?(nu(&quot;In page link (#&quot;+e+&quot;) not found in iFrame, so sending to parent&quot;),wu(0,0,&quot;inPageLink&quot;,&quot;#&quot;+e)):(t=function(e){e=e.getBoundingClientRect();var t={x:window.pageXOffset===u?document.documentElement.scrollLeft:window.pageXOffset,y:window.pageYOffset===u?document.documentElement.scrollTop:window.pageYOffset};return{x:parseInt(e.left,10)+parseInt(t.x,10),y:parseInt(e.top,10)+parseInt(t.y,10)}}(t=t),nu(&quot;Moving to in page link (#&quot;+e+&quot;) at x: &quot;+t.x+&quot; y: &quot;+t.y),wu(t.y,t.x,&quot;scrollToOffset&quot;))}function t(){var u=window.location.hash,t=window.location.href;&quot;&quot;!==u&amp;&amp;&quot;#&quot;!==u&amp;&amp;e(t)}return f.enable?Array.prototype.forEach&amp;&amp;document.querySelectorAll?(nu(&quot;Setting up location.hash handlers&quot;),Array.prototype.forEach.call(document.querySelectorAll(&#x27;a[href^=&quot;#&quot;]&#x27;),function(u){&quot;#&quot;!==u.getAttribute(&quot;href&quot;)&amp;&amp;uu(u,&quot;click&quot;,function(u){u.preventDefault(),e(this.getAttribute(&quot;href&quot;))})}),uu(window,&quot;hashchange&quot;,t),setTimeout(t,c)):Du(&quot;In page linking not fully supported in this browser! (See README.md for IE8 workaround)&quot;):nu(&quot;In page linking not enabled&quot;),{findTarget:e}}(),mu(&quot;init&quot;,&quot;Init message from host page&quot;),L()}function ou(u){var e=u.split(&quot;Callback&quot;);2===e.length&amp;&amp;(this[e=&quot;on&quot;+e[0].charAt(0).toUpperCase()+e[0].slice(1)]=this[u],delete this[u],Du(&quot;Deprecated: &#x27;&quot;+u+&quot;&#x27; has been renamed &#x27;&quot;+e+&quot;&#x27;. The old method will be removed in the next major version.&quot;))}function iu(e,t){u!==t&amp;&amp;&quot;&quot;!==t&amp;&amp;&quot;null&quot;!==t&amp;&amp;nu(&quot;Body &quot;+e+&#x27; set to &quot;&#x27;+(document.body.style[e]=t)+&#x27;&quot;&#x27;)}function au(u){var e={add:function(e){function t(){mu(u.eventName,u.eventType)}z[e]=t,uu(window,e,t,{passive:!0})},remove:function(u){var e=z[u];delete z[u],window.removeEventListener(u,e,!1)}};u.eventNames&amp;&amp;Array.prototype.map?(u.eventName=u.eventNames[0],u.eventNames.map(e[u.method])):e[u.method](u.eventName),nu(eu(u.method)+&quot; event listener: &quot;+u.eventType)}function cu(u){au({method:u,eventType:&quot;Animation Start&quot;,eventNames:[&quot;animationstart&quot;,&quot;webkitAnimationStart&quot;]}),au({method:u,eventType:&quot;Animation Iteration&quot;,eventNames:[&quot;animationiteration&quot;,&quot;webkitAnimationIteration&quot;]}),au({method:u,eventType:&quot;Animation End&quot;,eventNames:[&quot;animationend&quot;,&quot;webkitAnimationEnd&quot;]}),au({method:u,eventType:&quot;Input&quot;,eventName:&quot;input&quot;}),au({method:u,eventType:&quot;Mouse Up&quot;,eventName:&quot;mouseup&quot;}),au({method:u,eventType:&quot;Mouse Down&quot;,eventName:&quot;mousedown&quot;}),au({method:u,eventType:&quot;Orientation Change&quot;,eventName:&quot;orientationchange&quot;}),au({method:u,eventType:&quot;Print&quot;,eventNames:[&quot;afterprint&quot;,&quot;beforeprint&quot;]}),au({method:u,eventType:&quot;Ready State Change&quot;,eventName:&quot;readystatechange&quot;}),au({method:u,eventType:&quot;Touch Start&quot;,eventName:&quot;touchstart&quot;}),au({method:u,eventType:&quot;Touch End&quot;,eventName:&quot;touchend&quot;}),au({method:u,eventType:&quot;Touch Cancel&quot;,eventName:&quot;touchcancel&quot;}),au({method:u,eventType:&quot;Transition Start&quot;,eventNames:[&quot;transitionstart&quot;,&quot;webkitTransitionStart&quot;,&quot;MSTransitionStart&quot;,&quot;oTransitionStart&quot;,&quot;otransitionstart&quot;]}),au({method:u,eventType:&quot;Transition Iteration&quot;,eventNames:[&quot;transitioniteration&quot;,&quot;webkitTransitionIteration&quot;,&quot;MSTransitionIteration&quot;,&quot;oTransitionIteration&quot;,&quot;otransitioniteration&quot;]}),au({method:u,eventType:&quot;Transition End&quot;,eventNames:[&quot;transitionend&quot;,&quot;webkitTransitionEnd&quot;,&quot;MSTransitionEnd&quot;,&quot;oTransitionEnd&quot;,&quot;otransitionend&quot;]}),&quot;child&quot;===v&amp;&amp;au({method:u,eventType:&quot;IFrame Resized&quot;,eventName:&quot;resize&quot;})}function su(u,e,t,n){return e!==u&amp;&amp;(u in t||(Du(u+&quot; is not a valid option for &quot;+n+&quot;CalculationMethod.&quot;),u=e),nu(n+&#x27; calculation method set to &quot;&#x27;+u+&#x27;&quot;&#x27;)),u}function lu(){C=su(C,F,$,&quot;height&quot;)}function Fu(){R=su(R,M,V,&quot;width&quot;)}function Cu(){var u;!0===e?(cu(&quot;add&quot;),u=A&lt;0,window.MutationObserver||window.WebKitMutationObserver?u?Eu():r=function(){function u(u){function e(u){!1===u.complete&amp;&amp;(nu(&quot;Attach listeners to &quot;+u.src),u.addEventListener(&quot;load&quot;,n,!1),u.addEventListener(&quot;error&quot;,D,!1),i.push(u))}&quot;attributes&quot;===u.type&amp;&amp;&quot;src&quot;===u.attributeName?e(u.target):&quot;childList&quot;===u.type&amp;&amp;Array.prototype.forEach.call(u.target.querySelectorAll(&quot;img&quot;),e)}function e(u){nu(&quot;Remove listeners from &quot;+u.src),u.removeEventListener(&quot;load&quot;,n,!1),u.removeEventListener(&quot;error&quot;,D,!1),i.splice(i.indexOf(u),1)}function t(u,t,n){e(u.target),mu(t,n+&quot;: &quot;+u.target.src)}function n(u){t(u,&quot;imageLoad&quot;,&quot;Image loaded&quot;)}function D(u){t(u,&quot;imageLoadFailed&quot;,&quot;Image load failed&quot;)}function r(e){mu(&quot;mutationObserver&quot;,&quot;mutationObserver: &quot;+e[0].target+&quot; &quot;+e[0].type),e.forEach(u)}var o,i=[],a=window.MutationObserver||window.WebKitMutationObserver,c=(o=document.querySelector(&quot;body&quot;),c=new a(r),nu(&quot;Create body MutationObserver&quot;),c.observe(o,{attributes:!0,attributeOldValue:!1,characterData:!0,characterDataOldValue:!1,childList:!0,subtree:!0}),c);return{disconnect:function(){&quot;disconnect&quot;in c&amp;&amp;(nu(&quot;Disconnect body MutationObserver&quot;),c.disconnect(),i.forEach(e))}}}():(nu(&quot;MutationObserver not supported in this browser!&quot;),Eu())):nu(&quot;Auto Resize disabled&quot;)}function Eu(){0!==A&amp;&amp;(nu(&quot;setInterval: &quot;+A+&quot;ms&quot;),g=setInterval(function(){mu(&quot;interval&quot;,&quot;setInterval: &quot;+A)},Math.abs(A)))}function du(u,e){return e=e||document.body,e=null===(e=document.defaultView.getComputedStyle(e,null))?0:e[u],parseInt(e,10)}function fu(u,e){for(var t,n=e.length,D=0,r=eu(u),o=Date.now(),i=0;i&lt;n;i++)D&lt;(t=e[i].getBoundingClientRect()[u]+du(&quot;margin&quot;+r,e[i]))&amp;&amp;(D=t);return o=Date.now()-o,nu(&quot;Parsed &quot;+n+&quot; HTML elements&quot;),nu(&quot;Element position calculated in &quot;+o+&quot;ms&quot;),T/2&lt;(o=o)&amp;&amp;nu(&quot;Event throttle increased to &quot;+(T=2*o)+&quot;ms&quot;),D}function Au(u){return[u.bodyOffset(),u.bodyScroll(),u.documentElementOffset(),u.documentElementScroll()]}function gu(u,e){var t=document.querySelectorAll(&quot;[&quot;+e+&quot;]&quot;);return 0===t.length&amp;&amp;(Du(&quot;No tagged elements (&quot;+e+&quot;) found on page&quot;),document.querySelectorAll(&quot;body *&quot;)),fu(u,t)}function hu(){return document.querySelectorAll(&quot;body *&quot;)}function pu(e,t,n,D){function r(u,e){return!(Math.abs(u-e)&lt;=x)}n=u===n?$[C]():n,D=u===D?V[R]():D,r(l,n)||i&amp;&amp;r(N,D)||&quot;init&quot;===e?(yu(),wu(l=n,N=D,e)):e in{init:1,interval:1,size:1}||!(C in b||i&amp;&amp;R in b)?e in{interval:1}||nu(&quot;No change in size detected&quot;):vu(t)}function Bu(){Z=Date.now(),J=null,G=W.apply(q,X),J||(q=X=null)}function mu(u,e,t,n){k&amp;&amp;u in a?nu(&quot;Trigger event cancelled: &quot;+u):(u in{reset:1,resetPage:1,init:1}||nu(&quot;Trigger event: &quot;+e),(&quot;init&quot;===u?pu:Y)(u,e,t,n))}function yu(){k||(k=!0,nu(&quot;Trigger event lock on&quot;)),clearTimeout(O),O=setTimeout(function(){k=!1,nu(&quot;Trigger event lock off&quot;),nu(&quot;--&quot;)},c)}function bu(u){l=$[C](),N=V[R](),wu(l,N,u)}function vu(u){var e=C;C=F,nu(&quot;Reset trigger event: &quot;+u),yu(),bu(&quot;reset&quot;),C=e}function wu(e,t,n,D,r){!0===w&amp;&amp;(u===r?r=S:nu(&quot;Message targetOrigin: &quot;+r),nu(&quot;Sending message to host page (&quot;+(e=y+&quot;:&quot;+e+&quot;:&quot;+t+&quot;:&quot;+n+(u===D?&quot;&quot;:&quot;:&quot;+D))+&quot;)&quot;),_.postMessage(B+e,r))}function _u(){&quot;loading&quot;!==document.readyState&amp;&amp;window.parent.postMessage(&quot;[iFrameResizerChild]Ready&quot;,&quot;*&quot;)}}();var hljs=function(){var u={exports:{}};function e(u){return u instanceof Map?u.clear=u.delete=u.set=function(){throw Error(&quot;map is read-only&quot;)}:u instanceof Set&amp;&amp;(u.add=u.clear=u.delete=function(){throw Error(&quot;set is read-only&quot;)}),Object.freeze(u),Object.getOwnPropertyNames(u).forEach(function(t){var n=u[t];&quot;object&quot;!=_typeof(n)||Object.isFrozen(n)||e(n)}),u}u.exports=e,u.exports.default=e;var t=function(){function u(e){_classCallCheck(this,u),void 0===e.data&amp;&amp;(e.data={}),this.data=e.data,this.isMatchIgnored=!1}return _createClass(u,[{key:&quot;ignoreMatch&quot;,value:function(){this.isMatchIgnored=!0}}]),u}();function n(u){return u.replace(/&amp;/g,&quot;&amp;amp;&quot;).replace(/&lt;/g,&quot;&amp;lt;&quot;).replace(/&gt;/g,&quot;&amp;gt;&quot;).replace(/&quot;/g,&quot;&amp;quot;&quot;).replace(/&#x27;/g,&quot;&amp;#x27;&quot;)}function D(u){var e=Object.create(null);for(var t in u)e[t]=u[t];for(var n=arguments.length,D=new Array(n&gt;1?n-1:0),r=1;r&lt;n;r++)D[r-1]=arguments[r];return D.forEach(function(u){for(var t in u)e[t]=u[t]}),e}var r=function(u){return!!u.scope||u.sublanguage&amp;&amp;u.language},o=function(){function u(e,t){_classCallCheck(this,u),this.buffer=&quot;&quot;,this.classPrefix=t.classPrefix,e.walk(this)}return _createClass(u,[{key:&quot;addText&quot;,value:function(u){this.buffer+=n(u)}},{key:&quot;openNode&quot;,value:function(u){if(r(u)){var e;e=u.sublanguage?&quot;language-&quot;+u.language:function(u,e){var t=e.prefix;if(u.includes(&quot;.&quot;)){var n=u.split(&quot;.&quot;);return[&quot;&quot;.concat(t).concat(n.shift())].concat(_toConsumableArray(n.map(function(u,e){return&quot;&quot;.concat(u).concat(&quot;_&quot;.repeat(e+1))}))).join(&quot; &quot;)}return&quot;&quot;.concat(t).concat(u)}(u.scope,{prefix:this.classPrefix}),this.span(e)}}},{key:&quot;closeNode&quot;,value:function(u){r(u)&amp;&amp;(this.buffer+=&quot;&lt;/span&gt;&quot;)}},{key:&quot;value&quot;,value:function(){return this.buffer}},{key:&quot;span&quot;,value:function(u){this.buffer+=&#x27;&lt;span class=&quot;&#x27;.concat(u,&#x27;&quot;&gt;&#x27;)}}]),u}(),i=function(){var u=arguments.length&gt;0&amp;&amp;void 0!==arguments[0]?arguments[0]:{},e={children:[]};return Object.assign(e,u),e},a=function(){function u(){_classCallCheck(this,u),this.rootNode=i(),this.stack=[this.rootNode]}return _createClass(u,[{key:&quot;top&quot;,get:function(){return this.stack[this.stack.length-1]}},{key:&quot;root&quot;,get:function(){return this.rootNode}},{key:&quot;add&quot;,value:function(u){this.top.children.push(u)}},{key:&quot;openNode&quot;,value:function(u){var e=i({scope:u});this.add(e),this.stack.push(e)}},{key:&quot;closeNode&quot;,value:function(){if(this.stack.length&gt;1)return this.stack.pop()}},{key:&quot;closeAllNodes&quot;,value:function(){for(;this.closeNode(););}},{key:&quot;toJSON&quot;,value:function(){return JSON.stringify(this.rootNode,null,4)}},{key:&quot;walk&quot;,value:function(u){return this.constructor._walk(u,this.rootNode)}}],[{key:&quot;_walk&quot;,value:function(u,e){var t=this;return&quot;string&quot;==typeof e?u.addText(e):e.children&amp;&amp;(u.openNode(e),e.children.forEach(function(e){return t._walk(u,e)}),u.closeNode(e)),u}},{key:&quot;_collapse&quot;,value:function(e){&quot;string&quot;!=typeof e&amp;&amp;e.children&amp;&amp;(e.children.every(function(u){return&quot;string&quot;==typeof u})?e.children=[e.children.join(&quot;&quot;)]:e.children.forEach(function(e){u._collapse(e)}))}}]),u}(),c=function(u){_inherits(t,a);var e=_createSuper(t);function t(u){var n;return _classCallCheck(this,t),(n=e.call(this)).options=u,n}return _createClass(t,[{key:&quot;addKeyword&quot;,value:function(u,e){&quot;&quot;!==u&amp;&amp;(this.openNode(e),this.addText(u),this.closeNode())}},{key:&quot;addText&quot;,value:function(u){&quot;&quot;!==u&amp;&amp;this.add(u)}},{key:&quot;addSublanguage&quot;,value:function(u,e){var t=u.root;t.sublanguage=!0,t.language=e,this.add(t)}},{key:&quot;toHTML&quot;,value:function(){return new o(this,this.options).value()}},{key:&quot;finalize&quot;,value:function(){return!0}}]),t}();function s(u){return u?&quot;string&quot;==typeof u?u:u.source:null}function l(u){return E(&quot;(?=&quot;,u,&quot;)&quot;)}function F(u){return E(&quot;(?:&quot;,u,&quot;)*&quot;)}function C(u){return E(&quot;(?:&quot;,u,&quot;)?&quot;)}function E(){for(var u=arguments.length,e=new Array(u),t=0;t&lt;u;t++)e[t]=arguments[t];return e.map(function(u){return s(u)}).join(&quot;&quot;)}function d(){for(var u=arguments.length,e=new Array(u),t=0;t&lt;u;t++)e[t]=arguments[t];return&quot;(&quot;+(function(u){var e=u[u.length-1];return&quot;object&quot;==_typeof(e)&amp;&amp;e.constructor===Object?(u.splice(u.length-1,1),e):{}}(e).capture?&quot;&quot;:&quot;?:&quot;)+e.map(function(u){return s(u)}).join(&quot;|&quot;)+&quot;)&quot;}function f(u){return RegExp(u.toString()+&quot;|&quot;).exec(&quot;&quot;).length-1}var A=/\\[(?:[^\\\\\\]]|\\\\.)*\\]|\\(\\??|\\\\([1-9][0-9]*)|\\\\./;function g(u,e){var t=e.joinWith,n=0;return u.map(function(u){for(var e=n+=1,t=s(u),D=&quot;&quot;;t.length&gt;0;){var r=A.exec(t);if(!r){D+=t;break}D+=t.substring(0,r.index),t=t.substring(r.index+r[0].length),&quot;\\\\&quot;===r[0][0]&amp;&amp;r[1]?D+=&quot;\\\\&quot;+(Number(r[1])+e):(D+=r[0],&quot;(&quot;===r[0]&amp;&amp;n++)}return D}).map(function(u){return&quot;(&quot;.concat(u,&quot;)&quot;)}).join(t)}var h=&quot;[a-zA-Z]\\\\w*&quot;,p=&quot;[a-zA-Z_]\\\\w*&quot;,B=&quot;\\\\b\\\\d+(\\\\.\\\\d+)?&quot;,m=&quot;(-?)(\\\\b0[xX][a-fA-F0-9]+|(\\\\b\\\\d+(\\\\.\\\\d*)?|\\\\.\\\\d+)([eE][-+]?\\\\d+)?)&quot;,y=&quot;\\\\b(0b[01]+)&quot;,b={begin:&quot;\\\\\\\\[\\\\s\\\\S]&quot;,relevance:0},v={scope:&quot;string&quot;,begin:&quot;&#x27;&quot;,end:&quot;&#x27;&quot;,illegal:&quot;\\\\n&quot;,contains:[b]},w={scope:&quot;string&quot;,begin:&#x27;&quot;&#x27;,end:&#x27;&quot;&#x27;,illegal:&quot;\\\\n&quot;,contains:[b]},_=function(u,e){var t=D({scope:&quot;comment&quot;,begin:u,end:e,contains:[]},arguments.length&gt;2&amp;&amp;void 0!==arguments[2]?arguments[2]:{});t.contains.push({scope:&quot;doctag&quot;,begin:&quot;[ ]*(?=(TODO|FIXME|NOTE|BUG|OPTIMIZE|HACK|XXX):)&quot;,end:/(TODO|FIXME|NOTE|BUG|OPTIMIZE|HACK|XXX):/,excludeBegin:!0,relevance:0});var n=d(&quot;I&quot;,&quot;a&quot;,&quot;is&quot;,&quot;so&quot;,&quot;us&quot;,&quot;to&quot;,&quot;at&quot;,&quot;if&quot;,&quot;in&quot;,&quot;it&quot;,&quot;on&quot;,/[A-Za-z]+[&#x27;](d|ve|re|ll|t|s|n)/,/[A-Za-z]+[-][a-z]+/,/[A-Za-z][a-z]{2,}/);return t.contains.push({begin:E(/[ ]+/,&quot;(&quot;,n,/[.]?[:]?([.][ ]|[ ])/,&quot;){3}&quot;)}),t},S=_(&quot;//&quot;,&quot;$&quot;),x=_(&quot;/\\\\*&quot;,&quot;\\\\*/&quot;),k=_(&quot;#&quot;,&quot;$&quot;),O=Object.freeze({__proto__:null,MATCH_NOTHING_RE:/\\b\\B/,IDENT_RE:h,UNDERSCORE_IDENT_RE:p,NUMBER_RE:B,C_NUMBER_RE:m,BINARY_NUMBER_RE:y,RE_STARTERS_RE:&quot;!|!=|!==|%|%=|&amp;|&amp;&amp;|&amp;=|\\\\*|\\\\*=|\\\\+|\\\\+=|,|-|-=|/=|/|:|;|&lt;&lt;|&lt;&lt;=|&lt;=|&lt;|===|==|=|&gt;&gt;&gt;=|&gt;&gt;=|&gt;=|&gt;&gt;&gt;|&gt;&gt;|&gt;|\\\\?|\\\\[|\\\\{|\\\\(|\\\\^|\\\\^=|\\\\||\\\\|=|\\\\|\\\\||~&quot;,SHEBANG:function(){var u=arguments.length&gt;0&amp;&amp;void 0!==arguments[0]?arguments[0]:{},e=/^#![ ]*\\//;return u.binary&amp;&amp;(u.begin=E(e,/.*\\b/,u.binary,/\\b.*/)),D({scope:&quot;meta&quot;,begin:e,end:/$/,relevance:0,&quot;on:begin&quot;:function(u,e){0!==u.index&amp;&amp;e.ignoreMatch()}},u)},BACKSLASH_ESCAPE:b,APOS_STRING_MODE:v,QUOTE_STRING_MODE:w,PHRASAL_WORDS_MODE:{begin:/\\b(a|an|the|are|I&#x27;m|isn&#x27;t|don&#x27;t|doesn&#x27;t|won&#x27;t|but|just|should|pretty|simply|enough|gonna|going|wtf|so|such|will|you|your|they|like|more)\\b/},COMMENT:_,C_LINE_COMMENT_MODE:S,C_BLOCK_COMMENT_MODE:x,HASH_COMMENT_MODE:k,NUMBER_MODE:{scope:&quot;number&quot;,begin:B,relevance:0},C_NUMBER_MODE:{scope:&quot;number&quot;,begin:m,relevance:0},BINARY_NUMBER_MODE:{scope:&quot;number&quot;,begin:y,relevance:0},REGEXP_MODE:{begin:/(?=\\/[^/\\n]*\\/)/,contains:[{scope:&quot;regexp&quot;,begin:/\\//,end:/\\/[gimuy]*/,illegal:/\\n/,contains:[b,{begin:/\\[/,end:/\\]/,relevance:0,contains:[b]}]}]},TITLE_MODE:{scope:&quot;title&quot;,begin:h,relevance:0},UNDERSCORE_TITLE_MODE:{scope:&quot;title&quot;,begin:p,relevance:0},METHOD_GUARD:{begin:&quot;\\\\.\\\\s*[a-zA-Z_]\\\\w*&quot;,relevance:0},END_SAME_AS_BEGIN:function(u){return Object.assign(u,{&quot;on:begin&quot;:function(u,e){e.data._beginMatch=u[1]},&quot;on:end&quot;:function(u,e){e.data._beginMatch!==u[1]&amp;&amp;e.ignoreMatch()}})}});function T(u,e){&quot;.&quot;===u.input[u.index-1]&amp;&amp;e.ignoreMatch()}function N(u,e){void 0!==u.className&amp;&amp;(u.scope=u.className,delete u.className)}function M(u,e){e&amp;&amp;u.beginKeywords&amp;&amp;(u.begin=&quot;\\\\b(&quot;+u.beginKeywords.split(&quot; &quot;).join(&quot;|&quot;)+&quot;)(?!\\\\.)(?=\\\\b|\\\\s)&quot;,u.__beforeBegin=T,u.keywords=u.keywords||u.beginKeywords,delete u.beginKeywords,void 0===u.relevance&amp;&amp;(u.relevance=0))}function R(u,e){Array.isArray(u.illegal)&amp;&amp;(u.illegal=d.apply(void 0,_toConsumableArray(u.illegal)))}function j(u,e){if(u.match){if(u.begin||u.end)throw Error(&quot;begin &amp; end are not supported with match&quot;);u.begin=u.match,delete u.match}}function I(u,e){void 0===u.relevance&amp;&amp;(u.relevance=1)}var L=function(u,e){if(u.beforeMatch){if(u.starts)throw Error(&quot;beforeMatch cannot be used with starts&quot;);var t=Object.assign({},u);Object.keys(u).forEach(function(e){delete u[e]}),u.keywords=t.keywords,u.begin=E(t.beforeMatch,l(t.begin)),u.starts={relevance:0,contains:[Object.assign(t,{endsParent:!0})]},u.relevance=0,delete t.beforeMatch}},P=[&quot;of&quot;,&quot;and&quot;,&quot;for&quot;,&quot;in&quot;,&quot;not&quot;,&quot;or&quot;,&quot;if&quot;,&quot;then&quot;,&quot;parent&quot;,&quot;list&quot;,&quot;value&quot;];function H(u,e){return e?Number(e):function(u){return P.includes(u.toLowerCase())}(u)?0:1}var z={},U=function(u){console.error(u)},K=function(u){for(var e,t=arguments.length,n=new Array(t&gt;1?t-1:0),D=1;D&lt;t;D++)n[D-1]=arguments[D];(e=console).log.apply(e,[&quot;WARN: &quot;+u].concat(n))},W=function(u,e){z[&quot;&quot;.concat(u,&quot;/&quot;).concat(e)]||(console.log(&quot;Deprecated as of &quot;.concat(u,&quot;. &quot;).concat(e)),z[&quot;&quot;.concat(u,&quot;/&quot;).concat(e)]=!0)},q=Error();function X(u,e,t){for(var n=t.key,D=0,r=u[n],o={},i={},a=1;a&lt;=e.length;a++)i[a+D]=r[a],o[a+D]=!0,D+=f(e[a-1]);u[n]=i,u[n]._emit=o,u[n]._multi=!0}function G(u){(function(u){u.scope&amp;&amp;&quot;object&quot;==_typeof(u.scope)&amp;&amp;null!==u.scope&amp;&amp;(u.beginScope=u.scope,delete u.scope)})(u),&quot;string&quot;==typeof u.beginScope&amp;&amp;(u.beginScope={_wrap:u.beginScope}),&quot;string&quot;==typeof u.endScope&amp;&amp;(u.endScope={_wrap:u.endScope}),function(u){if(Array.isArray(u.begin)){if(u.skip||u.excludeBegin||u.returnBegin)throw U(&quot;skip, excludeBegin, returnBegin not compatible with beginScope: {}&quot;),q;if(&quot;object&quot;!=_typeof(u.beginScope)||null===u.beginScope)throw U(&quot;beginScope must be object&quot;),q;X(u,u.begin,{key:&quot;beginScope&quot;}),u.begin=g(u.begin,{joinWith:&quot;&quot;})}}(u),function(u){if(Array.isArray(u.end)){if(u.skip||u.excludeEnd||u.returnEnd)throw U(&quot;skip, excludeEnd, returnEnd not compatible with endScope: {}&quot;),q;if(&quot;object&quot;!=_typeof(u.endScope)||null===u.endScope)throw U(&quot;endScope must be object&quot;),q;X(u,u.end,{key:&quot;endScope&quot;}),u.end=g(u.end,{joinWith:&quot;&quot;})}}(u)}function J(u){function e(e,t){return RegExp(s(e),&quot;m&quot;+(u.case_insensitive?&quot;i&quot;:&quot;&quot;)+(u.unicodeRegex?&quot;u&quot;:&quot;&quot;)+(t?&quot;g&quot;:&quot;&quot;))}var t=function(){function u(){_classCallCheck(this,u),this.matchIndexes={},this.regexes=[],this.matchAt=1,this.position=0}return _createClass(u,[{key:&quot;addRule&quot;,value:function(u,e){e.position=this.position++,this.matchIndexes[this.matchAt]=e,this.regexes.push([e,u]),this.matchAt+=f(u)+1}},{key:&quot;compile&quot;,value:function(){0===this.regexes.length&amp;&amp;(this.exec=function(){return null});var u=this.regexes.map(function(u){return u[1]});this.matcherRe=e(g(u,{joinWith:&quot;|&quot;}),!0),this.lastIndex=0}},{key:&quot;exec&quot;,value:function(u){this.matcherRe.lastIndex=this.lastIndex;var e=this.matcherRe.exec(u);if(!e)return null;var t=e.findIndex(function(u,e){return e&gt;0&amp;&amp;void 0!==u}),n=this.matchIndexes[t];return e.splice(0,t),Object.assign(e,n)}}]),u}(),n=function(){function u(){_classCallCheck(this,u),this.rules=[],this.multiRegexes=[],this.count=0,this.lastIndex=0,this.regexIndex=0}return _createClass(u,[{key:&quot;getMatcher&quot;,value:function(u){if(this.multiRegexes[u])return this.multiRegexes[u];var e=new t;return this.rules.slice(u).forEach(function(u){var t=_slicedToArray(u,2),n=t[0],D=t[1];return e.addRule(n,D)}),e.compile(),this.multiRegexes[u]=e,e}},{key:&quot;resumingScanAtSamePosition&quot;,value:function(){return 0!==this.regexIndex}},{key:&quot;considerAll&quot;,value:function(){this.regexIndex=0}},{key:&quot;addRule&quot;,value:function(u,e){this.rules.push([u,e]),&quot;begin&quot;===e.type&amp;&amp;this.count++}},{key:&quot;exec&quot;,value:function(u){var e=this.getMatcher(this.regexIndex);e.lastIndex=this.lastIndex;var t=e.exec(u);if(this.resumingScanAtSamePosition())if(t&amp;&amp;t.index===this.lastIndex);else{var n=this.getMatcher(0);n.lastIndex=this.lastIndex+1,t=n.exec(u)}return t&amp;&amp;(this.regexIndex+=t.position+1,this.regexIndex===this.count&amp;&amp;this.considerAll()),t}}]),u}();if(u.compilerExtensions||(u.compilerExtensions=[]),u.contains&amp;&amp;u.contains.includes(&quot;self&quot;))throw Error(&quot;ERR: contains `self` is not supported at the top-level of a language.  See documentation.&quot;);return u.classNameAliases=D(u.classNameAliases||{}),function t(r,o){var i,a=r;if(r.isCompiled)return a;[N,j,G,L].forEach(function(u){return u(r,o)}),u.compilerExtensions.forEach(function(u){return u(r,o)}),r.__beforeBegin=null,[M,R,I].forEach(function(u){return u(r,o)}),r.isCompiled=!0;var c=null;return&quot;object&quot;==_typeof(r.keywords)&amp;&amp;r.keywords.$pattern&amp;&amp;(r.keywords=Object.assign({},r.keywords),c=r.keywords.$pattern,delete r.keywords.$pattern),c=c||/\\w+/,r.keywords&amp;&amp;(r.keywords=function u(e,t){var n=arguments.length&gt;2&amp;&amp;void 0!==arguments[2]?arguments[2]:&quot;keyword&quot;,D=Object.create(null);return&quot;string&quot;==typeof e?r(n,e.split(&quot; &quot;)):Array.isArray(e)?r(n,e):Object.keys(e).forEach(function(n){Object.assign(D,u(e[n],t,n))}),D;function r(u,e){t&amp;&amp;(e=e.map(function(u){return u.toLowerCase()})),e.forEach(function(e){var t=e.split(&quot;|&quot;);D[t[0]]=[u,H(t[0],t[1])]})}}(r.keywords,u.case_insensitive)),a.keywordPatternRe=e(c,!0),o&amp;&amp;(r.begin||(r.begin=/\\B|\\b/),a.beginRe=e(a.begin),r.end||r.endsWithParent||(r.end=/\\B|\\b/),r.end&amp;&amp;(a.endRe=e(a.end)),a.terminatorEnd=s(a.end)||&quot;&quot;,r.endsWithParent&amp;&amp;o.terminatorEnd&amp;&amp;(a.terminatorEnd+=(r.end?&quot;|&quot;:&quot;&quot;)+o.terminatorEnd)),r.illegal&amp;&amp;(a.illegalRe=e(r.illegal)),r.contains||(r.contains=[]),r.contains=(i=[]).concat.apply(i,_toConsumableArray(r.contains.map(function(u){return function(u){return u.variants&amp;&amp;!u.cachedVariants&amp;&amp;(u.cachedVariants=u.variants.map(function(e){return D(u,{variants:null},e)})),u.cachedVariants?u.cachedVariants:function u(e){return!!e&amp;&amp;(e.endsWithParent||u(e.starts))}(u)?D(u,{starts:u.starts?D(u.starts):null}):Object.isFrozen(u)?D(u):u}(&quot;self&quot;===u?r:u)}))),r.contains.forEach(function(u){t(u,a)}),r.starts&amp;&amp;t(r.starts,o),a.matcher=function(u){var e=new n;return u.contains.forEach(function(u){return e.addRule(u.begin,{rule:u,type:&quot;begin&quot;})}),u.terminatorEnd&amp;&amp;e.addRule(u.terminatorEnd,{type:&quot;end&quot;}),u.illegal&amp;&amp;e.addRule(u.illegal,{type:&quot;illegal&quot;}),e}(a),a}(u)}var Z=function(u){_inherits(t,_wrapNativeSuper(Error));var e=_createSuper(t);function t(u,n){var D;return _classCallCheck(this,t),(D=e.call(this,u)).name=&quot;HTMLInjectionError&quot;,D.html=n,D}return _createClass(t)}(),$=n,V=D,Y=Symbol(&quot;nomatch&quot;);return function(e){var n=Object.create(null),D=Object.create(null),r=[],o=!0,i=&quot;Could not find the language &#x27;{}&#x27;, did you forget to load/include a language module?&quot;,a={disableAutodetect:!0,name:&quot;Plain text&quot;,contains:[]},s={ignoreUnescapedHTML:!1,throwUnescapedHTML:!1,noHighlightRe:/^(no-?highlight)$/i,languageDetectRe:/\\blang(?:uage)?-([\\w-]+)\\b/i,classPrefix:&quot;hljs-&quot;,cssSelector:&quot;pre code&quot;,languages:null,__emitter:c};function f(u){return s.noHighlightRe.test(u)}function A(u,e,t){var n=&quot;&quot;,D=&quot;&quot;;&quot;object&quot;==_typeof(e)?(n=u,t=e.ignoreIllegals,D=e.language):(W(&quot;10.7.0&quot;,&quot;highlight(lang, code, ...args) has been deprecated.&quot;),W(&quot;10.7.0&quot;,&quot;Please use highlight(code, options) instead.\\nhttps://github.com/highlightjs/highlight.js/issues/2277&quot;),D=u,n=e),void 0===t&amp;&amp;(t=!0);var r={code:n,language:D};w(&quot;before:highlight&quot;,r);var o=r.result?r.result:g(r.language,r.code,t);return o.code=r.code,w(&quot;after:highlight&quot;,o),o}function g(u,e,D,r){var a=Object.create(null);function c(){if(b.keywords){var u=0;b.keywordPatternRe.lastIndex=0;for(var e,t=b.keywordPatternRe.exec(_),n=&quot;&quot;;t;){n+=_.substring(u,t.index);var D=p.case_insensitive?t[0].toLowerCase():t[0],r=(e=D,b.keywords[e]);if(r){var o=_slicedToArray(r,2),i=o[0],c=o[1];if(w.addText(n),n=&quot;&quot;,a[D]=(a[D]||0)+1,a[D]&lt;=7&amp;&amp;(S+=c),i.startsWith(&quot;_&quot;))n+=t[0];else{var s=p.classNameAliases[i]||i;w.addKeyword(t[0],s)}}else n+=t[0];u=b.keywordPatternRe.lastIndex,t=b.keywordPatternRe.exec(_)}n+=_.substring(u),w.addText(n)}else w.addText(_)}function l(){null!=b.subLanguage?function(){if(&quot;&quot;!==_){var u=null;if(&quot;string&quot;==typeof b.subLanguage){if(!n[b.subLanguage])return void w.addText(_);u=g(b.subLanguage,_,!0,v[b.subLanguage]),v[b.subLanguage]=u._top}else u=h(_,b.subLanguage.length?b.subLanguage:null);b.relevance&gt;0&amp;&amp;(S+=u.relevance),w.addSublanguage(u._emitter,u.language)}}():c(),_=&quot;&quot;}function F(u,e){for(var t=1,n=e.length-1;t&lt;=n;)if(u._emit[t]){var D=p.classNameAliases[u[t]]||u[t],r=e[t];D?w.addKeyword(r,D):(_=r,c(),_=&quot;&quot;),t++}else t++}function C(u,e){return u.scope&amp;&amp;&quot;string&quot;==typeof u.scope&amp;&amp;w.openNode(p.classNameAliases[u.scope]||u.scope),u.beginScope&amp;&amp;(u.beginScope._wrap?(w.addKeyword(_,p.classNameAliases[u.beginScope._wrap]||u.beginScope._wrap),_=&quot;&quot;):u.beginScope._multi&amp;&amp;(F(u.beginScope,e),_=&quot;&quot;)),b=Object.create(u,{parent:{value:b}})}function E(u,e,n){var D=function(u,e){var t=u&amp;&amp;u.exec(e);return t&amp;&amp;0===t.index}(u.endRe,n);if(D){if(u[&quot;on:end&quot;]){var r=new t(u);u[&quot;on:end&quot;](e,r),r.isMatchIgnored&amp;&amp;(D=!1)}if(D){for(;u.endsParent&amp;&amp;u.parent;)u=u.parent;return u}}if(u.endsWithParent)return E(u.parent,e,n)}function d(u){return 0===b.matcher.regexIndex?(_+=u[0],1):(O=!0,0)}var f={};function A(n,r){var i=r&amp;&amp;r[0];if(_+=n,null==i)return l(),0;if(&quot;begin&quot;===f.type&amp;&amp;&quot;end&quot;===r.type&amp;&amp;f.index===r.index&amp;&amp;&quot;&quot;===i){if(_+=e.slice(r.index,r.index+1),!o){var a=Error(&quot;0 width match regex (&quot;.concat(u,&quot;)&quot;));throw a.languageName=u,a.badRule=f.rule,a}return 1}if(f=r,&quot;begin&quot;===r.type)return function(u){for(var e=u[0],n=u.rule,D=new t(n),r=0,o=[n.__beforeBegin,n[&quot;on:begin&quot;]];r&lt;o.length;r++){var i=o[r];if(i&amp;&amp;(i(u,D),D.isMatchIgnored))return d(e)}return n.skip?_+=e:(n.excludeBegin&amp;&amp;(_+=e),l(),n.returnBegin||n.excludeBegin||(_=e)),C(n,u),n.returnBegin?0:e.length}(r);if(&quot;illegal&quot;===r.type&amp;&amp;!D){var c=Error(&#x27;Illegal lexeme &quot;&#x27;+i+&#x27;&quot; for mode &quot;&#x27;+(b.scope||&quot;&lt;unnamed&gt;&quot;)+&#x27;&quot;&#x27;);throw c.mode=b,c}if(&quot;end&quot;===r.type){var s=function(u){var t=u[0],n=e.substring(u.index),D=E(b,u,n);if(!D)return Y;var r=b;b.endScope&amp;&amp;b.endScope._wrap?(l(),w.addKeyword(t,b.endScope._wrap)):b.endScope&amp;&amp;b.endScope._multi?(l(),F(b.endScope,u)):r.skip?_+=t:(r.returnEnd||r.excludeEnd||(_+=t),l(),r.excludeEnd&amp;&amp;(_=t));do{b.scope&amp;&amp;w.closeNode(),b.skip||b.subLanguage||(S+=b.relevance),b=b.parent}while(b!==D.parent);return D.starts&amp;&amp;C(D.starts,u),r.returnEnd?0:t.length}(r);if(s!==Y)return s}if(&quot;illegal&quot;===r.type&amp;&amp;&quot;&quot;===i)return 1;if(k&gt;1e5&amp;&amp;k&gt;3*r.index)throw Error(&quot;potential infinite loop, way more iterations than matches&quot;);return _+=i,i.length}var p=y(u);if(!p)throw U(i.replace(&quot;{}&quot;,u)),Error(&#x27;Unknown language: &quot;&#x27;+u+&#x27;&quot;&#x27;);var B=J(p),m=&quot;&quot;,b=r||B,v={},w=new s.__emitter(s);!function(){for(var u=[],e=b;e!==p;e=e.parent)e.scope&amp;&amp;u.unshift(e.scope);u.forEach(function(u){return w.openNode(u)})}();var _=&quot;&quot;,S=0,x=0,k=0,O=!1;try{for(b.matcher.considerAll();;){k++,O?O=!1:b.matcher.considerAll(),b.matcher.lastIndex=x;var T=b.matcher.exec(e);if(!T)break;var N=A(e.substring(x,T.index),T);x=T.index+N}return A(e.substring(x)),w.closeAllNodes(),w.finalize(),m=w.toHTML(),{language:u,value:m,relevance:S,illegal:!1,_emitter:w,_top:b}}catch(t){if(t.message&amp;&amp;t.message.includes(&quot;Illegal&quot;))return{language:u,value:$(e),illegal:!0,relevance:0,_illegalBy:{message:t.message,index:x,context:e.slice(x-100,x+100),mode:t.mode,resultSoFar:m},_emitter:w};if(o)return{language:u,value:$(e),illegal:!1,relevance:0,errorRaised:t,_emitter:w,_top:b};throw t}}function h(u,e){e=e||s.languages||Object.keys(n);var t=function(u){var e={value:$(u),illegal:!1,relevance:0,_top:a,_emitter:new s.__emitter(s)};return e._emitter.addText(u),e}(u),D=e.filter(y).filter(v).map(function(e){return g(e,u,!1)});D.unshift(t);var r=_slicedToArray(D.sort(function(u,e){if(u.relevance!==e.relevance)return e.relevance-u.relevance;if(u.language&amp;&amp;e.language){if(y(u.language).supersetOf===e.language)return 1;if(y(e.language).supersetOf===u.language)return-1}return 0}),2),o=r[0],i=r[1],c=o;return c.secondBest=i,c}function p(u){var e=function(u){var e=u.className+&quot; &quot;;e+=u.parentNode?u.parentNode.className:&quot;&quot;;var t=s.languageDetectRe.exec(e);if(t){var n=y(t[1]);return n||(K(i.replace(&quot;{}&quot;,t[1])),K(&quot;Falling back to no-highlight mode for this block.&quot;,u)),n?t[1]:&quot;no-highlight&quot;}return e.split(/\\s+/).find(function(u){return f(u)||y(u)})}(u);if(!f(e)){if(w(&quot;before:highlightElement&quot;,{el:u,language:e}),u.children.length&gt;0&amp;&amp;(s.ignoreUnescapedHTML||(console.warn(&quot;One of your code blocks includes unescaped HTML. This is a potentially serious security risk.&quot;),console.warn(&quot;https://github.com/highlightjs/highlight.js/wiki/security&quot;),console.warn(&quot;The element with unescaped HTML:&quot;),console.warn(u)),s.throwUnescapedHTML))throw new Z(&quot;One of your code blocks includes unescaped HTML.&quot;,u.innerHTML);var t=u.textContent,n=e?A(t,{language:e,ignoreIllegals:!0}):h(t);u.innerHTML=n.value,function(u,e,t){var n=e&amp;&amp;D[e]||t;u.classList.add(&quot;hljs&quot;),u.classList.add(&quot;language-&quot;+n)}(u,e,n.language),u.result={language:n.language,re:n.relevance,relevance:n.relevance},n.secondBest&amp;&amp;(u.secondBest={language:n.secondBest.language,relevance:n.secondBest.relevance}),w(&quot;after:highlightElement&quot;,{el:u,result:n,text:t})}}var B=!1;function m(){&quot;loading&quot;!==document.readyState?document.querySelectorAll(s.cssSelector).forEach(p):B=!0}function y(u){return u=(u||&quot;&quot;).toLowerCase(),n[u]||n[D[u]]}function b(u,e){var t=e.languageName;&quot;string&quot;==typeof u&amp;&amp;(u=[u]),u.forEach(function(u){D[u.toLowerCase()]=t})}function v(u){var e=y(u);return e&amp;&amp;!e.disableAutodetect}function w(u,e){var t=u;r.forEach(function(u){u[t]&amp;&amp;u[t](e)})}for(var _ in&quot;undefined&quot;!=typeof window&amp;&amp;window.addEventListener&amp;&amp;window.addEventListener(&quot;DOMContentLoaded&quot;,function(){B&amp;&amp;m()},!1),Object.assign(e,{highlight:A,highlightAuto:h,highlightAll:m,highlightElement:p,highlightBlock:function(u){return W(&quot;10.7.0&quot;,&quot;highlightBlock will be removed entirely in v12.0&quot;),W(&quot;10.7.0&quot;,&quot;Please use highlightElement now.&quot;),p(u)},configure:function(u){s=V(s,u)},initHighlighting:function(){m(),W(&quot;10.6.0&quot;,&quot;initHighlighting() deprecated.  Use highlightAll() now.&quot;)},initHighlightingOnLoad:function(){m(),W(&quot;10.6.0&quot;,&quot;initHighlightingOnLoad() deprecated.  Use highlightAll() now.&quot;)},registerLanguage:function(u,t){var D=null;try{D=t(e)}catch(e){if(U(&quot;Language definition for &#x27;{}&#x27; could not be registered.&quot;.replace(&quot;{}&quot;,u)),!o)throw e;U(e),D=a}D.name||(D.name=u),n[u]=D,D.rawDefinition=t.bind(null,e),D.aliases&amp;&amp;b(D.aliases,{languageName:u})},unregisterLanguage:function(u){delete n[u];for(var e=0,t=Object.keys(D);e&lt;t.length;e++){var r=t[e];D[r]===u&amp;&amp;delete D[r]}},listLanguages:function(){return Object.keys(n)},getLanguage:y,registerAliases:b,autoDetection:v,inherit:V,addPlugin:function(u){(function(u){u[&quot;before:highlightBlock&quot;]&amp;&amp;!u[&quot;before:highlightElement&quot;]&amp;&amp;(u[&quot;before:highlightElement&quot;]=function(e){u[&quot;before:highlightBlock&quot;](Object.assign({block:e.el},e))}),u[&quot;after:highlightBlock&quot;]&amp;&amp;!u[&quot;after:highlightElement&quot;]&amp;&amp;(u[&quot;after:highlightElement&quot;]=function(e){u[&quot;after:highlightBlock&quot;](Object.assign({block:e.el},e))})})(u),r.push(u)}}),e.debugMode=function(){o=!1},e.safeMode=function(){o=!0},e.versionString=&quot;11.7.0&quot;,e.regex={concat:E,lookahead:l,either:d,optional:C,anyNumberOfTimes:F},O)&quot;object&quot;==_typeof(O[_])&amp;&amp;u.exports(O[_]);return Object.assign(e,O),e}({})}();function _typeof(u){return(_typeof=&quot;function&quot;==typeof Symbol&amp;&amp;&quot;symbol&quot;==typeof Symbol.iterator?function(u){return typeof u}:function(u){return u&amp;&amp;&quot;function&quot;==typeof Symbol&amp;&amp;u.constructor===Symbol&amp;&amp;u!==Symbol.prototype?&quot;symbol&quot;:typeof u})(u)}function _classCallCheck(u,e){if(!(u instanceof e))throw new TypeError(&quot;Cannot call a class as a function&quot;)}function _defineProperties(u,e){for(var t=0;t&lt;e.length;t++){var n=e[t];n.enumerable=n.enumerable||!1,n.configurable=!0,&quot;value&quot;in n&amp;&amp;(n.writable=!0),Object.defineProperty(u,_toPropertyKey(n.key),n)}}function _createClass(u,e,t){return e&amp;&amp;_defineProperties(u.prototype,e),t&amp;&amp;_defineProperties(u,t),Object.defineProperty(u,&quot;prototype&quot;,{writable:!1}),u}function _toPropertyKey(u){var e=_toPrimitive(u,&quot;string&quot;);return&quot;symbol&quot;===_typeof(e)?e:String(e)}function _toPrimitive(u,e){if(&quot;object&quot;!==_typeof(u)||null===u)return u;var t=u[Symbol.toPrimitive];if(void 0!==t){var n=t.call(u,e||&quot;default&quot;);if(&quot;object&quot;!==_typeof(n))return n;throw new TypeError(&quot;@@toPrimitive must return a primitive value.&quot;)}return(&quot;string&quot;===e?String:Number)(u)}&quot;object&quot;==(&quot;undefined&quot;==typeof exports?&quot;undefined&quot;:_typeof(exports))&amp;&amp;&quot;undefined&quot;!=typeof module&amp;&amp;(module.exports=hljs),function(){var u=function(u){var e=u.regex,t=/(?:[A-Z_a-z\\xAA\\xB5\\xBA\\xC0-\\xD6\\xD8-\\xF6\\xF8-\\u02C1\\u02C6-\\u02D1\\u02E0-\\u02E4\\u02EC\\u02EE\\u0370-\\u0374\\u0376\\u0377\\u037B-\\u037D\\u037F\\u0386\\u0388-\\u038A\\u038C\\u038E-\\u03A1\\u03A3-\\u03F5\\u03F7-\\u0481\\u048A-\\u052F\\u0531-\\u0556\\u0559\\u0560-\\u0588\\u05D0-\\u05EA\\u05EF-\\u05F2\\u0620-\\u064A\\u066E\\u066F\\u0671-\\u06D3\\u06D5\\u06E5\\u06E6\\u06EE\\u06EF\\u06FA-\\u06FC\\u06FF\\u0710\\u0712-\\u072F\\u074D-\\u07A5\\u07B1\\u07CA-\\u07EA\\u07F4\\u07F5\\u07FA\\u0800-\\u0815\\u081A\\u0824\\u0828\\u0840-\\u0858\\u0860-\\u086A\\u0870-\\u0887\\u0889-\\u088E\\u08A0-\\u08C9\\u0904-\\u0939\\u093D\\u0950\\u0958-\\u0961\\u0971-\\u0980\\u0985-\\u098C\\u098F\\u0990\\u0993-\\u09A8\\u09AA-\\u09B0\\u09B2\\u09B6-\\u09B9\\u09BD\\u09CE\\u09DC\\u09DD\\u09DF-\\u09E1\\u09F0\\u09F1\\u09FC\\u0A05-\\u0A0A\\u0A0F\\u0A10\\u0A13-\\u0A28\\u0A2A-\\u0A30\\u0A32\\u0A33\\u0A35\\u0A36\\u0A38\\u0A39\\u0A59-\\u0A5C\\u0A5E\\u0A72-\\u0A74\\u0A85-\\u0A8D\\u0A8F-\\u0A91\\u0A93-\\u0AA8\\u0AAA-\\u0AB0\\u0AB2\\u0AB3\\u0AB5-\\u0AB9\\u0ABD\\u0AD0\\u0AE0\\u0AE1\\u0AF9\\u0B05-\\u0B0C\\u0B0F\\u0B10\\u0B13-\\u0B28\\u0B2A-\\u0B30\\u0B32\\u0B33\\u0B35-\\u0B39\\u0B3D\\u0B5C\\u0B5D\\u0B5F-\\u0B61\\u0B71\\u0B83\\u0B85-\\u0B8A\\u0B8E-\\u0B90\\u0B92-\\u0B95\\u0B99\\u0B9A\\u0B9C\\u0B9E\\u0B9F\\u0BA3\\u0BA4\\u0BA8-\\u0BAA\\u0BAE-\\u0BB9\\u0BD0\\u0C05-\\u0C0C\\u0C0E-\\u0C10\\u0C12-\\u0C28\\u0C2A-\\u0C39\\u0C3D\\u0C58-\\u0C5A\\u0C5D\\u0C60\\u0C61\\u0C80\\u0C85-\\u0C8C\\u0C8E-\\u0C90\\u0C92-\\u0CA8\\u0CAA-\\u0CB3\\u0CB5-\\u0CB9\\u0CBD\\u0CDD\\u0CDE\\u0CE0\\u0CE1\\u0CF1\\u0CF2\\u0D04-\\u0D0C\\u0D0E-\\u0D10\\u0D12-\\u0D3A\\u0D3D\\u0D4E\\u0D54-\\u0D56\\u0D5F-\\u0D61\\u0D7A-\\u0D7F\\u0D85-\\u0D96\\u0D9A-\\u0DB1\\u0DB3-\\u0DBB\\u0DBD\\u0DC0-\\u0DC6\\u0E01-\\u0E30\\u0E32\\u0E40-\\u0E46\\u0E81\\u0E82\\u0E84\\u0E86-\\u0E8A\\u0E8C-\\u0EA3\\u0EA5\\u0EA7-\\u0EB0\\u0EB2\\u0EBD\\u0EC0-\\u0EC4\\u0EC6\\u0EDC-\\u0EDF\\u0F00\\u0F40-\\u0F47\\u0F49-\\u0F6C\\u0F88-\\u0F8C\\u1000-\\u102A\\u103F\\u1050-\\u1055\\u105A-\\u105D\\u1061\\u1065\\u1066\\u106E-\\u1070\\u1075-\\u1081\\u108E\\u10A0-\\u10C5\\u10C7\\u10CD\\u10D0-\\u10FA\\u10FC-\\u1248\\u124A-\\u124D\\u1250-\\u1256\\u1258\\u125A-\\u125D\\u1260-\\u1288\\u128A-\\u128D\\u1290-\\u12B0\\u12B2-\\u12B5\\u12B8-\\u12BE\\u12C0\\u12C2-\\u12C5\\u12C8-\\u12D6\\u12D8-\\u1310\\u1312-\\u1315\\u1318-\\u135A\\u1380-\\u138F\\u13A0-\\u13F5\\u13F8-\\u13FD\\u1401-\\u166C\\u166F-\\u167F\\u1681-\\u169A\\u16A0-\\u16EA\\u16EE-\\u16F8\\u1700-\\u1711\\u171F-\\u1731\\u1740-\\u1751\\u1760-\\u176C\\u176E-\\u1770\\u1780-\\u17B3\\u17D7\\u17DC\\u1820-\\u1878\\u1880-\\u18A8\\u18AA\\u18B0-\\u18F5\\u1900-\\u191E\\u1950-\\u196D\\u1970-\\u1974\\u1980-\\u19AB\\u19B0-\\u19C9\\u1A00-\\u1A16\\u1A20-\\u1A54\\u1AA7\\u1B05-\\u1B33\\u1B45-\\u1B4C\\u1B83-\\u1BA0\\u1BAE\\u1BAF\\u1BBA-\\u1BE5\\u1C00-\\u1C23\\u1C4D-\\u1C4F\\u1C5A-\\u1C7D\\u1C80-\\u1C88\\u1C90-\\u1CBA\\u1CBD-\\u1CBF\\u1CE9-\\u1CEC\\u1CEE-\\u1CF3\\u1CF5\\u1CF6\\u1CFA\\u1D00-\\u1DBF\\u1E00-\\u1F15\\u1F18-\\u1F1D\\u1F20-\\u1F45\\u1F48-\\u1F4D\\u1F50-\\u1F57\\u1F59\\u1F5B\\u1F5D\\u1F5F-\\u1F7D\\u1F80-\\u1FB4\\u1FB6-\\u1FBC\\u1FBE\\u1FC2-\\u1FC4\\u1FC6-\\u1FCC\\u1FD0-\\u1FD3\\u1FD6-\\u1FDB\\u1FE0-\\u1FEC\\u1FF2-\\u1FF4\\u1FF6-\\u1FFC\\u2071\\u207F\\u2090-\\u209C\\u2102\\u2107\\u210A-\\u2113\\u2115\\u2118-\\u211D\\u2124\\u2126\\u2128\\u212A-\\u2139\\u213C-\\u213F\\u2145-\\u2149\\u214E\\u2160-\\u2188\\u2C00-\\u2CE4\\u2CEB-\\u2CEE\\u2CF2\\u2CF3\\u2D00-\\u2D25\\u2D27\\u2D2D\\u2D30-\\u2D67\\u2D6F\\u2D80-\\u2D96\\u2DA0-\\u2DA6\\u2DA8-\\u2DAE\\u2DB0-\\u2DB6\\u2DB8-\\u2DBE\\u2DC0-\\u2DC6\\u2DC8-\\u2DCE\\u2DD0-\\u2DD6\\u2DD8-\\u2DDE\\u3005-\\u3007\\u3021-\\u3029\\u3031-\\u3035\\u3038-\\u303C\\u3041-\\u3096\\u309D-\\u309F\\u30A1-\\u30FA\\u30FC-\\u30FF\\u3105-\\u312F\\u3131-\\u318E\\u31A0-\\u31BF\\u31F0-\\u31FF\\u3400-\\u4DBF\\u4E00-\\uA48C\\uA4D0-\\uA4FD\\uA500-\\uA60C\\uA610-\\uA61F\\uA62A\\uA62B\\uA640-\\uA66E\\uA67F-\\uA69D\\uA6A0-\\uA6EF\\uA717-\\uA71F\\uA722-\\uA788\\uA78B-\\uA7CA\\uA7D0\\uA7D1\\uA7D3\\uA7D5-\\uA7D9\\uA7F2-\\uA801\\uA803-\\uA805\\uA807-\\uA80A\\uA80C-\\uA822\\uA840-\\uA873\\uA882-\\uA8B3\\uA8F2-\\uA8F7\\uA8FB\\uA8FD\\uA8FE\\uA90A-\\uA925\\uA930-\\uA946\\uA960-\\uA97C\\uA984-\\uA9B2\\uA9CF\\uA9E0-\\uA9E4\\uA9E6-\\uA9EF\\uA9FA-\\uA9FE\\uAA00-\\uAA28\\uAA40-\\uAA42\\uAA44-\\uAA4B\\uAA60-\\uAA76\\uAA7A\\uAA7E-\\uAAAF\\uAAB1\\uAAB5\\uAAB6\\uAAB9-\\uAABD\\uAAC0\\uAAC2\\uAADB-\\uAADD\\uAAE0-\\uAAEA\\uAAF2-\\uAAF4\\uAB01-\\uAB06\\uAB09-\\uAB0E\\uAB11-\\uAB16\\uAB20-\\uAB26\\uAB28-\\uAB2E\\uAB30-\\uAB5A\\uAB5C-\\uAB69\\uAB70-\\uABE2\\uAC00-\\uD7A3\\uD7B0-\\uD7C6\\uD7CB-\\uD7FB\\uF900-\\uFA6D\\uFA70-\\uFAD9\\uFB00-\\uFB06\\uFB13-\\uFB17\\uFB1D\\uFB1F-\\uFB28\\uFB2A-\\uFB36\\uFB38-\\uFB3C\\uFB3E\\uFB40\\uFB41\\uFB43\\uFB44\\uFB46-\\uFBB1\\uFBD3-\\uFC5D\\uFC64-\\uFD3D\\uFD50-\\uFD8F\\uFD92-\\uFDC7\\uFDF0-\\uFDF9\\uFE71\\uFE73\\uFE77\\uFE79\\uFE7B\\uFE7D\\uFE7F-\\uFEFC\\uFF21-\\uFF3A\\uFF41-\\uFF5A\\uFF66-\\uFF9D\\uFFA0-\\uFFBE\\uFFC2-\\uFFC7\\uFFCA-\\uFFCF\\uFFD2-\\uFFD7\\uFFDA-\\uFFDC]|\\uD800[\\uDC00-\\uDC0B\\uDC0D-\\uDC26\\uDC28-\\uDC3A\\uDC3C\\uDC3D\\uDC3F-\\uDC4D\\uDC50-\\uDC5D\\uDC80-\\uDCFA\\uDD40-\\uDD74\\uDE80-\\uDE9C\\uDEA0-\\uDED0\\uDF00-\\uDF1F\\uDF2D-\\uDF4A\\uDF50-\\uDF75\\uDF80-\\uDF9D\\uDFA0-\\uDFC3\\uDFC8-\\uDFCF\\uDFD1-\\uDFD5]|\\uD801[\\uDC00-\\uDC9D\\uDCB0-\\uDCD3\\uDCD8-\\uDCFB\\uDD00-\\uDD27\\uDD30-\\uDD63\\uDD70-\\uDD7A\\uDD7C-\\uDD8A\\uDD8C-\\uDD92\\uDD94\\uDD95\\uDD97-\\uDDA1\\uDDA3-\\uDDB1\\uDDB3-\\uDDB9\\uDDBB\\uDDBC\\uDE00-\\uDF36\\uDF40-\\uDF55\\uDF60-\\uDF67\\uDF80-\\uDF85\\uDF87-\\uDFB0\\uDFB2-\\uDFBA]|\\uD802[\\uDC00-\\uDC05\\uDC08\\uDC0A-\\uDC35\\uDC37\\uDC38\\uDC3C\\uDC3F-\\uDC55\\uDC60-\\uDC76\\uDC80-\\uDC9E\\uDCE0-\\uDCF2\\uDCF4\\uDCF5\\uDD00-\\uDD15\\uDD20-\\uDD39\\uDD80-\\uDDB7\\uDDBE\\uDDBF\\uDE00\\uDE10-\\uDE13\\uDE15-\\uDE17\\uDE19-\\uDE35\\uDE60-\\uDE7C\\uDE80-\\uDE9C\\uDEC0-\\uDEC7\\uDEC9-\\uDEE4\\uDF00-\\uDF35\\uDF40-\\uDF55\\uDF60-\\uDF72\\uDF80-\\uDF91]|\\uD803[\\uDC00-\\uDC48\\uDC80-\\uDCB2\\uDCC0-\\uDCF2\\uDD00-\\uDD23\\uDE80-\\uDEA9\\uDEB0\\uDEB1\\uDF00-\\uDF1C\\uDF27\\uDF30-\\uDF45\\uDF70-\\uDF81\\uDFB0-\\uDFC4\\uDFE0-\\uDFF6]|\\uD804[\\uDC03-\\uDC37\\uDC71\\uDC72\\uDC75\\uDC83-\\uDCAF\\uDCD0-\\uDCE8\\uDD03-\\uDD26\\uDD44\\uDD47\\uDD50-\\uDD72\\uDD76\\uDD83-\\uDDB2\\uDDC1-\\uDDC4\\uDDDA\\uDDDC\\uDE00-\\uDE11\\uDE13-\\uDE2B\\uDE3F\\uDE40\\uDE80-\\uDE86\\uDE88\\uDE8A-\\uDE8D\\uDE8F-\\uDE9D\\uDE9F-\\uDEA8\\uDEB0-\\uDEDE\\uDF05-\\uDF0C\\uDF0F\\uDF10\\uDF13-\\uDF28\\uDF2A-\\uDF30\\uDF32\\uDF33\\uDF35-\\uDF39\\uDF3D\\uDF50\\uDF5D-\\uDF61]|\\uD805[\\uDC00-\\uDC34\\uDC47-\\uDC4A\\uDC5F-\\uDC61\\uDC80-\\uDCAF\\uDCC4\\uDCC5\\uDCC7\\uDD80-\\uDDAE\\uDDD8-\\uDDDB\\uDE00-\\uDE2F\\uDE44\\uDE80-\\uDEAA\\uDEB8\\uDF00-\\uDF1A\\uDF40-\\uDF46]|\\uD806[\\uDC00-\\uDC2B\\uDCA0-\\uDCDF\\uDCFF-\\uDD06\\uDD09\\uDD0C-\\uDD13\\uDD15\\uDD16\\uDD18-\\uDD2F\\uDD3F\\uDD41\\uDDA0-\\uDDA7\\uDDAA-\\uDDD0\\uDDE1\\uDDE3\\uDE00\\uDE0B-\\uDE32\\uDE3A\\uDE50\\uDE5C-\\uDE89\\uDE9D\\uDEB0-\\uDEF8]|\\uD807[\\uDC00-\\uDC08\\uDC0A-\\uDC2E\\uDC40\\uDC72-\\uDC8F\\uDD00-\\uDD06\\uDD08\\uDD09\\uDD0B-\\uDD30\\uDD46\\uDD60-\\uDD65\\uDD67\\uDD68\\uDD6A-\\uDD89\\uDD98\\uDEE0-\\uDEF2\\uDF02\\uDF04-\\uDF10\\uDF12-\\uDF33\\uDFB0]|\\uD808[\\uDC00-\\uDF99]|\\uD809[\\uDC00-\\uDC6E\\uDC80-\\uDD43]|\\uD80B[\\uDF90-\\uDFF0]|[\\uD80C\\uD81C-\\uD820\\uD822\\uD840-\\uD868\\uD86A-\\uD86C\\uD86F-\\uD872\\uD874-\\uD879\\uD880-\\uD883\\uD885-\\uD887][\\uDC00-\\uDFFF]|\\uD80D[\\uDC00-\\uDC2F\\uDC41-\\uDC46]|\\uD811[\\uDC00-\\uDE46]|\\uD81A[\\uDC00-\\uDE38\\uDE40-\\uDE5E\\uDE70-\\uDEBE\\uDED0-\\uDEED\\uDF00-\\uDF2F\\uDF40-\\uDF43\\uDF63-\\uDF77\\uDF7D-\\uDF8F]|\\uD81B[\\uDE40-\\uDE7F\\uDF00-\\uDF4A\\uDF50\\uDF93-\\uDF9F\\uDFE0\\uDFE1\\uDFE3]|\\uD821[\\uDC00-\\uDFF7]|\\uD823[\\uDC00-\\uDCD5\\uDD00-\\uDD08]|\\uD82B[\\uDFF0-\\uDFF3\\uDFF5-\\uDFFB\\uDFFD\\uDFFE]|\\uD82C[\\uDC00-\\uDD22\\uDD32\\uDD50-\\uDD52\\uDD55\\uDD64-\\uDD67\\uDD70-\\uDEFB]|\\uD82F[\\uDC00-\\uDC6A\\uDC70-\\uDC7C\\uDC80-\\uDC88\\uDC90-\\uDC99]|\\uD835[\\uDC00-\\uDC54\\uDC56-\\uDC9C\\uDC9E\\uDC9F\\uDCA2\\uDCA5\\uDCA6\\uDCA9-\\uDCAC\\uDCAE-\\uDCB9\\uDCBB\\uDCBD-\\uDCC3\\uDCC5-\\uDD05\\uDD07-\\uDD0A\\uDD0D-\\uDD14\\uDD16-\\uDD1C\\uDD1E-\\uDD39\\uDD3B-\\uDD3E\\uDD40-\\uDD44\\uDD46\\uDD4A-\\uDD50\\uDD52-\\uDEA5\\uDEA8-\\uDEC0\\uDEC2-\\uDEDA\\uDEDC-\\uDEFA\\uDEFC-\\uDF14\\uDF16-\\uDF34\\uDF36-\\uDF4E\\uDF50-\\uDF6E\\uDF70-\\uDF88\\uDF8A-\\uDFA8\\uDFAA-\\uDFC2\\uDFC4-\\uDFCB]|\\uD837[\\uDF00-\\uDF1E\\uDF25-\\uDF2A]|\\uD838[\\uDC30-\\uDC6D\\uDD00-\\uDD2C\\uDD37-\\uDD3D\\uDD4E\\uDE90-\\uDEAD\\uDEC0-\\uDEEB]|\\uD839[\\uDCD0-\\uDCEB\\uDFE0-\\uDFE6\\uDFE8-\\uDFEB\\uDFED\\uDFEE\\uDFF0-\\uDFFE]|\\uD83A[\\uDC00-\\uDCC4\\uDD00-\\uDD43\\uDD4B]|\\uD83B[\\uDE00-\\uDE03\\uDE05-\\uDE1F\\uDE21\\uDE22\\uDE24\\uDE27\\uDE29-\\uDE32\\uDE34-\\uDE37\\uDE39\\uDE3B\\uDE42\\uDE47\\uDE49\\uDE4B\\uDE4D-\\uDE4F\\uDE51\\uDE52\\uDE54\\uDE57\\uDE59\\uDE5B\\uDE5D\\uDE5F\\uDE61\\uDE62\\uDE64\\uDE67-\\uDE6A\\uDE6C-\\uDE72\\uDE74-\\uDE77\\uDE79-\\uDE7C\\uDE7E\\uDE80-\\uDE89\\uDE8B-\\uDE9B\\uDEA1-\\uDEA3\\uDEA5-\\uDEA9\\uDEAB-\\uDEBB]|\\uD869[\\uDC00-\\uDEDF\\uDF00-\\uDFFF]|\\uD86D[\\uDC00-\\uDF39\\uDF40-\\uDFFF]|\\uD86E[\\uDC00-\\uDC1D\\uDC20-\\uDFFF]|\\uD873[\\uDC00-\\uDEA1\\uDEB0-\\uDFFF]|\\uD87A[\\uDC00-\\uDFE0]|\\uD87E[\\uDC00-\\uDE1D]|\\uD884[\\uDC00-\\uDF4A\\uDF50-\\uDFFF]|\\uD888[\\uDC00-\\uDFAF])(?:[0-9A-Z_a-z\\xAA\\xB5\\xB7\\xBA\\xC0-\\xD6\\xD8-\\xF6\\xF8-\\u02C1\\u02C6-\\u02D1\\u02E0-\\u02E4\\u02EC\\u02EE\\u0300-\\u0374\\u0376\\u0377\\u037B-\\u037D\\u037F\\u0386-\\u038A\\u038C\\u038E-\\u03A1\\u03A3-\\u03F5\\u03F7-\\u0481\\u0483-\\u0487\\u048A-\\u052F\\u0531-\\u0556\\u0559\\u0560-\\u0588\\u0591-\\u05BD\\u05BF\\u05C1\\u05C2\\u05C4\\u05C5\\u05C7\\u05D0-\\u05EA\\u05EF-\\u05F2\\u0610-\\u061A\\u0620-\\u0669\\u066E-\\u06D3\\u06D5-\\u06DC\\u06DF-\\u06E8\\u06EA-\\u06FC\\u06FF\\u0710-\\u074A\\u074D-\\u07B1\\u07C0-\\u07F5\\u07FA\\u07FD\\u0800-\\u082D\\u0840-\\u085B\\u0860-\\u086A\\u0870-\\u0887\\u0889-\\u088E\\u0898-\\u08E1\\u08E3-\\u0963\\u0966-\\u096F\\u0971-\\u0983\\u0985-\\u098C\\u098F\\u0990\\u0993-\\u09A8\\u09AA-\\u09B0\\u09B2\\u09B6-\\u09B9\\u09BC-\\u09C4\\u09C7\\u09C8\\u09CB-\\u09CE\\u09D7\\u09DC\\u09DD\\u09DF-\\u09E3\\u09E6-\\u09F1\\u09FC\\u09FE\\u0A01-\\u0A03\\u0A05-\\u0A0A\\u0A0F\\u0A10\\u0A13-\\u0A28\\u0A2A-\\u0A30\\u0A32\\u0A33\\u0A35\\u0A36\\u0A38\\u0A39\\u0A3C\\u0A3E-\\u0A42\\u0A47\\u0A48\\u0A4B-\\u0A4D\\u0A51\\u0A59-\\u0A5C\\u0A5E\\u0A66-\\u0A75\\u0A81-\\u0A83\\u0A85-\\u0A8D\\u0A8F-\\u0A91\\u0A93-\\u0AA8\\u0AAA-\\u0AB0\\u0AB2\\u0AB3\\u0AB5-\\u0AB9\\u0ABC-\\u0AC5\\u0AC7-\\u0AC9\\u0ACB-\\u0ACD\\u0AD0\\u0AE0-\\u0AE3\\u0AE6-\\u0AEF\\u0AF9-\\u0AFF\\u0B01-\\u0B03\\u0B05-\\u0B0C\\u0B0F\\u0B10\\u0B13-\\u0B28\\u0B2A-\\u0B30\\u0B32\\u0B33\\u0B35-\\u0B39\\u0B3C-\\u0B44\\u0B47\\u0B48\\u0B4B-\\u0B4D\\u0B55-\\u0B57\\u0B5C\\u0B5D\\u0B5F-\\u0B63\\u0B66-\\u0B6F\\u0B71\\u0B82\\u0B83\\u0B85-\\u0B8A\\u0B8E-\\u0B90\\u0B92-\\u0B95\\u0B99\\u0B9A\\u0B9C\\u0B9E\\u0B9F\\u0BA3\\u0BA4\\u0BA8-\\u0BAA\\u0BAE-\\u0BB9\\u0BBE-\\u0BC2\\u0BC6-\\u0BC8\\u0BCA-\\u0BCD\\u0BD0\\u0BD7\\u0BE6-\\u0BEF\\u0C00-\\u0C0C\\u0C0E-\\u0C10\\u0C12-\\u0C28\\u0C2A-\\u0C39\\u0C3C-\\u0C44\\u0C46-\\u0C48\\u0C4A-\\u0C4D\\u0C55\\u0C56\\u0C58-\\u0C5A\\u0C5D\\u0C60-\\u0C63\\u0C66-\\u0C6F\\u0C80-\\u0C83\\u0C85-\\u0C8C\\u0C8E-\\u0C90\\u0C92-\\u0CA8\\u0CAA-\\u0CB3\\u0CB5-\\u0CB9\\u0CBC-\\u0CC4\\u0CC6-\\u0CC8\\u0CCA-\\u0CCD\\u0CD5\\u0CD6\\u0CDD\\u0CDE\\u0CE0-\\u0CE3\\u0CE6-\\u0CEF\\u0CF1-\\u0CF3\\u0D00-\\u0D0C\\u0D0E-\\u0D10\\u0D12-\\u0D44\\u0D46-\\u0D48\\u0D4A-\\u0D4E\\u0D54-\\u0D57\\u0D5F-\\u0D63\\u0D66-\\u0D6F\\u0D7A-\\u0D7F\\u0D81-\\u0D83\\u0D85-\\u0D96\\u0D9A-\\u0DB1\\u0DB3-\\u0DBB\\u0DBD\\u0DC0-\\u0DC6\\u0DCA\\u0DCF-\\u0DD4\\u0DD6\\u0DD8-\\u0DDF\\u0DE6-\\u0DEF\\u0DF2\\u0DF3\\u0E01-\\u0E3A\\u0E40-\\u0E4E\\u0E50-\\u0E59\\u0E81\\u0E82\\u0E84\\u0E86-\\u0E8A\\u0E8C-\\u0EA3\\u0EA5\\u0EA7-\\u0EBD\\u0EC0-\\u0EC4\\u0EC6\\u0EC8-\\u0ECE\\u0ED0-\\u0ED9\\u0EDC-\\u0EDF\\u0F00\\u0F18\\u0F19\\u0F20-\\u0F29\\u0F35\\u0F37\\u0F39\\u0F3E-\\u0F47\\u0F49-\\u0F6C\\u0F71-\\u0F84\\u0F86-\\u0F97\\u0F99-\\u0FBC\\u0FC6\\u1000-\\u1049\\u1050-\\u109D\\u10A0-\\u10C5\\u10C7\\u10CD\\u10D0-\\u10FA\\u10FC-\\u1248\\u124A-\\u124D\\u1250-\\u1256\\u1258\\u125A-\\u125D\\u1260-\\u1288\\u128A-\\u128D\\u1290-\\u12B0\\u12B2-\\u12B5\\u12B8-\\u12BE\\u12C0\\u12C2-\\u12C5\\u12C8-\\u12D6\\u12D8-\\u1310\\u1312-\\u1315\\u1318-\\u135A\\u135D-\\u135F\\u1369-\\u1371\\u1380-\\u138F\\u13A0-\\u13F5\\u13F8-\\u13FD\\u1401-\\u166C\\u166F-\\u167F\\u1681-\\u169A\\u16A0-\\u16EA\\u16EE-\\u16F8\\u1700-\\u1715\\u171F-\\u1734\\u1740-\\u1753\\u1760-\\u176C\\u176E-\\u1770\\u1772\\u1773\\u1780-\\u17D3\\u17D7\\u17DC\\u17DD\\u17E0-\\u17E9\\u180B-\\u180D\\u180F-\\u1819\\u1820-\\u1878\\u1880-\\u18AA\\u18B0-\\u18F5\\u1900-\\u191E\\u1920-\\u192B\\u1930-\\u193B\\u1946-\\u196D\\u1970-\\u1974\\u1980-\\u19AB\\u19B0-\\u19C9\\u19D0-\\u19DA\\u1A00-\\u1A1B\\u1A20-\\u1A5E\\u1A60-\\u1A7C\\u1A7F-\\u1A89\\u1A90-\\u1A99\\u1AA7\\u1AB0-\\u1ABD\\u1ABF-\\u1ACE\\u1B00-\\u1B4C\\u1B50-\\u1B59\\u1B6B-\\u1B73\\u1B80-\\u1BF3\\u1C00-\\u1C37\\u1C40-\\u1C49\\u1C4D-\\u1C7D\\u1C80-\\u1C88\\u1C90-\\u1CBA\\u1CBD-\\u1CBF\\u1CD0-\\u1CD2\\u1CD4-\\u1CFA\\u1D00-\\u1F15\\u1F18-\\u1F1D\\u1F20-\\u1F45\\u1F48-\\u1F4D\\u1F50-\\u1F57\\u1F59\\u1F5B\\u1F5D\\u1F5F-\\u1F7D\\u1F80-\\u1FB4\\u1FB6-\\u1FBC\\u1FBE\\u1FC2-\\u1FC4\\u1FC6-\\u1FCC\\u1FD0-\\u1FD3\\u1FD6-\\u1FDB\\u1FE0-\\u1FEC\\u1FF2-\\u1FF4\\u1FF6-\\u1FFC\\u203F\\u2040\\u2054\\u2071\\u207F\\u2090-\\u209C\\u20D0-\\u20DC\\u20E1\\u20E5-\\u20F0\\u2102\\u2107\\u210A-\\u2113\\u2115\\u2118-\\u211D\\u2124\\u2126\\u2128\\u212A-\\u2139\\u213C-\\u213F\\u2145-\\u2149\\u214E\\u2160-\\u2188\\u2C00-\\u2CE4\\u2CEB-\\u2CF3\\u2D00-\\u2D25\\u2D27\\u2D2D\\u2D30-\\u2D67\\u2D6F\\u2D7F-\\u2D96\\u2DA0-\\u2DA6\\u2DA8-\\u2DAE\\u2DB0-\\u2DB6\\u2DB8-\\u2DBE\\u2DC0-\\u2DC6\\u2DC8-\\u2DCE\\u2DD0-\\u2DD6\\u2DD8-\\u2DDE\\u2DE0-\\u2DFF\\u3005-\\u3007\\u3021-\\u302F\\u3031-\\u3035\\u3038-\\u303C\\u3041-\\u3096\\u3099\\u309A\\u309D-\\u309F\\u30A1-\\u30FA\\u30FC-\\u30FF\\u3105-\\u312F\\u3131-\\u318E\\u31A0-\\u31BF\\u31F0-\\u31FF\\u3400-\\u4DBF\\u4E00-\\uA48C\\uA4D0-\\uA4FD\\uA500-\\uA60C\\uA610-\\uA62B\\uA640-\\uA66F\\uA674-\\uA67D\\uA67F-\\uA6F1\\uA717-\\uA71F\\uA722-\\uA788\\uA78B-\\uA7CA\\uA7D0\\uA7D1\\uA7D3\\uA7D5-\\uA7D9\\uA7F2-\\uA827\\uA82C\\uA840-\\uA873\\uA880-\\uA8C5\\uA8D0-\\uA8D9\\uA8E0-\\uA8F7\\uA8FB\\uA8FD-\\uA92D\\uA930-\\uA953\\uA960-\\uA97C\\uA980-\\uA9C0\\uA9CF-\\uA9D9\\uA9E0-\\uA9FE\\uAA00-\\uAA36\\uAA40-\\uAA4D\\uAA50-\\uAA59\\uAA60-\\uAA76\\uAA7A-\\uAAC2\\uAADB-\\uAADD\\uAAE0-\\uAAEF\\uAAF2-\\uAAF6\\uAB01-\\uAB06\\uAB09-\\uAB0E\\uAB11-\\uAB16\\uAB20-\\uAB26\\uAB28-\\uAB2E\\uAB30-\\uAB5A\\uAB5C-\\uAB69\\uAB70-\\uABEA\\uABEC\\uABED\\uABF0-\\uABF9\\uAC00-\\uD7A3\\uD7B0-\\uD7C6\\uD7CB-\\uD7FB\\uF900-\\uFA6D\\uFA70-\\uFAD9\\uFB00-\\uFB06\\uFB13-\\uFB17\\uFB1D-\\uFB28\\uFB2A-\\uFB36\\uFB38-\\uFB3C\\uFB3E\\uFB40\\uFB41\\uFB43\\uFB44\\uFB46-\\uFBB1\\uFBD3-\\uFC5D\\uFC64-\\uFD3D\\uFD50-\\uFD8F\\uFD92-\\uFDC7\\uFDF0-\\uFDF9\\uFE00-\\uFE0F\\uFE20-\\uFE2F\\uFE33\\uFE34\\uFE4D-\\uFE4F\\uFE71\\uFE73\\uFE77\\uFE79\\uFE7B\\uFE7D\\uFE7F-\\uFEFC\\uFF10-\\uFF19\\uFF21-\\uFF3A\\uFF3F\\uFF41-\\uFF5A\\uFF66-\\uFFBE\\uFFC2-\\uFFC7\\uFFCA-\\uFFCF\\uFFD2-\\uFFD7\\uFFDA-\\uFFDC]|\\uD800[\\uDC00-\\uDC0B\\uDC0D-\\uDC26\\uDC28-\\uDC3A\\uDC3C\\uDC3D\\uDC3F-\\uDC4D\\uDC50-\\uDC5D\\uDC80-\\uDCFA\\uDD40-\\uDD74\\uDDFD\\uDE80-\\uDE9C\\uDEA0-\\uDED0\\uDEE0\\uDF00-\\uDF1F\\uDF2D-\\uDF4A\\uDF50-\\uDF7A\\uDF80-\\uDF9D\\uDFA0-\\uDFC3\\uDFC8-\\uDFCF\\uDFD1-\\uDFD5]|\\uD801[\\uDC00-\\uDC9D\\uDCA0-\\uDCA9\\uDCB0-\\uDCD3\\uDCD8-\\uDCFB\\uDD00-\\uDD27\\uDD30-\\uDD63\\uDD70-\\uDD7A\\uDD7C-\\uDD8A\\uDD8C-\\uDD92\\uDD94\\uDD95\\uDD97-\\uDDA1\\uDDA3-\\uDDB1\\uDDB3-\\uDDB9\\uDDBB\\uDDBC\\uDE00-\\uDF36\\uDF40-\\uDF55\\uDF60-\\uDF67\\uDF80-\\uDF85\\uDF87-\\uDFB0\\uDFB2-\\uDFBA]|\\uD802[\\uDC00-\\uDC05\\uDC08\\uDC0A-\\uDC35\\uDC37\\uDC38\\uDC3C\\uDC3F-\\uDC55\\uDC60-\\uDC76\\uDC80-\\uDC9E\\uDCE0-\\uDCF2\\uDCF4\\uDCF5\\uDD00-\\uDD15\\uDD20-\\uDD39\\uDD80-\\uDDB7\\uDDBE\\uDDBF\\uDE00-\\uDE03\\uDE05\\uDE06\\uDE0C-\\uDE13\\uDE15-\\uDE17\\uDE19-\\uDE35\\uDE38-\\uDE3A\\uDE3F\\uDE60-\\uDE7C\\uDE80-\\uDE9C\\uDEC0-\\uDEC7\\uDEC9-\\uDEE6\\uDF00-\\uDF35\\uDF40-\\uDF55\\uDF60-\\uDF72\\uDF80-\\uDF91]|\\uD803[\\uDC00-\\uDC48\\uDC80-\\uDCB2\\uDCC0-\\uDCF2\\uDD00-\\uDD27\\uDD30-\\uDD39\\uDE80-\\uDEA9\\uDEAB\\uDEAC\\uDEB0\\uDEB1\\uDEFD-\\uDF1C\\uDF27\\uDF30-\\uDF50\\uDF70-\\uDF85\\uDFB0-\\uDFC4\\uDFE0-\\uDFF6]|\\uD804[\\uDC00-\\uDC46\\uDC66-\\uDC75\\uDC7F-\\uDCBA\\uDCC2\\uDCD0-\\uDCE8\\uDCF0-\\uDCF9\\uDD00-\\uDD34\\uDD36-\\uDD3F\\uDD44-\\uDD47\\uDD50-\\uDD73\\uDD76\\uDD80-\\uDDC4\\uDDC9-\\uDDCC\\uDDCE-\\uDDDA\\uDDDC\\uDE00-\\uDE11\\uDE13-\\uDE37\\uDE3E-\\uDE41\\uDE80-\\uDE86\\uDE88\\uDE8A-\\uDE8D\\uDE8F-\\uDE9D\\uDE9F-\\uDEA8\\uDEB0-\\uDEEA\\uDEF0-\\uDEF9\\uDF00-\\uDF03\\uDF05-\\uDF0C\\uDF0F\\uDF10\\uDF13-\\uDF28\\uDF2A-\\uDF30\\uDF32\\uDF33\\uDF35-\\uDF39\\uDF3B-\\uDF44\\uDF47\\uDF48\\uDF4B-\\uDF4D\\uDF50\\uDF57\\uDF5D-\\uDF63\\uDF66-\\uDF6C\\uDF70-\\uDF74]|\\uD805[\\uDC00-\\uDC4A\\uDC50-\\uDC59\\uDC5E-\\uDC61\\uDC80-\\uDCC5\\uDCC7\\uDCD0-\\uDCD9\\uDD80-\\uDDB5\\uDDB8-\\uDDC0\\uDDD8-\\uDDDD\\uDE00-\\uDE40\\uDE44\\uDE50-\\uDE59\\uDE80-\\uDEB8\\uDEC0-\\uDEC9\\uDF00-\\uDF1A\\uDF1D-\\uDF2B\\uDF30-\\uDF39\\uDF40-\\uDF46]|\\uD806[\\uDC00-\\uDC3A\\uDCA0-\\uDCE9\\uDCFF-\\uDD06\\uDD09\\uDD0C-\\uDD13\\uDD15\\uDD16\\uDD18-\\uDD35\\uDD37\\uDD38\\uDD3B-\\uDD43\\uDD50-\\uDD59\\uDDA0-\\uDDA7\\uDDAA-\\uDDD7\\uDDDA-\\uDDE1\\uDDE3\\uDDE4\\uDE00-\\uDE3E\\uDE47\\uDE50-\\uDE99\\uDE9D\\uDEB0-\\uDEF8]|\\uD807[\\uDC00-\\uDC08\\uDC0A-\\uDC36\\uDC38-\\uDC40\\uDC50-\\uDC59\\uDC72-\\uDC8F\\uDC92-\\uDCA7\\uDCA9-\\uDCB6\\uDD00-\\uDD06\\uDD08\\uDD09\\uDD0B-\\uDD36\\uDD3A\\uDD3C\\uDD3D\\uDD3F-\\uDD47\\uDD50-\\uDD59\\uDD60-\\uDD65\\uDD67\\uDD68\\uDD6A-\\uDD8E\\uDD90\\uDD91\\uDD93-\\uDD98\\uDDA0-\\uDDA9\\uDEE0-\\uDEF6\\uDF00-\\uDF10\\uDF12-\\uDF3A\\uDF3E-\\uDF42\\uDF50-\\uDF59\\uDFB0]|\\uD808[\\uDC00-\\uDF99]|\\uD809[\\uDC00-\\uDC6E\\uDC80-\\uDD43]|\\uD80B[\\uDF90-\\uDFF0]|[\\uD80C\\uD81C-\\uD820\\uD822\\uD840-\\uD868\\uD86A-\\uD86C\\uD86F-\\uD872\\uD874-\\uD879\\uD880-\\uD883\\uD885-\\uD887][\\uDC00-\\uDFFF]|\\uD80D[\\uDC00-\\uDC2F\\uDC40-\\uDC55]|\\uD811[\\uDC00-\\uDE46]|\\uD81A[\\uDC00-\\uDE38\\uDE40-\\uDE5E\\uDE60-\\uDE69\\uDE70-\\uDEBE\\uDEC0-\\uDEC9\\uDED0-\\uDEED\\uDEF0-\\uDEF4\\uDF00-\\uDF36\\uDF40-\\uDF43\\uDF50-\\uDF59\\uDF63-\\uDF77\\uDF7D-\\uDF8F]|\\uD81B[\\uDE40-\\uDE7F\\uDF00-\\uDF4A\\uDF4F-\\uDF87\\uDF8F-\\uDF9F\\uDFE0\\uDFE1\\uDFE3\\uDFE4\\uDFF0\\uDFF1]|\\uD821[\\uDC00-\\uDFF7]|\\uD823[\\uDC00-\\uDCD5\\uDD00-\\uDD08]|\\uD82B[\\uDFF0-\\uDFF3\\uDFF5-\\uDFFB\\uDFFD\\uDFFE]|\\uD82C[\\uDC00-\\uDD22\\uDD32\\uDD50-\\uDD52\\uDD55\\uDD64-\\uDD67\\uDD70-\\uDEFB]|\\uD82F[\\uDC00-\\uDC6A\\uDC70-\\uDC7C\\uDC80-\\uDC88\\uDC90-\\uDC99\\uDC9D\\uDC9E]|\\uD833[\\uDF00-\\uDF2D\\uDF30-\\uDF46]|\\uD834[\\uDD65-\\uDD69\\uDD6D-\\uDD72\\uDD7B-\\uDD82\\uDD85-\\uDD8B\\uDDAA-\\uDDAD\\uDE42-\\uDE44]|\\uD835[\\uDC00-\\uDC54\\uDC56-\\uDC9C\\uDC9E\\uDC9F\\uDCA2\\uDCA5\\uDCA6\\uDCA9-\\uDCAC\\uDCAE-\\uDCB9\\uDCBB\\uDCBD-\\uDCC3\\uDCC5-\\uDD05\\uDD07-\\uDD0A\\uDD0D-\\uDD14\\uDD16-\\uDD1C\\uDD1E-\\uDD39\\uDD3B-\\uDD3E\\uDD40-\\uDD44\\uDD46\\uDD4A-\\uDD50\\uDD52-\\uDEA5\\uDEA8-\\uDEC0\\uDEC2-\\uDEDA\\uDEDC-\\uDEFA\\uDEFC-\\uDF14\\uDF16-\\uDF34\\uDF36-\\uDF4E\\uDF50-\\uDF6E\\uDF70-\\uDF88\\uDF8A-\\uDFA8\\uDFAA-\\uDFC2\\uDFC4-\\uDFCB\\uDFCE-\\uDFFF]|\\uD836[\\uDE00-\\uDE36\\uDE3B-\\uDE6C\\uDE75\\uDE84\\uDE9B-\\uDE9F\\uDEA1-\\uDEAF]|\\uD837[\\uDF00-\\uDF1E\\uDF25-\\uDF2A]|\\uD838[\\uDC00-\\uDC06\\uDC08-\\uDC18\\uDC1B-\\uDC21\\uDC23\\uDC24\\uDC26-\\uDC2A\\uDC30-\\uDC6D\\uDC8F\\uDD00-\\uDD2C\\uDD30-\\uDD3D\\uDD40-\\uDD49\\uDD4E\\uDE90-\\uDEAE\\uDEC0-\\uDEF9]|\\uD839[\\uDCD0-\\uDCF9\\uDFE0-\\uDFE6\\uDFE8-\\uDFEB\\uDFED\\uDFEE\\uDFF0-\\uDFFE]|\\uD83A[\\uDC00-\\uDCC4\\uDCD0-\\uDCD6\\uDD00-\\uDD4B\\uDD50-\\uDD59]|\\uD83B[\\uDE00-\\uDE03\\uDE05-\\uDE1F\\uDE21\\uDE22\\uDE24\\uDE27\\uDE29-\\uDE32\\uDE34-\\uDE37\\uDE39\\uDE3B\\uDE42\\uDE47\\uDE49\\uDE4B\\uDE4D-\\uDE4F\\uDE51\\uDE52\\uDE54\\uDE57\\uDE59\\uDE5B\\uDE5D\\uDE5F\\uDE61\\uDE62\\uDE64\\uDE67-\\uDE6A\\uDE6C-\\uDE72\\uDE74-\\uDE77\\uDE79-\\uDE7C\\uDE7E\\uDE80-\\uDE89\\uDE8B-\\uDE9B\\uDEA1-\\uDEA3\\uDEA5-\\uDEA9\\uDEAB-\\uDEBB]|\\uD83E[\\uDFF0-\\uDFF9]|\\uD869[\\uDC00-\\uDEDF\\uDF00-\\uDFFF]|\\uD86D[\\uDC00-\\uDF39\\uDF40-\\uDFFF]|\\uD86E[\\uDC00-\\uDC1D\\uDC20-\\uDFFF]|\\uD873[\\uDC00-\\uDEA1\\uDEB0-\\uDFFF]|\\uD87A[\\uDC00-\\uDFE0]|\\uD87E[\\uDC00-\\uDE1D]|\\uD884[\\uDC00-\\uDF4A\\uDF50-\\uDFFF]|\\uD888[\\uDC00-\\uDFAF]|\\uDB40[\\uDD00-\\uDDEF])*/,n=[&quot;and&quot;,&quot;as&quot;,&quot;assert&quot;,&quot;async&quot;,&quot;await&quot;,&quot;break&quot;,&quot;case&quot;,&quot;class&quot;,&quot;continue&quot;,&quot;def&quot;,&quot;del&quot;,&quot;elif&quot;,&quot;else&quot;,&quot;except&quot;,&quot;finally&quot;,&quot;for&quot;,&quot;from&quot;,&quot;global&quot;,&quot;if&quot;,&quot;import&quot;,&quot;in&quot;,&quot;is&quot;,&quot;lambda&quot;,&quot;match&quot;,&quot;nonlocal|10&quot;,&quot;not&quot;,&quot;or&quot;,&quot;pass&quot;,&quot;raise&quot;,&quot;return&quot;,&quot;try&quot;,&quot;while&quot;,&quot;with&quot;,&quot;yield&quot;],D={$pattern:/[A-Za-z]\\w+|__\\w+__/,keyword:n,built_in:[&quot;__import__&quot;,&quot;abs&quot;,&quot;all&quot;,&quot;any&quot;,&quot;ascii&quot;,&quot;bin&quot;,&quot;bool&quot;,&quot;breakpoint&quot;,&quot;bytearray&quot;,&quot;bytes&quot;,&quot;callable&quot;,&quot;chr&quot;,&quot;classmethod&quot;,&quot;compile&quot;,&quot;complex&quot;,&quot;delattr&quot;,&quot;dict&quot;,&quot;dir&quot;,&quot;divmod&quot;,&quot;enumerate&quot;,&quot;eval&quot;,&quot;exec&quot;,&quot;filter&quot;,&quot;float&quot;,&quot;format&quot;,&quot;frozenset&quot;,&quot;getattr&quot;,&quot;globals&quot;,&quot;hasattr&quot;,&quot;hash&quot;,&quot;help&quot;,&quot;hex&quot;,&quot;id&quot;,&quot;input&quot;,&quot;int&quot;,&quot;isinstance&quot;,&quot;issubclass&quot;,&quot;iter&quot;,&quot;len&quot;,&quot;list&quot;,&quot;locals&quot;,&quot;map&quot;,&quot;max&quot;,&quot;memoryview&quot;,&quot;min&quot;,&quot;next&quot;,&quot;object&quot;,&quot;oct&quot;,&quot;open&quot;,&quot;ord&quot;,&quot;pow&quot;,&quot;print&quot;,&quot;property&quot;,&quot;range&quot;,&quot;repr&quot;,&quot;reversed&quot;,&quot;round&quot;,&quot;set&quot;,&quot;setattr&quot;,&quot;slice&quot;,&quot;sorted&quot;,&quot;staticmethod&quot;,&quot;str&quot;,&quot;sum&quot;,&quot;super&quot;,&quot;tuple&quot;,&quot;type&quot;,&quot;vars&quot;,&quot;zip&quot;],literal:[&quot;__debug__&quot;,&quot;Ellipsis&quot;,&quot;False&quot;,&quot;None&quot;,&quot;NotImplemented&quot;,&quot;True&quot;],type:[&quot;Any&quot;,&quot;Callable&quot;,&quot;Coroutine&quot;,&quot;Dict&quot;,&quot;List&quot;,&quot;Literal&quot;,&quot;Generic&quot;,&quot;Optional&quot;,&quot;Sequence&quot;,&quot;Set&quot;,&quot;Tuple&quot;,&quot;Type&quot;,&quot;Union&quot;]},r={className:&quot;meta&quot;,begin:/^(&gt;&gt;&gt;|\\.\\.\\.) /},o={className:&quot;subst&quot;,begin:/\\{/,end:/\\}/,keywords:D,illegal:/#/},i={begin:/\\{\\{/,relevance:0},a={className:&quot;string&quot;,contains:[u.BACKSLASH_ESCAPE],variants:[{begin:/([uU]|[bB]|[rR]|[bB][rR]|[rR][bB])?&#x27;&#x27;&#x27;/,end:/&#x27;&#x27;&#x27;/,contains:[u.BACKSLASH_ESCAPE,r],relevance:10},{begin:/([uU]|[bB]|[rR]|[bB][rR]|[rR][bB])?&quot;&quot;&quot;/,end:/&quot;&quot;&quot;/,contains:[u.BACKSLASH_ESCAPE,r],relevance:10},{begin:/([fF][rR]|[rR][fF]|[fF])&#x27;&#x27;&#x27;/,end:/&#x27;&#x27;&#x27;/,contains:[u.BACKSLASH_ESCAPE,r,i,o]},{begin:/([fF][rR]|[rR][fF]|[fF])&quot;&quot;&quot;/,end:/&quot;&quot;&quot;/,contains:[u.BACKSLASH_ESCAPE,r,i,o]},{begin:/([uU]|[rR])&#x27;/,end:/&#x27;/,relevance:10},{begin:/([uU]|[rR])&quot;/,end:/&quot;/,relevance:10},{begin:/([bB]|[bB][rR]|[rR][bB])&#x27;/,end:/&#x27;/},{begin:/([bB]|[bB][rR]|[rR][bB])&quot;/,end:/&quot;/},{begin:/([fF][rR]|[rR][fF]|[fF])&#x27;/,end:/&#x27;/,contains:[u.BACKSLASH_ESCAPE,i,o]},{begin:/([fF][rR]|[rR][fF]|[fF])&quot;/,end:/&quot;/,contains:[u.BACKSLASH_ESCAPE,i,o]},u.APOS_STRING_MODE,u.QUOTE_STRING_MODE]},c=&quot;[0-9](_?[0-9])*&quot;,s=&quot;(\\\\b(&quot;.concat(c,&quot;))?\\\\.(&quot;).concat(c,&quot;)|\\\\b(&quot;).concat(c,&quot;)\\\\.&quot;),l=&quot;\\\\b|&quot;+n.join(&quot;|&quot;),F={className:&quot;number&quot;,relevance:0,variants:[{begin:&quot;(\\\\b(&quot;.concat(c,&quot;)|(&quot;).concat(s,&quot;))[eE][+-]?(&quot;).concat(c,&quot;)[jJ]?(?=&quot;).concat(l,&quot;)&quot;)},{begin:&quot;(&quot;.concat(s,&quot;)[jJ]?&quot;)},{begin:&quot;\\\\b([1-9](_?[0-9])*|0+(_?0)*)[lLjJ]?(?=&quot;.concat(l,&quot;)&quot;)},{begin:&quot;\\\\b0[bB](_?[01])+[lL]?(?=&quot;.concat(l,&quot;)&quot;)},{begin:&quot;\\\\b0[oO](_?[0-7])+[lL]?(?=&quot;.concat(l,&quot;)&quot;)},{begin:&quot;\\\\b0[xX](_?[0-9a-fA-F])+[lL]?(?=&quot;.concat(l,&quot;)&quot;)},{begin:&quot;\\\\b(&quot;.concat(c,&quot;)[jJ](?=&quot;).concat(l,&quot;)&quot;)}]},C={className:&quot;comment&quot;,begin:e.lookahead(/# type:/),end:/$/,keywords:D,contains:[{begin:/# type:/},{begin:/#/,end:/\\b\\B/,endsWithParent:!0}]},E={className:&quot;params&quot;,variants:[{className:&quot;&quot;,begin:/\\(\\s*\\)/,skip:!0},{begin:/\\(/,end:/\\)/,excludeBegin:!0,excludeEnd:!0,keywords:D,contains:[&quot;self&quot;,r,F,a,u.HASH_COMMENT_MODE]}]};return o.contains=[a,F,r],{name:&quot;Python&quot;,aliases:[&quot;py&quot;,&quot;gyp&quot;,&quot;ipython&quot;],unicodeRegex:!0,keywords:D,illegal:/(&lt;\\/|-&gt;|\\?)|=&gt;/,contains:[r,F,{begin:/\\bself\\b/},{beginKeywords:&quot;if&quot;,relevance:0},a,C,u.HASH_COMMENT_MODE,{match:[/\\bdef/,/\\s+/,t],scope:{1:&quot;keyword&quot;,3:&quot;title.function&quot;},contains:[E]},{variants:[{match:[/\\bclass/,/\\s+/,t,/\\s*/,/\\(\\s*/,t,/\\s*\\)/]},{match:[/\\bclass/,/\\s+/,t]}],scope:{1:&quot;keyword&quot;,3:&quot;title.class&quot;,6:&quot;title.class.inherited&quot;}},{className:&quot;meta&quot;,begin:/^[\\t ]*@/,end:/(?=#)|$/,contains:[F,E,a]}]}};hljs.registerLanguage(&quot;python&quot;,u)}();var CopyButtonPlugin=function(){function u(){var e=arguments.length&gt;0&amp;&amp;void 0!==arguments[0]?arguments[0]:{};_classCallCheck(this,u),self.hook=e.hook,self.callback=e.callback}return _createClass(u,[{key:&quot;after:highlightElement&quot;,value:function(u){var e=u.el,t=u.text,n=Object.assign(document.createElement(&quot;button&quot;),{innerHTML:&quot;Copy&quot;,className:&quot;hljs-copy-button&quot;});n.dataset.copied=!1,e.parentElement.classList.add(&quot;hljs-copy-wrapper&quot;),e.parentElement.appendChild(n),e.parentElement.style.setProperty(&quot;--hljs-theme-background&quot;,window.getComputedStyle(e).backgroundColor),n.onclick=function(){if(navigator.clipboard){var u=t;hook&amp;&amp;&quot;function&quot;==typeof hook&amp;&amp;(u=hook(t,e)||t),navigator.clipboard.writeText(u).then(function(){n.innerHTML=&quot; Copied! &quot;,n.dataset.copied=!0;var u=Object.assign(document.createElement(&quot;div&quot;),{role:&quot;status&quot;,className:&quot;hljs-copy-alert&quot;,innerHTML:&quot;Copied to clipboard&quot;});e.parentElement.appendChild(u),setTimeout(function(){n.innerHTML=&quot;Copy&quot;,n.dataset.copied=!1,e.parentElement.removeChild(u),u=null},2e3)}).then(function(){if(&quot;function&quot;==typeof callback)return callback(u,e)})}}}}]),u}();!function(){hljs.addPlugin(new CopyButtonPlugin),hljs.highlightAll(),document.querySelectorAll(&quot;#gsk-scan .gsk-issue&quot;).forEach(function(u){u.addEventListener(&quot;click&quot;,function(e){e.preventDefault(),u.classList.toggle(&quot;open&quot;),u.classList.toggle(&quot;bg-zinc-700&quot;)})});var u=document.querySelectorAll(&quot;#gsk-scan [role=&#x27;tabpanel&#x27;]&quot;),e=document.querySelectorAll(&quot;#gsk-scan [data-tab-target]&quot;);e.forEach(function(t){t.addEventListener(&quot;click&quot;,function(n){n.preventDefault();var D=t.getAttribute(&quot;data-tab-target&quot;);u.forEach(function(u){u.classList.add(&quot;hidden&quot;)}),e.forEach(function(u){u.classList.remove(&quot;active&quot;)}),t.classList.add(&quot;active&quot;),document.getElementById(D).classList.remove(&quot;hidden&quot;)})})}();\n",
       "&lt;/script&gt;\n",
       "\" style=\"width: 100%; border: none;\" class=\"gsk-scan\"></iframe>\n",
       "<script>\n",
       "\"use strict\";\n",
       "\n",
       "function _typeof(obj) { \"@babel/helpers - typeof\"; return _typeof = \"function\" == typeof Symbol && \"symbol\" == typeof Symbol.iterator ? function (obj) { return typeof obj; } : function (obj) { return obj && \"function\" == typeof Symbol && obj.constructor === Symbol && obj !== Symbol.prototype ? \"symbol\" : typeof obj; }, _typeof(obj); }\n",
       "/*! iFrame Resizer (iframeSizer.min.js ) - v4.3.5 - 2023-03-08\n",
       " *  Desc: Force cross domain iframes to size to content.\n",
       " *  Requires: iframeResizer.contentWindow.min.js to be loaded into the target frame.\n",
       " *  Copyright: (c) 2023 David J. Bradshaw - dave@bradshaw.net\n",
       " *  License: MIT\n",
       " */\n",
       "!function (d) {\n",
       "  var c, u, a, v, x, I, M, r, f, k, i, l, z;\n",
       "  function m() {\n",
       "    return window.MutationObserver || window.WebKitMutationObserver || window.MozMutationObserver;\n",
       "  }\n",
       "  function F(e, n, i) {\n",
       "    e.addEventListener(n, i, !1);\n",
       "  }\n",
       "  function B(e, n, i) {\n",
       "    e.removeEventListener(n, i, !1);\n",
       "  }\n",
       "  function p(e) {\n",
       "    return x + \"[\" + (n = \"Host page: \" + (e = e), n = window.top !== window.self ? window.parentIFrame && window.parentIFrame.getId ? window.parentIFrame.getId() + \": \" + e : \"Nested host page: \" + e : n) + \"]\";\n",
       "    var n;\n",
       "  }\n",
       "  function t(e) {\n",
       "    return k[e] ? k[e].log : u;\n",
       "  }\n",
       "  function O(e, n) {\n",
       "    o(\"log\", e, n, t(e));\n",
       "  }\n",
       "  function E(e, n) {\n",
       "    o(\"info\", e, n, t(e));\n",
       "  }\n",
       "  function R(e, n) {\n",
       "    o(\"warn\", e, n, !0);\n",
       "  }\n",
       "  function o(e, n, i, t) {\n",
       "    !0 === t && \"object\" == _typeof(window.console) && console[e](p(n), i);\n",
       "  }\n",
       "  function w(e) {\n",
       "    function i() {\n",
       "      t(\"Height\"), t(\"Width\"), P(function () {\n",
       "        H(w), C(b), l(\"onResized\", w);\n",
       "      }, w, \"init\");\n",
       "    }\n",
       "    function n() {\n",
       "      var e = p.slice(I).split(\":\"),\n",
       "        n = e[1] ? parseInt(e[1], 10) : 0,\n",
       "        i = k[e[0]] && k[e[0]].iframe,\n",
       "        t = getComputedStyle(i);\n",
       "      return {\n",
       "        iframe: i,\n",
       "        id: e[0],\n",
       "        height: n + function (e) {\n",
       "          if (\"border-box\" !== e.boxSizing) return 0;\n",
       "          var n = e.paddingTop ? parseInt(e.paddingTop, 10) : 0,\n",
       "            e = e.paddingBottom ? parseInt(e.paddingBottom, 10) : 0;\n",
       "          return n + e;\n",
       "        }(t) + function (e) {\n",
       "          if (\"border-box\" !== e.boxSizing) return 0;\n",
       "          var n = e.borderTopWidth ? parseInt(e.borderTopWidth, 10) : 0,\n",
       "            e = e.borderBottomWidth ? parseInt(e.borderBottomWidth, 10) : 0;\n",
       "          return n + e;\n",
       "        }(t),\n",
       "        width: e[2],\n",
       "        type: e[3]\n",
       "      };\n",
       "    }\n",
       "    function t(e) {\n",
       "      var n = Number(k[b][\"max\" + e]),\n",
       "        i = Number(k[b][\"min\" + e]),\n",
       "        e = e.toLowerCase(),\n",
       "        t = Number(w[e]);\n",
       "      O(b, \"Checking \" + e + \" is in range \" + i + \"-\" + n), t < i && (t = i, O(b, \"Set \" + e + \" to min value\")), n < t && (t = n, O(b, \"Set \" + e + \" to max value\")), w[e] = \"\" + t;\n",
       "    }\n",
       "    function o() {\n",
       "      var t = e.origin,\n",
       "        o = k[b] && k[b].checkOrigin;\n",
       "      if (o && \"\" + t != \"null\" && !function () {\n",
       "        if (o.constructor !== Array) return e = k[b] && k[b].remoteHost, O(b, \"Checking connection is from: \" + e), t === e;\n",
       "        var e,\n",
       "          n = 0,\n",
       "          i = !1;\n",
       "        for (O(b, \"Checking connection is from allowed list of origins: \" + o); n < o.length; n++) if (o[n] === t) {\n",
       "          i = !0;\n",
       "          break;\n",
       "        }\n",
       "        return i;\n",
       "      }()) throw new Error(\"Unexpected message received from: \" + t + \" for \" + w.iframe.id + \". Message was: \" + e.data + \". This error can be disabled by setting the checkOrigin: false option or by providing of array of trusted domains.\");\n",
       "      return 1;\n",
       "    }\n",
       "    function a(e) {\n",
       "      return p.slice(p.indexOf(\":\") + v + e);\n",
       "    }\n",
       "    function s(i, t) {\n",
       "      var e, n, o;\n",
       "      e = function e() {\n",
       "        var e, n;\n",
       "        A(\"Send Page Info\", \"pageInfo:\" + (e = document.body.getBoundingClientRect(), n = w.iframe.getBoundingClientRect(), JSON.stringify({\n",
       "          iframeHeight: n.height,\n",
       "          iframeWidth: n.width,\n",
       "          clientHeight: Math.max(document.documentElement.clientHeight, window.innerHeight || 0),\n",
       "          clientWidth: Math.max(document.documentElement.clientWidth, window.innerWidth || 0),\n",
       "          offsetTop: parseInt(n.top - e.top, 10),\n",
       "          offsetLeft: parseInt(n.left - e.left, 10),\n",
       "          scrollTop: window.pageYOffset,\n",
       "          scrollLeft: window.pageXOffset,\n",
       "          documentHeight: document.documentElement.clientHeight,\n",
       "          documentWidth: document.documentElement.clientWidth,\n",
       "          windowHeight: window.innerHeight,\n",
       "          windowWidth: window.innerWidth\n",
       "        })), i, t);\n",
       "      }, n = 32, z[o = t] || (z[o] = setTimeout(function () {\n",
       "        z[o] = null, e();\n",
       "      }, n));\n",
       "    }\n",
       "    function r(e) {\n",
       "      e = e.getBoundingClientRect();\n",
       "      return W(b), {\n",
       "        x: Math.floor(Number(e.left) + Number(M.x)),\n",
       "        y: Math.floor(Number(e.top) + Number(M.y))\n",
       "      };\n",
       "    }\n",
       "    function d(e) {\n",
       "      var n = e ? r(w.iframe) : {\n",
       "          x: 0,\n",
       "          y: 0\n",
       "        },\n",
       "        i = {\n",
       "          x: Number(w.width) + n.x,\n",
       "          y: Number(w.height) + n.y\n",
       "        };\n",
       "      O(b, \"Reposition requested from iFrame (offset x:\" + n.x + \" y:\" + n.y + \")\"), window.top === window.self ? (M = i, c(), O(b, \"--\")) : window.parentIFrame ? window.parentIFrame[\"scrollTo\" + (e ? \"Offset\" : \"\")](i.x, i.y) : R(b, \"Unable to scroll to requested position, window.parentIFrame not found\");\n",
       "    }\n",
       "    function c() {\n",
       "      !1 === l(\"onScroll\", M) ? S() : C(b);\n",
       "    }\n",
       "    function u(e) {\n",
       "      var e = e.split(\"#\")[1] || \"\",\n",
       "        n = decodeURIComponent(e),\n",
       "        n = document.getElementById(n) || document.getElementsByName(n)[0];\n",
       "      n ? (n = r(n), O(b, \"Moving to in page link (#\" + e + \") at x: \" + n.x + \" y: \" + n.y), M = {\n",
       "        x: n.x,\n",
       "        y: n.y\n",
       "      }, c(), O(b, \"--\")) : window.top === window.self ? O(b, \"In page link #\" + e + \" not found\") : window.parentIFrame ? window.parentIFrame.moveToAnchor(e) : O(b, \"In page link #\" + e + \" not found and window.parentIFrame not found\");\n",
       "    }\n",
       "    function f(e) {\n",
       "      var n,\n",
       "        i = {};\n",
       "      i = 0 === Number(w.width) && 0 === Number(w.height) ? {\n",
       "        x: (n = a(9).split(\":\"))[1],\n",
       "        y: n[0]\n",
       "      } : {\n",
       "        x: w.width,\n",
       "        y: w.height\n",
       "      }, l(e, {\n",
       "        iframe: w.iframe,\n",
       "        screenX: Number(i.x),\n",
       "        screenY: Number(i.y),\n",
       "        type: w.type\n",
       "      });\n",
       "    }\n",
       "    function l(e, n) {\n",
       "      return T(b, e, n);\n",
       "    }\n",
       "    function m() {\n",
       "      switch (k[b] && k[b].firstRun && k[b] && (k[b].firstRun = !1), w.type) {\n",
       "        case \"close\":\n",
       "          N(w.iframe);\n",
       "          break;\n",
       "        case \"message\":\n",
       "          n = a(6), O(b, \"onMessage passed: {iframe: \" + w.iframe.id + \", message: \" + n + \"}\"), l(\"onMessage\", {\n",
       "            iframe: w.iframe,\n",
       "            message: JSON.parse(n)\n",
       "          }), O(b, \"--\");\n",
       "          break;\n",
       "        case \"mouseenter\":\n",
       "          f(\"onMouseEnter\");\n",
       "          break;\n",
       "        case \"mouseleave\":\n",
       "          f(\"onMouseLeave\");\n",
       "          break;\n",
       "        case \"autoResize\":\n",
       "          k[b].autoResize = JSON.parse(a(9));\n",
       "          break;\n",
       "        case \"scrollTo\":\n",
       "          d(!1);\n",
       "          break;\n",
       "        case \"scrollToOffset\":\n",
       "          d(!0);\n",
       "          break;\n",
       "        case \"pageInfo\":\n",
       "          s(k[b] && k[b].iframe, b), r = b, e(\"Add \", F), k[r] && (k[r].stopPageInfo = o);\n",
       "          break;\n",
       "        case \"pageInfoStop\":\n",
       "          k[b] && k[b].stopPageInfo && (k[b].stopPageInfo(), delete k[b].stopPageInfo);\n",
       "          break;\n",
       "        case \"inPageLink\":\n",
       "          u(a(9));\n",
       "          break;\n",
       "        case \"reset\":\n",
       "          j(w);\n",
       "          break;\n",
       "        case \"init\":\n",
       "          i(), l(\"onInit\", w.iframe);\n",
       "          break;\n",
       "        default:\n",
       "          0 === Number(w.width) && 0 === Number(w.height) ? R(\"Unsupported message received (\" + w.type + \"), this is likely due to the iframe containing a later version of iframe-resizer than the parent page\") : i();\n",
       "      }\n",
       "      function e(n, i) {\n",
       "        function t() {\n",
       "          k[r] ? s(k[r].iframe, r) : o();\n",
       "        }\n",
       "        [\"scroll\", \"resize\"].forEach(function (e) {\n",
       "          O(r, n + e + \" listener for sendPageInfo\"), i(window, e, t);\n",
       "        });\n",
       "      }\n",
       "      function o() {\n",
       "        e(\"Remove \", B);\n",
       "      }\n",
       "      var r, n;\n",
       "    }\n",
       "    var g,\n",
       "      h,\n",
       "      p = e.data,\n",
       "      w = {},\n",
       "      b = null;\n",
       "    if (\"[iFrameResizerChild]Ready\" === p) for (var y in k) A(\"iFrame requested init\", L(y), k[y].iframe, y);else x === (\"\" + p).slice(0, I) && p.slice(I).split(\":\")[0] in k ? (w = n(), b = w.id, k[b] && (k[b].loaded = !0), (h = w.type in {\n",
       "      \"true\": 1,\n",
       "      \"false\": 1,\n",
       "      undefined: 1\n",
       "    }) && O(b, \"Ignoring init message from meta parent page\"), !h && (h = !0, k[g = b] || (h = !1, R(w.type + \" No settings for \" + g + \". Message was: \" + p)), h) && (O(b, \"Received: \" + p), g = !0, null === w.iframe && (R(b, \"IFrame (\" + w.id + \") not found\"), g = !1), g && o() && m())) : E(b, \"Ignored: \" + p);\n",
       "  }\n",
       "  function T(e, n, i) {\n",
       "    var t = null,\n",
       "      o = null;\n",
       "    if (k[e]) {\n",
       "      if (\"function\" != typeof (t = k[e][n])) throw new TypeError(n + \" on iFrame[\" + e + \"] is not a function\");\n",
       "      o = t(i);\n",
       "    }\n",
       "    return o;\n",
       "  }\n",
       "  function g(e) {\n",
       "    e = e.id;\n",
       "    delete k[e];\n",
       "  }\n",
       "  function N(e) {\n",
       "    var n = e.id;\n",
       "    if (!1 === T(n, \"onClose\", n)) O(n, \"Close iframe cancelled by onClose event\");else {\n",
       "      O(n, \"Removing iFrame: \" + n);\n",
       "      try {\n",
       "        e.parentNode && e.parentNode.removeChild(e);\n",
       "      } catch (e) {\n",
       "        R(e);\n",
       "      }\n",
       "      T(n, \"onClosed\", n), O(n, \"--\"), g(e);\n",
       "    }\n",
       "  }\n",
       "  function W(e) {\n",
       "    null === M && O(e, \"Get page position: \" + (M = {\n",
       "      x: window.pageXOffset === d ? document.documentElement.scrollLeft : window.pageXOffset,\n",
       "      y: window.pageYOffset === d ? document.documentElement.scrollTop : window.pageYOffset\n",
       "    }).x + \",\" + M.y);\n",
       "  }\n",
       "  function C(e) {\n",
       "    null !== M && (window.scrollTo(M.x, M.y), O(e, \"Set page position: \" + M.x + \",\" + M.y), S());\n",
       "  }\n",
       "  function S() {\n",
       "    M = null;\n",
       "  }\n",
       "  function j(e) {\n",
       "    O(e.id, \"Size reset requested by \" + (\"init\" === e.type ? \"host page\" : \"iFrame\")), W(e.id), P(function () {\n",
       "      H(e), A(\"reset\", \"reset\", e.iframe, e.id);\n",
       "    }, e, \"reset\");\n",
       "  }\n",
       "  function H(o) {\n",
       "    function i(e) {\n",
       "      var n;\n",
       "      function i() {\n",
       "        Object.keys(k).forEach(function (e) {\n",
       "          function n(e) {\n",
       "            return \"0px\" === (k[i] && k[i].iframe.style[e]);\n",
       "          }\n",
       "          var i;\n",
       "          k[i = e] && null !== k[i].iframe.offsetParent && (n(\"height\") || n(\"width\")) && A(\"Visibility change\", \"resize\", k[i].iframe, i);\n",
       "        });\n",
       "      }\n",
       "      function t(e) {\n",
       "        O(\"window\", \"Mutation observed: \" + e[0].target + \" \" + e[0].type), h(i, 16);\n",
       "      }\n",
       "      !a && \"0\" === o[e] && (a = !0, O(r, \"Hidden iFrame detected, creating visibility listener\"), e = m()) && (n = document.querySelector(\"body\"), new e(t).observe(n, {\n",
       "        attributes: !0,\n",
       "        attributeOldValue: !1,\n",
       "        characterData: !0,\n",
       "        characterDataOldValue: !1,\n",
       "        childList: !0,\n",
       "        subtree: !0\n",
       "      }));\n",
       "    }\n",
       "    function e(e) {\n",
       "      var n;\n",
       "      n = e, o.id ? (o.iframe.style[n] = o[n] + \"px\", O(o.id, \"IFrame (\" + r + \") \" + n + \" set to \" + o[n] + \"px\")) : O(\"undefined\", \"messageData id not set\"), i(e);\n",
       "    }\n",
       "    var r = o.iframe.id;\n",
       "    k[r] && (k[r].sizeHeight && e(\"height\"), k[r].sizeWidth) && e(\"width\");\n",
       "  }\n",
       "  function P(e, n, i) {\n",
       "    i !== n.type && r && !window.jasmine ? (O(n.id, \"Requesting animation frame\"), r(e)) : e();\n",
       "  }\n",
       "  function A(n, i, t, o, e) {\n",
       "    function r() {\n",
       "      var e;\n",
       "      t && \"contentWindow\" in t && null !== t.contentWindow ? (e = k[o] && k[o].targetOrigin, O(o, \"[\" + n + \"] Sending msg to iframe[\" + o + \"] (\" + i + \") targetOrigin: \" + e), t.contentWindow.postMessage(x + i, e)) : R(o, \"[\" + n + \"] IFrame(\" + o + \") not found\");\n",
       "    }\n",
       "    function a() {\n",
       "      e && k[o] && k[o].warningTimeout && (k[o].msgTimeout = setTimeout(function () {\n",
       "        !k[o] || k[o].loaded || s || (s = !0, R(o, \"IFrame has not responded within \" + k[o].warningTimeout / 1e3 + \" seconds. Check iFrameResizer.contentWindow.js has been loaded in iFrame. This message can be ignored if everything is working, or you can set the warningTimeout option to a higher value or zero to suppress this warning.\"));\n",
       "      }, k[o].warningTimeout));\n",
       "    }\n",
       "    var s = !1;\n",
       "    o = o || t.id, k[o] && (r(), a());\n",
       "  }\n",
       "  function L(e) {\n",
       "    return e + \":\" + k[e].bodyMarginV1 + \":\" + k[e].sizeWidth + \":\" + k[e].log + \":\" + k[e].interval + \":\" + k[e].enablePublicMethods + \":\" + k[e].autoResize + \":\" + k[e].bodyMargin + \":\" + k[e].heightCalculationMethod + \":\" + k[e].bodyBackground + \":\" + k[e].bodyPadding + \":\" + k[e].tolerance + \":\" + k[e].inPageLinks + \":\" + k[e].resizeFrom + \":\" + k[e].widthCalculationMethod + \":\" + k[e].mouseEvents;\n",
       "  }\n",
       "  function s(t, i) {\n",
       "    function e(i) {\n",
       "      var e = m();\n",
       "      e && (e = e, t.parentNode) && new e(function (e) {\n",
       "        e.forEach(function (e) {\n",
       "          Array.prototype.slice.call(e.removedNodes).forEach(function (e) {\n",
       "            e === t && N(t);\n",
       "          });\n",
       "        });\n",
       "      }).observe(t.parentNode, {\n",
       "        childList: !0\n",
       "      }), F(t, \"load\", function () {\n",
       "        var e, n;\n",
       "        A(\"iFrame.onload\", i, t, d, !0), e = k[r] && k[r].firstRun, n = k[r] && k[r].heightCalculationMethod in f, !e && n && j({\n",
       "          iframe: t,\n",
       "          height: 0,\n",
       "          width: 0,\n",
       "          type: \"init\"\n",
       "        });\n",
       "      }), A(\"init\", i, t, d, !0);\n",
       "    }\n",
       "    function o(e) {\n",
       "      var n = e.split(\"Callback\");\n",
       "      2 === n.length && (this[n = \"on\" + n[0].charAt(0).toUpperCase() + n[0].slice(1)] = this[e], delete this[e], R(r, \"Deprecated: '\" + e + \"' has been renamed '\" + n + \"'. The old method will be removed in the next major version.\"));\n",
       "    }\n",
       "    function n(e) {\n",
       "      if (e = e || {}, k[r] = Object.create(null), k[r].iframe = t, k[r].firstRun = !0, k[r].remoteHost = t.src && t.src.split(\"/\").slice(0, 3).join(\"/\"), \"object\" != _typeof(e)) throw new TypeError(\"Options is not an object\");\n",
       "      Object.keys(e).forEach(o, e);\n",
       "      var n,\n",
       "        i = e;\n",
       "      for (n in l) Object.prototype.hasOwnProperty.call(l, n) && (k[r][n] = (Object.prototype.hasOwnProperty.call(i, n) ? i : l)[n]);\n",
       "      k[r] && (k[r].targetOrigin = !0 !== k[r].checkOrigin || \"\" === (e = k[r].remoteHost) || null !== e.match(/^(about:blank|javascript:|file:\\/\\/)/) ? \"*\" : e);\n",
       "    }\n",
       "    var r = function (e) {\n",
       "      if (\"string\" != typeof e) throw new TypeError(\"Invaild id for iFrame. Expected String\");\n",
       "      var n;\n",
       "      return \"\" === e && (t.id = (n = i && i.id || l.id + c++, null !== document.getElementById(n) && (n += c++), e = n), u = (i || {}).log, O(e, \"Added missing iframe ID: \" + e + \" (\" + t.src + \")\")), e;\n",
       "    }(t.id);\n",
       "    if (r in k && \"iFrameResizer\" in t) R(r, \"Ignored iFrame, already setup.\");else {\n",
       "      switch (n(i), O(r, \"IFrame scrolling \" + (k[r] && k[r].scrolling ? \"enabled\" : \"disabled\") + \" for \" + r), t.style.overflow = !1 === (k[r] && k[r].scrolling) ? \"hidden\" : \"auto\", k[r] && k[r].scrolling) {\n",
       "        case \"omit\":\n",
       "          break;\n",
       "        case !0:\n",
       "          t.scrolling = \"yes\";\n",
       "          break;\n",
       "        case !1:\n",
       "          t.scrolling = \"no\";\n",
       "          break;\n",
       "        default:\n",
       "          t.scrolling = k[r] ? k[r].scrolling : \"no\";\n",
       "      }\n",
       "      s(\"Height\"), s(\"Width\"), a(\"maxHeight\"), a(\"minHeight\"), a(\"maxWidth\"), a(\"minWidth\"), \"number\" != typeof (k[r] && k[r].bodyMargin) && \"0\" !== (k[r] && k[r].bodyMargin) || (k[r].bodyMarginV1 = k[r].bodyMargin, k[r].bodyMargin = k[r].bodyMargin + \"px\"), e(L(r)), k[r] && (k[r].iframe.iFrameResizer = {\n",
       "        close: N.bind(null, k[r].iframe),\n",
       "        removeListeners: g.bind(null, k[r].iframe),\n",
       "        resize: A.bind(null, \"Window resize\", \"resize\", k[r].iframe),\n",
       "        moveToAnchor: function moveToAnchor(e) {\n",
       "          A(\"Move to anchor\", \"moveToAnchor:\" + e, k[r].iframe, r);\n",
       "        },\n",
       "        sendMessage: function sendMessage(e) {\n",
       "          A(\"Send Message\", \"message:\" + (e = JSON.stringify(e)), k[r].iframe, r);\n",
       "        }\n",
       "      });\n",
       "    }\n",
       "    function a(e) {\n",
       "      var n = k[r][e];\n",
       "      1 / 0 !== n && 0 !== n && (t.style[e] = \"number\" == typeof n ? n + \"px\" : n, O(r, \"Set \" + e + \" = \" + t.style[e]));\n",
       "    }\n",
       "    function s(e) {\n",
       "      if (k[r][\"min\" + e] > k[r][\"max\" + e]) throw new Error(\"Value for min\" + e + \" can not be greater than max\" + e);\n",
       "    }\n",
       "  }\n",
       "  function h(e, n) {\n",
       "    null === i && (i = setTimeout(function () {\n",
       "      i = null, e();\n",
       "    }, n));\n",
       "  }\n",
       "  function e() {\n",
       "    \"hidden\" !== document.visibilityState && (O(\"document\", \"Trigger event: Visibility change\"), h(function () {\n",
       "      b(\"Tab Visible\", \"resize\");\n",
       "    }, 16));\n",
       "  }\n",
       "  function b(i, t) {\n",
       "    Object.keys(k).forEach(function (e) {\n",
       "      var n;\n",
       "      k[n = e] && \"parent\" === k[n].resizeFrom && k[n].autoResize && !k[n].firstRun && A(i, t, k[e].iframe, e);\n",
       "    });\n",
       "  }\n",
       "  function y() {\n",
       "    F(window, \"message\", w), F(window, \"resize\", function () {\n",
       "      var e;\n",
       "      O(\"window\", \"Trigger event: \" + (e = \"resize\")), h(function () {\n",
       "        b(\"Window \" + e, \"resize\");\n",
       "      }, 16);\n",
       "    }), F(document, \"visibilitychange\", e), F(document, \"-webkit-visibilitychange\", e);\n",
       "  }\n",
       "  function n() {\n",
       "    function t(e, n) {\n",
       "      if (n) {\n",
       "        if (!n.tagName) throw new TypeError(\"Object is not a valid DOM element\");\n",
       "        if (\"IFRAME\" !== n.tagName.toUpperCase()) throw new TypeError(\"Expected <IFRAME> tag, found <\" + n.tagName + \">\");\n",
       "        s(n, e), o.push(n);\n",
       "      }\n",
       "    }\n",
       "    for (var o, e = [\"moz\", \"webkit\", \"o\", \"ms\"], n = 0; n < e.length && !r; n += 1) r = window[e[n] + \"RequestAnimationFrame\"];\n",
       "    return r ? r = r.bind(window) : O(\"setup\", \"RequestAnimationFrame not supported\"), y(), function (e, n) {\n",
       "      var i;\n",
       "      switch (o = [], (i = e) && i.enablePublicMethods && R(\"enablePublicMethods option has been removed, public methods are now always available in the iFrame\"), _typeof(n)) {\n",
       "        case \"undefined\":\n",
       "        case \"string\":\n",
       "          Array.prototype.forEach.call(document.querySelectorAll(n || \"iframe\"), t.bind(d, e));\n",
       "          break;\n",
       "        case \"object\":\n",
       "          t(e, n);\n",
       "          break;\n",
       "        default:\n",
       "          throw new TypeError(\"Unexpected data type (\" + _typeof(n) + \")\");\n",
       "      }\n",
       "      return o;\n",
       "    };\n",
       "  }\n",
       "  function q(e) {\n",
       "    e.fn ? e.fn.iFrameResize || (e.fn.iFrameResize = function (i) {\n",
       "      return this.filter(\"iframe\").each(function (e, n) {\n",
       "        s(n, i);\n",
       "      }).end();\n",
       "    }) : E(\"\", \"Unable to bind to jQuery, it is not fully loaded.\");\n",
       "  }\n",
       "  \"undefined\" != typeof window && (c = 0, a = u = !1, v = \"message\".length, I = (x = \"[iFrameSizer]\").length, M = null, r = window.requestAnimationFrame, f = Object.freeze({\n",
       "    max: 1,\n",
       "    scroll: 1,\n",
       "    bodyScroll: 1,\n",
       "    documentElementScroll: 1\n",
       "  }), k = {}, i = null, l = Object.freeze({\n",
       "    autoResize: !0,\n",
       "    bodyBackground: null,\n",
       "    bodyMargin: null,\n",
       "    bodyMarginV1: 8,\n",
       "    bodyPadding: null,\n",
       "    checkOrigin: !0,\n",
       "    inPageLinks: !1,\n",
       "    enablePublicMethods: !0,\n",
       "    heightCalculationMethod: \"bodyOffset\",\n",
       "    id: \"iFrameResizer\",\n",
       "    interval: 32,\n",
       "    log: !1,\n",
       "    maxHeight: 1 / 0,\n",
       "    maxWidth: 1 / 0,\n",
       "    minHeight: 0,\n",
       "    minWidth: 0,\n",
       "    mouseEvents: !0,\n",
       "    resizeFrom: \"parent\",\n",
       "    scrolling: !1,\n",
       "    sizeHeight: !0,\n",
       "    sizeWidth: !1,\n",
       "    warningTimeout: 5e3,\n",
       "    tolerance: 0,\n",
       "    widthCalculationMethod: \"scroll\",\n",
       "    onClose: function onClose() {\n",
       "      return !0;\n",
       "    },\n",
       "    onClosed: function onClosed() {},\n",
       "    onInit: function onInit() {},\n",
       "    onMessage: function onMessage() {\n",
       "      R(\"onMessage function not defined\");\n",
       "    },\n",
       "    onMouseEnter: function onMouseEnter() {},\n",
       "    onMouseLeave: function onMouseLeave() {},\n",
       "    onResized: function onResized() {},\n",
       "    onScroll: function onScroll() {\n",
       "      return !0;\n",
       "    }\n",
       "  }), z = {}, window.jQuery !== d && q(window.jQuery), \"function\" == typeof define && define.amd ? define([], n) : \"object\" == (typeof module === \"undefined\" ? \"undefined\" : _typeof(module)) && \"object\" == _typeof(module.exports) && (module.exports = n()), window.iFrameResize = window.iFrameResize || n());\n",
       "}();\n",
       "(function(){iFrameResize({ checkOrigin: false }, '#scan-4947914384');})();\n",
       "</script>\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "display(results)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Generate a test suite from the Scan\n",
    "The objects produced by the scan can be used as fixtures to generate a test suite that integrate domain-specific issues. To create custom tests, refer to the Test your ML Model page."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T16:03:03.890497Z",
     "start_time": "2023-06-02T16:02:59.774550Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Executed 'F1 Score on data slice “`transcription` contains \"xyz\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x126210cd0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.32\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"subjective\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x1261111b0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.37\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"admission\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x1262f28f0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.38\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"daily\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12628cbb0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.38\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"coronary\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12615aa70>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.39\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"followup\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12245e260>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice “`transcription` contains \"count\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12629c790>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice “`transcription` contains \"lung\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x126138a90>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"function\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12615a3b0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.41\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"abc\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x1262250f0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"improved\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x126256bf0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"discharge\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12629e710>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"aspirin\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x126212590>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.43\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"greater\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x1261a2410>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.43\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"continue\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x126214d60>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.44\n",
      "               \n",
      "               \n",
      "Executed 'Invariance to “Add typos”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'transformation_function': <giskard.scanner.robustness.text_transformations.TextTypoTransformation object at 0x1263d1cf0>, 'threshold': 0.95, 'output_sensitivity': 0.05}: \n",
      "               Test succeed\n",
      "               Metric: 0.96\n",
      "                - [TestMessageLevel.INFO] 369 rows were perturbed\n",
      "               \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "               <h2><span style=\"color:red;\">×</span> Test suite failed</h2>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"xyz\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.32<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"subjective\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.37<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"admission\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.38<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"daily\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.38<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"coronary\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.39<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"followup\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.4<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: Precision on data slice “`transcription` contains \"count\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.4<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: Precision on data slice “`transcription` contains \"lung\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.4<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"function\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.41<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"abc\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.42<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"improved\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.42<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"discharge\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.42<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"aspirin\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.43<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"greater\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.43<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"continue\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.44<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: Invariance to “Add typos”</h3>\n",
       "               <h4><span style=\"color:green;\">✓</span> Test succeed</h4>\n",
       "               <p>Metric: 0.96<p>\n",
       "               <ul>\n",
       "               <li style=\"color:inherit;\">369 rows were perturbed</li>\n",
       "               </ul>\n",
       "               \n",
       "               "
      ],
      "text/plain": [
       "(False,\n",
       " [('F1 Score on data slice “`transcription` contains \"xyz\"”', \n",
       "                  Test failed\n",
       "                  Metric: 0.32\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"subjective\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.37\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"admission\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.38\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"daily\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.38\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"coronary\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.39\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"followup\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.4\n",
       "                  \n",
       "                  ),\n",
       "  ('Precision on data slice “`transcription` contains \"count\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.4\n",
       "                  \n",
       "                  ),\n",
       "  ('Precision on data slice “`transcription` contains \"lung\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.4\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"function\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.41\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"abc\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.42\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"improved\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.42\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"discharge\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.42\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"aspirin\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.43\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"greater\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.43\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"continue\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.44\n",
       "                  \n",
       "                  ),\n",
       "  ('Invariance to “Add typos”',\n",
       "   \n",
       "                  Test succeed\n",
       "                  Metric: 0.96\n",
       "                   - [TestMessageLevel.INFO] 369 rows were perturbed\n",
       "                  )])"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_suite = results.generate_test_suite(\"My first test suite\")\n",
    "test_suite.run()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Customize your suite by loading objects from the Giskard catalog\n",
    "\n",
    "The Giskard open source catalog will enable to load:\n",
    "* Tests such as metamorphic, performance, prediction & data drift, statistical tests, etc\n",
    "* Slicing functions such as detectors of toxicity, hate, emotion, etc\n",
    "* Transformation functions such as generators of typos, paraphrase, style tune, etc\n",
    "\n",
    "For demo purposes, we will load a simple unit test (test_f1) that checks if the test F1 score is above the given threshold. For more examples of tests and functions, refer to the Giskard catalog."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T16:03:54.282167Z",
     "start_time": "2023-06-02T16:03:50.135561Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Executed 'F1 Score on data slice “`transcription` contains \"xyz\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x126210cd0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.32\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"subjective\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x1261111b0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.37\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"admission\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x1262f28f0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.38\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"daily\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12628cbb0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.38\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"coronary\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12615aa70>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.39\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"followup\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12245e260>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice “`transcription` contains \"count\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12629c790>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice “`transcription` contains \"lung\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x126138a90>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"function\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12615a3b0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.41\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"abc\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x1262250f0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"improved\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x126256bf0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"discharge\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x12629e710>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"aspirin\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x126212590>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.43\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"greater\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x1261a2410>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.43\n",
      "               \n",
      "               \n",
      "Executed 'F1 Score on data slice “`transcription` contains \"continue\"”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x126214d60>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.44\n",
      "               \n",
      "               \n",
      "Executed 'Invariance to “Add typos”' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'transformation_function': <giskard.scanner.robustness.text_transformations.TextTypoTransformation object at 0x1263d1cf0>, 'threshold': 0.95, 'output_sensitivity': 0.05}: \n",
      "               Test failed\n",
      "               Metric: 0.94\n",
      "                - [TestMessageLevel.INFO] 371 rows were perturbed\n",
      "               \n",
      "Executed 'F1' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'threshold': 0.7}: \n",
      "               Test failed\n",
      "               Metric: 0.61\n",
      "               \n",
      "               \n",
      "Executed 'F1' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x12245e380>, 'dataset': <giskard.datasets.base.Dataset object at 0x122397c70>, 'threshold': 0.7}: \n",
      "               Test failed\n",
      "               Metric: 0.61\n",
      "               \n",
      "               \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "               <h2><span style=\"color:red;\">×</span> Test suite failed</h2>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"xyz\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.32<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"subjective\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.37<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"admission\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.38<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"daily\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.38<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"coronary\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.39<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"followup\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.4<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: Precision on data slice “`transcription` contains \"count\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.4<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: Precision on data slice “`transcription` contains \"lung\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.4<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"function\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.41<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"abc\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.42<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"improved\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.42<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"discharge\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.42<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"aspirin\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.43<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"greater\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.43<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1 Score on data slice “`transcription` contains \"continue\"”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.44<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: Invariance to “Add typos”</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.94<p>\n",
       "               <ul>\n",
       "               <li style=\"color:inherit;\">371 rows were perturbed</li>\n",
       "               </ul>\n",
       "               <h3>Test: F1</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.61<p>\n",
       "               <ul></ul>\n",
       "               <h3>Test: F1</h3>\n",
       "               <h4><span style=\"color:red;\">𐄂</span> Test failed</h4>\n",
       "               <p>Metric: 0.61<p>\n",
       "               <ul></ul>\n",
       "               \n",
       "               "
      ],
      "text/plain": [
       "(False,\n",
       " [('F1 Score on data slice “`transcription` contains \"xyz\"”', \n",
       "                  Test failed\n",
       "                  Metric: 0.32\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"subjective\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.37\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"admission\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.38\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"daily\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.38\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"coronary\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.39\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"followup\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.4\n",
       "                  \n",
       "                  ),\n",
       "  ('Precision on data slice “`transcription` contains \"count\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.4\n",
       "                  \n",
       "                  ),\n",
       "  ('Precision on data slice “`transcription` contains \"lung\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.4\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"function\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.41\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"abc\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.42\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"improved\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.42\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"discharge\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.42\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"aspirin\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.43\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"greater\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.43\n",
       "                  \n",
       "                  ),\n",
       "  ('F1 Score on data slice “`transcription` contains \"continue\"”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.44\n",
       "                  \n",
       "                  ),\n",
       "  ('Invariance to “Add typos”',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.94\n",
       "                   - [TestMessageLevel.INFO] 371 rows were perturbed\n",
       "                  ),\n",
       "  ('F1',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.61\n",
       "                  \n",
       "                  ),\n",
       "  ('F1',\n",
       "   \n",
       "                  Test failed\n",
       "                  Metric: 0.61\n",
       "                  \n",
       "                  )])"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_suite.add_test(testing.test_f1(model=wrapped_model, dataset=wrapped_data, threshold=0.7)).run()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Upload your suite to the Giskard server\n",
    "\n",
    "Upload your suite to the Giskard server to:\n",
    "* Compare models to decide which model to promote\n",
    "* Debug your tests to diagnose the issues\n",
    "* Create more domain-specific tests that are integrating business feedback\n",
    "* Share your results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# Uploading the test suite will automatically save the model, dataset, tests, slicing & transformation functions inside the Giskard UI server\n",
    "# Create a Giskard client after having install the Giskard server (see documentation)\n",
    "token = \"API_TOKEN\"  # Find it in Settings in the Giskard server\n",
    "\n",
    "client = GiskardClient(\n",
    "    url=\"http://localhost:19000\",  # URL of your Giskard instance\n",
    "    token=token\n",
    ")\n",
    "\n",
    "my_project = client.create_project(\"my_project\", \"PROJECT_NAME\", \"DESCRIPTION\")\n",
    "\n",
    "# Upload to the current project ✉️\n",
    "test_suite.upload(client, \"my_project\")"
   ]
  },
  {
   "cell_type": "markdown",
   "source": [
    "<div class=\"alert alert-info\">\n",
    "Connecting Google Colab with the Giskard server\n",
    "\n",
    "If you are using Google Colab and you want to install the Giskard server **locally**, you can run the Giskard server by executing this line in the terminal of your **local** machine (see the [documentation]https://docs.giskard.ai/en/latest/guides/installation_app/index.html)):\n",
    "\n",
    "> giskard server start\n",
    "\n",
    "Once the Giskard server is running, from the same terminal on your **local** machine, you can run:\n",
    "\n",
    "> giskard server expose\n",
    "\n",
    "This will provide you with the code snippets that you can copy and paste into your Colab notebook to establish a connection with your locally installed Giskard server\n",
    "</div>"
   ],
   "metadata": {
    "collapsed": false
   }
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.12"
  },
  "papermill": {
   "default_parameters": {},
   "duration": 1295.42731,
   "end_time": "2022-09-14T15:45:13.308486",
   "environment_variables": {},
   "exception": null,
   "input_path": "__notebook__.ipynb",
   "output_path": "__notebook__.ipynb",
   "parameters": {},
   "start_time": "2022-09-14T15:23:37.881176",
   "version": "2.3.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
