{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "# Medical transcript classification [sklearn]\n",
    "* Multiclass classification of medical transcript.\n",
    "* Reference notebook: <https://www.kaggle.com/code/leekahwin/text-classification-using-n-gram-0-8-f1/notebook>\n",
    "* Dataset: <https://www.kaggle.com/code/leekahwin/text-classification-using-n-gram-0-8-f1/input>\n",
    "\n",
    "By running this notebook, youâ€™ll create a whole test suite in a few lines of code. The model used here is a Random Forest classification model with the medical transcript dataset. Feel free to use your own model (tabular, text, or LLM).\n",
    "\n",
    "Youâ€™ll learn how to:\n",
    "\n",
    "* Detect vulnerabilities by scanning the model\n",
    "* Generate a test suite with domain-specific tests\n",
    "* Customize your test suite by loading a test from the Giskard catalog\n",
    "* Upload your model to the Giskard server to:\n",
    "    * Compare models to decide which one to promote\n",
    "    * Debug your tests to diagnose issues\n",
    "    * Share your results and collect business feedback from your team"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Install Giskard"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 207,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: giskard>=2.0.0b in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (2.0.0b12)\n",
      "Collecting giskard>=2.0.0b\n",
      "  Obtaining dependency information for giskard>=2.0.0b from https://files.pythonhosted.org/packages/ad/60/a2f0581afeb4368acfd165ecdd430adc4dc9f9559ce1caf5873cf7b02613/giskard-2.0.0b14-py3-none-any.whl.metadata\n",
      "  Using cached giskard-2.0.0b14-py3-none-any.whl.metadata (13 kB)\n",
      "Requirement already satisfied: cloudpickle>=1.1.1 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (2.2.1)\n",
      "Requirement already satisfied: zstandard>=0.10.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (0.21.0)\n",
      "Requirement already satisfied: mlflow-skinny>=2 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (2.5.0)\n",
      "Requirement already satisfied: numpy<1.24.0,>=1.22.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (1.23.5)\n",
      "Requirement already satisfied: scikit-learn>=1.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (1.0.2)\n",
      "Requirement already satisfied: scipy>=1.7.3 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (1.10.1)\n",
      "Requirement already satisfied: mixpanel>=4.4.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (4.10.0)\n",
      "Requirement already satisfied: requests>=2.19 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (2.31.0)\n",
      "Requirement already satisfied: pydantic<2,>=1.7 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (1.10.11)\n",
      "Requirement already satisfied: tqdm>=4.42.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (4.65.0)\n",
      "Requirement already satisfied: setuptools<68.0.0,>=39.1.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (67.8.0)\n",
      "Requirement already satisfied: pandas<2,>=1.3.4 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (1.5.3)\n",
      "Requirement already satisfied: xxhash>=3.2.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (3.2.0)\n",
      "Requirement already satisfied: langdetect>=1.0.9 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (1.0.9)\n",
      "Requirement already satisfied: chardet in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (5.1.0)\n",
      "Requirement already satisfied: jinja2>=3 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (3.1.2)\n",
      "Requirement already satisfied: requests-toolbelt>=0.9.1 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from giskard>=2.0.0b) (1.0.0)\n",
      "Requirement already satisfied: MarkupSafe>=2.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from jinja2>=3->giskard>=2.0.0b) (2.1.3)\n",
      "Requirement already satisfied: six in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from langdetect>=1.0.9->giskard>=2.0.0b) (1.16.0)\n",
      "Requirement already satisfied: urllib3 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mixpanel>=4.4.0->giskard>=2.0.0b) (1.26.15)\n",
      "Requirement already satisfied: click<9,>=7.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mlflow-skinny>=2->giskard>=2.0.0b) (8.1.6)\n",
      "Requirement already satisfied: databricks-cli<1,>=0.8.7 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mlflow-skinny>=2->giskard>=2.0.0b) (0.17.7)\n",
      "Requirement already satisfied: entrypoints<1 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mlflow-skinny>=2->giskard>=2.0.0b) (0.4)\n",
      "Requirement already satisfied: gitpython<4,>=2.1.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mlflow-skinny>=2->giskard>=2.0.0b) (3.1.31)\n",
      "Requirement already satisfied: pyyaml<7,>=5.1 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mlflow-skinny>=2->giskard>=2.0.0b) (6.0)\n",
      "Requirement already satisfied: protobuf<5,>=3.12.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mlflow-skinny>=2->giskard>=2.0.0b) (3.19.6)\n",
      "Requirement already satisfied: pytz<2024 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mlflow-skinny>=2->giskard>=2.0.0b) (2023.3)\n",
      "Requirement already satisfied: packaging<24 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mlflow-skinny>=2->giskard>=2.0.0b) (23.1)\n",
      "Requirement already satisfied: importlib-metadata!=4.7.0,<7,>=3.7.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mlflow-skinny>=2->giskard>=2.0.0b) (6.7.0)\n",
      "Requirement already satisfied: sqlparse<1,>=0.4.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from mlflow-skinny>=2->giskard>=2.0.0b) (0.4.4)\n",
      "Requirement already satisfied: python-dateutil>=2.8.1 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from pandas<2,>=1.3.4->giskard>=2.0.0b) (2.8.2)\n",
      "Requirement already satisfied: typing-extensions>=4.2.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from pydantic<2,>=1.7->giskard>=2.0.0b) (4.7.1)\n",
      "Requirement already satisfied: charset-normalizer<4,>=2 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from requests>=2.19->giskard>=2.0.0b) (3.1.0)\n",
      "Requirement already satisfied: idna<4,>=2.5 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from requests>=2.19->giskard>=2.0.0b) (3.4)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from requests>=2.19->giskard>=2.0.0b) (2023.5.7)\n",
      "Requirement already satisfied: joblib>=0.11 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from scikit-learn>=1.0->giskard>=2.0.0b) (1.3.1)\n",
      "Requirement already satisfied: threadpoolctl>=2.0.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from scikit-learn>=1.0->giskard>=2.0.0b) (3.1.0)\n",
      "Requirement already satisfied: pyjwt>=1.7.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from databricks-cli<1,>=0.8.7->mlflow-skinny>=2->giskard>=2.0.0b) (2.7.0)\n",
      "Requirement already satisfied: oauthlib>=3.1.0 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from databricks-cli<1,>=0.8.7->mlflow-skinny>=2->giskard>=2.0.0b) (3.2.2)\n",
      "Requirement already satisfied: tabulate>=0.7.7 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from databricks-cli<1,>=0.8.7->mlflow-skinny>=2->giskard>=2.0.0b) (0.9.0)\n",
      "Requirement already satisfied: gitdb<5,>=4.0.1 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from gitpython<4,>=2.1.0->mlflow-skinny>=2->giskard>=2.0.0b) (4.0.10)\n",
      "Requirement already satisfied: zipp>=0.5 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from importlib-metadata!=4.7.0,<7,>=3.7.0->mlflow-skinny>=2->giskard>=2.0.0b) (3.15.0)\n",
      "Requirement already satisfied: smmap<6,>=3.0.1 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from gitdb<5,>=4.0.1->gitpython<4,>=2.1.0->mlflow-skinny>=2->giskard>=2.0.0b) (5.0.0)\n",
      "Using cached giskard-2.0.0b14-py3-none-any.whl (385 kB)\n",
      "Installing collected packages: giskard\n",
      "  Attempting uninstall: giskard\n",
      "    Found existing installation: giskard 2.0.0b12\n",
      "    Uninstalling giskard-2.0.0b12:\n",
      "      Successfully uninstalled giskard-2.0.0b12\n",
      "Successfully installed giskard-2.0.0b14\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "pip install \"giskard>=2.0.0b\" -U"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Install necessary dependencies"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 208,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: nltk in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (3.8.1)\n",
      "Requirement already satisfied: click in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from nltk) (8.1.6)\n",
      "Requirement already satisfied: joblib in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from nltk) (1.3.1)\n",
      "Requirement already satisfied: regex>=2021.8.3 in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from nltk) (2023.6.3)\n",
      "Requirement already satisfied: tqdm in /Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages (from nltk) (4.65.0)\n"
     ]
    }
   ],
   "source": [
    "!pip install nltk"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Import libraries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 209,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:27.771693Z",
     "start_time": "2023-06-02T15:56:22.764215Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import os\n",
    "import string\n",
    "from pathlib import Path\n",
    "from typing import Iterable\n",
    "from urllib.request import urlretrieve\n",
    "\n",
    "import nltk\n",
    "import pandas as pd\n",
    "from nltk.corpus import stopwords\n",
    "from nltk.stem.snowball import SnowballStemmer\n",
    "from sklearn.ensemble import RandomForestClassifier\n",
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "from sklearn.metrics import classification_report\n",
    "from sklearn.model_selection import train_test_split\n",
    "from sklearn.pipeline import Pipeline\n",
    "from sklearn.preprocessing import FunctionTransformer\n",
    "\n",
    "import giskard\n",
    "from giskard import Dataset, Model, GiskardClient, testing"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Download NLTK stopwords corpus"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 210,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package stopwords to\n",
      "[nltk_data]     /Users/mathieuroques/nltk_data...\n",
      "[nltk_data]   Package stopwords is already up-to-date!\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 210,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Download list of english stopwords.\n",
    "nltk.download('stopwords')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Define constants"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 211,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:31.307739Z",
     "start_time": "2023-06-02T15:56:31.293380Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# Constants.\n",
    "LABELS_LIST = [\n",
    "    'Neurosurgery',\n",
    "    'ENT - Otolaryngology',\n",
    "    'Discharge Summary',\n",
    "    'General Medicine',\n",
    "    'Gastroenterology',\n",
    "    'Neurology',\n",
    "    'SOAP / Chart / Progress Notes',\n",
    "    'Obstetrics / Gynecology',\n",
    "    'Urology'\n",
    "]\n",
    "\n",
    "TEXT_COLUMN_NAME = \"transcription\"\n",
    "TARGET_COLUMN_NAME = \"medical_specialty\"\n",
    "\n",
    "RANDOM_SEED = 8888\n",
    "\n",
    "# Data.\n",
    "DATA_URL = os.path.join(\"ftp://sys.giskard.ai\", \"pub\", \"unit_test_resources\",\n",
    "                        \"medical_transcript_classification_dataset\", \"mtsamples.csv\")\n",
    "DATA_PATH = Path.home() / \".giskard\" / \"medical_transcript_classification_dataset\" / \"mtsamples.csv\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Dataset preparation"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Load data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 212,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:32.244129Z",
     "start_time": "2023-06-02T15:56:32.233725Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def fetch_from_ftp(url: str, file: Path) -> None:\n",
    "    \"\"\"Helper to fetch data from the FTP server.\"\"\"\n",
    "    if not file.parent.exists():\n",
    "        file.parent.mkdir(parents=True, exist_ok=True)\n",
    "\n",
    "    if not file.exists():\n",
    "        print(f\"Downloading data from {url}\")\n",
    "        urlretrieve(url, file)\n",
    "\n",
    "    print(f\"Data was loaded!\")\n",
    "\n",
    "\n",
    "def load_data() -> pd.DataFrame:\n",
    "    \"\"\"Load and initially preprocess data.\"\"\"\n",
    "    fetch_from_ftp(DATA_URL, DATA_PATH)\n",
    "\n",
    "    df = pd.read_csv(DATA_PATH)\n",
    "\n",
    "    # Drop useless columns.\n",
    "    df = df.drop(columns=['Unnamed: 0', \"description\", \"sample_name\", \"keywords\"])\n",
    "\n",
    "    # Trim text.\n",
    "    df = df.apply(lambda x: x.str.strip())\n",
    "\n",
    "    # Filter samples by label.\n",
    "    df = df[df[TARGET_COLUMN_NAME].isin(LABELS_LIST)]\n",
    "\n",
    "    # Drop rows with no transcript.\n",
    "    df = df[df[TEXT_COLUMN_NAME].notna()]\n",
    "\n",
    "    return df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 213,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:33.138579Z",
     "start_time": "2023-06-02T15:56:32.909490Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Data was loaded!\n"
     ]
    }
   ],
   "source": [
    "transcript_df = load_data()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Train-test split"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 214,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:33.407547Z",
     "start_time": "2023-06-02T15:56:33.370987Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "X_train, X_test, y_train, y_test = train_test_split(transcript_df[[TEXT_COLUMN_NAME]],\n",
    "                                                    transcript_df[TARGET_COLUMN_NAME],\n",
    "                                                    random_state=RANDOM_SEED)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Wrap dataset with Giskard"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 215,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:34.102804Z",
     "start_time": "2023-06-02T15:56:34.071403Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "raw_data = pd.concat([X_test, y_test], axis=1)\n",
    "wrapped_data = Dataset(raw_data,\n",
    "                       name=\"medical_transcript_dataset\",\n",
    "                       target=TARGET_COLUMN_NAME,\n",
    "                       column_types={TEXT_COLUMN_NAME: \"text\"})"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Model training"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Define preprocessing steps"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 216,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:35.135316Z",
     "start_time": "2023-06-02T15:56:35.106935Z"
    },
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "stemmer = SnowballStemmer(\"english\")\n",
    "stop_words = stopwords.words(\"english\")\n",
    "\n",
    "\n",
    "def preprocess_text(df: pd.DataFrame) -> pd.DataFrame:\n",
    "    \"\"\"Preprocess text.\"\"\"\n",
    "    # Lower.\n",
    "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(lambda x: x.lower())\n",
    "\n",
    "    # Remove punctuation.\n",
    "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(lambda x: x.translate(str.maketrans('', '', string.punctuation)))\n",
    "\n",
    "    # Tokenize.\n",
    "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(lambda x: x.split())\n",
    "\n",
    "    # Stem.\n",
    "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(lambda x: [stemmer.stem(word) for word in x])\n",
    "\n",
    "    # Remove stop-words.\n",
    "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(\n",
    "        lambda x: ' '.join([word for word in x if word not in stop_words]))\n",
    "\n",
    "    return df\n",
    "\n",
    "\n",
    "def adapt_vectorizer_input(df: pd.DataFrame) -> Iterable:\n",
    "    \"\"\"Adapt input for the vectorizers.\n",
    "\n",
    "    The problem is that vectorizers accept iterable, not DataFrame, but Series. Thus, we need to ravel dataframe with text have input single dimension.\n",
    "    Issue reference: https://stackoverflow.com/questions/50665240/valueerror-found-input-variables-with-inconsistent-numbers-of-samples-1-3185\"\"\"\n",
    "\n",
    "    df = df.iloc[:, 0]\n",
    "    return df\n",
    "\n",
    "\n",
    "text_preprocessor = FunctionTransformer(preprocess_text)\n",
    "vectorizer_input_adapter = FunctionTransformer(adapt_vectorizer_input)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Build estimator"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 217,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:54.806438Z",
     "start_time": "2023-06-02T15:56:35.867094Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                               precision    recall  f1-score   support\n",
      "\n",
      "            Discharge Summary       0.43      0.38      0.41        34\n",
      "         ENT - Otolaryngology       1.00      0.59      0.74        27\n",
      "             Gastroenterology       0.57      0.81      0.67        43\n",
      "             General Medicine       0.46      0.61      0.52        69\n",
      "                    Neurology       0.74      0.66      0.70        53\n",
      "                 Neurosurgery       0.73      0.79      0.76        24\n",
      "      Obstetrics / Gynecology       0.85      0.58      0.69        50\n",
      "SOAP / Chart / Progress Notes       0.40      0.36      0.38        33\n",
      "                      Urology       0.74      0.68      0.71        38\n",
      "\n",
      "                     accuracy                           0.61       371\n",
      "                    macro avg       0.66      0.61      0.62       371\n",
      "                 weighted avg       0.64      0.61      0.62       371\n",
      "\n"
     ]
    }
   ],
   "source": [
    "pipeline = Pipeline(steps=[\n",
    "    (\"text_preprocessor\", text_preprocessor),\n",
    "    (\"vectorizer_input_adapter\", vectorizer_input_adapter),\n",
    "    (\"vectorizer\", CountVectorizer(ngram_range=(1, 1))),\n",
    "    (\"estimator\", RandomForestClassifier(random_state=RANDOM_SEED))\n",
    "])\n",
    "\n",
    "pipeline.fit(X_train, y_train)\n",
    "y_pred = pipeline.predict(X_test)\n",
    "\n",
    "print(classification_report(y_test, y_pred))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "### Wrap model with Giskard"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 218,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T15:56:57.005694Z",
     "start_time": "2023-06-02T15:56:54.814070Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                               precision    recall  f1-score   support\n",
      "\n",
      "            Discharge Summary       0.43      0.38      0.41        34\n",
      "         ENT - Otolaryngology       1.00      0.59      0.74        27\n",
      "             Gastroenterology       0.57      0.81      0.67        43\n",
      "             General Medicine       0.46      0.61      0.52        69\n",
      "                    Neurology       0.74      0.66      0.70        53\n",
      "                 Neurosurgery       0.73      0.79      0.76        24\n",
      "      Obstetrics / Gynecology       0.85      0.58      0.69        50\n",
      "SOAP / Chart / Progress Notes       0.40      0.36      0.38        33\n",
      "                      Urology       0.74      0.68      0.71        38\n",
      "\n",
      "                     accuracy                           0.61       371\n",
      "                    macro avg       0.66      0.61      0.62       371\n",
      "                 weighted avg       0.64      0.61      0.62       371\n",
      "\n"
     ]
    }
   ],
   "source": [
    "wrapped_model = Model(pipeline.predict_proba,\n",
    "                      model_type=\"classification\",\n",
    "                      name=\"medical_transcript_classification\",\n",
    "                      feature_names=[TEXT_COLUMN_NAME],\n",
    "                      classification_labels=pipeline.classes_)\n",
    "\n",
    "# Validate wrapped model and data.\n",
    "print(classification_report(y_test, pipeline.classes_[wrapped_model.predict(wrapped_data).raw_prediction]))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Scan your model to find vulnerabilities\n",
    "With the Giskard scan feature, you can detect vulnerabilities in your model, including performance biases, unrobustness, data leakage, stochasticity, underconfidence, ethical issues, and more. For detailed information about the scan feature, please refer to our scan documentation."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 219,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T16:02:40.215531Z",
     "start_time": "2023-06-02T15:57:00.800368Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Your model is successfully validated.\n",
      "ðŸ”Ž Running scanâ€¦\n",
      "2023-08-11 10:38:11,043 pid:6545 MainThread giskard.scanner.logger INFO     Running detectors: ['PerformanceBiasDetector', 'TextPerturbationDetector', 'EthicalBiasDetector', 'DataLeakageDetector', 'StochasticityDetector', 'OverconfidenceDetector', 'UnderconfidenceDetector', 'SpuriousCorrelationDetector']\n",
      "Running detector PerformanceBiasDetectorâ€¦2023-08-11 10:38:11,044 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Running\n",
      "2023-08-11 10:38:11,044 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Calculating loss\n",
      "2023-08-11 10:38:11,112 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Loss calculated (took 0:00:00.067198)\n",
      "2023-08-11 10:38:11,112 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Finding data slices\n",
      "2023-08-11 10:38:41,764 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: 321 slices found (took 0:00:30.650746)\n",
      "2023-08-11 10:38:41,766 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Analyzing issues\n",
      "2023-08-11 10:38:41,782 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing 321 slices for performance issues.\n",
      "2023-08-11 10:38:44,209 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `language(transcription)` == \"en\"\tPrecision = 0.614 (global 0.612) Î”m = 0.003\tis_issue = False\n",
      "2023-08-11 10:38:44,223 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cautery\"\tPrecision = 0.741 (global 0.612) Î”m = 0.211\tis_issue = False\n",
      "2023-08-11 10:38:44,239 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subsequently\"\tPrecision = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:44,254 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"attention\"\tPrecision = 0.786 (global 0.612) Î”m = 0.284\tis_issue = False\n",
      "2023-08-11 10:38:44,268 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"coronary\"\tPrecision = 0.391 (global 0.612) Î”m = -0.360\tis_issue = True\n",
      "2023-08-11 10:38:44,284 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"external\"\tPrecision = 0.606 (global 0.612) Î”m = -0.009\tis_issue = False\n",
      "2023-08-11 10:38:44,298 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"equal\"\tPrecision = 0.655 (global 0.612) Î”m = 0.071\tis_issue = False\n",
      "2023-08-11 10:38:44,313 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"states\"\tPrecision = 0.535 (global 0.612) Î”m = -0.126\tis_issue = True\n",
      "2023-08-11 10:38:44,327 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"metzenbaum\"\tPrecision = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:38:44,341 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abnormalities\"\tPrecision = 0.656 (global 0.612) Î”m = 0.073\tis_issue = False\n",
      "2023-08-11 10:38:44,356 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualization\"\tPrecision = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:38:44,368 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilateral\"\tPrecision = 0.646 (global 0.612) Î”m = 0.056\tis_issue = False\n",
      "2023-08-11 10:38:44,381 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"head\"\tPrecision = 0.646 (global 0.612) Î”m = 0.056\tis_issue = False\n",
      "2023-08-11 10:38:44,394 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"estimated\"\tPrecision = 0.823 (global 0.612) Î”m = 0.344\tis_issue = False\n",
      "2023-08-11 10:38:44,418 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"numbness\"\tPrecision = 0.815 (global 0.612) Î”m = 0.332\tis_issue = False\n",
      "2023-08-11 10:38:44,438 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tubes\"\tPrecision = 0.783 (global 0.612) Î”m = 0.279\tis_issue = False\n",
      "2023-08-11 10:38:44,459 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"activity\"\tPrecision = 0.514 (global 0.612) Î”m = -0.161\tis_issue = True\n",
      "2023-08-11 10:38:44,478 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scan\"\tPrecision = 0.568 (global 0.612) Î”m = -0.071\tis_issue = True\n",
      "2023-08-11 10:38:44,496 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"single\"\tPrecision = 0.696 (global 0.612) Î”m = 0.137\tis_issue = False\n",
      "2023-08-11 10:38:44,510 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extended\"\tPrecision = 0.929 (global 0.612) Î”m = 0.518\tis_issue = False\n",
      "2023-08-11 10:38:44,524 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"breast\"\tPrecision = 0.444 (global 0.612) Î”m = -0.274\tis_issue = True\n",
      "2023-08-11 10:38:44,539 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"problems\"\tPrecision = 0.565 (global 0.612) Î”m = -0.076\tis_issue = True\n",
      "2023-08-11 10:38:44,553 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fascia\"\tPrecision = 0.824 (global 0.612) Î”m = 0.346\tis_issue = False\n",
      "2023-08-11 10:38:44,567 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tract\"\tPrecision = 0.450 (global 0.612) Î”m = -0.265\tis_issue = True\n",
      "2023-08-11 10:38:44,581 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lymph\"\tPrecision = 0.769 (global 0.612) Î”m = 0.257\tis_issue = False\n",
      "2023-08-11 10:38:44,595 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colonoscopy\"\tPrecision = 0.565 (global 0.612) Î”m = -0.076\tis_issue = True\n",
      "2023-08-11 10:38:44,610 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"region\"\tPrecision = 0.724 (global 0.612) Î”m = 0.184\tis_issue = False\n",
      "2023-08-11 10:38:44,624 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medication\"\tPrecision = 0.541 (global 0.612) Î”m = -0.117\tis_issue = True\n",
      "2023-08-11 10:38:44,638 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"father\"\tPrecision = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:44,652 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectus\"\tPrecision = 0.950 (global 0.612) Î”m = 0.553\tis_issue = False\n",
      "2023-08-11 10:38:44,666 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"function\"\tPrecision = 0.414 (global 0.612) Î”m = -0.324\tis_issue = True\n",
      "2023-08-11 10:38:44,680 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fever\"\tPrecision = 0.558 (global 0.612) Î”m = -0.089\tis_issue = True\n",
      "2023-08-11 10:38:44,691 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"procedure\"\tPrecision = 0.726 (global 0.612) Î”m = 0.187\tis_issue = False\n",
      "2023-08-11 10:38:44,705 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearance\"\tPrecision = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:44,721 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inserted\"\tPrecision = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:44,737 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suction\"\tPrecision = 0.826 (global 0.612) Î”m = 0.350\tis_issue = False\n",
      "2023-08-11 10:38:44,752 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removal\"\tPrecision = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:38:44,766 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removed\"\tPrecision = 0.790 (global 0.612) Î”m = 0.291\tis_issue = False\n",
      "2023-08-11 10:38:44,781 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reveals\"\tPrecision = 0.652 (global 0.612) Î”m = 0.066\tis_issue = False\n",
      "2023-08-11 10:38:44,795 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clear\"\tPrecision = 0.515 (global 0.612) Î”m = -0.158\tis_issue = True\n",
      "2023-08-11 10:38:44,810 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"age\"\tPrecision = 0.639 (global 0.612) Î”m = 0.044\tis_issue = False\n",
      "2023-08-11 10:38:44,824 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissection\"\tPrecision = 0.762 (global 0.612) Î”m = 0.245\tis_issue = False\n",
      "2023-08-11 10:38:44,841 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"small\"\tPrecision = 0.770 (global 0.612) Î”m = 0.259\tis_issue = False\n",
      "2023-08-11 10:38:44,855 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"enlarged\"\tPrecision = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:38:44,867 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"room\"\tPrecision = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:38:44,882 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"biopsy\"\tPrecision = 0.625 (global 0.612) Î”m = 0.021\tis_issue = False\n",
      "2023-08-11 10:38:44,899 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"irrigated\"\tPrecision = 0.854 (global 0.612) Î”m = 0.395\tis_issue = False\n",
      "2023-08-11 10:38:44,915 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"swelling\"\tPrecision = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:44,930 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"infection\"\tPrecision = 0.607 (global 0.612) Î”m = -0.009\tis_issue = False\n",
      "2023-08-11 10:38:44,943 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"exam\"\tPrecision = 0.663 (global 0.612) Î”m = 0.084\tis_issue = False\n",
      "2023-08-11 10:38:44,958 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"urinary\"\tPrecision = 0.486 (global 0.612) Î”m = -0.206\tis_issue = True\n",
      "2023-08-11 10:38:44,972 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"count\"\tPrecision = 0.400 (global 0.612) Î”m = -0.346\tis_issue = True\n",
      "2023-08-11 10:38:44,984 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"performed\"\tPrecision = 0.683 (global 0.612) Î”m = 0.116\tis_issue = False\n",
      "2023-08-11 10:38:44,999 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dorsal\"\tPrecision = 0.929 (global 0.612) Î”m = 0.518\tis_issue = False\n",
      "2023-08-11 10:38:45,013 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vicryl\"\tPrecision = 0.791 (global 0.612) Î”m = 0.293\tis_issue = False\n",
      "2023-08-11 10:38:45,028 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"counts\"\tPrecision = 0.892 (global 0.612) Î”m = 0.458\tis_issue = False\n",
      "2023-08-11 10:38:45,043 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"proximal\"\tPrecision = 0.889 (global 0.612) Î”m = 0.453\tis_issue = False\n",
      "2023-08-11 10:38:45,057 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extending\"\tPrecision = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:45,072 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medial\"\tPrecision = 0.778 (global 0.612) Î”m = 0.271\tis_issue = False\n",
      "2023-08-11 10:38:45,085 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cord\"\tPrecision = 0.939 (global 0.612) Î”m = 0.535\tis_issue = False\n",
      "2023-08-11 10:38:45,100 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissected\"\tPrecision = 0.881 (global 0.612) Î”m = 0.440\tis_issue = False\n",
      "2023-08-11 10:38:45,114 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recurrent\"\tPrecision = 0.654 (global 0.612) Î”m = 0.069\tis_issue = False\n",
      "2023-08-11 10:38:45,128 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cavity\"\tPrecision = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:38:45,142 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knee\"\tPrecision = 0.591 (global 0.612) Î”m = -0.034\tis_issue = False\n",
      "2023-08-11 10:38:45,157 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adequate\"\tPrecision = 0.816 (global 0.612) Î”m = 0.333\tis_issue = False\n",
      "2023-08-11 10:38:45,171 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"length\"\tPrecision = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:38:45,185 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complex\"\tPrecision = 0.522 (global 0.612) Î”m = -0.147\tis_issue = True\n",
      "2023-08-11 10:38:45,199 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"having\"\tPrecision = 0.523 (global 0.612) Î”m = -0.145\tis_issue = True\n",
      "2023-08-11 10:38:45,212 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mass\"\tPrecision = 0.596 (global 0.612) Î”m = -0.026\tis_issue = False\n",
      "2023-08-11 10:38:45,226 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"administered\"\tPrecision = 0.767 (global 0.612) Î”m = 0.253\tis_issue = False\n",
      "2023-08-11 10:38:45,241 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"blade\"\tPrecision = 0.900 (global 0.612) Î”m = 0.471\tis_issue = False\n",
      "2023-08-11 10:38:45,254 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denies\"\tPrecision = 0.459 (global 0.612) Î”m = -0.250\tis_issue = True\n",
      "2023-08-11 10:38:45,269 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthetic\"\tPrecision = 0.913 (global 0.612) Î”m = 0.492\tis_issue = False\n",
      "2023-08-11 10:38:45,281 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complications\"\tPrecision = 0.697 (global 0.612) Î”m = 0.140\tis_issue = False\n",
      "2023-08-11 10:38:45,296 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"erythema\"\tPrecision = 0.655 (global 0.612) Î”m = 0.071\tis_issue = False\n",
      "2023-08-11 10:38:45,311 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"local\"\tPrecision = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:38:45,326 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"delivered\"\tPrecision = 0.920 (global 0.612) Î”m = 0.504\tis_issue = False\n",
      "2023-08-11 10:38:45,340 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"direct\"\tPrecision = 0.731 (global 0.612) Î”m = 0.194\tis_issue = False\n",
      "2023-08-11 10:38:45,354 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"negative\"\tPrecision = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:45,368 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nasal\"\tPrecision = 0.594 (global 0.612) Î”m = -0.030\tis_issue = False\n",
      "2023-08-11 10:38:45,382 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"end\"\tPrecision = 0.757 (global 0.612) Î”m = 0.237\tis_issue = False\n",
      "2023-08-11 10:38:45,398 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suture\"\tPrecision = 0.804 (global 0.612) Î”m = 0.315\tis_issue = False\n",
      "2023-08-11 10:38:45,414 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incised\"\tPrecision = 0.900 (global 0.612) Î”m = 0.471\tis_issue = False\n",
      "2023-08-11 10:38:45,430 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"strength\"\tPrecision = 0.738 (global 0.612) Î”m = 0.206\tis_issue = False\n",
      "2023-08-11 10:38:45,443 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mri\"\tPrecision = 0.708 (global 0.612) Î”m = 0.158\tis_issue = False\n",
      "2023-08-11 10:38:45,458 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"areas\"\tPrecision = 0.607 (global 0.612) Î”m = -0.008\tis_issue = False\n",
      "2023-08-11 10:38:45,474 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"second\"\tPrecision = 0.679 (global 0.612) Î”m = 0.110\tis_issue = False\n",
      "2023-08-11 10:38:45,487 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"examination\"\tPrecision = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:45,502 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spinal\"\tPrecision = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:45,515 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"admission\"\tPrecision = 0.375 (global 0.612) Î”m = -0.387\tis_issue = True\n",
      "2023-08-11 10:38:45,530 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"open\"\tPrecision = 0.769 (global 0.612) Î”m = 0.257\tis_issue = False\n",
      "2023-08-11 10:38:45,544 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mouth\"\tPrecision = 0.700 (global 0.612) Î”m = 0.144\tis_issue = False\n",
      "2023-08-11 10:38:45,559 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pulmonary\"\tPrecision = 0.533 (global 0.612) Î”m = -0.128\tis_issue = True\n",
      "2023-08-11 10:38:45,574 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"flap\"\tPrecision = 0.727 (global 0.612) Î”m = 0.189\tis_issue = False\n",
      "2023-08-11 10:38:45,588 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"treatment\"\tPrecision = 0.518 (global 0.612) Î”m = -0.154\tis_issue = True\n",
      "2023-08-11 10:38:45,602 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"resection\"\tPrecision = 0.700 (global 0.612) Î”m = 0.144\tis_issue = False\n",
      "2023-08-11 10:38:45,617 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"contrast\"\tPrecision = 0.708 (global 0.612) Î”m = 0.158\tis_issue = False\n",
      "2023-08-11 10:38:45,633 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spine\"\tPrecision = 0.703 (global 0.612) Î”m = 0.148\tis_issue = False\n",
      "2023-08-11 10:38:45,649 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cm\"\tPrecision = 0.716 (global 0.612) Î”m = 0.171\tis_issue = False\n",
      "2023-08-11 10:38:45,668 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervical\"\tPrecision = 0.708 (global 0.612) Î”m = 0.158\tis_issue = False\n",
      "2023-08-11 10:38:45,696 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abc\"\tPrecision = 0.417 (global 0.612) Î”m = -0.319\tis_issue = True\n",
      "2023-08-11 10:38:45,712 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligated\"\tPrecision = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:38:45,729 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sinus\"\tPrecision = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:38:45,743 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oxygen\"\tPrecision = 0.591 (global 0.612) Î”m = -0.034\tis_issue = False\n",
      "2023-08-11 10:38:45,757 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distal\"\tPrecision = 0.795 (global 0.612) Î”m = 0.299\tis_issue = False\n",
      "2023-08-11 10:38:45,772 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sterile\"\tPrecision = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:45,786 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinical\"\tPrecision = 0.565 (global 0.612) Î”m = -0.076\tis_issue = True\n",
      "2023-08-11 10:38:45,800 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mm\"\tPrecision = 0.827 (global 0.612) Î”m = 0.351\tis_issue = False\n",
      "2023-08-11 10:38:45,814 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cardiac\"\tPrecision = 0.529 (global 0.612) Î”m = -0.135\tis_issue = True\n",
      "2023-08-11 10:38:45,829 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visit\"\tPrecision = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:45,844 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearing\"\tPrecision = 0.680 (global 0.612) Î”m = 0.111\tis_issue = False\n",
      "2023-08-11 10:38:45,858 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clamped\"\tPrecision = 0.960 (global 0.612) Î”m = 0.569\tis_issue = False\n",
      "2023-08-11 10:38:45,873 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectum\"\tPrecision = 0.650 (global 0.612) Î”m = 0.062\tis_issue = False\n",
      "2023-08-11 10:38:45,886 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"draped\"\tPrecision = 0.773 (global 0.612) Î”m = 0.263\tis_issue = False\n",
      "2023-08-11 10:38:45,902 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stomach\"\tPrecision = 0.850 (global 0.612) Î”m = 0.389\tis_issue = False\n",
      "2023-08-11 10:38:45,916 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wound\"\tPrecision = 0.638 (global 0.612) Î”m = 0.043\tis_issue = False\n",
      "2023-08-11 10:38:45,929 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"family\"\tPrecision = 0.573 (global 0.612) Î”m = -0.063\tis_issue = True\n",
      "2023-08-11 10:38:45,944 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nerve\"\tPrecision = 0.850 (global 0.612) Î”m = 0.389\tis_issue = False\n",
      "2023-08-11 10:38:45,958 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cut\"\tPrecision = 0.917 (global 0.612) Î”m = 0.498\tis_issue = False\n",
      "2023-08-11 10:38:45,972 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesions\"\tPrecision = 0.681 (global 0.612) Î”m = 0.113\tis_issue = False\n",
      "2023-08-11 10:38:45,987 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervix\"\tPrecision = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:46,002 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"technique\"\tPrecision = 0.783 (global 0.612) Î”m = 0.279\tis_issue = False\n",
      "2023-08-11 10:38:46,017 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"point\"\tPrecision = 0.615 (global 0.612) Î”m = 0.006\tis_issue = False\n",
      "2023-08-11 10:38:46,031 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"leg\"\tPrecision = 0.567 (global 0.612) Î”m = -0.074\tis_issue = True\n",
      "2023-08-11 10:38:46,048 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"s1\"\tPrecision = 0.556 (global 0.612) Î”m = -0.092\tis_issue = True\n",
      "2023-08-11 10:38:46,063 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"xyz\"\tPrecision = 0.318 (global 0.612) Î”m = -0.480\tis_issue = True\n",
      "2023-08-11 10:38:46,077 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"office\"\tPrecision = 0.476 (global 0.612) Î”m = -0.222\tis_issue = True\n",
      "2023-08-11 10:38:46,093 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gross\"\tPrecision = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:46,108 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"check\"\tPrecision = 0.560 (global 0.612) Î”m = -0.085\tis_issue = True\n",
      "2023-08-11 10:38:46,122 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"created\"\tPrecision = 0.950 (global 0.612) Î”m = 0.553\tis_issue = False\n",
      "2023-08-11 10:38:46,136 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hours\"\tPrecision = 0.558 (global 0.612) Î”m = -0.089\tis_issue = True\n",
      "2023-08-11 10:38:46,150 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"body\"\tPrecision = 0.816 (global 0.612) Î”m = 0.333\tis_issue = False\n",
      "2023-08-11 10:38:46,165 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"entered\"\tPrecision = 0.875 (global 0.612) Î”m = 0.430\tis_issue = False\n",
      "2023-08-11 10:38:46,180 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectal\"\tPrecision = 0.643 (global 0.612) Î”m = 0.051\tis_issue = False\n",
      "2023-08-11 10:38:46,194 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adenopathy\"\tPrecision = 0.778 (global 0.612) Î”m = 0.271\tis_issue = False\n",
      "2023-08-11 10:38:46,209 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterine\"\tPrecision = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:38:46,222 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"high\"\tPrecision = 0.488 (global 0.612) Î”m = -0.203\tis_issue = True\n",
      "2023-08-11 10:38:46,237 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"correct\"\tPrecision = 0.881 (global 0.612) Î”m = 0.440\tis_issue = False\n",
      "2023-08-11 10:38:46,252 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pupils\"\tPrecision = 0.706 (global 0.612) Î”m = 0.154\tis_issue = False\n",
      "2023-08-11 10:38:46,266 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"upper\"\tPrecision = 0.718 (global 0.612) Î”m = 0.173\tis_issue = False\n",
      "2023-08-11 10:38:46,281 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visual\"\tPrecision = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:38:46,295 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"revealed\"\tPrecision = 0.742 (global 0.612) Î”m = 0.213\tis_issue = False\n",
      "2023-08-11 10:38:46,311 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tumor\"\tPrecision = 0.583 (global 0.612) Î”m = -0.047\tis_issue = False\n",
      "2023-08-11 10:38:46,326 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distress\"\tPrecision = 0.486 (global 0.612) Î”m = -0.206\tis_issue = True\n",
      "2023-08-11 10:38:46,342 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"unremarkable\"\tPrecision = 0.632 (global 0.612) Î”m = 0.033\tis_issue = False\n",
      "2023-08-11 10:38:46,356 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"greater\"\tPrecision = 0.435 (global 0.612) Î”m = -0.289\tis_issue = True\n",
      "2023-08-11 10:38:46,372 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"internal\"\tPrecision = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:38:46,388 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recovery\"\tPrecision = 0.843 (global 0.612) Î”m = 0.378\tis_issue = False\n",
      "2023-08-11 10:38:46,405 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scope\"\tPrecision = 0.864 (global 0.612) Î”m = 0.411\tis_issue = False\n",
      "2023-08-11 10:38:46,419 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"skin\"\tPrecision = 0.723 (global 0.612) Î”m = 0.182\tis_issue = False\n",
      "2023-08-11 10:38:46,430 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"postoperative\"\tPrecision = 0.729 (global 0.612) Î”m = 0.191\tis_issue = False\n",
      "2023-08-11 10:38:46,443 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"evidence\"\tPrecision = 0.691 (global 0.612) Î”m = 0.130\tis_issue = False\n",
      "2023-08-11 10:38:46,457 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knife\"\tPrecision = 0.952 (global 0.612) Î”m = 0.557\tis_issue = False\n",
      "2023-08-11 10:38:46,473 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sac\"\tPrecision = 0.913 (global 0.612) Î”m = 0.492\tis_issue = False\n",
      "2023-08-11 10:38:46,490 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sutures\"\tPrecision = 0.636 (global 0.612) Î”m = 0.040\tis_issue = False\n",
      "2023-08-11 10:38:46,504 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterus\"\tPrecision = 0.647 (global 0.612) Î”m = 0.058\tis_issue = False\n",
      "2023-08-11 10:38:46,520 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inferior\"\tPrecision = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:38:46,536 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"similar\"\tPrecision = 0.724 (global 0.612) Î”m = 0.184\tis_issue = False\n",
      "2023-08-11 10:38:46,550 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bleeding\"\tPrecision = 0.727 (global 0.612) Î”m = 0.189\tis_issue = False\n",
      "2023-08-11 10:38:46,563 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"large\"\tPrecision = 0.647 (global 0.612) Î”m = 0.058\tis_issue = False\n",
      "2023-08-11 10:38:46,576 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"normal\"\tPrecision = 0.641 (global 0.612) Î”m = 0.048\tis_issue = False\n",
      "2023-08-11 10:38:46,589 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"posterior\"\tPrecision = 0.768 (global 0.612) Î”m = 0.255\tis_issue = False\n",
      "2023-08-11 10:38:46,605 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"chromic\"\tPrecision = 0.952 (global 0.612) Î”m = 0.557\tis_issue = False\n",
      "2023-08-11 10:38:46,631 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"round\"\tPrecision = 0.774 (global 0.612) Î”m = 0.265\tis_issue = False\n",
      "2023-08-11 10:38:46,659 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ct\"\tPrecision = 0.600 (global 0.612) Î”m = -0.019\tis_issue = False\n",
      "2023-08-11 10:38:46,677 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incision\"\tPrecision = 0.747 (global 0.612) Î”m = 0.221\tis_issue = False\n",
      "2023-08-11 10:38:46,692 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesion\"\tPrecision = 0.692 (global 0.612) Î”m = 0.131\tis_issue = False\n",
      "2023-08-11 10:38:46,705 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fashion\"\tPrecision = 0.807 (global 0.612) Î”m = 0.319\tis_issue = False\n",
      "2023-08-11 10:38:46,719 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vaginal\"\tPrecision = 0.586 (global 0.612) Î”m = -0.042\tis_issue = False\n",
      "2023-08-11 10:38:46,736 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subcuticular\"\tPrecision = 0.792 (global 0.612) Î”m = 0.294\tis_issue = False\n",
      "2023-08-11 10:38:46,752 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stable\"\tPrecision = 0.627 (global 0.612) Î”m = 0.025\tis_issue = False\n",
      "2023-08-11 10:38:46,766 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mucosa\"\tPrecision = 0.697 (global 0.612) Î”m = 0.139\tis_issue = False\n",
      "2023-08-11 10:38:46,782 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"shoulder\"\tPrecision = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:38:46,795 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"use\"\tPrecision = 0.681 (global 0.612) Î”m = 0.112\tis_issue = False\n",
      "2023-08-11 10:38:46,810 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nose\"\tPrecision = 0.794 (global 0.612) Î”m = 0.298\tis_issue = False\n",
      "2023-08-11 10:38:46,825 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rule\"\tPrecision = 0.450 (global 0.612) Î”m = -0.265\tis_issue = True\n",
      "2023-08-11 10:38:46,840 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denied\"\tPrecision = 0.773 (global 0.612) Î”m = 0.263\tis_issue = False\n",
      "2023-08-11 10:38:46,852 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"left\"\tPrecision = 0.693 (global 0.612) Î”m = 0.132\tis_issue = False\n",
      "2023-08-11 10:38:46,864 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prepped\"\tPrecision = 0.767 (global 0.612) Î”m = 0.253\tis_issue = False\n",
      "2023-08-11 10:38:46,878 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"diarrhea\"\tPrecision = 0.462 (global 0.612) Î”m = -0.246\tis_issue = True\n",
      "2023-08-11 10:38:46,894 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"improved\"\tPrecision = 0.419 (global 0.612) Î”m = -0.315\tis_issue = True\n",
      "2023-08-11 10:38:46,907 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"loss\"\tPrecision = 0.781 (global 0.612) Î”m = 0.276\tis_issue = False\n",
      "2023-08-11 10:38:46,921 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gentleman\"\tPrecision = 0.696 (global 0.612) Î”m = 0.137\tis_issue = False\n",
      "2023-08-11 10:38:46,938 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sensory\"\tPrecision = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:38:46,955 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"laterally\"\tPrecision = 0.846 (global 0.612) Î”m = 0.383\tis_issue = False\n",
      "2023-08-11 10:38:46,972 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"needle\"\tPrecision = 0.821 (global 0.612) Î”m = 0.343\tis_issue = False\n",
      "2023-08-11 10:38:46,987 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"findings\"\tPrecision = 0.691 (global 0.612) Î”m = 0.129\tis_issue = False\n",
      "2023-08-11 10:38:47,007 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tolerated\"\tPrecision = 0.698 (global 0.612) Î”m = 0.140\tis_issue = False\n",
      "2023-08-11 10:38:47,032 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"neck\"\tPrecision = 0.553 (global 0.612) Î”m = -0.096\tis_issue = True\n",
      "2023-08-11 10:38:47,061 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headache\"\tPrecision = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:47,082 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"condition\"\tPrecision = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:47,106 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wall\"\tPrecision = 0.763 (global 0.612) Î”m = 0.247\tis_issue = False\n",
      "2023-08-11 10:38:47,120 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abdominal\"\tPrecision = 0.615 (global 0.612) Î”m = 0.006\tis_issue = False\n",
      "2023-08-11 10:38:47,135 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"placed\"\tPrecision = 0.780 (global 0.612) Î”m = 0.274\tis_issue = False\n",
      "2023-08-11 10:38:47,157 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"supine\"\tPrecision = 0.764 (global 0.612) Î”m = 0.248\tis_issue = False\n",
      "2023-08-11 10:38:47,174 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prostate\"\tPrecision = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:38:47,191 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tube\"\tPrecision = 0.773 (global 0.612) Î”m = 0.263\tis_issue = False\n",
      "2023-08-11 10:38:47,211 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gait\"\tPrecision = 0.742 (global 0.612) Î”m = 0.213\tis_issue = False\n",
      "2023-08-11 10:38:47,227 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pelvis\"\tPrecision = 0.733 (global 0.612) Î”m = 0.199\tis_issue = False\n",
      "2023-08-11 10:38:47,243 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suctioned\"\tPrecision = 0.952 (global 0.612) Î”m = 0.557\tis_issue = False\n",
      "2023-08-11 10:38:47,258 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"passed\"\tPrecision = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:47,274 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subjective\"\tPrecision = 0.367 (global 0.612) Î”m = -0.401\tis_issue = True\n",
      "2023-08-11 10:38:47,288 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"able\"\tPrecision = 0.633 (global 0.612) Î”m = 0.034\tis_issue = False\n",
      "2023-08-11 10:38:47,302 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lumbar\"\tPrecision = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:47,316 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"running\"\tPrecision = 0.825 (global 0.612) Î”m = 0.348\tis_issue = False\n",
      "2023-08-11 10:38:47,323 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"patient\"\tPrecision = 0.597 (global 0.612) Î”m = -0.023\tis_issue = False\n",
      "2023-08-11 10:38:47,337 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pregnancy\"\tPrecision = 0.783 (global 0.612) Î”m = 0.279\tis_issue = False\n",
      "2023-08-11 10:38:47,351 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brain\"\tPrecision = 0.645 (global 0.612) Î”m = 0.054\tis_issue = False\n",
      "2023-08-11 10:38:47,366 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drainage\"\tPrecision = 0.630 (global 0.612) Î”m = 0.029\tis_issue = False\n",
      "2023-08-11 10:38:47,380 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"indications\"\tPrecision = 0.889 (global 0.612) Î”m = 0.453\tis_issue = False\n",
      "2023-08-11 10:38:47,393 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"taken\"\tPrecision = 0.713 (global 0.612) Î”m = 0.165\tis_issue = False\n",
      "2023-08-11 10:38:47,408 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"masses\"\tPrecision = 0.605 (global 0.612) Î”m = -0.011\tis_issue = False\n",
      "2023-08-11 10:38:47,425 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suite\"\tPrecision = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:47,440 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"day\"\tPrecision = 0.481 (global 0.612) Î”m = -0.214\tis_issue = True\n",
      "2023-08-11 10:38:47,456 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sponge\"\tPrecision = 0.886 (global 0.612) Î”m = 0.448\tis_issue = False\n",
      "2023-08-11 10:38:47,469 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reports\"\tPrecision = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:38:47,483 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vision\"\tPrecision = 0.778 (global 0.612) Î”m = 0.271\tis_issue = False\n",
      "2023-08-11 10:38:47,498 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"foley\"\tPrecision = 0.730 (global 0.612) Î”m = 0.193\tis_issue = False\n",
      "2023-08-11 10:38:47,512 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"presentation\"\tPrecision = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:47,525 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"20\"\tPrecision = 0.598 (global 0.612) Î”m = -0.023\tis_issue = False\n",
      "2023-08-11 10:38:47,540 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"daily\"\tPrecision = 0.385 (global 0.612) Î”m = -0.371\tis_issue = True\n",
      "2023-08-11 10:38:47,582 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"motor\"\tPrecision = 0.795 (global 0.612) Î”m = 0.300\tis_issue = False\n",
      "2023-08-11 10:38:47,602 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"epinephrine\"\tPrecision = 0.650 (global 0.612) Î”m = 0.062\tis_issue = False\n",
      "2023-08-11 10:38:47,622 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"speech\"\tPrecision = 0.704 (global 0.612) Î”m = 0.150\tis_issue = False\n",
      "2023-08-11 10:38:47,638 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"superior\"\tPrecision = 0.742 (global 0.612) Î”m = 0.213\tis_issue = False\n",
      "2023-08-11 10:38:47,653 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"section\"\tPrecision = 0.692 (global 0.612) Î”m = 0.131\tis_issue = False\n",
      "2023-08-11 10:38:47,668 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mild\"\tPrecision = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:38:47,683 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinic\"\tPrecision = 0.542 (global 0.612) Î”m = -0.115\tis_issue = True\n",
      "2023-08-11 10:38:47,697 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reviewed\"\tPrecision = 0.529 (global 0.612) Î”m = -0.135\tis_issue = True\n",
      "2023-08-11 10:38:47,712 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appeared\"\tPrecision = 0.765 (global 0.612) Î”m = 0.250\tis_issue = False\n",
      "2023-08-11 10:38:47,726 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hx\"\tPrecision = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:47,740 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consent\"\tPrecision = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:38:47,754 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"found\"\tPrecision = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:47,769 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bovie\"\tPrecision = 0.840 (global 0.612) Î”m = 0.373\tis_issue = False\n",
      "2023-08-11 10:38:47,782 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"weakness\"\tPrecision = 0.732 (global 0.612) Î”m = 0.196\tis_issue = False\n",
      "2023-08-11 10:38:47,796 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"obtained\"\tPrecision = 0.737 (global 0.612) Î”m = 0.205\tis_issue = False\n",
      "2023-08-11 10:38:47,811 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"informed\"\tPrecision = 0.854 (global 0.612) Î”m = 0.395\tis_issue = False\n",
      "2023-08-11 10:38:47,827 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"initially\"\tPrecision = 0.517 (global 0.612) Î”m = -0.155\tis_issue = True\n",
      "2023-08-11 10:38:47,842 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"status\"\tPrecision = 0.515 (global 0.612) Î”m = -0.158\tis_issue = True\n",
      "2023-08-11 10:38:47,856 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"care\"\tPrecision = 0.537 (global 0.612) Î”m = -0.122\tis_issue = True\n",
      "2023-08-11 10:38:47,872 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stool\"\tPrecision = 0.516 (global 0.612) Î”m = -0.156\tis_issue = True\n",
      "2023-08-11 10:38:47,886 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"following\"\tPrecision = 0.609 (global 0.612) Î”m = -0.004\tis_issue = False\n",
      "2023-08-11 10:38:47,901 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reflexes\"\tPrecision = 0.775 (global 0.612) Î”m = 0.267\tis_issue = False\n",
      "2023-08-11 10:38:47,917 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scalpel\"\tPrecision = 0.957 (global 0.612) Î”m = 0.563\tis_issue = False\n",
      "2023-08-11 10:38:47,932 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tenderness\"\tPrecision = 0.543 (global 0.612) Î”m = -0.112\tis_issue = True\n",
      "2023-08-11 10:38:47,947 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"compression\"\tPrecision = 0.636 (global 0.612) Î”m = 0.040\tis_issue = False\n",
      "2023-08-11 10:38:47,961 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headaches\"\tPrecision = 0.840 (global 0.612) Î”m = 0.373\tis_issue = False\n",
      "2023-08-11 10:38:47,976 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"closed\"\tPrecision = 0.806 (global 0.612) Î”m = 0.317\tis_issue = False\n",
      "2023-08-11 10:38:47,990 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"12\"\tPrecision = 0.541 (global 0.612) Î”m = -0.117\tis_issue = True\n",
      "2023-08-11 10:38:48,004 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"place\"\tPrecision = 0.828 (global 0.612) Î”m = 0.353\tis_issue = False\n",
      "2023-08-11 10:38:48,019 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualized\"\tPrecision = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:38:48,035 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"impression\"\tPrecision = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:48,051 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consistent\"\tPrecision = 0.828 (global 0.612) Î”m = 0.353\tis_issue = False\n",
      "2023-08-11 10:38:48,065 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluids\"\tPrecision = 0.786 (global 0.612) Î”m = 0.284\tis_issue = False\n",
      "2023-08-11 10:38:48,080 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"change\"\tPrecision = 0.725 (global 0.612) Î”m = 0.185\tis_issue = False\n",
      "2023-08-11 10:38:48,091 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"preoperative\"\tPrecision = 0.761 (global 0.612) Î”m = 0.243\tis_issue = False\n",
      "2023-08-11 10:38:48,105 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"changes\"\tPrecision = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:38:48,119 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilaterally\"\tPrecision = 0.646 (global 0.612) Î”m = 0.056\tis_issue = False\n",
      "2023-08-11 10:38:48,133 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"transverse\"\tPrecision = 0.812 (global 0.612) Î”m = 0.328\tis_issue = False\n",
      "2023-08-11 10:38:48,147 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"portion\"\tPrecision = 0.964 (global 0.612) Î”m = 0.576\tis_issue = False\n",
      "2023-08-11 10:38:48,159 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"noted\"\tPrecision = 0.651 (global 0.612) Î”m = 0.064\tis_issue = False\n",
      "2023-08-11 10:38:48,173 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"days\"\tPrecision = 0.450 (global 0.612) Î”m = -0.265\tis_issue = True\n",
      "2023-08-11 10:38:48,187 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carotid\"\tPrecision = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:38:48,202 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"arm\"\tPrecision = 0.591 (global 0.612) Î”m = -0.034\tis_issue = False\n",
      "2023-08-11 10:38:48,218 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"continue\"\tPrecision = 0.442 (global 0.612) Î”m = -0.277\tis_issue = True\n",
      "2023-08-11 10:38:48,234 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colon\"\tPrecision = 0.690 (global 0.612) Î”m = 0.127\tis_issue = False\n",
      "2023-08-11 10:38:48,251 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"iv\"\tPrecision = 0.758 (global 0.612) Î”m = 0.239\tis_issue = False\n",
      "2023-08-11 10:38:48,265 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ml\"\tPrecision = 0.717 (global 0.612) Î”m = 0.172\tis_issue = False\n",
      "2023-08-11 10:38:48,279 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lung\"\tPrecision = 0.400 (global 0.612) Î”m = -0.346\tis_issue = True\n",
      "2023-08-11 10:38:48,293 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspect\"\tPrecision = 0.787 (global 0.612) Î”m = 0.287\tis_issue = False\n",
      "2023-08-11 10:38:48,307 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ear\"\tPrecision = 0.619 (global 0.612) Î”m = 0.012\tis_issue = False\n",
      "2023-08-11 10:38:48,324 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stitch\"\tPrecision = 0.909 (global 0.612) Î”m = 0.486\tis_issue = False\n",
      "2023-08-11 10:38:48,338 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carefully\"\tPrecision = 0.826 (global 0.612) Î”m = 0.350\tis_issue = False\n",
      "2023-08-11 10:38:48,353 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"grasped\"\tPrecision = 0.840 (global 0.612) Î”m = 0.373\tis_issue = False\n",
      "2023-08-11 10:38:48,366 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"post\"\tPrecision = 0.552 (global 0.612) Î”m = -0.098\tis_issue = True\n",
      "2023-08-11 10:38:48,382 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"specimens\"\tPrecision = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:48,396 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"light\"\tPrecision = 0.674 (global 0.612) Î”m = 0.102\tis_issue = False\n",
      "2023-08-11 10:38:48,411 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"return\"\tPrecision = 0.514 (global 0.612) Î”m = -0.159\tis_issue = True\n",
      "2023-08-11 10:38:48,425 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"symptoms\"\tPrecision = 0.551 (global 0.612) Î”m = -0.099\tis_issue = True\n",
      "2023-08-11 10:38:48,439 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extremity\"\tPrecision = 0.619 (global 0.612) Î”m = 0.012\tis_issue = False\n",
      "2023-08-11 10:38:48,452 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"position\"\tPrecision = 0.794 (global 0.612) Î”m = 0.298\tis_issue = False\n",
      "2023-08-11 10:38:48,467 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscle\"\tPrecision = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:48,481 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"easily\"\tPrecision = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:38:48,495 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mg\"\tPrecision = 0.473 (global 0.612) Î”m = -0.227\tis_issue = True\n",
      "2023-08-11 10:38:48,510 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscles\"\tPrecision = 0.815 (global 0.612) Î”m = 0.332\tis_issue = False\n",
      "2023-08-11 10:38:48,523 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"months\"\tPrecision = 0.562 (global 0.612) Î”m = -0.081\tis_issue = True\n",
      "2023-08-11 10:38:48,536 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lateral\"\tPrecision = 0.797 (global 0.612) Î”m = 0.303\tis_issue = False\n",
      "2023-08-11 10:38:48,550 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bladder\"\tPrecision = 0.650 (global 0.612) Î”m = 0.062\tis_issue = False\n",
      "2023-08-11 10:38:48,561 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthesia\"\tPrecision = 0.764 (global 0.612) Î”m = 0.249\tis_issue = False\n",
      "2023-08-11 10:38:48,575 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"discharge\"\tPrecision = 0.424 (global 0.612) Î”m = -0.308\tis_issue = True\n",
      "2023-08-11 10:38:48,595 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspirin\"\tPrecision = 0.435 (global 0.612) Î”m = -0.289\tis_issue = True\n",
      "2023-08-11 10:38:48,613 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"difficulty\"\tPrecision = 0.734 (global 0.612) Î”m = 0.200\tis_issue = False\n",
      "2023-08-11 10:38:48,632 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oropharynx\"\tPrecision = 0.686 (global 0.612) Î”m = 0.121\tis_issue = False\n",
      "2023-08-11 10:38:48,653 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"right\"\tPrecision = 0.659 (global 0.612) Î”m = 0.077\tis_issue = False\n",
      "2023-08-11 10:38:48,669 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"description\"\tPrecision = 0.854 (global 0.612) Î”m = 0.395\tis_issue = False\n",
      "2023-08-11 10:38:48,683 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"midline\"\tPrecision = 0.836 (global 0.612) Î”m = 0.367\tis_issue = False\n",
      "2023-08-11 10:38:48,697 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"advanced\"\tPrecision = 0.759 (global 0.612) Î”m = 0.240\tis_issue = False\n",
      "2023-08-11 10:38:48,711 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"followup\"\tPrecision = 0.396 (global 0.612) Î”m = -0.353\tis_issue = True\n",
      "2023-08-11 10:38:48,725 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"08\"\tPrecision = 0.462 (global 0.612) Î”m = -0.246\tis_issue = True\n",
      "2023-08-11 10:38:48,741 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligament\"\tPrecision = 0.950 (global 0.612) Î”m = 0.553\tis_issue = False\n",
      "2023-08-11 10:38:48,756 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tone\"\tPrecision = 0.759 (global 0.612) Î”m = 0.240\tis_issue = False\n",
      "2023-08-11 10:38:48,773 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"saline\"\tPrecision = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:48,788 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"follow\"\tPrecision = 0.550 (global 0.612) Î”m = -0.101\tis_issue = True\n",
      "2023-08-11 10:38:48,802 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cancer\"\tPrecision = 0.622 (global 0.612) Î”m = 0.016\tis_issue = False\n",
      "2023-08-11 10:38:48,817 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drains\"\tPrecision = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:38:48,831 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"renal\"\tPrecision = 0.462 (global 0.612) Î”m = -0.246\tis_issue = True\n",
      "2023-08-11 10:38:48,844 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anterior\"\tPrecision = 0.756 (global 0.612) Î”m = 0.236\tis_issue = False\n",
      "2023-08-11 10:38:48,859 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"units\"\tPrecision = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:38:48,873 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"home\"\tPrecision = 0.519 (global 0.612) Î”m = -0.153\tis_issue = True\n",
      "2023-08-11 10:38:48,888 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hemostasis\"\tPrecision = 0.787 (global 0.612) Î”m = 0.286\tis_issue = False\n",
      "2023-08-11 10:38:48,905 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sleep\"\tPrecision = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:38:48,920 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"artery\"\tPrecision = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:38:48,933 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brought\"\tPrecision = 0.747 (global 0.612) Î”m = 0.221\tis_issue = False\n",
      "2023-08-11 10:38:48,947 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"underlying\"\tPrecision = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:48,962 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"membranes\"\tPrecision = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:48,975 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cc\"\tPrecision = 0.787 (global 0.612) Î”m = 0.287\tis_issue = False\n",
      "2023-08-11 10:38:48,985 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pain\"\tPrecision = 0.571 (global 0.612) Î”m = -0.066\tis_issue = True\n",
      "2023-08-11 10:38:48,999 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluid\"\tPrecision = 0.759 (global 0.612) Î”m = 0.241\tis_issue = False\n",
      "2023-08-11 10:38:49,007 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"history\"\tPrecision = 0.564 (global 0.612) Î”m = -0.078\tis_issue = True\n",
      "2023-08-11 10:38:49,023 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scissors\"\tPrecision = 0.852 (global 0.612) Î”m = 0.392\tis_issue = False\n",
      "2023-08-11 10:38:49,037 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"identified\"\tPrecision = 0.785 (global 0.612) Î”m = 0.282\tis_issue = False\n",
      "2023-08-11 10:38:51,485 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `language(transcription)` == \"en\"\tRecall = 0.614 (global 0.612) Î”m = 0.003\tis_issue = False\n",
      "2023-08-11 10:38:51,499 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cautery\"\tRecall = 0.741 (global 0.612) Î”m = 0.211\tis_issue = False\n",
      "2023-08-11 10:38:51,515 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subsequently\"\tRecall = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:51,530 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"attention\"\tRecall = 0.786 (global 0.612) Î”m = 0.284\tis_issue = False\n",
      "2023-08-11 10:38:51,544 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"coronary\"\tRecall = 0.391 (global 0.612) Î”m = -0.360\tis_issue = True\n",
      "2023-08-11 10:38:51,558 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"external\"\tRecall = 0.606 (global 0.612) Î”m = -0.009\tis_issue = False\n",
      "2023-08-11 10:38:51,573 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"equal\"\tRecall = 0.655 (global 0.612) Î”m = 0.071\tis_issue = False\n",
      "2023-08-11 10:38:51,587 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"states\"\tRecall = 0.535 (global 0.612) Î”m = -0.126\tis_issue = True\n",
      "2023-08-11 10:38:51,602 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"metzenbaum\"\tRecall = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:38:51,617 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abnormalities\"\tRecall = 0.656 (global 0.612) Î”m = 0.073\tis_issue = False\n",
      "2023-08-11 10:38:51,632 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualization\"\tRecall = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:38:51,645 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilateral\"\tRecall = 0.646 (global 0.612) Î”m = 0.056\tis_issue = False\n",
      "2023-08-11 10:38:51,658 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"head\"\tRecall = 0.646 (global 0.612) Î”m = 0.056\tis_issue = False\n",
      "2023-08-11 10:38:51,672 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"estimated\"\tRecall = 0.823 (global 0.612) Î”m = 0.344\tis_issue = False\n",
      "2023-08-11 10:38:51,686 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"numbness\"\tRecall = 0.815 (global 0.612) Î”m = 0.332\tis_issue = False\n",
      "2023-08-11 10:38:51,701 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tubes\"\tRecall = 0.783 (global 0.612) Î”m = 0.279\tis_issue = False\n",
      "2023-08-11 10:38:51,716 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"activity\"\tRecall = 0.514 (global 0.612) Î”m = -0.161\tis_issue = True\n",
      "2023-08-11 10:38:51,732 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scan\"\tRecall = 0.568 (global 0.612) Î”m = -0.071\tis_issue = True\n",
      "2023-08-11 10:38:51,749 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"single\"\tRecall = 0.696 (global 0.612) Î”m = 0.137\tis_issue = False\n",
      "2023-08-11 10:38:51,763 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extended\"\tRecall = 0.929 (global 0.612) Î”m = 0.518\tis_issue = False\n",
      "2023-08-11 10:38:51,777 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"breast\"\tRecall = 0.444 (global 0.612) Î”m = -0.274\tis_issue = True\n",
      "2023-08-11 10:38:51,792 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"problems\"\tRecall = 0.565 (global 0.612) Î”m = -0.076\tis_issue = True\n",
      "2023-08-11 10:38:51,806 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fascia\"\tRecall = 0.824 (global 0.612) Î”m = 0.346\tis_issue = False\n",
      "2023-08-11 10:38:51,820 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tract\"\tRecall = 0.450 (global 0.612) Î”m = -0.265\tis_issue = True\n",
      "2023-08-11 10:38:51,835 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lymph\"\tRecall = 0.769 (global 0.612) Î”m = 0.257\tis_issue = False\n",
      "2023-08-11 10:38:51,849 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colonoscopy\"\tRecall = 0.565 (global 0.612) Î”m = -0.076\tis_issue = True\n",
      "2023-08-11 10:38:51,864 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"region\"\tRecall = 0.724 (global 0.612) Î”m = 0.184\tis_issue = False\n",
      "2023-08-11 10:38:51,878 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medication\"\tRecall = 0.541 (global 0.612) Î”m = -0.117\tis_issue = True\n",
      "2023-08-11 10:38:51,893 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"father\"\tRecall = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:51,907 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectus\"\tRecall = 0.950 (global 0.612) Î”m = 0.553\tis_issue = False\n",
      "2023-08-11 10:38:51,921 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"function\"\tRecall = 0.414 (global 0.612) Î”m = -0.324\tis_issue = True\n",
      "2023-08-11 10:38:51,934 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fever\"\tRecall = 0.558 (global 0.612) Î”m = -0.089\tis_issue = True\n",
      "2023-08-11 10:38:51,945 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"procedure\"\tRecall = 0.726 (global 0.612) Î”m = 0.187\tis_issue = False\n",
      "2023-08-11 10:38:51,959 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearance\"\tRecall = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:51,975 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inserted\"\tRecall = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:51,991 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suction\"\tRecall = 0.826 (global 0.612) Î”m = 0.350\tis_issue = False\n",
      "2023-08-11 10:38:52,005 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removal\"\tRecall = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:38:52,018 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removed\"\tRecall = 0.790 (global 0.612) Î”m = 0.291\tis_issue = False\n",
      "2023-08-11 10:38:52,033 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reveals\"\tRecall = 0.652 (global 0.612) Î”m = 0.066\tis_issue = False\n",
      "2023-08-11 10:38:52,045 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clear\"\tRecall = 0.515 (global 0.612) Î”m = -0.158\tis_issue = True\n",
      "2023-08-11 10:38:52,060 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"age\"\tRecall = 0.639 (global 0.612) Î”m = 0.044\tis_issue = False\n",
      "2023-08-11 10:38:52,074 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissection\"\tRecall = 0.762 (global 0.612) Î”m = 0.245\tis_issue = False\n",
      "2023-08-11 10:38:52,089 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"small\"\tRecall = 0.770 (global 0.612) Î”m = 0.259\tis_issue = False\n",
      "2023-08-11 10:38:52,104 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"enlarged\"\tRecall = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:38:52,116 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"room\"\tRecall = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:38:52,129 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"biopsy\"\tRecall = 0.625 (global 0.612) Î”m = 0.021\tis_issue = False\n",
      "2023-08-11 10:38:52,145 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"irrigated\"\tRecall = 0.854 (global 0.612) Î”m = 0.395\tis_issue = False\n",
      "2023-08-11 10:38:52,162 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"swelling\"\tRecall = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:52,176 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"infection\"\tRecall = 0.607 (global 0.612) Î”m = -0.009\tis_issue = False\n",
      "2023-08-11 10:38:52,190 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"exam\"\tRecall = 0.663 (global 0.612) Î”m = 0.084\tis_issue = False\n",
      "2023-08-11 10:38:52,204 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"urinary\"\tRecall = 0.486 (global 0.612) Î”m = -0.206\tis_issue = True\n",
      "2023-08-11 10:38:52,219 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"count\"\tRecall = 0.400 (global 0.612) Î”m = -0.346\tis_issue = True\n",
      "2023-08-11 10:38:52,230 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"performed\"\tRecall = 0.683 (global 0.612) Î”m = 0.116\tis_issue = False\n",
      "2023-08-11 10:38:52,244 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dorsal\"\tRecall = 0.929 (global 0.612) Î”m = 0.518\tis_issue = False\n",
      "2023-08-11 10:38:52,259 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vicryl\"\tRecall = 0.791 (global 0.612) Î”m = 0.293\tis_issue = False\n",
      "2023-08-11 10:38:52,273 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"counts\"\tRecall = 0.892 (global 0.612) Î”m = 0.458\tis_issue = False\n",
      "2023-08-11 10:38:52,289 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"proximal\"\tRecall = 0.889 (global 0.612) Î”m = 0.453\tis_issue = False\n",
      "2023-08-11 10:38:52,302 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extending\"\tRecall = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:52,317 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medial\"\tRecall = 0.778 (global 0.612) Î”m = 0.271\tis_issue = False\n",
      "2023-08-11 10:38:52,331 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cord\"\tRecall = 0.939 (global 0.612) Î”m = 0.535\tis_issue = False\n",
      "2023-08-11 10:38:52,346 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissected\"\tRecall = 0.881 (global 0.612) Î”m = 0.440\tis_issue = False\n",
      "2023-08-11 10:38:52,360 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recurrent\"\tRecall = 0.654 (global 0.612) Î”m = 0.069\tis_issue = False\n",
      "2023-08-11 10:38:52,374 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cavity\"\tRecall = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:38:52,388 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knee\"\tRecall = 0.591 (global 0.612) Î”m = -0.034\tis_issue = False\n",
      "2023-08-11 10:38:52,402 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adequate\"\tRecall = 0.816 (global 0.612) Î”m = 0.333\tis_issue = False\n",
      "2023-08-11 10:38:52,421 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"length\"\tRecall = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:38:52,441 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complex\"\tRecall = 0.522 (global 0.612) Î”m = -0.147\tis_issue = True\n",
      "2023-08-11 10:38:52,461 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"having\"\tRecall = 0.523 (global 0.612) Î”m = -0.145\tis_issue = True\n",
      "2023-08-11 10:38:52,479 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mass\"\tRecall = 0.596 (global 0.612) Î”m = -0.026\tis_issue = False\n",
      "2023-08-11 10:38:52,496 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"administered\"\tRecall = 0.767 (global 0.612) Î”m = 0.253\tis_issue = False\n",
      "2023-08-11 10:38:52,511 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"blade\"\tRecall = 0.900 (global 0.612) Î”m = 0.471\tis_issue = False\n",
      "2023-08-11 10:38:52,524 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denies\"\tRecall = 0.459 (global 0.612) Î”m = -0.250\tis_issue = True\n",
      "2023-08-11 10:38:52,539 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthetic\"\tRecall = 0.913 (global 0.612) Î”m = 0.492\tis_issue = False\n",
      "2023-08-11 10:38:52,552 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complications\"\tRecall = 0.697 (global 0.612) Î”m = 0.140\tis_issue = False\n",
      "2023-08-11 10:38:52,568 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"erythema\"\tRecall = 0.655 (global 0.612) Î”m = 0.071\tis_issue = False\n",
      "2023-08-11 10:38:52,582 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"local\"\tRecall = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:38:52,597 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"delivered\"\tRecall = 0.920 (global 0.612) Î”m = 0.504\tis_issue = False\n",
      "2023-08-11 10:38:52,611 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"direct\"\tRecall = 0.731 (global 0.612) Î”m = 0.194\tis_issue = False\n",
      "2023-08-11 10:38:52,625 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"negative\"\tRecall = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:52,639 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nasal\"\tRecall = 0.594 (global 0.612) Î”m = -0.030\tis_issue = False\n",
      "2023-08-11 10:38:52,654 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"end\"\tRecall = 0.757 (global 0.612) Î”m = 0.237\tis_issue = False\n",
      "2023-08-11 10:38:52,670 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suture\"\tRecall = 0.804 (global 0.612) Î”m = 0.315\tis_issue = False\n",
      "2023-08-11 10:38:52,686 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incised\"\tRecall = 0.900 (global 0.612) Î”m = 0.471\tis_issue = False\n",
      "2023-08-11 10:38:52,701 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"strength\"\tRecall = 0.738 (global 0.612) Î”m = 0.206\tis_issue = False\n",
      "2023-08-11 10:38:52,714 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mri\"\tRecall = 0.708 (global 0.612) Î”m = 0.158\tis_issue = False\n",
      "2023-08-11 10:38:52,731 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"areas\"\tRecall = 0.607 (global 0.612) Î”m = -0.008\tis_issue = False\n",
      "2023-08-11 10:38:52,746 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"second\"\tRecall = 0.679 (global 0.612) Î”m = 0.110\tis_issue = False\n",
      "2023-08-11 10:38:52,758 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"examination\"\tRecall = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:52,773 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spinal\"\tRecall = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:52,787 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"admission\"\tRecall = 0.375 (global 0.612) Î”m = -0.387\tis_issue = True\n",
      "2023-08-11 10:38:52,801 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"open\"\tRecall = 0.769 (global 0.612) Î”m = 0.257\tis_issue = False\n",
      "2023-08-11 10:38:52,815 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mouth\"\tRecall = 0.700 (global 0.612) Î”m = 0.144\tis_issue = False\n",
      "2023-08-11 10:38:52,829 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pulmonary\"\tRecall = 0.533 (global 0.612) Î”m = -0.128\tis_issue = True\n",
      "2023-08-11 10:38:52,844 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"flap\"\tRecall = 0.727 (global 0.612) Î”m = 0.189\tis_issue = False\n",
      "2023-08-11 10:38:52,858 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"treatment\"\tRecall = 0.518 (global 0.612) Î”m = -0.154\tis_issue = True\n",
      "2023-08-11 10:38:52,872 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"resection\"\tRecall = 0.700 (global 0.612) Î”m = 0.144\tis_issue = False\n",
      "2023-08-11 10:38:52,887 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"contrast\"\tRecall = 0.708 (global 0.612) Î”m = 0.158\tis_issue = False\n",
      "2023-08-11 10:38:52,902 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spine\"\tRecall = 0.703 (global 0.612) Î”m = 0.148\tis_issue = False\n",
      "2023-08-11 10:38:52,916 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cm\"\tRecall = 0.716 (global 0.612) Î”m = 0.171\tis_issue = False\n",
      "2023-08-11 10:38:52,929 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervical\"\tRecall = 0.708 (global 0.612) Î”m = 0.158\tis_issue = False\n",
      "2023-08-11 10:38:52,943 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abc\"\tRecall = 0.417 (global 0.612) Î”m = -0.319\tis_issue = True\n",
      "2023-08-11 10:38:52,957 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligated\"\tRecall = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:38:52,973 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sinus\"\tRecall = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:38:52,987 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oxygen\"\tRecall = 0.591 (global 0.612) Î”m = -0.034\tis_issue = False\n",
      "2023-08-11 10:38:53,001 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distal\"\tRecall = 0.795 (global 0.612) Î”m = 0.299\tis_issue = False\n",
      "2023-08-11 10:38:53,015 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sterile\"\tRecall = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:53,029 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinical\"\tRecall = 0.565 (global 0.612) Î”m = -0.076\tis_issue = True\n",
      "2023-08-11 10:38:53,043 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mm\"\tRecall = 0.827 (global 0.612) Î”m = 0.351\tis_issue = False\n",
      "2023-08-11 10:38:53,057 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cardiac\"\tRecall = 0.529 (global 0.612) Î”m = -0.135\tis_issue = True\n",
      "2023-08-11 10:38:53,072 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visit\"\tRecall = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:53,086 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearing\"\tRecall = 0.680 (global 0.612) Î”m = 0.111\tis_issue = False\n",
      "2023-08-11 10:38:53,101 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clamped\"\tRecall = 0.960 (global 0.612) Î”m = 0.569\tis_issue = False\n",
      "2023-08-11 10:38:53,115 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectum\"\tRecall = 0.650 (global 0.612) Î”m = 0.062\tis_issue = False\n",
      "2023-08-11 10:38:53,129 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"draped\"\tRecall = 0.773 (global 0.612) Î”m = 0.263\tis_issue = False\n",
      "2023-08-11 10:38:53,144 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stomach\"\tRecall = 0.850 (global 0.612) Î”m = 0.389\tis_issue = False\n",
      "2023-08-11 10:38:53,158 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wound\"\tRecall = 0.638 (global 0.612) Î”m = 0.043\tis_issue = False\n",
      "2023-08-11 10:38:53,171 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"family\"\tRecall = 0.573 (global 0.612) Î”m = -0.063\tis_issue = True\n",
      "2023-08-11 10:38:53,185 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nerve\"\tRecall = 0.850 (global 0.612) Î”m = 0.389\tis_issue = False\n",
      "2023-08-11 10:38:53,199 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cut\"\tRecall = 0.917 (global 0.612) Î”m = 0.498\tis_issue = False\n",
      "2023-08-11 10:38:53,213 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesions\"\tRecall = 0.681 (global 0.612) Î”m = 0.113\tis_issue = False\n",
      "2023-08-11 10:38:53,228 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervix\"\tRecall = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:53,244 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"technique\"\tRecall = 0.783 (global 0.612) Î”m = 0.279\tis_issue = False\n",
      "2023-08-11 10:38:53,257 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"point\"\tRecall = 0.615 (global 0.612) Î”m = 0.006\tis_issue = False\n",
      "2023-08-11 10:38:53,271 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"leg\"\tRecall = 0.567 (global 0.612) Î”m = -0.074\tis_issue = True\n",
      "2023-08-11 10:38:53,292 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"s1\"\tRecall = 0.556 (global 0.612) Î”m = -0.092\tis_issue = True\n",
      "2023-08-11 10:38:53,312 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"xyz\"\tRecall = 0.318 (global 0.612) Î”m = -0.480\tis_issue = True\n",
      "2023-08-11 10:38:53,337 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"office\"\tRecall = 0.476 (global 0.612) Î”m = -0.222\tis_issue = True\n",
      "2023-08-11 10:38:53,358 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gross\"\tRecall = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:53,372 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"check\"\tRecall = 0.560 (global 0.612) Î”m = -0.085\tis_issue = True\n",
      "2023-08-11 10:38:53,387 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"created\"\tRecall = 0.950 (global 0.612) Î”m = 0.553\tis_issue = False\n",
      "2023-08-11 10:38:53,401 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hours\"\tRecall = 0.558 (global 0.612) Î”m = -0.089\tis_issue = True\n",
      "2023-08-11 10:38:53,415 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"body\"\tRecall = 0.816 (global 0.612) Î”m = 0.333\tis_issue = False\n",
      "2023-08-11 10:38:53,429 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"entered\"\tRecall = 0.875 (global 0.612) Î”m = 0.430\tis_issue = False\n",
      "2023-08-11 10:38:53,443 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectal\"\tRecall = 0.643 (global 0.612) Î”m = 0.051\tis_issue = False\n",
      "2023-08-11 10:38:53,459 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adenopathy\"\tRecall = 0.778 (global 0.612) Î”m = 0.271\tis_issue = False\n",
      "2023-08-11 10:38:53,473 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterine\"\tRecall = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:38:53,487 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"high\"\tRecall = 0.488 (global 0.612) Î”m = -0.203\tis_issue = True\n",
      "2023-08-11 10:38:53,501 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"correct\"\tRecall = 0.881 (global 0.612) Î”m = 0.440\tis_issue = False\n",
      "2023-08-11 10:38:53,516 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pupils\"\tRecall = 0.706 (global 0.612) Î”m = 0.154\tis_issue = False\n",
      "2023-08-11 10:38:53,529 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"upper\"\tRecall = 0.718 (global 0.612) Î”m = 0.173\tis_issue = False\n",
      "2023-08-11 10:38:53,543 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visual\"\tRecall = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:38:53,557 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"revealed\"\tRecall = 0.742 (global 0.612) Î”m = 0.213\tis_issue = False\n",
      "2023-08-11 10:38:53,572 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tumor\"\tRecall = 0.583 (global 0.612) Î”m = -0.047\tis_issue = False\n",
      "2023-08-11 10:38:53,586 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distress\"\tRecall = 0.486 (global 0.612) Î”m = -0.206\tis_issue = True\n",
      "2023-08-11 10:38:53,599 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"unremarkable\"\tRecall = 0.632 (global 0.612) Î”m = 0.033\tis_issue = False\n",
      "2023-08-11 10:38:53,614 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"greater\"\tRecall = 0.435 (global 0.612) Î”m = -0.289\tis_issue = True\n",
      "2023-08-11 10:38:53,630 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"internal\"\tRecall = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:38:53,645 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recovery\"\tRecall = 0.843 (global 0.612) Î”m = 0.378\tis_issue = False\n",
      "2023-08-11 10:38:53,661 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scope\"\tRecall = 0.864 (global 0.612) Î”m = 0.411\tis_issue = False\n",
      "2023-08-11 10:38:53,676 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"skin\"\tRecall = 0.723 (global 0.612) Î”m = 0.182\tis_issue = False\n",
      "2023-08-11 10:38:53,687 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"postoperative\"\tRecall = 0.729 (global 0.612) Î”m = 0.191\tis_issue = False\n",
      "2023-08-11 10:38:53,701 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"evidence\"\tRecall = 0.691 (global 0.612) Î”m = 0.130\tis_issue = False\n",
      "2023-08-11 10:38:53,715 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knife\"\tRecall = 0.952 (global 0.612) Î”m = 0.557\tis_issue = False\n",
      "2023-08-11 10:38:53,732 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sac\"\tRecall = 0.913 (global 0.612) Î”m = 0.492\tis_issue = False\n",
      "2023-08-11 10:38:53,748 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sutures\"\tRecall = 0.636 (global 0.612) Î”m = 0.040\tis_issue = False\n",
      "2023-08-11 10:38:53,762 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterus\"\tRecall = 0.647 (global 0.612) Î”m = 0.058\tis_issue = False\n",
      "2023-08-11 10:38:53,777 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inferior\"\tRecall = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:38:53,793 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"similar\"\tRecall = 0.724 (global 0.612) Î”m = 0.184\tis_issue = False\n",
      "2023-08-11 10:38:53,806 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bleeding\"\tRecall = 0.727 (global 0.612) Î”m = 0.189\tis_issue = False\n",
      "2023-08-11 10:38:53,820 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"large\"\tRecall = 0.647 (global 0.612) Î”m = 0.058\tis_issue = False\n",
      "2023-08-11 10:38:53,831 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"normal\"\tRecall = 0.641 (global 0.612) Î”m = 0.048\tis_issue = False\n",
      "2023-08-11 10:38:53,844 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"posterior\"\tRecall = 0.768 (global 0.612) Î”m = 0.255\tis_issue = False\n",
      "2023-08-11 10:38:53,858 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"chromic\"\tRecall = 0.952 (global 0.612) Î”m = 0.557\tis_issue = False\n",
      "2023-08-11 10:38:53,872 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"round\"\tRecall = 0.774 (global 0.612) Î”m = 0.265\tis_issue = False\n",
      "2023-08-11 10:38:53,886 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ct\"\tRecall = 0.600 (global 0.612) Î”m = -0.019\tis_issue = False\n",
      "2023-08-11 10:38:53,901 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incision\"\tRecall = 0.747 (global 0.612) Î”m = 0.221\tis_issue = False\n",
      "2023-08-11 10:38:53,914 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesion\"\tRecall = 0.692 (global 0.612) Î”m = 0.131\tis_issue = False\n",
      "2023-08-11 10:38:53,928 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fashion\"\tRecall = 0.807 (global 0.612) Î”m = 0.319\tis_issue = False\n",
      "2023-08-11 10:38:53,943 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vaginal\"\tRecall = 0.586 (global 0.612) Î”m = -0.042\tis_issue = False\n",
      "2023-08-11 10:38:53,959 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subcuticular\"\tRecall = 0.792 (global 0.612) Î”m = 0.294\tis_issue = False\n",
      "2023-08-11 10:38:53,976 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stable\"\tRecall = 0.627 (global 0.612) Î”m = 0.025\tis_issue = False\n",
      "2023-08-11 10:38:53,990 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mucosa\"\tRecall = 0.697 (global 0.612) Î”m = 0.139\tis_issue = False\n",
      "2023-08-11 10:38:54,005 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"shoulder\"\tRecall = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:38:54,018 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"use\"\tRecall = 0.681 (global 0.612) Î”m = 0.112\tis_issue = False\n",
      "2023-08-11 10:38:54,032 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nose\"\tRecall = 0.794 (global 0.612) Î”m = 0.298\tis_issue = False\n",
      "2023-08-11 10:38:54,047 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rule\"\tRecall = 0.450 (global 0.612) Î”m = -0.265\tis_issue = True\n",
      "2023-08-11 10:38:54,062 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denied\"\tRecall = 0.773 (global 0.612) Î”m = 0.263\tis_issue = False\n",
      "2023-08-11 10:38:54,074 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"left\"\tRecall = 0.693 (global 0.612) Î”m = 0.132\tis_issue = False\n",
      "2023-08-11 10:38:54,086 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prepped\"\tRecall = 0.767 (global 0.612) Î”m = 0.253\tis_issue = False\n",
      "2023-08-11 10:38:54,100 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"diarrhea\"\tRecall = 0.462 (global 0.612) Î”m = -0.246\tis_issue = True\n",
      "2023-08-11 10:38:54,116 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"improved\"\tRecall = 0.419 (global 0.612) Î”m = -0.315\tis_issue = True\n",
      "2023-08-11 10:38:54,128 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"loss\"\tRecall = 0.781 (global 0.612) Î”m = 0.276\tis_issue = False\n",
      "2023-08-11 10:38:54,143 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gentleman\"\tRecall = 0.696 (global 0.612) Î”m = 0.137\tis_issue = False\n",
      "2023-08-11 10:38:54,158 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sensory\"\tRecall = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:38:54,173 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"laterally\"\tRecall = 0.846 (global 0.612) Î”m = 0.383\tis_issue = False\n",
      "2023-08-11 10:38:54,187 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"needle\"\tRecall = 0.821 (global 0.612) Î”m = 0.343\tis_issue = False\n",
      "2023-08-11 10:38:54,199 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"findings\"\tRecall = 0.691 (global 0.612) Î”m = 0.129\tis_issue = False\n",
      "2023-08-11 10:38:54,214 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tolerated\"\tRecall = 0.698 (global 0.612) Î”m = 0.140\tis_issue = False\n",
      "2023-08-11 10:38:54,226 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"neck\"\tRecall = 0.553 (global 0.612) Î”m = -0.096\tis_issue = True\n",
      "2023-08-11 10:38:54,243 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headache\"\tRecall = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:54,263 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"condition\"\tRecall = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:54,285 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wall\"\tRecall = 0.763 (global 0.612) Î”m = 0.247\tis_issue = False\n",
      "2023-08-11 10:38:54,305 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abdominal\"\tRecall = 0.615 (global 0.612) Î”m = 0.006\tis_issue = False\n",
      "2023-08-11 10:38:54,319 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"placed\"\tRecall = 0.780 (global 0.612) Î”m = 0.274\tis_issue = False\n",
      "2023-08-11 10:38:54,335 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"supine\"\tRecall = 0.764 (global 0.612) Î”m = 0.248\tis_issue = False\n",
      "2023-08-11 10:38:54,349 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prostate\"\tRecall = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:38:54,362 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tube\"\tRecall = 0.773 (global 0.612) Î”m = 0.263\tis_issue = False\n",
      "2023-08-11 10:38:54,377 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gait\"\tRecall = 0.742 (global 0.612) Î”m = 0.213\tis_issue = False\n",
      "2023-08-11 10:38:54,392 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pelvis\"\tRecall = 0.733 (global 0.612) Î”m = 0.199\tis_issue = False\n",
      "2023-08-11 10:38:54,409 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suctioned\"\tRecall = 0.952 (global 0.612) Î”m = 0.557\tis_issue = False\n",
      "2023-08-11 10:38:54,423 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"passed\"\tRecall = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:54,438 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subjective\"\tRecall = 0.367 (global 0.612) Î”m = -0.401\tis_issue = True\n",
      "2023-08-11 10:38:54,452 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"able\"\tRecall = 0.633 (global 0.612) Î”m = 0.034\tis_issue = False\n",
      "2023-08-11 10:38:54,468 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lumbar\"\tRecall = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:54,483 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"running\"\tRecall = 0.825 (global 0.612) Î”m = 0.348\tis_issue = False\n",
      "2023-08-11 10:38:54,490 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"patient\"\tRecall = 0.597 (global 0.612) Î”m = -0.023\tis_issue = False\n",
      "2023-08-11 10:38:54,503 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pregnancy\"\tRecall = 0.783 (global 0.612) Î”m = 0.279\tis_issue = False\n",
      "2023-08-11 10:38:54,518 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brain\"\tRecall = 0.645 (global 0.612) Î”m = 0.054\tis_issue = False\n",
      "2023-08-11 10:38:54,534 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drainage\"\tRecall = 0.630 (global 0.612) Î”m = 0.029\tis_issue = False\n",
      "2023-08-11 10:38:54,549 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"indications\"\tRecall = 0.889 (global 0.612) Î”m = 0.453\tis_issue = False\n",
      "2023-08-11 10:38:54,562 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"taken\"\tRecall = 0.713 (global 0.612) Î”m = 0.165\tis_issue = False\n",
      "2023-08-11 10:38:54,577 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"masses\"\tRecall = 0.605 (global 0.612) Î”m = -0.011\tis_issue = False\n",
      "2023-08-11 10:38:54,593 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suite\"\tRecall = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:54,605 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"day\"\tRecall = 0.481 (global 0.612) Î”m = -0.214\tis_issue = True\n",
      "2023-08-11 10:38:54,621 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sponge\"\tRecall = 0.886 (global 0.612) Î”m = 0.448\tis_issue = False\n",
      "2023-08-11 10:38:54,634 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reports\"\tRecall = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:38:54,649 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vision\"\tRecall = 0.778 (global 0.612) Î”m = 0.271\tis_issue = False\n",
      "2023-08-11 10:38:54,663 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"foley\"\tRecall = 0.730 (global 0.612) Î”m = 0.193\tis_issue = False\n",
      "2023-08-11 10:38:54,678 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"presentation\"\tRecall = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:54,691 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"20\"\tRecall = 0.598 (global 0.612) Î”m = -0.023\tis_issue = False\n",
      "2023-08-11 10:38:54,705 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"daily\"\tRecall = 0.385 (global 0.612) Î”m = -0.371\tis_issue = True\n",
      "2023-08-11 10:38:54,719 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"motor\"\tRecall = 0.795 (global 0.612) Î”m = 0.300\tis_issue = False\n",
      "2023-08-11 10:38:54,734 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"epinephrine\"\tRecall = 0.650 (global 0.612) Î”m = 0.062\tis_issue = False\n",
      "2023-08-11 10:38:54,749 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"speech\"\tRecall = 0.704 (global 0.612) Î”m = 0.150\tis_issue = False\n",
      "2023-08-11 10:38:54,765 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"superior\"\tRecall = 0.742 (global 0.612) Î”m = 0.213\tis_issue = False\n",
      "2023-08-11 10:38:54,780 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"section\"\tRecall = 0.692 (global 0.612) Î”m = 0.131\tis_issue = False\n",
      "2023-08-11 10:38:54,794 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mild\"\tRecall = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:38:54,809 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinic\"\tRecall = 0.542 (global 0.612) Î”m = -0.115\tis_issue = True\n",
      "2023-08-11 10:38:54,824 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reviewed\"\tRecall = 0.529 (global 0.612) Î”m = -0.135\tis_issue = True\n",
      "2023-08-11 10:38:54,839 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appeared\"\tRecall = 0.765 (global 0.612) Î”m = 0.250\tis_issue = False\n",
      "2023-08-11 10:38:54,853 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hx\"\tRecall = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:54,867 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consent\"\tRecall = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:38:54,880 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"found\"\tRecall = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:54,895 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bovie\"\tRecall = 0.840 (global 0.612) Î”m = 0.373\tis_issue = False\n",
      "2023-08-11 10:38:54,908 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"weakness\"\tRecall = 0.732 (global 0.612) Î”m = 0.196\tis_issue = False\n",
      "2023-08-11 10:38:54,921 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"obtained\"\tRecall = 0.737 (global 0.612) Î”m = 0.205\tis_issue = False\n",
      "2023-08-11 10:38:54,937 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"informed\"\tRecall = 0.854 (global 0.612) Î”m = 0.395\tis_issue = False\n",
      "2023-08-11 10:38:54,952 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"initially\"\tRecall = 0.517 (global 0.612) Î”m = -0.155\tis_issue = True\n",
      "2023-08-11 10:38:54,967 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"status\"\tRecall = 0.515 (global 0.612) Î”m = -0.158\tis_issue = True\n",
      "2023-08-11 10:38:54,981 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"care\"\tRecall = 0.537 (global 0.612) Î”m = -0.122\tis_issue = True\n",
      "2023-08-11 10:38:54,997 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stool\"\tRecall = 0.516 (global 0.612) Î”m = -0.156\tis_issue = True\n",
      "2023-08-11 10:38:55,012 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"following\"\tRecall = 0.609 (global 0.612) Î”m = -0.004\tis_issue = False\n",
      "2023-08-11 10:38:55,026 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reflexes\"\tRecall = 0.775 (global 0.612) Î”m = 0.267\tis_issue = False\n",
      "2023-08-11 10:38:55,042 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scalpel\"\tRecall = 0.957 (global 0.612) Î”m = 0.563\tis_issue = False\n",
      "2023-08-11 10:38:55,056 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tenderness\"\tRecall = 0.543 (global 0.612) Î”m = -0.112\tis_issue = True\n",
      "2023-08-11 10:38:55,070 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"compression\"\tRecall = 0.636 (global 0.612) Î”m = 0.040\tis_issue = False\n",
      "2023-08-11 10:38:55,086 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headaches\"\tRecall = 0.840 (global 0.612) Î”m = 0.373\tis_issue = False\n",
      "2023-08-11 10:38:55,100 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"closed\"\tRecall = 0.806 (global 0.612) Î”m = 0.317\tis_issue = False\n",
      "2023-08-11 10:38:55,114 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"12\"\tRecall = 0.541 (global 0.612) Î”m = -0.117\tis_issue = True\n",
      "2023-08-11 10:38:55,127 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"place\"\tRecall = 0.828 (global 0.612) Î”m = 0.353\tis_issue = False\n",
      "2023-08-11 10:38:55,142 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualized\"\tRecall = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:38:55,158 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"impression\"\tRecall = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:38:55,172 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consistent\"\tRecall = 0.828 (global 0.612) Î”m = 0.353\tis_issue = False\n",
      "2023-08-11 10:38:55,190 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluids\"\tRecall = 0.786 (global 0.612) Î”m = 0.284\tis_issue = False\n",
      "2023-08-11 10:38:55,210 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"change\"\tRecall = 0.725 (global 0.612) Î”m = 0.185\tis_issue = False\n",
      "2023-08-11 10:38:55,226 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"preoperative\"\tRecall = 0.761 (global 0.612) Î”m = 0.243\tis_issue = False\n",
      "2023-08-11 10:38:55,246 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"changes\"\tRecall = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:38:55,262 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilaterally\"\tRecall = 0.646 (global 0.612) Î”m = 0.056\tis_issue = False\n",
      "2023-08-11 10:38:55,277 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"transverse\"\tRecall = 0.812 (global 0.612) Î”m = 0.328\tis_issue = False\n",
      "2023-08-11 10:38:55,292 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"portion\"\tRecall = 0.964 (global 0.612) Î”m = 0.576\tis_issue = False\n",
      "2023-08-11 10:38:55,304 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"noted\"\tRecall = 0.651 (global 0.612) Î”m = 0.064\tis_issue = False\n",
      "2023-08-11 10:38:55,317 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"days\"\tRecall = 0.450 (global 0.612) Î”m = -0.265\tis_issue = True\n",
      "2023-08-11 10:38:55,331 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carotid\"\tRecall = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:38:55,345 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"arm\"\tRecall = 0.591 (global 0.612) Î”m = -0.034\tis_issue = False\n",
      "2023-08-11 10:38:55,360 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"continue\"\tRecall = 0.442 (global 0.612) Î”m = -0.277\tis_issue = True\n",
      "2023-08-11 10:38:55,374 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colon\"\tRecall = 0.690 (global 0.612) Î”m = 0.127\tis_issue = False\n",
      "2023-08-11 10:38:55,390 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"iv\"\tRecall = 0.758 (global 0.612) Î”m = 0.239\tis_issue = False\n",
      "2023-08-11 10:38:55,403 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ml\"\tRecall = 0.717 (global 0.612) Î”m = 0.172\tis_issue = False\n",
      "2023-08-11 10:38:55,418 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lung\"\tRecall = 0.400 (global 0.612) Î”m = -0.346\tis_issue = True\n",
      "2023-08-11 10:38:55,431 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspect\"\tRecall = 0.787 (global 0.612) Î”m = 0.287\tis_issue = False\n",
      "2023-08-11 10:38:55,446 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ear\"\tRecall = 0.619 (global 0.612) Î”m = 0.012\tis_issue = False\n",
      "2023-08-11 10:38:55,462 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stitch\"\tRecall = 0.909 (global 0.612) Î”m = 0.486\tis_issue = False\n",
      "2023-08-11 10:38:55,477 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carefully\"\tRecall = 0.826 (global 0.612) Î”m = 0.350\tis_issue = False\n",
      "2023-08-11 10:38:55,491 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"grasped\"\tRecall = 0.840 (global 0.612) Î”m = 0.373\tis_issue = False\n",
      "2023-08-11 10:38:55,504 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"post\"\tRecall = 0.552 (global 0.612) Î”m = -0.098\tis_issue = True\n",
      "2023-08-11 10:38:55,520 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"specimens\"\tRecall = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:55,534 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"light\"\tRecall = 0.674 (global 0.612) Î”m = 0.102\tis_issue = False\n",
      "2023-08-11 10:38:55,550 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"return\"\tRecall = 0.514 (global 0.612) Î”m = -0.159\tis_issue = True\n",
      "2023-08-11 10:38:55,563 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"symptoms\"\tRecall = 0.551 (global 0.612) Î”m = -0.099\tis_issue = True\n",
      "2023-08-11 10:38:55,578 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extremity\"\tRecall = 0.619 (global 0.612) Î”m = 0.012\tis_issue = False\n",
      "2023-08-11 10:38:55,590 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"position\"\tRecall = 0.794 (global 0.612) Î”m = 0.298\tis_issue = False\n",
      "2023-08-11 10:38:55,604 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscle\"\tRecall = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:55,619 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"easily\"\tRecall = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:38:55,633 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mg\"\tRecall = 0.473 (global 0.612) Î”m = -0.227\tis_issue = True\n",
      "2023-08-11 10:38:55,649 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscles\"\tRecall = 0.815 (global 0.612) Î”m = 0.332\tis_issue = False\n",
      "2023-08-11 10:38:55,663 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"months\"\tRecall = 0.562 (global 0.612) Î”m = -0.081\tis_issue = True\n",
      "2023-08-11 10:38:55,676 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lateral\"\tRecall = 0.797 (global 0.612) Î”m = 0.303\tis_issue = False\n",
      "2023-08-11 10:38:55,691 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bladder\"\tRecall = 0.650 (global 0.612) Î”m = 0.062\tis_issue = False\n",
      "2023-08-11 10:38:55,702 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthesia\"\tRecall = 0.764 (global 0.612) Î”m = 0.249\tis_issue = False\n",
      "2023-08-11 10:38:55,716 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"discharge\"\tRecall = 0.424 (global 0.612) Î”m = -0.308\tis_issue = True\n",
      "2023-08-11 10:38:55,731 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspirin\"\tRecall = 0.435 (global 0.612) Î”m = -0.289\tis_issue = True\n",
      "2023-08-11 10:38:55,744 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"difficulty\"\tRecall = 0.734 (global 0.612) Î”m = 0.200\tis_issue = False\n",
      "2023-08-11 10:38:55,758 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oropharynx\"\tRecall = 0.686 (global 0.612) Î”m = 0.121\tis_issue = False\n",
      "2023-08-11 10:38:55,768 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"right\"\tRecall = 0.659 (global 0.612) Î”m = 0.077\tis_issue = False\n",
      "2023-08-11 10:38:55,782 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"description\"\tRecall = 0.854 (global 0.612) Î”m = 0.395\tis_issue = False\n",
      "2023-08-11 10:38:55,795 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"midline\"\tRecall = 0.836 (global 0.612) Î”m = 0.367\tis_issue = False\n",
      "2023-08-11 10:38:55,809 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"advanced\"\tRecall = 0.759 (global 0.612) Î”m = 0.240\tis_issue = False\n",
      "2023-08-11 10:38:55,824 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"followup\"\tRecall = 0.396 (global 0.612) Î”m = -0.353\tis_issue = True\n",
      "2023-08-11 10:38:55,838 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"08\"\tRecall = 0.462 (global 0.612) Î”m = -0.246\tis_issue = True\n",
      "2023-08-11 10:38:55,852 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligament\"\tRecall = 0.950 (global 0.612) Î”m = 0.553\tis_issue = False\n",
      "2023-08-11 10:38:55,866 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tone\"\tRecall = 0.759 (global 0.612) Î”m = 0.240\tis_issue = False\n",
      "2023-08-11 10:38:55,883 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"saline\"\tRecall = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:55,896 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"follow\"\tRecall = 0.550 (global 0.612) Î”m = -0.101\tis_issue = True\n",
      "2023-08-11 10:38:55,911 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cancer\"\tRecall = 0.622 (global 0.612) Î”m = 0.016\tis_issue = False\n",
      "2023-08-11 10:38:55,925 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drains\"\tRecall = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:38:55,939 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"renal\"\tRecall = 0.462 (global 0.612) Î”m = -0.246\tis_issue = True\n",
      "2023-08-11 10:38:55,952 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anterior\"\tRecall = 0.756 (global 0.612) Î”m = 0.236\tis_issue = False\n",
      "2023-08-11 10:38:55,966 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"units\"\tRecall = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:38:55,979 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"home\"\tRecall = 0.519 (global 0.612) Î”m = -0.153\tis_issue = True\n",
      "2023-08-11 10:38:55,995 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hemostasis\"\tRecall = 0.787 (global 0.612) Î”m = 0.286\tis_issue = False\n",
      "2023-08-11 10:38:56,024 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sleep\"\tRecall = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:38:56,041 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"artery\"\tRecall = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:38:56,057 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brought\"\tRecall = 0.747 (global 0.612) Î”m = 0.221\tis_issue = False\n",
      "2023-08-11 10:38:56,072 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"underlying\"\tRecall = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:56,087 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"membranes\"\tRecall = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:56,100 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cc\"\tRecall = 0.787 (global 0.612) Î”m = 0.287\tis_issue = False\n",
      "2023-08-11 10:38:56,111 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pain\"\tRecall = 0.571 (global 0.612) Î”m = -0.066\tis_issue = True\n",
      "2023-08-11 10:38:56,125 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluid\"\tRecall = 0.759 (global 0.612) Î”m = 0.241\tis_issue = False\n",
      "2023-08-11 10:38:56,133 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"history\"\tRecall = 0.564 (global 0.612) Î”m = -0.078\tis_issue = True\n",
      "2023-08-11 10:38:56,150 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scissors\"\tRecall = 0.852 (global 0.612) Î”m = 0.392\tis_issue = False\n",
      "2023-08-11 10:38:56,165 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"identified\"\tRecall = 0.785 (global 0.612) Î”m = 0.282\tis_issue = False\n",
      "2023-08-11 10:38:58,602 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `language(transcription)` == \"en\"\tAccuracy = 0.614 (global 0.612) Î”m = 0.003\tis_issue = False\n",
      "2023-08-11 10:38:58,615 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cautery\"\tAccuracy = 0.741 (global 0.612) Î”m = 0.211\tis_issue = False\n",
      "2023-08-11 10:38:58,630 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subsequently\"\tAccuracy = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:58,645 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"attention\"\tAccuracy = 0.786 (global 0.612) Î”m = 0.284\tis_issue = False\n",
      "2023-08-11 10:38:58,659 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"coronary\"\tAccuracy = 0.391 (global 0.612) Î”m = -0.360\tis_issue = True\n",
      "2023-08-11 10:38:58,673 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"external\"\tAccuracy = 0.606 (global 0.612) Î”m = -0.009\tis_issue = False\n",
      "2023-08-11 10:38:58,687 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"equal\"\tAccuracy = 0.655 (global 0.612) Î”m = 0.071\tis_issue = False\n",
      "2023-08-11 10:38:58,702 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"states\"\tAccuracy = 0.535 (global 0.612) Î”m = -0.126\tis_issue = True\n",
      "2023-08-11 10:38:58,716 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"metzenbaum\"\tAccuracy = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:38:58,732 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abnormalities\"\tAccuracy = 0.656 (global 0.612) Î”m = 0.073\tis_issue = False\n",
      "2023-08-11 10:38:58,751 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualization\"\tAccuracy = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:38:58,768 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilateral\"\tAccuracy = 0.646 (global 0.612) Î”m = 0.056\tis_issue = False\n",
      "2023-08-11 10:38:58,786 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"head\"\tAccuracy = 0.646 (global 0.612) Î”m = 0.056\tis_issue = False\n",
      "2023-08-11 10:38:58,802 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"estimated\"\tAccuracy = 0.823 (global 0.612) Î”m = 0.344\tis_issue = False\n",
      "2023-08-11 10:38:58,817 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"numbness\"\tAccuracy = 0.815 (global 0.612) Î”m = 0.332\tis_issue = False\n",
      "2023-08-11 10:38:58,831 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tubes\"\tAccuracy = 0.783 (global 0.612) Î”m = 0.279\tis_issue = False\n",
      "2023-08-11 10:38:58,847 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"activity\"\tAccuracy = 0.514 (global 0.612) Î”m = -0.161\tis_issue = True\n",
      "2023-08-11 10:38:58,863 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scan\"\tAccuracy = 0.568 (global 0.612) Î”m = -0.071\tis_issue = True\n",
      "2023-08-11 10:38:58,878 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"single\"\tAccuracy = 0.696 (global 0.612) Î”m = 0.137\tis_issue = False\n",
      "2023-08-11 10:38:58,892 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extended\"\tAccuracy = 0.929 (global 0.612) Î”m = 0.518\tis_issue = False\n",
      "2023-08-11 10:38:58,907 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"breast\"\tAccuracy = 0.444 (global 0.612) Î”m = -0.274\tis_issue = True\n",
      "2023-08-11 10:38:58,921 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"problems\"\tAccuracy = 0.565 (global 0.612) Î”m = -0.076\tis_issue = True\n",
      "2023-08-11 10:38:58,934 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fascia\"\tAccuracy = 0.824 (global 0.612) Î”m = 0.346\tis_issue = False\n",
      "2023-08-11 10:38:58,948 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tract\"\tAccuracy = 0.450 (global 0.612) Î”m = -0.265\tis_issue = True\n",
      "2023-08-11 10:38:58,962 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lymph\"\tAccuracy = 0.769 (global 0.612) Î”m = 0.257\tis_issue = False\n",
      "2023-08-11 10:38:58,976 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colonoscopy\"\tAccuracy = 0.565 (global 0.612) Î”m = -0.076\tis_issue = True\n",
      "2023-08-11 10:38:58,990 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"region\"\tAccuracy = 0.724 (global 0.612) Î”m = 0.184\tis_issue = False\n",
      "2023-08-11 10:38:59,004 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medication\"\tAccuracy = 0.541 (global 0.612) Î”m = -0.117\tis_issue = True\n",
      "2023-08-11 10:38:59,019 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"father\"\tAccuracy = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:59,033 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectus\"\tAccuracy = 0.950 (global 0.612) Î”m = 0.553\tis_issue = False\n",
      "2023-08-11 10:38:59,048 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"function\"\tAccuracy = 0.414 (global 0.612) Î”m = -0.324\tis_issue = True\n",
      "2023-08-11 10:38:59,061 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fever\"\tAccuracy = 0.558 (global 0.612) Î”m = -0.089\tis_issue = True\n",
      "2023-08-11 10:38:59,072 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"procedure\"\tAccuracy = 0.726 (global 0.612) Î”m = 0.187\tis_issue = False\n",
      "2023-08-11 10:38:59,086 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearance\"\tAccuracy = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:59,102 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inserted\"\tAccuracy = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:59,117 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suction\"\tAccuracy = 0.826 (global 0.612) Î”m = 0.350\tis_issue = False\n",
      "2023-08-11 10:38:59,131 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removal\"\tAccuracy = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:38:59,145 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"removed\"\tAccuracy = 0.790 (global 0.612) Î”m = 0.291\tis_issue = False\n",
      "2023-08-11 10:38:59,159 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reveals\"\tAccuracy = 0.652 (global 0.612) Î”m = 0.066\tis_issue = False\n",
      "2023-08-11 10:38:59,172 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clear\"\tAccuracy = 0.515 (global 0.612) Î”m = -0.158\tis_issue = True\n",
      "2023-08-11 10:38:59,186 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"age\"\tAccuracy = 0.639 (global 0.612) Î”m = 0.044\tis_issue = False\n",
      "2023-08-11 10:38:59,199 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissection\"\tAccuracy = 0.762 (global 0.612) Î”m = 0.245\tis_issue = False\n",
      "2023-08-11 10:38:59,214 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"small\"\tAccuracy = 0.770 (global 0.612) Î”m = 0.259\tis_issue = False\n",
      "2023-08-11 10:38:59,228 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"enlarged\"\tAccuracy = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:38:59,239 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"room\"\tAccuracy = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:38:59,252 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"biopsy\"\tAccuracy = 0.625 (global 0.612) Î”m = 0.021\tis_issue = False\n",
      "2023-08-11 10:38:59,268 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"irrigated\"\tAccuracy = 0.854 (global 0.612) Î”m = 0.395\tis_issue = False\n",
      "2023-08-11 10:38:59,283 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"swelling\"\tAccuracy = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:59,299 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"infection\"\tAccuracy = 0.607 (global 0.612) Î”m = -0.009\tis_issue = False\n",
      "2023-08-11 10:38:59,311 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"exam\"\tAccuracy = 0.663 (global 0.612) Î”m = 0.084\tis_issue = False\n",
      "2023-08-11 10:38:59,324 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"urinary\"\tAccuracy = 0.486 (global 0.612) Î”m = -0.206\tis_issue = True\n",
      "2023-08-11 10:38:59,338 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"count\"\tAccuracy = 0.400 (global 0.612) Î”m = -0.346\tis_issue = True\n",
      "2023-08-11 10:38:59,348 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"performed\"\tAccuracy = 0.683 (global 0.612) Î”m = 0.116\tis_issue = False\n",
      "2023-08-11 10:38:59,363 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dorsal\"\tAccuracy = 0.929 (global 0.612) Î”m = 0.518\tis_issue = False\n",
      "2023-08-11 10:38:59,377 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vicryl\"\tAccuracy = 0.791 (global 0.612) Î”m = 0.293\tis_issue = False\n",
      "2023-08-11 10:38:59,392 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"counts\"\tAccuracy = 0.892 (global 0.612) Î”m = 0.458\tis_issue = False\n",
      "2023-08-11 10:38:59,407 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"proximal\"\tAccuracy = 0.889 (global 0.612) Î”m = 0.453\tis_issue = False\n",
      "2023-08-11 10:38:59,422 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extending\"\tAccuracy = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:38:59,436 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"medial\"\tAccuracy = 0.778 (global 0.612) Î”m = 0.271\tis_issue = False\n",
      "2023-08-11 10:38:59,450 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cord\"\tAccuracy = 0.939 (global 0.612) Î”m = 0.535\tis_issue = False\n",
      "2023-08-11 10:38:59,466 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"dissected\"\tAccuracy = 0.881 (global 0.612) Î”m = 0.440\tis_issue = False\n",
      "2023-08-11 10:38:59,502 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recurrent\"\tAccuracy = 0.654 (global 0.612) Î”m = 0.069\tis_issue = False\n",
      "2023-08-11 10:38:59,522 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cavity\"\tAccuracy = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:38:59,538 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knee\"\tAccuracy = 0.591 (global 0.612) Î”m = -0.034\tis_issue = False\n",
      "2023-08-11 10:38:59,553 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adequate\"\tAccuracy = 0.816 (global 0.612) Î”m = 0.333\tis_issue = False\n",
      "2023-08-11 10:38:59,568 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"length\"\tAccuracy = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:38:59,582 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complex\"\tAccuracy = 0.522 (global 0.612) Î”m = -0.147\tis_issue = True\n",
      "2023-08-11 10:38:59,595 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"having\"\tAccuracy = 0.523 (global 0.612) Î”m = -0.145\tis_issue = True\n",
      "2023-08-11 10:38:59,609 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mass\"\tAccuracy = 0.596 (global 0.612) Î”m = -0.026\tis_issue = False\n",
      "2023-08-11 10:38:59,622 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"administered\"\tAccuracy = 0.767 (global 0.612) Î”m = 0.253\tis_issue = False\n",
      "2023-08-11 10:38:59,637 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"blade\"\tAccuracy = 0.900 (global 0.612) Î”m = 0.471\tis_issue = False\n",
      "2023-08-11 10:38:59,651 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denies\"\tAccuracy = 0.459 (global 0.612) Î”m = -0.250\tis_issue = True\n",
      "2023-08-11 10:38:59,665 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthetic\"\tAccuracy = 0.913 (global 0.612) Î”m = 0.492\tis_issue = False\n",
      "2023-08-11 10:38:59,679 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"complications\"\tAccuracy = 0.697 (global 0.612) Î”m = 0.140\tis_issue = False\n",
      "2023-08-11 10:38:59,694 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"erythema\"\tAccuracy = 0.655 (global 0.612) Î”m = 0.071\tis_issue = False\n",
      "2023-08-11 10:38:59,707 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"local\"\tAccuracy = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:38:59,722 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"delivered\"\tAccuracy = 0.920 (global 0.612) Î”m = 0.504\tis_issue = False\n",
      "2023-08-11 10:38:59,736 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"direct\"\tAccuracy = 0.731 (global 0.612) Î”m = 0.194\tis_issue = False\n",
      "2023-08-11 10:38:59,749 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"negative\"\tAccuracy = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:59,763 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nasal\"\tAccuracy = 0.594 (global 0.612) Î”m = -0.030\tis_issue = False\n",
      "2023-08-11 10:38:59,777 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"end\"\tAccuracy = 0.757 (global 0.612) Î”m = 0.237\tis_issue = False\n",
      "2023-08-11 10:38:59,793 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suture\"\tAccuracy = 0.804 (global 0.612) Î”m = 0.315\tis_issue = False\n",
      "2023-08-11 10:38:59,809 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incised\"\tAccuracy = 0.900 (global 0.612) Î”m = 0.471\tis_issue = False\n",
      "2023-08-11 10:38:59,827 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"strength\"\tAccuracy = 0.738 (global 0.612) Î”m = 0.206\tis_issue = False\n",
      "2023-08-11 10:38:59,842 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mri\"\tAccuracy = 0.708 (global 0.612) Î”m = 0.158\tis_issue = False\n",
      "2023-08-11 10:38:59,859 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"areas\"\tAccuracy = 0.607 (global 0.612) Î”m = -0.008\tis_issue = False\n",
      "2023-08-11 10:38:59,877 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"second\"\tAccuracy = 0.679 (global 0.612) Î”m = 0.110\tis_issue = False\n",
      "2023-08-11 10:38:59,890 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"examination\"\tAccuracy = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:38:59,904 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spinal\"\tAccuracy = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:38:59,917 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"admission\"\tAccuracy = 0.375 (global 0.612) Î”m = -0.387\tis_issue = True\n",
      "2023-08-11 10:38:59,932 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"open\"\tAccuracy = 0.769 (global 0.612) Î”m = 0.257\tis_issue = False\n",
      "2023-08-11 10:38:59,946 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mouth\"\tAccuracy = 0.700 (global 0.612) Î”m = 0.144\tis_issue = False\n",
      "2023-08-11 10:38:59,960 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pulmonary\"\tAccuracy = 0.533 (global 0.612) Î”m = -0.128\tis_issue = True\n",
      "2023-08-11 10:38:59,973 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"flap\"\tAccuracy = 0.727 (global 0.612) Î”m = 0.189\tis_issue = False\n",
      "2023-08-11 10:38:59,986 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"treatment\"\tAccuracy = 0.518 (global 0.612) Î”m = -0.154\tis_issue = True\n",
      "2023-08-11 10:39:00,000 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"resection\"\tAccuracy = 0.700 (global 0.612) Î”m = 0.144\tis_issue = False\n",
      "2023-08-11 10:39:00,014 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"contrast\"\tAccuracy = 0.708 (global 0.612) Î”m = 0.158\tis_issue = False\n",
      "2023-08-11 10:39:00,030 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"spine\"\tAccuracy = 0.703 (global 0.612) Î”m = 0.148\tis_issue = False\n",
      "2023-08-11 10:39:00,043 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cm\"\tAccuracy = 0.716 (global 0.612) Î”m = 0.171\tis_issue = False\n",
      "2023-08-11 10:39:00,057 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervical\"\tAccuracy = 0.708 (global 0.612) Î”m = 0.158\tis_issue = False\n",
      "2023-08-11 10:39:00,070 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abc\"\tAccuracy = 0.417 (global 0.612) Î”m = -0.319\tis_issue = True\n",
      "2023-08-11 10:39:00,084 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligated\"\tAccuracy = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:39:00,100 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sinus\"\tAccuracy = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:39:00,114 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oxygen\"\tAccuracy = 0.591 (global 0.612) Î”m = -0.034\tis_issue = False\n",
      "2023-08-11 10:39:00,129 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distal\"\tAccuracy = 0.795 (global 0.612) Î”m = 0.299\tis_issue = False\n",
      "2023-08-11 10:39:00,143 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sterile\"\tAccuracy = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:39:00,158 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinical\"\tAccuracy = 0.565 (global 0.612) Î”m = -0.076\tis_issue = True\n",
      "2023-08-11 10:39:00,171 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mm\"\tAccuracy = 0.827 (global 0.612) Î”m = 0.351\tis_issue = False\n",
      "2023-08-11 10:39:00,185 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cardiac\"\tAccuracy = 0.529 (global 0.612) Î”m = -0.135\tis_issue = True\n",
      "2023-08-11 10:39:00,208 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visit\"\tAccuracy = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:39:00,236 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appearing\"\tAccuracy = 0.680 (global 0.612) Î”m = 0.111\tis_issue = False\n",
      "2023-08-11 10:39:00,255 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clamped\"\tAccuracy = 0.960 (global 0.612) Î”m = 0.569\tis_issue = False\n",
      "2023-08-11 10:39:00,270 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectum\"\tAccuracy = 0.650 (global 0.612) Î”m = 0.062\tis_issue = False\n",
      "2023-08-11 10:39:00,283 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"draped\"\tAccuracy = 0.773 (global 0.612) Î”m = 0.263\tis_issue = False\n",
      "2023-08-11 10:39:00,300 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stomach\"\tAccuracy = 0.850 (global 0.612) Î”m = 0.389\tis_issue = False\n",
      "2023-08-11 10:39:00,314 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wound\"\tAccuracy = 0.638 (global 0.612) Î”m = 0.043\tis_issue = False\n",
      "2023-08-11 10:39:00,326 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"family\"\tAccuracy = 0.573 (global 0.612) Î”m = -0.063\tis_issue = True\n",
      "2023-08-11 10:39:00,340 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nerve\"\tAccuracy = 0.850 (global 0.612) Î”m = 0.389\tis_issue = False\n",
      "2023-08-11 10:39:00,354 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cut\"\tAccuracy = 0.917 (global 0.612) Î”m = 0.498\tis_issue = False\n",
      "2023-08-11 10:39:00,367 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesions\"\tAccuracy = 0.681 (global 0.612) Î”m = 0.113\tis_issue = False\n",
      "2023-08-11 10:39:00,382 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cervix\"\tAccuracy = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:39:00,395 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"technique\"\tAccuracy = 0.783 (global 0.612) Î”m = 0.279\tis_issue = False\n",
      "2023-08-11 10:39:00,408 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"point\"\tAccuracy = 0.615 (global 0.612) Î”m = 0.006\tis_issue = False\n",
      "2023-08-11 10:39:00,422 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"leg\"\tAccuracy = 0.567 (global 0.612) Î”m = -0.074\tis_issue = True\n",
      "2023-08-11 10:39:00,437 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"s1\"\tAccuracy = 0.556 (global 0.612) Î”m = -0.092\tis_issue = True\n",
      "2023-08-11 10:39:00,451 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"xyz\"\tAccuracy = 0.318 (global 0.612) Î”m = -0.480\tis_issue = True\n",
      "2023-08-11 10:39:00,465 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"office\"\tAccuracy = 0.476 (global 0.612) Î”m = -0.222\tis_issue = True\n",
      "2023-08-11 10:39:00,479 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gross\"\tAccuracy = 0.500 (global 0.612) Î”m = -0.183\tis_issue = True\n",
      "2023-08-11 10:39:00,493 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"check\"\tAccuracy = 0.560 (global 0.612) Î”m = -0.085\tis_issue = True\n",
      "2023-08-11 10:39:00,508 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"created\"\tAccuracy = 0.950 (global 0.612) Î”m = 0.553\tis_issue = False\n",
      "2023-08-11 10:39:00,522 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hours\"\tAccuracy = 0.558 (global 0.612) Î”m = -0.089\tis_issue = True\n",
      "2023-08-11 10:39:00,536 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"body\"\tAccuracy = 0.816 (global 0.612) Î”m = 0.333\tis_issue = False\n",
      "2023-08-11 10:39:00,551 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"entered\"\tAccuracy = 0.875 (global 0.612) Î”m = 0.430\tis_issue = False\n",
      "2023-08-11 10:39:00,565 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rectal\"\tAccuracy = 0.643 (global 0.612) Î”m = 0.051\tis_issue = False\n",
      "2023-08-11 10:39:00,580 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"adenopathy\"\tAccuracy = 0.778 (global 0.612) Î”m = 0.271\tis_issue = False\n",
      "2023-08-11 10:39:00,594 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterine\"\tAccuracy = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:39:00,608 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"high\"\tAccuracy = 0.488 (global 0.612) Î”m = -0.203\tis_issue = True\n",
      "2023-08-11 10:39:00,622 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"correct\"\tAccuracy = 0.881 (global 0.612) Î”m = 0.440\tis_issue = False\n",
      "2023-08-11 10:39:00,636 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pupils\"\tAccuracy = 0.706 (global 0.612) Î”m = 0.154\tis_issue = False\n",
      "2023-08-11 10:39:00,650 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"upper\"\tAccuracy = 0.718 (global 0.612) Î”m = 0.173\tis_issue = False\n",
      "2023-08-11 10:39:00,664 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visual\"\tAccuracy = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:39:00,679 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"revealed\"\tAccuracy = 0.742 (global 0.612) Î”m = 0.213\tis_issue = False\n",
      "2023-08-11 10:39:00,694 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tumor\"\tAccuracy = 0.583 (global 0.612) Î”m = -0.047\tis_issue = False\n",
      "2023-08-11 10:39:00,708 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"distress\"\tAccuracy = 0.486 (global 0.612) Î”m = -0.206\tis_issue = True\n",
      "2023-08-11 10:39:00,722 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"unremarkable\"\tAccuracy = 0.632 (global 0.612) Î”m = 0.033\tis_issue = False\n",
      "2023-08-11 10:39:00,737 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"greater\"\tAccuracy = 0.435 (global 0.612) Î”m = -0.289\tis_issue = True\n",
      "2023-08-11 10:39:00,753 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"internal\"\tAccuracy = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:39:00,768 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"recovery\"\tAccuracy = 0.843 (global 0.612) Î”m = 0.378\tis_issue = False\n",
      "2023-08-11 10:39:00,783 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scope\"\tAccuracy = 0.864 (global 0.612) Î”m = 0.411\tis_issue = False\n",
      "2023-08-11 10:39:00,797 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"skin\"\tAccuracy = 0.723 (global 0.612) Î”m = 0.182\tis_issue = False\n",
      "2023-08-11 10:39:00,807 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"postoperative\"\tAccuracy = 0.729 (global 0.612) Î”m = 0.191\tis_issue = False\n",
      "2023-08-11 10:39:00,820 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"evidence\"\tAccuracy = 0.691 (global 0.612) Î”m = 0.130\tis_issue = False\n",
      "2023-08-11 10:39:00,834 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"knife\"\tAccuracy = 0.952 (global 0.612) Î”m = 0.557\tis_issue = False\n",
      "2023-08-11 10:39:00,850 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sac\"\tAccuracy = 0.913 (global 0.612) Î”m = 0.492\tis_issue = False\n",
      "2023-08-11 10:39:00,866 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sutures\"\tAccuracy = 0.636 (global 0.612) Î”m = 0.040\tis_issue = False\n",
      "2023-08-11 10:39:00,879 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"uterus\"\tAccuracy = 0.647 (global 0.612) Î”m = 0.058\tis_issue = False\n",
      "2023-08-11 10:39:00,894 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"inferior\"\tAccuracy = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:39:00,913 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"similar\"\tAccuracy = 0.724 (global 0.612) Î”m = 0.184\tis_issue = False\n",
      "2023-08-11 10:39:00,930 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bleeding\"\tAccuracy = 0.727 (global 0.612) Î”m = 0.189\tis_issue = False\n",
      "2023-08-11 10:39:00,954 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"large\"\tAccuracy = 0.647 (global 0.612) Î”m = 0.058\tis_issue = False\n",
      "2023-08-11 10:39:00,969 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"normal\"\tAccuracy = 0.641 (global 0.612) Î”m = 0.048\tis_issue = False\n",
      "2023-08-11 10:39:00,986 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"posterior\"\tAccuracy = 0.768 (global 0.612) Î”m = 0.255\tis_issue = False\n",
      "2023-08-11 10:39:01,000 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"chromic\"\tAccuracy = 0.952 (global 0.612) Î”m = 0.557\tis_issue = False\n",
      "2023-08-11 10:39:01,015 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"round\"\tAccuracy = 0.774 (global 0.612) Î”m = 0.265\tis_issue = False\n",
      "2023-08-11 10:39:01,029 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ct\"\tAccuracy = 0.600 (global 0.612) Î”m = -0.019\tis_issue = False\n",
      "2023-08-11 10:39:01,042 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"incision\"\tAccuracy = 0.747 (global 0.612) Î”m = 0.221\tis_issue = False\n",
      "2023-08-11 10:39:01,056 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lesion\"\tAccuracy = 0.692 (global 0.612) Î”m = 0.131\tis_issue = False\n",
      "2023-08-11 10:39:01,069 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fashion\"\tAccuracy = 0.807 (global 0.612) Î”m = 0.319\tis_issue = False\n",
      "2023-08-11 10:39:01,083 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vaginal\"\tAccuracy = 0.586 (global 0.612) Î”m = -0.042\tis_issue = False\n",
      "2023-08-11 10:39:01,101 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subcuticular\"\tAccuracy = 0.792 (global 0.612) Î”m = 0.294\tis_issue = False\n",
      "2023-08-11 10:39:01,116 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stable\"\tAccuracy = 0.627 (global 0.612) Î”m = 0.025\tis_issue = False\n",
      "2023-08-11 10:39:01,131 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mucosa\"\tAccuracy = 0.697 (global 0.612) Î”m = 0.139\tis_issue = False\n",
      "2023-08-11 10:39:01,146 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"shoulder\"\tAccuracy = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:39:01,160 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"use\"\tAccuracy = 0.681 (global 0.612) Î”m = 0.112\tis_issue = False\n",
      "2023-08-11 10:39:01,173 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"nose\"\tAccuracy = 0.794 (global 0.612) Î”m = 0.298\tis_issue = False\n",
      "2023-08-11 10:39:01,187 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"rule\"\tAccuracy = 0.450 (global 0.612) Î”m = -0.265\tis_issue = True\n",
      "2023-08-11 10:39:01,201 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"denied\"\tAccuracy = 0.773 (global 0.612) Î”m = 0.263\tis_issue = False\n",
      "2023-08-11 10:39:01,211 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"left\"\tAccuracy = 0.693 (global 0.612) Î”m = 0.132\tis_issue = False\n",
      "2023-08-11 10:39:01,223 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prepped\"\tAccuracy = 0.767 (global 0.612) Î”m = 0.253\tis_issue = False\n",
      "2023-08-11 10:39:01,236 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"diarrhea\"\tAccuracy = 0.462 (global 0.612) Î”m = -0.246\tis_issue = True\n",
      "2023-08-11 10:39:01,251 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"improved\"\tAccuracy = 0.419 (global 0.612) Î”m = -0.315\tis_issue = True\n",
      "2023-08-11 10:39:01,264 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"loss\"\tAccuracy = 0.781 (global 0.612) Î”m = 0.276\tis_issue = False\n",
      "2023-08-11 10:39:01,278 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gentleman\"\tAccuracy = 0.696 (global 0.612) Î”m = 0.137\tis_issue = False\n",
      "2023-08-11 10:39:01,295 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sensory\"\tAccuracy = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:39:01,310 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"laterally\"\tAccuracy = 0.846 (global 0.612) Î”m = 0.383\tis_issue = False\n",
      "2023-08-11 10:39:01,325 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"needle\"\tAccuracy = 0.821 (global 0.612) Î”m = 0.343\tis_issue = False\n",
      "2023-08-11 10:39:01,338 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"findings\"\tAccuracy = 0.691 (global 0.612) Î”m = 0.129\tis_issue = False\n",
      "2023-08-11 10:39:01,356 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tolerated\"\tAccuracy = 0.698 (global 0.612) Î”m = 0.140\tis_issue = False\n",
      "2023-08-11 10:39:01,370 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"neck\"\tAccuracy = 0.553 (global 0.612) Î”m = -0.096\tis_issue = True\n",
      "2023-08-11 10:39:01,385 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headache\"\tAccuracy = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:39:01,399 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"condition\"\tAccuracy = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:39:01,412 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"wall\"\tAccuracy = 0.763 (global 0.612) Î”m = 0.247\tis_issue = False\n",
      "2023-08-11 10:39:01,425 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"abdominal\"\tAccuracy = 0.615 (global 0.612) Î”m = 0.006\tis_issue = False\n",
      "2023-08-11 10:39:01,436 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"placed\"\tAccuracy = 0.780 (global 0.612) Î”m = 0.274\tis_issue = False\n",
      "2023-08-11 10:39:01,451 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"supine\"\tAccuracy = 0.764 (global 0.612) Î”m = 0.248\tis_issue = False\n",
      "2023-08-11 10:39:01,466 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"prostate\"\tAccuracy = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:39:01,480 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tube\"\tAccuracy = 0.773 (global 0.612) Î”m = 0.263\tis_issue = False\n",
      "2023-08-11 10:39:01,495 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"gait\"\tAccuracy = 0.742 (global 0.612) Î”m = 0.213\tis_issue = False\n",
      "2023-08-11 10:39:01,509 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pelvis\"\tAccuracy = 0.733 (global 0.612) Î”m = 0.199\tis_issue = False\n",
      "2023-08-11 10:39:01,525 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suctioned\"\tAccuracy = 0.952 (global 0.612) Î”m = 0.557\tis_issue = False\n",
      "2023-08-11 10:39:01,539 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"passed\"\tAccuracy = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:39:01,555 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"subjective\"\tAccuracy = 0.367 (global 0.612) Î”m = -0.401\tis_issue = True\n",
      "2023-08-11 10:39:01,568 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"able\"\tAccuracy = 0.633 (global 0.612) Î”m = 0.034\tis_issue = False\n",
      "2023-08-11 10:39:01,583 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lumbar\"\tAccuracy = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:39:01,597 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"running\"\tAccuracy = 0.825 (global 0.612) Î”m = 0.348\tis_issue = False\n",
      "2023-08-11 10:39:01,602 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"patient\"\tAccuracy = 0.597 (global 0.612) Î”m = -0.023\tis_issue = False\n",
      "2023-08-11 10:39:01,618 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pregnancy\"\tAccuracy = 0.783 (global 0.612) Î”m = 0.279\tis_issue = False\n",
      "2023-08-11 10:39:01,631 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brain\"\tAccuracy = 0.645 (global 0.612) Î”m = 0.054\tis_issue = False\n",
      "2023-08-11 10:39:01,646 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drainage\"\tAccuracy = 0.630 (global 0.612) Î”m = 0.029\tis_issue = False\n",
      "2023-08-11 10:39:01,661 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"indications\"\tAccuracy = 0.889 (global 0.612) Î”m = 0.453\tis_issue = False\n",
      "2023-08-11 10:39:01,674 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"taken\"\tAccuracy = 0.713 (global 0.612) Î”m = 0.165\tis_issue = False\n",
      "2023-08-11 10:39:01,688 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"masses\"\tAccuracy = 0.605 (global 0.612) Î”m = -0.011\tis_issue = False\n",
      "2023-08-11 10:39:01,703 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"suite\"\tAccuracy = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:39:01,720 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"day\"\tAccuracy = 0.481 (global 0.612) Î”m = -0.214\tis_issue = True\n",
      "2023-08-11 10:39:01,746 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sponge\"\tAccuracy = 0.886 (global 0.612) Î”m = 0.448\tis_issue = False\n",
      "2023-08-11 10:39:01,769 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reports\"\tAccuracy = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:39:01,791 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"vision\"\tAccuracy = 0.778 (global 0.612) Î”m = 0.271\tis_issue = False\n",
      "2023-08-11 10:39:01,804 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"foley\"\tAccuracy = 0.730 (global 0.612) Î”m = 0.193\tis_issue = False\n",
      "2023-08-11 10:39:01,819 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"presentation\"\tAccuracy = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:39:01,831 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"20\"\tAccuracy = 0.598 (global 0.612) Î”m = -0.023\tis_issue = False\n",
      "2023-08-11 10:39:01,844 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"daily\"\tAccuracy = 0.385 (global 0.612) Î”m = -0.371\tis_issue = True\n",
      "2023-08-11 10:39:01,857 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"motor\"\tAccuracy = 0.795 (global 0.612) Î”m = 0.300\tis_issue = False\n",
      "2023-08-11 10:39:01,872 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"epinephrine\"\tAccuracy = 0.650 (global 0.612) Î”m = 0.062\tis_issue = False\n",
      "2023-08-11 10:39:01,888 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"speech\"\tAccuracy = 0.704 (global 0.612) Î”m = 0.150\tis_issue = False\n",
      "2023-08-11 10:39:01,902 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"superior\"\tAccuracy = 0.742 (global 0.612) Î”m = 0.213\tis_issue = False\n",
      "2023-08-11 10:39:01,917 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"section\"\tAccuracy = 0.692 (global 0.612) Î”m = 0.131\tis_issue = False\n",
      "2023-08-11 10:39:01,930 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mild\"\tAccuracy = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:39:01,944 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"clinic\"\tAccuracy = 0.542 (global 0.612) Î”m = -0.115\tis_issue = True\n",
      "2023-08-11 10:39:01,958 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reviewed\"\tAccuracy = 0.529 (global 0.612) Î”m = -0.135\tis_issue = True\n",
      "2023-08-11 10:39:01,971 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"appeared\"\tAccuracy = 0.765 (global 0.612) Î”m = 0.250\tis_issue = False\n",
      "2023-08-11 10:39:01,986 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hx\"\tAccuracy = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:39:01,999 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consent\"\tAccuracy = 0.833 (global 0.612) Î”m = 0.362\tis_issue = False\n",
      "2023-08-11 10:39:02,012 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"found\"\tAccuracy = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:39:02,025 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bovie\"\tAccuracy = 0.840 (global 0.612) Î”m = 0.373\tis_issue = False\n",
      "2023-08-11 10:39:02,038 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"weakness\"\tAccuracy = 0.732 (global 0.612) Î”m = 0.196\tis_issue = False\n",
      "2023-08-11 10:39:02,051 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"obtained\"\tAccuracy = 0.737 (global 0.612) Î”m = 0.205\tis_issue = False\n",
      "2023-08-11 10:39:02,066 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"informed\"\tAccuracy = 0.854 (global 0.612) Î”m = 0.395\tis_issue = False\n",
      "2023-08-11 10:39:02,082 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"initially\"\tAccuracy = 0.517 (global 0.612) Î”m = -0.155\tis_issue = True\n",
      "2023-08-11 10:39:02,097 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"status\"\tAccuracy = 0.515 (global 0.612) Î”m = -0.158\tis_issue = True\n",
      "2023-08-11 10:39:02,110 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"care\"\tAccuracy = 0.537 (global 0.612) Î”m = -0.122\tis_issue = True\n",
      "2023-08-11 10:39:02,125 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stool\"\tAccuracy = 0.516 (global 0.612) Î”m = -0.156\tis_issue = True\n",
      "2023-08-11 10:39:02,138 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"following\"\tAccuracy = 0.609 (global 0.612) Î”m = -0.004\tis_issue = False\n",
      "2023-08-11 10:39:02,152 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"reflexes\"\tAccuracy = 0.775 (global 0.612) Î”m = 0.267\tis_issue = False\n",
      "2023-08-11 10:39:02,167 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scalpel\"\tAccuracy = 0.957 (global 0.612) Î”m = 0.563\tis_issue = False\n",
      "2023-08-11 10:39:02,181 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tenderness\"\tAccuracy = 0.543 (global 0.612) Î”m = -0.112\tis_issue = True\n",
      "2023-08-11 10:39:02,195 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"compression\"\tAccuracy = 0.636 (global 0.612) Î”m = 0.040\tis_issue = False\n",
      "2023-08-11 10:39:02,208 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"headaches\"\tAccuracy = 0.840 (global 0.612) Î”m = 0.373\tis_issue = False\n",
      "2023-08-11 10:39:02,223 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"closed\"\tAccuracy = 0.806 (global 0.612) Î”m = 0.317\tis_issue = False\n",
      "2023-08-11 10:39:02,236 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"12\"\tAccuracy = 0.541 (global 0.612) Î”m = -0.117\tis_issue = True\n",
      "2023-08-11 10:39:02,250 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"place\"\tAccuracy = 0.828 (global 0.612) Î”m = 0.353\tis_issue = False\n",
      "2023-08-11 10:39:02,264 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"visualized\"\tAccuracy = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:39:02,280 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"impression\"\tAccuracy = 0.667 (global 0.612) Î”m = 0.090\tis_issue = False\n",
      "2023-08-11 10:39:02,294 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"consistent\"\tAccuracy = 0.828 (global 0.612) Î”m = 0.353\tis_issue = False\n",
      "2023-08-11 10:39:02,308 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluids\"\tAccuracy = 0.786 (global 0.612) Î”m = 0.284\tis_issue = False\n",
      "2023-08-11 10:39:02,322 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"change\"\tAccuracy = 0.725 (global 0.612) Î”m = 0.185\tis_issue = False\n",
      "2023-08-11 10:39:02,332 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"preoperative\"\tAccuracy = 0.761 (global 0.612) Î”m = 0.243\tis_issue = False\n",
      "2023-08-11 10:39:02,346 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"changes\"\tAccuracy = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:39:02,358 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bilaterally\"\tAccuracy = 0.646 (global 0.612) Î”m = 0.056\tis_issue = False\n",
      "2023-08-11 10:39:02,372 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"transverse\"\tAccuracy = 0.812 (global 0.612) Î”m = 0.328\tis_issue = False\n",
      "2023-08-11 10:39:02,386 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"portion\"\tAccuracy = 0.964 (global 0.612) Î”m = 0.576\tis_issue = False\n",
      "2023-08-11 10:39:02,397 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"noted\"\tAccuracy = 0.651 (global 0.612) Î”m = 0.064\tis_issue = False\n",
      "2023-08-11 10:39:02,411 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"days\"\tAccuracy = 0.450 (global 0.612) Î”m = -0.265\tis_issue = True\n",
      "2023-08-11 10:39:02,426 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carotid\"\tAccuracy = 0.818 (global 0.612) Î”m = 0.337\tis_issue = False\n",
      "2023-08-11 10:39:02,440 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"arm\"\tAccuracy = 0.591 (global 0.612) Î”m = -0.034\tis_issue = False\n",
      "2023-08-11 10:39:02,455 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"continue\"\tAccuracy = 0.442 (global 0.612) Î”m = -0.277\tis_issue = True\n",
      "2023-08-11 10:39:02,473 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"colon\"\tAccuracy = 0.690 (global 0.612) Î”m = 0.127\tis_issue = False\n",
      "2023-08-11 10:39:02,495 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"iv\"\tAccuracy = 0.758 (global 0.612) Î”m = 0.239\tis_issue = False\n",
      "2023-08-11 10:39:02,513 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ml\"\tAccuracy = 0.717 (global 0.612) Î”m = 0.172\tis_issue = False\n",
      "2023-08-11 10:39:02,532 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lung\"\tAccuracy = 0.400 (global 0.612) Î”m = -0.346\tis_issue = True\n",
      "2023-08-11 10:39:02,548 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspect\"\tAccuracy = 0.787 (global 0.612) Î”m = 0.287\tis_issue = False\n",
      "2023-08-11 10:39:02,563 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ear\"\tAccuracy = 0.619 (global 0.612) Î”m = 0.012\tis_issue = False\n",
      "2023-08-11 10:39:02,579 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"stitch\"\tAccuracy = 0.909 (global 0.612) Î”m = 0.486\tis_issue = False\n",
      "2023-08-11 10:39:02,594 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"carefully\"\tAccuracy = 0.826 (global 0.612) Î”m = 0.350\tis_issue = False\n",
      "2023-08-11 10:39:02,609 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"grasped\"\tAccuracy = 0.840 (global 0.612) Î”m = 0.373\tis_issue = False\n",
      "2023-08-11 10:39:02,624 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"post\"\tAccuracy = 0.552 (global 0.612) Î”m = -0.098\tis_issue = True\n",
      "2023-08-11 10:39:02,640 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"specimens\"\tAccuracy = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:39:02,654 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"light\"\tAccuracy = 0.674 (global 0.612) Î”m = 0.102\tis_issue = False\n",
      "2023-08-11 10:39:02,670 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"return\"\tAccuracy = 0.514 (global 0.612) Î”m = -0.159\tis_issue = True\n",
      "2023-08-11 10:39:02,682 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"symptoms\"\tAccuracy = 0.551 (global 0.612) Î”m = -0.099\tis_issue = True\n",
      "2023-08-11 10:39:02,696 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"extremity\"\tAccuracy = 0.619 (global 0.612) Î”m = 0.012\tis_issue = False\n",
      "2023-08-11 10:39:02,708 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"position\"\tAccuracy = 0.794 (global 0.612) Î”m = 0.298\tis_issue = False\n",
      "2023-08-11 10:39:02,720 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscle\"\tAccuracy = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:39:02,735 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"easily\"\tAccuracy = 0.800 (global 0.612) Î”m = 0.307\tis_issue = False\n",
      "2023-08-11 10:39:02,747 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"mg\"\tAccuracy = 0.473 (global 0.612) Î”m = -0.227\tis_issue = True\n",
      "2023-08-11 10:39:02,761 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"muscles\"\tAccuracy = 0.815 (global 0.612) Î”m = 0.332\tis_issue = False\n",
      "2023-08-11 10:39:02,774 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"months\"\tAccuracy = 0.562 (global 0.612) Î”m = -0.081\tis_issue = True\n",
      "2023-08-11 10:39:02,787 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"lateral\"\tAccuracy = 0.797 (global 0.612) Î”m = 0.303\tis_issue = False\n",
      "2023-08-11 10:39:02,800 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"bladder\"\tAccuracy = 0.650 (global 0.612) Î”m = 0.062\tis_issue = False\n",
      "2023-08-11 10:39:02,811 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anesthesia\"\tAccuracy = 0.764 (global 0.612) Î”m = 0.249\tis_issue = False\n",
      "2023-08-11 10:39:02,823 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"discharge\"\tAccuracy = 0.424 (global 0.612) Î”m = -0.308\tis_issue = True\n",
      "2023-08-11 10:39:02,839 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"aspirin\"\tAccuracy = 0.435 (global 0.612) Î”m = -0.289\tis_issue = True\n",
      "2023-08-11 10:39:02,852 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"difficulty\"\tAccuracy = 0.734 (global 0.612) Î”m = 0.200\tis_issue = False\n",
      "2023-08-11 10:39:02,867 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"oropharynx\"\tAccuracy = 0.686 (global 0.612) Î”m = 0.121\tis_issue = False\n",
      "2023-08-11 10:39:02,876 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"right\"\tAccuracy = 0.659 (global 0.612) Î”m = 0.077\tis_issue = False\n",
      "2023-08-11 10:39:02,889 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"description\"\tAccuracy = 0.854 (global 0.612) Î”m = 0.395\tis_issue = False\n",
      "2023-08-11 10:39:02,902 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"midline\"\tAccuracy = 0.836 (global 0.612) Î”m = 0.367\tis_issue = False\n",
      "2023-08-11 10:39:02,916 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"advanced\"\tAccuracy = 0.759 (global 0.612) Î”m = 0.240\tis_issue = False\n",
      "2023-08-11 10:39:02,931 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"followup\"\tAccuracy = 0.396 (global 0.612) Î”m = -0.353\tis_issue = True\n",
      "2023-08-11 10:39:02,944 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"08\"\tAccuracy = 0.462 (global 0.612) Î”m = -0.246\tis_issue = True\n",
      "2023-08-11 10:39:02,958 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"ligament\"\tAccuracy = 0.950 (global 0.612) Î”m = 0.553\tis_issue = False\n",
      "2023-08-11 10:39:02,972 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"tone\"\tAccuracy = 0.759 (global 0.612) Î”m = 0.240\tis_issue = False\n",
      "2023-08-11 10:39:02,989 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"saline\"\tAccuracy = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:39:03,003 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"follow\"\tAccuracy = 0.550 (global 0.612) Î”m = -0.101\tis_issue = True\n",
      "2023-08-11 10:39:03,016 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cancer\"\tAccuracy = 0.622 (global 0.612) Î”m = 0.016\tis_issue = False\n",
      "2023-08-11 10:39:03,030 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"drains\"\tAccuracy = 0.857 (global 0.612) Î”m = 0.401\tis_issue = False\n",
      "2023-08-11 10:39:03,043 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"renal\"\tAccuracy = 0.462 (global 0.612) Î”m = -0.246\tis_issue = True\n",
      "2023-08-11 10:39:03,058 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"anterior\"\tAccuracy = 0.756 (global 0.612) Î”m = 0.236\tis_issue = False\n",
      "2023-08-11 10:39:03,073 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"units\"\tAccuracy = 0.640 (global 0.612) Î”m = 0.046\tis_issue = False\n",
      "2023-08-11 10:39:03,086 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"home\"\tAccuracy = 0.519 (global 0.612) Î”m = -0.153\tis_issue = True\n",
      "2023-08-11 10:39:03,100 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"hemostasis\"\tAccuracy = 0.787 (global 0.612) Î”m = 0.286\tis_issue = False\n",
      "2023-08-11 10:39:03,116 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"sleep\"\tAccuracy = 0.714 (global 0.612) Î”m = 0.167\tis_issue = False\n",
      "2023-08-11 10:39:03,129 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"artery\"\tAccuracy = 0.688 (global 0.612) Î”m = 0.124\tis_issue = False\n",
      "2023-08-11 10:39:03,141 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"brought\"\tAccuracy = 0.747 (global 0.612) Î”m = 0.221\tis_issue = False\n",
      "2023-08-11 10:39:03,156 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"underlying\"\tAccuracy = 0.810 (global 0.612) Î”m = 0.323\tis_issue = False\n",
      "2023-08-11 10:39:03,170 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"membranes\"\tAccuracy = 0.750 (global 0.612) Î”m = 0.226\tis_issue = False\n",
      "2023-08-11 10:39:03,183 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"cc\"\tAccuracy = 0.787 (global 0.612) Î”m = 0.287\tis_issue = False\n",
      "2023-08-11 10:39:03,193 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"pain\"\tAccuracy = 0.571 (global 0.612) Î”m = -0.066\tis_issue = True\n",
      "2023-08-11 10:39:03,206 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"fluid\"\tAccuracy = 0.759 (global 0.612) Î”m = 0.241\tis_issue = False\n",
      "2023-08-11 10:39:03,214 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"history\"\tAccuracy = 0.564 (global 0.612) Î”m = -0.078\tis_issue = True\n",
      "2023-08-11 10:39:03,229 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"scissors\"\tAccuracy = 0.852 (global 0.612) Î”m = 0.392\tis_issue = False\n",
      "2023-08-11 10:39:03,246 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: Testing slice `transcription` contains \"identified\"\tAccuracy = 0.785 (global 0.612) Î”m = 0.282\tis_issue = False\n",
      "2023-08-11 10:39:03,259 pid:6545 MainThread giskard.scanner.logger INFO     PerformanceBiasDetector: 70 issues found (took 0:00:21.492897)\n",
      " 15 issues detected. (Took 0:00:52.233910)\n",
      "Running detector TextPerturbationDetectorâ€¦2023-08-11 10:39:03,289 pid:6545 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Running with transformations=['Transform to uppercase', 'Transform to lowercase', 'Transform to title case', 'Add typos', 'Punctuation Removal'] threshold=None output_sensitivity=None num_samples=None\n",
      "2023-08-11 10:39:04,440 pid:6545 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Testing `transcription` for perturbation `Transform to uppercase`\tFail rate: 0.000\n",
      "2023-08-11 10:39:05,530 pid:6545 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Testing `transcription` for perturbation `Transform to lowercase`\tFail rate: 0.000\n",
      "2023-08-11 10:39:06,618 pid:6545 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Testing `transcription` for perturbation `Transform to title case`\tFail rate: 0.000\n",
      "2023-08-11 10:39:07,774 pid:6545 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Testing `transcription` for perturbation `Add typos`\tFail rate: 0.049\n",
      "2023-08-11 10:39:14,254 pid:6545 MainThread giskard.scanner.logger INFO     TextPerturbationDetector: Testing `transcription` for perturbation `Punctuation Removal`\tFail rate: 0.000\n",
      " 0 issues detected. (Took 0:00:10.966312)\n",
      "Running detector EthicalBiasDetectorâ€¦2023-08-11 10:39:14,255 pid:6545 MainThread giskard.scanner.logger INFO     EthicalBiasDetector: Running with transformations=['Switch Gender', 'Switch Religion', 'Switch countries from high- to low-income and vice versa'] threshold=None output_sensitivity=None num_samples=None\n",
      "2023-08-11 10:39:18,002 pid:6545 MainThread giskard.scanner.logger INFO     EthicalBiasDetector: Testing `transcription` for perturbation `Switch Gender`\tFail rate: 0.000\n",
      "2023-08-11 10:39:20,936 pid:6545 MainThread giskard.scanner.logger INFO     EthicalBiasDetector: Testing `transcription` for perturbation `Switch Religion`\tFail rate: 0.000\n",
      "2023-08-11 10:39:36,344 pid:6545 MainThread giskard.scanner.logger INFO     EthicalBiasDetector: Testing `transcription` for perturbation `Switch countries from high- to low-income and vice versa`\tFail rate: 0.000\n",
      " 0 issues detected. (Took 0:00:22.089062)\n",
      "Running detector DataLeakageDetectorâ€¦2023-08-11 10:39:36,345 pid:6545 MainThread giskard.scanner.logger INFO     DataLeakageDetector: Running\n",
      " 0 issues detected. (Took 0:00:00.687579)\n",
      "Running detector StochasticityDetectorâ€¦2023-08-11 10:39:37,033 pid:6545 MainThread giskard.scanner.logger INFO     StochasticityDetector: Running\n",
      " 0 issues detected. (Took 0:00:00.649829)\n",
      "Running detector OverconfidenceDetectorâ€¦2023-08-11 10:39:37,683 pid:6545 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Running\n",
      "2023-08-11 10:39:37,684 pid:6545 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Calculating loss\n",
      "2023-08-11 10:39:37,691 pid:6545 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Loss calculated (took 0:00:00.007141)\n",
      "2023-08-11 10:39:37,691 pid:6545 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Finding data slices\n",
      "2023-08-11 10:39:49,901 pid:6545 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: 103 slices found (took 0:00:12.208729)\n",
      "2023-08-11 10:39:49,902 pid:6545 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Analyzing issues\n",
      "2023-08-11 10:39:49,911 pid:6545 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: Using overconfidence threshold = 0.2468526289804987\n",
      "2023-08-11 10:40:14,038 pid:6545 MainThread giskard.scanner.logger INFO     OverconfidenceDetector: 30 issues found (took 0:00:24.135935)\n",
      " 15 issues detected. (Took 0:00:36.356002)\n",
      "Running detector UnderconfidenceDetectorâ€¦2023-08-11 10:40:14,041 pid:6545 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: Running\n",
      "2023-08-11 10:40:14,042 pid:6545 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: Calculating loss\n",
      "2023-08-11 10:40:14,056 pid:6545 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: Loss calculated (took 0:00:00.013910)\n",
      "2023-08-11 10:40:14,057 pid:6545 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: Finding data slices\n",
      "2023-08-11 10:40:46,427 pid:6545 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: 286 slices found (took 0:00:32.369697)\n",
      "2023-08-11 10:40:46,428 pid:6545 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: Analyzing issues\n",
      "2023-08-11 10:41:08,066 pid:6545 MainThread giskard.scanner.logger INFO     UnderconfidenceDetector: 0 issues found (took 0:00:21.638200)\n",
      " 0 issues detected. (Took 0:00:54.025821)\n",
      "Running detector SpuriousCorrelationDetectorâ€¦2023-08-11 10:41:08,069 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: Running\n",
      "2023-08-11 10:41:10,990 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `text_length(transcription)` < 1743.000\tAssociation = 0.029\n",
      "2023-08-11 10:41:10,994 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `text_length(transcription)` >= 1743.000\tAssociation = 0.029\n",
      "2023-08-11 10:41:10,997 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `avg_digits(transcription)` < 0.012\tAssociation = 0.042\n",
      "2023-08-11 10:41:11,000 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `avg_digits(transcription)` >= 0.012\tAssociation = 0.042\n",
      "2023-08-11 10:41:11,004 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `language(transcription)` == \"en\"\tAssociation = 0.003\n",
      "2023-08-11 10:41:11,032 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"bleeding\"\tAssociation = 0.033\n",
      "2023-08-11 10:41:11,047 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"large\"\tAssociation = 0.018\n",
      "2023-08-11 10:41:11,089 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"normal\"\tAssociation = 0.046\n",
      "2023-08-11 10:41:11,204 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"cautery\"\tAssociation = 0.027\n",
      "2023-08-11 10:41:11,291 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"weakness\"\tAssociation = 0.037\n",
      "2023-08-11 10:41:11,726 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"initially\"\tAssociation = 0.008\n",
      "2023-08-11 10:41:12,079 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"examination\"\tAssociation = 0.065\n",
      "2023-08-11 10:41:12,096 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"status\"\tAssociation = 0.033\n",
      "2023-08-11 10:41:12,314 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"coronary\"\tAssociation = 0.019\n",
      "2023-08-11 10:41:12,428 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"equal\"\tAssociation = 0.030\n",
      "2023-08-11 10:41:12,445 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"external\"\tAssociation = 0.007\n",
      "2023-08-11 10:41:12,474 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"care\"\tAssociation = 0.029\n",
      "2023-08-11 10:41:12,491 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"states\"\tAssociation = 0.035\n",
      "2023-08-11 10:41:12,522 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"stool\"\tAssociation = 0.020\n",
      "2023-08-11 10:41:12,553 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"ct\"\tAssociation = 0.011\n",
      "2023-08-11 10:41:12,670 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"abnormalities\"\tAssociation = 0.012\n",
      "2023-08-11 10:41:12,730 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"admission\"\tAssociation = 0.034\n",
      "2023-08-11 10:41:12,745 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"following\"\tAssociation = 0.013\n",
      "2023-08-11 10:41:12,926 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"bilateral\"\tAssociation = 0.015\n",
      "2023-08-11 10:41:12,999 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"incision\"\tAssociation = 0.081\n",
      "2023-08-11 10:41:13,119 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"lesion\"\tAssociation = 0.008\n",
      "2023-08-11 10:41:13,180 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"open\"\tAssociation = 0.007\n",
      "2023-08-11 10:41:13,270 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"head\"\tAssociation = 0.017\n",
      "2023-08-11 10:41:13,343 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"estimated\"\tAssociation = 0.054\n",
      "2023-08-11 10:41:13,524 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"tenderness\"\tAssociation = 0.035\n",
      "2023-08-11 10:41:13,554 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"pulmonary\"\tAssociation = 0.025\n",
      "2023-08-11 10:41:13,569 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"compression\"\tAssociation = 0.024\n",
      "2023-08-11 10:41:13,598 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"flap\"\tAssociation = 0.033\n",
      "2023-08-11 10:41:13,666 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"12\"\tAssociation = 0.024\n",
      "2023-08-11 10:41:13,711 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"activity\"\tAssociation = 0.014\n",
      "2023-08-11 10:41:13,816 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"scan\"\tAssociation = 0.017\n",
      "2023-08-11 10:41:13,859 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"vaginal\"\tAssociation = 0.031\n",
      "2023-08-11 10:41:13,951 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"treatment\"\tAssociation = 0.011\n",
      "2023-08-11 10:41:13,983 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"breast\"\tAssociation = 0.008\n",
      "2023-08-11 10:41:14,168 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"resection\"\tAssociation = 0.006\n",
      "2023-08-11 10:41:14,227 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"problems\"\tAssociation = 0.025\n",
      "2023-08-11 10:41:14,287 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"fascia\"\tAssociation = 0.061\n",
      "2023-08-11 10:41:14,318 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"tract\"\tAssociation = 0.012\n",
      "2023-08-11 10:41:14,364 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"spine\"\tAssociation = 0.039\n",
      "2023-08-11 10:41:14,425 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"stable\"\tAssociation = 0.034\n",
      "2023-08-11 10:41:14,446 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"lymph\"\tAssociation = 0.018\n",
      "2023-08-11 10:41:14,869 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"colonoscopy\"\tAssociation = 0.029\n",
      "2023-08-11 10:41:14,899 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"region\"\tAssociation = 0.017\n",
      "2023-08-11 10:41:15,120 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"medication\"\tAssociation = 0.015\n",
      "2023-08-11 10:41:15,150 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"cervical\"\tAssociation = 0.025\n",
      "2023-08-11 10:41:15,210 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"shoulder\"\tAssociation = 0.009\n",
      "2023-08-11 10:41:15,238 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"changes\"\tAssociation = 0.025\n",
      "2023-08-11 10:41:15,266 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"bilaterally\"\tAssociation = 0.029\n",
      "2023-08-11 10:41:15,635 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"rectus\"\tAssociation = 0.034\n",
      "2023-08-11 10:41:15,681 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"abc\"\tAssociation = 0.014\n",
      "2023-08-11 10:41:15,698 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"ligated\"\tAssociation = 0.030\n",
      "2023-08-11 10:41:15,817 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"sinus\"\tAssociation = 0.017\n",
      "2023-08-11 10:41:15,860 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"oxygen\"\tAssociation = 0.016\n",
      "2023-08-11 10:41:15,923 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"function\"\tAssociation = 0.011\n",
      "2023-08-11 10:41:15,993 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"noted\"\tAssociation = 0.010\n",
      "2023-08-11 10:41:16,038 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"fever\"\tAssociation = 0.039\n",
      "2023-08-11 10:41:16,094 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"procedure\"\tAssociation = 0.134\n",
      "2023-08-11 10:41:16,153 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"clinical\"\tAssociation = 0.004\n",
      "2023-08-11 10:41:16,168 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"days\"\tAssociation = 0.033\n",
      "2023-08-11 10:41:16,229 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"appearance\"\tAssociation = 0.009\n",
      "2023-08-11 10:41:16,259 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"mm\"\tAssociation = 0.028\n",
      "2023-08-11 10:41:16,320 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"cardiac\"\tAssociation = 0.022\n",
      "2023-08-11 10:41:16,350 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"visit\"\tAssociation = 0.019\n",
      "2023-08-11 10:41:16,450 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"carotid\"\tAssociation = 0.019\n",
      "2023-08-11 10:41:16,527 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"arm\"\tAssociation = 0.010\n",
      "2023-08-11 10:41:16,544 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"nose\"\tAssociation = 0.034\n",
      "2023-08-11 10:41:16,562 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"suction\"\tAssociation = 0.029\n",
      "2023-08-11 10:41:16,592 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"removal\"\tAssociation = 0.019\n",
      "2023-08-11 10:41:16,651 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"rule\"\tAssociation = 0.012\n",
      "2023-08-11 10:41:16,842 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"reveals\"\tAssociation = 0.011\n",
      "2023-08-11 10:41:16,858 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"continue\"\tAssociation = 0.032\n",
      "2023-08-11 10:41:16,873 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"denied\"\tAssociation = 0.022\n",
      "2023-08-11 10:41:16,904 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"colon\"\tAssociation = 0.025\n",
      "2023-08-11 10:41:16,971 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"left\"\tAssociation = 0.035\n",
      "2023-08-11 10:41:17,292 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"stomach\"\tAssociation = 0.022\n",
      "2023-08-11 10:41:17,351 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"clear\"\tAssociation = 0.070\n",
      "2023-08-11 10:41:17,566 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"age\"\tAssociation = 0.033\n",
      "2023-08-11 10:41:17,664 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"dissection\"\tAssociation = 0.038\n",
      "2023-08-11 10:41:17,708 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"wound\"\tAssociation = 0.039\n",
      "2023-08-11 10:41:17,751 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"room\"\tAssociation = 0.054\n",
      "2023-08-11 10:41:17,765 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"family\"\tAssociation = 0.031\n",
      "2023-08-11 10:41:17,780 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"ml\"\tAssociation = 0.021\n",
      "2023-08-11 10:41:17,810 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"diarrhea\"\tAssociation = 0.030\n",
      "2023-08-11 10:41:17,853 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"biopsy\"\tAssociation = 0.012\n",
      "2023-08-11 10:41:17,886 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"swelling\"\tAssociation = 0.011\n",
      "2023-08-11 10:41:17,963 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"lung\"\tAssociation = 0.007\n",
      "2023-08-11 10:41:18,043 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"improved\"\tAssociation = 0.035\n",
      "2023-08-11 10:41:18,158 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"ear\"\tAssociation = 0.018\n",
      "2023-08-11 10:41:18,173 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"gentleman\"\tAssociation = 0.010\n",
      "2023-08-11 10:41:18,188 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"nerve\"\tAssociation = 0.037\n",
      "2023-08-11 10:41:18,221 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"sensory\"\tAssociation = 0.059\n",
      "2023-08-11 10:41:18,324 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"stitch\"\tAssociation = 0.031\n",
      "2023-08-11 10:41:18,444 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"infection\"\tAssociation = 0.026\n",
      "2023-08-11 10:41:18,461 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"tolerated\"\tAssociation = 0.044\n",
      "2023-08-11 10:41:18,476 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"lesions\"\tAssociation = 0.017\n",
      "2023-08-11 10:41:18,651 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"urinary\"\tAssociation = 0.015\n",
      "2023-08-11 10:41:18,665 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"neck\"\tAssociation = 0.062\n",
      "2023-08-11 10:41:18,744 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"headache\"\tAssociation = 0.016\n",
      "2023-08-11 10:41:18,810 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"count\"\tAssociation = 0.018\n",
      "2023-08-11 10:41:18,982 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"vicryl\"\tAssociation = 0.093\n",
      "2023-08-11 10:41:18,998 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"condition\"\tAssociation = 0.076\n",
      "2023-08-11 10:41:19,057 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"wall\"\tAssociation = 0.031\n",
      "2023-08-11 10:41:19,111 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"abdominal\"\tAssociation = 0.045\n",
      "2023-08-11 10:41:19,195 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"post\"\tAssociation = 0.016\n",
      "2023-08-11 10:41:19,404 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"return\"\tAssociation = 0.010\n",
      "2023-08-11 10:41:19,463 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"symptoms\"\tAssociation = 0.029\n",
      "2023-08-11 10:41:19,492 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"point\"\tAssociation = 0.005\n",
      "2023-08-11 10:41:19,537 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"leg\"\tAssociation = 0.012\n",
      "2023-08-11 10:41:19,569 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"medial\"\tAssociation = 0.014\n",
      "2023-08-11 10:41:19,644 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"cord\"\tAssociation = 0.043\n",
      "2023-08-11 10:41:19,674 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"dissected\"\tAssociation = 0.045\n",
      "2023-08-11 10:41:19,814 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"extremity\"\tAssociation = 0.029\n",
      "2023-08-11 10:41:19,890 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"recurrent\"\tAssociation = 0.008\n",
      "2023-08-11 10:41:19,921 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"prostate\"\tAssociation = 0.021\n",
      "2023-08-11 10:41:19,996 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"s1\"\tAssociation = 0.023\n",
      "2023-08-11 10:41:20,054 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"tube\"\tAssociation = 0.030\n",
      "2023-08-11 10:41:20,191 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"pelvis\"\tAssociation = 0.023\n",
      "2023-08-11 10:41:20,224 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"cavity\"\tAssociation = 0.032\n",
      "2023-08-11 10:41:20,318 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"mg\"\tAssociation = 0.051\n",
      "2023-08-11 10:41:20,396 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"passed\"\tAssociation = 0.006\n",
      "2023-08-11 10:41:20,456 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"xyz\"\tAssociation = 0.015\n",
      "2023-08-11 10:41:20,512 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"office\"\tAssociation = 0.025\n",
      "2023-08-11 10:41:20,587 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"knee\"\tAssociation = 0.018\n",
      "2023-08-11 10:41:20,635 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"months\"\tAssociation = 0.017\n",
      "2023-08-11 10:41:20,751 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"subjective\"\tAssociation = 0.043\n",
      "2023-08-11 10:41:20,780 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"lateral\"\tAssociation = 0.020\n",
      "2023-08-11 10:41:20,871 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"gross\"\tAssociation = 0.007\n",
      "2023-08-11 10:41:20,899 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"bladder\"\tAssociation = 0.034\n",
      "2023-08-11 10:41:21,004 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"lumbar\"\tAssociation = 0.026\n",
      "2023-08-11 10:41:21,022 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"check\"\tAssociation = 0.015\n",
      "2023-08-11 10:41:21,039 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"able\"\tAssociation = 0.019\n",
      "2023-08-11 10:41:21,265 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"patient\"\tAssociation = 0.020\n",
      "2023-08-11 10:41:21,281 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"pregnancy\"\tAssociation = 0.042\n",
      "2023-08-11 10:41:21,535 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"hours\"\tAssociation = 0.017\n",
      "2023-08-11 10:41:21,607 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"body\"\tAssociation = 0.018\n",
      "2023-08-11 10:41:21,693 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"brain\"\tAssociation = 0.044\n",
      "2023-08-11 10:41:21,753 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"discharge\"\tAssociation = 0.081\n",
      "2023-08-11 10:41:21,811 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"complex\"\tAssociation = 0.012\n",
      "2023-08-11 10:41:21,840 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"drainage\"\tAssociation = 0.008\n",
      "2023-08-11 10:41:21,870 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"aspirin\"\tAssociation = 0.015\n",
      "2023-08-11 10:41:21,885 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"having\"\tAssociation = 0.014\n",
      "2023-08-11 10:41:22,005 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"difficulty\"\tAssociation = 0.010\n",
      "2023-08-11 10:41:22,051 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"rectal\"\tAssociation = 0.021\n",
      "2023-08-11 10:41:22,092 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"right\"\tAssociation = 0.030\n",
      "2023-08-11 10:41:22,140 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"adenopathy\"\tAssociation = 0.025\n",
      "2023-08-11 10:41:22,168 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"taken\"\tAssociation = 0.070\n",
      "2023-08-11 10:41:22,274 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"mass\"\tAssociation = 0.014\n",
      "2023-08-11 10:41:22,463 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"masses\"\tAssociation = 0.032\n",
      "2023-08-11 10:41:22,550 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"uterine\"\tAssociation = 0.063\n",
      "2023-08-11 10:41:22,674 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"high\"\tAssociation = 0.009\n",
      "2023-08-11 10:41:22,691 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"followup\"\tAssociation = 0.023\n",
      "2023-08-11 10:41:22,706 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"day\"\tAssociation = 0.050\n",
      "2023-08-11 10:41:22,800 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"08\"\tAssociation = 0.011\n",
      "2023-08-11 10:41:22,895 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"ligament\"\tAssociation = 0.036\n",
      "2023-08-11 10:41:23,038 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"follow\"\tAssociation = 0.028\n",
      "2023-08-11 10:41:23,115 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"reports\"\tAssociation = 0.013\n",
      "2023-08-11 10:41:23,187 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"cancer\"\tAssociation = 0.006\n",
      "2023-08-11 10:41:23,435 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"denies\"\tAssociation = 0.061\n",
      "2023-08-11 10:41:23,520 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"vision\"\tAssociation = 0.011\n",
      "2023-08-11 10:41:23,549 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"renal\"\tAssociation = 0.012\n",
      "2023-08-11 10:41:23,564 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"visual\"\tAssociation = 0.028\n",
      "2023-08-11 10:41:23,580 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"foley\"\tAssociation = 0.025\n",
      "2023-08-11 10:41:23,660 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"erythema\"\tAssociation = 0.014\n",
      "2023-08-11 10:41:23,704 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"20\"\tAssociation = 0.016\n",
      "2023-08-11 10:41:23,722 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"daily\"\tAssociation = 0.031\n",
      "2023-08-11 10:41:23,826 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"epinephrine\"\tAssociation = 0.025\n",
      "2023-08-11 10:41:23,858 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"tumor\"\tAssociation = 0.009\n",
      "2023-08-11 10:41:23,887 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"anterior\"\tAssociation = 0.042\n",
      "2023-08-11 10:41:23,934 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"units\"\tAssociation = 0.014\n",
      "2023-08-11 10:41:24,021 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"home\"\tAssociation = 0.032\n",
      "2023-08-11 10:41:24,036 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"distress\"\tAssociation = 0.070\n",
      "2023-08-11 10:41:24,171 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"negative\"\tAssociation = 0.052\n",
      "2023-08-11 10:41:24,187 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"unremarkable\"\tAssociation = 0.037\n",
      "2023-08-11 10:41:24,219 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"superior\"\tAssociation = 0.025\n",
      "2023-08-11 10:41:24,236 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"section\"\tAssociation = 0.019\n",
      "2023-08-11 10:41:24,270 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"greater\"\tAssociation = 0.013\n",
      "2023-08-11 10:41:24,287 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"nasal\"\tAssociation = 0.034\n",
      "2023-08-11 10:41:24,340 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"sleep\"\tAssociation = 0.018\n",
      "2023-08-11 10:41:24,370 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"recovery\"\tAssociation = 0.090\n",
      "2023-08-11 10:41:24,388 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"scope\"\tAssociation = 0.041\n",
      "2023-08-11 10:41:24,430 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"mild\"\tAssociation = 0.036\n",
      "2023-08-11 10:41:24,606 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"clinic\"\tAssociation = 0.007\n",
      "2023-08-11 10:41:24,620 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"artery\"\tAssociation = 0.011\n",
      "2023-08-11 10:41:24,843 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"reviewed\"\tAssociation = 0.006\n",
      "2023-08-11 10:41:24,962 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"evidence\"\tAssociation = 0.009\n",
      "2023-08-11 10:41:25,125 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"suture\"\tAssociation = 0.058\n",
      "2023-08-11 10:41:25,417 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"sac\"\tAssociation = 0.029\n",
      "2023-08-11 10:41:25,432 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"hx\"\tAssociation = 0.047\n",
      "2023-08-11 10:41:25,461 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"pain\"\tAssociation = 0.031\n",
      "2023-08-11 10:41:25,477 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"fluid\"\tAssociation = 0.011\n",
      "2023-08-11 10:41:25,536 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"sutures\"\tAssociation = 0.034\n",
      "2023-08-11 10:41:25,596 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"uterus\"\tAssociation = 0.080\n",
      "2023-08-11 10:41:25,647 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"history\"\tAssociation = 0.031\n",
      "2023-08-11 10:41:25,678 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"scissors\"\tAssociation = 0.038\n",
      "2023-08-11 10:41:25,898 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"areas\"\tAssociation = 0.008\n",
      "2023-08-11 10:41:26,026 pid:6545 MainThread giskard.scanner.logger INFO     SpuriousCorrelationDetector: `transcription` contains \"second\"\tAssociation = 0.012\n",
      " 0 issues detected. (Took 0:00:17.959284)\n",
      "Scan completed: 30 issues found. (Took 0:03:14.984286)\n"
     ]
    }
   ],
   "source": [
    "results = giskard.scan(wrapped_model, wrapped_data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 220,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T16:02:41.204208Z",
     "start_time": "2023-06-02T16:02:40.220504Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<iframe id=\"scan-11917050016\" srcdoc=\"&lt;!doctype html&gt;\n",
       "&lt;html lang=&quot;en&quot;&gt;\n",
       "\n",
       "&lt;head&gt;\n",
       "    &lt;title&gt;Giskard Scan Results&lt;/title&gt;\n",
       "    &lt;base target=&quot;_blank&quot;&gt;\n",
       "    &lt;meta charset=&quot;utf-8&quot;&gt;\n",
       "    \n",
       "&lt;style&gt;pre code.hljs{display:block;overflow-x:auto;padding:1em}code.hljs{padding:3px 5px}\n",
       "/*!\n",
       "  Theme: GitHub Dark\n",
       "  Description: Dark theme as seen on github.com\n",
       "  Author: github.com\n",
       "  Maintainer: @Hirse\n",
       "  Updated: 2021-05-15\n",
       "\n",
       "  Outdated base version: https://github.com/primer/github-syntax-dark\n",
       "  Current colors taken from GitHub&#x27;s CSS\n",
       "*/.hljs{background:#0d1117;color:#c9d1d9}.hljs-doctag,.hljs-keyword,.hljs-meta .hljs-keyword,.hljs-template-tag,.hljs-template-variable,.hljs-type,.hljs-variable.language_{color:#ff7b72}.hljs-title,.hljs-title.class_,.hljs-title.class_.inherited__,.hljs-title.function_{color:#d2a8ff}.hljs-attr,.hljs-attribute,.hljs-literal,.hljs-meta,.hljs-number,.hljs-operator,.hljs-selector-attr,.hljs-selector-class,.hljs-selector-id,.hljs-variable{color:#79c0ff}.hljs-meta .hljs-string,.hljs-regexp,.hljs-string{color:#a5d6ff}.hljs-built_in,.hljs-symbol{color:#ffa657}.hljs-code,.hljs-comment,.hljs-formula{color:#8b949e}.hljs-name,.hljs-quote,.hljs-selector-pseudo,.hljs-selector-tag{color:#7ee787}.hljs-subst{color:#c9d1d9}.hljs-section{color:#1f6feb;font-weight:700}.hljs-bullet{color:#f2cc60}.hljs-emphasis{color:#c9d1d9;font-style:italic}.hljs-strong{color:#c9d1d9;font-weight:700}.hljs-addition{background-color:#033a16;color:#aff5b4}.hljs-deletion{background-color:#67060c;color:#ffdcd7}.hljs-copy-wrapper{overflow:hidden;position:relative}.hljs-copy-button:focus,.hljs-copy-wrapper:hover .hljs-copy-button{transform:translateX(0)}.hljs-copy-button{background-color:#2d2b57;background-color:var(--hljs-theme-background);background-image:url(&#x27;data:image/svg+xml;charset=utf-8,&lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; width=&quot;16&quot; height=&quot;16&quot; fill=&quot;none&quot; viewBox=&quot;0 0 24 24&quot;&gt;&lt;path fill=&quot;%23fff&quot; fill-rule=&quot;evenodd&quot; d=&quot;M6 5a1 1 0 0 0-1 1v14a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1V6a1 1 0 0 0-1-1h-2a1 1 0 1 1 0-2h2a3 3 0 0 1 3 3v14a3 3 0 0 1-3 3H6a3 3 0 0 1-3-3V6a3 3 0 0 1 3-3h2a1 1 0 0 1 0 2H6Z&quot; clip-rule=&quot;evenodd&quot;/&gt;&lt;path fill=&quot;%23fff&quot; fill-rule=&quot;evenodd&quot; d=&quot;M7 3a2 2 0 0 1 2-2h6a2 2 0 0 1 2 2v2a2 2 0 0 1-2 2H9a2 2 0 0 1-2-2V3Zm8 0H9v2h6V3Z&quot; clip-rule=&quot;evenodd&quot;/&gt;&lt;/svg&gt;&#x27;);background-position:50%;background-repeat:no-repeat;border:1px solid #ffffff22;border-radius:.25rem;color:#fff;height:2rem;position:absolute;right:1em;text-indent:-9999px;top:1em;transition:background-color .2s ease,transform .2s ease-out;width:2rem}.hljs-copy-button:hover{border-color:#ffffff44}.hljs-copy-button:active{border-color:#ffffff66}.hljs-copy-button[data-copied=true]{background-image:none;text-indent:0;width:auto}@media (prefers-reduced-motion){.hljs-copy-button{transition:none}}.hljs-copy-alert{clip:rect(0 0 0 0);-webkit-clip-path:inset(50%);clip-path:inset(50%);height:1px;overflow:hidden;position:absolute;white-space:nowrap;width:1px}\n",
       "/*! tailwindcss v3.3.2 | MIT License | https://tailwindcss.com*/*,:after,:before{border:0 solid #e5e7eb;box-sizing:border-box}:after,:before{--tw-content:&quot;&quot;}html{-webkit-text-size-adjust:100%;font-feature-settings:normal;font-family:ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji;font-variation-settings:normal;line-height:1.5;-moz-tab-size:4;-o-tab-size:4;tab-size:4}body{line-height:inherit;margin:0}hr{border-top-width:1px;color:inherit;height:0}abbr:where([title]){-webkit-text-decoration:underline dotted;text-decoration:underline dotted}h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight:inherit}a{color:inherit;text-decoration:inherit}b,strong{font-weight:bolder}code,kbd,pre,samp{font-family:ui-monospace,SFMono-Regular,Menlo,Monaco,Consolas,Liberation Mono,Courier New,monospace;font-size:1em}small{font-size:80%}sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}sub{bottom:-.25em}sup{top:-.5em}table{border-collapse:collapse;border-color:inherit;text-indent:0}button,input,optgroup,select,textarea{color:inherit;font-family:inherit;font-size:100%;font-weight:inherit;line-height:inherit;margin:0;padding:0}button,select{text-transform:none}[type=button],[type=reset],[type=submit],button{-webkit-appearance:button;background-color:transparent;background-image:none}:-moz-focusring{outline:auto}:-moz-ui-invalid{box-shadow:none}progress{vertical-align:baseline}::-webkit-inner-spin-button,::-webkit-outer-spin-button{height:auto}[type=search]{-webkit-appearance:textfield;outline-offset:-2px}::-webkit-search-decoration{-webkit-appearance:none}::-webkit-file-upload-button{-webkit-appearance:button;font:inherit}summary{display:list-item}blockquote,dd,dl,figure,h1,h2,h3,h4,h5,h6,hr,p,pre{margin:0}fieldset{margin:0}fieldset,legend{padding:0}menu,ol,ul{list-style:none;margin:0;padding:0}textarea{resize:vertical}input::-moz-placeholder,textarea::-moz-placeholder{color:#9ca3af;opacity:1}input::placeholder,textarea::placeholder{color:#9ca3af;opacity:1}[role=button],button{cursor:pointer}:disabled{cursor:default}audio,canvas,embed,iframe,img,object,svg,video{display:block;vertical-align:middle}img,video{height:auto;max-width:100%}[hidden]{display:none}*,:after,:before{--tw-border-spacing-x:0;--tw-border-spacing-y:0;--tw-translate-x:0;--tw-translate-y:0;--tw-rotate:0;--tw-skew-x:0;--tw-skew-y:0;--tw-scale-x:1;--tw-scale-y:1;--tw-pan-x: ;--tw-pan-y: ;--tw-pinch-zoom: ;--tw-scroll-snap-strictness:proximity;--tw-gradient-from-position: ;--tw-gradient-via-position: ;--tw-gradient-to-position: ;--tw-ordinal: ;--tw-slashed-zero: ;--tw-numeric-figure: ;--tw-numeric-spacing: ;--tw-numeric-fraction: ;--tw-ring-inset: ;--tw-ring-offset-width:0px;--tw-ring-offset-color:#fff;--tw-ring-color:rgba(59,130,246,.5);--tw-ring-offset-shadow:0 0 #0000;--tw-ring-shadow:0 0 #0000;--tw-shadow:0 0 #0000;--tw-shadow-colored:0 0 #0000;--tw-blur: ;--tw-brightness: ;--tw-contrast: ;--tw-grayscale: ;--tw-hue-rotate: ;--tw-invert: ;--tw-saturate: ;--tw-sepia: ;--tw-drop-shadow: ;--tw-backdrop-blur: ;--tw-backdrop-brightness: ;--tw-backdrop-contrast: ;--tw-backdrop-grayscale: ;--tw-backdrop-hue-rotate: ;--tw-backdrop-invert: ;--tw-backdrop-opacity: ;--tw-backdrop-saturate: ;--tw-backdrop-sepia: }::backdrop{--tw-border-spacing-x:0;--tw-border-spacing-y:0;--tw-translate-x:0;--tw-translate-y:0;--tw-rotate:0;--tw-skew-x:0;--tw-skew-y:0;--tw-scale-x:1;--tw-scale-y:1;--tw-pan-x: ;--tw-pan-y: ;--tw-pinch-zoom: ;--tw-scroll-snap-strictness:proximity;--tw-gradient-from-position: ;--tw-gradient-via-position: ;--tw-gradient-to-position: ;--tw-ordinal: ;--tw-slashed-zero: ;--tw-numeric-figure: ;--tw-numeric-spacing: ;--tw-numeric-fraction: ;--tw-ring-inset: ;--tw-ring-offset-width:0px;--tw-ring-offset-color:#fff;--tw-ring-color:rgba(59,130,246,.5);--tw-ring-offset-shadow:0 0 #0000;--tw-ring-shadow:0 0 #0000;--tw-shadow:0 0 #0000;--tw-shadow-colored:0 0 #0000;--tw-blur: ;--tw-brightness: ;--tw-contrast: ;--tw-grayscale: ;--tw-hue-rotate: ;--tw-invert: ;--tw-saturate: ;--tw-sepia: ;--tw-drop-shadow: ;--tw-backdrop-blur: ;--tw-backdrop-brightness: ;--tw-backdrop-contrast: ;--tw-backdrop-grayscale: ;--tw-backdrop-hue-rotate: ;--tw-backdrop-invert: ;--tw-backdrop-opacity: ;--tw-backdrop-saturate: ;--tw-backdrop-sepia: }.m-4{margin:1rem}.my-1{margin-bottom:.25rem;margin-top:.25rem}.my-2{margin-bottom:.5rem;margin-top:.5rem}.my-4{margin-top:1rem}.mb-4,.my-4{margin-bottom:1rem}.ml-1{margin-left:.25rem}.ml-2{margin-left:.5rem}.ml-4{margin-left:1rem}.mr-2{margin-right:.5rem}.mt-1{margin-top:.25rem}.mt-1\\.5{margin-top:.375rem}.mt-4{margin-top:1rem}.block{display:block}.inline-block{display:inline-block}.flex{display:flex}.table{display:table}.hidden{display:none}.h-11{height:2.75rem}.h-6{height:1.5rem}.h-max{height:-moz-max-content;height:max-content}.w-6{width:1.5rem}.w-full{width:100%}.w-min{width:-moz-min-content;width:min-content}.max-w-xl{max-width:36rem}.flex-grow{flex-grow:1}.table-auto{table-layout:auto}.cursor-pointer{cursor:pointer}.resize{resize:both}.list-disc{list-style-type:disc}.items-end{align-items:flex-end}.items-center{align-items:center}.space-x-1&gt;:not([hidden])~:not([hidden]){--tw-space-x-reverse:0;margin-left:calc(.25rem*(1 - var(--tw-space-x-reverse)));margin-right:calc(.25rem*var(--tw-space-x-reverse))}.overflow-hidden{overflow:hidden}.overflow-scroll{overflow:scroll}.overflow-x-auto{overflow-x:auto}.overflow-y-clip{overflow-y:clip}.text-ellipsis{text-overflow:ellipsis}.whitespace-nowrap{white-space:nowrap}.rounded{border-radius:.25rem}.rounded-full{border-radius:9999px}.rounded-sm{border-radius:.125rem}.rounded-t{border-top-left-radius:.25rem;border-top-right-radius:.25rem}.border{border-width:1px}.border-b{border-bottom-width:1px}.border-l{border-left-width:1px}.border-r{border-right-width:1px}.border-t{border-top-width:1px}.border-amber-200{--tw-border-opacity:1;border-color:rgb(253 230 138/var(--tw-border-opacity))}.border-blue-200{--tw-border-opacity:1;border-color:rgb(191 219 254/var(--tw-border-opacity))}.border-gray-500{--tw-border-opacity:1;border-color:rgb(107 114 128/var(--tw-border-opacity))}.border-gray-600{--tw-border-opacity:1;border-color:rgb(75 85 99/var(--tw-border-opacity))}.border-red-400{--tw-border-opacity:1;border-color:rgb(248 113 113/var(--tw-border-opacity))}.border-zinc-100\\/50{border-color:hsla(240,5%,96%,.5)}.border-zinc-500{--tw-border-opacity:1;border-color:rgb(113 113 122/var(--tw-border-opacity))}.border-b-gray-500{--tw-border-opacity:1;border-bottom-color:rgb(107 114 128/var(--tw-border-opacity))}.bg-amber-100\\/40{background-color:hsla(48,96%,89%,.4)}.bg-amber-200{--tw-bg-opacity:1;background-color:rgb(253 230 138/var(--tw-bg-opacity))}.bg-blue-300{--tw-bg-opacity:1;background-color:rgb(147 197 253/var(--tw-bg-opacity))}.bg-green-100\\/40{background-color:rgba(220,252,231,.4)}.bg-red-400{--tw-bg-opacity:1;background-color:rgb(248 113 113/var(--tw-bg-opacity))}.bg-zinc-500{--tw-bg-opacity:1;background-color:rgb(113 113 122/var(--tw-bg-opacity))}.bg-zinc-700{--tw-bg-opacity:1;background-color:rgb(63 63 70/var(--tw-bg-opacity))}.p-3{padding:.75rem}.p-4{padding:1rem}.px-1{padding-left:.25rem;padding-right:.25rem}.px-2{padding-left:.5rem;padding-right:.5rem}.px-3{padding-left:.75rem;padding-right:.75rem}.px-4{padding-left:1rem;padding-right:1rem}.py-0{padding-bottom:0;padding-top:0}.py-0\\.5{padding-bottom:.125rem;padding-top:.125rem}.py-2{padding-bottom:.5rem;padding-top:.5rem}.pt-1{padding-top:.25rem}.text-left{text-align:left}.text-center{text-align:center}.text-right{text-align:right}.align-middle{vertical-align:middle}.text-sm{font-size:.875rem;line-height:1.25rem}.text-xs{font-size:.75rem;line-height:1rem}.font-bold{font-weight:700}.font-medium{font-weight:500}.uppercase{text-transform:uppercase}.lowercase{text-transform:lowercase}.leading-4{line-height:1rem}.text-amber-100{--tw-text-opacity:1;color:rgb(254 243 199/var(--tw-text-opacity))}.text-amber-200{--tw-text-opacity:1;color:rgb(253 230 138/var(--tw-text-opacity))}.text-amber-900{--tw-text-opacity:1;color:rgb(120 53 15/var(--tw-text-opacity))}.text-blue-200{--tw-text-opacity:1;color:rgb(191 219 254/var(--tw-text-opacity))}.text-blue-300{--tw-text-opacity:1;color:rgb(147 197 253/var(--tw-text-opacity))}.text-blue-900{--tw-text-opacity:1;color:rgb(30 58 138/var(--tw-text-opacity))}.text-gray-400{--tw-text-opacity:1;color:rgb(156 163 175/var(--tw-text-opacity))}.text-green-50{--tw-text-opacity:1;color:rgb(240 253 244/var(--tw-text-opacity))}.text-red-400{--tw-text-opacity:1;color:rgb(248 113 113/var(--tw-text-opacity))}.text-white{--tw-text-opacity:1;color:rgb(255 255 255/var(--tw-text-opacity))}.text-zinc-100\\/90{color:hsla(240,5%,96%,.9)}.underline{text-decoration-line:underline}.filter{filter:var(--tw-blur) var(--tw-brightness) var(--tw-contrast) var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate) var(--tw-sepia) var(--tw-drop-shadow)}p a{text-decoration:underline}.tab-header{min-width:3rem}.active.tab-header{min-width:-moz-fit-content;min-width:fit-content}table.dataframe{max-width:100%;overflow:auto;width:100%}.dataframe tr{border-bottom:1px solid #555}.dataframe td,.dataframe th{padding:.5rem}.dataframe th{text-align:left!important}.first\\:border-t:first-child{border-top-width:1px}.hover\\:border-zinc-500:hover{--tw-border-opacity:1;border-color:rgb(113 113 122/var(--tw-border-opacity))}.hover\\:bg-zinc-400:hover{--tw-bg-opacity:1;background-color:rgb(161 161 170/var(--tw-bg-opacity))}.hover\\:bg-zinc-500:hover{--tw-bg-opacity:1;background-color:rgb(113 113 122/var(--tw-bg-opacity))}.hover\\:bg-zinc-700:hover{--tw-bg-opacity:1;background-color:rgb(63 63 70/var(--tw-bg-opacity))}.hover\\:text-white:hover{--tw-text-opacity:1;color:rgb(255 255 255/var(--tw-text-opacity))}.group.open .group-\\[\\.open\\]\\:inline-block{display:inline-block}.group.open .group-\\[\\.open\\]\\:hidden{display:none}.group.active .group-\\[\\.active\\]\\:border-gray-500{--tw-border-opacity:1;border-color:rgb(107 114 128/var(--tw-border-opacity))}.group.active .group-\\[\\.active\\]\\:border-b-zinc-800{--tw-border-opacity:1;border-bottom-color:rgb(39 39 42/var(--tw-border-opacity))}.group.active .group-\\[\\.active\\]\\:bg-zinc-800{--tw-bg-opacity:1;background-color:rgb(39 39 42/var(--tw-bg-opacity))}.peer.open~.peer-\\[\\.open\\]\\:table-row{display:table-row}:is(.dark .dark\\:bg-zinc-800){--tw-bg-opacity:1;background-color:rgb(39 39 42/var(--tw-bg-opacity))}:is(.dark .dark\\:bg-zinc-900){--tw-bg-opacity:1;background-color:rgb(24 24 27/var(--tw-bg-opacity))}:is(.dark .dark\\:fill-white){fill:#fff}:is(.dark .dark\\:text-white){--tw-text-opacity:1;color:rgb(255 255 255/var(--tw-text-opacity))}&lt;/style&gt;\n",
       "\n",
       "&lt;/head&gt;\n",
       "\n",
       "&lt;body&gt;\n",
       "    \n",
       "&lt;div class=&quot;dark&quot;&gt;\n",
       "    &lt;div id=&quot;gsk-scan&quot; class=&quot;dark:text-white dark:bg-zinc-800 rounded border border-gray-500&quot;&gt;\n",
       "        &lt;!-- TAB HEADER --&gt;\n",
       "&lt;div class=&quot;flex items-end pt-1 dark:bg-zinc-900 rounded-t&quot;&gt;\n",
       "    \n",
       "    &lt;div class=&quot;flex items-center px-4 dark:fill-white border-b border-gray-500 h-11&quot;&gt;\n",
       "        &lt;div&gt;\n",
       "            &lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; width=&quot;30&quot; height=&quot;15&quot; fill=&quot;none&quot;&gt;\n",
       "                &lt;path fill=&quot;#fff&quot; fill-rule=&quot;evenodd&quot;\n",
       "                    d=&quot;M22.504 1.549a4.196 4.196 0 0 1 2.573-.887v.002a3.783 3.783 0 0 1 2.706 1.086 3.783 3.783 0 0 1 1.126 2.69 3.771 3.771 0 0 1-1.126 2.69 3.77 3.77 0 0 1-2.706 1.085l-4.794.011-2.533 3.467L8.203 15l2.881-3.335a9.829 9.829 0 0 1-4.663-1.68H3.185L0 7.163h3.934C4.263 3.165 8.187 0 12.96 0c2.24 0 4.489.696 6.175 1.909a7.423 7.423 0 0 1 1.882 1.919 4.194 4.194 0 0 1 1.487-2.28ZM7.05 3.249l3.91 3.915h1.505L7.89 2.584a7.773 7.773 0 0 0-.84.665Zm4.079-2.008 5.923 5.923h1.503l-6.086-6.087c-.45.023-.898.078-1.34.164ZM4.574 8.226h-1.77l.784.693h1.584a8.454 8.454 0 0 1-.598-.693Zm9.479 0H5.984c1.469 1.477 3.656 2.377 5.977 2.422l2.092-2.422Zm-2.458 4.472 5.492-1.902 1.878-2.569h-3.508l-3.862 4.47Zm10.361-5.552h3.265a2.714 2.714 0 0 0 1.747-4.648 2.711 2.711 0 0 0-1.888-.773 3.127 3.127 0 0 0-3.123 3.124v2.297Zm3.659-3.73a.677.677 0 1 1-.134 1.348.677.677 0 0 1 .134-1.348Z&quot;\n",
       "                    clip-rule=&quot;evenodd&quot; /&gt;\n",
       "            &lt;/svg&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;span class=&quot;uppercase text-sm ml-2 py-2 leading-4 w-min&quot;&gt;30&amp;nbsp;issues\n",
       "            detected&lt;/span&gt;\n",
       "    &lt;/div&gt;\n",
       "\n",
       "    \n",
       "    \n",
       "    &lt;div data-tab-target=&quot;Performance&quot;\n",
       "        class=&quot;tab-header cursor-pointer group active&quot;&gt;\n",
       "        &lt;div\n",
       "            class=&quot;overflow-hidden text-ellipsis group-[.active]:bg-zinc-800 group-[.active]:border-b-zinc-800 group-[.active]:border-gray-500 border-l px-3 py-2 border-r border-t border-b border-gray-600 border-b-gray-500 h-11 whitespace-nowrap&quot;&gt;\n",
       "            Performance\n",
       "            \n",
       "            \n",
       "            &lt;span class=&quot;ml-1 rounded-full text-xs min-w-4 min-h-4 px-1 py-0.5 inline-block text-center bg-red-400&quot;&gt;\n",
       "                \n",
       "                        15\n",
       "                    &lt;/span&gt;\n",
       "                    \n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "    \n",
       "    &lt;div data-tab-target=&quot;Overconfidence&quot;\n",
       "        class=&quot;tab-header cursor-pointer group&quot;&gt;\n",
       "        &lt;div\n",
       "            class=&quot;overflow-hidden text-ellipsis group-[.active]:bg-zinc-800 group-[.active]:border-b-zinc-800 group-[.active]:border-gray-500 border-l px-3 py-2 border-r border-t border-b border-gray-600 border-b-gray-500 h-11 whitespace-nowrap&quot;&gt;\n",
       "            Overconfidence\n",
       "            \n",
       "            \n",
       "            &lt;span class=&quot;ml-1 rounded-full text-xs min-w-4 min-h-4 px-1 py-0.5 inline-block text-center bg-red-400&quot;&gt;\n",
       "                \n",
       "                        15\n",
       "                    &lt;/span&gt;\n",
       "                    \n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "    \n",
       "\n",
       "    \n",
       "    &lt;div class=&quot;flex-grow border-b border-gray-500 h-11&quot;&gt;&lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "&lt;!-- TAB HEADER END --&gt;\n",
       "        \n",
       "&lt;div id=&quot;Performance&quot; role=&quot;tabpanel&quot; class=&quot;m-4 mb-4&quot;&gt;\n",
       "    &lt;div class=&quot;p-3 bg-amber-100/40 rounded-sm w-full flex align-middle&quot;&gt;\n",
       "        &lt;div class=&quot;text-amber-100 mt-1.5&quot;&gt;\n",
       "            &lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; fill=&quot;none&quot; viewBox=&quot;0 0 24 24&quot; stroke-width=&quot;1.5&quot;\n",
       "                stroke=&quot;currentColor&quot; class=&quot;w-6 h-6&quot;&gt;\n",
       "                &lt;path stroke-linecap=&quot;round&quot; stroke-linejoin=&quot;round&quot;\n",
       "                    d=&quot;M12 9v3.75m-9.303 3.376c-.866 1.5.217 3.374 1.948 3.374h14.71c1.73 0 2.813-1.874 1.948-3.374L13.949 3.378c-.866-1.5-3.032-1.5-3.898 0L2.697 16.126zM12 15.75h.007v.008H12v-.008z&quot; /&gt;\n",
       "            &lt;/svg&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;div class=&quot;ml-2 text-amber-100 text-sm&quot;&gt;\n",
       "            \n",
       "            &lt;p&gt;\n",
       "                We found some data slices in your dataset on which your model performance is lower than average.\n",
       "                Performance bias may happen for different reasons:\n",
       "            &lt;/p&gt;\n",
       "            &lt;ul class=&quot;list-disc ml-4 my-1&quot;&gt;\n",
       "                &lt;li&gt;Not enough examples in the low-performing data slice in the training set&lt;/li&gt;\n",
       "                &lt;li&gt;Wrong labels in the training set in the low-performing data slice&lt;/li&gt;\n",
       "                &lt;li&gt;Drift between your training set and test set&lt;/li&gt;\n",
       "            &lt;/ul&gt;\n",
       "            &lt;p class=&quot;mt-1&quot;&gt;\n",
       "                To learn more about causes and solutions, check our &lt;a href=&quot;https://docs.giskard.ai/en/latest/getting-started/key_vulnerabilities/performance_bias/index.html&quot; target=&quot;_blank&quot; rel=&quot;noopener&quot;&gt;guide on performance bias.&lt;/a&gt;\n",
       "            &lt;/p&gt;\n",
       "            \n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "\n",
       "    &lt;div class=&quot;flex items-center space-x-1&quot;&gt;\n",
       "        &lt;h2 class=&quot;uppercase my-4 mr-2 font-medium&quot;&gt;Issues&lt;/h2&gt;\n",
       "        \n",
       "        &lt;span class=&quot;text-xs border rounded px-1 uppercase text-red-400 border-red-400&quot;&gt;15\n",
       "            major&lt;/span&gt;\n",
       "        \n",
       "        \n",
       "        \n",
       "    &lt;/div&gt;\n",
       "\n",
       "    &lt;div class=&quot;overflow-x-auto overflow-y-clip h-max&quot;&gt;\n",
       "    &lt;table class=&quot;table-auto w-full text-white&quot;&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;xyz&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.318\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -48.00% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            22 samples (5.93%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;xyz&amp;#34;,\n",
       "                the precision is 48.0%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.82)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3985&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Decreased ability to perfor...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.37)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;70&lt;/th&gt;\n",
       "      &lt;td&gt;XYZ, M.D. ,Suite 123, ABC Avenue ,City, STATE ...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (pÂ =Â 0.34)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;subjective&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.367\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -40.07% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            30 samples (8.09%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;subjective&amp;#34;,\n",
       "                the precision is 40.1%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2524&lt;/th&gt;\n",
       "      &lt;td&gt;POSTOPERATIVE DAY #1, TOTAL ABDOMINAL HYSTEREC...&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.24)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1299&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , I am following the patient today...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.74)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3313&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 66-year-old fem...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.73)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;admission&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.375\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -38.71% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            64 samples (17.25%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;admission&amp;#34;,\n",
       "                the precision is 38.7%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2911&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSES:,1.  Bilateral lower extre...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.67)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3312&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , The patient was seen and examine...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.79)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3440&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:  ,Complex open wound righ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.76)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;daily&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.385\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -37.14% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            65 samples (17.52%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;daily&amp;#34;,\n",
       "                the precision is 37.1%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3414&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (pÂ =Â 0.38)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3726&lt;/th&gt;\n",
       "      &lt;td&gt;ADMITTING DIAGNOSES:,1.  Fever.,2.  Otitis med...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.69)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3440&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:  ,Complex open wound righ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.76)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;coronary&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.391\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -36.05% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            23 samples (6.20%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;coronary&amp;#34;,\n",
       "                the precision is 36.0%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1441&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , The patient is not in acute dist...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.35)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1327&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS:,  The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.51)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3400&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:,1.  Epigastric pain.  Que...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.71)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;followup&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.396\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -35.31% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            48 samples (12.94%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;followup&amp;#34;,\n",
       "                the precision is 35.3%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3401&lt;/th&gt;\n",
       "      &lt;td&gt;ADMISSION DIAGNOSIS (ES):,1. Chronic obstructi...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.27)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1422&lt;/th&gt;\n",
       "      &lt;td&gt;PROBLEM LIST:,1.  Type 1 diabetes mellitus, in...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.41)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3317&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Followup on diabetes mellit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.85)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;count&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.400\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -34.63% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            50 samples (13.48%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;count&amp;#34;,\n",
       "                the precision is 34.6%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3461&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Abdominal pain.,HISTORY OF ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (pÂ =Â 0.72)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3414&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (pÂ =Â 0.38)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3312&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , The patient was seen and examine...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.79)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;lung&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.400\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -34.63% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            20 samples (5.39%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;lung&amp;#34;,\n",
       "                the precision is 34.6%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3723&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  This 3-year-old female pres...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.27)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3760&lt;/th&gt;\n",
       "      &lt;td&gt;TECHNIQUE: , Sequential axial CT images were o...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology (pÂ =Â 0.36)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3402&lt;/th&gt;\n",
       "      &lt;td&gt;REASON FOR ADMISSION: , Fever of unknown origi...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.70)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;function&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.414\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -32.37% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            29 samples (7.82%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;function&amp;#34;,\n",
       "                the precision is 32.4%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3400&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:,1.  Epigastric pain.  Que...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.71)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3933&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:,1.  Respiratory failure i...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.34)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3461&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Abdominal pain.,HISTORY OF ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (pÂ =Â 0.72)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;abc&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.417\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -31.90% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            24 samples (6.47%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;abc&amp;#34;,\n",
       "                the precision is 31.9%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1435&lt;/th&gt;\n",
       "      &lt;td&gt;REASON FOR VISIT:  ,Followup cervical spinal s...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology (pÂ =Â 0.78)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3461&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Abdominal pain.,HISTORY OF ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (pÂ =Â 0.72)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3451&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:  ,Blood in toilet.,HISTORY: ,...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (pÂ =Â 0.69)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;improved&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.419\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -31.46% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            31 samples (8.36%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;improved&amp;#34;,\n",
       "                the precision is 31.5%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3933&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:,1.  Respiratory failure i...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.34)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3440&lt;/th&gt;\n",
       "      &lt;td&gt;DISCHARGE DIAGNOSIS:  ,Complex open wound righ...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.76)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2910&lt;/th&gt;\n",
       "      &lt;td&gt;DIAGNOSES ON ADMISSION,1.  Cerebrovascular acc...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.76)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;discharge&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.424\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -30.78% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            85 samples (22.91%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;discharge&amp;#34;,\n",
       "                the precision is 30.8%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2647&lt;/th&gt;\n",
       "      &lt;td&gt;FINAL DIAGNOSES:,  Delivered pregnancy, choles...&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.67)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3395&lt;/th&gt;\n",
       "      &lt;td&gt;ADMITTING DIAGNOSIS:,  Aftercare of multiple t...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.70)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;130&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:, This 61-year-old male presen...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.36)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;greater&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.435\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -28.94% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            23 samples (6.20%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;greater&amp;#34;,\n",
       "                the precision is 28.9%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3230&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , This is a 61-yea...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.78)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1422&lt;/th&gt;\n",
       "      &lt;td&gt;PROBLEM LIST:,1.  Type 1 diabetes mellitus, in...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.41)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3946&lt;/th&gt;\n",
       "      &lt;td&gt;ADMISSION DIAGNOSIS: , Symptomatic cholelithia...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;Gastroenterology (pÂ =Â 0.70)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;aspirin&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.435\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -28.94% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            23 samples (6.20%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;aspirin&amp;#34;,\n",
       "                the precision is 28.9%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1330&lt;/th&gt;\n",
       "      &lt;td&gt;DIAGNOSIS:,  Polycythemia vera with secondary ...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.42)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.85)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3313&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 66-year-old fem...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.73)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;continue&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        PrecisionÂ =Â 0.442\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.612)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            -27.71% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            52 samples (14.02%)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;continue&amp;#34;,\n",
       "                the precision is 27.7%\n",
       "                lower than the global precision.    \n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3317&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Followup on diabetes mellit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.85)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1369&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.81)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3763&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Septal irritation.,HISTORY ...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.81)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "    &lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "\n",
       "&lt;div id=&quot;Overconfidence&quot; role=&quot;tabpanel&quot; class=&quot;m-4 mb-4 hidden&quot;&gt;\n",
       "    &lt;div class=&quot;p-3 bg-amber-100/40 rounded-sm w-full flex align-middle&quot;&gt;\n",
       "        &lt;div class=&quot;text-amber-100 mt-1.5&quot;&gt;\n",
       "            &lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; fill=&quot;none&quot; viewBox=&quot;0 0 24 24&quot; stroke-width=&quot;1.5&quot;\n",
       "                stroke=&quot;currentColor&quot; class=&quot;w-6 h-6&quot;&gt;\n",
       "                &lt;path stroke-linecap=&quot;round&quot; stroke-linejoin=&quot;round&quot;\n",
       "                    d=&quot;M12 9v3.75m-9.303 3.376c-.866 1.5.217 3.374 1.948 3.374h14.71c1.73 0 2.813-1.874 1.948-3.374L13.949 3.378c-.866-1.5-3.032-1.5-3.898 0L2.697 16.126zM12 15.75h.007v.008H12v-.008z&quot; /&gt;\n",
       "            &lt;/svg&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;div class=&quot;ml-2 text-amber-100 text-sm&quot;&gt;\n",
       "            \n",
       "            &lt;p&gt;\n",
       "                We found some data slices in your dataset containing significant number of overconfident predictions.\n",
       "                Overconfident predictions are rows that are incorrect but are predicted with high probabilities or\n",
       "                confidence scores. This happens when:\n",
       "            &lt;/p&gt;\n",
       "            &lt;ul class=&quot;list-disc ml-4 my-1&quot;&gt;\n",
       "                &lt;li&gt;There are not enough examples in the overconfident data slice in the training set&lt;/li&gt;\n",
       "                &lt;li&gt;Wrongly labeled examples in the training set in the overconfident data slice&lt;/li&gt;\n",
       "                &lt;li&gt;For imbalanced datasets, the model may assign high probabilities to predictions of the majority\n",
       "                    class&lt;/li&gt;\n",
       "            &lt;/ul&gt;\n",
       "            &lt;p class=&quot;mt-1&quot;&gt;\n",
       "                To learn more about causes and solutions, check our &lt;a href=&quot;https://docs.giskard.ai/en/latest/getting-started/key_vulnerabilities/overconfidence/index.html&quot; target=&quot;_blank&quot; rel=&quot;noopener&quot;&gt;guide on overconfidence issues.&lt;/a&gt;\n",
       "            &lt;/p&gt;\n",
       "            \n",
       "        &lt;/div&gt;\n",
       "    &lt;/div&gt;\n",
       "\n",
       "    &lt;div class=&quot;flex items-center space-x-1&quot;&gt;\n",
       "        &lt;h2 class=&quot;uppercase my-4 mr-2 font-medium&quot;&gt;Issues&lt;/h2&gt;\n",
       "        \n",
       "        &lt;span class=&quot;text-xs border rounded px-1 uppercase text-red-400 border-red-400&quot;&gt;11\n",
       "            major&lt;/span&gt;\n",
       "        \n",
       "        \n",
       "        &lt;span class=&quot;text-xs border rounded px-1 uppercase text-amber-200 border-amber-200&quot;&gt;4\n",
       "            medium&lt;/span&gt;\n",
       "        \n",
       "        \n",
       "    &lt;/div&gt;\n",
       "\n",
       "    &lt;div class=&quot;overflow-x-auto overflow-y-clip h-max&quot;&gt;\n",
       "    &lt;table class=&quot;table-auto w-full text-white&quot;&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;mass&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.762\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            27.57% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            16 out of 52 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;mass&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (16 samples, corresponding to\n",
       "                76.2% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;110&lt;/th&gt;\n",
       "      &lt;td&gt;1.  Pelvic tumor.,2.  Cystocele.,3.  Rectocele...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology (pÂ =Â 0.83)&lt;br&gt;Gastroente...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1369&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.81)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;58&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient retu...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.72)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;16&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.750\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            25.58% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            15 out of 38 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;16&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (15 samples, corresponding to\n",
       "                75.0% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1311&lt;/th&gt;\n",
       "      &lt;td&gt;PROBLEM LIST:,1.  Refractory hypertension, muc...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.81)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2530&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  The patient comes for three...&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.68)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;ct&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.750\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            25.58% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            15 out of 50 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;ct&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (15 samples, corresponding to\n",
       "                75.0% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3230&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , This is a 61-yea...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.78)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2910&lt;/th&gt;\n",
       "      &lt;td&gt;DIAGNOSES ON ADMISSION,1.  Cerebrovascular acc...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.76)&lt;br&gt;Neurology (pÂ =Â 0...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3940&lt;/th&gt;\n",
       "      &lt;td&gt;ADMITTING DIAGNOSIS: , Intractable migraine wi...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.71)&lt;br&gt;Neurology (pÂ =Â 0.17)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;having&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.742\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            24.23% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            23 out of 65 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;having&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (23 samples, corresponding to\n",
       "                74.2% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2941&lt;/th&gt;\n",
       "      &lt;td&gt;PREOPERATIVE DIAGNOSIS: , Brain tumors, multip...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurosurgery (pÂ =Â 0.77)&lt;br&gt;Urology (pÂ =Â 0.06)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;weight&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.742\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            24.23% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            23 out of 76 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;weight&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (23 samples, corresponding to\n",
       "                74.2% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1369&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS: , The patient is a...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.81)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;diet&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.739\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            23.76% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            17 out of 38 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;diet&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (17 samples, corresponding to\n",
       "                73.9% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3929&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Chest pain and fever.,HISTO...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.74)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3412&lt;/th&gt;\n",
       "      &lt;td&gt;ADMISSION DIAGNOSES: , Hyperglycemia, cholelit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.76)&lt;br&gt;Gastroenterology...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;today&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.737\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            23.38% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            28 out of 71 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;today&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (28 samples, corresponding to\n",
       "                73.7% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2941&lt;/th&gt;\n",
       "      &lt;td&gt;PREOPERATIVE DIAGNOSIS: , Brain tumors, multip...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurosurgery (pÂ =Â 0.77)&lt;br&gt;Urology (pÂ =Â 0.06)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;temperature&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.733\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            22.79% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            22 out of 60 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;temperature&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (22 samples, corresponding to\n",
       "                73.3% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3763&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Septal irritation.,HISTORY ...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.81)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1445&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , The patient states that she feel...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.79)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;discussed&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.727\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            21.78% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            16 out of 55 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;discussed&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (16 samples, corresponding to\n",
       "                72.7% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1435&lt;/th&gt;\n",
       "      &lt;td&gt;REASON FOR VISIT:  ,Followup cervical spinal s...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology (pÂ =Â 0.78)&lt;br&gt;SOAP / Chart / Progress ...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2733&lt;/th&gt;\n",
       "      &lt;td&gt;FINAL DIAGNOSES:,1.  Herniated nucleuses pulpo...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurosurgery&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.74)&lt;br&gt;Neurology (pÂ =Â 0...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;dr&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.725\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            21.48% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            37 out of 97 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;dr&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (37 samples, corresponding to\n",
       "                72.5% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3317&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Followup on diabetes mellit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.85)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;follow&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.722\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-red-400&quot;&gt;\n",
       "            20.93% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            26 out of 80 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;follow&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (26 samples, corresponding to\n",
       "                72.2% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1311&lt;/th&gt;\n",
       "      &lt;td&gt;PROBLEM LIST:,1.  Refractory hypertension, muc...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.81)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3412&lt;/th&gt;\n",
       "      &lt;td&gt;ADMISSION DIAGNOSES: , Hyperglycemia, cholelit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary (pÂ =Â 0.76)&lt;br&gt;Gastroenterology...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3312&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , The patient was seen and examine...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.79)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;wound&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.714\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-amber-200&quot;&gt;\n",
       "            19.60% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            15 out of 58 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;wound&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (15 samples, corresponding to\n",
       "                71.4% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;110&lt;/th&gt;\n",
       "      &lt;td&gt;1.  Pelvic tumor.,2.  Cystocele.,3.  Rectocele...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology (pÂ =Â 0.83)&lt;br&gt;Gastroente...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2941&lt;/th&gt;\n",
       "      &lt;td&gt;PREOPERATIVE DIAGNOSIS: , Brain tumors, multip...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurosurgery (pÂ =Â 0.77)&lt;br&gt;Urology (pÂ =Â 0.06)&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;2765&lt;/th&gt;\n",
       "      &lt;td&gt;PREOPERATIVE DIAGNOSIS:,  Rule out temporal ar...&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology&lt;/td&gt;\n",
       "      &lt;td&gt;Neurosurgery (pÂ =Â 0.75)&lt;br&gt;Gastroenterology (pÂ =...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;times&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.714\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-amber-200&quot;&gt;\n",
       "            19.60% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            15 out of 44 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;times&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (15 samples, corresponding to\n",
       "                71.4% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3317&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT:,  Followup on diabetes mellit...&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.85)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3763&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Septal irritation.,HISTORY ...&lt;/td&gt;\n",
       "      &lt;td&gt;ENT - Otolaryngology&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes (pÂ =Â 0.81)&lt;br&gt;Gene...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1378&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE: , This patient presents to the off...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.82)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;bladder&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.714\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-amber-200&quot;&gt;\n",
       "            19.60% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            15 out of 60 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;bladder&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (15 samples, corresponding to\n",
       "                71.4% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;110&lt;/th&gt;\n",
       "      &lt;td&gt;1.  Pelvic tumor.,2.  Cystocele.,3.  Rectocele...&lt;/td&gt;\n",
       "      &lt;td&gt;Urology&lt;/td&gt;\n",
       "      &lt;td&gt;Obstetrics / Gynecology (pÂ =Â 0.83)&lt;br&gt;Gastroente...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1435&lt;/th&gt;\n",
       "      &lt;td&gt;REASON FOR VISIT:  ,Followup cervical spinal s...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology (pÂ =Â 0.78)&lt;br&gt;SOAP / Chart / Progress ...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1404&lt;/th&gt;\n",
       "      &lt;td&gt;HISTORY OF PRESENT ILLNESS:,  The patient pres...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;Urology (pÂ =Â 0.75)&lt;br&gt;SOAP / Chart / Progress No...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "        &lt;tbody class=&quot;first:border-t border-b border-zinc-500&quot;&gt;\n",
       "            \n",
       "                &lt;tr class=&quot;gsk-issue text-sm group peer text-left cursor-pointer hover:bg-zinc-700&quot;&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;code class=&quot;mono text-blue-300&quot;&gt;`transcription` contains &amp;#34;infection&amp;#34;&lt;/code&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        Overconfidence rateÂ =Â 0.708\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            (GlobalÂ =Â 0.597)\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-amber-200&quot;&gt;\n",
       "            18.60% than global\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3&quot;&gt;\n",
       "        &lt;span class=&quot;text-gray-400&quot;&gt;\n",
       "            17 out of 61 samples\n",
       "        &lt;/span&gt;\n",
       "    &lt;/td&gt;\n",
       "    &lt;td class=&quot;p-3 text-xs text-right space-x-1&quot;&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;gsk-issue-detail-btn inline-block group-[.open]:hidden border border-zinc-100/50 text-zinc-100/90 hover:bg-zinc-500 hover:border-zinc-500 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;ShowÂ details&lt;/a&gt;\n",
       "        &lt;a href=&quot;#&quot;\n",
       "            class=&quot;hidden group-[.open]:inline-block gsk-issue-detail-btn border border-zinc-500 text-zinc-100/90 bg-zinc-500 hover:bg-zinc-400 hover:text-white px-2 py-0.5 rounded-sm&quot;&gt;HideÂ details&lt;/a&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "&lt;tr class=&quot;gsk-issue-detail text-left hidden  peer-[.open]:table-row border-b border-zinc-500 bg-zinc-700&quot;&gt;\n",
       "    &lt;td colspan=&quot;5&quot; class=&quot;p-3&quot;&gt;\n",
       "        &lt;h4 class=&quot;font-bold text-sm&quot;&gt;Description&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-sm&quot;&gt;\n",
       "            &lt;p class=&quot;my-2&quot;&gt;\n",
       "                For records in your dataset where `transcription` contains &amp;#34;infection&amp;#34;,\n",
       "                we found a significantly higher number of overconfident wrong predictions\n",
       "                (17 samples, corresponding to\n",
       "                70.8% of the wrong predictions in the data slice).\n",
       "            &lt;/p&gt;\n",
       "        &lt;/div&gt;\n",
       "        &lt;h4 class=&quot;font-bold mt-4 text-sm&quot;&gt;Examples&lt;/h4&gt;\n",
       "        &lt;div class=&quot;text-white max-w-xl text-sm overflow-scroll&quot; style=&quot;max-width: 920px&quot;&gt;\n",
       "            &lt;div&gt;\n",
       "&lt;style scoped&gt;\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "&lt;/style&gt;\n",
       "&lt;table border=&quot;1&quot; class=&quot;dataframe&quot;&gt;\n",
       "  &lt;thead&gt;\n",
       "    &lt;tr style=&quot;text-align: left;&quot;&gt;\n",
       "      &lt;th&gt;&lt;/th&gt;\n",
       "      &lt;th&gt;transcription&lt;/th&gt;\n",
       "      &lt;th&gt;medical_specialty&lt;/th&gt;\n",
       "      &lt;th&gt;Predicted `medical_specialty`&lt;/th&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/thead&gt;\n",
       "  &lt;tbody&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1385&lt;/th&gt;\n",
       "      &lt;td&gt;SUBJECTIVE:,  The patient is a 62-year-old whi...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.85)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;1435&lt;/th&gt;\n",
       "      &lt;td&gt;REASON FOR VISIT:  ,Followup cervical spinal s...&lt;/td&gt;\n",
       "      &lt;td&gt;SOAP / Chart / Progress Notes&lt;/td&gt;\n",
       "      &lt;td&gt;Neurology (pÂ =Â 0.78)&lt;br&gt;SOAP / Chart / Progress ...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "    &lt;tr&gt;\n",
       "      &lt;th&gt;3929&lt;/th&gt;\n",
       "      &lt;td&gt;CHIEF COMPLAINT: , Chest pain and fever.,HISTO...&lt;/td&gt;\n",
       "      &lt;td&gt;Discharge Summary&lt;/td&gt;\n",
       "      &lt;td&gt;General Medicine (pÂ =Â 0.74)&lt;br&gt;SOAP / Chart / Pr...&lt;/td&gt;\n",
       "    &lt;/tr&gt;\n",
       "  &lt;/tbody&gt;\n",
       "&lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "        &lt;/div&gt;\n",
       "    &lt;/td&gt;\n",
       "&lt;/tr&gt;\n",
       "            \n",
       "        &lt;/tbody&gt;\n",
       "        \n",
       "    &lt;/table&gt;\n",
       "&lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "\n",
       "\n",
       "\n",
       "        \n",
       "        &lt;div class=&quot;p-4&quot;&gt;\n",
       "    &lt;h2 class=&quot;my-4 mr-2 font-medium&quot;&gt;Debug your issues in the Giskard server&lt;/h2&gt;\n",
       "    \n",
       "    &lt;p class=&quot;text-sm my-1&quot;&gt;\n",
       "        Install the Giskard server app to:\n",
       "    &lt;/p&gt;\n",
       "    &lt;ul class=&quot;list-disc ml-4 my-1 text-sm&quot;&gt;\n",
       "        &lt;li&gt;Debug and diagnose your scan issues&lt;/li&gt;\n",
       "        &lt;li&gt;Save your scan result as a re-executable test suite to benchmark your model&lt;/li&gt;\n",
       "        &lt;li&gt;Extend your test suite with &lt;a href=&quot;https://docs.giskard.ai/en/latest/reference/tests/index.html&quot; target=&quot;_blank&quot; class=&quot;underline&quot; rel=&quot;noopener&quot;&gt;our catalog of ready-to-use tests&lt;/a&gt;&lt;/li&gt;\n",
       "    &lt;/ul&gt;\n",
       "    &lt;p class=&quot;text-sm my-1&quot;&gt;\n",
       "        You can find &lt;a href=&quot;https://docs.giskard.ai/en/latest/guides/installation_app/index.html&quot; target=&quot;_blank&quot; rel=&quot;noopener&quot;&gt;installation instructions here.&lt;/a&gt;\n",
       "    &lt;/p&gt;\n",
       "    &lt;div class=&quot;text-sm my-2&quot;&gt;\n",
       "        &lt;pre&gt;&lt;code class=&quot;language-python rounded&quot;&gt;from giskard import GiskardClient\n",
       "\n",
       "# Create a test suite from your scan results\n",
       "test_suite = results.generate_test_suite(&quot;My first test suite&quot;)\n",
       "\n",
       "# Upload your test suite to your Giskard server instance\n",
       "client = GiskardClient(&quot;http://localhost:19000&quot;, &quot;GISKARD_API_KEY&quot;)\n",
       "client.create_project(&quot;my_project_id&quot;, &quot;my_project_name&quot;)\n",
       "test_suite.upload(client, &quot;my_project_id&quot;)\n",
       "&lt;/code&gt;&lt;/pre&gt;\n",
       "    &lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "        \n",
       "    &lt;/div&gt;\n",
       "&lt;/div&gt;\n",
       "\n",
       "    \n",
       "&lt;script type=&quot;text/javascript&quot;&gt;\n",
       "&quot;use strict&quot;;function _typeof(u){return(_typeof=&quot;function&quot;==typeof Symbol&amp;&amp;&quot;symbol&quot;==typeof Symbol.iterator?function(u){return typeof u}:function(u){return u&amp;&amp;&quot;function&quot;==typeof Symbol&amp;&amp;u.constructor===Symbol&amp;&amp;u!==Symbol.prototype?&quot;symbol&quot;:typeof u})(u)}function _wrapNativeSuper(u){var e=&quot;function&quot;==typeof Map?new Map:void 0;return(_wrapNativeSuper=function(u){if(null===u||!_isNativeFunction(u))return u;if(&quot;function&quot;!=typeof u)throw new TypeError(&quot;Super expression must either be null or a function&quot;);if(void 0!==e){if(e.has(u))return e.get(u);e.set(u,t)}function t(){return _construct(u,arguments,_getPrototypeOf(this).constructor)}return t.prototype=Object.create(u.prototype,{constructor:{value:t,enumerable:!1,writable:!0,configurable:!0}}),_setPrototypeOf(t,u)})(u)}function _construct(u,e,t){return(_construct=_isNativeReflectConstruct()?Reflect.construct.bind():function(u,e,t){var n=[null];n.push.apply(n,e);var D=new(Function.bind.apply(u,n));return t&amp;&amp;_setPrototypeOf(D,t.prototype),D}).apply(null,arguments)}function _isNativeFunction(u){return-1!==Function.toString.call(u).indexOf(&quot;[native code]&quot;)}function _slicedToArray(u,e){return _arrayWithHoles(u)||_iterableToArrayLimit(u,e)||_unsupportedIterableToArray(u,e)||_nonIterableRest()}function _nonIterableRest(){throw new TypeError(&quot;Invalid attempt to destructure non-iterable instance.\\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.&quot;)}function _iterableToArrayLimit(u,e){var t=null==u?null:&quot;undefined&quot;!=typeof Symbol&amp;&amp;u[Symbol.iterator]||u[&quot;@@iterator&quot;];if(null!=t){var n,D,r,o,i=[],a=!0,c=!1;try{if(r=(t=t.call(u)).next,0===e){if(Object(t)!==t)return;a=!1}else for(;!(a=(n=r.call(t)).done)&amp;&amp;(i.push(n.value),i.length!==e);a=!0);}catch(u){c=!0,D=u}finally{try{if(!a&amp;&amp;null!=t.return&amp;&amp;(o=t.return(),Object(o)!==o))return}finally{if(c)throw D}}return i}}function _arrayWithHoles(u){if(Array.isArray(u))return u}function _inherits(u,e){if(&quot;function&quot;!=typeof e&amp;&amp;null!==e)throw new TypeError(&quot;Super expression must either be null or a function&quot;);u.prototype=Object.create(e&amp;&amp;e.prototype,{constructor:{value:u,writable:!0,configurable:!0}}),Object.defineProperty(u,&quot;prototype&quot;,{writable:!1}),e&amp;&amp;_setPrototypeOf(u,e)}function _setPrototypeOf(u,e){return(_setPrototypeOf=Object.setPrototypeOf?Object.setPrototypeOf.bind():function(u,e){return u.__proto__=e,u})(u,e)}function _createSuper(u){var e=_isNativeReflectConstruct();return function(){var t,n=_getPrototypeOf(u);if(e){var D=_getPrototypeOf(this).constructor;t=Reflect.construct(n,arguments,D)}else t=n.apply(this,arguments);return _possibleConstructorReturn(this,t)}}function _possibleConstructorReturn(u,e){if(e&amp;&amp;(&quot;object&quot;===_typeof(e)||&quot;function&quot;==typeof e))return e;if(void 0!==e)throw new TypeError(&quot;Derived constructors may only return object or undefined&quot;);return _assertThisInitialized(u)}function _assertThisInitialized(u){if(void 0===u)throw new ReferenceError(&quot;this hasn&#x27;t been initialised - super() hasn&#x27;t been called&quot;);return u}function _isNativeReflectConstruct(){if(&quot;undefined&quot;==typeof Reflect||!Reflect.construct)return!1;if(Reflect.construct.sham)return!1;if(&quot;function&quot;==typeof Proxy)return!0;try{return Boolean.prototype.valueOf.call(Reflect.construct(Boolean,[],function(){})),!0}catch(u){return!1}}function _getPrototypeOf(u){return(_getPrototypeOf=Object.setPrototypeOf?Object.getPrototypeOf.bind():function(u){return u.__proto__||Object.getPrototypeOf(u)})(u)}function _toConsumableArray(u){return _arrayWithoutHoles(u)||_iterableToArray(u)||_unsupportedIterableToArray(u)||_nonIterableSpread()}function _nonIterableSpread(){throw new TypeError(&quot;Invalid attempt to spread non-iterable instance.\\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.&quot;)}function _unsupportedIterableToArray(u,e){if(u){if(&quot;string&quot;==typeof u)return _arrayLikeToArray(u,e);var t=Object.prototype.toString.call(u).slice(8,-1);return&quot;Object&quot;===t&amp;&amp;u.constructor&amp;&amp;(t=u.constructor.name),&quot;Map&quot;===t||&quot;Set&quot;===t?Array.from(u):&quot;Arguments&quot;===t||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(t)?_arrayLikeToArray(u,e):void 0}}function _iterableToArray(u){if(&quot;undefined&quot;!=typeof Symbol&amp;&amp;null!=u[Symbol.iterator]||null!=u[&quot;@@iterator&quot;])return Array.from(u)}function _arrayWithoutHoles(u){if(Array.isArray(u))return _arrayLikeToArray(u)}function _arrayLikeToArray(u,e){(null==e||e&gt;u.length)&amp;&amp;(e=u.length);for(var t=0,n=new Array(e);t&lt;e;t++)n[t]=u[t];return n}function _classCallCheck(u,e){if(!(u instanceof e))throw new TypeError(&quot;Cannot call a class as a function&quot;)}function _defineProperties(u,e){for(var t=0;t&lt;e.length;t++){var n=e[t];n.enumerable=n.enumerable||!1,n.configurable=!0,&quot;value&quot;in n&amp;&amp;(n.writable=!0),Object.defineProperty(u,_toPropertyKey(n.key),n)}}function _createClass(u,e,t){return e&amp;&amp;_defineProperties(u.prototype,e),t&amp;&amp;_defineProperties(u,t),Object.defineProperty(u,&quot;prototype&quot;,{writable:!1}),u}function _toPropertyKey(u){var e=_toPrimitive(u,&quot;string&quot;);return&quot;symbol&quot;===_typeof(e)?e:String(e)}function _toPrimitive(u,e){if(&quot;object&quot;!==_typeof(u)||null===u)return u;var t=u[Symbol.toPrimitive];if(void 0!==t){var n=t.call(u,e||&quot;default&quot;);if(&quot;object&quot;!==_typeof(n))return n;throw new TypeError(&quot;@@toPrimitive must return a primitive value.&quot;)}return(&quot;string&quot;===e?String:Number)(u)}function _typeof(u){return(_typeof=&quot;function&quot;==typeof Symbol&amp;&amp;&quot;symbol&quot;==typeof Symbol.iterator?function(u){return typeof u}:function(u){return u&amp;&amp;&quot;function&quot;==typeof Symbol&amp;&amp;u.constructor===Symbol&amp;&amp;u!==Symbol.prototype?&quot;symbol&quot;:typeof u})(u)}!function(u){if(&quot;undefined&quot;!=typeof window){var e=!0,t=&quot;&quot;,n=0,D=&quot;&quot;,r=null,o=&quot;&quot;,i=!1,a={resize:1,click:1},c=128,s=!0,l=1,F=&quot;bodyOffset&quot;,C=F,E=!0,d=&quot;&quot;,f={},A=32,g=null,h=!1,p=!1,B=&quot;[iFrameSizer]&quot;,m=B.length,y=&quot;&quot;,b={max:1,min:1,bodyScroll:1,documentElementScroll:1},v=&quot;child&quot;,w=!0,_=window.parent,S=&quot;*&quot;,x=0,k=!1,O=null,T=16,N=1,M=&quot;scroll&quot;,R=M,j=window,I=function(){Du(&quot;onMessage function not defined&quot;)},L=function(){},P=function(){},H={height:function(){return Du(&quot;Custom height calculation function not defined&quot;),document.documentElement.offsetHeight},width:function(){return Du(&quot;Custom width calculation function not defined&quot;),document.body.scrollWidth}},z={},U=!1;try{var K=Object.create({},{passive:{get:function(){U=!0}}});window.addEventListener(&quot;test&quot;,Q,K),window.removeEventListener(&quot;test&quot;,Q,K)}catch(Au){}var W,q,X,G,J,Z,$={bodyOffset:function(){return document.body.offsetHeight+du(&quot;marginTop&quot;)+du(&quot;marginBottom&quot;)},offset:function(){return $.bodyOffset()},bodyScroll:function(){return document.body.scrollHeight},custom:function(){return H.height()},documentElementOffset:function(){return document.documentElement.offsetHeight},documentElementScroll:function(){return document.documentElement.scrollHeight},max:function(){return Math.max.apply(null,Au($))},min:function(){return Math.min.apply(null,Au($))},grow:function(){return $.max()},lowestElement:function(){return Math.max($.bodyOffset()||$.documentElementOffset(),fu(&quot;bottom&quot;,hu()))},taggedElement:function(){return gu(&quot;bottom&quot;,&quot;data-iframe-height&quot;)}},V={bodyScroll:function(){return document.body.scrollWidth},bodyOffset:function(){return document.body.offsetWidth},custom:function(){return H.width()},documentElementScroll:function(){return document.documentElement.scrollWidth},documentElementOffset:function(){return document.documentElement.offsetWidth},scroll:function(){return Math.max(V.bodyScroll(),V.documentElementScroll())},max:function(){return Math.max.apply(null,Au(V))},min:function(){return Math.min.apply(null,Au(V))},rightMostElement:function(){return fu(&quot;right&quot;,hu())},taggedElement:function(){return gu(&quot;right&quot;,&quot;data-iframe-width&quot;)}},Y=(W=pu,J=null,Z=0,function(){var u=Date.now(),e=T-(u-(Z=Z||u));return q=this,X=arguments,e&lt;=0||T&lt;e?(J&amp;&amp;(clearTimeout(J),J=null),Z=u,G=W.apply(q,X),J||(q=X=null)):J=J||setTimeout(Bu,e),G});uu(window,&quot;message&quot;,function(e){var t={init:function(){d=e.data,_=e.source,ru(),s=!1,setTimeout(function(){E=!1},c)},reset:function(){E?nu(&quot;Page reset ignored by init&quot;):(nu(&quot;Page size reset by host page&quot;),bu(&quot;resetPage&quot;))},resize:function(){mu(&quot;resizeParent&quot;,&quot;Parent window requested size check&quot;)},moveToAnchor:function(){f.findTarget(D())},inPageLink:function(){this.moveToAnchor()},pageInfo:function(){var u=D();nu(&quot;PageInfoFromParent called from parent: &quot;+u),P(JSON.parse(u)),nu(&quot; --&quot;)},message:function(){var u=D();nu(&quot;onMessage called from parent: &quot;+u),I(JSON.parse(u)),nu(&quot; --&quot;)}};function n(){return e.data.split(&quot;]&quot;)[1].split(&quot;:&quot;)[0]}function D(){return e.data.slice(e.data.indexOf(&quot;:&quot;)+1)}function r(){return e.data.split(&quot;:&quot;)[2]in{true:1,false:1}}B===(&quot;&quot;+e.data).slice(0,m)&amp;&amp;(!1===s?function(){var D=n();D in t?t[D]():(&quot;undefined&quot;==typeof module||!module.exports)&amp;&amp;&quot;iFrameResize&quot;in window||window.jQuery!==u&amp;&amp;&quot;iFrameResize&quot;in window.jQuery.prototype||r()||Du(&quot;Unexpected message (&quot;+e.data+&quot;)&quot;)}():r()?t.init():nu(&#x27;Ignored message of type &quot;&#x27;+n()+&#x27;&quot;. Received before initialization.&#x27;))}),uu(window,&quot;readystatechange&quot;,_u),_u()}function Q(){}function uu(u,e,t,n){u.addEventListener(e,t,!!U&amp;&amp;(n||{}))}function eu(u){return u.charAt(0).toUpperCase()+u.slice(1)}function tu(u){return B+&quot;[&quot;+y+&quot;] &quot;+u}function nu(u){h&amp;&amp;&quot;object&quot;==_typeof(window.console)&amp;&amp;console.log(tu(u))}function Du(u){&quot;object&quot;==_typeof(window.console)&amp;&amp;console.warn(tu(u))}function ru(){function a(u){return&quot;true&quot;===u}function s(u,e){return&quot;function&quot;==typeof u&amp;&amp;(nu(&quot;Setup custom &quot;+e+&quot;CalcMethod&quot;),H[e]=u,u=&quot;custom&quot;),u}var l;function F(u){wu(0,0,u.type,u.screenY+&quot;:&quot;+u.screenX)}function E(u,e){nu(&quot;Add event listener: &quot;+e),uu(window.document,u,F)}l=d.slice(m).split(&quot;:&quot;),y=l[0],n=u===l[1]?n:Number(l[1]),i=u===l[2]?i:a(l[2]),h=u===l[3]?h:a(l[3]),A=u===l[4]?A:Number(l[4]),e=u===l[6]?e:a(l[6]),D=l[7],C=u===l[8]?C:l[8],t=l[9],o=l[10],x=u===l[11]?x:Number(l[11]),f.enable=u!==l[12]&amp;&amp;a(l[12]),v=u===l[13]?v:l[13],R=u===l[14]?R:l[14],p=u===l[15]?p:a(l[15]),nu(&quot;Initialising iFrame (&quot;+window.location.href+&quot;)&quot;),&quot;iFrameResizer&quot;in window&amp;&amp;Object===window.iFrameResizer.constructor&amp;&amp;(l=window.iFrameResizer,nu(&quot;Reading data from page: &quot;+JSON.stringify(l)),Object.keys(l).forEach(ou,l),I=&quot;onMessage&quot;in l?l.onMessage:I,L=&quot;onReady&quot;in l?l.onReady:L,S=&quot;targetOrigin&quot;in l?l.targetOrigin:S,C=&quot;heightCalculationMethod&quot;in l?l.heightCalculationMethod:C,R=&quot;widthCalculationMethod&quot;in l?l.widthCalculationMethod:R,C=s(C,&quot;height&quot;),R=s(R,&quot;width&quot;)),nu(&quot;TargetOrigin for parent set to: &quot;+S),iu(&quot;margin&quot;,function(u,e){return-1!==e.indexOf(&quot;-&quot;)&amp;&amp;(Du(&quot;Negative CSS value ignored for margin&quot;),e=&quot;&quot;),e}(0,D=u===D?n+&quot;px&quot;:D)),iu(&quot;background&quot;,t),iu(&quot;padding&quot;,o),(l=document.createElement(&quot;div&quot;)).style.clear=&quot;both&quot;,l.style.display=&quot;block&quot;,l.style.height=&quot;0&quot;,document.body.appendChild(l),lu(),Fu(),document.documentElement.style.height=&quot;&quot;,document.body.style.height=&quot;&quot;,nu(&#x27;HTML &amp; body height set to &quot;auto&quot;&#x27;),nu(&quot;Enable public methods&quot;),j.parentIFrame={autoResize:function(u){return!0===u&amp;&amp;!1===e?(e=!0,Cu()):!1===u&amp;&amp;!0===e&amp;&amp;(e=!1,cu(&quot;remove&quot;),null!==r&amp;&amp;r.disconnect(),clearInterval(g)),wu(0,0,&quot;autoResize&quot;,JSON.stringify(e)),e},close:function(){wu(0,0,&quot;close&quot;)},getId:function(){return y},getPageInfo:function(u){&quot;function&quot;==typeof u?(P=u,wu(0,0,&quot;pageInfo&quot;)):(P=function(){},wu(0,0,&quot;pageInfoStop&quot;))},moveToAnchor:function(u){f.findTarget(u)},reset:function(){vu(&quot;parentIFrame.reset&quot;)},scrollTo:function(u,e){wu(e,u,&quot;scrollTo&quot;)},scrollToOffset:function(u,e){wu(e,u,&quot;scrollToOffset&quot;)},sendMessage:function(u,e){wu(0,0,&quot;message&quot;,JSON.stringify(u),e)},setHeightCalculationMethod:function(u){C=u,lu()},setWidthCalculationMethod:function(u){R=u,Fu()},setTargetOrigin:function(u){nu(&quot;Set targetOrigin: &quot;+u),S=u},size:function(u,e){mu(&quot;size&quot;,&quot;parentIFrame.size(&quot;+(u||&quot;&quot;)+(e?&quot;,&quot;+e:&quot;&quot;)+&quot;)&quot;,u,e)}},!0===p&amp;&amp;(E(&quot;mouseenter&quot;,&quot;Mouse Enter&quot;),E(&quot;mouseleave&quot;,&quot;Mouse Leave&quot;)),Cu(),f=function(){function e(e){e=e.split(&quot;#&quot;)[1]||e;var t=decodeURIComponent(e);t=document.getElementById(t)||document.getElementsByName(t)[0];u===t?(nu(&quot;In page link (#&quot;+e+&quot;) not found in iFrame, so sending to parent&quot;),wu(0,0,&quot;inPageLink&quot;,&quot;#&quot;+e)):(t=function(e){e=e.getBoundingClientRect();var t={x:window.pageXOffset===u?document.documentElement.scrollLeft:window.pageXOffset,y:window.pageYOffset===u?document.documentElement.scrollTop:window.pageYOffset};return{x:parseInt(e.left,10)+parseInt(t.x,10),y:parseInt(e.top,10)+parseInt(t.y,10)}}(t=t),nu(&quot;Moving to in page link (#&quot;+e+&quot;) at x: &quot;+t.x+&quot; y: &quot;+t.y),wu(t.y,t.x,&quot;scrollToOffset&quot;))}function t(){var u=window.location.hash,t=window.location.href;&quot;&quot;!==u&amp;&amp;&quot;#&quot;!==u&amp;&amp;e(t)}return f.enable?Array.prototype.forEach&amp;&amp;document.querySelectorAll?(nu(&quot;Setting up location.hash handlers&quot;),Array.prototype.forEach.call(document.querySelectorAll(&#x27;a[href^=&quot;#&quot;]&#x27;),function(u){&quot;#&quot;!==u.getAttribute(&quot;href&quot;)&amp;&amp;uu(u,&quot;click&quot;,function(u){u.preventDefault(),e(this.getAttribute(&quot;href&quot;))})}),uu(window,&quot;hashchange&quot;,t),setTimeout(t,c)):Du(&quot;In page linking not fully supported in this browser! (See README.md for IE8 workaround)&quot;):nu(&quot;In page linking not enabled&quot;),{findTarget:e}}(),mu(&quot;init&quot;,&quot;Init message from host page&quot;),L()}function ou(u){var e=u.split(&quot;Callback&quot;);2===e.length&amp;&amp;(this[e=&quot;on&quot;+e[0].charAt(0).toUpperCase()+e[0].slice(1)]=this[u],delete this[u],Du(&quot;Deprecated: &#x27;&quot;+u+&quot;&#x27; has been renamed &#x27;&quot;+e+&quot;&#x27;. The old method will be removed in the next major version.&quot;))}function iu(e,t){u!==t&amp;&amp;&quot;&quot;!==t&amp;&amp;&quot;null&quot;!==t&amp;&amp;nu(&quot;Body &quot;+e+&#x27; set to &quot;&#x27;+(document.body.style[e]=t)+&#x27;&quot;&#x27;)}function au(u){var e={add:function(e){function t(){mu(u.eventName,u.eventType)}z[e]=t,uu(window,e,t,{passive:!0})},remove:function(u){var e=z[u];delete z[u],window.removeEventListener(u,e,!1)}};u.eventNames&amp;&amp;Array.prototype.map?(u.eventName=u.eventNames[0],u.eventNames.map(e[u.method])):e[u.method](u.eventName),nu(eu(u.method)+&quot; event listener: &quot;+u.eventType)}function cu(u){au({method:u,eventType:&quot;Animation Start&quot;,eventNames:[&quot;animationstart&quot;,&quot;webkitAnimationStart&quot;]}),au({method:u,eventType:&quot;Animation Iteration&quot;,eventNames:[&quot;animationiteration&quot;,&quot;webkitAnimationIteration&quot;]}),au({method:u,eventType:&quot;Animation End&quot;,eventNames:[&quot;animationend&quot;,&quot;webkitAnimationEnd&quot;]}),au({method:u,eventType:&quot;Input&quot;,eventName:&quot;input&quot;}),au({method:u,eventType:&quot;Mouse Up&quot;,eventName:&quot;mouseup&quot;}),au({method:u,eventType:&quot;Mouse Down&quot;,eventName:&quot;mousedown&quot;}),au({method:u,eventType:&quot;Orientation Change&quot;,eventName:&quot;orientationchange&quot;}),au({method:u,eventType:&quot;Print&quot;,eventNames:[&quot;afterprint&quot;,&quot;beforeprint&quot;]}),au({method:u,eventType:&quot;Ready State Change&quot;,eventName:&quot;readystatechange&quot;}),au({method:u,eventType:&quot;Touch Start&quot;,eventName:&quot;touchstart&quot;}),au({method:u,eventType:&quot;Touch End&quot;,eventName:&quot;touchend&quot;}),au({method:u,eventType:&quot;Touch Cancel&quot;,eventName:&quot;touchcancel&quot;}),au({method:u,eventType:&quot;Transition Start&quot;,eventNames:[&quot;transitionstart&quot;,&quot;webkitTransitionStart&quot;,&quot;MSTransitionStart&quot;,&quot;oTransitionStart&quot;,&quot;otransitionstart&quot;]}),au({method:u,eventType:&quot;Transition Iteration&quot;,eventNames:[&quot;transitioniteration&quot;,&quot;webkitTransitionIteration&quot;,&quot;MSTransitionIteration&quot;,&quot;oTransitionIteration&quot;,&quot;otransitioniteration&quot;]}),au({method:u,eventType:&quot;Transition End&quot;,eventNames:[&quot;transitionend&quot;,&quot;webkitTransitionEnd&quot;,&quot;MSTransitionEnd&quot;,&quot;oTransitionEnd&quot;,&quot;otransitionend&quot;]}),&quot;child&quot;===v&amp;&amp;au({method:u,eventType:&quot;IFrame Resized&quot;,eventName:&quot;resize&quot;})}function su(u,e,t,n){return e!==u&amp;&amp;(u in t||(Du(u+&quot; is not a valid option for &quot;+n+&quot;CalculationMethod.&quot;),u=e),nu(n+&#x27; calculation method set to &quot;&#x27;+u+&#x27;&quot;&#x27;)),u}function lu(){C=su(C,F,$,&quot;height&quot;)}function Fu(){R=su(R,M,V,&quot;width&quot;)}function Cu(){var u;!0===e?(cu(&quot;add&quot;),u=A&lt;0,window.MutationObserver||window.WebKitMutationObserver?u?Eu():r=function(){function u(u){function e(u){!1===u.complete&amp;&amp;(nu(&quot;Attach listeners to &quot;+u.src),u.addEventListener(&quot;load&quot;,n,!1),u.addEventListener(&quot;error&quot;,D,!1),i.push(u))}&quot;attributes&quot;===u.type&amp;&amp;&quot;src&quot;===u.attributeName?e(u.target):&quot;childList&quot;===u.type&amp;&amp;Array.prototype.forEach.call(u.target.querySelectorAll(&quot;img&quot;),e)}function e(u){nu(&quot;Remove listeners from &quot;+u.src),u.removeEventListener(&quot;load&quot;,n,!1),u.removeEventListener(&quot;error&quot;,D,!1),i.splice(i.indexOf(u),1)}function t(u,t,n){e(u.target),mu(t,n+&quot;: &quot;+u.target.src)}function n(u){t(u,&quot;imageLoad&quot;,&quot;Image loaded&quot;)}function D(u){t(u,&quot;imageLoadFailed&quot;,&quot;Image load failed&quot;)}function r(e){mu(&quot;mutationObserver&quot;,&quot;mutationObserver: &quot;+e[0].target+&quot; &quot;+e[0].type),e.forEach(u)}var o,i=[],a=window.MutationObserver||window.WebKitMutationObserver,c=(o=document.querySelector(&quot;body&quot;),c=new a(r),nu(&quot;Create body MutationObserver&quot;),c.observe(o,{attributes:!0,attributeOldValue:!1,characterData:!0,characterDataOldValue:!1,childList:!0,subtree:!0}),c);return{disconnect:function(){&quot;disconnect&quot;in c&amp;&amp;(nu(&quot;Disconnect body MutationObserver&quot;),c.disconnect(),i.forEach(e))}}}():(nu(&quot;MutationObserver not supported in this browser!&quot;),Eu())):nu(&quot;Auto Resize disabled&quot;)}function Eu(){0!==A&amp;&amp;(nu(&quot;setInterval: &quot;+A+&quot;ms&quot;),g=setInterval(function(){mu(&quot;interval&quot;,&quot;setInterval: &quot;+A)},Math.abs(A)))}function du(u,e){return e=e||document.body,e=null===(e=document.defaultView.getComputedStyle(e,null))?0:e[u],parseInt(e,10)}function fu(u,e){for(var t,n=e.length,D=0,r=eu(u),o=Date.now(),i=0;i&lt;n;i++)D&lt;(t=e[i].getBoundingClientRect()[u]+du(&quot;margin&quot;+r,e[i]))&amp;&amp;(D=t);return o=Date.now()-o,nu(&quot;Parsed &quot;+n+&quot; HTML elements&quot;),nu(&quot;Element position calculated in &quot;+o+&quot;ms&quot;),T/2&lt;(o=o)&amp;&amp;nu(&quot;Event throttle increased to &quot;+(T=2*o)+&quot;ms&quot;),D}function Au(u){return[u.bodyOffset(),u.bodyScroll(),u.documentElementOffset(),u.documentElementScroll()]}function gu(u,e){var t=document.querySelectorAll(&quot;[&quot;+e+&quot;]&quot;);return 0===t.length&amp;&amp;(Du(&quot;No tagged elements (&quot;+e+&quot;) found on page&quot;),document.querySelectorAll(&quot;body *&quot;)),fu(u,t)}function hu(){return document.querySelectorAll(&quot;body *&quot;)}function pu(e,t,n,D){function r(u,e){return!(Math.abs(u-e)&lt;=x)}n=u===n?$[C]():n,D=u===D?V[R]():D,r(l,n)||i&amp;&amp;r(N,D)||&quot;init&quot;===e?(yu(),wu(l=n,N=D,e)):e in{init:1,interval:1,size:1}||!(C in b||i&amp;&amp;R in b)?e in{interval:1}||nu(&quot;No change in size detected&quot;):vu(t)}function Bu(){Z=Date.now(),J=null,G=W.apply(q,X),J||(q=X=null)}function mu(u,e,t,n){k&amp;&amp;u in a?nu(&quot;Trigger event cancelled: &quot;+u):(u in{reset:1,resetPage:1,init:1}||nu(&quot;Trigger event: &quot;+e),(&quot;init&quot;===u?pu:Y)(u,e,t,n))}function yu(){k||(k=!0,nu(&quot;Trigger event lock on&quot;)),clearTimeout(O),O=setTimeout(function(){k=!1,nu(&quot;Trigger event lock off&quot;),nu(&quot;--&quot;)},c)}function bu(u){l=$[C](),N=V[R](),wu(l,N,u)}function vu(u){var e=C;C=F,nu(&quot;Reset trigger event: &quot;+u),yu(),bu(&quot;reset&quot;),C=e}function wu(e,t,n,D,r){!0===w&amp;&amp;(u===r?r=S:nu(&quot;Message targetOrigin: &quot;+r),nu(&quot;Sending message to host page (&quot;+(e=y+&quot;:&quot;+e+&quot;:&quot;+t+&quot;:&quot;+n+(u===D?&quot;&quot;:&quot;:&quot;+D))+&quot;)&quot;),_.postMessage(B+e,r))}function _u(){&quot;loading&quot;!==document.readyState&amp;&amp;window.parent.postMessage(&quot;[iFrameResizerChild]Ready&quot;,&quot;*&quot;)}}();var hljs=function(){var u={exports:{}};function e(u){return u instanceof Map?u.clear=u.delete=u.set=function(){throw Error(&quot;map is read-only&quot;)}:u instanceof Set&amp;&amp;(u.add=u.clear=u.delete=function(){throw Error(&quot;set is read-only&quot;)}),Object.freeze(u),Object.getOwnPropertyNames(u).forEach(function(t){var n=u[t];&quot;object&quot;!=_typeof(n)||Object.isFrozen(n)||e(n)}),u}u.exports=e,u.exports.default=e;var t=function(){function u(e){_classCallCheck(this,u),void 0===e.data&amp;&amp;(e.data={}),this.data=e.data,this.isMatchIgnored=!1}return _createClass(u,[{key:&quot;ignoreMatch&quot;,value:function(){this.isMatchIgnored=!0}}]),u}();function n(u){return u.replace(/&amp;/g,&quot;&amp;amp;&quot;).replace(/&lt;/g,&quot;&amp;lt;&quot;).replace(/&gt;/g,&quot;&amp;gt;&quot;).replace(/&quot;/g,&quot;&amp;quot;&quot;).replace(/&#x27;/g,&quot;&amp;#x27;&quot;)}function D(u){var e=Object.create(null);for(var t in u)e[t]=u[t];for(var n=arguments.length,D=new Array(n&gt;1?n-1:0),r=1;r&lt;n;r++)D[r-1]=arguments[r];return D.forEach(function(u){for(var t in u)e[t]=u[t]}),e}var r=function(u){return!!u.scope||u.sublanguage&amp;&amp;u.language},o=function(){function u(e,t){_classCallCheck(this,u),this.buffer=&quot;&quot;,this.classPrefix=t.classPrefix,e.walk(this)}return _createClass(u,[{key:&quot;addText&quot;,value:function(u){this.buffer+=n(u)}},{key:&quot;openNode&quot;,value:function(u){if(r(u)){var e;e=u.sublanguage?&quot;language-&quot;+u.language:function(u,e){var t=e.prefix;if(u.includes(&quot;.&quot;)){var n=u.split(&quot;.&quot;);return[&quot;&quot;.concat(t).concat(n.shift())].concat(_toConsumableArray(n.map(function(u,e){return&quot;&quot;.concat(u).concat(&quot;_&quot;.repeat(e+1))}))).join(&quot; &quot;)}return&quot;&quot;.concat(t).concat(u)}(u.scope,{prefix:this.classPrefix}),this.span(e)}}},{key:&quot;closeNode&quot;,value:function(u){r(u)&amp;&amp;(this.buffer+=&quot;&lt;/span&gt;&quot;)}},{key:&quot;value&quot;,value:function(){return this.buffer}},{key:&quot;span&quot;,value:function(u){this.buffer+=&#x27;&lt;span class=&quot;&#x27;.concat(u,&#x27;&quot;&gt;&#x27;)}}]),u}(),i=function(){var u=arguments.length&gt;0&amp;&amp;void 0!==arguments[0]?arguments[0]:{},e={children:[]};return Object.assign(e,u),e},a=function(){function u(){_classCallCheck(this,u),this.rootNode=i(),this.stack=[this.rootNode]}return _createClass(u,[{key:&quot;top&quot;,get:function(){return this.stack[this.stack.length-1]}},{key:&quot;root&quot;,get:function(){return this.rootNode}},{key:&quot;add&quot;,value:function(u){this.top.children.push(u)}},{key:&quot;openNode&quot;,value:function(u){var e=i({scope:u});this.add(e),this.stack.push(e)}},{key:&quot;closeNode&quot;,value:function(){if(this.stack.length&gt;1)return this.stack.pop()}},{key:&quot;closeAllNodes&quot;,value:function(){for(;this.closeNode(););}},{key:&quot;toJSON&quot;,value:function(){return JSON.stringify(this.rootNode,null,4)}},{key:&quot;walk&quot;,value:function(u){return this.constructor._walk(u,this.rootNode)}}],[{key:&quot;_walk&quot;,value:function(u,e){var t=this;return&quot;string&quot;==typeof e?u.addText(e):e.children&amp;&amp;(u.openNode(e),e.children.forEach(function(e){return t._walk(u,e)}),u.closeNode(e)),u}},{key:&quot;_collapse&quot;,value:function(e){&quot;string&quot;!=typeof e&amp;&amp;e.children&amp;&amp;(e.children.every(function(u){return&quot;string&quot;==typeof u})?e.children=[e.children.join(&quot;&quot;)]:e.children.forEach(function(e){u._collapse(e)}))}}]),u}(),c=function(u){_inherits(t,a);var e=_createSuper(t);function t(u){var n;return _classCallCheck(this,t),(n=e.call(this)).options=u,n}return _createClass(t,[{key:&quot;addKeyword&quot;,value:function(u,e){&quot;&quot;!==u&amp;&amp;(this.openNode(e),this.addText(u),this.closeNode())}},{key:&quot;addText&quot;,value:function(u){&quot;&quot;!==u&amp;&amp;this.add(u)}},{key:&quot;addSublanguage&quot;,value:function(u,e){var t=u.root;t.sublanguage=!0,t.language=e,this.add(t)}},{key:&quot;toHTML&quot;,value:function(){return new o(this,this.options).value()}},{key:&quot;finalize&quot;,value:function(){return!0}}]),t}();function s(u){return u?&quot;string&quot;==typeof u?u:u.source:null}function l(u){return E(&quot;(?=&quot;,u,&quot;)&quot;)}function F(u){return E(&quot;(?:&quot;,u,&quot;)*&quot;)}function C(u){return E(&quot;(?:&quot;,u,&quot;)?&quot;)}function E(){for(var u=arguments.length,e=new Array(u),t=0;t&lt;u;t++)e[t]=arguments[t];return e.map(function(u){return s(u)}).join(&quot;&quot;)}function d(){for(var u=arguments.length,e=new Array(u),t=0;t&lt;u;t++)e[t]=arguments[t];return&quot;(&quot;+(function(u){var e=u[u.length-1];return&quot;object&quot;==_typeof(e)&amp;&amp;e.constructor===Object?(u.splice(u.length-1,1),e):{}}(e).capture?&quot;&quot;:&quot;?:&quot;)+e.map(function(u){return s(u)}).join(&quot;|&quot;)+&quot;)&quot;}function f(u){return RegExp(u.toString()+&quot;|&quot;).exec(&quot;&quot;).length-1}var A=/\\[(?:[^\\\\\\]]|\\\\.)*\\]|\\(\\??|\\\\([1-9][0-9]*)|\\\\./;function g(u,e){var t=e.joinWith,n=0;return u.map(function(u){for(var e=n+=1,t=s(u),D=&quot;&quot;;t.length&gt;0;){var r=A.exec(t);if(!r){D+=t;break}D+=t.substring(0,r.index),t=t.substring(r.index+r[0].length),&quot;\\\\&quot;===r[0][0]&amp;&amp;r[1]?D+=&quot;\\\\&quot;+(Number(r[1])+e):(D+=r[0],&quot;(&quot;===r[0]&amp;&amp;n++)}return D}).map(function(u){return&quot;(&quot;.concat(u,&quot;)&quot;)}).join(t)}var h=&quot;[a-zA-Z]\\\\w*&quot;,p=&quot;[a-zA-Z_]\\\\w*&quot;,B=&quot;\\\\b\\\\d+(\\\\.\\\\d+)?&quot;,m=&quot;(-?)(\\\\b0[xX][a-fA-F0-9]+|(\\\\b\\\\d+(\\\\.\\\\d*)?|\\\\.\\\\d+)([eE][-+]?\\\\d+)?)&quot;,y=&quot;\\\\b(0b[01]+)&quot;,b={begin:&quot;\\\\\\\\[\\\\s\\\\S]&quot;,relevance:0},v={scope:&quot;string&quot;,begin:&quot;&#x27;&quot;,end:&quot;&#x27;&quot;,illegal:&quot;\\\\n&quot;,contains:[b]},w={scope:&quot;string&quot;,begin:&#x27;&quot;&#x27;,end:&#x27;&quot;&#x27;,illegal:&quot;\\\\n&quot;,contains:[b]},_=function(u,e){var t=D({scope:&quot;comment&quot;,begin:u,end:e,contains:[]},arguments.length&gt;2&amp;&amp;void 0!==arguments[2]?arguments[2]:{});t.contains.push({scope:&quot;doctag&quot;,begin:&quot;[ ]*(?=(TODO|FIXME|NOTE|BUG|OPTIMIZE|HACK|XXX):)&quot;,end:/(TODO|FIXME|NOTE|BUG|OPTIMIZE|HACK|XXX):/,excludeBegin:!0,relevance:0});var n=d(&quot;I&quot;,&quot;a&quot;,&quot;is&quot;,&quot;so&quot;,&quot;us&quot;,&quot;to&quot;,&quot;at&quot;,&quot;if&quot;,&quot;in&quot;,&quot;it&quot;,&quot;on&quot;,/[A-Za-z]+[&#x27;](d|ve|re|ll|t|s|n)/,/[A-Za-z]+[-][a-z]+/,/[A-Za-z][a-z]{2,}/);return t.contains.push({begin:E(/[ ]+/,&quot;(&quot;,n,/[.]?[:]?([.][ ]|[ ])/,&quot;){3}&quot;)}),t},S=_(&quot;//&quot;,&quot;$&quot;),x=_(&quot;/\\\\*&quot;,&quot;\\\\*/&quot;),k=_(&quot;#&quot;,&quot;$&quot;),O=Object.freeze({__proto__:null,MATCH_NOTHING_RE:/\\b\\B/,IDENT_RE:h,UNDERSCORE_IDENT_RE:p,NUMBER_RE:B,C_NUMBER_RE:m,BINARY_NUMBER_RE:y,RE_STARTERS_RE:&quot;!|!=|!==|%|%=|&amp;|&amp;&amp;|&amp;=|\\\\*|\\\\*=|\\\\+|\\\\+=|,|-|-=|/=|/|:|;|&lt;&lt;|&lt;&lt;=|&lt;=|&lt;|===|==|=|&gt;&gt;&gt;=|&gt;&gt;=|&gt;=|&gt;&gt;&gt;|&gt;&gt;|&gt;|\\\\?|\\\\[|\\\\{|\\\\(|\\\\^|\\\\^=|\\\\||\\\\|=|\\\\|\\\\||~&quot;,SHEBANG:function(){var u=arguments.length&gt;0&amp;&amp;void 0!==arguments[0]?arguments[0]:{},e=/^#![ ]*\\//;return u.binary&amp;&amp;(u.begin=E(e,/.*\\b/,u.binary,/\\b.*/)),D({scope:&quot;meta&quot;,begin:e,end:/$/,relevance:0,&quot;on:begin&quot;:function(u,e){0!==u.index&amp;&amp;e.ignoreMatch()}},u)},BACKSLASH_ESCAPE:b,APOS_STRING_MODE:v,QUOTE_STRING_MODE:w,PHRASAL_WORDS_MODE:{begin:/\\b(a|an|the|are|I&#x27;m|isn&#x27;t|don&#x27;t|doesn&#x27;t|won&#x27;t|but|just|should|pretty|simply|enough|gonna|going|wtf|so|such|will|you|your|they|like|more)\\b/},COMMENT:_,C_LINE_COMMENT_MODE:S,C_BLOCK_COMMENT_MODE:x,HASH_COMMENT_MODE:k,NUMBER_MODE:{scope:&quot;number&quot;,begin:B,relevance:0},C_NUMBER_MODE:{scope:&quot;number&quot;,begin:m,relevance:0},BINARY_NUMBER_MODE:{scope:&quot;number&quot;,begin:y,relevance:0},REGEXP_MODE:{begin:/(?=\\/[^/\\n]*\\/)/,contains:[{scope:&quot;regexp&quot;,begin:/\\//,end:/\\/[gimuy]*/,illegal:/\\n/,contains:[b,{begin:/\\[/,end:/\\]/,relevance:0,contains:[b]}]}]},TITLE_MODE:{scope:&quot;title&quot;,begin:h,relevance:0},UNDERSCORE_TITLE_MODE:{scope:&quot;title&quot;,begin:p,relevance:0},METHOD_GUARD:{begin:&quot;\\\\.\\\\s*[a-zA-Z_]\\\\w*&quot;,relevance:0},END_SAME_AS_BEGIN:function(u){return Object.assign(u,{&quot;on:begin&quot;:function(u,e){e.data._beginMatch=u[1]},&quot;on:end&quot;:function(u,e){e.data._beginMatch!==u[1]&amp;&amp;e.ignoreMatch()}})}});function T(u,e){&quot;.&quot;===u.input[u.index-1]&amp;&amp;e.ignoreMatch()}function N(u,e){void 0!==u.className&amp;&amp;(u.scope=u.className,delete u.className)}function M(u,e){e&amp;&amp;u.beginKeywords&amp;&amp;(u.begin=&quot;\\\\b(&quot;+u.beginKeywords.split(&quot; &quot;).join(&quot;|&quot;)+&quot;)(?!\\\\.)(?=\\\\b|\\\\s)&quot;,u.__beforeBegin=T,u.keywords=u.keywords||u.beginKeywords,delete u.beginKeywords,void 0===u.relevance&amp;&amp;(u.relevance=0))}function R(u,e){Array.isArray(u.illegal)&amp;&amp;(u.illegal=d.apply(void 0,_toConsumableArray(u.illegal)))}function j(u,e){if(u.match){if(u.begin||u.end)throw Error(&quot;begin &amp; end are not supported with match&quot;);u.begin=u.match,delete u.match}}function I(u,e){void 0===u.relevance&amp;&amp;(u.relevance=1)}var L=function(u,e){if(u.beforeMatch){if(u.starts)throw Error(&quot;beforeMatch cannot be used with starts&quot;);var t=Object.assign({},u);Object.keys(u).forEach(function(e){delete u[e]}),u.keywords=t.keywords,u.begin=E(t.beforeMatch,l(t.begin)),u.starts={relevance:0,contains:[Object.assign(t,{endsParent:!0})]},u.relevance=0,delete t.beforeMatch}},P=[&quot;of&quot;,&quot;and&quot;,&quot;for&quot;,&quot;in&quot;,&quot;not&quot;,&quot;or&quot;,&quot;if&quot;,&quot;then&quot;,&quot;parent&quot;,&quot;list&quot;,&quot;value&quot;];function H(u,e){return e?Number(e):function(u){return P.includes(u.toLowerCase())}(u)?0:1}var z={},U=function(u){console.error(u)},K=function(u){for(var e,t=arguments.length,n=new Array(t&gt;1?t-1:0),D=1;D&lt;t;D++)n[D-1]=arguments[D];(e=console).log.apply(e,[&quot;WARN: &quot;+u].concat(n))},W=function(u,e){z[&quot;&quot;.concat(u,&quot;/&quot;).concat(e)]||(console.log(&quot;Deprecated as of &quot;.concat(u,&quot;. &quot;).concat(e)),z[&quot;&quot;.concat(u,&quot;/&quot;).concat(e)]=!0)},q=Error();function X(u,e,t){for(var n=t.key,D=0,r=u[n],o={},i={},a=1;a&lt;=e.length;a++)i[a+D]=r[a],o[a+D]=!0,D+=f(e[a-1]);u[n]=i,u[n]._emit=o,u[n]._multi=!0}function G(u){(function(u){u.scope&amp;&amp;&quot;object&quot;==_typeof(u.scope)&amp;&amp;null!==u.scope&amp;&amp;(u.beginScope=u.scope,delete u.scope)})(u),&quot;string&quot;==typeof u.beginScope&amp;&amp;(u.beginScope={_wrap:u.beginScope}),&quot;string&quot;==typeof u.endScope&amp;&amp;(u.endScope={_wrap:u.endScope}),function(u){if(Array.isArray(u.begin)){if(u.skip||u.excludeBegin||u.returnBegin)throw U(&quot;skip, excludeBegin, returnBegin not compatible with beginScope: {}&quot;),q;if(&quot;object&quot;!=_typeof(u.beginScope)||null===u.beginScope)throw U(&quot;beginScope must be object&quot;),q;X(u,u.begin,{key:&quot;beginScope&quot;}),u.begin=g(u.begin,{joinWith:&quot;&quot;})}}(u),function(u){if(Array.isArray(u.end)){if(u.skip||u.excludeEnd||u.returnEnd)throw U(&quot;skip, excludeEnd, returnEnd not compatible with endScope: {}&quot;),q;if(&quot;object&quot;!=_typeof(u.endScope)||null===u.endScope)throw U(&quot;endScope must be object&quot;),q;X(u,u.end,{key:&quot;endScope&quot;}),u.end=g(u.end,{joinWith:&quot;&quot;})}}(u)}function J(u){function e(e,t){return RegExp(s(e),&quot;m&quot;+(u.case_insensitive?&quot;i&quot;:&quot;&quot;)+(u.unicodeRegex?&quot;u&quot;:&quot;&quot;)+(t?&quot;g&quot;:&quot;&quot;))}var t=function(){function u(){_classCallCheck(this,u),this.matchIndexes={},this.regexes=[],this.matchAt=1,this.position=0}return _createClass(u,[{key:&quot;addRule&quot;,value:function(u,e){e.position=this.position++,this.matchIndexes[this.matchAt]=e,this.regexes.push([e,u]),this.matchAt+=f(u)+1}},{key:&quot;compile&quot;,value:function(){0===this.regexes.length&amp;&amp;(this.exec=function(){return null});var u=this.regexes.map(function(u){return u[1]});this.matcherRe=e(g(u,{joinWith:&quot;|&quot;}),!0),this.lastIndex=0}},{key:&quot;exec&quot;,value:function(u){this.matcherRe.lastIndex=this.lastIndex;var e=this.matcherRe.exec(u);if(!e)return null;var t=e.findIndex(function(u,e){return e&gt;0&amp;&amp;void 0!==u}),n=this.matchIndexes[t];return e.splice(0,t),Object.assign(e,n)}}]),u}(),n=function(){function u(){_classCallCheck(this,u),this.rules=[],this.multiRegexes=[],this.count=0,this.lastIndex=0,this.regexIndex=0}return _createClass(u,[{key:&quot;getMatcher&quot;,value:function(u){if(this.multiRegexes[u])return this.multiRegexes[u];var e=new t;return this.rules.slice(u).forEach(function(u){var t=_slicedToArray(u,2),n=t[0],D=t[1];return e.addRule(n,D)}),e.compile(),this.multiRegexes[u]=e,e}},{key:&quot;resumingScanAtSamePosition&quot;,value:function(){return 0!==this.regexIndex}},{key:&quot;considerAll&quot;,value:function(){this.regexIndex=0}},{key:&quot;addRule&quot;,value:function(u,e){this.rules.push([u,e]),&quot;begin&quot;===e.type&amp;&amp;this.count++}},{key:&quot;exec&quot;,value:function(u){var e=this.getMatcher(this.regexIndex);e.lastIndex=this.lastIndex;var t=e.exec(u);if(this.resumingScanAtSamePosition())if(t&amp;&amp;t.index===this.lastIndex);else{var n=this.getMatcher(0);n.lastIndex=this.lastIndex+1,t=n.exec(u)}return t&amp;&amp;(this.regexIndex+=t.position+1,this.regexIndex===this.count&amp;&amp;this.considerAll()),t}}]),u}();if(u.compilerExtensions||(u.compilerExtensions=[]),u.contains&amp;&amp;u.contains.includes(&quot;self&quot;))throw Error(&quot;ERR: contains `self` is not supported at the top-level of a language.  See documentation.&quot;);return u.classNameAliases=D(u.classNameAliases||{}),function t(r,o){var i,a=r;if(r.isCompiled)return a;[N,j,G,L].forEach(function(u){return u(r,o)}),u.compilerExtensions.forEach(function(u){return u(r,o)}),r.__beforeBegin=null,[M,R,I].forEach(function(u){return u(r,o)}),r.isCompiled=!0;var c=null;return&quot;object&quot;==_typeof(r.keywords)&amp;&amp;r.keywords.$pattern&amp;&amp;(r.keywords=Object.assign({},r.keywords),c=r.keywords.$pattern,delete r.keywords.$pattern),c=c||/\\w+/,r.keywords&amp;&amp;(r.keywords=function u(e,t){var n=arguments.length&gt;2&amp;&amp;void 0!==arguments[2]?arguments[2]:&quot;keyword&quot;,D=Object.create(null);return&quot;string&quot;==typeof e?r(n,e.split(&quot; &quot;)):Array.isArray(e)?r(n,e):Object.keys(e).forEach(function(n){Object.assign(D,u(e[n],t,n))}),D;function r(u,e){t&amp;&amp;(e=e.map(function(u){return u.toLowerCase()})),e.forEach(function(e){var t=e.split(&quot;|&quot;);D[t[0]]=[u,H(t[0],t[1])]})}}(r.keywords,u.case_insensitive)),a.keywordPatternRe=e(c,!0),o&amp;&amp;(r.begin||(r.begin=/\\B|\\b/),a.beginRe=e(a.begin),r.end||r.endsWithParent||(r.end=/\\B|\\b/),r.end&amp;&amp;(a.endRe=e(a.end)),a.terminatorEnd=s(a.end)||&quot;&quot;,r.endsWithParent&amp;&amp;o.terminatorEnd&amp;&amp;(a.terminatorEnd+=(r.end?&quot;|&quot;:&quot;&quot;)+o.terminatorEnd)),r.illegal&amp;&amp;(a.illegalRe=e(r.illegal)),r.contains||(r.contains=[]),r.contains=(i=[]).concat.apply(i,_toConsumableArray(r.contains.map(function(u){return function(u){return u.variants&amp;&amp;!u.cachedVariants&amp;&amp;(u.cachedVariants=u.variants.map(function(e){return D(u,{variants:null},e)})),u.cachedVariants?u.cachedVariants:function u(e){return!!e&amp;&amp;(e.endsWithParent||u(e.starts))}(u)?D(u,{starts:u.starts?D(u.starts):null}):Object.isFrozen(u)?D(u):u}(&quot;self&quot;===u?r:u)}))),r.contains.forEach(function(u){t(u,a)}),r.starts&amp;&amp;t(r.starts,o),a.matcher=function(u){var e=new n;return u.contains.forEach(function(u){return e.addRule(u.begin,{rule:u,type:&quot;begin&quot;})}),u.terminatorEnd&amp;&amp;e.addRule(u.terminatorEnd,{type:&quot;end&quot;}),u.illegal&amp;&amp;e.addRule(u.illegal,{type:&quot;illegal&quot;}),e}(a),a}(u)}var Z=function(u){_inherits(t,_wrapNativeSuper(Error));var e=_createSuper(t);function t(u,n){var D;return _classCallCheck(this,t),(D=e.call(this,u)).name=&quot;HTMLInjectionError&quot;,D.html=n,D}return _createClass(t)}(),$=n,V=D,Y=Symbol(&quot;nomatch&quot;);return function(e){var n=Object.create(null),D=Object.create(null),r=[],o=!0,i=&quot;Could not find the language &#x27;{}&#x27;, did you forget to load/include a language module?&quot;,a={disableAutodetect:!0,name:&quot;Plain text&quot;,contains:[]},s={ignoreUnescapedHTML:!1,throwUnescapedHTML:!1,noHighlightRe:/^(no-?highlight)$/i,languageDetectRe:/\\blang(?:uage)?-([\\w-]+)\\b/i,classPrefix:&quot;hljs-&quot;,cssSelector:&quot;pre code&quot;,languages:null,__emitter:c};function f(u){return s.noHighlightRe.test(u)}function A(u,e,t){var n=&quot;&quot;,D=&quot;&quot;;&quot;object&quot;==_typeof(e)?(n=u,t=e.ignoreIllegals,D=e.language):(W(&quot;10.7.0&quot;,&quot;highlight(lang, code, ...args) has been deprecated.&quot;),W(&quot;10.7.0&quot;,&quot;Please use highlight(code, options) instead.\\nhttps://github.com/highlightjs/highlight.js/issues/2277&quot;),D=u,n=e),void 0===t&amp;&amp;(t=!0);var r={code:n,language:D};w(&quot;before:highlight&quot;,r);var o=r.result?r.result:g(r.language,r.code,t);return o.code=r.code,w(&quot;after:highlight&quot;,o),o}function g(u,e,D,r){var a=Object.create(null);function c(){if(b.keywords){var u=0;b.keywordPatternRe.lastIndex=0;for(var e,t=b.keywordPatternRe.exec(_),n=&quot;&quot;;t;){n+=_.substring(u,t.index);var D=p.case_insensitive?t[0].toLowerCase():t[0],r=(e=D,b.keywords[e]);if(r){var o=_slicedToArray(r,2),i=o[0],c=o[1];if(w.addText(n),n=&quot;&quot;,a[D]=(a[D]||0)+1,a[D]&lt;=7&amp;&amp;(S+=c),i.startsWith(&quot;_&quot;))n+=t[0];else{var s=p.classNameAliases[i]||i;w.addKeyword(t[0],s)}}else n+=t[0];u=b.keywordPatternRe.lastIndex,t=b.keywordPatternRe.exec(_)}n+=_.substring(u),w.addText(n)}else w.addText(_)}function l(){null!=b.subLanguage?function(){if(&quot;&quot;!==_){var u=null;if(&quot;string&quot;==typeof b.subLanguage){if(!n[b.subLanguage])return void w.addText(_);u=g(b.subLanguage,_,!0,v[b.subLanguage]),v[b.subLanguage]=u._top}else u=h(_,b.subLanguage.length?b.subLanguage:null);b.relevance&gt;0&amp;&amp;(S+=u.relevance),w.addSublanguage(u._emitter,u.language)}}():c(),_=&quot;&quot;}function F(u,e){for(var t=1,n=e.length-1;t&lt;=n;)if(u._emit[t]){var D=p.classNameAliases[u[t]]||u[t],r=e[t];D?w.addKeyword(r,D):(_=r,c(),_=&quot;&quot;),t++}else t++}function C(u,e){return u.scope&amp;&amp;&quot;string&quot;==typeof u.scope&amp;&amp;w.openNode(p.classNameAliases[u.scope]||u.scope),u.beginScope&amp;&amp;(u.beginScope._wrap?(w.addKeyword(_,p.classNameAliases[u.beginScope._wrap]||u.beginScope._wrap),_=&quot;&quot;):u.beginScope._multi&amp;&amp;(F(u.beginScope,e),_=&quot;&quot;)),b=Object.create(u,{parent:{value:b}})}function E(u,e,n){var D=function(u,e){var t=u&amp;&amp;u.exec(e);return t&amp;&amp;0===t.index}(u.endRe,n);if(D){if(u[&quot;on:end&quot;]){var r=new t(u);u[&quot;on:end&quot;](e,r),r.isMatchIgnored&amp;&amp;(D=!1)}if(D){for(;u.endsParent&amp;&amp;u.parent;)u=u.parent;return u}}if(u.endsWithParent)return E(u.parent,e,n)}function d(u){return 0===b.matcher.regexIndex?(_+=u[0],1):(O=!0,0)}var f={};function A(n,r){var i=r&amp;&amp;r[0];if(_+=n,null==i)return l(),0;if(&quot;begin&quot;===f.type&amp;&amp;&quot;end&quot;===r.type&amp;&amp;f.index===r.index&amp;&amp;&quot;&quot;===i){if(_+=e.slice(r.index,r.index+1),!o){var a=Error(&quot;0 width match regex (&quot;.concat(u,&quot;)&quot;));throw a.languageName=u,a.badRule=f.rule,a}return 1}if(f=r,&quot;begin&quot;===r.type)return function(u){for(var e=u[0],n=u.rule,D=new t(n),r=0,o=[n.__beforeBegin,n[&quot;on:begin&quot;]];r&lt;o.length;r++){var i=o[r];if(i&amp;&amp;(i(u,D),D.isMatchIgnored))return d(e)}return n.skip?_+=e:(n.excludeBegin&amp;&amp;(_+=e),l(),n.returnBegin||n.excludeBegin||(_=e)),C(n,u),n.returnBegin?0:e.length}(r);if(&quot;illegal&quot;===r.type&amp;&amp;!D){var c=Error(&#x27;Illegal lexeme &quot;&#x27;+i+&#x27;&quot; for mode &quot;&#x27;+(b.scope||&quot;&lt;unnamed&gt;&quot;)+&#x27;&quot;&#x27;);throw c.mode=b,c}if(&quot;end&quot;===r.type){var s=function(u){var t=u[0],n=e.substring(u.index),D=E(b,u,n);if(!D)return Y;var r=b;b.endScope&amp;&amp;b.endScope._wrap?(l(),w.addKeyword(t,b.endScope._wrap)):b.endScope&amp;&amp;b.endScope._multi?(l(),F(b.endScope,u)):r.skip?_+=t:(r.returnEnd||r.excludeEnd||(_+=t),l(),r.excludeEnd&amp;&amp;(_=t));do{b.scope&amp;&amp;w.closeNode(),b.skip||b.subLanguage||(S+=b.relevance),b=b.parent}while(b!==D.parent);return D.starts&amp;&amp;C(D.starts,u),r.returnEnd?0:t.length}(r);if(s!==Y)return s}if(&quot;illegal&quot;===r.type&amp;&amp;&quot;&quot;===i)return 1;if(k&gt;1e5&amp;&amp;k&gt;3*r.index)throw Error(&quot;potential infinite loop, way more iterations than matches&quot;);return _+=i,i.length}var p=y(u);if(!p)throw U(i.replace(&quot;{}&quot;,u)),Error(&#x27;Unknown language: &quot;&#x27;+u+&#x27;&quot;&#x27;);var B=J(p),m=&quot;&quot;,b=r||B,v={},w=new s.__emitter(s);!function(){for(var u=[],e=b;e!==p;e=e.parent)e.scope&amp;&amp;u.unshift(e.scope);u.forEach(function(u){return w.openNode(u)})}();var _=&quot;&quot;,S=0,x=0,k=0,O=!1;try{for(b.matcher.considerAll();;){k++,O?O=!1:b.matcher.considerAll(),b.matcher.lastIndex=x;var T=b.matcher.exec(e);if(!T)break;var N=A(e.substring(x,T.index),T);x=T.index+N}return A(e.substring(x)),w.closeAllNodes(),w.finalize(),m=w.toHTML(),{language:u,value:m,relevance:S,illegal:!1,_emitter:w,_top:b}}catch(t){if(t.message&amp;&amp;t.message.includes(&quot;Illegal&quot;))return{language:u,value:$(e),illegal:!0,relevance:0,_illegalBy:{message:t.message,index:x,context:e.slice(x-100,x+100),mode:t.mode,resultSoFar:m},_emitter:w};if(o)return{language:u,value:$(e),illegal:!1,relevance:0,errorRaised:t,_emitter:w,_top:b};throw t}}function h(u,e){e=e||s.languages||Object.keys(n);var t=function(u){var e={value:$(u),illegal:!1,relevance:0,_top:a,_emitter:new s.__emitter(s)};return e._emitter.addText(u),e}(u),D=e.filter(y).filter(v).map(function(e){return g(e,u,!1)});D.unshift(t);var r=_slicedToArray(D.sort(function(u,e){if(u.relevance!==e.relevance)return e.relevance-u.relevance;if(u.language&amp;&amp;e.language){if(y(u.language).supersetOf===e.language)return 1;if(y(e.language).supersetOf===u.language)return-1}return 0}),2),o=r[0],i=r[1],c=o;return c.secondBest=i,c}function p(u){var e=function(u){var e=u.className+&quot; &quot;;e+=u.parentNode?u.parentNode.className:&quot;&quot;;var t=s.languageDetectRe.exec(e);if(t){var n=y(t[1]);return n||(K(i.replace(&quot;{}&quot;,t[1])),K(&quot;Falling back to no-highlight mode for this block.&quot;,u)),n?t[1]:&quot;no-highlight&quot;}return e.split(/\\s+/).find(function(u){return f(u)||y(u)})}(u);if(!f(e)){if(w(&quot;before:highlightElement&quot;,{el:u,language:e}),u.children.length&gt;0&amp;&amp;(s.ignoreUnescapedHTML||(console.warn(&quot;One of your code blocks includes unescaped HTML. This is a potentially serious security risk.&quot;),console.warn(&quot;https://github.com/highlightjs/highlight.js/wiki/security&quot;),console.warn(&quot;The element with unescaped HTML:&quot;),console.warn(u)),s.throwUnescapedHTML))throw new Z(&quot;One of your code blocks includes unescaped HTML.&quot;,u.innerHTML);var t=u.textContent,n=e?A(t,{language:e,ignoreIllegals:!0}):h(t);u.innerHTML=n.value,function(u,e,t){var n=e&amp;&amp;D[e]||t;u.classList.add(&quot;hljs&quot;),u.classList.add(&quot;language-&quot;+n)}(u,e,n.language),u.result={language:n.language,re:n.relevance,relevance:n.relevance},n.secondBest&amp;&amp;(u.secondBest={language:n.secondBest.language,relevance:n.secondBest.relevance}),w(&quot;after:highlightElement&quot;,{el:u,result:n,text:t})}}var B=!1;function m(){&quot;loading&quot;!==document.readyState?document.querySelectorAll(s.cssSelector).forEach(p):B=!0}function y(u){return u=(u||&quot;&quot;).toLowerCase(),n[u]||n[D[u]]}function b(u,e){var t=e.languageName;&quot;string&quot;==typeof u&amp;&amp;(u=[u]),u.forEach(function(u){D[u.toLowerCase()]=t})}function v(u){var e=y(u);return e&amp;&amp;!e.disableAutodetect}function w(u,e){var t=u;r.forEach(function(u){u[t]&amp;&amp;u[t](e)})}for(var _ in&quot;undefined&quot;!=typeof window&amp;&amp;window.addEventListener&amp;&amp;window.addEventListener(&quot;DOMContentLoaded&quot;,function(){B&amp;&amp;m()},!1),Object.assign(e,{highlight:A,highlightAuto:h,highlightAll:m,highlightElement:p,highlightBlock:function(u){return W(&quot;10.7.0&quot;,&quot;highlightBlock will be removed entirely in v12.0&quot;),W(&quot;10.7.0&quot;,&quot;Please use highlightElement now.&quot;),p(u)},configure:function(u){s=V(s,u)},initHighlighting:function(){m(),W(&quot;10.6.0&quot;,&quot;initHighlighting() deprecated.  Use highlightAll() now.&quot;)},initHighlightingOnLoad:function(){m(),W(&quot;10.6.0&quot;,&quot;initHighlightingOnLoad() deprecated.  Use highlightAll() now.&quot;)},registerLanguage:function(u,t){var D=null;try{D=t(e)}catch(e){if(U(&quot;Language definition for &#x27;{}&#x27; could not be registered.&quot;.replace(&quot;{}&quot;,u)),!o)throw e;U(e),D=a}D.name||(D.name=u),n[u]=D,D.rawDefinition=t.bind(null,e),D.aliases&amp;&amp;b(D.aliases,{languageName:u})},unregisterLanguage:function(u){delete n[u];for(var e=0,t=Object.keys(D);e&lt;t.length;e++){var r=t[e];D[r]===u&amp;&amp;delete D[r]}},listLanguages:function(){return Object.keys(n)},getLanguage:y,registerAliases:b,autoDetection:v,inherit:V,addPlugin:function(u){(function(u){u[&quot;before:highlightBlock&quot;]&amp;&amp;!u[&quot;before:highlightElement&quot;]&amp;&amp;(u[&quot;before:highlightElement&quot;]=function(e){u[&quot;before:highlightBlock&quot;](Object.assign({block:e.el},e))}),u[&quot;after:highlightBlock&quot;]&amp;&amp;!u[&quot;after:highlightElement&quot;]&amp;&amp;(u[&quot;after:highlightElement&quot;]=function(e){u[&quot;after:highlightBlock&quot;](Object.assign({block:e.el},e))})})(u),r.push(u)}}),e.debugMode=function(){o=!1},e.safeMode=function(){o=!0},e.versionString=&quot;11.7.0&quot;,e.regex={concat:E,lookahead:l,either:d,optional:C,anyNumberOfTimes:F},O)&quot;object&quot;==_typeof(O[_])&amp;&amp;u.exports(O[_]);return Object.assign(e,O),e}({})}();function _typeof(u){return(_typeof=&quot;function&quot;==typeof Symbol&amp;&amp;&quot;symbol&quot;==typeof Symbol.iterator?function(u){return typeof u}:function(u){return u&amp;&amp;&quot;function&quot;==typeof Symbol&amp;&amp;u.constructor===Symbol&amp;&amp;u!==Symbol.prototype?&quot;symbol&quot;:typeof u})(u)}function _classCallCheck(u,e){if(!(u instanceof e))throw new TypeError(&quot;Cannot call a class as a function&quot;)}function _defineProperties(u,e){for(var t=0;t&lt;e.length;t++){var n=e[t];n.enumerable=n.enumerable||!1,n.configurable=!0,&quot;value&quot;in n&amp;&amp;(n.writable=!0),Object.defineProperty(u,_toPropertyKey(n.key),n)}}function _createClass(u,e,t){return e&amp;&amp;_defineProperties(u.prototype,e),t&amp;&amp;_defineProperties(u,t),Object.defineProperty(u,&quot;prototype&quot;,{writable:!1}),u}function _toPropertyKey(u){var e=_toPrimitive(u,&quot;string&quot;);return&quot;symbol&quot;===_typeof(e)?e:String(e)}function _toPrimitive(u,e){if(&quot;object&quot;!==_typeof(u)||null===u)return u;var t=u[Symbol.toPrimitive];if(void 0!==t){var n=t.call(u,e||&quot;default&quot;);if(&quot;object&quot;!==_typeof(n))return n;throw new TypeError(&quot;@@toPrimitive must return a primitive value.&quot;)}return(&quot;string&quot;===e?String:Number)(u)}&quot;object&quot;==(&quot;undefined&quot;==typeof exports?&quot;undefined&quot;:_typeof(exports))&amp;&amp;&quot;undefined&quot;!=typeof module&amp;&amp;(module.exports=hljs),function(){var u=function(u){var e=u.regex,t=/(?:[A-Z_a-z\\xAA\\xB5\\xBA\\xC0-\\xD6\\xD8-\\xF6\\xF8-\\u02C1\\u02C6-\\u02D1\\u02E0-\\u02E4\\u02EC\\u02EE\\u0370-\\u0374\\u0376\\u0377\\u037B-\\u037D\\u037F\\u0386\\u0388-\\u038A\\u038C\\u038E-\\u03A1\\u03A3-\\u03F5\\u03F7-\\u0481\\u048A-\\u052F\\u0531-\\u0556\\u0559\\u0560-\\u0588\\u05D0-\\u05EA\\u05EF-\\u05F2\\u0620-\\u064A\\u066E\\u066F\\u0671-\\u06D3\\u06D5\\u06E5\\u06E6\\u06EE\\u06EF\\u06FA-\\u06FC\\u06FF\\u0710\\u0712-\\u072F\\u074D-\\u07A5\\u07B1\\u07CA-\\u07EA\\u07F4\\u07F5\\u07FA\\u0800-\\u0815\\u081A\\u0824\\u0828\\u0840-\\u0858\\u0860-\\u086A\\u0870-\\u0887\\u0889-\\u088E\\u08A0-\\u08C9\\u0904-\\u0939\\u093D\\u0950\\u0958-\\u0961\\u0971-\\u0980\\u0985-\\u098C\\u098F\\u0990\\u0993-\\u09A8\\u09AA-\\u09B0\\u09B2\\u09B6-\\u09B9\\u09BD\\u09CE\\u09DC\\u09DD\\u09DF-\\u09E1\\u09F0\\u09F1\\u09FC\\u0A05-\\u0A0A\\u0A0F\\u0A10\\u0A13-\\u0A28\\u0A2A-\\u0A30\\u0A32\\u0A33\\u0A35\\u0A36\\u0A38\\u0A39\\u0A59-\\u0A5C\\u0A5E\\u0A72-\\u0A74\\u0A85-\\u0A8D\\u0A8F-\\u0A91\\u0A93-\\u0AA8\\u0AAA-\\u0AB0\\u0AB2\\u0AB3\\u0AB5-\\u0AB9\\u0ABD\\u0AD0\\u0AE0\\u0AE1\\u0AF9\\u0B05-\\u0B0C\\u0B0F\\u0B10\\u0B13-\\u0B28\\u0B2A-\\u0B30\\u0B32\\u0B33\\u0B35-\\u0B39\\u0B3D\\u0B5C\\u0B5D\\u0B5F-\\u0B61\\u0B71\\u0B83\\u0B85-\\u0B8A\\u0B8E-\\u0B90\\u0B92-\\u0B95\\u0B99\\u0B9A\\u0B9C\\u0B9E\\u0B9F\\u0BA3\\u0BA4\\u0BA8-\\u0BAA\\u0BAE-\\u0BB9\\u0BD0\\u0C05-\\u0C0C\\u0C0E-\\u0C10\\u0C12-\\u0C28\\u0C2A-\\u0C39\\u0C3D\\u0C58-\\u0C5A\\u0C5D\\u0C60\\u0C61\\u0C80\\u0C85-\\u0C8C\\u0C8E-\\u0C90\\u0C92-\\u0CA8\\u0CAA-\\u0CB3\\u0CB5-\\u0CB9\\u0CBD\\u0CDD\\u0CDE\\u0CE0\\u0CE1\\u0CF1\\u0CF2\\u0D04-\\u0D0C\\u0D0E-\\u0D10\\u0D12-\\u0D3A\\u0D3D\\u0D4E\\u0D54-\\u0D56\\u0D5F-\\u0D61\\u0D7A-\\u0D7F\\u0D85-\\u0D96\\u0D9A-\\u0DB1\\u0DB3-\\u0DBB\\u0DBD\\u0DC0-\\u0DC6\\u0E01-\\u0E30\\u0E32\\u0E40-\\u0E46\\u0E81\\u0E82\\u0E84\\u0E86-\\u0E8A\\u0E8C-\\u0EA3\\u0EA5\\u0EA7-\\u0EB0\\u0EB2\\u0EBD\\u0EC0-\\u0EC4\\u0EC6\\u0EDC-\\u0EDF\\u0F00\\u0F40-\\u0F47\\u0F49-\\u0F6C\\u0F88-\\u0F8C\\u1000-\\u102A\\u103F\\u1050-\\u1055\\u105A-\\u105D\\u1061\\u1065\\u1066\\u106E-\\u1070\\u1075-\\u1081\\u108E\\u10A0-\\u10C5\\u10C7\\u10CD\\u10D0-\\u10FA\\u10FC-\\u1248\\u124A-\\u124D\\u1250-\\u1256\\u1258\\u125A-\\u125D\\u1260-\\u1288\\u128A-\\u128D\\u1290-\\u12B0\\u12B2-\\u12B5\\u12B8-\\u12BE\\u12C0\\u12C2-\\u12C5\\u12C8-\\u12D6\\u12D8-\\u1310\\u1312-\\u1315\\u1318-\\u135A\\u1380-\\u138F\\u13A0-\\u13F5\\u13F8-\\u13FD\\u1401-\\u166C\\u166F-\\u167F\\u1681-\\u169A\\u16A0-\\u16EA\\u16EE-\\u16F8\\u1700-\\u1711\\u171F-\\u1731\\u1740-\\u1751\\u1760-\\u176C\\u176E-\\u1770\\u1780-\\u17B3\\u17D7\\u17DC\\u1820-\\u1878\\u1880-\\u18A8\\u18AA\\u18B0-\\u18F5\\u1900-\\u191E\\u1950-\\u196D\\u1970-\\u1974\\u1980-\\u19AB\\u19B0-\\u19C9\\u1A00-\\u1A16\\u1A20-\\u1A54\\u1AA7\\u1B05-\\u1B33\\u1B45-\\u1B4C\\u1B83-\\u1BA0\\u1BAE\\u1BAF\\u1BBA-\\u1BE5\\u1C00-\\u1C23\\u1C4D-\\u1C4F\\u1C5A-\\u1C7D\\u1C80-\\u1C88\\u1C90-\\u1CBA\\u1CBD-\\u1CBF\\u1CE9-\\u1CEC\\u1CEE-\\u1CF3\\u1CF5\\u1CF6\\u1CFA\\u1D00-\\u1DBF\\u1E00-\\u1F15\\u1F18-\\u1F1D\\u1F20-\\u1F45\\u1F48-\\u1F4D\\u1F50-\\u1F57\\u1F59\\u1F5B\\u1F5D\\u1F5F-\\u1F7D\\u1F80-\\u1FB4\\u1FB6-\\u1FBC\\u1FBE\\u1FC2-\\u1FC4\\u1FC6-\\u1FCC\\u1FD0-\\u1FD3\\u1FD6-\\u1FDB\\u1FE0-\\u1FEC\\u1FF2-\\u1FF4\\u1FF6-\\u1FFC\\u2071\\u207F\\u2090-\\u209C\\u2102\\u2107\\u210A-\\u2113\\u2115\\u2118-\\u211D\\u2124\\u2126\\u2128\\u212A-\\u2139\\u213C-\\u213F\\u2145-\\u2149\\u214E\\u2160-\\u2188\\u2C00-\\u2CE4\\u2CEB-\\u2CEE\\u2CF2\\u2CF3\\u2D00-\\u2D25\\u2D27\\u2D2D\\u2D30-\\u2D67\\u2D6F\\u2D80-\\u2D96\\u2DA0-\\u2DA6\\u2DA8-\\u2DAE\\u2DB0-\\u2DB6\\u2DB8-\\u2DBE\\u2DC0-\\u2DC6\\u2DC8-\\u2DCE\\u2DD0-\\u2DD6\\u2DD8-\\u2DDE\\u3005-\\u3007\\u3021-\\u3029\\u3031-\\u3035\\u3038-\\u303C\\u3041-\\u3096\\u309D-\\u309F\\u30A1-\\u30FA\\u30FC-\\u30FF\\u3105-\\u312F\\u3131-\\u318E\\u31A0-\\u31BF\\u31F0-\\u31FF\\u3400-\\u4DBF\\u4E00-\\uA48C\\uA4D0-\\uA4FD\\uA500-\\uA60C\\uA610-\\uA61F\\uA62A\\uA62B\\uA640-\\uA66E\\uA67F-\\uA69D\\uA6A0-\\uA6EF\\uA717-\\uA71F\\uA722-\\uA788\\uA78B-\\uA7CA\\uA7D0\\uA7D1\\uA7D3\\uA7D5-\\uA7D9\\uA7F2-\\uA801\\uA803-\\uA805\\uA807-\\uA80A\\uA80C-\\uA822\\uA840-\\uA873\\uA882-\\uA8B3\\uA8F2-\\uA8F7\\uA8FB\\uA8FD\\uA8FE\\uA90A-\\uA925\\uA930-\\uA946\\uA960-\\uA97C\\uA984-\\uA9B2\\uA9CF\\uA9E0-\\uA9E4\\uA9E6-\\uA9EF\\uA9FA-\\uA9FE\\uAA00-\\uAA28\\uAA40-\\uAA42\\uAA44-\\uAA4B\\uAA60-\\uAA76\\uAA7A\\uAA7E-\\uAAAF\\uAAB1\\uAAB5\\uAAB6\\uAAB9-\\uAABD\\uAAC0\\uAAC2\\uAADB-\\uAADD\\uAAE0-\\uAAEA\\uAAF2-\\uAAF4\\uAB01-\\uAB06\\uAB09-\\uAB0E\\uAB11-\\uAB16\\uAB20-\\uAB26\\uAB28-\\uAB2E\\uAB30-\\uAB5A\\uAB5C-\\uAB69\\uAB70-\\uABE2\\uAC00-\\uD7A3\\uD7B0-\\uD7C6\\uD7CB-\\uD7FB\\uF900-\\uFA6D\\uFA70-\\uFAD9\\uFB00-\\uFB06\\uFB13-\\uFB17\\uFB1D\\uFB1F-\\uFB28\\uFB2A-\\uFB36\\uFB38-\\uFB3C\\uFB3E\\uFB40\\uFB41\\uFB43\\uFB44\\uFB46-\\uFBB1\\uFBD3-\\uFC5D\\uFC64-\\uFD3D\\uFD50-\\uFD8F\\uFD92-\\uFDC7\\uFDF0-\\uFDF9\\uFE71\\uFE73\\uFE77\\uFE79\\uFE7B\\uFE7D\\uFE7F-\\uFEFC\\uFF21-\\uFF3A\\uFF41-\\uFF5A\\uFF66-\\uFF9D\\uFFA0-\\uFFBE\\uFFC2-\\uFFC7\\uFFCA-\\uFFCF\\uFFD2-\\uFFD7\\uFFDA-\\uFFDC]|\\uD800[\\uDC00-\\uDC0B\\uDC0D-\\uDC26\\uDC28-\\uDC3A\\uDC3C\\uDC3D\\uDC3F-\\uDC4D\\uDC50-\\uDC5D\\uDC80-\\uDCFA\\uDD40-\\uDD74\\uDE80-\\uDE9C\\uDEA0-\\uDED0\\uDF00-\\uDF1F\\uDF2D-\\uDF4A\\uDF50-\\uDF75\\uDF80-\\uDF9D\\uDFA0-\\uDFC3\\uDFC8-\\uDFCF\\uDFD1-\\uDFD5]|\\uD801[\\uDC00-\\uDC9D\\uDCB0-\\uDCD3\\uDCD8-\\uDCFB\\uDD00-\\uDD27\\uDD30-\\uDD63\\uDD70-\\uDD7A\\uDD7C-\\uDD8A\\uDD8C-\\uDD92\\uDD94\\uDD95\\uDD97-\\uDDA1\\uDDA3-\\uDDB1\\uDDB3-\\uDDB9\\uDDBB\\uDDBC\\uDE00-\\uDF36\\uDF40-\\uDF55\\uDF60-\\uDF67\\uDF80-\\uDF85\\uDF87-\\uDFB0\\uDFB2-\\uDFBA]|\\uD802[\\uDC00-\\uDC05\\uDC08\\uDC0A-\\uDC35\\uDC37\\uDC38\\uDC3C\\uDC3F-\\uDC55\\uDC60-\\uDC76\\uDC80-\\uDC9E\\uDCE0-\\uDCF2\\uDCF4\\uDCF5\\uDD00-\\uDD15\\uDD20-\\uDD39\\uDD80-\\uDDB7\\uDDBE\\uDDBF\\uDE00\\uDE10-\\uDE13\\uDE15-\\uDE17\\uDE19-\\uDE35\\uDE60-\\uDE7C\\uDE80-\\uDE9C\\uDEC0-\\uDEC7\\uDEC9-\\uDEE4\\uDF00-\\uDF35\\uDF40-\\uDF55\\uDF60-\\uDF72\\uDF80-\\uDF91]|\\uD803[\\uDC00-\\uDC48\\uDC80-\\uDCB2\\uDCC0-\\uDCF2\\uDD00-\\uDD23\\uDE80-\\uDEA9\\uDEB0\\uDEB1\\uDF00-\\uDF1C\\uDF27\\uDF30-\\uDF45\\uDF70-\\uDF81\\uDFB0-\\uDFC4\\uDFE0-\\uDFF6]|\\uD804[\\uDC03-\\uDC37\\uDC71\\uDC72\\uDC75\\uDC83-\\uDCAF\\uDCD0-\\uDCE8\\uDD03-\\uDD26\\uDD44\\uDD47\\uDD50-\\uDD72\\uDD76\\uDD83-\\uDDB2\\uDDC1-\\uDDC4\\uDDDA\\uDDDC\\uDE00-\\uDE11\\uDE13-\\uDE2B\\uDE3F\\uDE40\\uDE80-\\uDE86\\uDE88\\uDE8A-\\uDE8D\\uDE8F-\\uDE9D\\uDE9F-\\uDEA8\\uDEB0-\\uDEDE\\uDF05-\\uDF0C\\uDF0F\\uDF10\\uDF13-\\uDF28\\uDF2A-\\uDF30\\uDF32\\uDF33\\uDF35-\\uDF39\\uDF3D\\uDF50\\uDF5D-\\uDF61]|\\uD805[\\uDC00-\\uDC34\\uDC47-\\uDC4A\\uDC5F-\\uDC61\\uDC80-\\uDCAF\\uDCC4\\uDCC5\\uDCC7\\uDD80-\\uDDAE\\uDDD8-\\uDDDB\\uDE00-\\uDE2F\\uDE44\\uDE80-\\uDEAA\\uDEB8\\uDF00-\\uDF1A\\uDF40-\\uDF46]|\\uD806[\\uDC00-\\uDC2B\\uDCA0-\\uDCDF\\uDCFF-\\uDD06\\uDD09\\uDD0C-\\uDD13\\uDD15\\uDD16\\uDD18-\\uDD2F\\uDD3F\\uDD41\\uDDA0-\\uDDA7\\uDDAA-\\uDDD0\\uDDE1\\uDDE3\\uDE00\\uDE0B-\\uDE32\\uDE3A\\uDE50\\uDE5C-\\uDE89\\uDE9D\\uDEB0-\\uDEF8]|\\uD807[\\uDC00-\\uDC08\\uDC0A-\\uDC2E\\uDC40\\uDC72-\\uDC8F\\uDD00-\\uDD06\\uDD08\\uDD09\\uDD0B-\\uDD30\\uDD46\\uDD60-\\uDD65\\uDD67\\uDD68\\uDD6A-\\uDD89\\uDD98\\uDEE0-\\uDEF2\\uDF02\\uDF04-\\uDF10\\uDF12-\\uDF33\\uDFB0]|\\uD808[\\uDC00-\\uDF99]|\\uD809[\\uDC00-\\uDC6E\\uDC80-\\uDD43]|\\uD80B[\\uDF90-\\uDFF0]|[\\uD80C\\uD81C-\\uD820\\uD822\\uD840-\\uD868\\uD86A-\\uD86C\\uD86F-\\uD872\\uD874-\\uD879\\uD880-\\uD883\\uD885-\\uD887][\\uDC00-\\uDFFF]|\\uD80D[\\uDC00-\\uDC2F\\uDC41-\\uDC46]|\\uD811[\\uDC00-\\uDE46]|\\uD81A[\\uDC00-\\uDE38\\uDE40-\\uDE5E\\uDE70-\\uDEBE\\uDED0-\\uDEED\\uDF00-\\uDF2F\\uDF40-\\uDF43\\uDF63-\\uDF77\\uDF7D-\\uDF8F]|\\uD81B[\\uDE40-\\uDE7F\\uDF00-\\uDF4A\\uDF50\\uDF93-\\uDF9F\\uDFE0\\uDFE1\\uDFE3]|\\uD821[\\uDC00-\\uDFF7]|\\uD823[\\uDC00-\\uDCD5\\uDD00-\\uDD08]|\\uD82B[\\uDFF0-\\uDFF3\\uDFF5-\\uDFFB\\uDFFD\\uDFFE]|\\uD82C[\\uDC00-\\uDD22\\uDD32\\uDD50-\\uDD52\\uDD55\\uDD64-\\uDD67\\uDD70-\\uDEFB]|\\uD82F[\\uDC00-\\uDC6A\\uDC70-\\uDC7C\\uDC80-\\uDC88\\uDC90-\\uDC99]|\\uD835[\\uDC00-\\uDC54\\uDC56-\\uDC9C\\uDC9E\\uDC9F\\uDCA2\\uDCA5\\uDCA6\\uDCA9-\\uDCAC\\uDCAE-\\uDCB9\\uDCBB\\uDCBD-\\uDCC3\\uDCC5-\\uDD05\\uDD07-\\uDD0A\\uDD0D-\\uDD14\\uDD16-\\uDD1C\\uDD1E-\\uDD39\\uDD3B-\\uDD3E\\uDD40-\\uDD44\\uDD46\\uDD4A-\\uDD50\\uDD52-\\uDEA5\\uDEA8-\\uDEC0\\uDEC2-\\uDEDA\\uDEDC-\\uDEFA\\uDEFC-\\uDF14\\uDF16-\\uDF34\\uDF36-\\uDF4E\\uDF50-\\uDF6E\\uDF70-\\uDF88\\uDF8A-\\uDFA8\\uDFAA-\\uDFC2\\uDFC4-\\uDFCB]|\\uD837[\\uDF00-\\uDF1E\\uDF25-\\uDF2A]|\\uD838[\\uDC30-\\uDC6D\\uDD00-\\uDD2C\\uDD37-\\uDD3D\\uDD4E\\uDE90-\\uDEAD\\uDEC0-\\uDEEB]|\\uD839[\\uDCD0-\\uDCEB\\uDFE0-\\uDFE6\\uDFE8-\\uDFEB\\uDFED\\uDFEE\\uDFF0-\\uDFFE]|\\uD83A[\\uDC00-\\uDCC4\\uDD00-\\uDD43\\uDD4B]|\\uD83B[\\uDE00-\\uDE03\\uDE05-\\uDE1F\\uDE21\\uDE22\\uDE24\\uDE27\\uDE29-\\uDE32\\uDE34-\\uDE37\\uDE39\\uDE3B\\uDE42\\uDE47\\uDE49\\uDE4B\\uDE4D-\\uDE4F\\uDE51\\uDE52\\uDE54\\uDE57\\uDE59\\uDE5B\\uDE5D\\uDE5F\\uDE61\\uDE62\\uDE64\\uDE67-\\uDE6A\\uDE6C-\\uDE72\\uDE74-\\uDE77\\uDE79-\\uDE7C\\uDE7E\\uDE80-\\uDE89\\uDE8B-\\uDE9B\\uDEA1-\\uDEA3\\uDEA5-\\uDEA9\\uDEAB-\\uDEBB]|\\uD869[\\uDC00-\\uDEDF\\uDF00-\\uDFFF]|\\uD86D[\\uDC00-\\uDF39\\uDF40-\\uDFFF]|\\uD86E[\\uDC00-\\uDC1D\\uDC20-\\uDFFF]|\\uD873[\\uDC00-\\uDEA1\\uDEB0-\\uDFFF]|\\uD87A[\\uDC00-\\uDFE0]|\\uD87E[\\uDC00-\\uDE1D]|\\uD884[\\uDC00-\\uDF4A\\uDF50-\\uDFFF]|\\uD888[\\uDC00-\\uDFAF])(?:[0-9A-Z_a-z\\xAA\\xB5\\xB7\\xBA\\xC0-\\xD6\\xD8-\\xF6\\xF8-\\u02C1\\u02C6-\\u02D1\\u02E0-\\u02E4\\u02EC\\u02EE\\u0300-\\u0374\\u0376\\u0377\\u037B-\\u037D\\u037F\\u0386-\\u038A\\u038C\\u038E-\\u03A1\\u03A3-\\u03F5\\u03F7-\\u0481\\u0483-\\u0487\\u048A-\\u052F\\u0531-\\u0556\\u0559\\u0560-\\u0588\\u0591-\\u05BD\\u05BF\\u05C1\\u05C2\\u05C4\\u05C5\\u05C7\\u05D0-\\u05EA\\u05EF-\\u05F2\\u0610-\\u061A\\u0620-\\u0669\\u066E-\\u06D3\\u06D5-\\u06DC\\u06DF-\\u06E8\\u06EA-\\u06FC\\u06FF\\u0710-\\u074A\\u074D-\\u07B1\\u07C0-\\u07F5\\u07FA\\u07FD\\u0800-\\u082D\\u0840-\\u085B\\u0860-\\u086A\\u0870-\\u0887\\u0889-\\u088E\\u0898-\\u08E1\\u08E3-\\u0963\\u0966-\\u096F\\u0971-\\u0983\\u0985-\\u098C\\u098F\\u0990\\u0993-\\u09A8\\u09AA-\\u09B0\\u09B2\\u09B6-\\u09B9\\u09BC-\\u09C4\\u09C7\\u09C8\\u09CB-\\u09CE\\u09D7\\u09DC\\u09DD\\u09DF-\\u09E3\\u09E6-\\u09F1\\u09FC\\u09FE\\u0A01-\\u0A03\\u0A05-\\u0A0A\\u0A0F\\u0A10\\u0A13-\\u0A28\\u0A2A-\\u0A30\\u0A32\\u0A33\\u0A35\\u0A36\\u0A38\\u0A39\\u0A3C\\u0A3E-\\u0A42\\u0A47\\u0A48\\u0A4B-\\u0A4D\\u0A51\\u0A59-\\u0A5C\\u0A5E\\u0A66-\\u0A75\\u0A81-\\u0A83\\u0A85-\\u0A8D\\u0A8F-\\u0A91\\u0A93-\\u0AA8\\u0AAA-\\u0AB0\\u0AB2\\u0AB3\\u0AB5-\\u0AB9\\u0ABC-\\u0AC5\\u0AC7-\\u0AC9\\u0ACB-\\u0ACD\\u0AD0\\u0AE0-\\u0AE3\\u0AE6-\\u0AEF\\u0AF9-\\u0AFF\\u0B01-\\u0B03\\u0B05-\\u0B0C\\u0B0F\\u0B10\\u0B13-\\u0B28\\u0B2A-\\u0B30\\u0B32\\u0B33\\u0B35-\\u0B39\\u0B3C-\\u0B44\\u0B47\\u0B48\\u0B4B-\\u0B4D\\u0B55-\\u0B57\\u0B5C\\u0B5D\\u0B5F-\\u0B63\\u0B66-\\u0B6F\\u0B71\\u0B82\\u0B83\\u0B85-\\u0B8A\\u0B8E-\\u0B90\\u0B92-\\u0B95\\u0B99\\u0B9A\\u0B9C\\u0B9E\\u0B9F\\u0BA3\\u0BA4\\u0BA8-\\u0BAA\\u0BAE-\\u0BB9\\u0BBE-\\u0BC2\\u0BC6-\\u0BC8\\u0BCA-\\u0BCD\\u0BD0\\u0BD7\\u0BE6-\\u0BEF\\u0C00-\\u0C0C\\u0C0E-\\u0C10\\u0C12-\\u0C28\\u0C2A-\\u0C39\\u0C3C-\\u0C44\\u0C46-\\u0C48\\u0C4A-\\u0C4D\\u0C55\\u0C56\\u0C58-\\u0C5A\\u0C5D\\u0C60-\\u0C63\\u0C66-\\u0C6F\\u0C80-\\u0C83\\u0C85-\\u0C8C\\u0C8E-\\u0C90\\u0C92-\\u0CA8\\u0CAA-\\u0CB3\\u0CB5-\\u0CB9\\u0CBC-\\u0CC4\\u0CC6-\\u0CC8\\u0CCA-\\u0CCD\\u0CD5\\u0CD6\\u0CDD\\u0CDE\\u0CE0-\\u0CE3\\u0CE6-\\u0CEF\\u0CF1-\\u0CF3\\u0D00-\\u0D0C\\u0D0E-\\u0D10\\u0D12-\\u0D44\\u0D46-\\u0D48\\u0D4A-\\u0D4E\\u0D54-\\u0D57\\u0D5F-\\u0D63\\u0D66-\\u0D6F\\u0D7A-\\u0D7F\\u0D81-\\u0D83\\u0D85-\\u0D96\\u0D9A-\\u0DB1\\u0DB3-\\u0DBB\\u0DBD\\u0DC0-\\u0DC6\\u0DCA\\u0DCF-\\u0DD4\\u0DD6\\u0DD8-\\u0DDF\\u0DE6-\\u0DEF\\u0DF2\\u0DF3\\u0E01-\\u0E3A\\u0E40-\\u0E4E\\u0E50-\\u0E59\\u0E81\\u0E82\\u0E84\\u0E86-\\u0E8A\\u0E8C-\\u0EA3\\u0EA5\\u0EA7-\\u0EBD\\u0EC0-\\u0EC4\\u0EC6\\u0EC8-\\u0ECE\\u0ED0-\\u0ED9\\u0EDC-\\u0EDF\\u0F00\\u0F18\\u0F19\\u0F20-\\u0F29\\u0F35\\u0F37\\u0F39\\u0F3E-\\u0F47\\u0F49-\\u0F6C\\u0F71-\\u0F84\\u0F86-\\u0F97\\u0F99-\\u0FBC\\u0FC6\\u1000-\\u1049\\u1050-\\u109D\\u10A0-\\u10C5\\u10C7\\u10CD\\u10D0-\\u10FA\\u10FC-\\u1248\\u124A-\\u124D\\u1250-\\u1256\\u1258\\u125A-\\u125D\\u1260-\\u1288\\u128A-\\u128D\\u1290-\\u12B0\\u12B2-\\u12B5\\u12B8-\\u12BE\\u12C0\\u12C2-\\u12C5\\u12C8-\\u12D6\\u12D8-\\u1310\\u1312-\\u1315\\u1318-\\u135A\\u135D-\\u135F\\u1369-\\u1371\\u1380-\\u138F\\u13A0-\\u13F5\\u13F8-\\u13FD\\u1401-\\u166C\\u166F-\\u167F\\u1681-\\u169A\\u16A0-\\u16EA\\u16EE-\\u16F8\\u1700-\\u1715\\u171F-\\u1734\\u1740-\\u1753\\u1760-\\u176C\\u176E-\\u1770\\u1772\\u1773\\u1780-\\u17D3\\u17D7\\u17DC\\u17DD\\u17E0-\\u17E9\\u180B-\\u180D\\u180F-\\u1819\\u1820-\\u1878\\u1880-\\u18AA\\u18B0-\\u18F5\\u1900-\\u191E\\u1920-\\u192B\\u1930-\\u193B\\u1946-\\u196D\\u1970-\\u1974\\u1980-\\u19AB\\u19B0-\\u19C9\\u19D0-\\u19DA\\u1A00-\\u1A1B\\u1A20-\\u1A5E\\u1A60-\\u1A7C\\u1A7F-\\u1A89\\u1A90-\\u1A99\\u1AA7\\u1AB0-\\u1ABD\\u1ABF-\\u1ACE\\u1B00-\\u1B4C\\u1B50-\\u1B59\\u1B6B-\\u1B73\\u1B80-\\u1BF3\\u1C00-\\u1C37\\u1C40-\\u1C49\\u1C4D-\\u1C7D\\u1C80-\\u1C88\\u1C90-\\u1CBA\\u1CBD-\\u1CBF\\u1CD0-\\u1CD2\\u1CD4-\\u1CFA\\u1D00-\\u1F15\\u1F18-\\u1F1D\\u1F20-\\u1F45\\u1F48-\\u1F4D\\u1F50-\\u1F57\\u1F59\\u1F5B\\u1F5D\\u1F5F-\\u1F7D\\u1F80-\\u1FB4\\u1FB6-\\u1FBC\\u1FBE\\u1FC2-\\u1FC4\\u1FC6-\\u1FCC\\u1FD0-\\u1FD3\\u1FD6-\\u1FDB\\u1FE0-\\u1FEC\\u1FF2-\\u1FF4\\u1FF6-\\u1FFC\\u203F\\u2040\\u2054\\u2071\\u207F\\u2090-\\u209C\\u20D0-\\u20DC\\u20E1\\u20E5-\\u20F0\\u2102\\u2107\\u210A-\\u2113\\u2115\\u2118-\\u211D\\u2124\\u2126\\u2128\\u212A-\\u2139\\u213C-\\u213F\\u2145-\\u2149\\u214E\\u2160-\\u2188\\u2C00-\\u2CE4\\u2CEB-\\u2CF3\\u2D00-\\u2D25\\u2D27\\u2D2D\\u2D30-\\u2D67\\u2D6F\\u2D7F-\\u2D96\\u2DA0-\\u2DA6\\u2DA8-\\u2DAE\\u2DB0-\\u2DB6\\u2DB8-\\u2DBE\\u2DC0-\\u2DC6\\u2DC8-\\u2DCE\\u2DD0-\\u2DD6\\u2DD8-\\u2DDE\\u2DE0-\\u2DFF\\u3005-\\u3007\\u3021-\\u302F\\u3031-\\u3035\\u3038-\\u303C\\u3041-\\u3096\\u3099\\u309A\\u309D-\\u309F\\u30A1-\\u30FA\\u30FC-\\u30FF\\u3105-\\u312F\\u3131-\\u318E\\u31A0-\\u31BF\\u31F0-\\u31FF\\u3400-\\u4DBF\\u4E00-\\uA48C\\uA4D0-\\uA4FD\\uA500-\\uA60C\\uA610-\\uA62B\\uA640-\\uA66F\\uA674-\\uA67D\\uA67F-\\uA6F1\\uA717-\\uA71F\\uA722-\\uA788\\uA78B-\\uA7CA\\uA7D0\\uA7D1\\uA7D3\\uA7D5-\\uA7D9\\uA7F2-\\uA827\\uA82C\\uA840-\\uA873\\uA880-\\uA8C5\\uA8D0-\\uA8D9\\uA8E0-\\uA8F7\\uA8FB\\uA8FD-\\uA92D\\uA930-\\uA953\\uA960-\\uA97C\\uA980-\\uA9C0\\uA9CF-\\uA9D9\\uA9E0-\\uA9FE\\uAA00-\\uAA36\\uAA40-\\uAA4D\\uAA50-\\uAA59\\uAA60-\\uAA76\\uAA7A-\\uAAC2\\uAADB-\\uAADD\\uAAE0-\\uAAEF\\uAAF2-\\uAAF6\\uAB01-\\uAB06\\uAB09-\\uAB0E\\uAB11-\\uAB16\\uAB20-\\uAB26\\uAB28-\\uAB2E\\uAB30-\\uAB5A\\uAB5C-\\uAB69\\uAB70-\\uABEA\\uABEC\\uABED\\uABF0-\\uABF9\\uAC00-\\uD7A3\\uD7B0-\\uD7C6\\uD7CB-\\uD7FB\\uF900-\\uFA6D\\uFA70-\\uFAD9\\uFB00-\\uFB06\\uFB13-\\uFB17\\uFB1D-\\uFB28\\uFB2A-\\uFB36\\uFB38-\\uFB3C\\uFB3E\\uFB40\\uFB41\\uFB43\\uFB44\\uFB46-\\uFBB1\\uFBD3-\\uFC5D\\uFC64-\\uFD3D\\uFD50-\\uFD8F\\uFD92-\\uFDC7\\uFDF0-\\uFDF9\\uFE00-\\uFE0F\\uFE20-\\uFE2F\\uFE33\\uFE34\\uFE4D-\\uFE4F\\uFE71\\uFE73\\uFE77\\uFE79\\uFE7B\\uFE7D\\uFE7F-\\uFEFC\\uFF10-\\uFF19\\uFF21-\\uFF3A\\uFF3F\\uFF41-\\uFF5A\\uFF66-\\uFFBE\\uFFC2-\\uFFC7\\uFFCA-\\uFFCF\\uFFD2-\\uFFD7\\uFFDA-\\uFFDC]|\\uD800[\\uDC00-\\uDC0B\\uDC0D-\\uDC26\\uDC28-\\uDC3A\\uDC3C\\uDC3D\\uDC3F-\\uDC4D\\uDC50-\\uDC5D\\uDC80-\\uDCFA\\uDD40-\\uDD74\\uDDFD\\uDE80-\\uDE9C\\uDEA0-\\uDED0\\uDEE0\\uDF00-\\uDF1F\\uDF2D-\\uDF4A\\uDF50-\\uDF7A\\uDF80-\\uDF9D\\uDFA0-\\uDFC3\\uDFC8-\\uDFCF\\uDFD1-\\uDFD5]|\\uD801[\\uDC00-\\uDC9D\\uDCA0-\\uDCA9\\uDCB0-\\uDCD3\\uDCD8-\\uDCFB\\uDD00-\\uDD27\\uDD30-\\uDD63\\uDD70-\\uDD7A\\uDD7C-\\uDD8A\\uDD8C-\\uDD92\\uDD94\\uDD95\\uDD97-\\uDDA1\\uDDA3-\\uDDB1\\uDDB3-\\uDDB9\\uDDBB\\uDDBC\\uDE00-\\uDF36\\uDF40-\\uDF55\\uDF60-\\uDF67\\uDF80-\\uDF85\\uDF87-\\uDFB0\\uDFB2-\\uDFBA]|\\uD802[\\uDC00-\\uDC05\\uDC08\\uDC0A-\\uDC35\\uDC37\\uDC38\\uDC3C\\uDC3F-\\uDC55\\uDC60-\\uDC76\\uDC80-\\uDC9E\\uDCE0-\\uDCF2\\uDCF4\\uDCF5\\uDD00-\\uDD15\\uDD20-\\uDD39\\uDD80-\\uDDB7\\uDDBE\\uDDBF\\uDE00-\\uDE03\\uDE05\\uDE06\\uDE0C-\\uDE13\\uDE15-\\uDE17\\uDE19-\\uDE35\\uDE38-\\uDE3A\\uDE3F\\uDE60-\\uDE7C\\uDE80-\\uDE9C\\uDEC0-\\uDEC7\\uDEC9-\\uDEE6\\uDF00-\\uDF35\\uDF40-\\uDF55\\uDF60-\\uDF72\\uDF80-\\uDF91]|\\uD803[\\uDC00-\\uDC48\\uDC80-\\uDCB2\\uDCC0-\\uDCF2\\uDD00-\\uDD27\\uDD30-\\uDD39\\uDE80-\\uDEA9\\uDEAB\\uDEAC\\uDEB0\\uDEB1\\uDEFD-\\uDF1C\\uDF27\\uDF30-\\uDF50\\uDF70-\\uDF85\\uDFB0-\\uDFC4\\uDFE0-\\uDFF6]|\\uD804[\\uDC00-\\uDC46\\uDC66-\\uDC75\\uDC7F-\\uDCBA\\uDCC2\\uDCD0-\\uDCE8\\uDCF0-\\uDCF9\\uDD00-\\uDD34\\uDD36-\\uDD3F\\uDD44-\\uDD47\\uDD50-\\uDD73\\uDD76\\uDD80-\\uDDC4\\uDDC9-\\uDDCC\\uDDCE-\\uDDDA\\uDDDC\\uDE00-\\uDE11\\uDE13-\\uDE37\\uDE3E-\\uDE41\\uDE80-\\uDE86\\uDE88\\uDE8A-\\uDE8D\\uDE8F-\\uDE9D\\uDE9F-\\uDEA8\\uDEB0-\\uDEEA\\uDEF0-\\uDEF9\\uDF00-\\uDF03\\uDF05-\\uDF0C\\uDF0F\\uDF10\\uDF13-\\uDF28\\uDF2A-\\uDF30\\uDF32\\uDF33\\uDF35-\\uDF39\\uDF3B-\\uDF44\\uDF47\\uDF48\\uDF4B-\\uDF4D\\uDF50\\uDF57\\uDF5D-\\uDF63\\uDF66-\\uDF6C\\uDF70-\\uDF74]|\\uD805[\\uDC00-\\uDC4A\\uDC50-\\uDC59\\uDC5E-\\uDC61\\uDC80-\\uDCC5\\uDCC7\\uDCD0-\\uDCD9\\uDD80-\\uDDB5\\uDDB8-\\uDDC0\\uDDD8-\\uDDDD\\uDE00-\\uDE40\\uDE44\\uDE50-\\uDE59\\uDE80-\\uDEB8\\uDEC0-\\uDEC9\\uDF00-\\uDF1A\\uDF1D-\\uDF2B\\uDF30-\\uDF39\\uDF40-\\uDF46]|\\uD806[\\uDC00-\\uDC3A\\uDCA0-\\uDCE9\\uDCFF-\\uDD06\\uDD09\\uDD0C-\\uDD13\\uDD15\\uDD16\\uDD18-\\uDD35\\uDD37\\uDD38\\uDD3B-\\uDD43\\uDD50-\\uDD59\\uDDA0-\\uDDA7\\uDDAA-\\uDDD7\\uDDDA-\\uDDE1\\uDDE3\\uDDE4\\uDE00-\\uDE3E\\uDE47\\uDE50-\\uDE99\\uDE9D\\uDEB0-\\uDEF8]|\\uD807[\\uDC00-\\uDC08\\uDC0A-\\uDC36\\uDC38-\\uDC40\\uDC50-\\uDC59\\uDC72-\\uDC8F\\uDC92-\\uDCA7\\uDCA9-\\uDCB6\\uDD00-\\uDD06\\uDD08\\uDD09\\uDD0B-\\uDD36\\uDD3A\\uDD3C\\uDD3D\\uDD3F-\\uDD47\\uDD50-\\uDD59\\uDD60-\\uDD65\\uDD67\\uDD68\\uDD6A-\\uDD8E\\uDD90\\uDD91\\uDD93-\\uDD98\\uDDA0-\\uDDA9\\uDEE0-\\uDEF6\\uDF00-\\uDF10\\uDF12-\\uDF3A\\uDF3E-\\uDF42\\uDF50-\\uDF59\\uDFB0]|\\uD808[\\uDC00-\\uDF99]|\\uD809[\\uDC00-\\uDC6E\\uDC80-\\uDD43]|\\uD80B[\\uDF90-\\uDFF0]|[\\uD80C\\uD81C-\\uD820\\uD822\\uD840-\\uD868\\uD86A-\\uD86C\\uD86F-\\uD872\\uD874-\\uD879\\uD880-\\uD883\\uD885-\\uD887][\\uDC00-\\uDFFF]|\\uD80D[\\uDC00-\\uDC2F\\uDC40-\\uDC55]|\\uD811[\\uDC00-\\uDE46]|\\uD81A[\\uDC00-\\uDE38\\uDE40-\\uDE5E\\uDE60-\\uDE69\\uDE70-\\uDEBE\\uDEC0-\\uDEC9\\uDED0-\\uDEED\\uDEF0-\\uDEF4\\uDF00-\\uDF36\\uDF40-\\uDF43\\uDF50-\\uDF59\\uDF63-\\uDF77\\uDF7D-\\uDF8F]|\\uD81B[\\uDE40-\\uDE7F\\uDF00-\\uDF4A\\uDF4F-\\uDF87\\uDF8F-\\uDF9F\\uDFE0\\uDFE1\\uDFE3\\uDFE4\\uDFF0\\uDFF1]|\\uD821[\\uDC00-\\uDFF7]|\\uD823[\\uDC00-\\uDCD5\\uDD00-\\uDD08]|\\uD82B[\\uDFF0-\\uDFF3\\uDFF5-\\uDFFB\\uDFFD\\uDFFE]|\\uD82C[\\uDC00-\\uDD22\\uDD32\\uDD50-\\uDD52\\uDD55\\uDD64-\\uDD67\\uDD70-\\uDEFB]|\\uD82F[\\uDC00-\\uDC6A\\uDC70-\\uDC7C\\uDC80-\\uDC88\\uDC90-\\uDC99\\uDC9D\\uDC9E]|\\uD833[\\uDF00-\\uDF2D\\uDF30-\\uDF46]|\\uD834[\\uDD65-\\uDD69\\uDD6D-\\uDD72\\uDD7B-\\uDD82\\uDD85-\\uDD8B\\uDDAA-\\uDDAD\\uDE42-\\uDE44]|\\uD835[\\uDC00-\\uDC54\\uDC56-\\uDC9C\\uDC9E\\uDC9F\\uDCA2\\uDCA5\\uDCA6\\uDCA9-\\uDCAC\\uDCAE-\\uDCB9\\uDCBB\\uDCBD-\\uDCC3\\uDCC5-\\uDD05\\uDD07-\\uDD0A\\uDD0D-\\uDD14\\uDD16-\\uDD1C\\uDD1E-\\uDD39\\uDD3B-\\uDD3E\\uDD40-\\uDD44\\uDD46\\uDD4A-\\uDD50\\uDD52-\\uDEA5\\uDEA8-\\uDEC0\\uDEC2-\\uDEDA\\uDEDC-\\uDEFA\\uDEFC-\\uDF14\\uDF16-\\uDF34\\uDF36-\\uDF4E\\uDF50-\\uDF6E\\uDF70-\\uDF88\\uDF8A-\\uDFA8\\uDFAA-\\uDFC2\\uDFC4-\\uDFCB\\uDFCE-\\uDFFF]|\\uD836[\\uDE00-\\uDE36\\uDE3B-\\uDE6C\\uDE75\\uDE84\\uDE9B-\\uDE9F\\uDEA1-\\uDEAF]|\\uD837[\\uDF00-\\uDF1E\\uDF25-\\uDF2A]|\\uD838[\\uDC00-\\uDC06\\uDC08-\\uDC18\\uDC1B-\\uDC21\\uDC23\\uDC24\\uDC26-\\uDC2A\\uDC30-\\uDC6D\\uDC8F\\uDD00-\\uDD2C\\uDD30-\\uDD3D\\uDD40-\\uDD49\\uDD4E\\uDE90-\\uDEAE\\uDEC0-\\uDEF9]|\\uD839[\\uDCD0-\\uDCF9\\uDFE0-\\uDFE6\\uDFE8-\\uDFEB\\uDFED\\uDFEE\\uDFF0-\\uDFFE]|\\uD83A[\\uDC00-\\uDCC4\\uDCD0-\\uDCD6\\uDD00-\\uDD4B\\uDD50-\\uDD59]|\\uD83B[\\uDE00-\\uDE03\\uDE05-\\uDE1F\\uDE21\\uDE22\\uDE24\\uDE27\\uDE29-\\uDE32\\uDE34-\\uDE37\\uDE39\\uDE3B\\uDE42\\uDE47\\uDE49\\uDE4B\\uDE4D-\\uDE4F\\uDE51\\uDE52\\uDE54\\uDE57\\uDE59\\uDE5B\\uDE5D\\uDE5F\\uDE61\\uDE62\\uDE64\\uDE67-\\uDE6A\\uDE6C-\\uDE72\\uDE74-\\uDE77\\uDE79-\\uDE7C\\uDE7E\\uDE80-\\uDE89\\uDE8B-\\uDE9B\\uDEA1-\\uDEA3\\uDEA5-\\uDEA9\\uDEAB-\\uDEBB]|\\uD83E[\\uDFF0-\\uDFF9]|\\uD869[\\uDC00-\\uDEDF\\uDF00-\\uDFFF]|\\uD86D[\\uDC00-\\uDF39\\uDF40-\\uDFFF]|\\uD86E[\\uDC00-\\uDC1D\\uDC20-\\uDFFF]|\\uD873[\\uDC00-\\uDEA1\\uDEB0-\\uDFFF]|\\uD87A[\\uDC00-\\uDFE0]|\\uD87E[\\uDC00-\\uDE1D]|\\uD884[\\uDC00-\\uDF4A\\uDF50-\\uDFFF]|\\uD888[\\uDC00-\\uDFAF]|\\uDB40[\\uDD00-\\uDDEF])*/,n=[&quot;and&quot;,&quot;as&quot;,&quot;assert&quot;,&quot;async&quot;,&quot;await&quot;,&quot;break&quot;,&quot;case&quot;,&quot;class&quot;,&quot;continue&quot;,&quot;def&quot;,&quot;del&quot;,&quot;elif&quot;,&quot;else&quot;,&quot;except&quot;,&quot;finally&quot;,&quot;for&quot;,&quot;from&quot;,&quot;global&quot;,&quot;if&quot;,&quot;import&quot;,&quot;in&quot;,&quot;is&quot;,&quot;lambda&quot;,&quot;match&quot;,&quot;nonlocal|10&quot;,&quot;not&quot;,&quot;or&quot;,&quot;pass&quot;,&quot;raise&quot;,&quot;return&quot;,&quot;try&quot;,&quot;while&quot;,&quot;with&quot;,&quot;yield&quot;],D={$pattern:/[A-Za-z]\\w+|__\\w+__/,keyword:n,built_in:[&quot;__import__&quot;,&quot;abs&quot;,&quot;all&quot;,&quot;any&quot;,&quot;ascii&quot;,&quot;bin&quot;,&quot;bool&quot;,&quot;breakpoint&quot;,&quot;bytearray&quot;,&quot;bytes&quot;,&quot;callable&quot;,&quot;chr&quot;,&quot;classmethod&quot;,&quot;compile&quot;,&quot;complex&quot;,&quot;delattr&quot;,&quot;dict&quot;,&quot;dir&quot;,&quot;divmod&quot;,&quot;enumerate&quot;,&quot;eval&quot;,&quot;exec&quot;,&quot;filter&quot;,&quot;float&quot;,&quot;format&quot;,&quot;frozenset&quot;,&quot;getattr&quot;,&quot;globals&quot;,&quot;hasattr&quot;,&quot;hash&quot;,&quot;help&quot;,&quot;hex&quot;,&quot;id&quot;,&quot;input&quot;,&quot;int&quot;,&quot;isinstance&quot;,&quot;issubclass&quot;,&quot;iter&quot;,&quot;len&quot;,&quot;list&quot;,&quot;locals&quot;,&quot;map&quot;,&quot;max&quot;,&quot;memoryview&quot;,&quot;min&quot;,&quot;next&quot;,&quot;object&quot;,&quot;oct&quot;,&quot;open&quot;,&quot;ord&quot;,&quot;pow&quot;,&quot;print&quot;,&quot;property&quot;,&quot;range&quot;,&quot;repr&quot;,&quot;reversed&quot;,&quot;round&quot;,&quot;set&quot;,&quot;setattr&quot;,&quot;slice&quot;,&quot;sorted&quot;,&quot;staticmethod&quot;,&quot;str&quot;,&quot;sum&quot;,&quot;super&quot;,&quot;tuple&quot;,&quot;type&quot;,&quot;vars&quot;,&quot;zip&quot;],literal:[&quot;__debug__&quot;,&quot;Ellipsis&quot;,&quot;False&quot;,&quot;None&quot;,&quot;NotImplemented&quot;,&quot;True&quot;],type:[&quot;Any&quot;,&quot;Callable&quot;,&quot;Coroutine&quot;,&quot;Dict&quot;,&quot;List&quot;,&quot;Literal&quot;,&quot;Generic&quot;,&quot;Optional&quot;,&quot;Sequence&quot;,&quot;Set&quot;,&quot;Tuple&quot;,&quot;Type&quot;,&quot;Union&quot;]},r={className:&quot;meta&quot;,begin:/^(&gt;&gt;&gt;|\\.\\.\\.) /},o={className:&quot;subst&quot;,begin:/\\{/,end:/\\}/,keywords:D,illegal:/#/},i={begin:/\\{\\{/,relevance:0},a={className:&quot;string&quot;,contains:[u.BACKSLASH_ESCAPE],variants:[{begin:/([uU]|[bB]|[rR]|[bB][rR]|[rR][bB])?&#x27;&#x27;&#x27;/,end:/&#x27;&#x27;&#x27;/,contains:[u.BACKSLASH_ESCAPE,r],relevance:10},{begin:/([uU]|[bB]|[rR]|[bB][rR]|[rR][bB])?&quot;&quot;&quot;/,end:/&quot;&quot;&quot;/,contains:[u.BACKSLASH_ESCAPE,r],relevance:10},{begin:/([fF][rR]|[rR][fF]|[fF])&#x27;&#x27;&#x27;/,end:/&#x27;&#x27;&#x27;/,contains:[u.BACKSLASH_ESCAPE,r,i,o]},{begin:/([fF][rR]|[rR][fF]|[fF])&quot;&quot;&quot;/,end:/&quot;&quot;&quot;/,contains:[u.BACKSLASH_ESCAPE,r,i,o]},{begin:/([uU]|[rR])&#x27;/,end:/&#x27;/,relevance:10},{begin:/([uU]|[rR])&quot;/,end:/&quot;/,relevance:10},{begin:/([bB]|[bB][rR]|[rR][bB])&#x27;/,end:/&#x27;/},{begin:/([bB]|[bB][rR]|[rR][bB])&quot;/,end:/&quot;/},{begin:/([fF][rR]|[rR][fF]|[fF])&#x27;/,end:/&#x27;/,contains:[u.BACKSLASH_ESCAPE,i,o]},{begin:/([fF][rR]|[rR][fF]|[fF])&quot;/,end:/&quot;/,contains:[u.BACKSLASH_ESCAPE,i,o]},u.APOS_STRING_MODE,u.QUOTE_STRING_MODE]},c=&quot;[0-9](_?[0-9])*&quot;,s=&quot;(\\\\b(&quot;.concat(c,&quot;))?\\\\.(&quot;).concat(c,&quot;)|\\\\b(&quot;).concat(c,&quot;)\\\\.&quot;),l=&quot;\\\\b|&quot;+n.join(&quot;|&quot;),F={className:&quot;number&quot;,relevance:0,variants:[{begin:&quot;(\\\\b(&quot;.concat(c,&quot;)|(&quot;).concat(s,&quot;))[eE][+-]?(&quot;).concat(c,&quot;)[jJ]?(?=&quot;).concat(l,&quot;)&quot;)},{begin:&quot;(&quot;.concat(s,&quot;)[jJ]?&quot;)},{begin:&quot;\\\\b([1-9](_?[0-9])*|0+(_?0)*)[lLjJ]?(?=&quot;.concat(l,&quot;)&quot;)},{begin:&quot;\\\\b0[bB](_?[01])+[lL]?(?=&quot;.concat(l,&quot;)&quot;)},{begin:&quot;\\\\b0[oO](_?[0-7])+[lL]?(?=&quot;.concat(l,&quot;)&quot;)},{begin:&quot;\\\\b0[xX](_?[0-9a-fA-F])+[lL]?(?=&quot;.concat(l,&quot;)&quot;)},{begin:&quot;\\\\b(&quot;.concat(c,&quot;)[jJ](?=&quot;).concat(l,&quot;)&quot;)}]},C={className:&quot;comment&quot;,begin:e.lookahead(/# type:/),end:/$/,keywords:D,contains:[{begin:/# type:/},{begin:/#/,end:/\\b\\B/,endsWithParent:!0}]},E={className:&quot;params&quot;,variants:[{className:&quot;&quot;,begin:/\\(\\s*\\)/,skip:!0},{begin:/\\(/,end:/\\)/,excludeBegin:!0,excludeEnd:!0,keywords:D,contains:[&quot;self&quot;,r,F,a,u.HASH_COMMENT_MODE]}]};return o.contains=[a,F,r],{name:&quot;Python&quot;,aliases:[&quot;py&quot;,&quot;gyp&quot;,&quot;ipython&quot;],unicodeRegex:!0,keywords:D,illegal:/(&lt;\\/|-&gt;|\\?)|=&gt;/,contains:[r,F,{begin:/\\bself\\b/},{beginKeywords:&quot;if&quot;,relevance:0},a,C,u.HASH_COMMENT_MODE,{match:[/\\bdef/,/\\s+/,t],scope:{1:&quot;keyword&quot;,3:&quot;title.function&quot;},contains:[E]},{variants:[{match:[/\\bclass/,/\\s+/,t,/\\s*/,/\\(\\s*/,t,/\\s*\\)/]},{match:[/\\bclass/,/\\s+/,t]}],scope:{1:&quot;keyword&quot;,3:&quot;title.class&quot;,6:&quot;title.class.inherited&quot;}},{className:&quot;meta&quot;,begin:/^[\\t ]*@/,end:/(?=#)|$/,contains:[F,E,a]}]}};hljs.registerLanguage(&quot;python&quot;,u)}();var CopyButtonPlugin=function(){function u(){var e=arguments.length&gt;0&amp;&amp;void 0!==arguments[0]?arguments[0]:{};_classCallCheck(this,u),self.hook=e.hook,self.callback=e.callback}return _createClass(u,[{key:&quot;after:highlightElement&quot;,value:function(u){var e=u.el,t=u.text,n=Object.assign(document.createElement(&quot;button&quot;),{innerHTML:&quot;Copy&quot;,className:&quot;hljs-copy-button&quot;});n.dataset.copied=!1,e.parentElement.classList.add(&quot;hljs-copy-wrapper&quot;),e.parentElement.appendChild(n),e.parentElement.style.setProperty(&quot;--hljs-theme-background&quot;,window.getComputedStyle(e).backgroundColor),n.onclick=function(){if(navigator.clipboard){var u=t;hook&amp;&amp;&quot;function&quot;==typeof hook&amp;&amp;(u=hook(t,e)||t),navigator.clipboard.writeText(u).then(function(){n.innerHTML=&quot;Â Copied!Â &quot;,n.dataset.copied=!0;var u=Object.assign(document.createElement(&quot;div&quot;),{role:&quot;status&quot;,className:&quot;hljs-copy-alert&quot;,innerHTML:&quot;Copied to clipboard&quot;});e.parentElement.appendChild(u),setTimeout(function(){n.innerHTML=&quot;Copy&quot;,n.dataset.copied=!1,e.parentElement.removeChild(u),u=null},2e3)}).then(function(){if(&quot;function&quot;==typeof callback)return callback(u,e)})}}}}]),u}();!function(){hljs.addPlugin(new CopyButtonPlugin),hljs.highlightAll(),document.querySelectorAll(&quot;#gsk-scan .gsk-issue&quot;).forEach(function(u){u.addEventListener(&quot;click&quot;,function(e){e.preventDefault(),u.classList.toggle(&quot;open&quot;),u.classList.toggle(&quot;bg-zinc-700&quot;)})});var u=document.querySelectorAll(&quot;#gsk-scan [role=&#x27;tabpanel&#x27;]&quot;),e=document.querySelectorAll(&quot;#gsk-scan [data-tab-target]&quot;);e.forEach(function(t){t.addEventListener(&quot;click&quot;,function(n){n.preventDefault();var D=t.getAttribute(&quot;data-tab-target&quot;);u.forEach(function(u){u.classList.add(&quot;hidden&quot;)}),e.forEach(function(u){u.classList.remove(&quot;active&quot;)}),t.classList.add(&quot;active&quot;),document.getElementById(D).classList.remove(&quot;hidden&quot;)})})}();\n",
       "&lt;/script&gt;\n",
       "\n",
       "&lt;/body&gt;\n",
       "\n",
       "&lt;/html&gt;\" style=\"width: 100%; border: none;\" class=\"gsk-scan\"></iframe>\n",
       "<script>\n",
       "\"use strict\";\n",
       "\n",
       "function _typeof(obj) { \"@babel/helpers - typeof\"; return _typeof = \"function\" == typeof Symbol && \"symbol\" == typeof Symbol.iterator ? function (obj) { return typeof obj; } : function (obj) { return obj && \"function\" == typeof Symbol && obj.constructor === Symbol && obj !== Symbol.prototype ? \"symbol\" : typeof obj; }, _typeof(obj); }\n",
       "/*! iFrame Resizer (iframeSizer.min.js ) - v4.3.5 - 2023-03-08\n",
       " *  Desc: Force cross domain iframes to size to content.\n",
       " *  Requires: iframeResizer.contentWindow.min.js to be loaded into the target frame.\n",
       " *  Copyright: (c) 2023 David J. Bradshaw - dave@bradshaw.net\n",
       " *  License: MIT\n",
       " */\n",
       "!function (d) {\n",
       "  var c, u, a, v, x, I, M, r, f, k, i, l, z;\n",
       "  function m() {\n",
       "    return window.MutationObserver || window.WebKitMutationObserver || window.MozMutationObserver;\n",
       "  }\n",
       "  function F(e, n, i) {\n",
       "    e.addEventListener(n, i, !1);\n",
       "  }\n",
       "  function B(e, n, i) {\n",
       "    e.removeEventListener(n, i, !1);\n",
       "  }\n",
       "  function p(e) {\n",
       "    return x + \"[\" + (n = \"Host page: \" + (e = e), n = window.top !== window.self ? window.parentIFrame && window.parentIFrame.getId ? window.parentIFrame.getId() + \": \" + e : \"Nested host page: \" + e : n) + \"]\";\n",
       "    var n;\n",
       "  }\n",
       "  function t(e) {\n",
       "    return k[e] ? k[e].log : u;\n",
       "  }\n",
       "  function O(e, n) {\n",
       "    o(\"log\", e, n, t(e));\n",
       "  }\n",
       "  function E(e, n) {\n",
       "    o(\"info\", e, n, t(e));\n",
       "  }\n",
       "  function R(e, n) {\n",
       "    o(\"warn\", e, n, !0);\n",
       "  }\n",
       "  function o(e, n, i, t) {\n",
       "    !0 === t && \"object\" == _typeof(window.console) && console[e](p(n), i);\n",
       "  }\n",
       "  function w(e) {\n",
       "    function i() {\n",
       "      t(\"Height\"), t(\"Width\"), P(function () {\n",
       "        H(w), C(b), l(\"onResized\", w);\n",
       "      }, w, \"init\");\n",
       "    }\n",
       "    function n() {\n",
       "      var e = p.slice(I).split(\":\"),\n",
       "        n = e[1] ? parseInt(e[1], 10) : 0,\n",
       "        i = k[e[0]] && k[e[0]].iframe,\n",
       "        t = getComputedStyle(i);\n",
       "      return {\n",
       "        iframe: i,\n",
       "        id: e[0],\n",
       "        height: n + function (e) {\n",
       "          if (\"border-box\" !== e.boxSizing) return 0;\n",
       "          var n = e.paddingTop ? parseInt(e.paddingTop, 10) : 0,\n",
       "            e = e.paddingBottom ? parseInt(e.paddingBottom, 10) : 0;\n",
       "          return n + e;\n",
       "        }(t) + function (e) {\n",
       "          if (\"border-box\" !== e.boxSizing) return 0;\n",
       "          var n = e.borderTopWidth ? parseInt(e.borderTopWidth, 10) : 0,\n",
       "            e = e.borderBottomWidth ? parseInt(e.borderBottomWidth, 10) : 0;\n",
       "          return n + e;\n",
       "        }(t),\n",
       "        width: e[2],\n",
       "        type: e[3]\n",
       "      };\n",
       "    }\n",
       "    function t(e) {\n",
       "      var n = Number(k[b][\"max\" + e]),\n",
       "        i = Number(k[b][\"min\" + e]),\n",
       "        e = e.toLowerCase(),\n",
       "        t = Number(w[e]);\n",
       "      O(b, \"Checking \" + e + \" is in range \" + i + \"-\" + n), t < i && (t = i, O(b, \"Set \" + e + \" to min value\")), n < t && (t = n, O(b, \"Set \" + e + \" to max value\")), w[e] = \"\" + t;\n",
       "    }\n",
       "    function o() {\n",
       "      var t = e.origin,\n",
       "        o = k[b] && k[b].checkOrigin;\n",
       "      if (o && \"\" + t != \"null\" && !function () {\n",
       "        if (o.constructor !== Array) return e = k[b] && k[b].remoteHost, O(b, \"Checking connection is from: \" + e), t === e;\n",
       "        var e,\n",
       "          n = 0,\n",
       "          i = !1;\n",
       "        for (O(b, \"Checking connection is from allowed list of origins: \" + o); n < o.length; n++) if (o[n] === t) {\n",
       "          i = !0;\n",
       "          break;\n",
       "        }\n",
       "        return i;\n",
       "      }()) throw new Error(\"Unexpected message received from: \" + t + \" for \" + w.iframe.id + \". Message was: \" + e.data + \". This error can be disabled by setting the checkOrigin: false option or by providing of array of trusted domains.\");\n",
       "      return 1;\n",
       "    }\n",
       "    function a(e) {\n",
       "      return p.slice(p.indexOf(\":\") + v + e);\n",
       "    }\n",
       "    function s(i, t) {\n",
       "      var e, n, o;\n",
       "      e = function e() {\n",
       "        var e, n;\n",
       "        A(\"Send Page Info\", \"pageInfo:\" + (e = document.body.getBoundingClientRect(), n = w.iframe.getBoundingClientRect(), JSON.stringify({\n",
       "          iframeHeight: n.height,\n",
       "          iframeWidth: n.width,\n",
       "          clientHeight: Math.max(document.documentElement.clientHeight, window.innerHeight || 0),\n",
       "          clientWidth: Math.max(document.documentElement.clientWidth, window.innerWidth || 0),\n",
       "          offsetTop: parseInt(n.top - e.top, 10),\n",
       "          offsetLeft: parseInt(n.left - e.left, 10),\n",
       "          scrollTop: window.pageYOffset,\n",
       "          scrollLeft: window.pageXOffset,\n",
       "          documentHeight: document.documentElement.clientHeight,\n",
       "          documentWidth: document.documentElement.clientWidth,\n",
       "          windowHeight: window.innerHeight,\n",
       "          windowWidth: window.innerWidth\n",
       "        })), i, t);\n",
       "      }, n = 32, z[o = t] || (z[o] = setTimeout(function () {\n",
       "        z[o] = null, e();\n",
       "      }, n));\n",
       "    }\n",
       "    function r(e) {\n",
       "      e = e.getBoundingClientRect();\n",
       "      return W(b), {\n",
       "        x: Math.floor(Number(e.left) + Number(M.x)),\n",
       "        y: Math.floor(Number(e.top) + Number(M.y))\n",
       "      };\n",
       "    }\n",
       "    function d(e) {\n",
       "      var n = e ? r(w.iframe) : {\n",
       "          x: 0,\n",
       "          y: 0\n",
       "        },\n",
       "        i = {\n",
       "          x: Number(w.width) + n.x,\n",
       "          y: Number(w.height) + n.y\n",
       "        };\n",
       "      O(b, \"Reposition requested from iFrame (offset x:\" + n.x + \" y:\" + n.y + \")\"), window.top === window.self ? (M = i, c(), O(b, \"--\")) : window.parentIFrame ? window.parentIFrame[\"scrollTo\" + (e ? \"Offset\" : \"\")](i.x, i.y) : R(b, \"Unable to scroll to requested position, window.parentIFrame not found\");\n",
       "    }\n",
       "    function c() {\n",
       "      !1 === l(\"onScroll\", M) ? S() : C(b);\n",
       "    }\n",
       "    function u(e) {\n",
       "      var e = e.split(\"#\")[1] || \"\",\n",
       "        n = decodeURIComponent(e),\n",
       "        n = document.getElementById(n) || document.getElementsByName(n)[0];\n",
       "      n ? (n = r(n), O(b, \"Moving to in page link (#\" + e + \") at x: \" + n.x + \" y: \" + n.y), M = {\n",
       "        x: n.x,\n",
       "        y: n.y\n",
       "      }, c(), O(b, \"--\")) : window.top === window.self ? O(b, \"In page link #\" + e + \" not found\") : window.parentIFrame ? window.parentIFrame.moveToAnchor(e) : O(b, \"In page link #\" + e + \" not found and window.parentIFrame not found\");\n",
       "    }\n",
       "    function f(e) {\n",
       "      var n,\n",
       "        i = {};\n",
       "      i = 0 === Number(w.width) && 0 === Number(w.height) ? {\n",
       "        x: (n = a(9).split(\":\"))[1],\n",
       "        y: n[0]\n",
       "      } : {\n",
       "        x: w.width,\n",
       "        y: w.height\n",
       "      }, l(e, {\n",
       "        iframe: w.iframe,\n",
       "        screenX: Number(i.x),\n",
       "        screenY: Number(i.y),\n",
       "        type: w.type\n",
       "      });\n",
       "    }\n",
       "    function l(e, n) {\n",
       "      return T(b, e, n);\n",
       "    }\n",
       "    function m() {\n",
       "      switch (k[b] && k[b].firstRun && k[b] && (k[b].firstRun = !1), w.type) {\n",
       "        case \"close\":\n",
       "          N(w.iframe);\n",
       "          break;\n",
       "        case \"message\":\n",
       "          n = a(6), O(b, \"onMessage passed: {iframe: \" + w.iframe.id + \", message: \" + n + \"}\"), l(\"onMessage\", {\n",
       "            iframe: w.iframe,\n",
       "            message: JSON.parse(n)\n",
       "          }), O(b, \"--\");\n",
       "          break;\n",
       "        case \"mouseenter\":\n",
       "          f(\"onMouseEnter\");\n",
       "          break;\n",
       "        case \"mouseleave\":\n",
       "          f(\"onMouseLeave\");\n",
       "          break;\n",
       "        case \"autoResize\":\n",
       "          k[b].autoResize = JSON.parse(a(9));\n",
       "          break;\n",
       "        case \"scrollTo\":\n",
       "          d(!1);\n",
       "          break;\n",
       "        case \"scrollToOffset\":\n",
       "          d(!0);\n",
       "          break;\n",
       "        case \"pageInfo\":\n",
       "          s(k[b] && k[b].iframe, b), r = b, e(\"Add \", F), k[r] && (k[r].stopPageInfo = o);\n",
       "          break;\n",
       "        case \"pageInfoStop\":\n",
       "          k[b] && k[b].stopPageInfo && (k[b].stopPageInfo(), delete k[b].stopPageInfo);\n",
       "          break;\n",
       "        case \"inPageLink\":\n",
       "          u(a(9));\n",
       "          break;\n",
       "        case \"reset\":\n",
       "          j(w);\n",
       "          break;\n",
       "        case \"init\":\n",
       "          i(), l(\"onInit\", w.iframe);\n",
       "          break;\n",
       "        default:\n",
       "          0 === Number(w.width) && 0 === Number(w.height) ? R(\"Unsupported message received (\" + w.type + \"), this is likely due to the iframe containing a later version of iframe-resizer than the parent page\") : i();\n",
       "      }\n",
       "      function e(n, i) {\n",
       "        function t() {\n",
       "          k[r] ? s(k[r].iframe, r) : o();\n",
       "        }\n",
       "        [\"scroll\", \"resize\"].forEach(function (e) {\n",
       "          O(r, n + e + \" listener for sendPageInfo\"), i(window, e, t);\n",
       "        });\n",
       "      }\n",
       "      function o() {\n",
       "        e(\"Remove \", B);\n",
       "      }\n",
       "      var r, n;\n",
       "    }\n",
       "    var g,\n",
       "      h,\n",
       "      p = e.data,\n",
       "      w = {},\n",
       "      b = null;\n",
       "    if (\"[iFrameResizerChild]Ready\" === p) for (var y in k) A(\"iFrame requested init\", L(y), k[y].iframe, y);else x === (\"\" + p).slice(0, I) && p.slice(I).split(\":\")[0] in k ? (w = n(), b = w.id, k[b] && (k[b].loaded = !0), (h = w.type in {\n",
       "      \"true\": 1,\n",
       "      \"false\": 1,\n",
       "      undefined: 1\n",
       "    }) && O(b, \"Ignoring init message from meta parent page\"), !h && (h = !0, k[g = b] || (h = !1, R(w.type + \" No settings for \" + g + \". Message was: \" + p)), h) && (O(b, \"Received: \" + p), g = !0, null === w.iframe && (R(b, \"IFrame (\" + w.id + \") not found\"), g = !1), g && o() && m())) : E(b, \"Ignored: \" + p);\n",
       "  }\n",
       "  function T(e, n, i) {\n",
       "    var t = null,\n",
       "      o = null;\n",
       "    if (k[e]) {\n",
       "      if (\"function\" != typeof (t = k[e][n])) throw new TypeError(n + \" on iFrame[\" + e + \"] is not a function\");\n",
       "      o = t(i);\n",
       "    }\n",
       "    return o;\n",
       "  }\n",
       "  function g(e) {\n",
       "    e = e.id;\n",
       "    delete k[e];\n",
       "  }\n",
       "  function N(e) {\n",
       "    var n = e.id;\n",
       "    if (!1 === T(n, \"onClose\", n)) O(n, \"Close iframe cancelled by onClose event\");else {\n",
       "      O(n, \"Removing iFrame: \" + n);\n",
       "      try {\n",
       "        e.parentNode && e.parentNode.removeChild(e);\n",
       "      } catch (e) {\n",
       "        R(e);\n",
       "      }\n",
       "      T(n, \"onClosed\", n), O(n, \"--\"), g(e);\n",
       "    }\n",
       "  }\n",
       "  function W(e) {\n",
       "    null === M && O(e, \"Get page position: \" + (M = {\n",
       "      x: window.pageXOffset === d ? document.documentElement.scrollLeft : window.pageXOffset,\n",
       "      y: window.pageYOffset === d ? document.documentElement.scrollTop : window.pageYOffset\n",
       "    }).x + \",\" + M.y);\n",
       "  }\n",
       "  function C(e) {\n",
       "    null !== M && (window.scrollTo(M.x, M.y), O(e, \"Set page position: \" + M.x + \",\" + M.y), S());\n",
       "  }\n",
       "  function S() {\n",
       "    M = null;\n",
       "  }\n",
       "  function j(e) {\n",
       "    O(e.id, \"Size reset requested by \" + (\"init\" === e.type ? \"host page\" : \"iFrame\")), W(e.id), P(function () {\n",
       "      H(e), A(\"reset\", \"reset\", e.iframe, e.id);\n",
       "    }, e, \"reset\");\n",
       "  }\n",
       "  function H(o) {\n",
       "    function i(e) {\n",
       "      var n;\n",
       "      function i() {\n",
       "        Object.keys(k).forEach(function (e) {\n",
       "          function n(e) {\n",
       "            return \"0px\" === (k[i] && k[i].iframe.style[e]);\n",
       "          }\n",
       "          var i;\n",
       "          k[i = e] && null !== k[i].iframe.offsetParent && (n(\"height\") || n(\"width\")) && A(\"Visibility change\", \"resize\", k[i].iframe, i);\n",
       "        });\n",
       "      }\n",
       "      function t(e) {\n",
       "        O(\"window\", \"Mutation observed: \" + e[0].target + \" \" + e[0].type), h(i, 16);\n",
       "      }\n",
       "      !a && \"0\" === o[e] && (a = !0, O(r, \"Hidden iFrame detected, creating visibility listener\"), e = m()) && (n = document.querySelector(\"body\"), new e(t).observe(n, {\n",
       "        attributes: !0,\n",
       "        attributeOldValue: !1,\n",
       "        characterData: !0,\n",
       "        characterDataOldValue: !1,\n",
       "        childList: !0,\n",
       "        subtree: !0\n",
       "      }));\n",
       "    }\n",
       "    function e(e) {\n",
       "      var n;\n",
       "      n = e, o.id ? (o.iframe.style[n] = o[n] + \"px\", O(o.id, \"IFrame (\" + r + \") \" + n + \" set to \" + o[n] + \"px\")) : O(\"undefined\", \"messageData id not set\"), i(e);\n",
       "    }\n",
       "    var r = o.iframe.id;\n",
       "    k[r] && (k[r].sizeHeight && e(\"height\"), k[r].sizeWidth) && e(\"width\");\n",
       "  }\n",
       "  function P(e, n, i) {\n",
       "    i !== n.type && r && !window.jasmine ? (O(n.id, \"Requesting animation frame\"), r(e)) : e();\n",
       "  }\n",
       "  function A(n, i, t, o, e) {\n",
       "    function r() {\n",
       "      var e;\n",
       "      t && \"contentWindow\" in t && null !== t.contentWindow ? (e = k[o] && k[o].targetOrigin, O(o, \"[\" + n + \"] Sending msg to iframe[\" + o + \"] (\" + i + \") targetOrigin: \" + e), t.contentWindow.postMessage(x + i, e)) : R(o, \"[\" + n + \"] IFrame(\" + o + \") not found\");\n",
       "    }\n",
       "    function a() {\n",
       "      e && k[o] && k[o].warningTimeout && (k[o].msgTimeout = setTimeout(function () {\n",
       "        !k[o] || k[o].loaded || s || (s = !0, R(o, \"IFrame has not responded within \" + k[o].warningTimeout / 1e3 + \" seconds. Check iFrameResizer.contentWindow.js has been loaded in iFrame. This message can be ignored if everything is working, or you can set the warningTimeout option to a higher value or zero to suppress this warning.\"));\n",
       "      }, k[o].warningTimeout));\n",
       "    }\n",
       "    var s = !1;\n",
       "    o = o || t.id, k[o] && (r(), a());\n",
       "  }\n",
       "  function L(e) {\n",
       "    return e + \":\" + k[e].bodyMarginV1 + \":\" + k[e].sizeWidth + \":\" + k[e].log + \":\" + k[e].interval + \":\" + k[e].enablePublicMethods + \":\" + k[e].autoResize + \":\" + k[e].bodyMargin + \":\" + k[e].heightCalculationMethod + \":\" + k[e].bodyBackground + \":\" + k[e].bodyPadding + \":\" + k[e].tolerance + \":\" + k[e].inPageLinks + \":\" + k[e].resizeFrom + \":\" + k[e].widthCalculationMethod + \":\" + k[e].mouseEvents;\n",
       "  }\n",
       "  function s(t, i) {\n",
       "    function e(i) {\n",
       "      var e = m();\n",
       "      e && (e = e, t.parentNode) && new e(function (e) {\n",
       "        e.forEach(function (e) {\n",
       "          Array.prototype.slice.call(e.removedNodes).forEach(function (e) {\n",
       "            e === t && N(t);\n",
       "          });\n",
       "        });\n",
       "      }).observe(t.parentNode, {\n",
       "        childList: !0\n",
       "      }), F(t, \"load\", function () {\n",
       "        var e, n;\n",
       "        A(\"iFrame.onload\", i, t, d, !0), e = k[r] && k[r].firstRun, n = k[r] && k[r].heightCalculationMethod in f, !e && n && j({\n",
       "          iframe: t,\n",
       "          height: 0,\n",
       "          width: 0,\n",
       "          type: \"init\"\n",
       "        });\n",
       "      }), A(\"init\", i, t, d, !0);\n",
       "    }\n",
       "    function o(e) {\n",
       "      var n = e.split(\"Callback\");\n",
       "      2 === n.length && (this[n = \"on\" + n[0].charAt(0).toUpperCase() + n[0].slice(1)] = this[e], delete this[e], R(r, \"Deprecated: '\" + e + \"' has been renamed '\" + n + \"'. The old method will be removed in the next major version.\"));\n",
       "    }\n",
       "    function n(e) {\n",
       "      if (e = e || {}, k[r] = Object.create(null), k[r].iframe = t, k[r].firstRun = !0, k[r].remoteHost = t.src && t.src.split(\"/\").slice(0, 3).join(\"/\"), \"object\" != _typeof(e)) throw new TypeError(\"Options is not an object\");\n",
       "      Object.keys(e).forEach(o, e);\n",
       "      var n,\n",
       "        i = e;\n",
       "      for (n in l) Object.prototype.hasOwnProperty.call(l, n) && (k[r][n] = (Object.prototype.hasOwnProperty.call(i, n) ? i : l)[n]);\n",
       "      k[r] && (k[r].targetOrigin = !0 !== k[r].checkOrigin || \"\" === (e = k[r].remoteHost) || null !== e.match(/^(about:blank|javascript:|file:\\/\\/)/) ? \"*\" : e);\n",
       "    }\n",
       "    var r = function (e) {\n",
       "      if (\"string\" != typeof e) throw new TypeError(\"Invaild id for iFrame. Expected String\");\n",
       "      var n;\n",
       "      return \"\" === e && (t.id = (n = i && i.id || l.id + c++, null !== document.getElementById(n) && (n += c++), e = n), u = (i || {}).log, O(e, \"Added missing iframe ID: \" + e + \" (\" + t.src + \")\")), e;\n",
       "    }(t.id);\n",
       "    if (r in k && \"iFrameResizer\" in t) R(r, \"Ignored iFrame, already setup.\");else {\n",
       "      switch (n(i), O(r, \"IFrame scrolling \" + (k[r] && k[r].scrolling ? \"enabled\" : \"disabled\") + \" for \" + r), t.style.overflow = !1 === (k[r] && k[r].scrolling) ? \"hidden\" : \"auto\", k[r] && k[r].scrolling) {\n",
       "        case \"omit\":\n",
       "          break;\n",
       "        case !0:\n",
       "          t.scrolling = \"yes\";\n",
       "          break;\n",
       "        case !1:\n",
       "          t.scrolling = \"no\";\n",
       "          break;\n",
       "        default:\n",
       "          t.scrolling = k[r] ? k[r].scrolling : \"no\";\n",
       "      }\n",
       "      s(\"Height\"), s(\"Width\"), a(\"maxHeight\"), a(\"minHeight\"), a(\"maxWidth\"), a(\"minWidth\"), \"number\" != typeof (k[r] && k[r].bodyMargin) && \"0\" !== (k[r] && k[r].bodyMargin) || (k[r].bodyMarginV1 = k[r].bodyMargin, k[r].bodyMargin = k[r].bodyMargin + \"px\"), e(L(r)), k[r] && (k[r].iframe.iFrameResizer = {\n",
       "        close: N.bind(null, k[r].iframe),\n",
       "        removeListeners: g.bind(null, k[r].iframe),\n",
       "        resize: A.bind(null, \"Window resize\", \"resize\", k[r].iframe),\n",
       "        moveToAnchor: function moveToAnchor(e) {\n",
       "          A(\"Move to anchor\", \"moveToAnchor:\" + e, k[r].iframe, r);\n",
       "        },\n",
       "        sendMessage: function sendMessage(e) {\n",
       "          A(\"Send Message\", \"message:\" + (e = JSON.stringify(e)), k[r].iframe, r);\n",
       "        }\n",
       "      });\n",
       "    }\n",
       "    function a(e) {\n",
       "      var n = k[r][e];\n",
       "      1 / 0 !== n && 0 !== n && (t.style[e] = \"number\" == typeof n ? n + \"px\" : n, O(r, \"Set \" + e + \" = \" + t.style[e]));\n",
       "    }\n",
       "    function s(e) {\n",
       "      if (k[r][\"min\" + e] > k[r][\"max\" + e]) throw new Error(\"Value for min\" + e + \" can not be greater than max\" + e);\n",
       "    }\n",
       "  }\n",
       "  function h(e, n) {\n",
       "    null === i && (i = setTimeout(function () {\n",
       "      i = null, e();\n",
       "    }, n));\n",
       "  }\n",
       "  function e() {\n",
       "    \"hidden\" !== document.visibilityState && (O(\"document\", \"Trigger event: Visibility change\"), h(function () {\n",
       "      b(\"Tab Visible\", \"resize\");\n",
       "    }, 16));\n",
       "  }\n",
       "  function b(i, t) {\n",
       "    Object.keys(k).forEach(function (e) {\n",
       "      var n;\n",
       "      k[n = e] && \"parent\" === k[n].resizeFrom && k[n].autoResize && !k[n].firstRun && A(i, t, k[e].iframe, e);\n",
       "    });\n",
       "  }\n",
       "  function y() {\n",
       "    F(window, \"message\", w), F(window, \"resize\", function () {\n",
       "      var e;\n",
       "      O(\"window\", \"Trigger event: \" + (e = \"resize\")), h(function () {\n",
       "        b(\"Window \" + e, \"resize\");\n",
       "      }, 16);\n",
       "    }), F(document, \"visibilitychange\", e), F(document, \"-webkit-visibilitychange\", e);\n",
       "  }\n",
       "  function n() {\n",
       "    function t(e, n) {\n",
       "      if (n) {\n",
       "        if (!n.tagName) throw new TypeError(\"Object is not a valid DOM element\");\n",
       "        if (\"IFRAME\" !== n.tagName.toUpperCase()) throw new TypeError(\"Expected <IFRAME> tag, found <\" + n.tagName + \">\");\n",
       "        s(n, e), o.push(n);\n",
       "      }\n",
       "    }\n",
       "    for (var o, e = [\"moz\", \"webkit\", \"o\", \"ms\"], n = 0; n < e.length && !r; n += 1) r = window[e[n] + \"RequestAnimationFrame\"];\n",
       "    return r ? r = r.bind(window) : O(\"setup\", \"RequestAnimationFrame not supported\"), y(), function (e, n) {\n",
       "      var i;\n",
       "      switch (o = [], (i = e) && i.enablePublicMethods && R(\"enablePublicMethods option has been removed, public methods are now always available in the iFrame\"), _typeof(n)) {\n",
       "        case \"undefined\":\n",
       "        case \"string\":\n",
       "          Array.prototype.forEach.call(document.querySelectorAll(n || \"iframe\"), t.bind(d, e));\n",
       "          break;\n",
       "        case \"object\":\n",
       "          t(e, n);\n",
       "          break;\n",
       "        default:\n",
       "          throw new TypeError(\"Unexpected data type (\" + _typeof(n) + \")\");\n",
       "      }\n",
       "      return o;\n",
       "    };\n",
       "  }\n",
       "  function q(e) {\n",
       "    e.fn ? e.fn.iFrameResize || (e.fn.iFrameResize = function (i) {\n",
       "      return this.filter(\"iframe\").each(function (e, n) {\n",
       "        s(n, i);\n",
       "      }).end();\n",
       "    }) : E(\"\", \"Unable to bind to jQuery, it is not fully loaded.\");\n",
       "  }\n",
       "  \"undefined\" != typeof window && (c = 0, a = u = !1, v = \"message\".length, I = (x = \"[iFrameSizer]\").length, M = null, r = window.requestAnimationFrame, f = Object.freeze({\n",
       "    max: 1,\n",
       "    scroll: 1,\n",
       "    bodyScroll: 1,\n",
       "    documentElementScroll: 1\n",
       "  }), k = {}, i = null, l = Object.freeze({\n",
       "    autoResize: !0,\n",
       "    bodyBackground: null,\n",
       "    bodyMargin: null,\n",
       "    bodyMarginV1: 8,\n",
       "    bodyPadding: null,\n",
       "    checkOrigin: !0,\n",
       "    inPageLinks: !1,\n",
       "    enablePublicMethods: !0,\n",
       "    heightCalculationMethod: \"bodyOffset\",\n",
       "    id: \"iFrameResizer\",\n",
       "    interval: 32,\n",
       "    log: !1,\n",
       "    maxHeight: 1 / 0,\n",
       "    maxWidth: 1 / 0,\n",
       "    minHeight: 0,\n",
       "    minWidth: 0,\n",
       "    mouseEvents: !0,\n",
       "    resizeFrom: \"parent\",\n",
       "    scrolling: !1,\n",
       "    sizeHeight: !0,\n",
       "    sizeWidth: !1,\n",
       "    warningTimeout: 5e3,\n",
       "    tolerance: 0,\n",
       "    widthCalculationMethod: \"scroll\",\n",
       "    onClose: function onClose() {\n",
       "      return !0;\n",
       "    },\n",
       "    onClosed: function onClosed() {},\n",
       "    onInit: function onInit() {},\n",
       "    onMessage: function onMessage() {\n",
       "      R(\"onMessage function not defined\");\n",
       "    },\n",
       "    onMouseEnter: function onMouseEnter() {},\n",
       "    onMouseLeave: function onMouseLeave() {},\n",
       "    onResized: function onResized() {},\n",
       "    onScroll: function onScroll() {\n",
       "      return !0;\n",
       "    }\n",
       "  }), z = {}, window.jQuery !== d && q(window.jQuery), \"function\" == typeof define && define.amd ? define([], n) : \"object\" == (typeof module === \"undefined\" ? \"undefined\" : _typeof(module)) && \"object\" == _typeof(module.exports) && (module.exports = n()), window.iFrameResize = window.iFrameResize || n());\n",
       "}();\n",
       "(function(){iFrameResize({ checkOrigin: false }, '#scan-11917050016');})();\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "display(results)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Generate a test suite from the Scan\n",
    "The objects produced by the scan can be used as fixtures to generate a test suite that integrate domain-specific issues. To create custom tests, refer to the Test your ML Model page."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 221,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T16:03:03.890497Z",
     "start_time": "2023-06-02T16:02:59.774550Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Executed 'Precision on data slice â€œ`transcription` contains \"xyz\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x329abc2e0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.32\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"subjective\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x33bf00040>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.37\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"admission\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x3351e7700>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.38\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"daily\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x33bf0a920>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.38\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"coronary\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x30209bb50>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.39\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"followup\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334e45000>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"count\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x3283453c0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"lung\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x32a102e60>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"function\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x2c05a5300>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.41\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"abc\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x327f85a80>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"improved\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x33d5e1cc0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"discharge\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334d9b160>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"greater\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x33c79d120>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.43\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"aspirin\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334d9b280>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.43\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"continue\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x32a100b80>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.44\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"mass\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x329abf0d0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.76\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"16\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334e44b50>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.75\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"ct\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334209ae0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.75\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"having\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x329abddb0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.74\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"weight\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x3348633a0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.74\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"diet\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334531840>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.74\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"today\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334862f20>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.74\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"temperature\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x33d5e02e0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.73\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"discussed\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x329ae6920>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.73\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"dr\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x32a0978e0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.73\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"follow\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x3347eab00>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.72\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"wound\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x32a1005e0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.71\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"times\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x3341b6e60>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.71\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"bladder\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334862860>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.71\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"infection\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x2c64ffac0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.71\n",
      "               \n",
      "               \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "    .alert {\n",
       "        padding: 16px;\n",
       "        border-radius: 4px;\n",
       "        display: flex;\n",
       "        align-items: center;\n",
       "        flex-wrap: wrap;\n",
       "        gap: 4px;\n",
       "    }\n",
       "\n",
       "    .test-result {\n",
       "        border-radius: 4px;\n",
       "        padding: 0 12px;\n",
       "        display: flex;\n",
       "        gap: 4px;\n",
       "        align-items: center;\n",
       "    }\n",
       "\n",
       "    .alert-error, .test-card-failed .test-result {\n",
       "        color: rgb(255, 82, 82);\n",
       "        background: rgb(243, 226, 226);\n",
       "        fill: currentcolor;\n",
       "\n",
       "    }\n",
       "\n",
       "    .test-card-error .test-result {\n",
       "        color: #856404;\n",
       "        background: #fff3cd;\n",
       "        fill: currentcolor;\n",
       "    }\n",
       "\n",
       "    .alert-success, .test-card-passed .test-result {\n",
       "        color: rgb(76, 175, 80);\n",
       "        background: rgb(226, 243, 226);\n",
       "        fill: currentcolor;\n",
       "    }\n",
       "\n",
       "    .alert svg {\n",
       "        height: 24px;\n",
       "        width: 24px;\n",
       "        border-radius: 50%;\n",
       "        background: rgba(0, 0, 0, 10%);\n",
       "    }\n",
       "\n",
       "    .test-result svg {\n",
       "        height: 16px;\n",
       "        width: 16px;\n",
       "    }\n",
       "\n",
       "    .card-container {\n",
       "        display: flex;\n",
       "        flex-direction: column;\n",
       "        gap: 16px;\n",
       "    }\n",
       "\n",
       "    .test-card {\n",
       "        border-radius: 4px;\n",
       "        border: 1px solid #e0e0e0;\n",
       "        color: rgb(98, 98, 98);\n",
       "        background: #fff;\n",
       "    }\n",
       "\n",
       "    .param {\n",
       "        display: flex;\n",
       "        flex-direction: column;\n",
       "    }\n",
       "\n",
       "    .test-card .test-name, .test-card .param-value {\n",
       "        color: #000\n",
       "    }\n",
       "\n",
       "    .test-card-row {\n",
       "        padding: 10px;\n",
       "        display: flex;\n",
       "        gap: 20px;\n",
       "        align-items: center;\n",
       "        flex-wrap: wrap;\n",
       "    }\n",
       "\n",
       "    .test-card-row:last-child {\n",
       "        border-top: 1px solid #dee2e6;\n",
       "    }\n",
       "\n",
       "    .spacer {\n",
       "        flex-grow: 1;\n",
       "    }\n",
       "\n",
       "    .test-card-failed .metric {\n",
       "        color: #b71c1c;\n",
       "    }\n",
       "</style>\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "<div class='dark'>\n",
       "    <div id='gsk-suite'>\n",
       "        <div class='alert alert-error'>\n",
       "  \n",
       "  <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "    <path\n",
       "      d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "  </svg>\n",
       "  <strong>Test suite failed.</strong>\n",
       "  <span>To debug your failing test and diagnose the issue, please run the Giskard server (<a\n",
       "    href='https://docs.giskard.ai/en/latest/guides/installation_app/index.html'>see documentation</a>)</span>\n",
       "  \n",
       "</div>\n",
       "        <div class='card-container'>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;xyz&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.31818</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;xyz&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;subjective&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.36667</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;subjective&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;admission&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.375</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;admission&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;daily&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.38462</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;daily&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;coronary&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.3913</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;coronary&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;followup&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.39583</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;followup&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;count&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.4</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;count&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;lung&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.4</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;lung&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;function&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.41379</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;function&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;abc&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.41667</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;abc&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;improved&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.41935</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;improved&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;discharge&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.42353</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;discharge&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;greater&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.43478</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;greater&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;aspirin&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.43478</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;aspirin&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;continue&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.44231</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;continue&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;mass&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.7619</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;mass&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;16&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.75</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;16&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;ct&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.75</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;ct&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;having&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.74194</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;having&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;weight&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.74194</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;weight&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;diet&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.73913</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;diet&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;today&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.73684</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;today&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;temperature&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.73333</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;temperature&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;discussed&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.72727</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;discussed&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;dr&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.72549</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;dr&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;follow&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.72222</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;follow&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;wound&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.71429</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;wound&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;times&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.71429</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;times&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;bladder&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.71429</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;bladder&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;infection&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.70833</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;infection&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "</div>\n",
       "    </div>\n",
       "</div>\n"
      ],
      "text/plain": [
       "<TestSuiteResult (failed)>"
      ]
     },
     "execution_count": 221,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_suite = results.generate_test_suite(\"My first test suite\")\n",
    "test_suite.run()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Customize your suite by loading objects from the Giskard catalog\n",
    "\n",
    "The Giskard open source catalog will enable to load:\n",
    "\n",
    "* Tests such as metamorphic, performance, prediction & data drift, statistical tests, etc\n",
    "* Slicing functions such as detectors of toxicity, hate, emotion, etc\n",
    "* Transformation functions such as generators of typos, paraphrase, style tune, etc\n",
    "\n",
    "For demo purposes, we will load a simple unit test (test_f1) that checks if the test F1 score is above the given threshold. For more examples of tests and functions, refer to the Giskard catalog."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 222,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-06-02T16:03:54.282167Z",
     "start_time": "2023-06-02T16:03:50.135561Z"
    },
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Executed 'Precision on data slice â€œ`transcription` contains \"xyz\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x329abc2e0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.32\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"subjective\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x33bf00040>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.37\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"admission\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x3351e7700>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.38\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"daily\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x33bf0a920>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.38\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"coronary\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x30209bb50>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.39\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"followup\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334e45000>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"count\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x3283453c0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"lung\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x32a102e60>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.4\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"function\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x2c05a5300>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.41\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"abc\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x327f85a80>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"improved\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x33d5e1cc0>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"discharge\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334d9b160>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.42\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"greater\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x33c79d120>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.43\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"aspirin\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334d9b280>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.43\n",
      "               \n",
      "               \n",
      "Executed 'Precision on data slice â€œ`transcription` contains \"continue\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x32a100b80>, 'threshold': 0.581266846361186}: \n",
      "               Test failed\n",
      "               Metric: 0.44\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"mass\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x329abf0d0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.76\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"16\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334e44b50>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.75\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"ct\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334209ae0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.75\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"having\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x329abddb0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.74\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"weight\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x3348633a0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.74\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"diet\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334531840>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.74\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"today\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334862f20>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.74\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"temperature\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x33d5e02e0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.73\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"discussed\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x329ae6920>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.73\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"dr\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x32a0978e0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.73\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"follow\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x3347eab00>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.72\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"wound\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x32a1005e0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.71\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"times\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x3341b6e60>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.71\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"bladder\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x334862860>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.71\n",
      "               \n",
      "               \n",
      "Executed 'Overconfidence on data slice â€œ`transcription` contains \"infection\"â€' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'slicing_function': <giskard.slicing.slice.QueryBasedSliceFunction object at 0x2c64ffac0>, 'threshold': 0.6569444444444444, 'p_threshold': 0.2468526289804987}: \n",
      "               Test failed\n",
      "               Metric: 0.71\n",
      "               \n",
      "               \n",
      "Executed 'F1' with arguments {'model': <giskard.models.function.PredictionFunctionModel object at 0x286d01720>, 'dataset': <giskard.datasets.base.Dataset object at 0x286d5e2f0>, 'threshold': 0.7}: \n",
      "               Test failed\n",
      "               Metric: 0.61\n",
      "               \n",
      "               \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "    .alert {\n",
       "        padding: 16px;\n",
       "        border-radius: 4px;\n",
       "        display: flex;\n",
       "        align-items: center;\n",
       "        flex-wrap: wrap;\n",
       "        gap: 4px;\n",
       "    }\n",
       "\n",
       "    .test-result {\n",
       "        border-radius: 4px;\n",
       "        padding: 0 12px;\n",
       "        display: flex;\n",
       "        gap: 4px;\n",
       "        align-items: center;\n",
       "    }\n",
       "\n",
       "    .alert-error, .test-card-failed .test-result {\n",
       "        color: rgb(255, 82, 82);\n",
       "        background: rgb(243, 226, 226);\n",
       "        fill: currentcolor;\n",
       "\n",
       "    }\n",
       "\n",
       "    .test-card-error .test-result {\n",
       "        color: #856404;\n",
       "        background: #fff3cd;\n",
       "        fill: currentcolor;\n",
       "    }\n",
       "\n",
       "    .alert-success, .test-card-passed .test-result {\n",
       "        color: rgb(76, 175, 80);\n",
       "        background: rgb(226, 243, 226);\n",
       "        fill: currentcolor;\n",
       "    }\n",
       "\n",
       "    .alert svg {\n",
       "        height: 24px;\n",
       "        width: 24px;\n",
       "        border-radius: 50%;\n",
       "        background: rgba(0, 0, 0, 10%);\n",
       "    }\n",
       "\n",
       "    .test-result svg {\n",
       "        height: 16px;\n",
       "        width: 16px;\n",
       "    }\n",
       "\n",
       "    .card-container {\n",
       "        display: flex;\n",
       "        flex-direction: column;\n",
       "        gap: 16px;\n",
       "    }\n",
       "\n",
       "    .test-card {\n",
       "        border-radius: 4px;\n",
       "        border: 1px solid #e0e0e0;\n",
       "        color: rgb(98, 98, 98);\n",
       "        background: #fff;\n",
       "    }\n",
       "\n",
       "    .param {\n",
       "        display: flex;\n",
       "        flex-direction: column;\n",
       "    }\n",
       "\n",
       "    .test-card .test-name, .test-card .param-value {\n",
       "        color: #000\n",
       "    }\n",
       "\n",
       "    .test-card-row {\n",
       "        padding: 10px;\n",
       "        display: flex;\n",
       "        gap: 20px;\n",
       "        align-items: center;\n",
       "        flex-wrap: wrap;\n",
       "    }\n",
       "\n",
       "    .test-card-row:last-child {\n",
       "        border-top: 1px solid #dee2e6;\n",
       "    }\n",
       "\n",
       "    .spacer {\n",
       "        flex-grow: 1;\n",
       "    }\n",
       "\n",
       "    .test-card-failed .metric {\n",
       "        color: #b71c1c;\n",
       "    }\n",
       "</style>\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "<div class='dark'>\n",
       "    <div id='gsk-suite'>\n",
       "        <div class='alert alert-error'>\n",
       "  \n",
       "  <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "    <path\n",
       "      d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "  </svg>\n",
       "  <strong>Test suite failed.</strong>\n",
       "  <span>To debug your failing test and diagnose the issue, please run the Giskard server (<a\n",
       "    href='https://docs.giskard.ai/en/latest/guides/installation_app/index.html'>see documentation</a>)</span>\n",
       "  \n",
       "</div>\n",
       "        <div class='card-container'>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;xyz&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.31818</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;xyz&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;subjective&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.36667</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;subjective&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;admission&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.375</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;admission&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;daily&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.38462</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;daily&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;coronary&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.3913</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;coronary&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;followup&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.39583</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;followup&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;count&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.4</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;count&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;lung&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.4</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;lung&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;function&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.41379</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;function&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;abc&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.41667</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;abc&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;improved&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.41935</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;improved&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;discharge&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.42353</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;discharge&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;greater&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.43478</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;greater&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;aspirin&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.43478</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;aspirin&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Precision on data slice â€œ`transcription` contains &#34;continue&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.44231</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;continue&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.581266846361186</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;mass&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.7619</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;mass&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;16&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.75</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;16&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;ct&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.75</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;ct&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;having&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.74194</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;having&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;weight&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.74194</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;weight&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;diet&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.73913</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;diet&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;today&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.73684</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;today&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;temperature&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.73333</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;temperature&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;discussed&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.72727</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;discussed&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;dr&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.72549</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;dr&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;follow&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.72222</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;follow&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;wound&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.71429</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;wound&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;times&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.71429</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;times&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;bladder&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.71429</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;bladder&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test Overconfidence on data slice â€œ`transcription` contains &#34;infection&#34;â€</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.70833</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>slicing_function</span>\n",
       "        <span class='param-value'>`transcription` contains &#34;infection&#34;</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.6569444444444444</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>p_threshold</span>\n",
       "        <span class='param-value'>0.2468526289804987</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "  <div class='test-card test-card-failed'>\n",
       "    <div class='test-card-row'>\n",
       "      <span class='test-name'>Test F1</span>\n",
       "      <div class='spacer'></div>\n",
       "      \n",
       "      <span class='metric'>Measured <strong>Metric = 0.61186</strong></span>\n",
       "      \n",
       "      <span class='test-result'>\n",
       "      \n",
       "        <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><title>close</title>\n",
       "          <path\n",
       "            d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z' />\n",
       "        </svg>\n",
       "        Failed\n",
       "      \n",
       "    </span>\n",
       "    </div>\n",
       "    <div class='test-card-row'>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>model</span>\n",
       "        <span class='param-value'>e1791018-69e4-4583-ba7e-78bf305f3109</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>dataset</span>\n",
       "        <span class='param-value'>medical_transcript_dataset</span>\n",
       "      </div>\n",
       "      \n",
       "      <div class='param'>\n",
       "        <span>threshold</span>\n",
       "        <span class='param-value'>0.7</span>\n",
       "      </div>\n",
       "      \n",
       "    </div>\n",
       "  </div>\n",
       "  \n",
       "</div>\n",
       "    </div>\n",
       "</div>\n"
      ],
      "text/plain": [
       "<TestSuiteResult (failed)>"
      ]
     },
     "execution_count": 222,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_suite.add_test(testing.test_f1(model=wrapped_model, dataset=wrapped_data, threshold=0.7)).run()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## Upload your suite to the Giskard server\n",
    "\n",
    "Upload your suite to the Giskard server to:\n",
    "\n",
    "* Compare models to decide which model to promote\n",
    "* Debug your tests to diagnose the issues\n",
    "* Create more domain-specific tests that are integrating business feedback\n",
    "* Share your results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 223,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "ConnectionError",
     "evalue": "HTTPConnectionPool(host='localhost', port=19000): Max retries exceeded with url: /api/v2/settings/ml-worker-connect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x327fc8640>: Failed to establish a new connection: [Errno 61] Connection refused'))",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mConnectionRefusedError\u001b[0m                    Traceback (most recent call last)",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/urllib3/connection.py:174\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    173\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m--> 174\u001b[0m     conn \u001b[39m=\u001b[39m connection\u001b[39m.\u001b[39;49mcreate_connection(\n\u001b[1;32m    175\u001b[0m         (\u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_dns_host, \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mport), \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mtimeout, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mextra_kw\n\u001b[1;32m    176\u001b[0m     )\n\u001b[1;32m    178\u001b[0m \u001b[39mexcept\u001b[39;00m SocketTimeout:\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/urllib3/util/connection.py:95\u001b[0m, in \u001b[0;36mcreate_connection\u001b[0;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[1;32m     94\u001b[0m \u001b[39mif\u001b[39;00m err \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m---> 95\u001b[0m     \u001b[39mraise\u001b[39;00m err\n\u001b[1;32m     97\u001b[0m \u001b[39mraise\u001b[39;00m socket\u001b[39m.\u001b[39merror(\u001b[39m\"\u001b[39m\u001b[39mgetaddrinfo returns an empty list\u001b[39m\u001b[39m\"\u001b[39m)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/urllib3/util/connection.py:85\u001b[0m, in \u001b[0;36mcreate_connection\u001b[0;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[1;32m     84\u001b[0m     sock\u001b[39m.\u001b[39mbind(source_address)\n\u001b[0;32m---> 85\u001b[0m sock\u001b[39m.\u001b[39;49mconnect(sa)\n\u001b[1;32m     86\u001b[0m \u001b[39mreturn\u001b[39;00m sock\n",
      "\u001b[0;31mConnectionRefusedError\u001b[0m: [Errno 61] Connection refused",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mNewConnectionError\u001b[0m                        Traceback (most recent call last)",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:703\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[1;32m    702\u001b[0m \u001b[39m# Make the request on the httplib connection object.\u001b[39;00m\n\u001b[0;32m--> 703\u001b[0m httplib_response \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_make_request(\n\u001b[1;32m    704\u001b[0m     conn,\n\u001b[1;32m    705\u001b[0m     method,\n\u001b[1;32m    706\u001b[0m     url,\n\u001b[1;32m    707\u001b[0m     timeout\u001b[39m=\u001b[39;49mtimeout_obj,\n\u001b[1;32m    708\u001b[0m     body\u001b[39m=\u001b[39;49mbody,\n\u001b[1;32m    709\u001b[0m     headers\u001b[39m=\u001b[39;49mheaders,\n\u001b[1;32m    710\u001b[0m     chunked\u001b[39m=\u001b[39;49mchunked,\n\u001b[1;32m    711\u001b[0m )\n\u001b[1;32m    713\u001b[0m \u001b[39m# If we're going to release the connection in ``finally:``, then\u001b[39;00m\n\u001b[1;32m    714\u001b[0m \u001b[39m# the response doesn't need to know about the connection. Otherwise\u001b[39;00m\n\u001b[1;32m    715\u001b[0m \u001b[39m# it will also try to release it and we'll have a double-release\u001b[39;00m\n\u001b[1;32m    716\u001b[0m \u001b[39m# mess.\u001b[39;00m\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:398\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[1;32m    397\u001b[0m     \u001b[39melse\u001b[39;00m:\n\u001b[0;32m--> 398\u001b[0m         conn\u001b[39m.\u001b[39;49mrequest(method, url, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mhttplib_request_kw)\n\u001b[1;32m    400\u001b[0m \u001b[39m# We are swallowing BrokenPipeError (errno.EPIPE) since the server is\u001b[39;00m\n\u001b[1;32m    401\u001b[0m \u001b[39m# legitimately able to close the connection after sending a valid response.\u001b[39;00m\n\u001b[1;32m    402\u001b[0m \u001b[39m# With this behaviour, the received response is still readable.\u001b[39;00m\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/urllib3/connection.py:244\u001b[0m, in \u001b[0;36mHTTPConnection.request\u001b[0;34m(self, method, url, body, headers)\u001b[0m\n\u001b[1;32m    243\u001b[0m     headers[\u001b[39m\"\u001b[39m\u001b[39mUser-Agent\u001b[39m\u001b[39m\"\u001b[39m] \u001b[39m=\u001b[39m _get_default_user_agent()\n\u001b[0;32m--> 244\u001b[0m \u001b[39msuper\u001b[39;49m(HTTPConnection, \u001b[39mself\u001b[39;49m)\u001b[39m.\u001b[39;49mrequest(method, url, body\u001b[39m=\u001b[39;49mbody, headers\u001b[39m=\u001b[39;49mheaders)\n",
      "File \u001b[0;32m/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/http/client.py:1283\u001b[0m, in \u001b[0;36mHTTPConnection.request\u001b[0;34m(self, method, url, body, headers, encode_chunked)\u001b[0m\n\u001b[1;32m   1282\u001b[0m \u001b[39m\u001b[39m\u001b[39m\"\"\"Send a complete request to the server.\"\"\"\u001b[39;00m\n\u001b[0;32m-> 1283\u001b[0m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_send_request(method, url, body, headers, encode_chunked)\n",
      "File \u001b[0;32m/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/http/client.py:1329\u001b[0m, in \u001b[0;36mHTTPConnection._send_request\u001b[0;34m(self, method, url, body, headers, encode_chunked)\u001b[0m\n\u001b[1;32m   1328\u001b[0m     body \u001b[39m=\u001b[39m _encode(body, \u001b[39m'\u001b[39m\u001b[39mbody\u001b[39m\u001b[39m'\u001b[39m)\n\u001b[0;32m-> 1329\u001b[0m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mendheaders(body, encode_chunked\u001b[39m=\u001b[39;49mencode_chunked)\n",
      "File \u001b[0;32m/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/http/client.py:1278\u001b[0m, in \u001b[0;36mHTTPConnection.endheaders\u001b[0;34m(self, message_body, encode_chunked)\u001b[0m\n\u001b[1;32m   1277\u001b[0m     \u001b[39mraise\u001b[39;00m CannotSendHeader()\n\u001b[0;32m-> 1278\u001b[0m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_send_output(message_body, encode_chunked\u001b[39m=\u001b[39;49mencode_chunked)\n",
      "File \u001b[0;32m/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/http/client.py:1038\u001b[0m, in \u001b[0;36mHTTPConnection._send_output\u001b[0;34m(self, message_body, encode_chunked)\u001b[0m\n\u001b[1;32m   1037\u001b[0m \u001b[39mdel\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_buffer[:]\n\u001b[0;32m-> 1038\u001b[0m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49msend(msg)\n\u001b[1;32m   1040\u001b[0m \u001b[39mif\u001b[39;00m message_body \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m   1041\u001b[0m \n\u001b[1;32m   1042\u001b[0m     \u001b[39m# create a consistent interface to message_body\u001b[39;00m\n",
      "File \u001b[0;32m/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/http/client.py:976\u001b[0m, in \u001b[0;36mHTTPConnection.send\u001b[0;34m(self, data)\u001b[0m\n\u001b[1;32m    975\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mauto_open:\n\u001b[0;32m--> 976\u001b[0m     \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mconnect()\n\u001b[1;32m    977\u001b[0m \u001b[39melse\u001b[39;00m:\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/urllib3/connection.py:205\u001b[0m, in \u001b[0;36mHTTPConnection.connect\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    204\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39mconnect\u001b[39m(\u001b[39mself\u001b[39m):\n\u001b[0;32m--> 205\u001b[0m     conn \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_new_conn()\n\u001b[1;32m    206\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_prepare_conn(conn)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/urllib3/connection.py:186\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    185\u001b[0m \u001b[39mexcept\u001b[39;00m SocketError \u001b[39mas\u001b[39;00m e:\n\u001b[0;32m--> 186\u001b[0m     \u001b[39mraise\u001b[39;00m NewConnectionError(\n\u001b[1;32m    187\u001b[0m         \u001b[39mself\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39mFailed to establish a new connection: \u001b[39m\u001b[39m%s\u001b[39;00m\u001b[39m\"\u001b[39m \u001b[39m%\u001b[39m e\n\u001b[1;32m    188\u001b[0m     )\n\u001b[1;32m    190\u001b[0m \u001b[39mreturn\u001b[39;00m conn\n",
      "\u001b[0;31mNewConnectionError\u001b[0m: <urllib3.connection.HTTPConnection object at 0x327fc8640>: Failed to establish a new connection: [Errno 61] Connection refused",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mMaxRetryError\u001b[0m                             Traceback (most recent call last)",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/requests/adapters.py:486\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[0;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[1;32m    485\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m--> 486\u001b[0m     resp \u001b[39m=\u001b[39m conn\u001b[39m.\u001b[39;49murlopen(\n\u001b[1;32m    487\u001b[0m         method\u001b[39m=\u001b[39;49mrequest\u001b[39m.\u001b[39;49mmethod,\n\u001b[1;32m    488\u001b[0m         url\u001b[39m=\u001b[39;49murl,\n\u001b[1;32m    489\u001b[0m         body\u001b[39m=\u001b[39;49mrequest\u001b[39m.\u001b[39;49mbody,\n\u001b[1;32m    490\u001b[0m         headers\u001b[39m=\u001b[39;49mrequest\u001b[39m.\u001b[39;49mheaders,\n\u001b[1;32m    491\u001b[0m         redirect\u001b[39m=\u001b[39;49m\u001b[39mFalse\u001b[39;49;00m,\n\u001b[1;32m    492\u001b[0m         assert_same_host\u001b[39m=\u001b[39;49m\u001b[39mFalse\u001b[39;49;00m,\n\u001b[1;32m    493\u001b[0m         preload_content\u001b[39m=\u001b[39;49m\u001b[39mFalse\u001b[39;49;00m,\n\u001b[1;32m    494\u001b[0m         decode_content\u001b[39m=\u001b[39;49m\u001b[39mFalse\u001b[39;49;00m,\n\u001b[1;32m    495\u001b[0m         retries\u001b[39m=\u001b[39;49m\u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mmax_retries,\n\u001b[1;32m    496\u001b[0m         timeout\u001b[39m=\u001b[39;49mtimeout,\n\u001b[1;32m    497\u001b[0m         chunked\u001b[39m=\u001b[39;49mchunked,\n\u001b[1;32m    498\u001b[0m     )\n\u001b[1;32m    500\u001b[0m \u001b[39mexcept\u001b[39;00m (ProtocolError, \u001b[39mOSError\u001b[39;00m) \u001b[39mas\u001b[39;00m err:\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:787\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[1;32m    785\u001b[0m     e \u001b[39m=\u001b[39m ProtocolError(\u001b[39m\"\u001b[39m\u001b[39mConnection aborted.\u001b[39m\u001b[39m\"\u001b[39m, e)\n\u001b[0;32m--> 787\u001b[0m retries \u001b[39m=\u001b[39m retries\u001b[39m.\u001b[39;49mincrement(\n\u001b[1;32m    788\u001b[0m     method, url, error\u001b[39m=\u001b[39;49me, _pool\u001b[39m=\u001b[39;49m\u001b[39mself\u001b[39;49m, _stacktrace\u001b[39m=\u001b[39;49msys\u001b[39m.\u001b[39;49mexc_info()[\u001b[39m2\u001b[39;49m]\n\u001b[1;32m    789\u001b[0m )\n\u001b[1;32m    790\u001b[0m retries\u001b[39m.\u001b[39msleep()\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/urllib3/util/retry.py:592\u001b[0m, in \u001b[0;36mRetry.increment\u001b[0;34m(self, method, url, response, error, _pool, _stacktrace)\u001b[0m\n\u001b[1;32m    591\u001b[0m \u001b[39mif\u001b[39;00m new_retry\u001b[39m.\u001b[39mis_exhausted():\n\u001b[0;32m--> 592\u001b[0m     \u001b[39mraise\u001b[39;00m MaxRetryError(_pool, url, error \u001b[39mor\u001b[39;00m ResponseError(cause))\n\u001b[1;32m    594\u001b[0m log\u001b[39m.\u001b[39mdebug(\u001b[39m\"\u001b[39m\u001b[39mIncremented Retry for (url=\u001b[39m\u001b[39m'\u001b[39m\u001b[39m%s\u001b[39;00m\u001b[39m'\u001b[39m\u001b[39m): \u001b[39m\u001b[39m%r\u001b[39;00m\u001b[39m\"\u001b[39m, url, new_retry)\n",
      "\u001b[0;31mMaxRetryError\u001b[0m: HTTPConnectionPool(host='localhost', port=19000): Max retries exceeded with url: /api/v2/settings/ml-worker-connect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x327fc8640>: Failed to establish a new connection: [Errno 61] Connection refused'))",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mConnectionError\u001b[0m                           Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[223], line 5\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[39m# Uploading the test suite will automatically save the model, dataset, tests, slicing & transformation functions inside the Giskard UI server\u001b[39;00m\n\u001b[1;32m      2\u001b[0m \u001b[39m# Create a Giskard client after having install the Giskard server (see documentation)\u001b[39;00m\n\u001b[1;32m      3\u001b[0m token \u001b[39m=\u001b[39m \u001b[39m\"\u001b[39m\u001b[39mAPI_TOKEN\u001b[39m\u001b[39m\"\u001b[39m  \u001b[39m# Find it in Settings in the Giskard server\u001b[39;00m\n\u001b[0;32m----> 5\u001b[0m client \u001b[39m=\u001b[39m GiskardClient(\n\u001b[1;32m      6\u001b[0m     url\u001b[39m=\u001b[39;49m\u001b[39m\"\u001b[39;49m\u001b[39mhttp://localhost:19000\u001b[39;49m\u001b[39m\"\u001b[39;49m,  \u001b[39m# URL of your Giskard instance\u001b[39;49;00m\n\u001b[1;32m      7\u001b[0m     token\u001b[39m=\u001b[39;49mtoken\n\u001b[1;32m      8\u001b[0m )\n\u001b[1;32m     10\u001b[0m my_project \u001b[39m=\u001b[39m client\u001b[39m.\u001b[39mcreate_project(\u001b[39m\"\u001b[39m\u001b[39mmy_project\u001b[39m\u001b[39m\"\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39mPROJECT_NAME\u001b[39m\u001b[39m\"\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39mDESCRIPTION\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[1;32m     12\u001b[0m \u001b[39m# Upload to the current project âœ‰ï¸\u001b[39;00m\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/client/giskard_client.py:86\u001b[0m, in \u001b[0;36mGiskardClient.__init__\u001b[0;34m(self, url, token, hf_token)\u001b[0m\n\u001b[1;32m     83\u001b[0m \u001b[39mif\u001b[39;00m hf_token:\n\u001b[1;32m     84\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_session\u001b[39m.\u001b[39mcookies[\u001b[39m\"\u001b[39m\u001b[39mspaces-jwt\u001b[39m\u001b[39m\"\u001b[39m] \u001b[39m=\u001b[39m hf_token\n\u001b[0;32m---> 86\u001b[0m server_settings \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_session\u001b[39m.\u001b[39;49mget(\u001b[39m\"\u001b[39;49m\u001b[39msettings/ml-worker-connect\u001b[39;49m\u001b[39m\"\u001b[39;49m)\u001b[39m.\u001b[39mjson()\n\u001b[1;32m     87\u001b[0m analytics\u001b[39m.\u001b[39minit_server_info(server_settings)\n\u001b[1;32m     89\u001b[0m analytics\u001b[39m.\u001b[39mtrack(\u001b[39m\"\u001b[39m\u001b[39mInit GiskardClient\u001b[39m\u001b[39m\"\u001b[39m, {\u001b[39m\"\u001b[39m\u001b[39mclient version\u001b[39m\u001b[39m\"\u001b[39m: giskard\u001b[39m.\u001b[39m__version__})\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/requests/sessions.py:602\u001b[0m, in \u001b[0;36mSession.get\u001b[0;34m(self, url, **kwargs)\u001b[0m\n\u001b[1;32m    594\u001b[0m \u001b[39m\u001b[39m\u001b[39mr\u001b[39m\u001b[39m\"\"\"Sends a GET request. Returns :class:`Response` object.\u001b[39;00m\n\u001b[1;32m    595\u001b[0m \n\u001b[1;32m    596\u001b[0m \u001b[39m:param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[1;32m    597\u001b[0m \u001b[39m:param \\*\\*kwargs: Optional arguments that ``request`` takes.\u001b[39;00m\n\u001b[1;32m    598\u001b[0m \u001b[39m:rtype: requests.Response\u001b[39;00m\n\u001b[1;32m    599\u001b[0m \u001b[39m\"\"\"\u001b[39;00m\n\u001b[1;32m    601\u001b[0m kwargs\u001b[39m.\u001b[39msetdefault(\u001b[39m\"\u001b[39m\u001b[39mallow_redirects\u001b[39m\u001b[39m\"\u001b[39m, \u001b[39mTrue\u001b[39;00m)\n\u001b[0;32m--> 602\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mrequest(\u001b[39m\"\u001b[39;49m\u001b[39mGET\u001b[39;49m\u001b[39m\"\u001b[39;49m, url, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/requests_toolbelt/sessions.py:76\u001b[0m, in \u001b[0;36mBaseUrlSession.request\u001b[0;34m(self, method, url, *args, **kwargs)\u001b[0m\n\u001b[1;32m     74\u001b[0m \u001b[39m\u001b[39m\u001b[39m\"\"\"Send the request after generating the complete URL.\"\"\"\u001b[39;00m\n\u001b[1;32m     75\u001b[0m url \u001b[39m=\u001b[39m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mcreate_url(url)\n\u001b[0;32m---> 76\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39msuper\u001b[39;49m(BaseUrlSession, \u001b[39mself\u001b[39;49m)\u001b[39m.\u001b[39;49mrequest(\n\u001b[1;32m     77\u001b[0m     method, url, \u001b[39m*\u001b[39;49margs, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs\n\u001b[1;32m     78\u001b[0m )\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/requests/sessions.py:589\u001b[0m, in \u001b[0;36mSession.request\u001b[0;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[1;32m    584\u001b[0m send_kwargs \u001b[39m=\u001b[39m {\n\u001b[1;32m    585\u001b[0m     \u001b[39m\"\u001b[39m\u001b[39mtimeout\u001b[39m\u001b[39m\"\u001b[39m: timeout,\n\u001b[1;32m    586\u001b[0m     \u001b[39m\"\u001b[39m\u001b[39mallow_redirects\u001b[39m\u001b[39m\"\u001b[39m: allow_redirects,\n\u001b[1;32m    587\u001b[0m }\n\u001b[1;32m    588\u001b[0m send_kwargs\u001b[39m.\u001b[39mupdate(settings)\n\u001b[0;32m--> 589\u001b[0m resp \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49msend(prep, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49msend_kwargs)\n\u001b[1;32m    591\u001b[0m \u001b[39mreturn\u001b[39;00m resp\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/requests/sessions.py:703\u001b[0m, in \u001b[0;36mSession.send\u001b[0;34m(self, request, **kwargs)\u001b[0m\n\u001b[1;32m    700\u001b[0m start \u001b[39m=\u001b[39m preferred_clock()\n\u001b[1;32m    702\u001b[0m \u001b[39m# Send the request\u001b[39;00m\n\u001b[0;32m--> 703\u001b[0m r \u001b[39m=\u001b[39m adapter\u001b[39m.\u001b[39;49msend(request, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs)\n\u001b[1;32m    705\u001b[0m \u001b[39m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[1;32m    706\u001b[0m elapsed \u001b[39m=\u001b[39m preferred_clock() \u001b[39m-\u001b[39m start\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/requests/adapters.py:519\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[0;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[1;32m    515\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39misinstance\u001b[39m(e\u001b[39m.\u001b[39mreason, _SSLError):\n\u001b[1;32m    516\u001b[0m         \u001b[39m# This branch is for urllib3 v1.22 and later.\u001b[39;00m\n\u001b[1;32m    517\u001b[0m         \u001b[39mraise\u001b[39;00m SSLError(e, request\u001b[39m=\u001b[39mrequest)\n\u001b[0;32m--> 519\u001b[0m     \u001b[39mraise\u001b[39;00m \u001b[39mConnectionError\u001b[39;00m(e, request\u001b[39m=\u001b[39mrequest)\n\u001b[1;32m    521\u001b[0m \u001b[39mexcept\u001b[39;00m ClosedPoolError \u001b[39mas\u001b[39;00m e:\n\u001b[1;32m    522\u001b[0m     \u001b[39mraise\u001b[39;00m \u001b[39mConnectionError\u001b[39;00m(e, request\u001b[39m=\u001b[39mrequest)\n",
      "\u001b[0;31mConnectionError\u001b[0m: HTTPConnectionPool(host='localhost', port=19000): Max retries exceeded with url: /api/v2/settings/ml-worker-connect (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x327fc8640>: Failed to establish a new connection: [Errno 61] Connection refused'))"
     ]
    }
   ],
   "source": [
    "# Uploading the test suite will automatically save the model, dataset, tests, slicing & transformation functions inside the Giskard UI server\n",
    "# Create a Giskard client after having install the Giskard server (see documentation)\n",
    "token = \"API_TOKEN\"  # Find it in Settings in the Giskard server\n",
    "\n",
    "client = GiskardClient(\n",
    "    url=\"http://localhost:19000\",  # URL of your Giskard instance\n",
    "    token=token\n",
    ")\n",
    "\n",
    "my_project = client.create_project(\"my_project\", \"PROJECT_NAME\", \"DESCRIPTION\")\n",
    "\n",
    "# Upload to the current project âœ‰ï¸\n",
    "test_suite.upload(client, \"my_project\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "<div class=\"alert alert-info\">\n",
    "Connecting Google Colab with the Giskard server\n",
    "\n",
    "If you are using Google Colab and you want to install the Giskard server **locally**, you can run the Giskard server by executing this line in the terminal of your **local** machine (see the [documentation](https://docs.giskard.ai/en/latest/guides/installation_app/index.html)):\n",
    "\n",
    "> giskard server start\n",
    "\n",
    "Once the Giskard server is running, from the same terminal on your **local** machine, you can run:\n",
    "\n",
    "> giskard server expose --token \\<ngrok_API_token\\>\n",
    "\n",
    "Read the flowing [instructions](https://docs.giskard.ai/en/latest/cli/ngrok/index.html) in order to get the `ngrok_API_token`. This will provide you with the code snippets that you can copy and paste into your Colab notebook to establish a connection with your locally installed Giskard server\n",
    "</div>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 224,
   "id": "39789833",
   "metadata": {},
   "outputs": [],
   "source": [
    "from giskard.testing.tests.calibration import test_overconfidence_rate"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 225,
   "id": "d7ce359e",
   "metadata": {},
   "outputs": [],
   "source": [
    "a = test_overconfidence_rate(model=wrapped_model, dataset=wrapped_data, debug=True).execute()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 226,
   "id": "8f45092d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>transcription</th>\n",
       "      <th>medical_specialty</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>3419</th>\n",
       "      <td>CURRENT HISTORY:,  A 94-year-old female from t...</td>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2665</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Chronic venous hyper...</td>\n",
       "      <td>Neurosurgery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3295</th>\n",
       "      <td>S:,  The patient is here today with his mom fo...</td>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2552</th>\n",
       "      <td>Pap smear in November 2006 showed atypical squ...</td>\n",
       "      <td>Obstetrics / Gynecology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3880</th>\n",
       "      <td>ADMISSION DIAGNOSES:,1.  Severe menometrorrhag...</td>\n",
       "      <td>Discharge Summary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2593</th>\n",
       "      <td>ADMITTING DIAGNOSIS:,  Intrauterine pregnancy ...</td>\n",
       "      <td>Obstetrics / Gynecology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1299</th>\n",
       "      <td>SUBJECTIVE: , I am following the patient today...</td>\n",
       "      <td>SOAP / Chart / Progress Notes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3440</th>\n",
       "      <td>DISCHARGE DIAGNOSIS:  ,Complex open wound righ...</td>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3311</th>\n",
       "      <td>CHIEF COMPLAINT:,  The patient is here for two...</td>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2733</th>\n",
       "      <td>FINAL DIAGNOSES:,1.  Herniated nucleuses pulpo...</td>\n",
       "      <td>Neurosurgery</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>86 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          transcription  \\\n",
       "3419  CURRENT HISTORY:,  A 94-year-old female from t...   \n",
       "2665  PREOPERATIVE DIAGNOSIS: , Chronic venous hyper...   \n",
       "3295  S:,  The patient is here today with his mom fo...   \n",
       "2552  Pap smear in November 2006 showed atypical squ...   \n",
       "3880  ADMISSION DIAGNOSES:,1.  Severe menometrorrhag...   \n",
       "...                                                 ...   \n",
       "2593  ADMITTING DIAGNOSIS:,  Intrauterine pregnancy ...   \n",
       "1299  SUBJECTIVE: , I am following the patient today...   \n",
       "3440  DISCHARGE DIAGNOSIS:  ,Complex open wound righ...   \n",
       "3311  CHIEF COMPLAINT:,  The patient is here for two...   \n",
       "2733  FINAL DIAGNOSES:,1.  Herniated nucleuses pulpo...   \n",
       "\n",
       "                  medical_specialty  \n",
       "3419               General Medicine  \n",
       "2665                   Neurosurgery  \n",
       "3295               General Medicine  \n",
       "2552        Obstetrics / Gynecology  \n",
       "3880              Discharge Summary  \n",
       "...                             ...  \n",
       "2593        Obstetrics / Gynecology  \n",
       "1299  SOAP / Chart / Progress Notes  \n",
       "3440               General Medicine  \n",
       "3311               General Medicine  \n",
       "2733                   Neurosurgery  \n",
       "\n",
       "[86 rows x 2 columns]"
      ]
     },
     "execution_count": 226,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a.output_df.df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 257,
   "id": "77d7ecb6",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Discharge Summary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>SOAP / Chart / Progress Notes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Obstetrics / Gynecology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>Discharge Summary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>Discharge Summary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>SOAP / Chart / Progress Notes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>Discharge Summary</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>86 rows Ã— 1 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                0\n",
       "0               Discharge Summary\n",
       "1                       Neurology\n",
       "2   SOAP / Chart / Progress Notes\n",
       "3                General Medicine\n",
       "4         Obstetrics / Gynecology\n",
       "..                            ...\n",
       "81              Discharge Summary\n",
       "82               General Medicine\n",
       "83              Discharge Summary\n",
       "84  SOAP / Chart / Progress Notes\n",
       "85              Discharge Summary\n",
       "\n",
       "[86 rows x 1 columns]"
      ]
     },
     "execution_count": 257,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.DataFrame(wrapped_model.predict(a.output_df).prediction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 256,
   "id": "1a85de16",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "General Medicine                 23\n",
       "SOAP / Chart / Progress Notes    16\n",
       "Neurology                        11\n",
       "Discharge Summary                10\n",
       "Obstetrics / Gynecology           8\n",
       "Urology                           6\n",
       "Neurosurgery                      5\n",
       "ENT - Otolaryngology              4\n",
       "Gastroenterology                  3\n",
       "Name: medical_specialty, dtype: int64"
      ]
     },
     "execution_count": 256,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a.output_df.df.medical_specialty.value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 227,
   "id": "fe5d3c20",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "General Medicine                 24\n",
       "Discharge Summary                16\n",
       "SOAP / Chart / Progress Notes    16\n",
       "Neurology                         9\n",
       "Gastroenterology                  7\n",
       "Neurosurgery                      7\n",
       "Obstetrics / Gynecology           4\n",
       "Urology                           3\n",
       "dtype: int64"
      ]
     },
     "execution_count": 227,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.DataFrame(wrapped_model.predict(a.output_df).prediction).value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 310,
   "id": "02b1d40f",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_og = a.output_df.df.reset_index(drop=True)\n",
    "# tf-idf of the predictions\n",
    "predictions_transcripts = df_og[pd.DataFrame(wrapped_model.predict(a.output_df).prediction)[0]== \"Neurology\" ].transcription\n",
    "\n",
    "# tf-idf of the ground truth\n",
    "gt_trancripts_ = wrapped_data.df[wrapped_data.df.medical_specialty==\"Neurology\"].transcription\n",
    "gt_trancripts = a.output_df.df[a.output_df.df.medical_specialty==\"Neurology\"].transcription"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 311,
   "id": "a6d0d8c2",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def get_keywords(gt_trancripts):\n",
    "    \"\"\"\n",
    "    This function takes in the ground truth transcripts and returns the top 20 keywords\n",
    "    \"\"\"\n",
    "    tfidf_vectorizer = TfidfVectorizer( stop_words='english',ngram_range=(1, 2\n",
    "                                                                        ))\n",
    "    tfidf_weights = tfidf_vectorizer.fit_transform(gt_trancripts.tolist())\n",
    "\n",
    "\n",
    "    # Get the terms found in the text\n",
    "    terms = tfidf_vectorizer.get_feature_names_out()\n",
    "\n",
    "    # Get the TF-IDF weights for each term\n",
    "    weights = tfidf_weights.toarray()[0]\n",
    "\n",
    "    # Create a dict mapping terms to their TF-IDF weights\n",
    "    weights_dict = {}\n",
    "    for i in range(len(terms)):\n",
    "        weights_dict[terms[i]] = weights[i]\n",
    "\n",
    "    sorted_footballers_by_goals = sorted(weights_dict.items(), key=lambda x:x[1], reverse=True)\n",
    "    converted_dict = pd.DataFrame(sorted_footballers_by_goals)\n",
    "    converted_dict= converted_dict.set_index([0])\n",
    "    return converted_dict.head(20).index.to_list()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 312,
   "id": "c608b865",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['discogram',\n",
       " 'lumbar discogram',\n",
       " 'oblique',\n",
       " 'needle',\n",
       " 'fluoroscopic',\n",
       " 'lateral fluoroscopic',\n",
       " 'proceeded',\n",
       " 'disc',\n",
       " 'disc space',\n",
       " 'l5 s1',\n",
       " 's1',\n",
       " 'lumbar',\n",
       " 'ap',\n",
       " 'l5',\n",
       " 'space',\n",
       " 'diagnosis low',\n",
       " 'fluoroscopic projections',\n",
       " 'oblique lateral',\n",
       " 'using oblique',\n",
       " 'using']"
      ]
     },
     "execution_count": 312,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "get_keywords(gt_trancripts=gt_trancripts_)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 319,
   "id": "9616ed38",
   "metadata": {},
   "outputs": [],
   "source": [
    "from wordcloud import WordCloud\n",
    "import matplotlib.pyplot as plt\n",
    "from sklearn.feature_extraction.text import TfidfVectorizer\n",
    "def pred_word_cloud(all_text,stop_words=stop_words):\n",
    "    tfidf_vectorizer = TfidfVectorizer( stop_words='english',ngram_range=(1, 2\n",
    "                                                                        ))\n",
    "    tfidf_weights = tfidf_vectorizer.fit_transform(all_text)\n",
    "\n",
    "\n",
    "    # Get the terms found in the text\n",
    "    terms = tfidf_vectorizer.get_feature_names_out()\n",
    "\n",
    "    # Get the TF-IDF weights for each term\n",
    "    weights = tfidf_weights.toarray()[0]\n",
    "\n",
    "    # Create a dict mapping terms to their TF-IDF weights\n",
    "    weights_dict = {}\n",
    "    for i in range(len(terms)):\n",
    "        weights_dict[terms[i]] = weights[i]\n",
    "        \n",
    "    wordcloud = WordCloud(background_color=\"white\",stopwords = stop_words, max_words = 50,prefer_horizontal=1).generate_from_frequencies(weights_dict) #stop_words+list(bow.index)\n",
    "\n",
    "    plt.imshow(wordcloud)\n",
    "    plt.axis(\"off\")\n",
    "    plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 320,
   "id": "77f85b7a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAgMAAAEMCAYAAABZZbUfAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8pXeV/AAAACXBIWXMAAA9hAAAPYQGoP6dpAAEAAElEQVR4nOz9Z5wd2XXeC/8r18mxc+5GIwODOAmTyJnhMIoURSpQpGxaybJ8nX1tWbIt27LkK0v3vk6yoiVZViCpyDgkh5MzBjk2Oud4cq74fqjTB2igATQwmET2ox80PNVVtSvs2nvttZ71LMF1XZdNbGITm9jEJjbxPQvxnb6ATWxiE5vYxCY28c5i0xjYxCY2sYlNbOJ7HJvGwCY2sYlNbGIT3+PYNAY2sYlNbGITm/gex6YxsIlNbGITm9jE9zg2jYFNbGITm9jEJr7HsWkMbGITm9jEJjbxPY5NY2ATm9jEJjaxie9xbBoDm9jEJjaxiU18j2PTGNjEJjaxiU1s4nsc8jt9AZtYH67rcimzwlOTo+xvbuP+jp63/RqG0ytEdJ1mf7CxbSyb5vWFaXAFfnjH3rf9mjbx3sJvvPoar03PkCqXEQSBf/nwg+xtbSWkae/0pX1XoWQYfOXiEF88fQbDtq+7X2ckzD9+4AhbEwkEQbhj7buuy1cvDvGtkREylSrdkQg/c89hOiIRxDvYzibeOmwaA+9SuEDFMpkvFdhixN+Ra3h5bort8aY1xkDVNlkul9n8vDexEcwXCgynUiwWiwhAoVrDdjbLodxp2I7LSqnExeVlajcwBqqWRcU0ceGOf8NPXhrmjblZSobJYrHIodl22sLhTWPgPYJNY+Adhu04lEyD2WIe8IwATZLoCUdxActxWCqXuJRewXIdVFGiOxxFkSQM2yJTrZCpVnABnywTVnWims5sMY/julQsE0kQ0WWZXK1KZyhCSFWpWjaZapmCaQAQUjWimk5AUalYJgulAi/NTlIyDaK6jigIbIkmAHBch5JpMpxZwXQcFFGiJRAkpGq4rsNKpUymWsF2XRRRIqiotAVDAMwUcpiOdx9ly8CwHXrCEfyKiunYzBXzVC0Lx3XRZZmQqtHkC1A0DZbLJWzXwXXBch3CqkZrIETFMimbJo7r0BYMA5CqlKnZFpokk/D53+7XuolNvK2QRIHWUJBdLS1ULRPbcbFdB9N2mMnnsR3nLb+Gk/PzZMred58ql7m4vMLHtm8afu8VbBoD7yBc16VqWxxfnOcPzh3DdcF2HdqDYf7Nfe8DoGjUeHZ6nFfnp8nVqkR1Hz9/7yO0+IMsl0t8a2KE52fGsV2XrlCYe9q6eF/3AP/73AlMx2Y0m8GvKPRFYrw0O8k/Ong/+5vbmcpn+ebEMOdSiwDsTrbycFcfe5OtTOdz/P7ZY5xaXmCmmOf0ygKaJPHrj3wEgIplcT61xH878SqpSomI5uNzO/dxd1sXNcviyfFhXpiZoGZbRHWd3YkWfnzvIUQEvnDxNKlqhfZAmEuZFVYqJf7F3Q+xI9FMqlLm988eZyqfo2qbtAVCHG7t4Ie27eViepkvXDxD1fYMhVy1yr7mVn7yrrsZyaS4kFqiYBj8zL67AXhhZoKFUoHeSIwP9m19Z17wJjbxNsGnKHx42zYOdnRQqNUoGQZFwyBTqfL/PPc8+VoN5y0uUCtLIoIgILgukiCgSdJb2t4m7iw2jYF3EC6wWCryb176Nv/7w5+mNRBCveIDEoDlSokP9m7lMzvvwnFc/vY3/pzxbJqKaTKcTfHi7CS/98FPIiDw26de59TyAm2BMIIgcKilg62xJAXDGwj+77sf4uj8DI7jMl3Mka1V+YMPfRqAX3ntWY4tzBLTfGyLJ/kPDzzOT37zr3i0Z4DP7tzXuB6AXK1Kky/Arzz0BKok8bNPfZmpfJYmf4CqZfG1sSH+5+MfJ6rpPD8zwRcvnuHsyhK7E80ASIJAVzjM39t/z5rztgZC/Py9j6CIErbr8lunXuf56Ul+cJvHTZgv5fn01j18dGA7JdPgx77+Jd7XPcBANE7JNPhfZ4/xd7kbAbiYXkaXZd7X3f/WvsRNbOJdAAHPM9gbja7Zbrsu//3lVygaxltuDPydgwf54pmzZCoVdjQ18UN79yCJmxz19wo2jYF3EFXLIm/UPFe4P4AmSQ1Sz+qH2xWK0uQPoIoShmsT1X0Yjs1iuchUPstYNsX//dyTgDdJdwTDVCwTgLCmY+PiAmLdUjdsi/lSAdeFvkisEc/rCkWoWCYzxRxbYwlEQUDAG2Sujvk1+QO4Lg3LP6JquECmWmGpXGK2mOMXX/4OiihSsSwcXMqmwepQlPD52RpLrjmv7TjMlwp84eJpqraFLEicWJpDlaTGs2j1hwirGpIgoIgiPeEYK5UynaEwzf4gdzW18sbCDEFFI+nzk/R5z20Tm/hux/XIgILr3nlywHXwiZ07eHRgAMd1UCWJhN+/yS16D2HTGHgHIYsiqihRtS2qloUuyUiCwJX2uyZJyKLnfkPwVtWuCz5ZIazqhDWdx3sGEQVwXQhrWoPwJ4kCkiAgCt5/BTxvRFBVydaqZCqVRjvZahVZFAkpHstbwIvL266L67prBhulPsGubhPr1r8sikQ0nYCs8khXH0FFxQUUQaQnHG1M/rIooklru16mVmE8l2G2WODjW3ZgOQ6zxTyparluzkDeqFG1LVw8YylTrRCQFVRRotkf4FBrJ6/NzxDRdNqDYfoisTvKmN7EJjaxPgRBIKLrRHT9nb6UTdwmbtkYSGVLLKULFEpVDu/uua3B1nFcylWDSxNLtCRCxMJ+/D71ls/zXocsioQ1jZ5wlBOLc0R1H5IgoMsK/dG4x/itT+JXI6xqdIbCbInGafL7kQQRx3UJq96q+EZYNRYmcllOLM4B3kTbE47SFvCIfoIgENF00tUyxxfn8CkKO+JNjXOsd026JBNRdQZjCVr8QfyKAq53n83+wJpjrr4r03YoGgZFs0ZE07170TRKdYIjeNkVU4UcZ5YXKJoGogBJvx9dVlAkkcFYgq+NDZGpVdiRaKIrFLnhc9jEJjaxiU14uGVj4PTQLF97/hwnLk7z7d/5+7dlDNi2w8xChv/429/kE+/fy5H9/fR3JW/5PO91iIJAXPfz43sO8b/OvAF4K/eOYJh/dvhBJEHEJ8nIV7i6/YrSMCICisqRjh5+4+SriIi4uOxrbudHtu9Fk2RkQUQRJVRRQhIEZFFEl2Q6QxHiuo+5Yp7/dvxlAPY0tbIz0URX2JtABeBgSwcnl+Y4uTRPezDMLz/4ASRBRBFFXDwCJHixSlWS8CsKLf4QH+jdwv+5cBLTthEQiOk+/t2RR5EkEU2W0ST5mtBDVNfpCIWpWha/c/ooUV0nX/MMlFXDIaConFic49zKIvlalbvbOukMRTyjAwgpGgFFQREkwqpGUN3MZd/EJjaxiY3gHQ8TOI77lhNb3s3wyTL3t3dzf3v3NX/bk2xhT7Kl8VsVJf7zwx9as88nB3fxycFd1xz7Tw8dWbe9/c3tjf+9s07oux4+v/sAcKDxWxAEtsaSmLaXDungSVj+wr3vW3PcxwZ28LGBHeue82f33bvudl2S2ZNs4Q8/9Klr/rbaPxI+P0/0DvJQZ++653Bch+VyiU9v203TFdoIm3jnsMo72cQmNvHuxjtiDMiyyEB3E7/5b38Yn6agqe+4TfKO4WrPyng2w8XUMkvlEkvlIofbOuiPxrEdlyfHh7Fsmwe7e8lVq1Qti9ZAiJlCjrCm0RYM0eQPENEux+1KpsHF1AqvzE7hui4fHtjqnbtUIlUtk6qUeaSrj95ojIupZd5YmEUSBB7v3UJrMMTZ5UXOLHvph4/3bsF2HZ6aGGU8l2E4k2JPspUdyaY1WRB34jmswVXG4tX7nl5a4LWFad5YmKUzGGZPspXITRTuXNfl+Nwcr0xNM5xKMZcvkK1UqFoWGzVN//1j7+eutjYS/mvDMiOpFE9eGubPTp8B4BcffT/3dXcRUFVc12U8m+WF8QnOLS0xlc2SKVdwcIloOi3BIF3RCHtaW3hicNDje9zg+diOw2KxyKtT01xaSTGdy7FYLJKrVTFsB12WCKoaSb+f/niM9w30s7OpieBtqgCemp/n9ZlZzi8tMZnNkq1U8SkyzcEgA/E4B9vbeXTLAIooIovSbTPKDdvm/OIipxYWOLOwxHgmTa5aQxQEfIpC0u9nb2srHxgcoDsaJaC+u0ONF5aXOTU/z5mFRUbTGdLlckNTYyP47L67eGJwC33xd0aEDKBmWbwwMcmvvfAiRcO47n7JgJ+/d889PDrQ/6YzCnLVKmPpDK9NT3Nybp6FYpGiYSAIEFJVWkMhDnd28kBPD52RML66p/B6eGp0lK9dHOLMwiJ9sRi/9f0fJ1ut8p2RUZ6fmGB4JYVPkbm3q4sPbh1kR3MzqiThui5fG7rE02NjjKa8jK62UIhP7trJ/T3dNAUCN72X5VKJSysrXFxe4dLyCuPZDPlqraEa6VMUmgN+BuJx7mpvY2dzMwNv0/u+7Vl4/Uj2Bo8VBBRZoim2uXq7GkWzRqZaIe7zsT2R5OzyIoulEgFFRRSgJRDk5ZkpbMdBk2XyRo1srcpiuUjc50O/iph3bmWJVKXM3uZWwqrG8zOTiHgExNZAiF3JZl6fn8GnKGSqFSqWxUOdPbQEQlxMrTBfLKBJ3mTy5Nglvm9wBzHdR9UyOdjaQZMvgPQWk/REQaAvEuNTW3fTURcVuhLtoTBHxB62xpLEdR8hVUUU1h+AbMchXanwlQsXeX5igvlCgWylStk0qVkW9i14qWqW3QiVXA3TdsjXaswXCgCkymXKdeW3lyYm+ZsLF5jK5ciUKxQMg5pl4bouqiwzlc0ynEoxk8vxwcHB67ZfNAzGUmlemJjk7NIiC4UiuWqVomFQNk2M+kQjiV5oZ1JRGE6lOLu4xIH2No709rC/rQ1lA4ac67pULYu/Pn+BZ8bGmMnlSVcqFGreQCaLIvOFAuPpDGcXFzm9uMDn9u1DFIQNnf9KOK7LWDrNVy8OcWFpmdl8nlS53GiLeshrOpdjOpfjzOICd3d2cqizg31tbe8qxTvHdTFsmy+eOcPRmVmmsjlS5TL5Wo2qad5Sfysaxi3t/1ahYprMFwo3NAZsx2n099vB6nEvTEzw6tQ0F5eXmcsXWKl/R1Z98lQkielcnqlsjqMzs9zV1sp93V3sbW297rmrpkWqXGY6l6NqWdQsiz89dZqXJ6cYSaUaZOpctUa+VuN9/f082NvD8+MT/MmpU4ynM+RqNWzHYblUomh4/fKerk66r0rtBE84Llup8lfnz3NhaZnFYpFspUK2WiVf79OrBqEsiszn84xnspycX6AnFuVgRzuf2r0bZZVI/hbhNo0Bb6VSKNeYms9QKFYxLRtJEmmKh+htj6Eq8jUXPrOYZTFVIJcvN7Z1t8dpjocIB9dnobqui2U7zCxmyeTKlCoGtu2gqTLNiRBNsSChgLamrbmlHOlcCdt2GOhuYmx6hUKpimU7KLJEIhakvyOBLF/7cE3LJpMvs5Iuki1UqJkWkiSiKjIBXSUZC9Ca9PL4LctmJVvi4vgi7U0R+joTKPK1A9/o1DKpXBlZEtm/o8t7gtd5p6btMfjbAiH2NLXw0swUy5UySZ+fkOqt/lOVMoZjY5teiMV0HEzbxnW5ZoU+V8hjuy77W9pJ+vz8xdA54j4fPWGPgLgtnuSvhy9gOQ5x3U9nKIwkiliuzVQ+S8UyiWg6Tf4Ac4U8IVUjomlYToDBWOKmq9Y7AUEQSPoCJH3rW95eGuHGVAaXSiWOzc7yNxcucm5xEUEQiOo6nZGwZ3CJAhXTJF2ukKlU1ki7doTDtASDBFQFv6LQGgqiyRv7hEqGwUKhSNk0+MrFi3xndAzHddEkCVWW0TStMTBVTJOqbdMUDHgE0us8X8dxmc3n+calS1xYXgZAFECVZHyKQkzXkUQR07Ypmyb5qpdBMpnNslAoYLsuIVVlR/ONw0UA+VqNc4tLfPnCRU7Oz2PYdiNdtSUYQBK8drLVKgvFAjO5HB3hMNlq9ZaqoRm2zXKpxDeGLvGVCxeZLxQwbBtJFIn5dDRJxsVboZYMg9F0mrF0muWSN8GqksTulpabtvN2oWpZnJib58sXLjK0vELFNNEVhfZQiLCmIQoChmNTrBksFApULKtxbEjT6I/HCKkafkVmazJB8B32foiCQEswyJGebgo1A8txsOrjz3gmU1cCffMGi+04nFta4htDl3h1apqZfL7xvUR0HUWScHEbfS69XGE0nWYml6NQq+FXFPrj8RsahpbjkKtWObu4yDOjYywWi8iiSEzXWS6XmcxmsV1v0dUTi/LVoSHOLy2jyxIRTaNkmuRrNU7OL9AWChP16esaA27dIHzy0jAXl5ep1t+xJAiENI2IrjeyyAzLIl2tkqvVmM7lGE2nWSgU2JZMsre19ZYN61vBbRkDguAN0MMTS/z106cZnV6hUKqhKhIHdnbzox85RFtTGF1b6645eWGG594Y4cylWUzboVwx+OxHD/O+e7ayM3itJee6Lq4LCyt5nnzhPGeH55lbzlE1TGIhP3fv6eHBgwPs6G9dk41w7Pw0r5wcp1iu8nc+eT9f+MYxxmZWKFUMfLrCvm2d/O1P3ENTPIiqrH0E6VyZY+emeOXUOCNTy+QKFVRVIRzQ6GyJcv/+Pj7y0G4AaqbN2ZF5fvX3vs3779nKz/zQg4SD+prB23YcvvbCed44O0ks7G8YAzfCaidZndi91D7vQ3SB+zu6GctlmC96q05F9DIJxHUmDkEQwHWwbbsuTSwgIKBKntSvUnd/BVWV7nAEXZb5+tgQht1H1bYaOgOiIPBY7wABRWm4/UqmgV9WLqc+vsthOQ6n5xf4oxOnOLvohT6Sfj8H2tu4u9Oz6jVZYr5Q4NjsHMdn5xjPZLBdFwE43NnBBwcHGUwkaAt7AlEbve9stcqJ+TnOLS7xzeERAopCUPNc9wm/H1WSWCmXWSoWKZsmUV3nQFvbDc+pKzKiKJKulFElCV2W8SsKcb+PjnCEtlAQVZLJ16rM5vOMpTOkymUM22Ysk+HbIyO4rsvWZHLdvnPlcxtPZ/j9Y8d4fWYG8IzOsKbRFg6xu6UFn6yQr1aZK+QZy2RIlSv8z1dfJ+733bBwzpVwXJdspcoL45P8j9dex6x7HEKaRtzvZ29rC0m/H9txyVQqjGcyTGQzFGsGZxcXSZfLpMpl/s3734cuX7sYebthOw4rpRK/c/QNTs0vNDJkemJRPrR1K4OJBIrkrUDH0mm+MzrGxeVlLMdBwDM+f2z/fva2tDQMz3daxEcWRe5qa+XnH3mYgmFQMkyKhkG+WuU3X3udyWx2jUFzO7Adh3y1yu8dPcZ3RkepWpbXD1SVtlCInS0thDXV6y/VKueXllgseN/NuaUlZuuGw8/eew8+RbmhQVCzLP7k5GlSlQr98Tgd4RBl0+Kp0VGqpslMLs8bM7O0BoN8a3iEnmiUHc1NSILIZDbLqfl5bNfl1elp+mIx3t9/bVhEqvfhVLmEKAgEFAVNlglqKgPxBG3BIJoi4zgumUqZE/MLrJRK1GybfK3G8bl5/vfxk/z7xx8l/BaOtbdlDLguVA2Tf/s/vs6DBwf42CN7EEWB4YllvvbcWdLZIp/92GH27+hac+GP37+N+/f3kcqWODU0y3//k+dv2k6pXOPn/r+/YTldYudAKx9//x7iET8XRhf52vNnOTM8y/vv2cZnPnJoTVvlisGF0QV+8X98jYcODbB/ZyeGYXNpYomvPneWbKHMT336CFu6mxrHua7LH335dY6fn0YQBD7y0G6aYgGyxSpTc2nypSrp3OXc/IBPpaslyt5tnTzz+jCf+9jda7wUruuylCoyNr2CIAgcOTBwXY/AjbA90URHMMyfD51lKVRiezyJLskkfX6ytSrtwRDjucy6Lrm72zo5sTjHb5x4DV1RuLejm+Vy8Zr9Zgt5htIrXEgtY9oOLvBgRw9HF2Y5vjhHayDE7mQztuvSGgiyUCzwGyde4/GeLextbkGV3v28j5GVFK9Nz/DG7CzgkTd/8dH3c293F9Gr8qM/vmMHZxcX+YVvP8XF5RVcIF/1dA5647Fbbvu16WlKhsnwygrNgQD/7MEHeKC3h7jPt2bwKJsmQ8srLBaLdEciNwzGqZLEQDzGD+3Zy3AqxUe3b+OutlaaAgHkqwYkw7aZymb51ede4JnxcRzXZSSVRhZFPrd/Hy3B64fsRlIpnh0f5+mx8ca2Iz3dfGz7dj66Y/uaMFHNspjLF/jV51/g6bExForX9rXroVCrcXxujl9+9lnMugGxr62VD24d5NN79hBQlDXfeNWy+MbQJX75uefJVirMFQp8a3iEva2t/MDuXW+ax/JmsVwq88bsHC9MTACeZsgP7N7FTxw6SGsotGZfx3X5sQP7+bEvfonhVBrDtqlaJsulEn3xd49ehiAIaLJMR2Rt2q7ruvzNhQvMX+XduFW4rstCscjXhy7xtaEhwFuM7Gxu4nP79/OJnTs8UuoVY6ztOPzJ6TP8nxMnGUunyVar/PbrR9nT0sJ93d1EdO26z892Xb588SK//qEPcm93F62hEFXT4r++8gp/fPIURcPg3NISl1ZW6I5G+Z1PfoLWYBBJEJgvFPipv/obLqVSZCoVZvI5ZvJ5eq7yDoiCgF+R+aG9exlJpRlMxDnc2cme1pZrXP+u61I0DH7p6Wd5aWqq4Rn7+qVL/PQ9h9FiMfSbcCJuF7dpDLjgunzmw4d4/P7tDRd/uWogiPD8G6OcHZ6nvTlKa/JyjFdVZBRZxqcpZPKVm8b2MvkyX33uLFPzGX72Rx7i8J4e2pq8kpiP3ruNrrYYT792iReOjbJ7Sxt7t3U0HqzlODgu/MDj+/jow7vx6QoukM2XURWJZ16/xPnRBWJhP8lYENd1MS2b0ellggGNRw4P8v2P3eWtxl0X2/HEdyRp7SDbFA/y+L1befnEKEfPTXHv3t419/zS8VFWMkW622I8sP/m0rh7mlrYmWxqpBP+3f2HEQURSRDYkWy6rPxXFxly6x4B23XXHfya/QEe7Rngwa5erwiSKOHgIuLFXQXgF+5/BFWSsV2HBzp7wPWILJIg0B4K8+GBreCCJssoosi+5jZ2JpsxbZuAol4z8bxbcWphgeHUCuAx3D+xcyfbm5KE1yHSyaJIVzTK5w8e4F8++S1cYGhlhVPzC3xs+/Zbbvvk/AK6LLOrpYX/9KEn6A6HUWX5msneJ8vsbmlmZ3PThlaBvbEYP3n4EHZd9U0WxXW/K0UU6YlG+XePP8rp//PHpOsFZYqGwbG5OT60det1DY9Xp6f59shI4/eRnm4+u28f93V3XRMCUCWJrmiEX/7A4/ydv/wrRlIpyqZ50/sAeGVqmr84e66x/33dXXx23z4e7utFXycco0kSH9q2lXSlwhfPnGEklaZoGPzeG8f40NZB5HqBrXcKM/kcr09PN34/0t/HvV1d6xLNBCCgKPzk3Yf5j888x3KpRKpc4YWJSX7i0MG38arfWZiOw0gqxf9641hj24e3beWj27fxYG/vun1UEkU+vXsXIVXla0NDPDM2jgv8/15+hc5ImLDect2+LQiwq7mZ/e3tjfeiSCLfv3Mnf3nufIMXocoyn9u/j4Tf3xjvgprGo1sGGMtkMGybfLXGbO5aYwA8g+BvHdiP7TgNDs/1xs2AqvLPHnoA94UX+PrQJSqmx/05Nb9AzOej7S0yBm57FBcEgf07O4mGfOiagq4phAI69+3tRRRgdinH3FLummNEUUAURSTx5h9puWpw4sIMsbCf7f0ttDdFGtkHAZ/Gffv6aE2GPU/Dpdm1B7te4YwDu7oIBTR0TcGnKURDfu69qxfHcZmaz7CYKjQOkSWRcFCnUKpybmSe0ekVZEmst6cS9Gv4tLWrk6BfZ+dAG7GQnxPnp1lKF9ZcxvEL02iqTE97nET05mxTRZLQr3C9+xUVXZZRJK/6X1BVUSUJTZLRZS82rNXdw+t1LkkUUWWZoKISUrxjfbLScDkKgkBQ9UISej10ENI0FFFEEr1qh0FFJaCqDStWkST8skJY098zIQLwmLyZSrXxuz8Rw38dN6L37BW2JBKNbdlqlZVS+Zp9NwLLceiPx/jYju30RCJosnzdsI4iSWiyvKFnK4siflUhpGlr3ul69yOLIkm/n90tLfjrsWfL9khQ1yNCpstlJtIZprKXv+UPDG5hSyKOto4rXqirXUZ8Oo9tGVg302I9FGs1Lq2scHphoeHh+tDWQXa1NOOrewTWa0uTJN7X30dHOIwsiliOw1yhwMXlFUo3ILi9HSjWDOYLlz0jHZEwyYB/XSNPEAQkUWQwkWgY9VXTZC6ff9uu992AiUyG0wuLpOvqqE0BP4c7O9nf3t4I/VzZD1Z/+xSF+7q7ONDe3sgkmsnlOLOwyPwNnqGIwEA87i1+6u9FFAQ6IuE1Y4Msil7Mvv59rb6v1lCosU/NtshVq+u2szqehDQNv6Kg1EOM6/VpURCI+3wMxONrOAgr5RK1NxmCuRHelDHQ0RxFli+fQhQFejoSyJJEtlAhk7+9gXMVhmkzPZ8hEQsQDfmv4SC0N0WIhnyYlrff1ZAkgc6WaEMuF7y0xt72BKIokM6VyRerjfsRRZG7tnXSmggzs5jlmy9e4KlXhzgzPMdiqoBhXvsiFFkiEQ0w2NvM6PQKi6kCNcPCcVxyhQoTsymSsSA97fE3nUK52lFudfJddavdiIx25fmvbmO9dm/3Wt5JVOoZA6uI6r4bejVkUVwjr2pYFlVrY6vc9c7VE41xf3fXupPo24HVAawrEmnUlVj1DlwPC8Uiy6VyY7Xuk2X2tLQQv8Ekv6qaeaijfcPytIvFUiNrADxZ7d0tLTRtwJjojETqxE7PwKlZFsMrKUrG7b2rOwWPuHn52YZUbV0Px5WIXuHNsF33HTdo3m7M5HKMpFJY9ZLL/fE4vbHohozK5mCQvnisMYFWLYtLqdQag+xqCIJAayiELK4d2/yKsmaBJYsCnZHwGkNOFATCmtbwOli20yAHvllIokjSH1jT/0uG2XgubwXehDEAQb+GcEUKl4BAOKAjiALVmkm19uY+Rtt2yBbKRIN+ZOnaS9VUGb2+Us8WKmv/WCc5hvz6mji9KAgNkl+lalC9asB49N5tPHhoC+GgztOvDfFbX3yRv3zqFK+fmWRm0SMvXrmKEgTPwLh/fx/LmQKzi1ky+TK24zA6vUIqV6a3PUF/5+0pLNquheNujID13QDXdev37FyzWnVdF8d1cNz1P4jVY911jgWuUcARGxuvjzUSym/C+PErCp2RMIPJd15p80ojyHVdatb1+9dUNkeudtlgTgYCtIZCDdXHG2FrMklIUzeUhDyeybBSKnsS3EBXJELSH7hpfFQQBFRJoikQIO7zNbaPZtJrJuJ3BMLVP4VbSsler0jYdzvmC0WmstnG7z0tLRsueCQKAi2BIDtbLmfHjKXTLN6AtyIAsat4O6vwKXIjlKqIEjGf7ypJddaUaV7N7LpTCNQ9squoXZGC+FbgDrC+Vj9fD97Frm67Ax1ZALdRquaqluttrft8bvDMHPfK8629xpZEiE8+dhdPHNnB+dEFvvbcWV4/Pcmzrw3T15ngpz59hHvv6l1zjCxJPHrPNv70a8e4MLpAR0uU+/f189wbIyiyxM6Blts2BlK1BXxSgJBy66S1W8XVE+itTHxrDaQ3995z5jK6FEQVdWQuTwY2FjW7DLgE5Oi114BD1ljCL0dQRR2JtRyKuM9P+ApBpqlcjpptXVOIaRU1y2Iqd9k9Hta0NRPOraAzEqYlGHhLBvfGk9/gQCFJa42a64UIAGbyefK1mnecINAXi22IIyIIAjGfj7Cmo8nyTVdMq5kO4A3q/fEYsiTe8NquREjV1qTdLRdLG85ieKsQUFSaApeJmQvFAtlq5br9zXVdpnI5LMe7bk2Wb0js/G7EYrHITO6yW78vFluX03M9RH06vbFo4/d4OsNSqXT9A+qr+/W+S1HwDAFZFAmo6nX2ERpTiIOLcxNjwIUNf6eCKKxpc6Pfwu3ito0B14XlTJFkLIgsXWZ2LmcKOI5LKKARCr45bXhFkWhLRlhYyWOus3qp1EyK5Rqu66wh7a3CcVxWskWS0UDj47Mdh6V0AddxiYR8BP3r5+36dYX9OzrZM9jGcrrI3zxzmueODvObX3yRe/b2rvE2CALEIn4O7+5meHKZUxdnObCji+feGOGevT20JsNI0u1NAufzL9Ph2/K2GAMpY468maY3sOuWRaXmq2PYrk2Hb8ubEqRycRkqHKPTt5W42oosXzYGLMekYnsf9nrGgOVaDBWO0RvYTVxtQZLWuhb3tbUymkpxcn4eF/jSmbPc3dVJ3OdbVy9goVDkD48db0y2W5IJ9rTeXg57SNXwKW9hjrjrcmF5mQtLy0xks8zl8ywXS+RrNU+AyLaxbBvDcSjWahueKAu1Gkb92xOARMB/SwZNQFXwbcAYyNUqlOshGNt1+calYZ4eG9+w69Kw7TWrsnyt9pa6VDeCzkiYe7s7+ebwMADfGRmlPeSlYq6nlmg5Dr/3xjFyVc/4ivt8HOnteVuv+Z1G2TQpXBEaifl9Nw2tXAmfLBPTLxvshVqNyg0IrAKgytINR6zbEc66ERaKRcbSac4uLjGaTrNc9ISLyqZFtf6tGrZNpa478nbhto0Bx3E5dXGGBw4OIPu8Sd+yHY6emaJmWLQ1helofnNV4/y6yv4dnXz9+XOMT68QCfqIRy4P8MfOTTG3lCMU0Nkz2H7N8aZlc/LCDA8fHkRTvWGlZli8cdZT8OtsjdKS8FJ8rjS6hHqIQZYEZEmkJRmipz1BU3yByfk0V3tDVmOkd+/tZWYxy+j0Mi8eHyWTK3N4Vw+JK4yRm8FyTApWmkuFY/ikECUrj+kaZI0lRotncFyb7sA2qnaZgpmm4hTxS2Es16BV70MApkpDuLgMBPdSsYuU7DwVu0DZKtAT2E5EaSJrLDNTGUYQBPoCexARmCidZ7YyQtHM0qJ3E1LipI15VmqzWK5FUusgqbazXJthpTaLg83W0EEc12akeIqimSVrLNOsdxFVkkyXL5EzV5AEme7AdgJShLHiafJWGlXU6fZvJ662XvNsHNdmqnSB6fIQPilAu6+fpNbJSPEkrmsTUZpw1TZqToWh/OtU7DKyqKBLQWzXYrx4mglBJCTHaPX10KR52g6DySR7Wlt5dXqGyWyW5VKJ33j1Ne7p6uSu1jZaQkFUUSJdKXN+aZk3Zmc5Ob8AeF6Bwx0d3Nt1c52I9SCJ4lui1JiuVDi/5IkBjacz5GtVKqZXEtuwLCzHE7Fy6qWoHW5thWFYNvbqpCp4btFbMfZUSULewEBaMa1GOiF4E6P1JuLltuNsdAH2lqE5GGR/WzuDiQQjqRSZapWvDV1isVTiff19dIbDKJJMyTCYzed4ZWqaY7OzVC0LX10050M3UKD8boPjut57v6IfqOsUNbsRJFFEu0L4rWpZNzV8xQ14sd/sl+u4Lifm5vjW8Cgj6RTLpRKFmkHZNDAsu5795qVJrnqu3+6aPbcpOuRlBbx0cgy/T6Up7jEqlzNFXjs9QSIaoLs1RvIKuWHHcTFMC8uyKddMcsUqrutSrpnki1WyhQqyJKKrMpLkMTYDPpV79vbyzOvDvHRynErNpK8zia7JpHNlnnntEsVyjb6OBNv6Wq66Rq/Nl06MEQ7qxCMBHNdlfjnHa6cnaEmE6G6LEQuvGhcu6VyZi2MLIAhEQj78uscmzZeqTM+nqdTMuvGwftfY3tdCMhZkaHyRV06OE/CpDHQnCQY27iEp2TlSxjwgoIk+XFwKZoZlYYailUEVdabKQ9TsMo5ro0sB8maKsl0AF3TJjywqqKLOTOUSlmviuA6iIOKXwsyURxD9MnkrTao2T7uvH1lQERFwXQfbMfHJQRRRw8Emay6TNhZp8/WjiT4kQUISPAJcxSqxUJmgRe/BcS1cHPxSEEVQWa7NkDWXqFglZFFlrHiGweABVox5TKdKUI8iC9ePB4uChCrqgMB0+RJJrQtZUCjZFapOCReXnLlMxS5huga2YzeCSaIgoYl+TMdgtjzaMAZCdVLaE4Nb+Itz50mVy5ycnydbqTKSSpPw+5BFiXy1yngmy0QmQ7ZaJaxpPDrQz6GODtquyg/fKDwD87YOvS6mczlOzS/wndFRXpyYJFOp1AVOVBI+PxGfTqCeQSKJIrIoIAoiQyvLTGVzGyI7CVeNk+4V//9OwmvmckMJv4+4z7+hrKP10BWN3tKK8q2AJst0RsJ8fOcO/ujESVLlMhOZDAWjRqZSoTkQQBYlqpbJUqnEucUlMpUqQVXlrrZW3tffR99taFq8l7FKdl41WNcPEF8fV1ZThY3xfN5qMm/FNBlaWeEvzp7j1ekZFotFapaFJsu0hUJEw3o9K0xCESVkSUQSROYLeWbyeRZuQIC8k7itr0WWREIBnan5DM8eHWnUGJheSDM+m+LhQ1vo70oSCXruGtd1qdQMZhayZPNlihWDkakVbMdlcSXPpYklDNNC1xV6WmPEIwEURcLvUzm4s4s9W9s5c2mOVLbEQOcKoaDO5FyaU0Oz9HbEObCri572tcUcvE4Ak/Npnn7tEslYEKOedTA+m+LRe7fR0xYnFLgcQ05lS3zjxQvUDIv25gixsA9ZkrxrnFzCsmweODBwXTOxozlKV2uMoYklTg3N0teZoCUZRlc3nheaN1Nkaot0+bcRV1uZKJ8jYyxiuxY+KURMbWaseJqaUyGiJGn3bWGuMoLtWCzVpomrrWwJ7iOkxHhy/g/QJD8xtZmk2kGbr58n5/+AJq0TERGfHKRN7/MmcFEjrCSIqS0MBPciIFKxixhODUmQ2RG+G4CaXSGkxBEEUIxFFqqT9AV3E5Lj+KQQfcHdCIiMpE/huDYhJY4qaowWT9EX2I0sKPiUAM1aFz75+vHQuNpKk9aJ6Rq8lvo6B2OP0ax3M1seoeZUcXEpmjk0yYdfiKCIKuBSc8o0aZ00692s1GYZLhy/3CeArckEPmUXk9kcz42PUzY81bJzS0uN/cR6Gp4qSTQHA2xLJPnMXXcxmExsWH74rYZh2xybneWrF728avCMndZgkO5olG3JBN3RKE2BAH5VQa+XmVZEkT88cZJ0ubIhY0CVJKRVkrDr8ShuZXi2N1iVdDWVEuq1KGIxDnV03LTozPXQEQ4T0d/ZEtYCnkfpR/buYWh5hVenp0iVK8zlC8znC43nuJrB4xEh/fTH43xs+3Ye7uu97ft/L2L1GaiS1OibNcu67JnaAGzHoXpFSFmXZdR3UAfFqtcv+PKFi3zp7Dkc10WXZZqCATrCYQ61d9AVjZDw+wlpKj5FRZclNEni+YlJnh4de/caA4IgEPSr7Nveyec+dpg/+srrPPXKRbKFCgGfygcf2MHnPnY38cjanPrhyWV+64svcvrS3Jrtr5wa55VT3mAmIPBTnz7Chx7cSVM8iCgI6JrCz//UE/zVd07x+ulJnnzxAqVKjeZEmMfu284jhwfZ1nuttrqX7aDz8z/1QX7nz1/itTOTlCsG4YDOBx/YwY9/8v5rVuyJaID25ginLs5wYWyBfKmKKkkkYwF2bmnj7j09PHrPthsKWOwcaGVmMcOLx8Z45PAg6jq1Cm4Ex3WwsVEEtb5ilzDdGo5roUpBJEFmd/QBZivDGHYVSZDqE6GXeeDiIosqAoLHyMf1JmA5VF+JO/jlEO1KP2ElzrHsU2wJ7qfLv7WxfHVdp54l4qKJPizp8mp4vHSGnLkCCISUOJ7jeVX2WGhIJ1tuDVlQUUQFnxzkYPxxYmoze6JHmC4PcSr3HL2BXfQH9q7rdtalIJrkw7FtXJyGxe/iIriea6/TP8jX559HlwJ0+bfRG9hJypjHJ4VQRR1RkBrXtwpVkuiPxfhPTzzOZ77wJcYzGSzHQRZF7LqA02oltC3xOPf1dPORbVsbugDvFkxms3xjaJhnr1AF/Mi2rfzQ3j3saGq6bozTqUtPixtccQfquhbgrbpy1eqGJvfV1VnZNDdkdKzqJEC9VHYyyecPHiC5gUpw72aIgkBE1/mlDzzGv/rmt3hlappstYomy1iO41VglGViPh99sRiPbhngsYF+or4bp71+t8KvKIRUtdFnrq4PcjNUTIvsFVoiYU17Rw2qbKXCqYUF/vfxEw3jb1dLMx/aupUfuWuvF3a7zrji1UF4+xYft9zSgwcHOHKgH7cu6vOv/+6H6jUE3Eb44Oo0QEEQ2Lu1nf/yc5/CcW48kMiSeI3KX8Cn8iMfPsgPffBAo17BalueyMq153EcLw1toCvJL/2Dj3pqfbgIrH+NAPGIn5/81P2NNlYHtEb+vSjcdBB1HBdcAb9P5bH7tl1T++BmSGrtWK7JCyt/SUJtp+aU6fJvI6wkOJV5npjaTLPWjelcG0/t9u9AEOCZpS+gCBo9gZ2U7WsFN7LGMhljibnKKLZrgeuAC0E5iib6+M7Sn7E9fJiwnLjmWEHwPAZlq0DZLjQm8qjSxFxljGeXv8TW0AG2hQ9zIfcaK6U5wkqcFr2HpeoMM5UhMsYiDs6GWbWu65I1FzmXe5nl2gyqqOOTQ7TqPZStArZjkqnNowo3Xwnarst0Nse/+ua3uLSyQmckwid37eSTu3cRq+fECwgN3ogoCG95VcbbwbNj40znc7iAJHqT5z88cj8Jv/+mpLt8rYp9k+9wFd2RCOH6Ctt2XUZT6Q0T8yqmSbZSobyB2P+WeJymgBeyc12XoeWVdzwb4E7AxYtb//2/+QonFxbQJInHBgb4V+97mGQ9h1wAEARE8ATZ3oX97e1CWzhMTyzGcj2zZGQlzV2tbXSEryWIr4dMxStYtIreeIzmdzAjYzSd5qXJqYYh0BeL8cldu/iB3buQb/KeK5b5tqbH3rIxIEnimoQtVdzYylcUxdty16xaTbIkwS0SOl3XE0JSxZvf5mo761Ud3ChqhsW5kXnmV3Ic3NVV1zO4tXPoUoB23wBhOe7F7V0bTfIhCQrhpgSCIKKLPjrcgcb+ISVGb2A3muiFZZq0LgQEdCmA49qIgliPv8OR5CfQpQAuLs1aFw42ASmCJvloEjoIyVEMp0ZQ9mL6fYFd2FfoHHT7t9OsdeO4FpKgeEWPRJ123wAJrR3LMQnKEURBZm/0QUzHRBQkdNGHJMr45CCWYyAg4JOD13gFBAT2RB5AFX0ooooq6jzY9ElCcoK7og9jujUEvPs5lXmOe5MfIShHKJgZJssXOBz/AJoYQBEVWvVeIspag2Yik+Gp0VHOLS1huy4/uGc37x/op8nvf0srgt1pLJWKDZa0LIjsaGrCt6pqeKMDXRhaTt2QYX0lemMxovrlcN9CschsLk9QVddlxF+J88vL5I3ahsIK25qStAaDCHjei0srK8wXCkR1vaGW+F5E0TD45vAwZxYXKRkGDwxu4bP79tEaDL6n1DvfLnRHIwwm4436IcdmZ3mwr5ed3JzE57guc4UCp+bnG9sGE4nb5vncCRQMY41qaV88Rksw0NAvuBHm8gVm84Wb7HXn8O4IgL5HYdsOpYrhpSq6LpcmlrkwtoCqSBzZ378hFvXVkAQZXZTQNJ83UQqXiVWrhEKBtaSY1YkePO+HTwquux9AXPOqQ7qutx/Q2M+LKWs4OIiIdQnbtS42nxREF/24eHm4q9ClAJrox8VBFLz7VsREw7W/uq8sqPU2WSNYtQpBEAhfMYFLgkxC86r3RdTLWg2WY+KTAphOjaKVw3JNYkrzmmN1yY9+VXrhUrHEmYVFSqZZ5xAk6QiH31OGAFCvlVH/UVcWvBknumZZjKXTzOTzG3a9toaCtIaChDWtUXv99ZkZz60dv/Ek/crU9BqX7Y2Q9PvpjcXoikSYyuUoGgbPj0/gVxS2NTW9q0I0t4KqaXJsdo5CrYbrurSGQmxvvn4Y53sd3ZEI25OeYVuxLKZzOYaWlxm4QlnwehhPZzwjsi4y5FMUdjQ10f4OGgOrQmmrWK0ae7PePJpKM5nNki6/ORXfW8GmMfAmYNo2S+kCLx4fxbYdjl+YoVI12bO1jYM7by8FDVZTFa/tLtfbvmafqwyI22njarGea/dZ36r1zimts+3y3uI6BsDtQBQkuvzbmSpfxHYtAnKEnsDOmx5XMC7rxYuC0MjpVevlclev+d2OoKqi1ENdjuuwWCxSqaekXa2m5rouNdtmoVDgqZFR0uXyhklZEd0TcemJRTmz4JV9fm58gu5olJjfR0RbWxHOraeHLZVKvDQ5RaZSud6p18CvqmxNJtnX3sZsPo/tunx7ZJS4309AVUkGAviuI+PsuC41y0unrNaLurSFQ+8KA8J0HKayXjhHEITGtRq2jfIe6m9vF1qCQbY1JemPxzm3tESuVuON2VmSAT8hTWtUF72yaiF44YGXp6Y4OT/fkHDuj8XYmkySDGysPsZbAVWS8F+hL5IqV8hVqxiWfQ2fzK2nFGYrVZ4fH2cklaK0QQ/encB3pTEgCgKSJL4pl//1cGXaimk6TM1n+Itvn6JSM+hpi/Phh3Zyz57eawiUm7izEAWRJr2TJr3zlo6TRWGNLv+XL1ykbJgc7uy4fhGZ1X/1NKV3wyTjKbN5A6NpOxydnmFkJcWO5iaCV+ilu4DjOIym0nx7ZJTfev31W85fPtDWzmKhxNmFRVzg9ZkZgqqKYdt8cOvgGqKbadssl0r8z9de59T8PKZtI7CxZMSDHe04rsPLU1OslMoMrazwe28c48zCAj+0dy93tbWua4RWTIuxdIaLy8tcWF6iYpr8u8cefVcw8UU8giB4Rsvx2Tm+ePosH92+jZZgYN3CUqu/GvU/uLHBcOWY5K79g/eur3r4jrt+hscaqd3rKCSu386qSuzaREC33s7Vbd2oHUkU6Y3F+PzBA/yLJ7+J7bo8Oz7BUrGEaTt8ctfONTVRXNfFcl2+OjRUL2GcaZznbx3cT0c4tKHqn28VEj4/PdHLejvnFhc5t7jE3tZWuqKRNd+p67pULYuvX7rEb75+lGy1uuFv507gu9IY+NCDO3nigR0bJqjdKizLIZcv4/d7pY4fOjjQsPxFYdPSfzdjIB7n8S0DvD4zA8C3hod5amRk3XcmCp7hkAgE2JZMcHdXJ4c6OtjdcnsqhHcSj9bv4dLKCmXTpGJZ/MRf/TUH2j2Rm6ZgAEkQmCsUOLOwwHQuR65aQ5NlfuLQQf7qvFd7fiPY0dyE47pcXF7m6bExAJ4ZG+PV6Wn+68uvsK+tFb+iUDAMZvN5xtIZCrUavbEYbaFgY2K/GYKqyqGODn71iSf4R1/7euN8f30+z1cuDhHz+egMh/GpMgICVdMiXakwXyg0RFt0Weaerq53XHBoFSFN5ft37eSlyUkc1+XM4iLnlpb476++es2+Al6KZdznoysSYWdLEw/39d1U6Kpm2+RrNfLVKsWa4fWH+r+iaVA0zcaEXKjVeGpklKlsrq4OqeBTvKqnQU2jKRBAldZX5DMdh6JhkKlUKBkGZcPrd2XT8ISTcoUG6bNimrwyNYXrugQ01WtHlvGrXjtxnw+foqxLokv6/TyxdZDhlRR/fOoURcPg3NISv/idp/kfr77G3tZWIrqG63qVRM8sLLBSLjeIrRFN4/t37eSj27atW9b97cSWZIKHrT7+9PRpctUaNdvmf584wZOXLnFvdze98RiSIJCrVpnO5Ti7uMhMLo8myzy+xeOFfePS8Ntyrd+VxoAoCqw6sl3X5eS5Gbo74sQi/g2nVN0INcMiX6wiy54WgrhBEuV7ARljlm/N/1eqV2QhNOsDPNH2jxG4OeHJdV0qdpkzudP0BPpIqAkUUfHIZ9V5AnIQn+QR0pZqi0SUKD7JEzNaD7ZrU7HLnMqeZHd4L0E5iPQmnndLMMg9XV082NvbGKBtL3Vk3f1N26ZqWayUSpycn+cbkWEe6e/jp+8+fNMV21uJgKLwyV27CKgqXzx9hlK9IuPJ+XkuLC8h15+RVZc1lUSR/nicn777EPva2nhhYnLDxoAgCPQn4vyTB+7HrHMGarZNpV5iN1OpIAoCtuti1oup9MZi/PwjDzORyfL8xPiGjAFBEAioKgc62vm1D3+I33/jGBeWl8lWq1iOQ7pcplCrNTwzqyGJq4vDJHy+Oy7wdLvQFYW7Ozt5qK+PozMz5Gu1667MASzD8PpbucyF5WWeHh3nw9u28vmDB7wKeevc2Kn5ef7w+EmGUyv1LCovGXe1nUK9TfDc6X9+9iyadLmEtnjFv5975CH2trauWyVwLJ3mS2fO8vzEZL1gmMdbcepx8XS50jAGSqbJU6OjvDw1veb8q/9+bP8+jvT0rCuqJAgCuizztw/uJ6hrfPPSMEMrK15YtlTipcnJxmrfdhwqpontuiiiyI7mZh7bMsAP7NrlGTXvcEdQJYmBeJyfe/hhfvnZ5ygaBqbjsFgq8dTICGrda2Q7DqZjU7NsYj4fP3vvPextbeHs4hJPXhp+W7wD7ypjwHVdajWLS+OLlEoGuibT250knSlRLFWpVE0kUaSjLUoiHuTshVmqNZOAX6OzPUY4qHNuaJ6a4W1ra4ngui6vHRtjeSVPV3ucRDxIczJ0TbuGaXH89BSO45KIBWhuChPwaZw6P4NtOyTjAZqSIfw+ldPnZxBFL33QsmyKZYPxqWUMwybgV9E1BVEUSMZDBAMakzMpAJLxIJdGFzFMm1jETzwWQNcULgzP4zguXe0xImEflu1wfsjTY+jpTBCPBfD73h5GteXUWKlNULGzjW2qeGvFeWRRpmSXqdlVbNdGduX6pF7BVyf02a7NSm0FTdQQEFBEBUmQEBHJmdl6JoHaUDwsWiUM16inJELNqeETfdiup0MgICLfJGtkJJXiwvIyJ+fmG3XiV/OQNVlek5Ln4oURDMsiW61SNAzvX83AdGwOd3Swt7Wl8TG/3ZBEkcGER5b0KwpHZ2aZymbJ12oUagYuoEsSUZ/OlkSCgUScva2t3NvVRdznozkYIJDZeJ/yyTL98Tg/uu8uemJRhlMp5vMFUpUKRcNAlSTCmkZ3JEJ/PMZ93d3c1eaRVU8vbLz/SKJIUFU52N6Otd/m/NIy45k007k8c/k8JcPAsCwcvAIyfkUh6vPRFPDTHAzSHYlwd1cXyrvAQF8ulbi0ssJr0zNM53KYjoNPkQkoKn5FuSbcZLsuhm2Tq1apWhZl0yRbrfKt4RH2tLSwr62NxDrx75JhMpXNNlzkN4LtuqTK1+dxFGq16/JJqpbFfMHT1b8ZHNclV6016ixcjXSlgmFfX39CFASag0EerWf6nF9eZnhlhYlM1ns+htHwpLQEg/TGYmxNJtjZ3MyulmZaQ++OAk+rZY7v7+7mJw4f4sTcHNM5r1R3sVajYBhoskxAVWgLhOgKR7inu4uH+npp8vtZLpVJ+P2svA1EwneVMQAwt5BlejZNuWLi0xUs2yGdKVGuGJimjSgKFMs1DvhVTp6dBqCjPUYiFqBSMbk4PE/NtAgFdUrlGu2tUaZmUvh0hWBAx79OYSLHcalUTI6emCAS8eO6EPBrFIo1zg/NIQgCqUyAStVk60ALS8t5r4hScwTHdTEMi+VUkYmpFTrbYoRDOtWahabKBPwqYxPLnq67KvPGqUmSieCalej5oTlESaRcqdHVHkeWRF47Pk5bS4R4LEA4tLGa8O8GCIKAgjeJe6sTb2CxsSlYBcKKly/suA5Fq0jJKlG2KwTlAH4pQM2psWKsgAsxNU5UjaKKCqIgYrs2jutguiYrtWXa9Q4yRhpFVNAlHfk63dl1XRaLRV6YmOS58XFOzM0jiSJ7W1tpDQWJ6Dp+RVmT3+24nnpYxTJZKBSYzGRZKZcpmSYXlpZ5dmycrcnEdY0Bv6LQHYlwuLMDgG3J5HVrsjuOi2layLKEKG68THLUp3NXmxd7bA0FObe4xEqpTMUycfGq5rUEA2xNJtieaGJrIoFf8wSH9rW1NSajzkh4Q4RTTZZ5fHALHeEwpxcWGE6lmSvkKdRq+BSFhN9HdyTKnpYW7uvu8gz3cJgdzU3MFfIICMT8NxfTEQSBqE/nia2D7GltYTSd4cLSEqdnFqhhU7VMbNvLeIkH/bQGg3RFI/TEomyJx2m/QU6667rYtle3QFHeOoMhXS5zen6Bb42M8OSlYSRRoDcWpTkQJOH3qjnKV3gpXbwMkYppslQqMlcosFQskalUGFpZ4YWJCVpDwXWNgaius7ulhaD25hcMiRuk2AZVlcFEnHSl40230xEOb4jTsS2ZpCsSYV9bGyfm5zmzsMBSqUTFNBER8KsKSX+APW0tHGxv31B57bjfx7ZkU53EKZHw+9aVvd7R3IRfUbBdZ930RFEQiPp8HGzvoGZbbE0kaQpeyxVTJIn2cIjP3LWXnmiUoeVlZvJ5MuUKluvgV1Siuk5bKMj2piYe6e9Dr5NlW0NBHuztZSafoycaxSe/dTyYd5Ux4AIvvDbCobt66GyPUiob/NGfv0p3e5zurgQ9nXEMw+Ivv3qC7VtaMU2b7Vtb2buzE79P5SvfPIWmybS2RMgXK7x8dJR//NOP0dYa5e4DfWwbaEFcZyByXZdK1cS0bPbv7qK3K4Fp2Tz9wkUiYR9NiRDj0yuspIvs3t7Bgb09vHFqgqphoioyTYkgj9y/lT+ez9LeFiPgUzl9boZazZNuzeQrCIBh2CynCvytH7wPXVcYHlvi1LlpohE/bS0Rjp6coFwx6O9pwrEdHr5vK4l48C0dtN4uiIhMlSfxS3500dfIKkgZK2TNLJ2+Ltp8KudyZxAFienyFAPBAXzSDjRprZhQ2SoxUhgmKAc5kztNt7+bdt/6A9Qq6ekrF4f401OnmcxmCagq93Z18i8ffpjWYPCGKpGrJKg/OH6CL1+4wNnFJSzH4emxMX7swD6uN+30xKL0xPbx2f37bvpsLNNmaSFPLBHE51NuqcKlKkm0BoN85q67rr1ux22ExYqFGoV0BTEm4Per/L1779lwG1djZ0vzmprxN8JgMsFgMsFPHj50W221h8O0h8Pc39XFpeA8WwZbEASBatWiUq7R1Hz9id+xHS+z5grCmWU5VMrewiKRvPOrx9X+9uLkJF++cJFnxsYRgPcP9PPjhw6xs7mJ0E1K8tr1/vWVCxf52tAlAF6anOKB3l52rcNXOdDRzoGOawu1XXk9Rn0h9WZI1VsSCf7xA0du+/jbhV9RGv3oB/fsftPnu7+7m/u7u9dsW62bI0piw0j+xUfff8PzaLLMwY52/vDTP3DTNldLen9gYAsf3LIFURQ3FLLe29rKr334gzfd707g3aV36UK1ZqIoEqrqWUaWVSeFBHWCfg1ZlnCBlqYwn/q+g1QqJl/79hlOnpmmXDHRNAVNk+nrbuLjH9oHcNO8TkkSaW0O89lP3cvZoTmeev4C5y/OU66Y6LqCqsrs39PNow9uX/d407T54t+8wf2H+tnS6xVSQvRcco7tYNcrv0mSQFMihCCuDkw21ZqFz6eiqjKPPbSD+w4NsLW/hR/91L385ddP8MrRURaXrlURfK9BFmTCSgRZuGx/uq7La6lXiSlxmrVmTMdkujJN3swTUsJooo7tXuuyjCox9sUO8EbmKBkzgywqjfDD1XDx2Oa/f+w4M7mcJ0ccj/H/fPAJuiLhRnrejSAKAj9y194GcdBxXSaz2Q2r+N0MpVKN114ZYXkxh7VOqe7bQa1q8tLzQ5imd76FhSzH3hhjeTH/riHX3QpM0+Zrf32cStnEMh3KpRrZzI1dp+fOzjA6srTmfk3TolisUshvLOXxdlC1bL5yYYiXJqc8OWKfj1/+wOMc7GgnuAEBJVEQeKi3l49s29bYNpvPU7zNKo6lssE/+MUv8vtfeuW2jv9ewH/+7W/zN0+dZn4p95a2819+/2n+/MkTzC1m39J2bgfvKs+AIMBD9w1y/MwUL70+gs+ncnh/L+l0ac1+rusyO5fh5Plp5hdzKIqEJAs8cM8Wvv3ceZaW8zQlQ3S2eeSU7s44x09PMTGdYtuWFvq6m9acz7IcFpZyvHZigkymRHMyhD+gcv/hAb721BkSsSKtzWEcO4woCjz78hDjUx5Zp1q1AJdzF+eQJJFUpkQwoNHREuWvv36Cro44K6kCHW3XEmXaW6PUDItnXhoiky3R2R4DF1ZSRU6dn8Gs50zfauWudxKrXIDp8iQlq0g1UKEv0E/Byl/e5lRp09sQBIFHWx5npjKNXJFp09vYHtrBZGkCWZRRRQ1ZkFmsLjJdnvREgkLbaNPb8Ut+8kaeNl87ATl4Xde65ThMZjOUDAPbdYnpOj3RaEM9b6MVzVZL8a5WVFMl6bbqmrqui+u4fPmvj1GrWsQTAcJhH47j8OLzQ7iOS2d3nN7+ZpqbQ3zz66exTJu+gWbCER/TkymWlwrE4n5s2yWeDLJ1exvf+eZZXNdl155OYvEAmXSJb3z1FOlUkd6+JsrlGuWSwQvPXcR5xmXHrg46u+KIosCLz13Eshzu2t9DU3OIWtXijaNjVMoG3b1JOjvjmJbNsdfHKJcMjjy8jWrFIJspk8uVyWZK3P/AVpaXC/j9Klu2tqLrCt/6+mkOHO67ZvVeKRuMjiwycmmBWs2kf6CF3r4kkxMrjAwvYhg273t0B+GIn7Onp5meSuHzq7gu2JbNmQtzLC7mSCRCbNnaQrFQ5fgb48zOZBBFgSMPbcO2bV545iKW7bB9Rxt9A8309jXx+iujVCoGLS0RevqSFItVXnxuiFy2TCTq5679PUxPpUiniuRzFXRdYfuuDrZuu7bU9vUwncuSrVYxbBtNlumPxQioqic4s4FzeOXTJRRZQqxrE3gFo26fDLcqGf9ege04TM1lSKWL9HTGaYq/tcJBf/czD6KqMj7t9tzwC8t50tkSlm2zd/v1051//AfvR5YlfPo7n/Z6Nd5VxgDgTeAuVGsWqioRiwYolarEowF8uookiTz20A7CYR+D/S10dcRRZYlkIkjAr3P/YS8dw+9TCYd8CILAXbs6KRSrqIpMdJ38f1EUCId87BhsZaDHKzkcjfjx6SoP3TuIqkoEAxoBv4auKezf081gfzPxaIBoxFuRfuTxPSRiQUJBHU2TcR2P0RsO6ti2QyioE434OXL3lkZdBL9fpaczwUP3DqLrCpGwD5+m4Lqwa1sbg31NNCdDREK3RuB7JyEiEJQDHIwdQhV1InVvgF9a3aYRUaLoko/+QD9hJYxP8qFLOgE5SLe/x8sYECRiagxFVAjJYe6O30tQDhJRooiCiON6JYu7fF0Eb1D9ELyUqAZ7u553fataASOpFOlyGbdezKilnrp3qzBNm9lpj4DVv6WZRCKI47hMTaZINoWIx4Nks2XOnJyivSOKZdkEghozM2mqwwZGzaarN0mxUKGQr1Au1bAth67uBIGgxuJCjnLJoK09iqbKDGxpIdkcZmkhhyyLhCMhmprDzE6nyaaLBFb7Z1jn3JlpBra0EAhqzE6neeiR7URinrFiGBZd3QmmJlNMjC5hWQ6CKBCJ+OnoiHH29AyRqA9RELh0cZ629ijlsoFtX+vZsWybTLpENlvm8D39RGMB/EGdRFOIUslgcSHLpaF5AgEdy3Lo6IwTDOpcODuDIArE4gFKpRq5XNlTztRkWlqjWJZDJl3k0sV59h3owedXUTWZ/i0txOIBREkk2RRiYT5LPl/BdeHk8UnCET/NzWEcx+Xoq6OIkoDPp9LVk8CyHM6dnmbrttaNv2PHWaM6B6zJjd8IFvIeT2V1Ak8G/Oi3WOfkvQzHdrkwsoBpWtcQvt8KJGJvLmQ0NZdmKVW46SQfj7579WfeVb1LEARCQZ3tg22Nj8CLq1wWbdA0mQN7vXhPIKDV07tocAH27uxsFENajcn0dCYag9LVRZBWt4VDOtsCLQ1p3tVjD+ztxnacRoEjgD07ro1PtzStXf24rksiHqwfIzRSnXZvvxzbU2SJWNTPgUh3I767uvKMRn2NNt/p9JhbgSCIBOQgO8K71myXRfmabW0+71kErpjM9boBcaWSYkgMsTuyp7GP6ZiUrBIJNUFSa0IXr0+wFPCIdKuktZptk6qUmc3naQvdWKXOracdLhWLPDc+zlQ2C3iSottvUBnwRrAsm/m5DOGwn8FtrUSjATJpTxGxozNGX38Lbxwd49LFOUzTIhDQaGoOszCfo1ioYlsuPT0JRocXybsVcrkypmXz2BN7SCSDfPWvj2OZNtt3thOO+BgYbMHnVynWU2E7u+Js3d7O0IU5lhZzxBNBgkGdltYIly7OU6tZ6D4Fy7LZuafLI+wWq5glm0jUT0vNYn4u48VA4wFaWiP0DTTzyovDNLf0IUoiw0MLVKsmiWQQVbt2iHFdF8uykSWRXXu8HPpspoQoisTiAWzHYW4mjaJIdPck6epOEAzpOI5bNwaCFPJVspllAErFGqomE4sHcF2X+bkMDz6ynXDERzQWoG+gufHdNzWHyecqFAoVXNdldHiRIw9uo60jSjpV5PlnLtDVk6C9PUZXT4JctsKp4xO39I4DioJSz9V3HIdMxStb3B4O3bAE9urqfblc5o3ZWY7PzTV8ggOJBBF9/X7uuC6maTO3mCWbr+C4LgLg01UGe+teUAFK5RqTs2kyuRKW7RCPBEjGAoRDPq/ddJHFlTym5RURkySR9uYI8WgASRKpVA3SuTKO7VIzTI/UbTkoisSOLa3IkkiuUCGTKzeuQxS80Ggs7CPg1xpcloXlPLlChWrNxHVBkgQG+5pRZIlS2WB6PsPxs1NoqoyiyKSzJWRJYtdWT5rctGxyhSrLqQJVw0LAm2hjET/hoO6lMi/nsW0H07KpGRalsgG4DPa1EPCrOI5LvlhlbjFLpWrSkgzRFA+tydyq1kxyhQqLKwUsy8Z1IRjQiEe9Z2eYNhMzKU5dmGElUyIS8pGoT/hbepoJ+FVsxyVfqDC7mKNSNWiKB2lOhglc0Y7juJQrBkupAtlCBbc+H0TDPrraY8iSRKlcY2ElT7FUq5NgXWKRAG3NYXz6myePvquMgVWsVoy7GdarPAisS8xYzwhY26Zw3VoCt6NgJQjCholgV+97o2v5bsdGJJerdpXF2iK7I3vRpPVzr1chiSJt4RBhTaNsmhRqNS6tpPjGpWE+tWsXat0Ve+UZXNcrfGw7DiXD4BtDw/zRiZMsFUsIeLnjjw4M3F55UdcLS6mqjG05mIaFbTteye2gD1Xzrsd1XGzbRZIlBFFgcFsrzS0Rzpye9tQ1lfp+9eqaguB5HTwb2jM+RUnAMCwUVWoMLqGQD0VZnai8gVmSvfDHwcP9BEM6pVKNcNjXeCaz02nGRpYoFqu0d8auUqJzPV6PAIlkENP0jJ1ctsQnPnU34fD6Xi1NlwkEL09uly7Ms7CQw3Eckk2hRr480CitLYqi91wsG8uysWwHy7I5e3qa1EoRRREJhX31yqHeOLCa7eNxkDyypmXZ2JaDbTmNe3FsF9tyGn0pENDQdZWiVGUd2soN0RoMEdN96HV9/YlMhieHh/ng4CDNdY/SGglnLhNVy6bJM6NjfPniRU7MeenFsijyQE/PdQvumKbN4kqev3jyJOcuzVEzPV2JrrYYP/f3ngA8z8TUXIZvPHeOo6cmKZar7N/ZyWNHdnBgdzeO63LszBTfeO4c2bw3GWmazEffv4cPPLgDv09lOV3k+ddHyBcq5ItVJmdT5ApVYhE/v/LPP04oqDM8vsQrJ8Y5cW7ay/6SRB6+Z5D7DvSxc4s3keeLVb71wgXODc95k6zt4NcVfv5nP0g07Gd0coW/+OYJzlyYRVEkTl2YRVNlwkGdX//5H0AQIJUpcfzcNM+8conlVAFRFDi4p4d79/eyf6dnxD7/+gi5fIVSpUYmV2Z0chnbdfmFv/8hBnubqRkWF0bm+eJXj3NpYomPPbqHDz+yi76uyzVQllIFjp6e5JvPX6BSMbAcm+0DrTxwaAuP3DNIvlDhi189xrnheUplA1WVOXZmCoB/8dOPM9jXTK1mcnFskS985RjDE0s89sB2vu+xvWzpuRyurtZMxqZX+Maz5zg3PI9l2SiKxKE93Xz+0/cT8ImMz6T42tNnuTS+RM2wsG2be/b388kn7qK7PX5rnXQdvCuNgU1s4kYIykG2h3ZsaF8BL0f+s/v3NbIJUuUyv/Lsc3zh9Gn2trbSFgoR0jRkUaRme/XQl0tlZnM5Li4vr9EHbwr4eai3l49u33b9Rm8A3aew/2Afv/nfvsWFszO0tkcJR66dMCNRP0ce2saf/fHLNDWFaO+IsR5JoaU1zI5dnfzJH76Epiv09TczuK0FWZYY2NLCF/74VXbt6Vw3XtzTm6S7N8nXv3KS1rYIPX1NyOtkrgii4E3ysxmqFRMEUBSRxYUcQxe8CnF793cTTwSpVkx6+5oYGV4kENRuaoSvQpRESsUq6VSRQr6KIAjsP9DLuTMzvPbyCPFkCE1XWF4qMDw0z9CFefK5Ml1dCRzHJZctUamYBIO65yUUoK09xsTYMl/8k1c5dHc/0aifl1+8xPioRypMJEO8/7FdPP3tc5SKNSIxP/ceGWR25uZ59NeDl4Ip8aFtWzEdm2fGxnGBX3vhRf701ClP8yEeJxnwKtcZtkXJMMlUqszmcpyYn6d8hWKgLIo8vmWAj2zbSsy3vmF1aWyRrz5zlkrV5P/7158iFNCpGiaVqknQr1EqG176qmXT3hzld//TA6SzZX7uV/+aN8KTdLXHaIqH2LO9g3v29xLwadi2w/hMin/yS3/O/Qf7G6vlmmHx1afP8kv/9GMM9jWjKTJVwyQU8Ay73q4k3R1xPv+p+zyNldk0v/rb30ISBXZuacNxXH73Cy9RKht8+H27OXJwAEWWWErlScaCSJLIob3d7N7Wzq//7lN0tcd48NAW+rouFyBzgS99/TimZfMzn32Q/q4kxXKNX/vtb/PMy0NEQ34GerwJ/aVjo+zb1clP/PARutpi5ItVgn7Nq2arSBw5OMCRgwP881/5qzUr9VUcPTXJxEyKjz++lw8/sstTny1UkCSxriUT5Bf+rw/xJ19+g1SmSH93ko+8b23Gg9+nct/+Pu7b38cv/PpXCPqvzSZ56Y1Rjp6exAV+6z/+CJqqUChVPQXH+nV99TtnaE6EePTIdg7u7sI0bVKZErHonam9sGkMbOI9h1sJm6yGXX547x4UUeRbIyMcnZnFcV2msjnm84WGPrynA+7iuJ5713JdrLqimk9RuLuzg0cH+vnQ1q23fB1XXk8gqPHTf/8xXBdkWUKSRRzb8xZIssg992/h4N196LrCT/3Mo7h4k68gCNx3xOOXtLVHPY+CKKAoMp//6fch4OXOy4rkcWue2EO15mXECHVvg6JKCAJ8/AcOI0oCsiTyU3/Pa0NTJaR66ll7R6xhe/T1N9PRGceo83gATp2Yolisct8DW2lqCqH7FBRFZqGaxedXufveAWR5fcXKYFDn4KH+NeI2e/Z1s21HG7btNFJpVU2mqzuJZdlIkojjuvh8Kp1dcR54ZDuu46L7PI7NgUO99efpGR+KIrH/UC+79nbh2A4+v6ev8JHv29/I2NB0BUkS+dSP3NsI06mKxL4DPSiq9wz9/iSf/6mHb/kdP7ZlgKiuE9Z0vnrxIrbrslAoslLyQgBivdjXlV4B23EwbBsXj7DaE43yxNYtfG7ffiK6ft3+tpIpMjq5zP/1Y480Jjqf5omfXYktPU08cKgfgGjER0syjCiK5ApVkrEgswsZnn55iFLZwMXFMGxK5VrdPV43TmSRPdva6e9OEvBpCILXh1evbSVT5OT5Gc4OzSEIYNsuE9Np7tre2XBtHzs7xed/4D52Dbah1t91Uzy0bkj0yjoN1J/VSrrIzHyW8yPznLk462WY1V3+O7a0spQqNIyB/p4k2/pbaW/2Qs2hgLYm1fRm2DnYRr5U5S+fPMnETIojB/vp7UzUy9Nf4c296v3fKibn0uSLVX70E3ejaQpC/VpXz+e6Lg/evYUnnzvPueF5Tl2Y4b79fWzpbbpjNXjeMmOgZGXImwukatPkzUUqdgHLrdWJX7fodwP2RT9GQutGk9YnYDiuzWJ1hHO5b2M4ZURk/HKMh5o/f81+ZTvLbPk8OXOBipWl5pSxXRNF1FFFH5oYwC/HiKkdJLUeVNHfKMu7EdiuRc0uslKbIGcuULIy1OwSplPBdA0ERCRRRhE0VDFAUI6T0LpJaN233Nb14LouWXOOVG2KnLlA0Uph2GUc10YRdXQpSECO06T306wPIKHUO7FwxyoLgkvVLpIzF0jVJskYcxhOGcsxsF0LWVBRJT9hpYmw0kpUaSOurfIx7iBPQhAIahrvG+inOxrlSE8PZxcXGU9nSFcqlE0T07KwHQdFklAkiZCmEdY0EgE/A/E425ua6IvF6I5GrrtC29ileC78cMRfd2d72SLSFcQjs2JQrRgookgorOO614bOrlzBuy51d7yXqYDguch1n4Kmy+sOfoGgtuZcV7dxpbaFokjIsojPpzTOFYn6kRWJSMRPOOJHEGBuNsPMVArTtNl/sH1dTQ/w+D2avvZvmiajqt51XBnmkxsDnVB/ft71XP0GNNUbyoQrjtU0Ly34ynPK6wycwaDEajmYq5+TKEJIufX37VMUdrU0E1C9/x6fm2OmrjxXqNU8CV3HQRZFFElCk2UimkZTMEBvLEZvLEpfNEZ/Ik7Cf+P27bpoVcCvIdQnVO82hDUeIV1XCIe8SUyqhyZdvPh7oVTlC189xn0H+mlOhFBkiWy+zMvHvVoUlyM2AsGA5mU6iJffieu6VGsmz7w8RNWweOjuLUTDPsoVg4mZVOM6XKBWM733rVyuQnkr2hq27eA4DgPdST7++F1rpKejYT/tLdHGb7+u4tOUxnu/1Ym6szXKg4e20NkaJZ0r89xrw5wdmmPHljYO7um++Qk2CMuycRyHgE9dUxRtFYIgsL2/FU2RG7yCbz5/nvPDMR68ewstyevrbWwUd9QY8CpI1ViqjpGqTZAyplipTdYn3XzdGLBuK1WuL3A3EbUVjfWNAReXgrnEUP45KnYeEYmw2sqDTX+7sU/VLpAx55ivXGCmfIasMU/ZylxjDOhiEL8cI6F1Y4Xuo13fjnqdPPbV+wYoWWmKVpqSlSJvLrFcGyNrzFOy0lSdIqZdwXRrdWNAQRE0NClAUE6S1Hpo1geIq11E1TYC8u3FgBzXwXJqLNfGmSmfZrk2TtaYo2CtYDhlbMdClXzoUpCgnCBtTFGyMjRr/fjlWH2gePNpLxU7T86YJ23MsFwbZ6U2Tro2i+GUMB2vH8iihib6CSstRNQ24monbdZ2mrR+dCl4R4wiuGxW9ESjNAc8Rb6+WIzRdJpUuUzZMDFsG9t1UEQJVZYIKiphXSNZ338wkSCgqjdVzwOwTAvbcnBsB0ES0XQFXDBqJrWqiaoraLqCbdlkVwqUcmUSrVGCUc9AKBerlPIVgvV4uygK1CoGlmkjKxKqrmAaFo7t1Cd/AV9Aw6zZLM2mcR2XaFOIYMS/4VS2m+129eAUTwQJhvTGihs8DoKmKTQ3h0ncBgN8vevY6OAtrMMTut45r90H7qjxWT9bwu8nUk9l7QyHmczlSJXK5Gue3LC1agyIEroiE9V0mkNB+uMxeqLRRtGgm0FTZYIBnam5NN3tMURFwHFcLNtBkS/3V3E9LpLr4jgO5YrByfMzfPYTd7NtoKXB5l/vvtYzqAAqNZNL40u0NoU5sLuLZCzI+eEFFEVcc3w8GmA5XSRbqBCpGyeG6RFKxSuMGVkSvdj4VfLImibj96nIssRgXzNd9bTtStXwskuuyLpYdeffLmRZoq05TEdrlJHJJVLpImeGPGLnlcaAJHmGV824vsTyjRCoa+hML2To6YwjAlZdKXP1uaiqxO5t7RRLNcZnVlhayfP86yPs2tr27jIGPOa1Rao2zasrf8JC9RIV+0oBh1VimPffm3kHvFJDAoIgIiIhCtJNiWVXwsHGtMs4roUoyBhOiYXqJc7lnuJi/pl1jzGcMoZTpkgKjEmmy6cASKhdNzQGACzXYKp8ivHiUearF8kac9fd18XBcSxMKpTtLBljlunyKVTRR3dgP9vDjzAQvAdZuDE57przug6GUyZtzPDqyp8wWT6B417bOat2gapdIGvMM1M+y0jhVQ7FP0l3YD+O66CKtx+D8gxCg4XKJc7lvs1s5RwFc3ndfVefd8FaYbZyDllQiatd3N/0Odp9O9Cl0B0zCFbhUxQ6IxE6I5Gb73ybKGTKlAsVbMsGQaCjvxnHccilCqzMZwmEfXQNtlEt1xg+OcnEhTkOvm8nW/f1YJkW1XKNWtVAqK+WjJpJejFHKV/BF9Bo62sis5THrFlYpo3ruPTsaKdUrHDy+YtYps3ue7cQ3PPW1XFva49es62rO0FXd+Lanb+HIYsiyUCAJ7YOvmVtxCMB+rubeOHoCIN9Tfh9GoZhYVo2rRuYJFzXm3h8uhdyKVeMOkFv6pYmUqueWSBJIqblkM1XeOPMJOWy0RjHBEFg745OhkYXCQc1gn4VSRTJ5Cu0JsPomowk1b1PIR/ZXIV0tkQmV0YSvWyzaNhPR2uUmYUsL70xyhMP7QQ8sp9fV2ltCjfCRTeCbTtUqga27TYE4AqlGvliBZ+uIksiy+kClapJPBqgszXGw/cM8n/+5nUyubWCVwGfBgispItk6imvQb9nsHjtmGsyG4qlGvlCBZ/Pa6ejNcrCcp5Xjo2xvb8FRZYoVwwkSSQRDaAoEpOzacJBHb9PpbcziabIvHrim1Rrt2eAXI07Ygy4uFhulXRtmr+Y/nmqduGa1b8uBtDqqz3LMShaK9f1EMiCik+KEJTjhJQmIkobca0L5QYpZOvBwaFs5/BLYY6n/4ZLhRdYro3fwhlEugP7bzo5ujis1CZ4dvG3Kds3LxZyPRhOhZHCyyxWhqnZBfZEP7xhA8jFxXAqTBTf4MXlPyBrzm+43Zy5wDOLv8X2yPvpCx7EL4XhNitpW67BhdzTPLf0O9Sc0s0PuOrYpdooX575Dxxp+jG2hO4jrnbf0CDaqJDK25meefKFi2SXC8Rawpx7bZTP/NOP4DoO5UKNzHKBr//hC/zD//ez2JZNrVKjWq6haLLn3pdEFqfTzE8sE2+J0NwR59gzF6iUqhSzZRYmVvihf/QEz/3VUYKRAJZpMz+xzI/+849gGRaVUhUARb1+nYYb4b2Uxvpew1vVV7f2NxOL+vjDP3+Vf/If/gLDslEVid6OBP/6H3z4piOIIku0NUX48CO7+H9/9ylM26G9OcJdOzppa45sKJtKEASaEyEeODTAy8fG+Pv/5s/w6SpPPLiDZDzYIBiKosCP/+D9/NlX3uDLT53hN//4RRzXJRzQ+Df/8CP0tMeRJBFZFvnw+3bz67/7FL/+u0/hutDWFOG//uIPIgCf/vABjp2d5hvPnuULXz2GbTs0JUI89sA2PvnEPjYirju7mOUP/vxVJmZSTM9nGJla5oWjIzTFg/zkDx9ha18LZy/N8eyrl7gwsoAseWJB779/G/cd6F9zrsN39VCqGHzl6dM8+fx5wgGdn//7H2JrXzNLqQK/+4WXGJ9OMbOQ4dzwHK8cHyMRC/DTn3mQwd5mjhzspyUR4q++eZKf+YU/w3YcfJrCPft6+YkfPoKiSDz36jBHz0yQzpaRJRGfrvK5T95Dd3vspve6EQjuHZClcnGYK1/g2cXfYr56idVJxCdFaNO3czDx/SS0HiQkqHsFDKfMUP4FzuWeImvM4nBZhvVI0+foDRwmprbXy+aKyIJ6wxK6tmsxWniFby/8Vyr18ru6FOLT3b/CpfyLDBdeImPM4uIgItPiGySuduCToiiihotL0UyRNqbJGQuU7QyKoPMTW/7wpi5rLxXN5ssz/4G5yvlG+wISrb6txJQOIkozQSWJKgYAl4qdI2vMsVgdYbZybs35BER8Upgf7f0vBJUkknBzm83F5Wjqz7mQe4aV2vgaz4tfitKk9dET2E9QSXpVGp0yRWuFmfJ55ipncXFRRT+6GEQSVTLGTOP4Dt8ufqjnP9+0hHHeXGK8dJTnFn8Hw6my2g9EJCJqK72BQ0TVdnQpiISM4ZRZqU0wVT5F3lzEcC5LxKqCj92xJ9gT/SBNWt912yxVDX7tG8/z1NmRdf+uKzKH+7v41594Pz717VH9+s6XXkOURPY9sI1XnjzFzsMD5FIFMst5lmbTXDw6xs/99k8gKRKnXhxi4uIc3/d3HkGsM+/PHx1jYWqF3fduobkjzl/8xrexTAfNp+A6Lg9/4hDf+rOX2ffQdvxBH+deH+HQ+3YSb4nyzF++jqopPPDR/Q0y4Cocx6VkGPz+829wbHyOpXwR07aJ+nUe2tbHDxzeTUskdN2U3U28OZRrJn/y8kn+8MVj6/5dEkXu29LNP/nwgzSFNi5Os0pArFZNLMvBxfU0SiTBW7EKUCzVUGRxTT56sVxDqtcr8HQEvPosUA8pyBKGaTVId47jYphebNvvU9cdC2qGiWnaWLaX7qoqMmbdOFnldazyBkzLqWvCuB6x1q8hXaG1spp7v5pZIdZ1aMDry5ZlUzOthoaMRwCVG+3UDAvHcZFlcU3oYBW27VCpmZ5kvOM2ODOiIOCrk0sN08a4og0vDOGpQ14ZLnEcB8P0Vv1eqq93P7IkYjsO1bpn4HrtrF5PzbA8nYf6M1FkCV33KlyWK4bHLXBdVtOHdU1Zw994M7gjnoGcscB8dYgVY4rVCSAoJ+n272Nf/KPE1S40MbCGEapLQbaFH0IT/YwUX2GidPkDmasMkVB7afPdXvrWKmzHZKTwCsOFl8ibS/jlKG36NraE7vf4B6IfSVAR8AhEllvDcMpUrDwlO03RTKGJfoSbWJmCICAhMxC6F3Cp2WVafFtp1QcJyglUyY8i6Ciihiis1q82MNwKJSvDbPkcR9NfwnRq1PnsVO0CY8WjbA8/hE++uUt7unSa2fI5snWDp35l9AcP0+3fT7tvO345hixq4IKDheXU6AvezVTpJOdz36FgLVG00hsyPq5G1S6yULnEmcyTayb1qNJBb/AAW0L3EZQTKKIPSVAQEHBcm3b/TvqCdzNbOctE8RgLVa8wi+FWmCgeRxIUjiQ/572ndQYfF5di1SBdWl9rXldkCtXa2yroLAiQWcxx+qUh5idWuPeJPcxPLpPPlLAMu5G37hHsBMqFKqdfHuauB7aRTxVYnFph6uI8kiQSiYfoGmxlYSpFuVDFF9AIhH0IojdYK5qXgeC6q3FzgfRCjgtvjLP73i1rritbrvCHLx7n22eHSRXKVOty1+lShWLVoGbZPL57C/t61i96s4k3C5eycf2+KgoChWqtIZq2UawSAgPrpKytIhy81qt6dYrbemXSr1TUkyQB300MRU1V0K4yun2s/b0qiHQjWuSq9kponeuG+sSvyqjX8YAB12RTXA1JEtdN87sSmnrZuLgRRFFE18R125RE8YbvZhXyVQbG1Xiry9jfEWMgay6wUh3HvGISSGo9dAf20ebbfo2r20vjkomqbfQGD1KyM8xXLjbcykvVUTK+WSzHQBZv/wHYrslE8RhZY46w0kK7fwcDwXvoDRxCFa9PrrIcg5pTomilEAV5w267Dt8uJGQs16RZH6BJ673uJLaKmNpBQI4zUz7NQnUYw/FiUQ4Oc5Vz9Afvxsf1jYHVUMtk6RhpYwrTrTb+1qT10R+8h57AAWLq+gN8zO3EJ4Up21kmSyfIGDPr8gxuhrQxzWzlLEu1sca2qNJOd+AutoUfotPvKQhe3RdCJImrXfjkCKKgkDOXqNhZAHLmPHPl86zUJmnR37qY652HgCiJ6AGNrsEWAmE/idYoiipj1CwCIR2hPqjGWyL0bG1rrA4EUSDWHMEy7ToBELq3tuELaBRzFXS/iqor9O/sJBwPovlUOvqa0f1emld7b5J82NfwMqyialrMZwt859wIUyvZNcZRzbSYTud4/uI43Yko29qa3jYvyibeOhiGxcRMitl5730nYgHaWyI0JW6N3Hni7DSJWIBkPPiWT0jfTTh3aR5NlWltChMM3NwYeKdxR4yBvLFA2phes63Nt50O/86bxryjajtJrY+I0spSbRSAkpWiYC1TtQsExdsnJDnYzFcvIgsqfcFDbA0/SJd/702Pk0UVWVQJyLcWi1lND7wVSIJCQu1mV+Qx8uZywxgAl+XaOJZbu/EJXLDcGhOl42uIegIi2yOP0Bs4SFRtu+7hgiCQ0LrZHfkAtmuRMxdu2RhwXJvp8hkmSycbxwqI9AT2sy388E2fuShItOhb0EQ/85WLjBePAi62a5I3lxgrvv6eMgZkVaKlK8HhR3c3WO5b9/XgOJ7c7JVCPD3b2uje2opt2QgCRBIh9j+0tjpmW28TLd0JcOvxZAHufvyyPHP0Cub+rnu24Dhe6tWVyFeqjC9nGF++PqdlbDnN+HKaVLFMZ/ytI1hu4q3HqtTuN54+x/nheQRBYPf2do4cGrhlY+C5Vy6xe3s7uqZsGgO3gNeOjxON+PDrfd87xoCnKbDU+C0iEVFaiSgbK+4RkGMk9b6GMQBQsQrkzEWCyptnJ3f697Aj/D7a/Tvf9LneKgyGH+RU9ht14p+3bssaC9ju5RjUenCwmS2fJ28uYbleiVMBAb8cZWf4/QTkjT2/Vt9WUsYUC9VLLFdHb37AFSiYyyxULrJyBTkzpDSxJXQfXYE9NzjyMgQE/FKMA7GPMVF8o+HxKNsZLuSf5t7kZy5Phu9CXEm96dvR4dVFv2p1vlqn4moIgoB8kyI018vZv96+V++fLVUYX765st5Kscx0OveeMgaufPbv1v7xdqNaM5mey/CNZ87xf/7b54lG/EiicFvhsn/0k4/e8ev7XsDf+eH73+lLuCXcEaaQ6VbXkr+kALKo3jTWvgpZUNGltVWjbIwrVslvDvtiHyOqXltc6N0ERdAJyUl08fJzsNwqjmtyI1a/49pMlU5gu5clcyVBYSB4rxcKuYV0zJjSQbfv5p6TqzFWOkrOXFyzbWvoAcJKC7eSv62KPjr8e1FFX+M427UominPY3EFyfTdCMu1SRt5kr0x2nqTNz/gbYRpO1QM86b7GZa1of3eLXBdlz+d+h+cyr5Mzrx9GeHvNlSqBrlChXjUTzDgEfPgTqspbOK7CXdQdOhyN3Nc84ar2avhYGNf5ZoWEe+A+I2AJgZIar3XVS58N2D1OcmihiQqXCnBsKrYeD3DysUhZUytce2LgkyrvhXpFvgOgKe6qN260bRSnaBqF9Zsa9a3oEvBWzJGEEBCJqQ0YxkzDQPHwSZVmyQgx26L3Ph2wXBMClYJXVNvaSX/dkCWRLQNlMBVJek9VSpXEAQebv4oQTmC/138jd8ppDIlzl2a46Wjo14hIFGgsy3GgT3d7NnejmFYPPnseS6OLDC/lCOTLfPrv/UUkihwYE839x0cIOBXef61YWo1i2K5RiZbYnbBK5f84z9yhNbmiFc4Z3KZp1+6xNJKnofu3cqBPV1rxG1M02ZyJsUbp6cYmfDKWquKxP7dXbzv/m1omswbpyY5emqSdLaE47goisRjD2xn17Z2FFliYibFl756nPsP9nHq/EyjdsCu7e08/uCODY1fruty7PQUx85MsZIqesqOssijD2xn19Y2VjIlhseWyObKjEws09edRJFF8qUqs/NZfvBjB+ntSmAYNhPTKV58fYSVTBFRFNk+0MLOrW1s39KK67q8/MYopbJBuWKQzVeYrKsrfvqjB9nS1wyuy8jEMt958SJLqQK7t3Xw4D1b6GiNNq7XsmyWU0VeOjrKxdFFLNtGkUQGepr4+BN3oesKJ8/NcPLcNHOLOWzbQZZEHrl/KzsG2wgFdFYyRX73T17kvoP9DI8tkcoUkSSRvTs7eeLhnRuuCXIl7shXv6rcZ9QJgLZrYjhlTKe6IW0A06lRtnJrtsmi9qYncAGRkNKEJgXe1ZPIKjxhpbUv0a3/3/Xgui55cxHHvbxqFhCJqu0ItygrrNfVEG8VWXOOmr1WUyCitKAIt6YLsWo4+KQwOSRsPGPAxaVkpXHdt9cz4LgupmMyXJgEQSCqhIgoIVRRZqw0g+O6xNUwYSWIIsqMFqexXYeoEsJybCp2lYVqCtOxCMg6siAjCSJhJYBP0pmtLKGKCj5ZZ668hOXahJUAYSWIiMBMZRHHdWjVk/glnapjMFNeRBAE2vQkISWAKm7MYPapCongjfUyBCDs00jeZL93Gzp8ve/0JbytEAWRZCyAKHoKfZlsiZeOjrBne7tHQI34aUqGqNYsZFmivSXiZaaEfXUJYpeFpTyXxhaJRnwkYkE62qLUahaKIiPgMeD9Po225jCvnRhn20CeanWtx2hsaoXzl+aZmk3T0hRGwEuP8/vUBlfGBSJhH2pd8rpaM/jOi0N0d8SJRvxkcxVePTZGd0eMeDSArqvkCxVefH2Eew/0N2otbAThkN4IhVSrBs+8fImO1iilUo3ZhSxzC1lamsKcPj9DPOYnHPJRMyzODc3R1hxheHyJS2OLaKpMR6tX+2NyJo1p2bQ0hYmGfSyuFLg0uogkiXS2xeisPzdNk70qooKXgdHaHOHU+RlikQDlirHmOmcXclwYnmd0cpmWplCjgmgwsPZeQ0GdZttBACpVkxdeGyERCxL0a5TLBi+/MUZHa5SAX0WWw+SLVZ579RL3HugjHPLdcorwHZkhVwWCitYK4Ll2C9YKBXOFuNZ5w2Nt16JiZckZl0VyRCQ0MYhfir6p6xIFkYjSsuFwxZ3EqvaAVZfedbBxXBsHB9d16hN8faJ3vf9Vq9cOuDU4FK3UGl0BURAJyolbvm9F9OGTNi5ruRqrLZjLazJJAGpOiZy5gGjdWhdzXPtacRYXqk4R521NEPSknct2lZdWThJU/AwGu+u64SLHMxcA6PC10BNoo1mLM1qcIST76fA1Y7s2BavMWHGa+eoK3f42NMnTHe8LdOKTdC7mx4moIVr0BK+nz+KXNTp8LZiOheXaHE9fAAEGQ1Xa9SZKVpmXV04SUYNoooouqRs2BsK6Rlc8gq7IVM31CaIhn0ZrNERLdH2Cmeu6VJ0yFbuEU8+P9PgpIXTRh+FUMV0T27UxnRqqqOMRQT3vVlCOoAgqDg41u0LNqWDX+7su+dFEHUVUcV2Xsl2sF41ysF0L2zVxgVhdd8PB8RYRdhHLsQjIITTRhyzKV1yvg+EY1JwKhlNr5N/7pAA+KYCAUL+fy9+ddz9BNNEr4Ww4NYpWHl3yU7MrODiIgogm+gjIty65/GahazLtrRFam0MEfBqZXJmXjo7wwmsj/N3PPYQiSzx07yA7t7Zx+sIsw+NLfOb7716THrcq7zs1m6azfYBHj2yjpSlMsVQj4Pe8WqtSv4N9zbx2YmLdFL6T56YZnVimORnic5+6B1WRyRerSKKIpsq4rktzIkS8rqAnCrCwlOff/tpX+NRH9hMN+7Edh0Kpik9X+cBDO5AkkaMnJ/jvf/AsK6kCfl1BFG+uQtqUCBGN+FFkEUkSWVjO84u/9hU++pjHWVrVYvjM99/NP/33f05TIsjeHR10tkY5dX6W+w4NcPLcDMPjS3z0sT30dMYpVwy+9NXjnL04x+5t7UTr8uBzizk6WqONFX+pVEPXL9c/6O9por+niaHRRXT92u9zeHyRN05NEgn7+NwP3IOqypTLBo7joNVTE5PxYMO4URWZheU8v/Rfvs6RwwP0d3taMYViFV1TeP+RbUTDfkYml/mXv/yXLK8UPLnmd8IYiKtdNOtbGjniADOlMwTlRD2l7bIU5SpWB/y8scRSdZSl2mXRmIAcJ6q0ElLeXNzV+/DDt+aqfhNYndjrgsMUzRRzlfNkjXkK1jJFK0XNLlJ1ClhOrTHIWY6Jg3lNqOSm7bmeSWE45au8Bx6B8FbvWxJk5LoWwkYzCmzXxHSq18Tz/3L6X99S2zeGS80q4d5qYfk3CQeHslWlYtd4vPU+2nxN5M0ir6XOEFVCNGkxhgqT5MwCH21/iEOxnRzLXMB0TDRJpU1P0tIa548mvkJ3oA3TMRkvzmL4vIktZeQQ6h6HlVqGf9DzGSRB4nxulJHiNDE1RLOe4NXUGfZEthBWgtiuzWMt9xJTw4i3YOxFAz62tiXZ39PO62PT2FflsgvAIzv6uau7nYhvfY+O7Vqcyb3GKytPUbILuC74JB8PNX2Yu6L3M1w8y0xlnKyxwkRpiMHQXlzXIW0sUnHKfLj1MwwEd5I3M5zLvcGZ3GtkzRSiILEvej+7wgfp8PXhYHM08yw1u0LRypMxVlioTuHi8pN9/4qk1krZLjJWusDzy19loTLDYy2fZE/0Hpq0y5kzpmswXr7IyczLjJcuYLkWqqBzd+IR7kt8AFXUOJt7naPp58iZ6boUt8ojzd/HnsjdiILEaOk8X5n7I+6JP8qp7KsUrRxhJcqO8EEebf6E9+zeJtKi67pUqp77/ivfPk29JhWpTAmjLrBzK+Iz/T1JBnubaF2t6HednP7rYXxqBcd1eeyhHQ1tgUjI17hWgLnFLN958SLLqeLq0EixVPPEker7SJLIBx7a0TBE4tEAyXiIdLZMZ9uNM7pWzzG/mOXpl4dYWMrjAsL/n73/jpMjO8978e+p2DlPT86DGWRgASywOXJJLpdBS4oiRdGiKCtYsi1ZsmzJ1762ZNmWZf90HX7XClaiZIm2RDHnzcvNi5wzJufYPR0r3j+qpzEDzCAOsIvlPPzsh5iuU1Wn0jnvecPzuJDNeYqLsiShqTKNdTE0TSYS8lGXjtJUH8d2XAolg5nZPKPjGfYd6uPshaX5T+2tKSanc9W/mxvidLWlq30LXed9GxnPMD2b48efvhtdU6riT4uvZ2wiw6tvn6N3aNrjEAFm5wqYpr2Eh+Kx+3tIJTzp53g0QG0qwmy2SKPlwHUWMKyKMVDvX4/hFDg7/3q1Rny4eIyCnaFgzbEj8SPoUoBL01eGikfYO/1lRgonl/y+KfYEzcHrT2S7HMLj97+NaTNjpTP05w9wfv5NJsoXvBW7u9jVv/Cvm1/lejTQxmVGhECgCo8r+3qxsG/5mowBl7KTuyEVyuuFfZVEylsBVSjU+1N8uuWDvDy5j4QWJSD7KdplUnoMTVbZldhITF1+hViwS3xv7DUeTO2k3lfDaGkSIQQ2niiS7XpsYoqQSWoxFp6X6VqYjkVIj6BJKk/U3UNaT+CXdVJ6jG+MvMTmyDq6Qs0k9GvP+k9HQvzup5/kD55/k7fPDzKeyWG5DrWREJ/YtZkP37WBVHjlEIEkZNoC6+lp24Ym+bBci/HSEH818F9ZH9kBgOWYKELll9b9e/6fM/+ce5PvozO0kXkrw+G5N2kL9vDc+JcJqTE+1vg5avQGinaBvx38A1wcNEknqdcCcHjuTbbH7uXB1JPEtBQFK0dYjSEhEVZibI3uYVNkF//zwr9Dky4f+d6eeYnhYh9+Ocgvr/sPqJJGzsqiST70SviyLdDDutCWihcDJsqj/EXff6Y9uJ6IGsd0TOaMaSzX5O+3/3McXE5k9/PS5NfZFttDUqutkJbdehRLJvuPDPAnX3yV3/mXT9NYG6NUNvn+Syf48rcPXPfxfJp6RZKbq2Fh4lpulHEcl+x8if/4/36ff/T5R9i+uZlwUGd2rsBn/tGfLWkr4DJGwwUlxKt98d4Kucx/+v1n+fyP38ff//FWIiE/mWyBn1hynqUiTZIQVZpl1/WOIyTYvqWZf/trH11iVAnBEkpmXVOqUt43hMp1rTQ6Z+dL/Nc/eYEPPbaZT33sbhKxANn5Ej/9q395WduA/6JQWLVWyb1yaHklrIoxoAiVtK+T+1Kf5aXxP8LGxMUlY45xdO579OcPENMa0KWgp03gGuTMKebMUebNySWVCF3h++gI7SauXTm8cK24HVa767qUnTx7p/+WkeJJ5oxRSvY8tmss01qgS35U4UOWNGShVlbkGnPGKGU7f51Z88s/9Bu/blFlSbwWXH9Y486B6VhMlec4nDnNvFUgooYIqQG26Ot4bfIg82aeGl8CWUhQgr2zx+nNDRFRg+TsEoZtcCbbh0/SyNtFFCETVUM8P/4Wzf468laRuoqU7+Kn1ehPYzgWh+dOk9MLNPhrUIXCpGNzNtePYZvY1VDTtUMSgmjAx089uJNP7NqMYXshGU2RSYWCxEP+K8pXe9/0DCfnD2A5nuZ9yS5SsHKVRFcXXfbhl/0Vt7+fhJYmoaWxsSnZBeatWSbLo1zIn2KwcA5N6DjYTJXHiGspstZc1RhI+xqo9TUT12pQhEpYjSItosQWyFVBs+VG1tHiABISm6N346uEBSIVZU6P8tZhzpzhTO4IJdurXDKcMnkr5xnYrhfKk4TMluhu/EoQEETUOAE5zLyVJaGlr+sZ3AxM06ZcNnEcl1Q8iK4pHD01zPn+iavvvByWv23XjOaGBH1D0/zgrXN8ui6KIsvkC0ZVcbBUNrEsm0jIRzjoo2yYvHmwd1mNhksluq8VrusZSZZlEw7qREI+LMvmzQO9S5kcr3LoWDRATSLM9Eye1/ad5/H7exBCVBMfvRDBIgPhunt6EelUhFgkwPOvnuZzPxZHVeSqkJGuK9X7FgxoRMOekfrG/gtVqugluPS+3UTHVsUYEEIiIMdoC+5gY/Qx+vIHKNhz2K5RdY/PGaMVF7SE7VqU7GzVCBAIdClAvX8Dm6LvI6E1Xbco0TuFBdnm45lnOZ97i4wxtoQFMKLWElHTBOUEfiWCLgVRJR+KUJGEgiTkiiqjwpG57zJV7rvmkkqBl18hkC5bnduujXKDin/XPsELZHE5CYlAYl34/lWRQoYKKZF/3bLnupWQhMAna9T7UyS0KDE1TFyLoMsam6Kd+GUfETVEWPGjSiqtgXriaph6fw1hJYAt69yd2EzalyCmhtFljYgaouyY1WPV6HFiWpjNsXVVD1ZYDdESqMN0LAKyj4QewS/5MF2LBn+apBajwV+DX77OBE0hUIS4IQ4Bx3Uo2QX2z75CSq/FrwWQkMnb85U8mAX+eBlNaAgEslDQJB+apCMjezk0lbBYTE3QFdxUHcjag+tJ6fVE1YvS3QE5hF8OoFZYSOXrHK7KThGfHCCsxKpGzkIiseM6GE6Zg3OvEVOTRPU4sqRStgvsx4ZFppZAkNDS3rcmBIpQUYSK7Zi4V1rirTI0TSGZCNHcGOc7zx8j4NcolgxM07kmutvrwexcnuNnRpnLFpmcnufEmVEs06ahboJtG5tIxoNsqfBpDI7M8NXvHvK8XrZLe0uSLesbCfg1ujtqOXRiiMHRWWRJIpsreoly8urcNCEEAb/Kuo40J06PMDE1j6JIzOdKlZLKawulaarM5vUNgMuxU8PMZQo4jqeP0FAb5Z4d7VyDmjSFosHBowPMZYsMjc4ymynwA03hfP8kOza3kIgF6GyroVAyOHVujK9+9xCS5Ok+1CTD3LOjnYBfo7Othr7Bab713FE0VWE+V6zINt+6/LdVS7FXJI2oVsdd8Y9SdgoMF4+Tt7y6XxeHgj2Ha19MOhLIaJLH2a/LAaJqHZuiT9AZ2rPqsrW3ErZrkDXHOTTzDebMcdzKql5CIaLV0RrYRq2vm4TeRFStJajEvRXNMlbwQP4gc8Ywy/kTloMQUsWgUJZ4IRZUJBWhcj0jleeWc7Dca68zVyXfZW5SCYmtsSfxyeHrOv9KEMITW1Jvgpr6RqBICnEtQkQLVd5ZT1wE4L7UdmzXRiBVf9uZuJzUqjmwlHjLdV3q/ClkpMrKzNs3rl1M3NQllVpfkrSewMZBrqyGXdcl7Ut4/UDcVoIdx7UpWDkOzb3Gz3f+S+p9bYDLcLFvSbvKXVr0t7SkqkUWCqqk0ejvYHfiMUKqd92mYyIJgVQxGhba3kzyryo0XNep5Dd4U7uDl53tuC4lu8Chudf5bOsv0RbsQREq46Why8KKQnjez6X3+/ZX7Pt9Ki2Nce7e1sqBo4PousKGdXW0NSWWaAiAtzIPh3y0NScve08EgkQsgG3bKxoRuYI3WY2MZYiE/RSKZc72TTA9m6ejJUUiFmR9Vx2qIpEvlNh7eADHcfDpKn6fypb1jUTCPh7c08XRUyP0DU4Ti/jZuqGRbRuaCPi8ioOgX2Nde5rF99PvU2lpiHs5BFd5x6WKrPEDuzs5dnqE/uFZImEfOzY3s3VDE8GAjuM4JKKBSo6FoLEuRjwWQFVlQgGNhtooqiKzeX0jwYDGi6+f5q0Dvdi2QzTiR18UEohFAhiGRSS8vLKCYVicOj/OyNgcmqpg2w4XBqaYmsnR1VpDNOKnrSmJrinMZQvsPzKA7TjomkJ3Ry177mojFNS5b2cHx06PsP/oAEG/zu7trWxZ30AsEkBInmHY3VG7xNhRVJmWxkRF8OkdKi1cgEAi5etgS/T9lOwsecujPlWFjiK0KkOeLGn4pBBJvZVaXxcN/o3U+9ejy3dWSRPAvDXNycwLzJoj1d8kFEJqivfX/zKN/g3XvEK+kUjPglelaJuLshFcilYWXQtd15Dl4uA4JvbVKJAXzi1E9dku9k7YWMT1JsJK6o4y7JaDEAJlhZiwfAPXdqXjXa3t9ey72nBwMF0DScjoUgAJwZQxwYnstceqBRBR4sS1GubMKY5n97Ez/hAA0+VxfLKfkBq9pnfWq9bx1N0Wqg4c18Z27WpiZb2/hZFiP0czb9Pgb0FGIW/PVwwStXI9XmWAjMy8Ocex7N5FvX33obkhwU98fA8/8fE9V2wXiwa4e1srd29rvWybJAne//CV2VibG+L8zGceuGIbWZbY2N3Axu6Vha0+8dQOPvHUjiW/fejxi6ykWzY08vu/85kl27va0/zGP/rgFc+9GEIInn7yLp5+8q4lvz/52OYlx1zAP/7pR6v/TiVCbNvYXP1764Ymtm5YOUT92P1XFs+LRQP8/R+//6p9bmlM8NOfWrndh5/YyoefWJo3t/h6mhvi/Onv/b0l22sSIf7vf/LUVc+9ElbVGHBci5PZF3h54s8o2hkUoRHXGvhgw6+R0JovmxgWLHBxMfXhjkPemuZ09pUlv6X0Vu5KfIxm/2au57oMp1Q1mK4VAkFca6RcyldJelzXZtYYIqLWXddkXLZzzFfKQ68HKV8rJSdXTR4FmDWG0KXgZcySa7gzoQqNlF7HnsRj/FX/fwdcknotTf4OEmr6irkGiyEQfKDuxziZPcDhuTd4fvyr2K5NXE9xT+JxtsXuvSZ+jKHiBV6d+h4TpWHGS8PMlCfZO/MyCS3N47U/QpO/nd2Jx7iQO8HBudf4vdO/juPa+GQ/u+IPsyf5OAktzZ7EY3x9+C8wXYOYmqAztKkaEljDGn6YsGrGgOmUmTNHeWPqrynZWcAl7evknuSnSWrNV1XvuxPh1TEXyVpLE3iCSoLWwPZrJv3xEhALlOwslnNtq/IFCCGR1FuZKJ2vkvQ4rs1o8TTNwe3AtcftC3aWOXPsus4PUKN3MlMeXGIMjBVPE1Xr14yB9wiEEKioPJL+CPck3we4KEJFlXS2x+4loITpCm2mLdBT/c4/2/rLRNQYslCIKHFag+u8f6sxtsXupTu81QtJuSBLCkE5jCIUQLAr/hCWY+FfwVtY62vig3U/humYWK6JJKQKa6lCWI0hkNAknY7QBur8LRgVeXCBREAOoUs+BIKHap5iT+IxXFxkoaBLPrZEdxPXPK9Wd3grv9j5W0vO3Rro4seaf56wEr0h79Aa1vBuxKoZAyV7nuHCcbLmhJdVLIWIa43U+dejLFP2816A7drYjrlEFwC8/InAdSoeTpbOU3by112mJ5Cp0Ts4LX5QLSywsRgpnfQSnITvmo2wvDXNdLn/us4PnsjRYOEQ08ZA9bfB/GEa/VuIqrV3fKhgDR6EkIiqCaKX2JcRNQbgTdyLHnXad9F9rEoaATxGURmFoBK+ImlPSLlykqMm6WhazZX7i8AnB/BdIfwYVROX2cvhyvUAHkGRfykTqi770eXlY8ZrWMOdilVLTSw7ecaKp6txa03yE5CjBK7yUd/ZuLF6ziVHcD2p3v78gcv4/a8FkpCo9XVV6qS9Sd9xbSZLvWTM8atLIFdgOmXmjFEmSxeuuw81ehsRtQ5NujjojpXOMlk+z7w5dZP3yF303xrWsIY1rOFWYNU8A7ZrULBnF/1tYboGtmNWVoaXsxDe6ZAldVnGPtMpk7OmiWn1V9j7oiGQNSc4ln2OgjV33X0QQqI+0ENITVG0c1huCXAp2hnOzr+CIj1eyddYQeioUks9Z4wwVjqzREb6WhFWa6j39zBnDDNcPA54xuHJ7Iu4OGyNPVWtbLja81/I+nYX0TYvaDbcqlfHdV1cl0oSmovjUiUGWQwBICoZLkIgSV51wcLfa7h9qFYHVGhmF56hi/f/l8LjFgC5wjEgidtbjbFaWCDicRynct1XuuZKdUfleqXKPbgTr3sNtx6rZgzIQsUvX/QCFOxZLuTeQriwJf5BImotmuS/bWxdtws+KUhab2esdLb626wxxKHZb/FI7c9ecd+SM89o8TQvjv8hOXOKG1/9CrZEP8AR97tLKKHfnv4SlmPSE3mIhsCGFfeeNgZ4eeJPGC2eXLHN1dAdvh+BYKrcS7nCkzBaPEXWnGSkeJJ7U58loTVfExvkvDXJcOE4g4UjGE6BbbEP0RS4cUbKq41986UyI7NZDvSNcHhglOHZLDO5AtliiaJp4bguQV0jqGuEdI1YwMeGhjSbm+vorktRFw0R0G9v2eMPOxzXpWCYnBye4NjQGOfGpxnLzDOTL5ErlZkvlrEdB02RCfl0YgE/rakYW1vq2dRUS2c6sSLt8juJq72rRdNibG6et84PcnRwjJG5LNO5PHP5EgXDwnEdAppKQNMI+jTilXd1fUOartokLckoId97M2y7hpvDqhkDATlOZ2gPJzLPVd3COXOKE9nnOZ97E2kFOV2BjCxUNMlPSEmS9nXQEtxOTG3Ar1y7aM47AYEgqCToCt+3xBiYNyc5kXkeyy3RGbqXqFpXKZuUMJwCc8YQY6WzjBXPMlE+T96cQREKca2Jkp1j3pq8rj4AdEceZMYYJG/NVCsCXByOZZ5hqHiUOl83TYEthJQEIDCdEnl7lonSeXpzez1OCCEIynE0KcCsOXxd98InR2gN3sX9NZ/jxfE/quY+FKxZLuT2Mlo8TVrvIK43EZBjaHIABRXLLWM4RYr2PHlrlqw5QdYcx3LKmG6ZuNZwUwECgSfLe+mb57ouRwbHeP74OY4OjTM0PUfJtChbFpbtYDuLVpyAZZeYL5YRQiBLghMjE3zr0Cl8qkJTIsrujiY+fNcGaqOhG6rxvRH84fNv8fyJcxTK184LcSXUxcL82O4tfGBr96oc71ZgJldgf+8wey8Msb9/mNl8EcO0MWwb23GwKyvmBfa5kmWTK5tMZvP0Ts7w5rkBNEXxBG2uU9VzMWRJ0FWb5Lc+8QThVZpcvXdVWdYgOD48zmtn+jnYP8L58WmKhknZsrEWrtlZ/K46zJcMpHkYkCROjU6iHT6FriikwkHu62rhR3ZtojYSQr0JOuJ3Aq7r8m8Pfp/7a9vZmmgg7b/9YlG3C6fnJtg3NcAPxi7Qn5vhP979EboiKULqrTHmVs0Y0KQAad861kce5ULuLQyngINN2clTdvJX2FMgISEJlTlpmBlj0Js4fF00BjbSEtzGApXJuxF+JUprcAfH554jY4556oTYFO0Mvbm9zBlj+ORQlWtggX0xZ02Tt2Yp2VkkIbM59kF8cpiRwonrMgYWoMshOkL3YLsWJzIvUna8/IOyk2O6PEDRzjJtDKBJHq2m41oYTrEyAY8hkGj0b6be38OsMVzhTbj2aVgSsldFEdzJXYmPcTLzAiV7HhcH0ylW/5sxBlElH5JQkZAqteEWllvGdEqUnfwSBsZLkzOvF0IIfOrFDDHLdpgrFHn5ZC8/ON3L+YlpJrN5cuUrl3QuuKMBTJuK8l8ZAcwViszkCvROzfL4xi62tdSTCgdvWVhjATP5AgPTc+RK11eOuhJsx2G+dH3VLLcDrutiOQ5vnx/kxZMX6J2YYXg2y2hmHsu+SsKt62K7HhWYYdsUjNUxnBRJIqhrSylvbxYCfJpSJdqxHYdC2eSZY2d542w/FyZnGMvkyBRKVzyM45Hte9RNtkO5olIpgOlcgXzJYGg2wwPd7WxtqaM5Ebvl7+pqQQjBI/VdNAfjBJX3tjcu6QuyJdGAKsn85yMvULLN6hh0K7CqPAOSkEjozfTnD+C9etfScbcq72vZ3mp1snyBaWOAgjVDQImR0tretTQEquQjpbfRFb6XM/OvkbemsVwDF4eMOU7GHF9xXwkZnxymxtfBxujjOK5Fzpy+oX4IBHX+blxsctYMw4WjFColnpZbJmuOk12hL6rwkfS10hHaTVNgE+fm37jmp7fkOJKPuFbP5uj7KVpzTJZ7yZlTlBxP8atoZyna2es4niepfDPVKKIywAoBhmUzOpfl8MAoX9t/nIMDo8vypF8PXGC+ZHB6bIqz41MUDQvTttnSXEd9LHzbvATvVdiOQ7ZY5vzENN85dJpnj58jfxXD7c6FZ7gKITBtm+lcgWODY3xt/wmODo5h2jenA+LiGbHnJqa5MDlNplAiXzawOxxaUrE7Jo/i4fqud7oLtwUpX5CUL0hTMMb/OPHqLV8Q37QxsJAEl7OmGCgcYSB/ANP1NAc8Lm/pihdR5cxzXVzsaohh1hiiaGWwXIP31f1jZC6lA313QOApI95b81lsbPrzB8gYYxXhlovXs7i9R2Er45cjpH2d3F/zOWr0dspOnqAS49oNqaXQJD+N/k2E1Rpen/hf9BUOYDpl3Iqe/FK2da8fslBI6W3sSX6Kev9GJAQRtfaG+yAJhRq9nYfTP8eZ+Vfoy+9luHgS2zGqYjbuZdUB4uL/hEdVrUgqca2ZBv9GwsqNS1kLBAFVxXVhYj7Hs8fO8b/fOMRYJnf1na8TjgvPHjvLyGyGD2zp5pN7thB5F8al7xS4rku+bHBkYJQ/fmkvB/tHrr5TBQsjxZ1UgyIAv+oNyTP5Im+eG+T3n3ud4dnrrzK6GhwXfnC6j8GZDP1Ts/zcY3uI+pca3a7rYjo2hnNxHBMINElBlSQs1/EEs1wXy3VRKmEXp6Ka55MVZCHhAqZjYzk2TkWvT5UkVCEjSxKu61bP4SXyeoySAAFVQ0LgArbrULa9HB5dVlCkpaEeby5yvHO5jvfwhUCTZDTJC4eYjo1Z7Yd3RXrlegAs16FkW2iSjFnpk0CgSjK6fPXp0qncM6fSFyE86nAAy7Er/Zarf5cdTwcDPBp3TZZRhHRNc51buc9l264Il3nQJBlVkq9K5XwpbtoYMN0SR2a/y4nMC0yUL8bNo2otCb2FlN5OUI4tq4TnYGE5BiU7R86aYrR4mow5Wt1ecuY5kXmezbEnqPWtQxXv3tpeVfh4NP3zzBrDjBVP01c4wFjxNHlrBtPx3HqKpBNUEqT0VpoCm2n0byLt66ryr+tSiKCSJKjEqlTO1wtZaMTVJp5q/BcMF48zkD/EeOmsl09gzuJg45PDhJQECb2ZjtAeOkP3VLP9TbdETGvkZlwxQgiCSpy74h9lc+z9zJuT9OUOMFQ8QsYYo2hnKDsFTKdUyRcJ4JcjhNQkMa2eJv9m6v0bCCpxFKFXj3mjfQn4NDKFEv/9mdd5/Ww/s/ni1Xe8CRwfniBXNsgWS/zKkw/e0nO9l2E7Ll/dd5yv7T/BmbFrZ8bUFJmI34cAypZF0TAxrxZOeDdAQMCnkSuW+fK+Y3zjwEkm568UYr159E7OMl8sY1g2/9fHHl3y1ZuOzdcHjvHFc/uYKRdxgYim8zM99/LBpg28OnaeY7NjjBQy7Jsa5OG6DizXpT83y7xR5Ne3vY+7a1qYLOZ4dvg03xk6yWghiyIkPtqyiccbu9kUr8dyHf7q3D7ypsFUOc9Qfo7TmQlc1+ULD3+GtlCCOaPI25MD/OmZNzmXmeIfbnyADzRtoD18Udiq7FjsnRzgmwPHeXtyAMO2CMgqn+zYzme6dhFUVL4xcJy/6z3EWHEe23EIKCr/YMP9vK+xB0WSeHOij9859Byf6riLbw0cZ6qcp9Yf5vGGbn5hw9VphrNGiS/1HmIoP8fhmRESup/2cJK8ZfLK6Hl+ZcvD3JtuR5Nlnh0+zRfOvEXRspCEoCOS5Efbt/GhpitTRS/GUD7DX5x9m4PTw8yUC6hC4rNdu3iyecN151PchDHg2SGvTf4Ffbn9zBqe1S6j8WD6p2gKbCWi1nhCOgt0BpeM556HtrJWdG0s12DvzJc5nX15icjRmfnXiaoNqNLKxoCETHvobn6y/feXrIA1yXfLFRAXT1QxrY6wmqI9dLcn5bxIavaiV0BBESqypFZK5i7uvzn6BOvC91cT8AJyDOk6HtPiY9X5uqnR27Fds7IqX7DLRYWxzZNOVipqgB7LnE6jfyM/2/WF6nFkoV63YMxCP1R8xLQGtsRSbIg+ioON617klF/spZCEhBCyd2/E5ffmRiAJCGgq//2Z19jXO3TVeGtDPEw84Cfk0/FX4rfZYpmJbJ7pXJ78NSbrDc1k+O6RM2xrbeDBnjbUa5E8u05sa6mnbFpM5QrMF8vMl8rkygaFskGhbGLcpFv5ncZfvnqAZ46e4cLEzBXbtSRj3Leulc1NtXSkE6QjoeqqyMXFtGzGMjkOD4zy4onz9E7Okile+T1YgCxJBHWV2miYukiIVDhIIuSnJhykszaJX1sdZU7whsegpvKnP9jLwb5RpnNXVi+ti4WIBfyEfToBTUWRJbLFElPzBWZyxWu+xpl8ke8cOc2961q4t6ulWhkjSxLbEg1s3/Mx/LKKYdv05ab5twee4eG6TgAMx1tFf+Ghz/BTP/giH2/byrZEI9OlPM8On+auZBN/fPoNwqrOr215lKZgjHmzzH8+/DyykAgqGo3BGC7w7MhpHm/o5pPt20n5gmSNEq2hBKokk9SDPN6wjj3pVv7Ba1/Cp6hc6vf5ev8xjs+OEVJ0/uLhz6BLCnNGkajmJ6hoCGBbop5tyQZ0ScHFZTA3x7/e/112pZpJ+UOUbIuxYpaSbfFf7nkay3V4bbyXv7lwgA81b6QpEL2m0N+cUeTXtz3OWxP9vDx6jp9dfx/dkRRnM1PU+SPsTDWxK9XM3TUthBSNOaPEvskB/vDE69dkDLiVHKb/cOhZHm1Yx8fbtpLQg+TMMv/sra9T4wuyq6blugyCGzYGXNdlonSeocIxMuY4DhYSCj2Rh2gL7SSq1l3nJOy5iDZGHme4cJyCNVedvKbL/Vfl7PdEc279xH81LKgI3mg/NDmAtkqCTYqkoXB9STZCSChCIyKtjk67EAIZBVm+3p6sDkqmxSun+7gwMc1srnhZAk7Yp9NeE6envoaOmgQNiQi6IqPJMrLshbgM26JoWMzliwzPZjkxMsHb5wcrSYTLw3ZcpnMFvrLvGHs6mlCkmzdsLsVdrQ20pmKUTAvT8jLqLdvBtB1M28awbAqGQdGwKJsWk/N5vnnwJMZyuujvIjiuy+hslldO99I7NYvlLL+q1xWFRzd28MTmdbQkYyRCfqJ+32UTtOO4xIMBaiJB1jfUcHRwjL0Xhnjr/OCKfUhHgjzU4yXYBTSNgO6V6/lUpfpf2K+jyKuXE2I7Lq+e7qN/eo6ZfOGydzWoqzTEImxuqmNdXZL6WASfqqAqMqrsvV+GZVMyTbLFMuOZHEcGx3jr3ABF01oxP8ZxXeaLZb6y7zibGmvxa2pFJRPmjBIHpgaxKhwcs+UC48UsVsUt7ZNVfD6FhkAURUg0BKK0hOIIAaczE8yWC5zLTjFvlhkpZAmrOoZjM5CfpT4QYaqUpzEY8+65L0R7OEFHOEFA0UjoATTJU3iVhcAvafhkFU2Sq673xTg1N47l2DzU0EljIIYsBDHdjyw8ZVHHdZkzShyeGaZke/cja5YZK2YxHNtbnbpeWOD+2nYaAhEQgr75aTRJZrqUpyEQuabi+JQvSL0/QlTz4VdUOiNJFCHoy81Ssi0s12GmXODtyQFkIWHYFr3zM4yX5nFwkdwre0MdXGaNAifnxik7Foemh/HLKqZjM2MUGSvOkzXLt8cYcHAYLZ5m3pysstzJQmFd+D6iaj3qdSd9eTHqtK+DoBJDERpmhUAnb84sIfVZwxquFYbtcHxojExxaZa8ANpq4h5fQFMtmxq92vNY0L/iR1g0TMYzOdpScfyqwr7eYTKF0oqTVdmyOdA3wvDcPC3JGD51VfN1aYhHaIgvX367kIGfLxsUDZOSadE3Ocv3j5x59xsDjsu+3mEuTMysWCkR9umsq0vy1Pb13L+uFU1ZviQPPJW+kE8j5NNoScaoj4WJ+HxM5wqcG18+YVcIQWsqzn3rWqmP3Z4SZ8d1OT48znzJuMwQaEnGWFeXZENDmu0t9fTU1xAL+leMC5dNi5l8keZEFJ+icHhghKlcYcVn77guB/uGGZzJEPJpBHSNomXwyth5yo6FJinVCdisGAbgxad1WUEWAl1WiGg+wqqOLimYjk3BMpg3y9iugyrJlG1vHN+ZamZdNE1wUZlcjS9EUg9Wf7uWGP1izJYLhFSd9nACpbJ698lq9frKtsWr4xco2RaKkLzYPFzMV6gcRxKC7mgapRJ39ysqAUXzlGGvMQklqvqrsXu/ohLXAowr8ziVnIbpUoFXx3sxbAufolZzqSzH9oy2ayBnW7i3C8aO4Xj39qG6DhqCUXzXef9uwjPgMFI8sWTFLgmV5uDWSvz5+rGQPKZLIVTJh2l7bi7TLd807e8afjjhuu5lhoAkBGGfzkd3bORD23qoi4Wrg8eV4NdU2mritKRi7O5s5ne+8SKHBkaZms8v+3a6rkumUGLvhSGifh++6O0TbRJCoMoysYCfWMALrwk8joR3M1zXxbBtvnHwxIrlnpIQtNXE+eTuLTy2sfO6ji8JQWc6SSzgx3Rs/tv3X1t2ghzP5Dg5MkF3Xeq2GQPAskZr2Kfz5NYentjSRVdt8ppCTrqqUB8LUxftZmd7I//juTd589wAwzOZFUfSTLHMwf4REiE/zckoM+UCf3bmLf7soc+wMVaLLATnslP8r3P7FvVPLEniq3CFVv5ykYRAFRL3pdv5hY33V8sBLcdBCJAQXrIfHsfCzXE/eMmKZfuiF2ThWm3XIWsU+cKZt/m9e36EXalm/LLKYH6WPz/z9pLjCAR+eTEvzvWr6sqLPIHePbq4f9k26Z2f5ovn9/Olxz9PSzBGzizzwshZXhu/Vjp4ryRfk2Q+3bGDB+o68MlKNYlSEsv5Tq6Mm/BxucyZo9iVFbuEgk8OoUuha1brW/aorovtGkvqyz399LUSrTWsDpLhAP/sqYf46Yd30ZSIXpMhsBiSENRGQ/zWjz7Boxs7r0o6893Dp5nI5m66jPGHAYZlMzqb5a0rhGGaElHu62rhI3dde6LVpUgE/Xzm3m1016XQV/DYvHV+kOeOn3tHn5tPU/hnTz3Ezzyyiw0N6evOPRFCkI6E+LUPPciP3r2ZaODK4csXjp+jd3IW23Up2iYCSGh+fLLCcD7DtwdPXNf56/wRmkIxBvOzfHfwRIU22mUgP8tUKV81BK4Fyz2HheMBbIjWUbYtvt5/zHP748Xu580ShmNTsL05Jab68csqU6U8X+8/xu2uOTEdrypCAlJ6AFkIDs+M8Np475J2i6/t0t9lIagPRGgOxXh9/AL7JgeqbRfCMvZ1vrc36be8aHu4ODftynddl5w1zbw5tYh4RhBSk8tWI6xhDdeLTY1pntq+nie39Syx1m8EYZ/Oh7f3IAv4328eWbHdqdEJMsUSTuUjXsPKyJXKHOgbvuL4vLujiXvXtd70uWRJ4hN3b+aPXnhr2VLTmbwXRjg2NM7mptrbXtrcnIjysZ0b+dD29eg3yRQY1DUeXN+OYdv8/nNvrtju7Pg0U/N5VCHRHIzxibZt/Prb30QIaAhE6YqkaAxEr/k9loTgH218kFfGLvD9odP85dm9WI5DOhDmE61beX/T+msqgTsxO8b/uXCQvtw0p2bHmSjO8+3B4zQF4/z0ut2sj9Xy8fatvD05wDNDp/jk81/Adh0CisZHWjbxdNsWGgNRPt62ld898jyW41DjD7Et0UBjMFoti7wdiGg63dE0H2zayE+8+Ff4FZX1sTRhVac+cJHS/+/6DvPaeC9jxXkmSzn+3cFnSPgC7Ew189GWzbSG4vz2zif5Wv8x/uepN/jdI89jOw4NwSi/svkRuqNXVvW8FDcxwwoCcgQJCa9S0sF0SmTMUSI3IVt7IvMcOWtmSQZ+na/nBnIQ3hm8MHie3swsCZ+fp7s2rcoxXxnqZd/EMBFN56c37XpX8i3cCYgFfGxtrufRDZ2rEr+XhKCrNsWmpgyxwBnmVqhUKJkWE5kcc4USydDqJIe+V5E3TE6OTK5oCwgBbTUJOtKJm2LN8+q/vSTMlSoCXBeyxRKnRifZ3FR74ye7AYR9Ot31KT64pRtdkVehqkbQFI9wV2sD8aB/xfLasmUxNZ9nKlegJhLkM507mWzM4eISUDSimp8dqSbCqo+tiQa6Iqmqa/9fbHsf6yI1+BWVkKrTHIyhSBL1gQiP1nexMVZL3jJwAb+s0lTZLhC8r6GbgmVQu0LCW2Mwyo+2byNvlslbBspCPL5yHEkIIqqPnckmGgIRZstFXDzug8ZgFJ+sogiJH+/cwWQpj+M6+GSVpB5ka6KetD+MLinclWzi/7fno0vOvT6a5lc2P0xHOHlVL2JA1XhfYw+O6xDX/DxU18nGWC1hVWdjrJak7hEJhVWdz3bt5InGbmQhEdP8yJLEB5vWV0Xl9tS00hFOUrIt8lYZv6yiSjJxLUDK58lqt4YSPN22lUfri5RsExcIKBpNweh1h1xueEQUQhDXGhksHK1a8ZZb5nzubTZGHkWXQ9dsEDiuRdkpMFE6z/ncW5ScBZY6gSLp1PvXo4o7g7ylPzvHkakxmkKrJ908mMuwb2yYpD9QKca7dpiOjWnbWK5LRLszDKpbhfX1NWxprqMxsXox4IhfpyEeoTUVZ25gdNk2rgsT2Tyz+eKaMXAVlE2Lgem5FbeHfTqpcGDVRIYa4hEifh+KLC1LbVwwTHonr1zaeCvQmoqxvaWBllRs1Y4Z1DXqY2HaaxLM5YdXyHOBmVyRyWyeumiYnliaHpZWFnVFPBKwSyfu+2rbq/8OqTr1Ae8788kSzaE4zaH4in1rW8QXsByimp+tiSvzzChCUOMPUeNfOTenO5qm+5KhuSOSrP67LhChLrB0fEj6giQrk+/VoEnyEu6DllCclsp1p+QQKd/Fvq2P1bI+trKRuXjfleBXNHqiq1P5dePGABL1/vWcyr5c0R5wsV2LI7PfIaW3ktCa0KQAslArls7ikILHK+Dx0puUnDyzxiAHZr7OaPFk1SsgC5WoWk+dvwflHS4ZvFaEVI2kz0/C9+4gSMoaZWZKBYqWxdZU3TvdnXcMkhDct66VrS11q0oRLIQgHvSzuamWwysYAwCT87lbTnb0XoBh24xnV2aHTEdChH2rU9InhCCoayRCfgKaSrZ4uS5D0TAZvIJxcisghMchce+6llV/VwOaxvbWeg71j6yYCzGbL95ysqM7BaWiQalkYpk2iVSYfK6EUbawba+0UpYF/oCOz6ciSZfLrJumRblkUS6ZWJYNQiDLEpoq4w9oKKrn9ZnPFimXLXRdIRT2eZTUpk2paFDIl9E0hWg8iCQJHMfFMm1mpnNEYwE0XUFehe/hpoyBjvAeDs99F8MpUHbyuDhMG/18Y+i3aQ5soymwibSvk4haiyp8FRUCC8spkTNnmDGHGS+eZbx0lulyPw5Ls3pDSooHan4SnxS8qaTE24lPdm95p7uwBIcmRjk6NYbh2D/UxkDUr7OpqZbW5JUt7RtB2KfTmU5esU2u5JX4reHKsG2H+WUm5QUkggF0ZXXzhyJ+Hz51eWPAsGxmcrfXiIv4dLrrUnTXXV/M91rgUxXW1V6Z3rtgmCuWdC5nQFwawlhoc+2hjUrm/6JDL7fvpee+HeHSvW+c59UXTnLu9Cj/7c/+Pv/7z1/h0N5exscy2LZLui7KBz92Fw88sp6a2ghLFr2uS//5Sd5+/RyvvHCCwf5pFEWitj7Gxi1NPPmxu1i3oQHXdfk/f/Eae18/xz0PdfP5f/AoAAO9k7z07HG+/MU32bajlX/5H36UYMhHqWRw/vQ4//Tnv8A/+9cfZde9XcSTN1+pdFNflYzKwzU/zf6Zr3I293o16a/s5LmQe4ve/D6vCkAsVSfwiAfdStKhUyEXuuiik1DoCO1mc+z9tIfu5l2rUnQH4NTsJOcy07SEY+90V95RPLS+/ZYpCQY0lcYV6v0XUKjU+q/hyrAd54rMeUFdRVVWd2EQ1DS0FRL0LMchd5uVHO9d10pzMsatqALVFJmWZNT7DlZIzCibFkVjZZI3B4uxwiF8SpyAksIvx5ZsLzkZynaGmHZ9SZ4TpeO4roNPjhHTW1Y8d8meQ5ciVaryWw3TtBgbmeM3f+1vKBYMOnvqufv+dRhlm7dfP8uf//4LzE3nuP/RDfRsbKju942/28crL5yg/8IkPRsbuf/h9RiGxfkz47z9+jneeOUM/+o/fIJ1GxqoSUeIJ4IMXLhIuz06NMv4yBy6rnDm5AiW6S2Wc9kSg/1eu/Z1dQSCq3MfbipnACCuN7Et/hQJvZkz868yXvL0CRxscG1vrX+NFQ4SCk2BzXSE9lDv7yGhNSO/A1UE3+49zRuj/Xy6exszpQIvDfUymp/HchxS/gD/ZMf9JH2BajJJwTL5v19/lqliHtt1SfoCbKup46c37VrxHOfnZjg0OcLe8SHGC3mPbKKyzS8rdMWS/NrOBy+6CSsDwzfOn+TN0QHmyiVkSaIhFOZH122mI+rRdgLkTYM/PrqXc3PTnJiZIGOUODQ5ypGpser5n2jp4rPrt//QqOptaqwj7NNvyWpCkT052yuhbFoY1poxcDW4eAbBlbavdqWf7TgruswFAuk2fyPddSlSocAteVelSmjkSkJkhm1RvsK7KpBwcckY/VhOCb8/tmT7vDHMZOnkNRsDHpvtCeaMXoJKLbK08rdkOwZle76iU3ObcqBcjwirVDL5yZ9/hNr6KJquYNsO23e18bu/+VVOHhumsSVJz0ZvpT/QO8Vbr5zBKFk8/L5NfPCj2wmFfTiOy+xMnuOHB/jin73C17+0j3/wK+8nkQwRifoZ6LtoDEyMZzAMi607Wnn7tbNMjGfw+VUK+TITY3MEQzrJVAhVWx2q85ueaVXJR42vA00OoktBknpLRYwmWxGjKWI7BjZ2hbrBU+yThIImBdClILocxC9Hiahp6nzdNAY2EVQS71gFwVAuw2sjA7SEY8yUihQsA12WEUDGKFWZqxYgIWgJx/ArKocnRxkv5AgoK2Uou8wbZV4cPM+p2SnKtkVjKMJEIcfe8SF8ssL6RA31ocgSh4hh2wzlsrw20o/lOGiyTMEyOTgxgiQEP7d5N3Gfx0gmCUFY00n5g556FYKgolEXuOhKCmu3ZmJ8t6IlGcWv3RrDUpbEVasTTNtekalwDRchECiyhO0sz5RnWFZV0W61ULbsZZMHwXu2q6k/cC1ojEcJ+2/N2CdJAp+mXtHXukBpvcD5MlM+j+OaqHKQsFKHKgVQJD8FaxLTKVTbTZVOIQmFWaOXgnU5s6PrOlhumZnyOWzXxC8n8Msxz9NQPIzpFFClII5rYjpFZsu9OFhoUhCfHEOTQkyVT+O6Nn45houL7RhMl8/iujY+JYomRZCFwpzRh+2ahNVGBBKWU8B0i5hOkZCSxifH0eRrSwoEkBWJzu5aNm5tIhTyISqx+1QqQjwRYmY6x+zMxVyXMydHGBmaoaWthl33dNLVU1/dVpOO4NgOsUSQY4cGyGVLhCN+orEgmbkBikUTXVeYnsphmjabtjVzcG8vYyNz1NRGKRTKTE3OU1MbRa/kKqwGVmV0VCUfKb2VpNZMm7mLwcIRZo1h5s1J8tYshpPHdMpeOY9QUISGKnyE1BRhNUlEqSWuNVHnX4ci3h2TVM40eHW4n9pAiMeaO0kHgpiOTf/8HDHdt6Q2VlcUfumu+8gbBn9w5E32jQ9f8dhDuSzf7z+LJiv8WPcWHmlq5/zcNDOlIj5F4bHmTv7ehruW7FO0TPrn50j7gzzdtYmkz89wPss3L5ziiycP87GOjQRVDZ+ioMsKH+nYgOM6FC2Tc5lpNidr+YWte6rHC6jqdUtc3qmQhKAuFl71WPPF40uoV6kDtxwH21kjHboaZMlbuZbN5eP0uZKBaa2OMbDgDciXjRVXwoosXZVUajUhgNpI8Kqephs/vlgxJLIA23ErxpFL3ppgvHQM2zHwKzEc3SSlr0eTglXuFxebgjXFSPEAmhTEsOdZzuvg4mK5JcaLRynas8S0VuJaJ4qkMlfuwyfHKknlBoadYyD/GqrkJ6TWE1VbUFSd6fIZdClCVGvGcS1KTobh/F5kSSWmtRJWG1AkH2PFIxStGeoDOxDIlOxZDGeesp0npjWT8q2/LmNAVWXuursdXVcRlfiNJAl8AZV4Msj4SIZiwQutuC6cOj5MsWDgC6j4/CoDvZNLjpebLxGJBjg1OEwuWyIY1omnQpSKJnMzOcIRP7PTOWzLoau7jngixMjQDF09dRTyBtOTOZpak0irGEta1dFRCImYVk9Mq79640uw2E23+N/XaxjczL6LYdo2Y4V5/uf7fgR9ETXlPfWXx7IWzhLSvMn4arWo+ydGyFkG96fr+WjnBgSwo7aRu2sbOT07xdm5ZaxqQBaCf3f/+4nrPmRJYr1Rpj4Q5tmBc5ybmyLlD+BTPMW2dMB70f2qJ+wRVDXqgtcnaflegMCjZq2NhtFukTGwcJ4rYs0OuCaoikxdNLxi0t7wbJZc2cB13VVZNNiOw3hmnvkV8gJUWSYevH2VTD5NpSYSIqDdOlmvq921ixS+Jufmn6E99AhBpZaMMcD5+edI6euXtLecMgP512gK7Cak1jFROsF48fAy55XwSVHaw48xXTpL3hpnzuyjK/wEdYFthNR6YlorPilCwZomZ41xT80v4ZO9WkDHtWkK3MNQ4S0s16BozTJdPkNYq6Mj/D5koXpqflh0hB9nvHiMjDGAg40mBYnrXaR9mzgy81eoUoio1nzN90ySBLUNMST58runqgqO4+As8i5NjHku/peeOc7Lzx5f8bhCQD5foiWdoq4+iuM4DPZNEQjqZOYK+PwqrR1p2rtqGeidIjdfIp8rMTGW4ZEnNq1qCOtdQ+tnGhaD58Zp39CAY7sU8yUs0yZec3014ZZpk58vIoQgmrjxDMuQpnFfQ2s1Dr+aKFoGAq8udjFkSUKWxLIfa0BR2ZBIE1/kldBlpTrBz5SKlOy1mPSlkCWJ2kWStmt4dyOoaWxoSHNieGLZ7dO5AiNzWaZzBVLha1/ZLQcXODEySbZYXtFrE/brbG66PVU4Aq90cjWVEG8WllNCET5koYKgSj+/GC4upltEkXwokg9ZaMtKnhvOPEP5tyna0wSVWiShVOTML4dAEJBTV5ROd7Cw3DK6FGVh1MyaQ0yUTlC25wirjYAA10VCJiDHK3LtV8ieXAFCCHw+leVH50vh4lbep7vubmf3/euQrvBMWztqiCdCJFIhIrEAA/3TSJUSxHRdlEjUz/rNDfzg+ZNMTcyTmSuQyxZZt75+WePkRvGuMQYK8yVe+cYBWrvrsUyLQq6EUTKvaAyc2t9Lsi5Ksj5WtZBMw6KQLSLJ0k0ZA7KQiOsVgZdVnki2pur5Tu9pzs7N8OpwH/fUNTMw75EV6bLMpuTlJBKKJBHVvfrTan8E1UnOcd1VT6x6L0AIKpKsVxUCW8O7ACGfxl2tDXxl77EVh+tjg+O0peLXLVJ0KVzX5YXj51b0CoBXkrqt5TaV5AoIaAqSEO+Kd1USMs3B++jL/aAiI6xT69tE3hpnuLCXrDFEUKlBk4LU+7fTO/8ifjmO4eSRVhSrc5k3JzyBOxcCysoluYvHXcspUbCmGMy/wXT5LD45RkLvIKa1cSbzbfLWJCG1DnBxXYt5cxzHdbBd4xKD4sZv7LXPA4JUOoyqyqTSYbbc1ULqCvNYKOJDUWT8AZ2m5iSToxlM0yYY0qmrj6EoEh1dtXztb95moG+SfK6MLEu0dtS8e8MECzDKJmP9U2SmcziOSyIdIVkfY7Rvisz0PEII2jc24gvq9J8coVgoY5RMZiYy2LZN36kRcpkigZAP13UpFw36To2SzxbRdIXmdXU4tsPBV05R0xCntaeBeDpCIh3h/NFByiWTeE3YU0ArmZw51I9l2oTjQVL1MbLTOXLZIqVCmUDYR6ohTqoutuQaBFy3gM21oiMaZ1dtE+cy0/yf00c4ODFK0TLwKypbUrVsq2m4bB+BQL1qf9asgUshuHpy3xrePQjoKhsa0gR9GvmysayBe3p0kqZEhO2t9SSCN8boaNo245kcb54bJL+COmJQ16iNhmlOxG7oHNcLgUBXV5Zivt0QSMS1Nmy3jOPaaFKAgJJEET7iWgdBOYUqh/DJMVQpQNE3V2WKFVzuUZWFTlxvRxY6iuRHEhKaFEYgkdC70KUwquRHCBlVDlLv34EsvHCJEDKK5Ceud+BX4kTURnxyDEXo1Po3o0leP2Sh4pdjqFIITQ4hKv9TJB++SglknX8bQXV1WPuWvW8CunrqObi3l7nZAuOjGdatb6huW4DruliWg6JIVc9DXWOMqckshbxBY0uCVG0ESZZobEni2A5D/dPYtkMw7COWCK7qQvWWjJLlosG5Y0OM9k4SjgVwHJdwPMhI3yQDZ8awDAtVV4gmw5w7NkgxV0LVFI97wHGZGJphfrZAss6LFZmGxcCZUcYGptE0BUmSCMcD9J4cIT9fQvdrKJpMLBVipG8S23JQNQXHdhgfnObc0UEc2yGaDJPLFJgenSOXKVAumgRCPvLZ0mXGwK2CEIJ0IMTGZJrhXJaDEyMM5TIkfH7uqWvhvoYWuuOXk4J4lRjX/+AX6h5+WBXzhPCIVt4l4+sargJdUWhJRmlLxTkzNrWsvPDgTIZD/aMcHRhjT2dzZQK99ids2h6R0N4LQ5wancS0l69caIiHWVebJBa8fWyiPkW5oe/8VkAICb8Sp1nZs0grxluQNCl3L2nr4tIWerDSZnnJX0XSiWudxLQ2BEv1Fmp8S/MQdDlMU3B39W9ZqASU5LKehPbQo7jY1WO6rktC77rsHAtoDK5c8r1a2LiliXR9jKmJLHtfP0trew3BkH6RQdCyKZctcF3qm+KoqoLPr1HfEOf86THmsyW6NzZQUxtFkgQ1tRH8AY2RwRkUVSaZCuH3r25eyS0xBmzLITM1Tyga4KN//xEAHMdh855OahrjDJ0d48jrZ/EFNNo2NFLXnMS2bU4d6ENWZHp2tHPh2BDZWY8SMxj2s+ORDYz0TjDaN8XxvRf45C++j4b2Grbd183W+7urdIxb7l3H6QN95LNFjLLFS1/bzyMf20mqIc7540O89NW9NLTV0La+gfaNjVw4PsSxN86y85ENN3y9K020yyUzLvx2YGIEIQT/aPu9/Pj6bV6bG+7BytBkGUkISraFs7g/rH74490IIbzV1rtmubWGK0IIgaYofHzXZv7Hc28wnStc1sZ2HI4Pj/Nfvvcqv/2j76e7LoWqyFd8pxe+O8d1mczm2dc7xH/53qsrGgICuKezhcc3da3WpV0TrtewuT0QVzVQVjIALmsnBGKVp51Lj3krznG9/enoruPh923ipWeP8YPnT9B3foK7dnfgD+iUigbTk1n6zk/S2V3L537+UeLJEP6ARmNrkqGBaUpFk0jUT11DFCEEqirT2JJk4MIk/qDO1h1tq97vW3bHQrFAdYJ2XZc3vnuYmYksqqYQigZwbIdSwUBVZXxBDdu0l82MdF2XH3zzAJmpeYKRALpPw7E9IaOrfTOu62IUDfSAhqp5q0O7wuIUTYXxBXUkRV4ySd4o7Ap/QME0Kds2RctkzigRVLSKMtdFGI5Nf3a24iW4uSSoq6E1HOPkzAQvDl7g3voWagMhSrZFjT9IR/TK4iDvFawlD95ZkCXB0zs38eqZPg72jyyr6VAyLc6NT/P3/+Tv+OhdG3l0Ywc99TVXTCocmMnw3LFzvHK6lyODo5TN5Q0B8FgA71vXSnfdlal7Vxtr7+p7Bx/6kbvYvK2Zw/v7eeX5E3zz7/ZhWTY+n0Y8GaSts4ZHP7ClyiAYCOp0dKXJ58rEEkFqaiPE4hff566eOoYHZtB0hfbO1aeqvmXGwKWWopAE2ekcRtkiGA0gJMHW+9Zxcl8vR988RyDkQ9UVpkbmOPz6GfpODGNZNvVtKQQwM5ElM53DF9CrdZ71rTVcOD7E5MgsXVtaCMX87H3uOP2nRwnHAkTiQe59chsvfnkvkizhD+ps2t1Z9Tisxmc3lp/npaFevnBiP47rMlUsULRMzsxOcmJmAkkIPty+noeb2tlS0QZQhERPvIbXR/v5zTef5/f2vwp4FQNN4Qjbaur5dM82/PLNrxIebmpHlWS+dv4E/8+BV5GERNLn50NtPT80xsAa7jyoisTnH9qJ+prESyd7l+UBcIFC2eS7R07zypk+In6dRNBPXTRMUNc8sRfbJl82GM/kGM/mmC+WyZWNFQ0BIaCrNsXPPHI3Gxpq3oWr9DUsRr5s8N1jZ3j+9HnmCkVkSaIzleBfPPkwunJj46fruhiWzf/9zefYmEzx5Of28Pd+7hHqm+LLZu//43/+JKZhEwovLUGVJImG5gTxZIhd93ZilE1c15sLFVlC0xUi0YAXIgcURaImHeEP/urnkGWJVDpS7b8Qgo/92G4ef3IrsiQRjq5+6OqWGAO+gE7XlubqpC2EoLWngVA0gGXa6H4NBDS2p3EdF6Nsovk0mjprCUZ8tG9oIFETwcUlWRsjmgwTqSRLqLrqlWkIWL+jjcxMHlmWiCSC6H6Nri3NpJsT6D6VZH2MYNiHUTJxHRd/yEc4HiCfKZKsi6JoCg1tNSiXkHDsqW0irGqsj199MAioGutiST7asXKYYVOylqQvUE3v+9r5E2SMEh3RJBFNR5OkKg3rRDHPGxX2w8eaOxHAtpp6QqqOX1nKHCYLr8Lg13c9xK7aRmL65S9Iyh9kV10jAVVlND+Pi0tI1VmfuL0rnjWs4Vqx8M111SZ5bGMnjuPy0qkLmMuwBLrAXKHEXKFUYYJUifhn0JSKJ9B1MEyb+VKZwlWEojRZpjYW5tN7ttJdlyJ0G8mG1nD9yJcNLkzN8PUjJ2hLxGlNxFBlmUTQf9M06y7QPz1HZ02CZG2Y9tTyCychBA1NKy+qNE3xFAdjV090FUKgagqd3ctXryRTYZKpW8cVc4uMAY22DUsz4hs70tS1Jj2u70U1l5GE5wZZXDIXWaYksLG9BiEt5Qlv6a7Hth1wXeTKhL5pz9JyI9d12f5gj0dSgqgaKAuoa0lS17I0KWV7uoHt6csz+pdDRNPZWdvIztrGq7Z1XJeSbfG3Z4/SEIxwT10zu2ob8ckKDl6Y4Xt9Z3hluI83Rwd5tKkDhGBTspZNyct1rxdoh39x2z0rnlOTZdoicdoiq6PWNzYzz+x8gWLp4sAaC/lJx0OEAmuD5xpWDxG/jz2dLaiyzHSuwKnRSYqGuWLNjO245MvGitUBV0LYp9MYj7Cro4kPbesh5NPWXPbvcmRLZc5MTHN0eJzP37eT9mQCuZKgp8o3xw8jBGyor6ExGiGg3l466ncKtzXLQl7mAV0rg5K8AoXm1XScF7tZ3mnYrsNYfp5jU+M82drNI03tNIS8+lMXF8dxOTs3zdHpcebKpXdloeBze0/z3L4znBmcxHXBtGweuauLn3j/Du7qbnqnu7eG9xhqIkEeXt9ORzrBb3/tBU6OTFA0rVWrjpElCU2R2NHWwPu3dPORuzYgiXfHeLGGK2O+VGZoNkN9JMzutmZCq0ThLIRAVxR+88OPr8rx7hSsFWDfRshCoi4QIqhqPDt4nrJjc39DK0FVY6ZY4M3RQV4cOs9EIcdn129/V65MnrpvE/dubmM6W2B0Kstvf+GZd7pLa3iPQ1cVumqT/PnPfpJvHDjO37x1hKND4zd1TEkIArrKh7ev50Pb1tOZThAN3D7a4TXcPMqWxXyxTEDX3iXFmHc27mhj4MUDnlxyXTLChtbL3ejvNgjAp6j89n1P8NzAOZ7tP8eXzx6jZFn4FIWkP8imZC0/uWEHDza2vStf8EhQJ+jXqEuESUVvbSXEGtYA3krNsh0O9A3zpb3H6J2cvaxNQFPxaSqKJJEtljAsG1l4ojw+TSWgaSRDflpTMZoSMVpSMdbVJkmFAgR9OqosrXkD3iE4rstoZp5vHD7JqbFJxrM5CqZJIuDnkZ52Hl7XXo3Zu67L73zvZU6Pe+2m80XKpsXTf/jXSELQFI9wT3szP/OAx4NQMEy+tP8ob1wY4Dc//Dj/Z98RDgyMkCsbJIIBHu1p5zN3b6s++wMDI/z7775EwfAIr57evpHH1newLn15jpVlO4xm5vni3sOcHptktlDCcV1iAR9Pbe7hsfUdpEJBDMtmPJvj/+w7wpnxi+3iAT9PbOjkQ1vWE6nkpxQMg1/50nf40KZupnIFzk5Oc2Z8CkUSbGuq5yNbN9BTm/LKpVcZd7QxsPfkICG/jnaHMMwJ4fFk70w3ENF0Jgt58qaB6dioskxE81EXCNEYihBUb51Qyc3A008ASXiiKmtYw61GplDi2NAYf/36IS5MzFyWCNhTX8N961ppTcbQVJmyaWE7jkehK0mokqcq6ddUon4fEb9ONOAjHvR7Zb9rRsA7DkkIDNuiMR6hsyaBJAlmcgVeOHWeoKbhU1Xqo17y3L0dLaxLJ+mdmuXI8BhDsxk+cdcmNEUm6vfRHI9Wj+u6LrOFIsdHJvjSgWNIQrCnvRnT9sbc5CUMlvXRMJ/atYVC2eD3X36LmUJhWeIr0/Ym+D9+dS/j8zlak3G2NunIQiJvGoT9WjWJUQhvvCxbFhvq06iyjOM6zOaLfOPwSdqSCdbX1RAL+HBcl+G5LF8/cpK6SIhk0M+jPR0UDIMDAyPEA37AZWvT9YsBXg13xix6CVzXpWzanB6YoCEVpad19WsubxWEEKT8QVL+tVX1GtZwNeRKZc6NT/O9I2d4+VTvkm0L0tSPb+rkiU1dtNXEb6ky5RpuDQSeZ6c7nSIW8FMXCeFXVS5MzfDq+X7OTkzTWZOsGgP3d7biui7HRycoWxYFw+STO7cQ0DxZ9uXCq0XT5MTIBD+6YxMdNUmEAMO2L6Ocr4+G+eSOzZRMiy+8cWDFPs+XypwZn+LZk+f4yNb1PLSujaZ4FFWSGZ/PEfZpVRp0SXiy3Bvr03TVJIlXJv0LU7N89/gZzkxMURcNEVsUpjo/OUNXTZI97c101CTIFEocHhrj5NgktdHwmjFg2Q6WbWNaDqPTWSZmc4QDOsWySTZfqrZTZG8lcKnGvO042LaDaTk4FbWshdWDpshIkqjSWVq2Q6lsEvBplC0Lp6JCtXDcsuGtPhbOt+C28froYDsOQZ/mrVJshwVRNEnyGPFkSVy2InFdt6IlbmPZTiVJSiBJAqWyuhGXJDd5hpGFZTvV41q2g2HZS/df1EcAx3GxnYt9XUjIEpX22gp9XMMabhdc1+XCxAwvnDjPV/YtlYFdmECe2raezz+4k8AqJY+t4Z1BxKfz2PpObMfFcR1cFzY31tEYi1AwDKZyFW6YSugHFthVpUrCn3xFDRKfqrCpMc0jPR3XVHZ4NQGg0cw8b/QOosiCn33g7iXljI3xpaJEkhBE/V7owHKcKsldQNOoj4SZmM+Ru0QsqyUe5aF1bTzQ1eYdMxphd1szJ0YnmMjmrtr/G8EdZQwcOTfMDw5d4PXjffSPzmI7nlHw6pGlK4YHt3XwwT3r+cCepXzXQxNz/ODwBb735in6x2Y9zudYiE3tdXzq8e1saK1FCJgvlDlwZojf+INv8ce/8Sn+8nv7ONU3jq7KPHnPRn7koc38py++wMm+cVRF5v4t7fzyjz2EJAQvHTzHi/vPcW5kij/5jU/xP7/+BntPDjA+Mw9CsK4pxc995F7Wt6UJX5KwZFo2h8+O8PKh87x2tJexmXkCukprXZwdPU08/eAWGmqiS/YpmxZ/+LXXee1IL//gR+6jpyXNSwfP8bVXjjE2nUVTZTrqkzy0vZOffPIin/jodJbD50Z46eA5Tg9MMpvN4+JJtnY0JfmpJ3ezsa2WgG9tkF3DO4NcyeDbh07xlf2X68FHAj62tdTzyx+47x3o2RpWG0XT4ssHjrGvf5jzkzPMFAqVUtEy/nXty3JMXA/8qsrm+tpVW9xkimUGZ+bYWFdLUL9yGarrQt4w+Ku3DvFW7yAjmSzzJU82O28YbG9uwLpEQrsrnSLqXzo/hHUV27GXJeBaDdxRxkA6HubuDS201sUZGJ/j668cpS4ZYVNHHT3NF1WoGlIRmmuX1tV//61TvLD/LIfOjdDTUsM9m1qxbIehiTkOnx3mwJkh/u3PfJCelovHsW2H//o3L9PdXMPG9lr6x2b562f2caxvlHjIz5bOesZncnzvrVM8sK2dbV0e14BhWQxPZPhXf/QdJCHY2dOErirMZAu8drSP3/zz7/MLT9/Pno0t1MRCVY/AH3/zTd442sd0tsCGtloe39VNvljm3PAU333zFG8c7ePf/4OnaExFL/N6zOWKvHTwPAfODHH0/Ch7NrYQ8utMzOVQZRm/b2l8//zIFN998yTTmTx3b2gmFvJjWjbjs/PsOzXAb3/hGf7159/PhtY1g2AN7wxePtXL6bEpiuXLyYJaElGe3rlxzXP1HoBpO/zH773M2Ykp7ulo4cd3b6UhGkGTZX79q98nvArkT5IQtyS37FpK18ey8/z+y29ycnSKT+3aTE9dDYlAAMd1+cd/883LxnIAv6rcMtXclXBHGQOJSAC/rtJWnyAS8vHdN0+SiARY35Lmvs3t1XY+TcGne5Of67oMT2Z483gfI9NZdnQ38vGHtxIPew9jNlvgRN84f/rtt/jma8epiQWJLqiUCUiEAzy0vRPbdjhwZoi/+v5+SiWTxx7eil9XOXx2hJN94/SOzLCpvb5yTi9DVgjBxx/ZSk0shCJL5EsGrXUJ/uzbb/PGsT6iQR81MY9gad+pQQ6cHkJIgofv6uSJu7uJhfwYls3oVJYj50b4+qvH+MrLR/j8h3YTDy9NfDEtm96RabZ2NfC5J++msSaKpsgUyhZCQMi/9IPqbEjxo496AkmpaBBdU3Acl5lMnkQkwFdfPsr5oWlq4+E1Y2AN7whOjU4wWfFYXYqQT6cjvUan/V6A4zq83TfIXc0NbGuqY0NdmpCu4bpeNcDNegUWsJpmo19TiAf8HB0ex7AsfKqyoncgbxjs7R/mvo5WtjbV0xyPoqsKJcNkvlSuhqCX9PUdsHHvKGMg4NMI+DRsx2FyLocsS/g0hVjYT30qsuJ+ZwcnOT88jSJJPLCtg7s3tFS3GZZNLOznyy8fYd+pQZ5+cEt14hQImuvidDWlKhOriRAQDwfoaUkTD/nJFcpIkmAqk1/yUGVJ0NmYZNf65upkatk2yUiAL798mLODk/S0pHlgazsusP/UIKPTWTZ31PPYji529jRXj9VamyDo13j58HlePdLLjzy4hXDAh7KIcMlxXTRVpjkd46Htnctam4tRlwxTEw9V428LyKUiWLbDl188zNhMlvlCeYUjrGENtxajc/PMl5Z//xRZIqCtGanvBbiul5AX0FRCuo6myOTLBmcmppkrFKmLXM5Ie6uxENd3XZY1RuN+P13pJC+f6eXg4Cjr0kkiPk+iuGRa2I5XOqgpMrbjkisZhHWNUEUzYzZf5OzEFPOl8rtGXv6OMgZuFEfOjzKXK9Jen6S1Ns5UJr9ku+O41CfCHDo3wlyuSDpR4X8WUBsPo8gyfl0lEtARQtCU9jiwFUVG1xQ0RaZkLGVFUxWZXeubl+geKLJMUzpGczrOuaEpJmbnsR0HSUgcPjdM2bRoq0uwfd1SJr+AT6U+GWFLRz3feeMkE7M5amIhwpfQ/25sq2NrV8NVDQFYkPl0yRfLmLaD67i4uBimTbCik50tlCldhc99DWu4VZgvlZct6wIoWzYzhSL1Me9bXQsX3LkQQtCajNM3Pce5iSn8qkK+bPKlA0cxbPu2ka+VLYuyZVM2LQqGge24FAyTmXyRqVweWZKI+n1IQlAbCbGnvZn/vfcIX3z7EI+v76Q1GUeT5Wqy4+72ZjTFj67ItCSinByb5MLUDLOFIqOZeb5/4qy3oHuXvLo/FMbA6HSWYtnktaO9vHa094ptc8UypUqMUgABXUGSvOzShQzTgE+9GCvCK2VxL3H1SEKQjoeXfZHT8RAXRqYplAxyhTLhoK+SLKgRCepo6uWTuabKNNZEEcD4zDxtdfHLjIFENHBNRECu6xkB/WOzfP+tUxy5MMpMpkC+ZFA2LQzT8ixi1+VdYrSu4YcQQV1b0bDtm5zhi68f5LeefgJFli5bXa0ZB+8MFj+Ha30GmizzLz74CP/9hdf5Hy+9ScE0SQWD/MTubaRCAWRxe2Lnr58f4NtHT/PsybOUK0bo3x04xpcPHkOTZdpTCb7wuU8QC/jxayo9tSn+x6c/wu+//Bb/85W9zBaKuEBNKMhP3nMXViW8URcJ888/8DD//jsv8qt/9x0UIdGWjPPk5m6SwQC+q2gfXLynt/ad/qEwBhZu5vrWNI/v6r7iy7W+tfaSZLulUszgSVDezHPxJllPOGlxcGgll5S3kepGcXmXgAoh0FW0GgAGJ+b4/lun+NoPjqKqMh/cs4HGVIRwwIcsC2ZzJf7dGs3wGt5h1EfDhHWNmVzhsm2T8wWeOXqOgakMT27tpru+hrpoiHjQG6gl1gyCdwK2ZVMqlDFLJvHa2DXtIwRsqKvh9370yUpWvZdv5VOVauhVWWZcW19XQ0sswujAFJRtbNm+TMMmoKn83IN387l7d3gu+iv04/7OVna1NvIvnnz48oFYgCwEkUUZ/ook0VmT5N997IklJYNSpe+67E2vuiLTk4zzu+97hHAyjKLK1XJ2x3ERgqqwUkDT+KvPfxJVltEVBcd2sAyLzMw8n961hU/u3LzsvVgN3LHGwPV854lIAE1VSIQD3LWusZq0txziYT9l8+ZLNxzXZXa+QGvd5WqBs/NFXBd0TSGgqwgEqViIwfFZCiUD07IvWxGZtsNUxkumioe9Ae9GcfjsMEfOj+C4Lp99/y52rm8i6PNWYaZlc25o+t3iuVrDDzG2tzZwcnSS/um5y7a5rkvRMDk9Nsl8qUzYp+PXFDTZG2ivZggI4Q3muqrgUxWCukYs4KM2GqY5GaMzncCvqletN1/DUliGTWm+RDFfvg5jQKDISyfaa4EiS6iuYPjQAEFFoaYxcZkxIIQgoGkEriG9RFPky3KoFmAZFqf3XyC4ox2hKVWVXUUWhOWrVDu44JoOomARbdDR/ct3Zn42z/C5Ubp3dCAqHC+WaWGUDOancyRqYysK9q0G7khjQCCqeuWm5emVXwntDUkOnBmiZJhkcsVqCeDi8WKhvG+BtOdmYTsOZ4em2NxRjyJ7ZEG245DNlxifnSfgU4mF/CgVi7CrMcXwZIaJmRyD43N0NF6UVTZMm7lckQuj08RCfhKR4E1RAU/M5pjOFNBUhT0bW2lMR6vhjLn5IqPTmXelYuIarg2O6ylgWo6NuYgEazpXYJnE5Sosx2G+VGYym0OWZVRZQpGkCgW1uGmN+OtFT0MNnekE58ammMkXl21TKJucG5++oeMvGAO6IhPQNaJ+nZpIiMZ4hK7aJA3xCPXRMI2JKLoir3kaFsEsm4z1T5KbK4DrEoj4aeyqY6xvgux0DkWTcV2XoTOjJOvj+II6pmExcm6Mpu4GRs6PUS4aqLpCsj5OIVuklC9TLhkoqkyiLk4sHbmMYA2g99ggZtlE82nIisT06Cy9xwaZHpklmgoTq4ngD/sZODWMWbaIJELEa6OousLQmVEs00YPaASjQULRAAOnh7HKFrHaKLGaKLpf5cLRARzbIRgJ4A/5sG2H/c8eQZIkEnVRgtEAiiozcHoU13FI1scJRPzgwmjvBJZpEYoFiSRDyLJM37FByiWTurYabMumkC0ydHYMISDdkkJRZcb7pzjw3DEUVSGejhKMBSjlywydGcUoGTQ7LpLrMj+TY7x/Csd2iNVGiSRC5DMFJodmEBLUt6cJRoMoy4Sbr4Q70hhAQDjgQ1Vk5gslJmZzZPMlT8va9SzGBRZCgG1dDbx2tJfB8VlePHCODW11+DSvFMTFMwJMy8ZyXJJh/6p00bQc9p0a5P4t7cRCPiRJUDYsTvSNMzY9z+b2ehpqokiSx3h494YWjpwfoX98hteO9hIL+9FUz400M1/gzMAkZwYn2d7VSE3MKwW8UciyVB3Ybcfx6rhdr9phZDrL/lODl8VgF/5eYEY0bafqQbEdj/GwbHhljFJl4lgbPFcXluNQMixM28aphJq80JK75G+r8jwKhknRMCgaFiXTYmQ2i+WsbOiWTIsLEzO8fWEIn6oS1FX8muq5PBUFVZGr8r4CUWXDlIRA4MmR6xUNgNVAazLGXW0NjMxmefP84Kp47BbDchysskG+DDP5IkMAwxMABHWVDQ1pdnc089imTlqTMXwVutsfdriuy/xsnmOvnmbo7Ciu61LXWkO6OUX/iWHmprLUNCbAhWOvn2HjnnWkGuIU5ovs/f5hQvEgB54/RmYqSygWZN2OdqZHZshMzZPPFtF8Gh2bm9n5/q2XnBjKhTL7njmMWTaJpaPUNCU9psrD/TiOQzwdo3VDIx1bWznw3FHmJrM0ratn/e4ukvUxXvvaPiRFItWUoLGzjobOWvY/e4TM5Dxd29tYt6OdeG2MN791AEmWaOispaYpiRCCQy8eJxD207a5mVRjAlmWeP0b+8B16bqrjaauelRd5dWvvo2qqzR119O2qYlA2M+Jt84SrYlgmW3IssX4wBQvfPFVQokguz+wDX/Iz/C5MQ69eJxgNEDn9lYaO+vITs9z6u1zRGvCOLaD6biMnB9n7/cPIysS3Ts7aO5poP/kMPueOUwoFuSBj92NHtB/OIwBgUcs1FgT5VT/BF9/5ShzuQJ1iQiFkkFjTZSOhiRdTTUIIehpSfP+u7v5/tun+f7bp7kwOsM9G1sI+nUM02JyLs/JvnHWNdfw+Q/dTTR08waB7Tj0jc7w7//iWXpaawj7fUxnCzzz9ilsx+XxXeu4e/3F8sHHd67j2IVRXjl8gT/79lscvTDKpvY65vMljl4Y5ezQJAFN5R9+/AFiN9m/7uYazg5O8vKh8/zuX7/AB/asx7JsekdnuDDskbwsN+iVDYvR6Sy5okG+ZDA2kwVgJlvgdP8EAoHf51VdNNXEkOW1gXM1MZnN89LJCxwZGCVfNrzJ3jQpGha5UplC5TfDdm6oXGk2X+Qr+45fRv27AFmSCOoqIZ+OT1Xwqwp+TSWoawR9GolggJ1tjbx/y7qbvdQq3r95HY3xKAh4+eSVk39XE/myyb7eYQ72j/Klvcf41Q/ez/s2r8OvKmtGLtB7dIBwIsjGe9ahqDKlfBlVU9hwzzp6jw0wPTwLAprW1XHucB9jveMEIn40n8orX3kbWZFo3diEWbb49h8/T9ddbbSsb6JlQyO52RzP/OXLlxkDpmlx6OUTJGpjbHloA7UtKfKZAv0nhth4Xw8dW5o5f7ifQy8dZ/tjm3j00/dx4fAAE4NTnHzzDHue2sFo3wQ/+W9+lLrWmmru1uOfeYAz+y8wNTjDweePUdeepq6thgc+vht/0Atd5LMFmrrr+cDnHiYUCzJwaph9zxwhURultrWGgy8cY3pkjs0P9DDWN8mv/tHPola4boySwe4nt/Pmtw5imxaGgEK2iFE2+dDPPEY8HUNWJFxcWjc28tTPPoZc8UTpAY27HtvM3u8fxnVc+k8NMTk0Q9umJh78+B4AZsbmKGSLKKrCh37mMZJ18RvKabsjjQHwXPy/+ulHeG7vGd483sd33jhJybCIBn28b1c3tfHwkvbv372eje117Ds1yLNvn+FrrxzDtGw0RSYW9tNen/CIfsKBVan7DPk0/vVPvZ/vvHmSfSeHmJzL4QIdDQk+9+RutnTULzE6hICf/9i93LOplVcOX+CtE/28ebwPn6bSmIry1H2b+Mh9G+lsSt306mT3xlZCAZ1ULMQPDp3nv/7tywR1jdb6ODvXN/OR+zfxq//9a0u1DFyXUwPj/PYXnmViNrek0uBk3zhnhyaRKjoPsbCfv/hXn7lpo2UNSzGbL/LamT5eOdNXufcXn4GLl316K8M7tuOQLZaZL5W95FcqY07FMxAL+FBlaVWNAUWS6KiJ81MP7GRmvsCp0clVI6G5FtiOw1Q2x2999XlGZ7M8saV7jewIWL+7i2/84bNMDk7TvqWZR37svmXj2Rv2dHH89dOcnchQ157mgad38+xf/oBoOoLu10jWx/nYP/wAJ944QzQVIhj1U8qVliXiwXUp5koEowHUS9gEYzVh/CEfsiKTyxR4+e/eZHZsjmR9AkVVME0LIQTJ+lh1xZydmufgi8eYGc+QbkoiKTLlYplSoUw4HkJaLixWGXot06ZcKJNoiKP5VB78xB5C0QAukGyIXdFgDEWDdO/qIJaO8PX/8Qyb7+umc3sbcHUtGKNk4Lou4fjFqrFYOsrOJ7ZS35Hma//v99nx2Ca6d3USil2fGN4daQws3LCGVJSn7tvIfVvaKJZNHMdFUSRS0eBlDH2qItOQjPLwdp2NbXUUSgaO6yIJgarIBH0a6XgYv64CLls7G/hv/+TjdDQkCeoqsiTR05Lmv/3y0zTXxghWKg56WtP8zi98mFQ0SGBRFYIQgkjIx6ce385T926olqoEfRpNNVGCfr2anLRwPX5dZVN7HfWpCE/c3YNhWkiSwK+rhAM+auMhzyW76IVRFZmnH9rCfVvaaaqJEglcPQlHkSU6GpLEHvHzyF2dnva7JBHQVaIhP8logN/4e48TCugkI94LJQlBR0OKf/mTT1zRXSuEx6cQXGMtXHW4uJ5r+zZOhsv2oxKaWPIDXpKrfaWkhGs+vheGOtA3zKH+Ec6MTTEyN0/f1Ow7cu0uXgjlK/tPIMsSrttJZ23yqvu9l2FZNvlMgcxUltnxLENnR0k1JDj++mkuHB2gOF/k9N7zdO/sIJIIk5vNY5Ytoqkw2x/bxIHnjzE3kSFRHyfd7N1LsVCktcJ8KCsyPbs6eOYvf8D4wCQ1TcnqhLeQ0LcwNLqOw8TgNOWi4XksI97CRCxKCHUBx3aYGJjCKls4tkM0FaF9czPf/8LLzE7MUdOcpK4tTaohTuO6Ol78mzdo29REMOKn++5O9n7vEIVMgdq2GnS/hqopS5JOC/NFJgan2f/sEXqPDnB63wXq2tNeQuLe85TyJUzDxnVcfAGNVFOC7/75S2y4u5NQPMTcRIajr5zy9t17nlhthHymyMHnjzE1MkN9Ry3RVJi5iSwXjvRTypewTOeGSsLvSGNgAT5NobEmSuMl4j0rQdcU0lqIdPzqjFaJSIB7NrUu+S0a8rPnkt9iIT+7Frn7qxDeBNpcG7+mcj+oGBBBH5Ggj7a6a9oFWZJorUvQWnd9q5WQXyfk12m5RMNhAVs6G5bt246epmXbr2ENqwHDspjI5nnj3AAH+0Y4NTrJ8EyGXNlYcZ+FygBPwe7Kx3dct5pg6VzniDk0k+GNswPEAn5aU/FbVuJ1J+DcwT5qW1KkW1JEk2FO7z1P+ukkyfq4l4xt2vhDPhDQua2VRH0M3a+haApN3fVkpuaxTZtgLEA8HUVIEvHaGJpPI5oMs+3hjZedU5Ilkg0J2je3IKsSoViQUDxI111tRGu8kr2a5iTbHt5IfUcaISR8QR1ZkfGHfPiCOhvv6a66/nW/RnNPA0KSCIT9SLJEJBEi1ZSgY2sLgWiAQCSA5lNRVIVN9/UwN5ElEPIRTUWI1kSYm8gQigaJ10WJJEMoqsLGe3sQlXdDkiV8AZ369jS6XyNW43lEVFUm1RAnGA3Q0FmLP+wHXDbsWcfsRAY9oKPqyqLtPoKxALGaKAJBbi5PMBogEPKh+TSCET+1LSliNRFqW1No+vVP7Xe0MbCGNazhvYOSaTGRzfHGuQH+5wtvMzmfu8zTsFDD7VMVdNVj/1RlT75WkaVq+GIl2AvS3Y6NadtYtidXbtp2NenStFeuTjo6NE4yHOTh9e3UvAM0ue8WXDjcT9dd7TR0prFMm2/90bOE4kG2PrThsrY9d3cu+TscD3HPUztwbG8Fu5A/sABfUOfhH73nsuMIIdD9Gg9/8h7siofo0oVWc3cDzd0NuK5L++ZmZFle4g2456kd1X/7Qz667mqnfUtLNUa/gPd99kFsy0ZIUnWlv+nebmzTRlKkagjh0U/dh207XsJ0pd3ic/gCOnVtNdS11Szpp+u41LZ5OW2SfDHZevP9PVimhVLJTUnWx2nuWbowi6bCNK9vwLYcZMXb12lM0LKh8bLjXQ/WjIE13PGQKhntV9p2O1K+rpTLIaSb74Pgytf6TkOWBDdalu+6LqdHJ3nm6Fn+4pX9K+Y+RAM+drY1cldrA5uballXlyLs16/7nnilxA5TuQLjmRwDU3OcGp3k9XP9nB2bWtHNmi8bnBuf4oWTF/jUnq3LN7oCrvT8vPt3m95VSSC5K/RDXD12vesD2zj22imOvHKCQNjP4z/xIMp1VjhJN+FZuZq3VQiBcg0qhVdqtxxnwaXXKIRYQjl/rRCSQJEuP68QAvUaqnG8fl88ryRLN3U/AYT7blFJeI/g2b2n+d6bpzh0bpg//Y1P05yOXXOYYA3Xj4X4sr1CyZwQIAkJ9Qat5WvtgwtXzKWQJIEspJtyLS9wB6xGXP5WQAiBLIkqm9r1YF/vEP/7jcM8e+zsstenyBIf2NLNLzy+h9pICE1RqmWOC+e+Hiwe9haXZxqWzV+/foi/eGU/c4XSsvsmgn52tjfyXz/7kes+59VyPmTp1r+r4L2rK71FUuU5Kld4jo5T0TNxXUAgyaIat1/DnYk1z8Aa7mgIISqsYbeOmeta+iBg1errV4IkCbRlVhN3MhaYBL/01lH29w4vawjEg352tTfxi++7h/pYGFWWb9o7snjSEpWsNRkv9+BjOzZyfGicg/0jTC9DhZwtlTk7Nk3ZstDkaycjEsIzlG7EWFotLPT1ZkjLwOOUYG2N857CdY8srlPEtgcwSt+/ZItASCF8gZ+qvnCua+A4U5QLf3fZcYTQ8AU/jxBXoXK8w9Ben+R9u7rZ1F5HNOR711rKjuPiOA5lwyLg1961/VzDexsucGx4nNOjk8zkL594FUmiMR7hQ9t6aE1euWRrNSCEoDYaYltrPcOzmWWNAcv2mBrzJQM16F+j7v4hg+O6DAzPcOTkMMl4kD3b224oVPBuw/UbA24O2zxKcf4/IyvdlV8FICPJSXyBn1rUtoxjDVCc/0/IShfe6s1T2RFSAF/g78F7zBjoakrR1ZR6p7txVbiui2HYzGWL+P1XFvBYwxquhsxsASEJNE3GtwL3+nJwXZc3zg4wky8u6xUI+3U6a5M80NO2ir29OjbU1/D2+UFgctntC5wL0YDv+oRS1nDHw3VcLvRP8XffOcD6zjp2bmn5YTUGSrhuDoSfcOLPEWLB3eQZBEsbG7huBhCE4v9/JCnJxSJSCcQPDynNQqxugUzDUx703MsXf1u4NxfJZBbXzi5ud9lvlQQ1F6pyygvuz+WOZ9kOhZLBXLZAfTrq0dkuYq1ZaLcgZUzlUJfyHKxhDa7r8oPnT+Dzq7S019CzseHqO1X2c1yX18/2r1g62FmbZFtzPYFbHIK5FHXRMCF9ZaPGdWGuUKQxEXkHA1RruBksvH+LqbWvaT8W+FSk95SQ1Q0YAwVct4gk4khyI1cOHJm4ThYh4khSA0L64WbuyuZL/N03D1A2LTpbUrQ2JQn4Nb79wjFc1+W+nR0IIRibzDA5nWNiep5dW1vp7kjjuvDt549hmBYbOuvoaK3BMC1efOMMlmXz0J51pJNhpufyvHWwl0LRoKejFkkS5IsGM7N5pmZzPLi7i862GkbGMhw6PkhdOsL6zjoOnRhkeHSOmbkCc9kCD+7porkhwcmzoxw9NUzZtGltTPCBhzYSDl2futga3vt44XtH6eypI5a4dtYz14WiYXJufIryCmJjLYkoXe8Awc9CqeJKWNA0WcOdi6nZPC+/eYaaRIi7t7URuEaPliwJHr6nmwd3d3mlfO8Rg+AGPQMlhBTjavSJrmviuPMIKQ5idTJkXbeIbV0gN/dPCUT+DYq6EUm6SDpkW/2U8n+G65bwh34eWekAwDKPYpbfxiy/huuMI6QaVH03inYPqraNBa+GY09iGfsxjTewrbO4TgYhxZDVjajaPWi+xxf1pYxVfotC7r8Qjv8R5eJXsIyDOPYECA3N/yS+wKcQws9spsCFgSlSyRB7tnsv3tDoHIdPDPHIPd3Eo35e2XuOQtEgGvbT3pzk4Xu7een1M2TnS9QkQ9TVRNhzVxs+XaV3YJqBkRniET+pRIiX3zzD7u1tCCGYyxb4iR/ZjU9XOXh8EIDujjSP1ffw7eePkk6FqUtH2GDUc653AhfIzJdQVZmN3fXUpSN869kj5PMGIOhqS6PIErbrvmde/DWsDhzHYXY6x9joHI0tyeviQ7Ych5lcEcteeaegrhF+B9gsM8USpSuooQo8voPbUwi4hluBuWyBF18/zSP3dFd5C64FC15TSXpv+YRuIDW5jOvM4zozFHP/FVwHIYWQ5HoUdSuS3LZo0jdxnRyum6WY+0MEGkJoCLkGVbu70vZ6b6iMEBFcZw7L2I8kJZYYA5ZxGNs6j5DiXjvXxbEHMIrPYNsDSEojQnSBa2AZR3GdPACqdpEowqUMro2sbgBkXCeLbfXiOhlkpR1Jbveu0XVw3CyWcYRy4Uu4bqGyvRmwkUSYhbCIYVjk8mUiIR/JRAhVkbGHZ8jmSqQSQRKxIIWiSS5fJhr2E48FqauJUDYsMvNFQkGdcMhHKhFCCEHZNMkXygSDOuGgj+aGBMGATrFkomsqNUlPm2HBnZWoHC9XKGPbLpqqEPBrmJYNrottO2iqQjIepDYVIZsrEQrqTM3mGZvMkk6GWdeeRr2KEpZtOwwPTHPh3ARzM3kK+TLlsoWiSCRrwqzrqae2IUokupQu2rEd8vkyZ0+N0nd+gkK+jOO4yLKE369R1xhn/aYGwlH/El5yy7IZ6p9maGCasZE5CvkyALpPJRzxk66LsmFzI4Ggfpkc6tmTo/Sen2B2Jke5aCLJEjV1ETZsaqKmNkIguDSfxbYdsnMFLpwbp+/8JIV8Gdd1URQZf0Cjtj7Gpq1NBMO+agzRrdzb3nMTjAzOMDGepVjw+ujza0RjAdJ1Ubo31BMI6uRzZQ7v62NyIsO9D6/nrVfOkM0UqG2I0dpeQ11DnJefO05mtkAqHaatM03PxoYl15aZzTPQN835M6Nk5gqepntQp6Y2wsYtTaQWScPm82VOHhnkzMlRHn9yK/lcib7zE4yNzGKaNsGQj4amBM2tSZpaU9V3arB/isG+aUaGZhgbnmU+W+TMyRFKRYMjB/qrfenorqWzu46WtsvzaBzXpWCYV7QfFOmi+ujtxOR8gcIVWA8lIYgH/WvpAncoSmWT2bkCA8OzmJa9JtnODRgDQviR5ASSXINpvA1uGZCQ5DSuM43u/yQuQYSQAA1JiiPL9VjGQcCqHCOE68yh+38MpAhCXHs3hNCQ5DSKuhXbPIyjbsBV1lUHN8vYC0jIcjuSnPJqe419WOZRJCmO7vsQktyK44xRyv0plnkEhHbRGBA+JLkBRbsLWd2AEHEcu5dS4W+wzKPY5mkkuf2SXtmYxhvo/o8hK+sRUgjXLeKFULxrk2UJXVOYyRQYn8wS8Gs4jovfpzI1m6NsWKiKhK6pCCHw6WrVTakoEooiM58rMTaRxefz2mi6QrlsUiyZdLSkiIZ9mKaN37c0virLEj5drfCqg2na5PJlZuby5AtlsrmSZwz4VHTNc4+6QDwWYHB0lt7BKSQhuHtb6/LiHYtgmTaH9/ez/+0LZGbzFAsGhmEhgEBQZ+KeTnbsbmfDluYlhsXcXIHes+O89OxxBvumKJUMbNtFAJquUNcYp64hSiCooy66vKH+afa+cY7Tx0eYGMtQKhk4jouqyoRCPpraUrR3pfEH9OrAbZo246NzvPLCSc6eHiUzk/f6KAmisQCZmQJbd7TS1pnGH7i4Kp2ZmufsyVFe/8FpBvumq+cSQqDrCrUNMVraU94+iyawgd4p3nrlDOfPjjM1OU+52keFSNRPx7paWttTBII6xXyZfW+e4/CBfoIhH2++coaBvikamuJs2NLEpm0tvP7SKUZH5kgkQ+zY00FjU4Jw1H+xj6fG2PfGOU6fHCE/X0YI0HwKqZoI+VyZBx5dTzDkQ5YlSgVv8v72V/aTSkfIzOY5fmSQ8ZE5yoaFJEm0dqTYuqONUNhPLBEABNOTOc6dHuXU8WGGB2awTJvJ8UzVmFiAYzskk6FljQHXdbFsmyu5Exaog283+iZnV+QZkIRAV2ViAd+7lgBqAa7rMjqRZT5fIh4JkIgFmM0WmZ7JYZi2926oMtGwn9qayBKj0nFcZjN5ZuYKlA0Lx/E8g36fSjoVxu/TLgulOI5LoWSQnS+SnS9hmDYIz7WuKjIBv046GUJRLy8Pnc0UyOZKFAplDMtGEgK/TyMW9RMO6Oj65XkjrutSNiwmp3MUimUM065qzmiqQiTkIxL2EQzouK7L5HSObK7IzFyB42dGyOZKjIxnOHZquBomEJIgnQyTToarHDG24zA1k2N8ch6nwmkiSYJQ0EdrU6IqCb9c/wzTZmYuz1y2iGnaCEBRZIIBjdpUGE27qITpOC7jU1lmMwVqUxF0XWFmNk+2It6kKBIBn0Yk7CdeWVCt1it43caArKxHkupRtD1Ich2uk8My3sYsvUwh+3vIyiYUbTugI8n1aP4fQVZ7EFItuCVs6wxm6QUK2d9FVnpQtB0IcW3aAot6geb/MIXsf0KxelH1e3DdAGBgGm+hqJtRtIvsYOXi15CkGlT9ARR1GwCSFEPV78cov4BZfhVCvwR4hoqi7gB1G+CA6yJJ21HUI7j2GLbVi+qlkCzpj6JuRfV9AEmKLNvjWDRAa3OSF18/w/hkluaGBLWpMN0dtXzvpRP4NIV17Wlsx8WqfAjgGQL16SjpZIhX3jrL2GSW9pYUDbVRWiXBi6+foViykBVBOOiJHy3ObJUlCXkRiYmqekbF5EyOQ8cHGR7PcPLcGGXDIhS4uHpWFZlSxdAoly3yRYMzfZPs2KSjyCu7bQ3D4vUfnGI+U6KtK01dQ4xAUGd2ap7nvnuUr//t28zN5mloSpBKX7xXxw4N8K0v7+fE0UE+8JHt1DfGEQJy2RLDgzOcODbkUZdeMvh89W/e5tDeXhRVYte9XTQ2JSgWDcZH5xgenOHC2TF0Xb0oYOK6ZGbzfOmv3uD57xxhw5Ymtu5so6Y2QrFg8NKzx/nrP3+F0ZFZPvSxHXQvSobb+8Z5nv32YS6cHeeJp7bR0JzAqXgLRoZmOHVsCHkZJrC//avXObyvj3giyLZdbdQ3xCkWyowMzzLUP01/7yTaJQPd5HiWL3/xTR58bAOyInH+zBjnTo9x7NAg3RvqSaUjHD3Yz743zrN+UyO77unEcRxee/k0P3j+BGdPjrJjTwcPPtaIZTn0X5jkxJFB9r91gWQqxKZtLYTCvso9gVy+zN/+r9fw+zUi0QAPvW8jsiLz1qtnOLy/n8G+Kfx+jSee8r6rdG2EbTvaaOtIMzw0w//+81fo6qljy/YWOrsvCmvU1EZJ1S7/TQghKl6ehdTXy5ErG2SK5RXft9WGx0zo8ub5AYZnM8u20VWFdCRE4AoJhu8WuC5845nDvH2oj8fu7+EDj2zihVdP8Z0XjjE5k0MISKfCPLC7i5/+sfuq+zmOS6ls8tLrZ3nh9VMMj2colUx0TaGzrYZPfngH6ztriYYDS0KHpbLJ6fNjvLr3PPuODDA9kwMBfp9KMhaip7OWn3h6N8l4EGmR98yyHN462MvbB/s4dX6Mqdk8qiLT2ZrioXvWsWNzC21NiSWLkYWJtn94hq989yCnL4wzPZOnbFhomkJNPMg9Ozq4b1cnW9Y34LguL7x+mjcPXGBgeIbZbBHbdvjms0f49vPHqkO6psr82Id38qmP7CJU8Q4ahs0Lr57ir766l2LJwHZcFFli55YWfuuffhj/MqGshf6NTWT47ovHeWXvOWZm8wghiEUCrO+q4yeevpum+jia6vFVmJbNt547yvdePsHf+8QemupifO+l4+w/OkCxZBIJ+VjXlua+XZ28/+GNKLJXnbcauIEwgYKQkihSJalH8gwEWenBzgxhFL/iTf5CB6TK5Hp3dW9Z6UbRdmEahzGK30aSG5e4+a8NMpr+QUrSn2BbpzDLe1H1+zHLL+M4k8jqJhR1V7W1bV3AtF+hXPwSS2+cC7jISidg4+UNlLCM/RilF7CMt3DsEVxsXLeEJCVQWGZgEjKyun1RZcUyd02WqK+J8Bv/8APYToXLutKVDevqwXWXVAks4HOLOLp//Rcv33f7xmbvN1mqXtnmRVzW9+7suNhNIfjHn3+02u6+XR3gLm9Z/urPvo+/+eY+muvjPPXYZlRV4b/8yfOs76hd9sVfQCjk41/9h0/i86tLVcRcWLehgb/+0x8wMjTLyWPDPPjYxUliYizD0MAUO3Z38Au/8gFkpfLRuwsVEs6ydJsnjw4Rjvi5/5H1fOpz9y85HxWZZUm+eIEzUzkO7+/je18/yMc+eTdPf3oPtQ2xauXEUx/fyb/5tf/D/jfPY1sO/3TjR6v7Dg9MMz05z933dvGL//SDF+/bVfp4/NAAtfVRHn9yK089vfOqfQQIhnS272rjx3/qAYYGZ/hff/wyh/b2EosH+NlfeoJSyeC//c63OX9mjN5z4+zc08GZEyO8/MxxMpkCn/zsffz45x+onqtcMjl/Zoz/65e/yJ//wYv8s3/zMULhpWpYAvjQj+zg8Q9trXptPvCR7XzhD17k9R+c5uXnjleNgfqmOPVNcSzT5tTxYRRFpqY2yvotzdx971Iu+pWgyhJ1kdAVh7ILEzOcHB5nR9u1VSjcLGzH5WsHjnN0cGxFz0AyFGBne+Nt6c9qIZsr8dbBPianc7x1qJcHd3cRCfuZyxQolkyC/qUhsf7hGf72W/v51nNH2LG5hcfvX08k7GM+V+LF18/wr/7TN/jcJ/8/9v47yrIsO+8Df9ff511479L7qsws21XVrtqg0Q00gG6gSQokSIACR+CipJEomiHFmSVpJK0RpRG5RM8BSIIE0A2g0V1oX1VdVV0+vfeR4f3z5tozf9wXLyIyTLrIqsyq+HpVZ+Z9595zrj377P3tbz/FC09to6djiRj+R989xmvvXGVqrsBzTwzR25nBsl3msyXGJnO8/u5VfuNrT63wCriuzz/93Z/yg9fO05SOsmtbO/3dTVRrNq++dYV/9fs/Y9e2a/zVX3uGvduXnoOFXIVT58f4n/7p93E9n8f2dvP43h5SiTA1y+XUhfGgMmF9JS9JEnu2t9PRmmBuocTl6zN875WzfPrZXTy2r6fhUZVlid7OFOayYj+GrvKFT+3lwO5uShWLt49f5zs/OrPhNfd8wU9+dpFvvXSCK8MzPHtkkE89vQPfF0xO53n7+HXePnGdf/i3fo5dQ+3EY0vE7FKpxu//8bvEoiayLPGlT+9DURROnR/l7OUJTl0YAwle/MSu24Zu7xT3ECZY+9UNwgft+P4sCG/jtqgoajdCLADrx+XWHQOApKEZn8RzL+Dab6HpR7GrP0BRt6OofUhyjMXJXogamvECmvEMqrZvzbEvEghr5d/Dtd9DCJdQ9L9AVvuRJB2r8sc49pvrj0ky2MhCW542uOhakyQpmAikpRl5Y5Llyn3X37ZWvwGWv4SNzMM1xwuH9nZzc2yBl14+g66pvPjcLiLh26yGJAiF9TVSdQR9A4HbvVaxyS2UV+yWTEXINMU4f2aUH//5KR5/cohkOhI86EI0qoDdej49fU1cvjjBsXeu0daR5IlntxMKa8hycG2lW/aZnsxx+kQQ037uM7tJpiPLVjaCRDJM70AzuYUyE6MLFAtVorFAPCrTHCMaMzl3eoQfv3SKI89sIxav8wM2GGNvfzM3rs3w3ptXSaYiPP7E4IZjBNB1lfbONJIskWmKEgrrmGGd7r4MkgSmqWOGNGRFplp1ADj+7g0W5kt09qR56rntK1ZsuqGRbooxsK2VKxcnyc6XsWrOij6PPrOdwe1taNqSql4sZtLUGicaM5mbKSzd5sXx3jLutc5lPciSRNjQ6EzFmcgVcNYgcV2emiUVCfHC7kE6krEHmtZaqNY4eXOS/+vH75CrVNdt1xqP8tyOW0OFDzc8z2diOk9LJsb/++/8IolYCEWW8OqSwsu9idOzBU6eG+X1d67whU/u5de+cphkPPAAeJ7Pp5/dyd/7n7/NT9++jKbK9HxlyRi4MTqPZbscPdDHb/7as6iq0uDNuK6P5bjEY6HGs1mp2lwdnuUHr53n8L4ePv2Jnezb0YGmKvgCnn9yG7/7zbe5eG2aP/zOMfb+14Ex4Ps+F65O8u//5F08T/D3f+eLbOtvJhY1Uerv1c9/dj+GrmDWvW4SsH2gFc/zmZ4t1MMkEj2dKY4c6CEWWZqMNVVe4YWUJIhFTYb6dHwhmJzJ33ZB/v6pm7xzYphsocLXf/4wv/Jzj2HoalASu2bzuRd28/f/1z/jD77zPl/70uM8+djAiv0FsGdHO1/70uPB2CT45FPbeOWty/zBn73Pq29d5lNPb//wjIG1IRA4CD+PrPaDtFFcWQA+ws8iqf33NoRFV7bxNJ57Dde5iucN49rvoZkvIittdWJiXTdbbkKSNGQ5vqYxEGge1DkHzhmEqKBqjwchjEUPiGTUeQDrDuoOh37Lx7O+Ip2eLRAJG40HdyFbJpEIoWtL9bHvZLK/H5TKFrWaQyxqousKna1JQqZOqVxDlmXamuO3JXMJAYV8hQtnxpidKVAqVKlWHRzHpZivMTWRo6k5juuuZGoPbm/j+c/u4SffP80PvnOKc6dG6ehO09WbYWBbGx1dqTXP9ZMv7kXVFMZuzvHSnxzj7MkRBra30j/USntHklRmZWW5csliZqqAEII/+6P3iMSCj8dyXD4/QSFfJZGMUC5ZDXf6nv3dVCs2P3v1It/7sxOcOz1KZ0+G7t4MfYMttHem1rwmn/25A7zyw7PMzxb57jff59T7Nxjc0cbg9jZa25MkkuFV+yiKTDgSGF6qptS14mUi9bROWZaQZbmuUxFMolOTOapVm5Ebc3zzP7xF+BbDLQif5LEtl0rZwrZX1lJoaU8QS4RWXGdZkdF1BVWVsa31ay/cCyQpqNewu7OVhXIVx1vtdavYDlem5vjDd07znz37GMmwuW589l5RtR2uzcxzamSSn164wUyhtG5546ZYmIGW9IeS7ng/EEBzJsqu7W10t6eQ5fUzwcanc1wZnqFmuTx7eJD2lkTjuySEIBzS6WhLMD6VY3wqR7VmN7yFi271qdkCw6Pz7NrWhqEvGr5L17RBYK3YnLk0TqlU49Debrb3tZBZVmI+ZGp0t6e4dnOWa8OzVKo2IVMjm68yNpljerbAnu3t7N3RTjoVWfF9utXfLEkSRr3QkGlq6JoCUhA6DZn6hqmFwbMqoejBs3cnpNZL16cZm8gSNnVeeGo7zZlo47y9SMCB2NbXwvDYAuNTOcoVa4VR1tuZZve2wJOxGB4JmzqdbUliUZOJ6dym8mnueib2/SzCLyJJISQpjMBH+Dl89xq+P4+ufQGoPzh+Gd+fQ5LCSFIoiAqKEp57Bd+fRlM/iyTdeV7yrVC0HchKM65zGtd6E9+fQNUPIMuL5SKDWKSq78f3JvDcYXx/vi5+FHgMBC4SKhKh+pjzBIU3moP0SWHjeTfxvTGEWC1Ner/wfZ9yxWZkfIHOtmRj4p/PloMXSwvcyK7nU6s56JrSIJy4rke1vrpbfFl9IYICIghMQ2vEoWzbRdMUNHUpNuXULWPTUBFCUKna5OuKhDoKsaiJYai4bjQgTRnqhsaH63gUClXee/Mqp44NUylbuI6H6/l4rket6lCrr2JvDRF3dKdQ1W1UKzan3h/m0vkJbg7P0Xw5ztREjoOP99M70Iymr9SCP3C4D9fzOHPc4NqVaU6+f4PRm3OM3ZxncHsbQzva6B0IngdJkrAdt5FxcO3y9Cr3/CJa2hK0dSRXfMB6B1pQVAXX9Th97CbnT49y88YsbR0pJsay7H+sl/6hlhUcDYDHnhjAshzOnRrl5vVZTrx3g9GReSbGsgztaKN/qJXu3pWTy/JqaFLgZgkqnS3LpAhW4UvXslyq4boetarN5QsTa96rSNQgEm1GN9VVlfmiUWPNOuiLE4e/TjGo+4EkwZGBLk6OTFCsrTYGhIDpQokfnL5MX1OKPV2ttMQjRAz9njX+fV/geB6lms18qcLYQo6TI5McGx7n5M3JdfeTJYntbc3s72kjHV1twD3saG2Os2Og9baF02bnS0xM5QN+TanKsTMjq1KKfV9QrToUijWqNadhDOwYaGVuocTIxAI/fuMi+WKVlkyMZCJMIhZqGAuLqFo214ZnEUCxbHHp+jQ3JxZWtMkWKnieT7FsUa5YmIbKQq7C3EIJ1/PYta2NZCL8oWSdbITxyVxw/k0xtve3rPhNUWSiYYPBvmauj8wxny1TLNVIJZfmw872JN0dK3kSi4ZLJGywkFvpXb1f3LUx4DnncZ2zyEobitKDEA6ucxzXegfw0c3PIknB6sX3JrBrP0BWu+rpdj6eex239hoIG938ZH1ivjfIchJF3YnnBmx/pAiqtn9pNV+HHvoK1eI/xbF+hiRF0IynQfh4/mSdC5BAV16sH7MNz7uB65xDdUdAVAP9AOc00iZX5vB8n2rV4ebYPMMj80QjBqF6pkClZtcnqsCVV65YjIwtkIyHaG8NjIZS2WJ0PIvre/R3N+F6Qa0Btz7R93UH16FQrDI3XyIRC9HcFIRPCsUaC9kShqHR1Z7CdX1qloNluyj1CdLzfHK5CvliDdNQ6epYe+W7iFKpxoUzY/yf/8uf09Qc4+gz2xja0UZza4JIxCC7UOb/989fWXNfw9Do7mviL/615/jU5/by7s+ucur4MJfPTwR/f/wmf+O//jyt7YnG+ACiMZPnP7OHo08PMTme441XLvDmTy/xg++eJBo12f9YH7/zt7+IrtcnVqiHMOBLv3Q44Daso52QSISIJ5ZUMs2QxtCONga2tTDxhSxvvnqRk8eGOX3iJm+9don9h3r5L//+z5NIhhtjlCSJWDzEi186yNPP72RidIG3Xr/EG69c4DvffJ9MU5THnhjkr/3OZxpjbGCNYW3kB5LqWe89/c189ucOrChxeit6+poxTQ1nmXdA05R7qrB5P94pWZJ4YdcAf3rsHHPFCo63Orffdj1GF/L8j3/2Cl86tItP7Ohje1sTyXBoqexvPeZ1K49jUX0zqEoYrGxrjku2XOXixCyvX7rBm1dHyJWr63oDFhELGTy3s49PbO+75/P9MBEJ6aSTt198lco18sUqlu3yv/+rlzdsK0kEYapk8O8XntpONGryg1fP8eM3LvLyzy6xY7CVA7u7OLini11DbUtue0nCcTzmFkoA/O4fvbXhsxSPmVSqNulkhGKpSqlsocgBwfphVEXNF6t4vk8sYqztypeguZ6xUKpY5IsrjYFYxCS+hsCbXPcSuq6/qSmRd++jl0IIP0+l/PsIfxwIo6jdqPpR4vG/szLtTtIAjWrxf8f3JgAdWWlG1fYRS/9+vbbB/U2wmn4Y35+lWnwNI/wrSHJ8xTGD+tDPIMXjONarWLXvUi39Y0BDlttQjSfRQ19otDdjv4Vd/S529c8oVP8IScpghH8Z3fhMIMO8iRgZW+DClUmiYYOrwzMM9TcjSRI1y+XEmVE6WhM0Z2JYtkupVMOyXX7vj97mt3/9eWbni4xP5ZAlibeO3eA3/+KznLkwzvRsgUwqyomzI/zGrz3D1RszFEs1dF3jx69f5Otffpzp2SJTs3lKZYuezjRd7YFxMTNb5NrNWZrSEVqb41y8MsX4VBbb9ujuSN3WGJgcy/LD75zEtlz+1t/9EkM72olEl1YC506P3pGLt6M7zS/86lG+/LUjTI5n+fYfvMu3/+g99h/q5annttO9RpqaGdLpH2qhf6iFb/zGJ/jxn5/mpz86x9tvXObg4T6e/+weNE0hEjFobolzAYm9B7ro7mvGMDd+DVaHdiQ6u9P8yl96ml/6C09x7fI03/vTY7z0J8d5/ScXeOq57bS0rSbFRqIG23a1s21XO9/4jef47rfe56c/Ps/br19i555OPvm5vbe9NhuhtT2BGdJJpSMcPNxHZ/eDV/zcjE9wSzzCV4/s5TvHL3BseHzddhXb4Q/fOc0fvnOamGnQk0myo72J7kyCeMgkpGvoqoImK9QcF9t1sVyXqu0yVywzkSswnS8xlS8yVyxzNx5WWYK/+eLTfGJHH02xe/dmfpiQZXnDssSLWDSaDF3lt//ScxsaiB2tCRLxpQkrGjF44cltPHN4gPHpPD/86XnefP86/+nb7/GH3z3Gkf29/IO/9XOo6tIxF6e0X/vKETKp6LqpcrqukkqEV76PUt1zdduz+vCw0WO25HlcTR7XNCUIZXxAuGtjQNX2oqhDmJFfR+ARrEcUkAwkaTHvMTgrWenCiHwDI/wLy9rKIGlIUowg2+A+b6OkB+Q9ScUwf64evlj98Va1nShqP2b4VxGLegeNcS89zIrShxn5q5jhb9RDCApIiy7BxZTCRfKUiW58mmTLq8hyGri7okvlikWxaPHJZ3Yyu1AiZOrIsoxhqKSTSyvShVyZ4ZF5ZueLlCs2vhAkE2Hms2XOX5qgrztDyAyKDaUSEY4e6qdQrGJZDvMLJcJhg0N7u3Ecl5m5Im31lL5zCxNcvj7Nvl2dKIpMJKwTXzZ5D/Q2UbMcrg3PcuXGDPv3dG14PotEIYBkOoq+LKwghODNVy+RXSiRSq+M43uej6iz6pdb0LIcEAuPPD3En33zPfK5CjXLaRxvsfKiosgrXGmKIrNjdydjN+e5dH6ChflS46Vr60xx6OgAP/3xef7sW+/zq7/+LB1d6VUvouN4+L7fyG32PL+uKcCKuJ4sQ3NrnMeeGOSlPzlOLltuxOIXx+h5PqqqNFytAWFLZu+BHq5fmWZ6Mkd2ocRdyfetgSNPD3Hu9BgjN+b44XdP8ld+O1DLXH5uQgismoOuq2tmPtwtJFkiXucZlApVSsW1Gfjr7l8f3BcO7MDzfGzX48zY1G33K9UsLk/Ncn12AbUexlgsJS0hIZZ5BIQAX/h4vsDzryK7egAAqudJREFUgz/v1BCQJYmoqfM3X3yGF/duIx5+eCuRbhYSMZN0MsLEdJ69OzvobE2uaxAoykpRqOWpyd1tSb7xC0f4xc8f5OS5UX769hXeOTnM6+9e5cjBXmIRE11TaW9OcJIxtg+0sGuojWhkbblzSaKuwwKJeIh4zMRzfUbHs7f16qw6FpuXn78emjJRbozOkctXsCxnhZ4ALGpA5HE9j0TUbOgGfFi4h2yCQEUQ1s4dXtlWrU/6sXsY2p3B90bx3TFkuQNF280iX2H1WBbHvfFYJEkjSBEM1Asd4XCpeImKW8FUTLpCXdj+FJ7wMBUTH5+8XaYrlOJi8SQVr0JUjdIZ6mTamqboFDEUAxkZRVLYm1ha/UUjJom4yTvHrjM+mWOor5lqzaZQrDE+meP85UkO7dMoFKtk82Vsx21MapIUMHsLpRrpVLQxmSqKhGmqaJqCEJBJRylXbE6cHWFyOs/enZ1YjlvPlfVZyJYRQKFYY3quwOh4FsPQSMTD5Iu1RtihUKqtUle4FZGoSc9AM+++eZUfffcUu/Z1Ek+EqFYdhq/NMHJjFttyV72EtarN+29f48yJEXr6mkg3RTFMDdfxmJ8tcurYTVQ1EL9ZTrYrl2r8x3/7BpGoQVNLnFg8hBnSKJcsLp4d5/yZsYZrf3EiTqYi7NjdwZGnhzj1/jCaqtA/1EKmOYamqZSKVaYmcqiaQmd3hqef3wFAtWLx+ssXuXl9hs7uNOlMDCOkYVsOUxM5zp4cRVVl+odaGiQ/IQSlYo3f+xev0twSJ90UJRoLYZgq5WKNMydHuXZ5ikjEqOfm3/vXSZIk+gdbOHikj5Pv3eDt1y+jqgp9gy0YpobnepSKNSbHc2Saojz9wk7St5Ar77XfdFOMdCbK6Mg877xxORBjiYdwLJd0c5Smlvht+4roGk9v68XQVNT3ZE7cnNiwvQAcz18zA2GzkAiZDLVl+PTuQZ7b2Uc8bKJuMnnxYURXe4qBniZOnhvl3RPDfPULh4LFxi2PZ0AClhrkwEbhH0lCkQOhtJiqEAkbbOtrYWauyOvvXmV2vojrBvctEtbZv7OT7796jtMXxmltitOcia/qy/d9XG9JDr05E6OtOYGmKbx/+ia/8PmDtDTF7nglrWkKhq4FhOdC7YHwYbb1tXBzbL6ezjnMs0eGGufluh7ZfIWzFydIxsO0NMWIRz/cwn2blE3wwSGYDD3ADgSPnLN43iiqfhBJzmy61T5dm2aqNoXlWRiKge3bdIY6Ga+OB+xU2UCXdQSCml8ja2cpuSUEgtHKKLqsI7kSru8iS/IKYyAZD9HTmWZ8Kkdbc5xYJEhT01SF9tYEuq4Eyn0hneZMFM8LVpqGrjIzV8SyXXq7MiiKRLlikUyGA5EhWaajNUEopNPZniSbq7CQrdCUjpJMhCiVAxnjzrYkkbCBLAUvczRs0NwUQ18kqUlBnK6jPUmibN12qoonw+zZ382xoWucOXGThbki0XhgveeyZXr6m5mZLqwiqfm+YG66wNmTI4yPzBONBxOm7wkqZYuZqTyHjg4wsK2VeLxO9BTge4IrFyfxXJ9ozCQU0TFNDdt2mZ0qAIJDRwLi4aLnwDA12jqSvPDiHr73pye4Vl+ZxxJhVFXGqjnkcxXaOpIkU5EVY5waz3L6+E1u3pglFgsFk6znUyxUWZgr8fgTA4ExUM8CQAQfscvnJxi+NkMkahIK6Rimim25TE3kMEM6u/Z20tPfdN8rlUQqwmNHBhC+4OT7wxx75zpjN+fRdAXfDzwChVyVA4f78Nz1dffvBpIUKEs+drSfs6dGGRmew3V9wlEDBOw50B14um5jDEiSRFc6gaEpVCwbSYIbs1mKNQv3AU74ayFi6HSl4wy1ZjjY28Gndg3Smoh+5D0Ci2hrjrN9oIW2lgRvHb9OT2eatuY4pqkh10m4+WINRZbIpKJ018OHlu1y/soktuORiIUImRqqquA6HuPTeXKFKoosN9L/IPi27Rxqo6cjzdlLEyRiISzbJR4LFDJt26NStXBdH9PU2L2tHYBo2KC7I8X2gVZOnR/j9XevsL2/lWQiFEi913lWsizRlI7Sklm5CDQNjVjUxDRUro/McXV4lqZ0FEWW8YUgHjVJxExkWW54+Ko1B8/z8XyfUsWqiwq5LOQrRGy3bgDJS2TKwVZujMwxt1DilTcvk4yHidXfi2LZ4urwDNNzBT5xdIiujhSGoWLZm5uxczd45IwB8BCiFGQGeMO49tsIUcQMfWPTexIILhYvIiPTbDTjCY8LhQscTh3m/YX3mbfnaTVb+UzrZwAYjAwSVaPMW/PcLN8k5+R4uulpFuwF5q15HH9lXncyESYRD7FjqG1Fqk9TOkp/71JcPJ2K0N/ThO8Lnn0iqJR1/tIkruvz6U/s5K1j18kVqmzrb2lUFDxyaJG7EaWrPY0QouHqC5kBkWiwr7kx8ScTYQ4f7OPwwb5GvyFToykdRdTlfW+HVDrC408OsjBb5LWfnOfi+XGEgEQyzKHD/Xz1G08AgkKuihFa8uCYpkbPQDO793Vx7fIUYyPz+L7ADGmk01H2Huzhi7/4GJ3d6UZdAqmuZ3DoSD+XL0wyM5WneK2K8H0SqQhdPRl27+/iyNNDZJpWfggiUZPPfGE/iWSEt1+/zPDVGcZGx/Bcj2QqSndfhp17O9m+q2PZGHUGt7eRXSgxfH2WseF5fN8nHDXJNMU48Hgvn/vyITq60o3rLMkS4bDB408McuXSJNOTOcolKxhjOkpffxP7H+/jwON9jTFKclBHIJEMo+ta41wj9W3msusWjhrEE+EVksn7H+ulvTNJ32ALr//kPFcvT+HUFdkSyQi9A008drS/4b0IxqiTzgRhnbUKUZmmXh/P6mdAqqcz/OKvPUkyHeHsyRFuXJ2mVnNIJMP09DfhbVCIaDlkWaI1EeNXnzrAYFuG3//ZKS5PzTJfqtS9AJtjwKzqt76S1RQZXVUYbMnwpUM7OTrYTU8m+dBLDm820skIu7e188KT2/jj75/k9775Nv09TTRnoqiqQqFQ5crwDO0tCZ481N8wBspVmz/8zjGmZgv0dmVobYoRjRiUyjbDo/MMj83TlI6wd0cHZn3CNA2Nvu4Mn31uFy/95Czfe+Ucpy6MMdjbjGlo5ItVxqdyCCHYMdDaMAZkWWKgp4kvfWYf127O8u++9Q67t7c30u5s22V8KkckrPPskaFVxkA4FEgBd3ekOHVhjFjUaMj/er5g/84O9u/qwtDlwBCwHC5dn6Zac6hZDsOj8/ieIJuvcPLcGLFIkBYej5nsHAzEvHYMtDIzV2RiJsfP3r9GpWbT2xl8i2fmSpw6P0YqEebzL+xhsKeZDxuSEHcZbPmQIfwirnOS4sJvAhqa8RS6+QWM8C9uel++8Hlp8iXiapy4FieiRggpIbpCXfxw+ofMWDM0G818uuXTvJ99n4JTIK4F4YXp2jRVv8pTmafI2llmrVls3+bnO35+U8ZWKFaZmMpz5cY0LU0xDu7taeTQbmELHwV4vuDc2DTv3xjj7WsjvHd9DHuTPBrL0ZqIsq2tif1dbTy9rZcDPe2spQb6KML3Bf/iP7zO9396nmePDPGXvvoErc13FrZ1XI+fvXeNP3/lLDdG5sgVqni+IJOMsK2/mWePDLF3Rwc99QnOsl3+9Psnef/MCMOj88xnywghiMdCdLYm2b29jS9+ai8DPU1relneOXGDN49d58zFccYmsziuTyoRpqs9xZ7t7Tx7eJA9O1YqUTqOx/DYPH/43WOcvzLJ3EIZy3YImzodbUk++dR2jh7sY9stqX0QSCePT+b4F//hdS7fmKFUtgJxsXSU3/jaU3zi6BCmqVGqWFy5PsPv/IM/2PB6ybJEf08Tv/u//fqK6z+XLfHnL5/lh6+dZ2auGITWkhH2bu/g13/lSdpbEw3ehWW7/N433+abLx3nG794lM8/v5vW5pUh+TePXeff/sGbTEzn+YP/668RDd8dV209PHrGgPABF+EXgw2SVucDrE06ub++BNPWNG/OvYkvfFJ6ilazFYCskyWshJGRmbfniakxrpevo8s6uqzjCY+KV3lgxsAiec7z/IAlrG5OiegtbOFhQVAnIOAF2J5Hvlrj/NgM4wt5pvIlJvNFJhbyFC0by3GwHA/b87BdD0UOVvu6omBoKlFTJxkOkYmGSUdCNMcjdKeTbG9roikWwdAUVEVBU+Q6L+CjYQwIEbi3bcdF01RChnrbYmPL93VcLyhS5IkGSU+WJGatd2iK7CBqtKAqSoPLZNkujuvheWIZv2kxkyEo1rae4JHjeDiuh+t6eP4SN0qRJVQ1ICqqt2gJLNaSsCwHt07yhYB9I8tBsSJVU1YVVFq+b7Vm10nMwXZZlggZWkOJMyio5VMs3Z4cqygyidhS7H+RS2HZHrbjNsYnS8E5mYa24noIEdSEqFkupqFh6Mqq+2U7XqNIWiK+mjB/r3jkjIEPEosEwllrFk946JJOSA0FD5HwUGUVBFT9KrqsU3AKKJKCUi/L7AmPtJ7G8R0s38IXPu2h9g/5rLZwO4xXzjJZvQAIDqW/ioxy3y/c5cJrzNVuoMgqTzT9hc0Z6McIix/ufLVGxXao2Q5Vx6VqO7ieV88QCAzkRRKbJAUaBLIU5GXrqoKhqsGfmkpY14iHTHR1dQW9zcB45QyXCz+l5M4DoEg6Q7Fn2B5/btP7Wg9CCFxRYbZ6HMvLo8kR0sYeyu4ktp/H8UpIkkLSGCKidjBRfgNXVNDlBEljO6aSZrLyJq5fxVCSJPR+BD4Xs/+OuD5AXO8nqrUT1bo/sHPawoPBll95A0iShC7pdIY6AwsPgbyG1HKiLnyZ0BLIm5EuuYUPFUVnlqnqJQQ+B4SHLN1/ru+CNcp49QyqdPcuPSF8ql4eX3hEtdUaC/eLYD0gKLpz6HIIU3lw2T/3CkmSUBWJTDTMoyIELKGgSDoSMjO1a1heiYze84GPI2ddwfFKgIQvHOZrZ3BEGSH84FslXBZq5zDCKXL2VVTJRNeDb1rBvkHZmUDg4/plXL9MxtyL7ecBH1lSN12M7VHC4rzgCwdJklE2KFb3sOPjexfvEsEqY+PLpUj3v4LcwocPTTYJKwnCSpL7zf3fDLjCZqZ2janqpQfWh+NbjJVPkbM3Tunbwp2jLbSDp5r+Es+3/ud0hveiyx9G6phgpvoeEb2bjsizpI09TFXfwvLyhNQmWkKPkzH3MFc7iydsZFRiWjcpYzu6HGO2egJFNohonSiywYJ1nrDaTkhpock8QLN5kIj2aFVwvB+4fg1feIhl3wVfuNh+Ade/O7l6IQSuX0UIf8XxVrbx8YRN1Z3H8vK4voUvXDx/aZvnW/Xw+f1hyzOwhS3cgsHYUwzGnvqwh9HAvDXClcJr+HgMxZ95AD0IZq1rnM19nx2JF2gL7XgAfXz8IEsKshJClQ0MObopHqZ7gevbKJKKImm4koQvgvQ1XYmjyhF8zwMBYbWFwcQvMlV5i5vF75E0duKKCiGpGVXSCWl9ZIzF1Ogl3WdRD8s8irjbKPlE9T3SxnYiSnNj+vaxsLwCmhxGl2+vv9PoG5+xylt0hI+iEQ50XG65jo6okrdvcGL+XxDXeumJfIKo1k7Vm+fUwr8hofXRG32BlDGIoaxWPb0bbBkDW9jCQ46p6gVK7hxhdWM56HuFQDBWOYWzYVXOLTyakOiNvchE+XVu+t9Hl2N0Rp6j5K70AAkEBWeYmcp7VNxpFNlElhR6Yi9yNf8tStIYYbWNmN5HHEjqg0xW3mTBukDG3EfK2P7hnN59QuAzVnqdijdHzcujKzE6Qkcx1QRXC9+j5s4TVltoDx9B4HI5920Sei9pY4ik3k/SGORK/rvocoyUMUREbcXxK1zMfwtPWCT0PuJaD2VnkpI7Tc1bIK51k9D7kCSZi7lvkbdukNIHSeh9JIy+FePzhYPnWwzEPk935Bk0OYLl5Si7bn3bJ9DkUKCUe594aI0Bx6/g+jU0OYwiGZtieQrh4+Ni+2VMOYF0G7f/BwUhBFeKb3Gl+CZlN3vfx5MkmZjaxCdbfxNDefSqqwFMV68yXr3IbO06FS+PIqkktFa6w/toDQ0Ruc3EeDr7EhV3gSazn4zRz7nc98nbU3jCwVAidIb3szvxmcZqreTMcSL7J8zWrjeOkTH6eL71r6/bR96eZLRykpnaVUruAq6/km1syBEOpr5Ce3g3ABIyvnCZqJ7nYu5lyt48EjIhJUlbaAc74p9EkQLJ0pIzx8XCq8zWrjBTu0bFy6FIGn888ncax28P7aIrfIDuyIG7vr4Ak9WLjJSPM18bZqp2iaqX4+TCt7lWfLPRpj/6BL2Rx0gbS7HumldkrHKKyeolcvY4nrCJqS30RA7RYm4jobchhGDBHuVC/sfocogmo59z+R8SUuLsTX6BmldkvHKGeesm7aHd7E5+lrCapOjMcC73Q3Q5TLPZz2ztOrPWdWpekbCSZCj+DK3mdiLqyroLZTfLbO0ao5WTZK0xBD5RtYmW0DY6Q3tJGysJbgVnhvfm/xOd4X1k9F7mrBtMVM5RcueQJZWu8H56Io+RMXrqhGGbm+VjzNauU3CmqXlFfDwiSobO8B5aQ9vJGL33dB8eNKJaFz3Rz+LhIKOiyTFSYieaHEOVTVTJZEfy1wirbbSHn8YTDrKkoitxFCnEYDxI21YkHVWOIEkSbZGncPwysqTd94r0w4Wg4s0RVptJG9txRY2x8hvsSn2dttBBCs4otldi3rpET/QThNUm2kIHSRvb0eQIMgpNxm7yzkgwZ4kqE5V3USSDsNpEzV0gb90kpKaIqq20hPZRsEfIWlfojX2SkJKhI/wEEa0NTV79rZbR0OQoNe8iVwovkdIHCalpNDlCzctypfAdMsY24loPEa31vq7EQ2sMVL0sZWeWhNZFSN2cPMqABGNjeQVM+eF6gAvODKOVsxSc6fs+loRMxuhuuAMfJQghyDmTXCy8znj1PFl7AssrIUsKETVN0Zmj4uXYEX8OVdLXNRKz9ijz1jBVr8C8NUrFzS32gOPXsLyVtQBkSSWiZqiqReatG+SdKRx/dUndxTF6wuZy8XXmatdxhU1IiWNLGhOVc9h+mZjWQlOkH32ZMeYLD8svcbnwU2p+EZBwfIuaN0remSSld9JsDqJJJkgSqqShy5EGQUuRNExlyQ2py6H7IizJKKiSvmyMEppsruhDq68Ql87b4UrxDSYqZym7WRRJQ0YlZ4/j+FUqbpaB2FNE1SYsr8RE9TwyEl79WRwpn0CXw8iSQsmZwxE1rhRfpz28E1UycPwaM7WrOH6VgjOFJ1x84SEhMW8N4+YD5beO0B5CajDOmldguPQuE9ULFOwpVNlAQqbgTFHzi5SdOXYnXiSmtTSeF8ev1TNGIGuNYftlHFFFkbT681HGE/aK6zVRvUDRmcETTsAhEjKz1lWqXhbLLxNWUo0xPSyQJAlVChHVuxpPe1DBYcmYliWNhDEAgKoZS63qi6W43teISS9uC6nNmCJd3/ZwlQ6+W3jCRpOjgfvdzVJyJig5E8FiVArjSTaWl0OTImhymJDaRFhtRpICdUJTTVFwRxG4+MKl5E6iSREMJYGEgi88POEQUjMktF5KziSWP4MmB8eLqK2ElPSai1NF1gipTaT0QarePGU3mB9SxkCwzZ2n6EyiSMajZwwE7EsPyysCAlUOIRHEsXw8fOGiSiYVZ5acfQMZOfgwyiEUVAQCx68g8NHlSJ3J6dVvhIcmh1AkHSF8bL8MCBTZQJUMXN+i6mWpeXniWheeX8MTDkJ4+Hh1L4SGJ2xcv4ZAoEhaYAE+4g/8owKBYLR8mvP5Vyi6s8t/wLbHydvTVLw8naE9JPWN0zRrXpGZ2lUUSWNH/AViWgtCeFh+GUOJsrwWQEhJsD/1JSy3xJncn3O1+LMNR1nxcpzLfZ+41kpf5DADsadwhc3bs/+eGesq7aGdHMn86orJwRM2ZXeBicpZ9qe+TEzNUPHyTFUvcjb3A66X3iautQUTshyjL3qUnsghfOEyXbtCXGvlyWVpiZocQl9jNXGnSOjtGEoE17epugXmrOv0RY+wM/7JRhtDiWLIkfpZC0ruLKcXvgOSRIs5RH/kKLoSZqR8nGvFtyi584TVNEOxIOvB9S1cUcPxK+xNfJ5Xav+ES4VXaTW30xneS7cS58eT/5g5a5iY2tzoZ9a6jo9Pb+Qxmo0BJEnmevEtLhV+SkiJE1IShNTdCCGYt25ysfAKJWeW9tBuhmLPIEsqU9WLjFfOcKH6E+JaKzsTn77FnSqYq11HljRazW30R58goqQpOjMYSnRFVoUsKXi+TVLvIKG1EdWaEAgu5H7CWOUkvvBoD+1+6IyBJdxZZb/1Jva1JqpH3QhYDk9Y2H4JV1SRJY352kUq7mzD67HILZAlHdevYHmFYO6SZBy/jONXcf0arrCQJQ3w8YVHRG0hrLawYF1Ck8PIkrYs+0JCkQ0sv4DqmyiygSLpK8YloaDLYTLGDiy/yHT1BI5fRpOjwTatwGTlGLZ//xV1P3BjwMel6s5zIf9n+MKhI/wYimRQdmeouLOUnTnaQgcQuNh+ieHy67hFi87w4yT0HjwcbhRfxvIKDMZexBM2FXeWijtP2Z2jO/IEGXM7NS/P9cLL+Di0hvbSbOyk6mUZK79LVGuj2dzFbO0CeXuEqpel6i7QG32WhN7DvHWVycpxXGGT1HsZjH2G0AOK125hJQQ+Z3I/ouoV1vzdxyXvTHOl9BZH0l/d8FgVLw9IfLXnfySpdW6YDSJJEio6qpYmpMZR5fW9UT4eU9WLVL0Ce5NfYEfiU0TUFEII9iY/z1uz/46Km6fiZW8xBlxUyeDZlt+kK7y/MZ5WcxuT1QuMlc+wJ/E5AFRZbxg7phKMx1AiK9z19wtTiWIqUXzhYSgRFEkjoqbX7cMXDudyP6TgTnMk86vsSny67q4X9EQO4vo2o5VTnM//kKHYEtExpXexPf4CreY2Okv7GSufpsUc4rHMV/GFx2vT/4yyM0/NL6FKOiCoeQWOZn6VttBOTCWKQNAXPcKsdYPJ6kVMJU5HPfxyJvfnlJxZuiOHeK71Nxvpm33Rw1wpvM7789/knfn/xPb480jLUn+FEExZV/i5zr9LZ3gfIWXtiTyohqjyQttvr9guhKBJ7+NHk/8bFS/PnHWdttAHFzv3xRJ5T2I1+WwLd46J8rv4eJhKgrbw4/jCpuLNY/lFVNlArVeuTei9TFVPkLWv02TuxlQSjJZfp2CPUNU6UeUw3ZFPcHbh32F6c0S1Dgw5zmI9zQCLtTUlkno/14s/JKn3kzF3ktT7VozL9otkratMV09SdRdoCz9GWG1hrnqG6dppqu48HZEnSOr93C8+cGOgYI8zUTlGWMkQ09qYqBxHCI+43knG2MHOxC9wIfcn2H6ZqNZCxthBs7mLs9k/QpEM2sOH2B7/OWaq55ipnUdCoCtRmkN72GXu4OT871F25wipKdLGAH2x+gcAGVUO0xk5wkTlOAhBzcsjSxpt4UOk9QHen/sXtIT2oMlhWkL7UCQd169ueQU+YOScCTzhrPu749fI2ZO3PY4pR4moGVJaJ/dTEXAVBNh+FYQIGOPLng95RXrpyj5lSSGkxukK71+xTlMknYTWznTtMj4Pb2jHFx4jlZPE1BZSehdhZdFADuoTpIxuZq3rzFnDeCzdP00ONciPITlGVM0s88xIaHIEHw8hvPolkwgpycBzIUdWjKEjtIeR8rEV4bSJynniWivNRj8KK1dWCb2dzvBeji98i7wzRUJrrxscwec4o3eT0rsx5bvXVpAkiYiaQZcjVL1CPfT0wUAIwX//ysscn5jgUHsHf+upp8iEH01+0MOA/tiLRLVWdDnW8Hh0RZ5BQml4RSRJYiD2Ir5wAKkRojuQ/g0Efn16lwGZoy3/ZaDjgFzfX9R/k+iJPlf/t8K2+JcaHA15DRKgISdoDR2iydwTfFtQGiW6m0L7lm27f/7bB+8ZEC6uX8VUUyiSQUf4MWpePtimJNHkEIEEikBGJaQk68QKQc4ZQVQ8an7A+sSbw6+3M+UEuhRGCB9X1BDCR1eit8RUF2/WYvRMoEg6phxHk8P4wiOudVJyppmunSOhddETfQb1A8gPbjEH2Z14gaIzi+1bOH4Fy69gexXsuvvJ9qsbTpIfFdwu3Ucg7ug6KLKOqcQ3nSgqSTJtoR1oSojxyllCSoK+yGF8PK4UX8fzbeJmCzF1pUCQKunBS7+my1VGrJtt/HBA1FfsFTfLGzP/mvfn/3DF72V3gapXwJDDOF6VxfdMCtgJ9VbBRzT4iEn1LcCyey4ReC2WKz8utjXroUHHtxo8BMevocpGwKq+ZXWsSHrDs2B5JXxtWW0DCUJK8g70QYKxjZRPMl29TN6ZpOLlcPwqvnCZt0aIqpsvBnU72K7XkAj+ILwCc5UKZ6enOTU1xW8ePkxYe/gFds5MT3Npbg7H9/i1ffvXaCERUVvQlSiqZAZu/MYzZxBwJ5aubfDcLhkHQbs1Yv1ojbf51nuzPFQloTb+vdY9DESh5Lq3a1EiWwIhVm+7T3zgxoChRIlpHeScESRkImpTEM+nGjCpGxd2MUaztM31a1TcBUruFFGtFV/4yFJgjymSyuIFMZUkiqSTs2+iSiYhJYWuRKl5eRasa5ScaQrOOK6ooUhGvY9gX0XScUWNsjuNIUfqohBiUxeWayFtdKLKGpZXwRNOPdZq4fh2wGEQDp6wcXwbV1hMVa8wb41g+eUHO7APAaYSw/LLeOsIaSiSSugOGMxLz8XmQkImprYwGH2aeWuYS4VXmLduIksKRWeO9vAeusIH0OSV9TIkSUZedzyCh0Hg6HaQkdFkk4iaXsXoj9cJTCElXj/PZZP98hdIkrjdC+ULj7Wuh08wmcuS3HhnZUmpZwqtfl4C0RY30Kpf9p4vnc/SODcay1jlNBfyP6nzkkwSWlt91ShRdhfWHOuDxicHBtjd0kJnPIapPvhPebZa5cTkBOdmZ3H9D7as9L3i0twcJ6cmMZR1uBBIpIwhTCWBLK8kJK+3iLhTw2uz2klrvC9rbbtffODGgKkkyZhD5OxhPGxEnURoAko91hdW02i+WZ/UF7dlkOvs6qqXRQhBSEkiSyqGkmgQLyJqMwmtC0lSmKmdJ2vfCFYDUhB/qXl5JEmm6mUD2U052tg3qrVieyWE8AgpKTQ5woJ1jbDahHqL+3EzIUkSUTVN9JaP660QQuDhYnllji/8GWU3+5EzBiQkmoxeyu5CY+W38ncZU47StImx87uFJEkYSoTu8AGy9hhztRtUvTyqZNJk9DMUe5pmc3Bt0tU9vMCNGPcDnHAkAs8EG3hlJCTCagrZU+mPHqUzvHftVZGkbsi5uB0EYPtlXGHj1+WgF8+9Un9/FzMShBCYSgy3niESSOwujckR1UZapqncm/CPJ1wuF17jZvkYA9En6Y0eJqP3ElLiCAICY8mZvf2BNhGSJPG5oaEPrD8hBPlajbMzM1Qc567Fej5oLI7v6sI8I/k8nbG1w0CSJJO4JU7/ccUHbgyosklC7+FQ5q8g8JbFVJawI/GlVfvtSv4CQF260QvcK2tYVPvSX2/8vcnYUW+76AbspS20fk72ocyvcy73LcJKhiNNv4Uqmbw183/QGtqL8RDotQckNw1VTWLUP4YfNUhI7E1+hlnrBrZdvWUClDDkCBmjm8HYE5vWZzAJLvv3ir+v/Ogtf+ZGysfQZIMD6a9wOP0rD8xVq0oGCIHn2400u2Uj2pR+NdkIZE6XpfItHh8W5bgVOkN7OV/4EZpsktQ766mPAZbfq/uLYQqqXoG8M0lISTZY/YIgq0KRVKLaUv339tAuxitnWbBu1kMGZp3FICjY00xXLxFTW4mqTfeUiukLj/HKWRJaB4Oxp+mLHK6PB2atazh+dZWZtvyZEiv+X2z4TG08DtH4bzlkSUKR1n4OxLJ9pHq7xX8vpRrS+G35eJbv6wvBbKXMuZkZepNJHC+oELkcmiyv2l8s/rn491v6lKVlT9o64xcskSVXfg1Wj3v5mF0huDw3z2SxSHs0umq8EtQrVG6RL+FD1hm4N9WkgNn7oProDB9hvnaJs9k/QpY0diR/HlNJ3uUYt3DvkBiMHkWWNK4V32a0cpqCM4cmmXSGd7Ez8Rzd4f2YcnRTe/XxcPwatl+m4mZx/RqSpJB1xjHkcD2nX1/xPE3XrhJR04SVeMNAfRBoMgcYr5xhunaZt2Z/j67IATzfxlRiRNUm4vr95RdLSLSGdnC19CZXiz/DFTatoe24vhUcX2slpMZRJJ2D6V9ktHKSE9k/5Wb5OAPRJ4hqTdTcIllnDMe3iKoZDqa+cl8jMpQoP53+Z/RFjtAe2oUsqVzMv8xc7ToHU19mMLokF/145pepuFmGS++Td6bZnfgsqqwxWj7NROUsVS/PC62/vUGIZmPIkkzG6GO8cpqbpeMNb8SCPcrZ3PcoOtPEtJZb9gp4LY5fw/Ir1LwinnCpeUXyziS6HEKTw6iSxp26e09PT/PSpUv87skTK7b//I6d/KNPfYqovtp7KYDfP32af3viBJ8ZGODr+/bx6o0b/NH5c0yXShiKQlssxqG2dn7niSdImGZjcnV8n2+dP8/3Ll/iejbLQrVKzXWZLZd5+l/9yxX9SMDbv/XXSZorQ2PnZmZ45cZ1zk5PcyOXY7ZcxheC9miMQ+3tPNvbyxe3b1/3ClQdh9du3uRnIyOcmJxgrFDAF4KYYdAejbK3pYXfePwwXfE4EnBlfp5Xhof50dUrXF1YoGzbeEJwfWGBP7lwYcWxH+/o4N//0i8/kIqVjyI+NGPgXi2xu9nvXvqIqW2EIinaw48BoMuRVbmfW3hwWIyFdYV20Wz08pj/ZXzhItVj1YYSQV+DKHY/yNkTjJZPcjL7bYTwqXlFLL+MhMSfjf5DZGR0JcLj6V9mW/zZxn6d4X2Mlk/w1uy/4/jCnyIRCAOFlCTN5iD90SN0hPfc9/i6w/uxvAI3Su9xrfQmV4qvo8kmA9En6Yscvm9jACT6o0fI2eNMVi9wofATzueD1f++5BcxlAghgrQ7U4nyQuvfYLj8PtO1y5zMfhvHr6JKBiElQZPZT0q/v8I1EhBTWxiKPU3Bmeb4wp9g1dMOH8t8lcHo040+JEkirrZyOPN1xitnGCmf4J25f48gyCZpMbfRHTlAd+TQPXsrFEnnSOZrgGCieo6RynFkFGJaCwdTX2G8cjrILlmG0fIprpfeYaR8HCF8yu4Cjl/jYuFlbpaPIUkyca2FF1r/Bkm9447G0ZtI8KXt2+lPpShZFn984QKj+Tye8Dd02/tC4Poeb4zcZLxQ4MLcLDubmjjQ2spCtcpEoch3L10kW63wd597npZoYGhLQEcsyvP9/Tze2cml2Tl+eO0qbdEoX9qxA2MZT0ECzFsIhQL40wsXeHtsFEWWGUilONzRgesLLszO8LPRES7Pz1O2bb62d+/KfYWgYFn8D6/9lNNTU9Rcl+ZIhC9s2wZIzFcqzFbKnJ2dJWmaDWMiouv0p5J8ZnCQp3t6+da5cxRti95kkk8NDKzoozMW2/IILMNDq0D4YUGR9VXpSVv44KEr4RXqfXeL/ugTNJtD6+aOL4chR2gy+tgZ/9S6bVRJb3y0PeEEAjvOHAm9g4zcV1/hBR/eqpdjpnaFmpenyRhAk016IgdJ6u1rhnZMJcqO+CfpDO8j0kjXWzY+JUJneB9hNUXensIRFqqkkTZ6N6WksSQFpM3B2FNkjF7KbhZP2CiSTos5WE8DXMy3V8gYvUiSQlrvDia5+ngMOUpUa2pM1HGthYOpLxNS4uhKiMDoOEqLua0hDywhc7jpa6T1bqJqM5YflNqVkOgM76PV306rOYMrLHQ5TFtoe5AeuIyToMgaGaMPXQ4R11opuXMNYyCut5LSuzCUlSmKYSXBwdQvoMvhFYqLa14fZFJGN3uSn6PkzDUm/rCapCO8h6iawRG1Fc9aVG2mM7S3XvlyrWOuFHS6E0R1nf5UikwkguW6vDk6ykz5ztMZJ4pFFFnmxaEhjnR2EtF0yrbN5fl5vnnuLO+MjXEznyes60R1HUWWGUpnaI1GcTwfXVb48fVrpEMhPj0wsMoToctynRy6dI4H29roiMeIaDpN4TAxQ8f3BcMd7fzo6lXOz87y6vAwv7J37wrvgOP7/PDaVd4dGyNhmjzW0cETnV20xYKU1JJlkavVcHyf0DKjJGGa7Gpqpjsex/Z8fnztKq7v0RWP8/lbOBYhTdsyBpZhyxjYwkcSPZFDd9w2rCYbH/bbQQgfx69yNvd9ZBR6o4dpMQcbmQOusJmonOVm+Tg3yu9x1P8GmmzQEd5DB2sf31Cit6mSKJHUO0ne54p7o+MDtJjbaDG33ba1Khu0mIO0mIMbtotqTexOfnbFtu7IwRX/liWZA6mfb/zbsgKZaB+PhNZOQm+7ozMwlDCG0k+TeWfiKyE1wd7k5++orSRJ6FKI/ujRNX+PrwoRQNroIm103dHx7xSaoqApCnHTRAhB3DBQ7iJt1qvv8439++mOJxoTYX8qxfnZGV66fJmRfJ6ueJyoriNLEp3xwMBxPI/rCwtIBKvv7ZkmEreEBNbC8319jXEvxucBDrkdzFcqXJqb5+z0dIMPsBj3r7ku37l4kVytxnN9ffz8jp0c6Vz5/C9yA5YfN1o3ZIQQ2J5HSNPQFIWkabKrefV92sISHo5KPVvYwiMCT7iU3Rw3y8fojT5Gf/Qw7aFdNBkDNBkDtJrb6QjtJa13N0IOD7d6wMOKrWu22eiOxzna2UlPIrliRRzVdR5r70ACZsolys7maZnEDANNUfDrE3zVcag6Dr4QJEyTVMgkW62sIEV6QlC0LN4aGyMVCvFsT+8qQwAC4uRyQ+BhxCJ5cpEAuRYa5MgPOUNjyzOwhS3cBRRJI6wkCasJLuRfxvFrtIZ2BJoUwmK+NsyV4hvknSnSRnfgUt+yubfwECATDtOXXB2GUmSZmBGEXUqWvYp1fz+wPI+Xr1/n+MQEF+fmGC8WKFoWVdfF8Tzcupt/+URYtm2uzs/j+T4HWttoeoSVFX0hsFyPXLVGe3xt0vNkvoihKUR1fRXv4oPEljGwhS3cJXQlzIvt/3cu5H/CjdI7XCy8jOvbSJKMqcSIq230R4/SGz380JTJ3sIWQqpK3LwD/YdNWKH6QlCybf6bH3yfi3NzxOp8hye6ukiHQ5iKwnsTE7w1MsJcpbJiX9f3KTtBxciIpj30q/+N4Po+VccmV62uawycGJskEwnRn0nRtswYuDQ9i+cLdrQ1NzI8HiS2jIEtbOEuEJSuVWgL7USRNKpuDlvU6hkPUqDDrySJac3EtbZ7Ehn6OCOiptmX+jlsr3xH5M8t3DmCnPwPZmKtOA5vjoxwYnKSznicp7t7eLK7m+ZwmLAeTPCzlQrHJyZW7StLEmpdMbDiuo+E2uHwfJb5SoWyZVOybHa2NtMWjzKZL3J9PouuKuxsbcZyXK7PLzCazeP6Pk2RMNlKlXy1xmS+SFjX2NnajK4qvD8yTq5aI1ut0p1K0JmIozxAw2jLGNjCFu4SkiQRUuJ3RVLcwp3BVGIMRDdPUGoLy3Hvhmkg8ENDROh2qLkuxycnyFZrfH7bNl7o7+NI5xKhUgiBoShrhiRUWSZRD1vMlctUXbdRnfGux3xXe9w7xnMFJgoFbM/D8wVnJqaIGT1UHIeFciUwbkQg6TyazXNldh7f9ynUajiehyd8qo5D2Q7CNLtam5ktlZkrVxispbFc94GzaD4yxkDARhUIPHzh15W+fJZ00RaLptTLR0pB0aLFwhMblbfdwhbuBcEzGdQ1Fyw+k4vliG5RoUNCkoLqmnJdlbNRLe0h8C4IETD8PeEihLdUhIWl92jxXbrdR3v5dfGFV68psNbx1Pp1efDnH0xwAh+/rnLqr7hXa5FAb/2WSEhLhWykh+O+bRYCtcDg3rq+3zAK1rs3ru8zVSohEGzPNNERW+nlsVyXhWqV+VtCBACmqtIZi6MrCsO5HPOVCrbvYSgrp6vbKTnKstzITvCWEfQexPNUtGxUWWGoOcPO1mb+px/+lANd7XQlE3i+4P2RcQSCiVwRCRjIpAhpGo7nMemWaI1G6Uun6EzG+F9+/DrPDfbRlUrQEovy2Z1DD9QjsIiPjjGAIOdMMlW9zFztJll7grwzjeWXqHllHL8GSOiySUgNyqi2hbbTZm6j1Rxs1I7fwhY2C75wmbVuMlY5y5w1TMGZpewuUHaz9QqUge6+JhmYapSk1kGT0UOz0Uez2U9raJAPbm2zMWpekcnqZU7nf8BU9TJVr4gQPoYcJm100R89wmD0CCm9o1EieD0IPGZrw9ysnOJ68V1mrGHceuXBgHPRzGDsCYaiTxDVMmjSvdc5uFO4wqLi5pm3bjJjDTNvjVB05qh4eWyvii3K2F6gLyBLSr3WQYywkiCqZYhrzaT0DtpC22k1h9YsR/soQ5YkQppKwjAYyeepuS5xw1j36VRlieZIIFW9UK1Ssu0Vv3/74kXeGx+n7DgrdAKCfWXSoRCf7O/np8PDfP/qFQxF4XPbVqa9CsCrhxBu5RVIUqAVmjRNbuZyTBQKOL7fkEx+EDBVlVQojLFIiFxjKb+7vZmXzi7w7s0xdrY28/XH9vHSuUtEdJ1kyMRQ1KVKlA9spGvjkTYGBD7ztRHGq+e5VnqPyeplPOEsW3F4dWvQb1j2nmdR84rk7Ckmq5dRJHXdWtJ3CllS6I0c4PmW3yCkbsU5l8PyyszUrvPd8f91047ZEd7Jp1r/OlE1/VCJhgghcIXDWOUMo+XTjFbOkrMncYWDL9z68+g3vAQAnvDwhINlVyg680xUzyNLKppkENXSbI89y874c0TVNIq8eUzjd+a/yXDpGAvWeGObLMkcyXyVA6kvNDT8feFxtfgOlwpvMFw+juPX8ITTWDF7nk2tWmKmdp1z+R/TFznE/tTnaTb61rw+BWeGd+e/yVjlLAVnFlfY9YJUAtdzsLwKRWeW6do1TmZfYkf8WbbFnqI9tGPTzh3A8x1K7gI3yycZr5xnzrpJ0Z2vfz/c+v1a9BKs9Ob4wsMVDrZfpejOMWvdCDwZBN6MqJahK7yHnvB+usJ7CKvJTRivz3ihwFylQs11qTgOE8UitucyWSzy8vXrpMNhwppGcyRCyjQbGQKbAUmSaIlE+URvL9++eJH/7kc/ZG9LCykzhOt7FGyb33niyUb1xIim8+LgEP/p9Gn+8OwZri3M81hHB6osc3JyipOTk0gSDKbTTBQKq/oyVJX/8qmnGS8UeHdsjIuzs3zn0kV2NDejy0rgbi/kyddq/KNPfZq+ZHIVyU6SJA53djJXqXBuZobfeem7HOnsQiCoOg7N4Qi/tn+tssabg9lSmWtzCxwfneDmQo63h0fpS6coWhbZapW5cpmzk9NY7trZG63xGJemZ/nXbx3j6YEedrY0NbgUDwKPpDEghKDi5biQf4XJ6hUW7DEKzgxVr3D7feuhBOof4c2AjELFzW/lk68BgY8rbIru5lV1q7htjVK2Dws84TJVvczFwuvMWTcpODOUnHkcUbuDvYNnMjAMgi01SlS9Io5vM127Rld4D52hXXVvAdzvusHyypTc7Kr7UnBmqbh5YnVlw4uF17hceJOJ6nmqXn7tkQsnKLtt21wXLrZf5emmbxDXmhuhDl94VNw8b839R0bKpym587jCXutoDQPJsatcLrxB1S0gIdNqDt2z8bf4bmatCWatG8zUrjNnDVN05ii7WWpeEUdYd3lEUS+dzIpVoOWXsbzAQLpROsZg7CgD0SP1MNC9jb/murx0+TJvjY7iCR/X97mRzVJ1Xa4tLPB7p05iKAqqrPBsbw9PdnWzv+3OBJvuFF3xOL+waze5msVEocBsuYwqy+h1UZ/lWgG6orA9k+EvHzrEO2NjXJmfZzSfR1MUIprOF7dvR5VlhnPZVcYABJ6I7kSC3z56lLdHx7i6MM/1bJbruRxqPWShKQrtsShhfX0lwRf6+pGQeOPmTYazWSaKRTRZJqzp7G+9XxnvJezrCI6VCAXSyL90cA+dyQS269EUDVOxHfoyKU6OTZKJhPm1xw+QCod48/pNnhnopS0eI24aaKrC1w7tI6xr7GhpoiUaoea6dMRjyA84VPDIGQOe71D2slwrvsf5/Ktk7XEsf3XcaQtb+CDg14WF5qybXC7+jIuF16i5xU0wVgSusJizhsna45ScuaCAkrBoM7cH8fQH4BWpeUXK7gJRNU3emeFK8U3Gq+cpuwu33dcTDjl7Csev0Rbazr7EZ1EIPAyWX+FG+RhXim/XhZhuzxD38Viwx/GES1hNkta70GTzrs9biMAgHa2cZbp6lenaVWZq18k5k3d1nDuFJxzyzjQFZ4a52k1sv4qhRGkxBjDqssz3ArfmItk+yVig/Lde/r2hqI0YswT0JpO80N9PVzxOU2Sl/LFlOVSKFuRdPt03wPZMBh2ZctnCMFQUpR539wP1woNtbXx5xw5OT0+Tt2p4vsBQFdoi0RUrc0WWSZomP7djB0kzxGg+T8m2UGSZ/lSaF/r6EAQlhn3BqmJBkiQR0jSe7+snEwpzYW6W4WyWbK2GBIQ1jXQoTH8qRUxfP1wxkEoFhY10nYtzs+RqNTRFIWGY9KVWay7cKzqTKz3CB7s61vztPW8cQ1VpiobRFQVFlulLJ0mGQ402j3UH+zZHVTKR8CqVxQeFR8oY8IVHxctzvXSMn878a2y/xu2UyhYJSYu41eW3hS3cK4TwsbwS45ULnMh+l+HysQfSjyccxqrnmLNuMlm9wmfafpu41oy8Thnv+0HVK1B052imn6vFtxgpn1nTI7AeBD4VN8/783/MUPQJImoKX3jk7SnemfvDuvfu7t69gjPDqeyfsyP+LBmjB+UuP1sCQc0r8cr0vyRnT26aR/CO+vWLXCi8iuWXebb5L9Jk9KHKd1/7JKLrfK6ln09netixM5gsZDkgL/q+wPf9xr+FCLynnhcYXM/39fFCfz++7zfaLrbLLpSZGllAmbD5Xz79WWLxELOzBWZn8jS3xAmHDXxfUKvZCB9CpspXdu3iK7t24Xk+QgRZBoqyerKSJIndzS3s3kAGeE9LC1/ZuWvd38OaxtGuLo52rZR2FkLg+8E5Lu97kSTo+wJZltAUhV3NzexqXip57Xt+QPD8EEKMBzpbOT81y9s3RtEUmU9u6yesrx/+k+tlnj8IPDLGgBCCojPHjdL7/GTqn+Nx+xdaQqqXmE0E9drxsf0qZSdLzS9+AKPewkcVQggsv8p7C9/iYuE1cvbUA++z5pcYLh/jmyP/D77c/XdI612oGJv6USu7WbL2BFWvyFtzf0DNu/v3xMcja09wufgztseeoejOcy73ExbssXsaUzCplnl//k/5ZOtfu2tejiwpRNR04FX4kNQgr5feQ5dD7E1+hv7I4Xu6Z5NTOYqFKvF4GE1TaG6OgQylYpXZuSLxeIhMJkapVKNWtbEdD+ELOrvSgCCXq5DLlYnFQjQ1Be1OHB9m5OY8z72wi1g8hON4WDUH1/XRtGB6KBaqvPXmFUqlGvv397B9ZztCCGZni9SqNoah0d6R3NwLdht4rs/sZI6xG3N0DTTT2plsGEaO5VLIV2huS6y579xMAU1TCEcMjNCSYbY8O+FBGQq96RS96c3zSGwmHhljoOaXOJv7Eady39/QEAgpCfoih+iNHKAttJ2ommmkJwmAerpQ1p5gpHySkfIpxqsX1ohfroSMiqGEiapp4loLUa2JiJIirCaJqhkyRveqymhbAFUySOkdPNn0NWpeGcsrY/tVbL+K5ZepeSVsrxIQ0+7AwHsYEBAFbX40+U8Yq5yldAcudBmFqJYhpXcQUVKE1AQKCo6wsbwyNb/AROUSll/e0IUeZM1M8b3xf8wTTV9jIHoE4z6qO96KsptlonIRkBrufBmFiJqiPbyDhNaK49eYrl1lwZ7A8tavmncu9zIZo4d56yYXCq+s+C2mNpE2uknrnaiyzlxtJIjhu3NrHssXLleKb/JU09cxlMia1R83goTEweQXeXvuD+4oPKBKBjGtiZiaIaKmMZQIhhxucCQsr0LOGWemNozj1+4o7HGl+Ba6HCaqNtFyhwWVbkW1YnPj+gznz43zK19/grGxBbILZWzb5frVaX7p609w/P0b1KoO4YjBpQsT/KW/8gmuXp6iWKxRrdqM3Jzja7/2JL7nI0sSmqaga8H1VBSZifEs2WyZRDJMU1MM1/OoVW0c20XTFIQQnDs7xuRkDkWWaO9IfeDGgKLIKKqC47gcf+MyL/7SYWRdxvd8alWb3EKZptb4mpP6hRM3CUdNegZbaO1cMgaGr0xTq9js2NcF8ofjOfgw8cgYAzdLJ5ioXqLi5tb8XUKm2eznsfTP02z0EVUzhJTYilKnixBCoBo6YTVBe2gnk9VLnMi+RGmdD5EpR9kef5bB2FF0OYQmm2iSgSLpqLKOKunocugjl060GVAklYiaYm/iM3jCxRNeg1nvCxevztxeJI3ZfpUbpfe4WT6F5Zc/7OGvCU84nMh+N4ile7l1JwIJmbCaZHv8mRXPpCLpqFJAevKFXz9/h4qXZ8EeZ7p6henatXVX0gKfrD3OhfwreMJld+KTm6aTUfHyTNYukXOmEPjEtGaGok/SE9lPXGtBl81GuG6ieonz+ZeZt0bWPNaCPcaF/E+puDksL7iXmhRiT/JTdIZ2kdTb0eUwsiRT88pM165ws3ySK8W31jye7VeYs25iKJF7Yuj3Rg5yofAqJXcBdxlZUEImoqZJ6m1kjB5ajH4iagpdDqHKBqpkoNR1DyAID3m4OH6ViptnxrrOVPUyE5VLG3ocPeEwXbvKjdJ792YMCEEkYtA/0MLUZI5iscbEeJZcroJpaoRCOq7rUas5RGMGA4OtzM4UqFZtxsezVKs2mqoQCuk4jkc0ahKNmSQSIdraE40wg25oaLqK7wcr5VgsRCRqoOsqLfXVdntHivm5EgsLJYSAXbsfVEXNdSCBbqhE4yFmJnJA4P6fHF3g2oUJzLDB4M52alWbm1dnuHllCsfxaGpNkM9WWJgtMjm6gK6r7DjQjW6onH73OtMTORbminT3N9PRm0FVPz7f9IfeGAhWYRYjldMs2KNrErNkFEJKnF3xFxiMPkFIiW24cpAkCV0JoSshomqGqJah7GU5nf1B3UOwMqYpEBhymKTWTrPZt8ln+NGGJMloktmoX78eFkVtbL9KyZlnvHIBi4fPGHD8Gll7gkuFNyi5C/jCXbOdIUdJ6R30RPYzFHuStN6FoURQpLVfuUUeS7M9S1rvJKl3MFY5w3jlIj6r+3CExWT1ckBMM/tpMQc27fxc36bi5lAkjcHoEXbEn6E9tKNRphkC/k5YTWL7FXL2RD09cCVsv8JY5Syub9fPT2Ig+jg7Ys/SYg6scvdH1BSKpDFZvbSmt0UgWLDHyBi992QMxLRmmo0BCs4sOXsSU4mS0juJay0ktFYSeitpvYuM0VP/hshsRPZbFE9KGZ0ktXYiapprxXepeHnW40UUnBkmqhepekVMOXpXq09JkvA8n1KphusGsXIzpKMUqlSrNoapoWsqkgSmqZPJxIjGTFzXxzA1qlUby3Iw6+10XUXTFXRDJRIN7m21alEpWxTyVaan8mTSUTRNQdNUqlWHfK5COKyjyBKSBLWag22v/Q48SEiShKLI6MZqISLX8cjNl0AI8gtlFmYL5ObLAY9AkvBcr1ElUJYlLp0aZdehHmpVh2rZQtOUNTkQH3U89MYAQMldYLxynoKzdnqaLodoC23j8cyXUVg/zWQtaLJBWu/gaOaXGS4dJ+9MryIYWX6Zscp5omqaJqOHRdWxLWweApEQFUOOoCuhh7LAjxCCspvjeuk9JqsX122nSgbNRi+7Ei+wP/W5OyL6LarZJfRgUuoK76HN3EbZ+Tfk3ak1J9uyl2W8eoFY4XUyRvemEQoFPp4QJLVWDqZ+jozevUrjQJYU0non+5Ivcj73MiV3fs3U2qw93jg/XQ5zJPNLNJl96MsMi0Uk9TY8sYf20I51vQML1jhWZP3QxHoIVAIleiL7qda9OU1GLzvjz9ET3h+EbdYx1DY+pkKz0Uda76QzvAvbq3Ct9O66YceqV2DeGiVrTdAe2n5X/YXDBgvzZS5fmqRUrJFIhBgcbEHXFSbGspimRiRqEouZGKaKLEuEwzqqKjMw0MzEuMrMTIGQqROJmsiyhGFohMNLrvJSsUapWCW3UGJ8dIFt21qRFR3dUHFsl2tXp2nvSJLPV5FlCV1XcJyHo3aArMikm+P0DLVy6p1rCCC/UMb3BE1tcUIRAwmJmcks8WSEzt4Mnb1N/PP/+SWOvrCTlvYEuq7w+LPbG16SjxMeemNAILhROrZhOlLK6OSJpq/ftSGwCFlSialNPJb6ed5b+GPyzvSqNjPWVfSywYH0F9Ck0BpH2cJHHa6wma3d4L25b23YbjB2lF3xF9gef/qe+zKVGIOxJwipcb4/8Y/J2VNresVy9iSncz9gV/w5MkYP0iaFqmRJ4bH0l4lrzeuKHSmSRkRJsT3+LKey31vher8VuhxmR/wTtIYGG4JGayGkJBiKPcWV4tustbrOORONkMO9oC9ykBazH8e3yNzGW3U3UCSNhNbOi+1/k/90878la0+saxA4fo3J6gXaQtvuSnfg8NEBHj/SX2fwB/vF4iG6utPBNgAJXvjUnsY+n/7sXgBaWxMMDrWuaCdJEgcO9q7oo6U1wRe+tLrmxtPPbA8yEbwgE6G3r4mOzoAIp2kfvCs98MrQeEQWMyiWNgT/9W5rZWx4jvdev8zAjna++LWjvPnjCmZIJ5oIoxnLyydLUJcvBume6iE8ynj4ll+rIBitnMb2q2v+GlaSNBv9dGyCQtnO5POBqt0aL6gnXArOHFeL79x3P1t4NDFWOceV4ltUN4gLD0SP8FjqywzGjt53fxISHaGdfKLlL9MV2btOK4HllXhz7vfrktubAxmFbbGn0eSNDV9V1hiMPXHbVbWhRNiXevG2xoouh2g3t637e8GZWfdbcCdQJJ2Y2kRK3/wYt4SEqUQ5mvllEvr6gj+2X2W8eoF7TW9ea36S7rAqz522W29fRV2aMlRVRlU/nCkkO1vi4qlRfvr9M5x57wan3r7GyLUZzh+/yU///DTnT9zk+JtXWZgtUC7VKGTLZGeLXDo9Sq26tpHW1BbH93z+4z97hUunx3Cdh0vY7EHjkfAMzFSv4/hrrzoWNcHvll28Fkw5SlxrZcGeWDO32vFrzNSusyv+wn33tYVHC4HC4CVGK2fWbSOjsi/5WdJG1127nNfCohu6O7yPmdo18vYkeWdmzbGNlM+wYI+TMXrWdMHfDRRJI641E1GTtyXFSig06T0Nct16xwsrCZr0ntuuhAPCaRpNMtZUb6x6xdtm/mw43vps+CDWe4tFcXoiB7lQ+Cl5e2rNsXrCIWtP3JNi6Vor1Ttdvd7vKvfW/T/MVXM0EWL73k7aulLYlktTaxzd1EikInT0ZrCqDk1tCU68dY1ozOTrv/VJEskwr7x0iqc/s5uWjiShsIGiKnz1L3+CWCJE//Y2mtsSWDWHTEsc5WNEHoSH3BgQQuALl7KbXVfRLawmG9Kp94PFD29Uy2Aq0TWNAde3yNkTUCdDbeHjg4IzzYI9sW7GiYxCqzlIq7kNU4luat9B1ssOsvb4msYACKpenqnqJSJqclOMgZjWhCLdPuwmIRFSYuhyiJpXCqS+b4Euh4hpTXeUeitJMqqsoyshXNdeFRp0fbtRf2QzFgAPAhE1RUxrxlRilNz5Vb8vSjPfC4QIXPWW5RIK6x8rN/Zy6IaKbkRJZla+a+GIsWKbGdZwLBff8ymXa6QyEVo6kiTTS236tgVSwpquEomZQcqlcvvqmx81PNRhAoGP41trMvwXsVg5bLMQUZOY8tofLVc4FJy1J4MtfLQxVbtKwZlZk8gHwQQ6EDtKWE1uilfgVrSaQ3SH92+4Ur9ZPkXZzd5RvfmNEBgD66vGLUdQ2latp0yufd6GEiGu3Y0OvBQw7df4PPl4uL7zgakI3i0WS/3G1RYi6triMoviZywrq3un8H2BY3sBMc7/6Kqo+r5gOltkLl+mdh/ZCgM72jFDOsOXp7h2YZKDTw0RjqxfwEmSJBRV+dgZAvCwewbwb5trrkr6ipSn+4UmmSjrlGD1hVcnMm75BT4uWPxYj5bPUFiDWLoIRdYYij2B+gAMAYCY1kTG6CGmNZN31lY7HC4fZ3fiU7Sa/n0RCVVJI642cTdPeVhJoEramiRCQ46Q0JrX2GttSIChxJCk6TXXAJ6wcYWDxua995uNhNa8rjEAAsev4eMjUyfD1c9Tou53rF/6xvb6v33Pp1azyefKNLevvQhafGZv3feOty2rvru47YOeHGuOw//6h6+SjoX5+ad2s6//3krMt3amaO18OBX/HjY81MaAX9d+3wiKpGzqSkyW1HUFXBYr8G3h4wWBz2T18pouXwBNMkjq7bQY/TxIMzGiphiMHuF49jtr/m77VebtEZrtXlLGvRPkFoWi7gaaElrXba/LYSJq5i6OJqHK+rqR/aAU9MNN7tLkEKq0UQlhge1XUBSVy2cmuHBqhPnZIk1tSWoViz2HeolEDd59/TKVssXgznZ27OmkWrV597VLtHQkGdjeBmvEta2aw/lTIxx74wqSLPHc5/bR3dfE1QsTnHznOo7rsedQD31DbVy7OMnF06PYlsPnv3qYybEFsnNFcgtlyqUaTzy3k537u1fl838QEAJURfnAtPk/7niojQEJCXmDNCQIyFPuJroMl+qrr4YsKRhyZMsr8LGCIGsFsrvrPRemEqfVGHrg2giGEqUttAPWMQYAcvYEOWfqvowBSVIwldhd7aNI6rrnr0r6XUt1KxvwAQKBqocjt309qLKOuk5K5iIWCYT5XJlKxeLAkX4mx7JBXQHLQZJgYabAV3/9WcyQTjiiUynb7NjXzfWLk+uGCWpVh3KxhhDw3It76eprYnJ0gWKhSld/E7sO9BCOGpimzuDONkJhnbHhOc4eH8bzfKKxEDv3pUmkw7z6vTMM7Gj7wI0BQ1P5L77yDJqqkI5tpXJ/EHi4jQFJvq3uuuNb95VqdCtsr7JuXXMZZav+wMcMQS2AyQ09QrpsktQ3rzb6egg8EBvXqC8685Td7H31I6Pcdegt4DKsbSYrkop+mxTFWyEhr2t0C+4+1v5BQ5LkO85Z8D2BhESqKcbcTKFRkQ8kVF2ltSPZcNO7rt8oKLQedEOltSNFrWIzO5UnFg+RnS/heT6JVIS2utt8YmSe7HwJx3ZJpMKM3phD11XMFo1Ma5ym1ji5hdKHwk1QZJn+9vQH3u/HGQ+1MSAjY8hhFFnD99d++INiN5tXgbDi5bG9ytrjkRRCd7li2sKjDYEg70yvKzsMoMoGMfXuM1p84WP5Fp5wCSuRRkGt9fvRiappZJR1s2vKbnbd+h13CkmS0DZ0ca+1z/qTnyypd388NpACXi4u89DizhP6F6V1ZUWqa+EH+ymKvILs5roetYpNMV+hWrYol2qomrJKP1/TFDLNMaoVizd/ch7d0BqjqVVt5mcLGKbG6PVZpieyOI5HpiXeEPLRdBUzpAWSvPdxmYUQlGs2uVKVmu3iev6KdEpNVYiGDNpSS9/U2XyJXLGK6ween2jIIBMLEzZX8rgc1yNfrpEtVmjPxImYKzMrhBCMz+XxfEE8YpKKLhmjtuNSqFiUqhY1xw2q25oaiYhJ2NRR5CUP11y+jOW4GPUKjvlyFdcTyHLgvWiKRwgZ9yZ297DhoTYGJElGq6cl5e21JVmLzhw5e7KxUrjXm7KoXpWzJyh7a6+sVFknqd8bkWULjyoEeWcKb4MYtS6HSCzzDNzpqtXyawyXrzNvz3E49SRGvajWrfsvPtOKpGIqcQwlSs0rrJmnXnIXKLs57if9VSJI79ssyJKCsonHe5BYee0Xr7BYtWWNPVf8NSjGdWehDFmWkJXAEJTlwDCQpMV/L01MpXyV4avTHHvzCmM35rhwcpQ9h3pINa1coCzMFbl+eYqr5ydAgCxD31Ar50+N8v5rl7lxeZrBne04TlDUaGGuiFVzUFUZRZGQ5KXn5l41+hdJkcevjvPtn53l0tgc2WIF2/Hw/KBaYndLkk/s6+e/+uXnG/v96Rvn+JOfnWEuX8bzfJ7dN8B/9tnHeXx714rjzxXKfPftC/zhqyf5h3/pRZ7c3YOqKMv6Fvxv33yNYsXiC0/s5KvP7mvc27G5Aq+cvMrrZ65zbWIeVZE5NNjBF5/YxdGdPcQjS16xP3vrHBdHZxjqyCBLMt979wLZUhVDUxloz/BXPneEg0OdKB+BKocPtTEAAW+gJ3yAS26BqldY9fuCPcpk9RI1v4Qp319+90j5NHlnel0lN1OO0Rt5jK1cgo8PhBCUnPk18+cXocnmilS8qlel5lWwhIUvfFqNdpCg4OTJOzkSWpKUlsaUQ5hKCEVSsH2rYQzknRwVr0xYiZDQkiv6kpBIaK1BqeM1jOOaV8Ly7167/9Y+1suouffjPZyaAGvBxyNnT1J0Zim5C5TcBSpujqpXrBdysnCFhSecQPcAt/53B1+4uMKuc4/ujOR4+NntHH5mG5Is0TvY0pAbliTYvqej0S6RjnDwiUEOHB0AIZBkeU01wpb2JE0tcR5/agikwNsgSfDsZ/bw9Kd24Xs+an2l63s+vi9WKgnW+wb4nX/wFWT57r93ruczNpfnH/zbH/CZx7fx337tBQY7MlydmOPv/Ovv8eWndvO5wzvYP7BycfUbnz/CX/zMY8wXyvztf/kS2jrCP22pGIMdGZqTUX547DKP7+hqGAO+EBTKFu9fHuUXntnLto4lr93kfIH/5+/9EMt12dPXxm/93JNULYf/8JPj/O6P3uf8zWn+5lc/saKv0ZkcJ66M09EU56998QniYZOrE3P85PgV/rt/9RK///f+IplYGEV5tOeFR8IYGIge4Ubp2JrGgCdcsvYEZ3I/5Ej6q/fURyBu5HEq9711a9MrqITV5F0XF9nCo4+Kl8cX66/yZBQ0aWk1caF4lslqUKAnrsUpuyVSWgZXOBSdAiey7/Hljl9GlVe/fldKlyi7JWpelQV7juebP0NICTcyXIKCP6F1XfICD8e3qHnlexc/qufKbxaCsT68kiY5e5I5a4R56yaztWEW7DFsv4YvnKDkNi6+8PCFXxdBEnWXeuBNXOIwiMb/7sa/Hky2wf1cpfK3bIKRGpP0bYSgJAlZkZGXreoX95UkubHaD7bV2R7rrGrvdYKr2S4nrwUCbc/vH+TQUCeGphALGRwcbGc2VyJXqq5wyQPIsoypSyQiJrqqrHumkiTRlopxYKCDV09dpWY5GGpQnKlqObxz8SaeL9jX305Pa8CR8IXg22+do1yz+dRjQ3z5qT0ko6GgpL0i88dvnOXs8BTnh6fZ1dvSuCaeL1Bkmf/mV16guyWFqsh0tyRJREL8D//hx9yYnCeka8TCdxcKe9jw0BsDINFqDpLS26l6BWx/dTy/5C5wpfgWnaFdtJrbbsviXQ4hBJZfZrh0nInqRWxvbTJiUu+gI7Rz09XltvCwQ+D41rpFsmBRNW/pmXN9B1e46LJGTE1QdsuAhOs75Jwsc9Ys600Ws9Y0tfoz6AkPV7grXdNSwFHYiJzmCw/Xt7mfmkVrCf7cDx6mNZMQPrZfY94aYaJ6gQV7jKIzt8ILsNH9fhSw3uT+QUkKe75PvlRFlmUSEbMe0w/6i4dMKraD662+xovGjizXFQA3GF5TIsLe/jb+5I0z3JhaYEdXM5GQQc12OX5lnO7mBG2pGJE630AIOHN9kkTUpKc5RVs61jBGdvW2kjhxhamFAiOzOXb1Lnn6DE2lJRllqLMJQ1PrarXQ25rC8wULxSqO93Cnut4JHnpjQJIkYloTHaGdlNwF5qybq9rYfoWp6mUuFl5DlhTiWguGHLlt3NP2q1TdoKTo2dyPKTpza34EFEmlJTRIT+TAA1GX28LDCwG3VbuTkVdU4lNlDVMxCSlhMnoTs9Y0BSeH5VuU3GJ9ggfHt7G8GlWvQtEpEFGj+MLH9m1kZEJKeA1WvYQq6Wz0lQxKEN97uq3Eox//XA+2V6XsZVmwxrlZPsmV4psU3bkNCaJbuHvIskTY0EAI8uUq5ZqFqWtUajbFqkU8bDZIefeKRMRkqCODqWucHZ6iIxPHNDRKNYszNybZ3dtGMhpCrXtChBCMzeVpTkaZyRXrnosAnudTqtpYjstMbiUh3dRVEhETU1/2jqtKw8ioWg6e/2gbj/AIGAOL2Jl4npK7wII1js/qF9cTDscWvs28NcqO+LN0hfcQ11oDhjZLFqao170U+MzWhrlRep/LxTeZs4bX7TuqNtEd3kNPZP8DObctPNwIiKtrr+QlZGRJWSG4ExgHCoqkIEsysiRTcku4voOMhCoF7O6cU2DOmmWyOo6ERMZooifcy3h1jJy9gCaFiSiRFat0CdBkY8OV9v0aAx9FBNoEHjPWDS4VXuNy4WcU16kzcWdY7puRVm2/l3DBRwmGprJvoB3PF7x7cRRD0+hrS3FzOsul0Rm+/sJB2tL3l5llaCpNiQh7+lo5dmmMIzu6iYVMZrIlro7P81c+d4T4LdLD5ZrN9PVJzl6fXJML0Z6OU66tfHdURSZirjzO8j2FEB+J2/zIGAMZvYed8ecRQnAm/8N12w2XjzNcPoEmGUS1DC3mIGElgSbrCASOV6Ps5ZiqXqHi5fFuoyhoyjE+1fZbdIX33LaC2xY+qth4lXzrd+Bg8vEV/+4MddfbCYTwG2l4GT3DM03P8wxLbOruUB9doV4EYsmQvW2Pa434o7myvx8cm/82J7LfpeBM31PFQABVMtDlEIYSQZdD6LKJKhsoko4qaWiSiSYbFJwZ5u0xsvb4Jp/FowFdVdjR1cL/7SvP8O9/fIz/9OpJTE0lEw/z9U8e5Bee3kdTYmMNmTtBSNf47OHt/OM/eo2phSKO4/H+pVEiIZ0nd/cSvWUSj4VM9vS28czefvavIXGsqfKKbAII3v574FA+cnhkjAFJkugM78JQwjjC4mLhpxu0FjjCIm9PUXIXkAk+votpQr7wcMXqimgr+kMmpMT5TNvfoDO0G0OOfmRdp1vYGAHRbz3Cno8vXDzhNkJI6z4nIlD3u+1zJFjhzVo+sYu6rv1Gk1ngrbhz3sxHHTWvyPncK7y38C2q7topmbciobWSNrpJ650k9XaSWhtRLYMqGciSEoRvJKkexlkqixyoMEpcL72HX3jjY2sMLJZzfu30dY7u6uGxoU729rehyDIRUycWMtgMJomhqzy9u4//U32DS6OzKLLEpbFZntndh6mpK7ItJElib38bY7M5NDUQNVo1yUvSx1b++JExBiBI4UrrXRxIfR6AG6Vj9UJGa73cgVvQvwd1wpCSoMnoYXv8GbrCezCUyKayq7fw6EAiWA1utNJedMvfjk+yWXXnHd/ecDqTJOWuSLQfZVS9IjPVq5zNv7xMf2Ft6HKYFrOf7vA+UnonETWFqcQwlSimEkWXw8jcWUU7Q46s4JF83CCEwPW8gNjX00xPS4ru5tSaqZDL4fsCz/epWgHB0HV9LMfFclxURV6dfSBJJKMhhjqamJjPkytVmcmVeeGFoSUS4rK2nzwwyO/96Bgnr44TNjS2dzZj6CqlqsV0toSmynXdgEdqatwUPHJnbCgRusP7ArY0MFu7TtGd2xRJ4pASJ6KmaTJ66Q7vYXfik+hyZMsj8LGGhLFBKh8ESoKub9+15O49QVCvDLj+pKZICqr8aKc5bQaEEBTsaYbLx5mqXVq3nYRMTGum2eijP/o4Q9EniGjpLbLwfUIIgSTBbL7MuZtT5EpVJCmIwcfCJi3JKKlYqDHBT2eLTC0UWShWKFYsCpUaQghOX5+kYjmEdK2uLRBp7CNJEpois3+gnbM3phiby2NoCvv621at8CUJDg518u6lEWayJX52dpjJ+SK6plC1HBaKFVqSUfb2t8PH0I57JJ92SZIZiB2hxeznTO5HXCy8viYBMAgNrP5oLrn1AstRJiCAdYX3sj32DJ3hnST1jlX7beHjCImQkkCSlHXnXx8PR1SBtUvKbiaCMEF1XVe3hIIqGRgfhGHykMMTLlO1q5zPv7phu5AaZ2f8E+xJfIpms3+Tev/4kgeFqKdsux67elp498IILx+/AgSTdzSks62zic8f2cmz+/qJh4MY/fEr47z0zgXOXJ9sHCtbrDLy8gkAwqbO3/jyU7x4eMcqD8HRnT2cuDqO63kMdTQx2LFaHlySJDLxML/x+aP87NwwPzs7zLdeO43luCRjIfrb0vS1plFkiYITCM/VhIsj+9h4FOwaUc1oGBmyJBEJ6dSEQ9G10O0lTpmERFjVUeVHx6P8SBoDi7hU/BnXiu+umW6oyyGiapqKW8ARVXzhI0tBARZDjhBS4qT0DlJ6J01GL+2h7cS1Zh6ujOgtfNiQJIhpTeuW5wVwvBp5e4aEtnERoc2AwCfrTOKvo24XUqJbxbTqmKxeYqJ6YcOsAQmZT7X+Fj2RA0SUzat77+M98loF94PpbIn/17//MZoq87d+6RN0NiXQFAUhBHOFCv/qz9/hB+9fomI5/PJzQZbW54/s4PNHdtx1X5IkcXh7F4f/q1+5o/YtySi/+MxefvGZvat+q3oOVwtz/NJP/s3SxtbgO/C9Pz/Ljz7/N8iYEVRFprc9xe/+o6/z1Z/8G8qvrCSiR1Wdf/vcNziY6eJRwSNnDAghcIXFy9P/imvFd6h4OZZb4LocZmf8OYZiT9BmbrslxUda+p8kIaM20sKC+N7GhWK28PGDhEza6N7QZWz7VXL25ANPPXV9h4qbqxfSWnvVGdUyRNQ0W0ZtYAzM1VYvFBYhIbE/+Tk6QjsJK4lNffcdL5At/jiiVLO4OZPl5NVx/j//+ZfZ2dNCJKQ3nsj+dsFPTlwhX66RKy2Fd+/n+t/Nvhu1dXyfvF1FAv7J079Mi7lEHFeQSOhm4xi+L8jaVUDw9w58lgOZzobXQEGiP3b3xcs+TDxyxkDFy3Gl8CY3Su9RcbMrqrepks5j6S/THzlExuglrD54t+0WPuqQSOmdyBu8Ko6oUXRmH/hIXGFRdGY3ZMNHlDQRZeu5B8g5k7fREpAZjB0lpMQ39PzcC2y/huNvnLb8UYWmKISNQJDn9I0JYhGD5kTAvapYNhdHZrk6PkdXc4KeluSHO9hb4AqPkmMTUXWG4s20hWIr9CSUZURyIQRFxyKk6PTG0uxItKxoayiPVir6I2UM2H6VBWuci4XXKTgzK36TkGkP7WB77GnSRtcHQ+bawkceEhJxrRlDCVNylTULFtl+hawT6LAv7vUg4Pg1FuyJDdtEtXTdM/DxRVA3wKfsZql56xVtktBlkxZz8IGQLWt+cU3p9I8DNFUmHQtzeEc3V8YDYywWMkCSsByXkekcETPgDQx2PlyrZ8/3KbsWUc0gouqE1fVVbH0CYyCi6bdt+yjgkTIGis4sE9XzjFROrfpNkw0Opb9ExuhGk8019t7CFu4ekiQRVdNE1QxFZ66eyroSNa/MbO0GnnCRUW+bPnWvsPwyU7XLG7ZJam117svHG67vYHmVeubFaiiSQkRNEVFTm5o1sFiwqOjM1UOYHw4Wy/X6QiBLdx7+FCJI7fMFKLJ0V/suQpFlmpMR/vqXnuQ/vHyc105fp1Cp4Xo+YUOnvz3Nl5/ew4GBdrqak3d7ao1xivqft5IJ7weu8Cm5FhFVJ2dXUaQgrKzIMmFFQ5OXUkt9ISg6VUKqRtmxmauVCLwHAXlQk5VHSrPgkTEGhBBMVC9yufjmqt8USSOqNrE99uxWzH8LDwQdoV2U3Hksa7Ux4AmbojvPVPUqbaFtKA/gtRJCUHFzDJeOr9tGlXTSRjdJ/cETGR922KKyYdlpGZWImnogPpyaXyJrT1J2sw/g6HcGv87mz1drtMXvvLia4/lcnZknX63RlU7QlYzfU/+GprKnr43/8Te+eE/73w6u71O1XWqOQ8tdnN+dHDdnVxkt5/i1V34PX/iEVY3BWBN/dcdTPN3Sh67Uyz8LQd6uMVbK8V+986cgBRkG/dE0//muZznS1E1cMx+ZOenRMQbwydtTzNWGV/2mySYpo3NLgnULDwz90ceZqV1bM3MFwPVtLhVep9noQ1E2/7UqONPM1K5TcufXbdMXOURMbULaks1Gvk3VRUmSUKXNUcG7FdeK71L5EA0BCNzdVdtmvlShNRa9Y2+VKssoskS+WmPieoGux/Y82IHeIxzPo2xZFKvWphoD7eE4f2HwMHuT7TSZERzf52Ypy1szN/hv3v1T/o8nf4k9qXYSuklE0/n6wGPsTLQQ101AYr5W5uXJK/z997/L397/GZ5u7ac1dH81GD4oPDLGgONbASlnjVoCQgj8emGWLYNgCw8CGaOblNFJqJqg6uVX/e4Jh2uld9mf+jxJqe22FTPvFuPVC9woHduQPNgbOURETT8yK5EHCU0ObUgKFCKQdd5MLYDFYkiXCq9TdhfueD/H85gplHjr+kiQ4dDV1ijv25qIkgqHeOPKTbrTCSQJzk/MUnMdetJJulIJ8tUaZ8ensV2X1niUfZ1t5Ko1LkzOEDF0drU34wu4OjPPpak5XN9nqDlNRzLOaDbPjdkFHM/nmaFeWuIRIrpORNfIlTcWcjs/Ns34QoGybZMMmdRcjz1dLZRrNpcm56g5Ls9s7yURNrk5m+XS5CyyLPN4fyepaIhrU/Ncm55HUWQe6++kPRlDu4V0V7Eczo5OMbaQx/N9WhNRjgx2c3Z0mvlihUTYZFub4K0rIwy2ZkhHQ9iOx5tXbvL09l5O3JhgoVwhpGvs6mhhtlgiV65RrFkossyOjmYGW5beGVWSSekhHmvqxlBUfCHojqToiiR4f26Ed2Zv0haKk9BN5Hp2wcFMF5ocjNuKunREEhybG+FsdpKOcGLLGNhsLFYCW+vl9YRNyVmg6M4SVTObzgzewhZMJUqT0UeT0cto5fSq3wU+OXuS0fIZ9FiIuNzEZq06C84sU9UrzNRurNNCIqTEaQttJ6Rs3irpUYYiaSiSjsTapE8fl4qXwxc+EmJTDChPOEzXrjJdu3ZXiqiO67FQrnJuYoY97S3IksRCuUrVcYgYOqlwiItTsxiagiLLXJicYWd7M6osk6/UmCmWmStV8H0fCQm7zQMEruczX6oggGy5wnS+xEK5QnMsUPCTpCVxoLlSmSszc6QiJqoir5qU18LofJ6xhTymrlKq2mQrVfqbUwjAdj1mCyWuTs0x1JZhIlvg2vQCQ+0ZZDkgEo4t5Lkxk2WwLY0i3VqqO8Bsscxsocx8KSBjSgTueSEENdtpcCOKVYuJbAHbDSrajs3nuT69wOh8noptY2hqIHHs+/giuN+u53N2dIrBliXCbeAxUkgbS0WUYpqBqaq0hWKMlLKUXavRVkEitaxtVDNIGSHawnFmakUW1ggrPqx4ZOSRFNlAlfQV5VwX4QqbrD3BjdKxIK7rVXB9G194AdFEfDyVwLawuWgzh+iNHEBeh3Am8DlfeIU5axjbr913f0IIXN9mpHyKqeplyt7aq01FUmgzt5ExutGV+68E96hDkiRkScaQw2jrZAp4wqXozFHzy5siDuQJl7KX40zuh1S8/IqU59vvG5D2TFXjYE8HrYkoAkGxZuN4wXHmyxUqtkPNcag6Dl/ct4MD3e11boBLazzCtpYmmmMRTFWlORZlqCVDvlIDIRipT4p9mRRf3LeD3R0thHWNjmScvZ2t9GZSnJ+YwXbvfNz5Sg1NUdjT1YaqyI2JuDkeYW93K/3NaS5NzpGvWgggHjY52NtOUzQcSBUjkYyEONDbQSYWQl6DCDhXKKMoMh3JOIMtGTLxCJoi09ecoiURpVALJuaudIKx+TxnRwNvRSJscmI4yPBpT8bIRMMcH54gX6nRHI9ydLCL/T1tnL45eVvf0KJCTUIPUfUcXLH+8xIcSyKhmTi+h+3f+fX8sPHoGAOSgqlECSvJNX93hcUPJv+//GTqn3M290MmqhcpuvO4wgZEwyi4n/+28PFGk9lLT+QgzUbfum3GK+c4sfAS14rv3Nezs5geN1W7zJuzv8949cK6bTXJ5ImmX1534vu4Iqm3EVPXT11zhc1w6RiWX7nn91vUvy15e4oL+Vc5nfsBvnDv6hhRQ2d3RwvfeGI/f/T+Gd66NspssYyEwKsX7nE9HyEEhqqSjiwZfP1NKVRF5tVLN7g+t8BjvR0kwmbDgbroS605bpAZY+iNZ/LtG6O8fPEap0YnCetqfcW9/Nzqf65zbVRFRlMVNEXG0BQkJE4OT/DKuWscvzFOxNTwhU86EuKZHb08u6OXP3r7DO/fGCekazy3q5/Dg518850zHLs+juWsvm67O1uYzhV5++oIE7kCRwe6A6/FLUPa3dXCfKnC8RvjXBif4ZN7BrFcF1WVMTSVlniUrz21H1mSiIcMIoaBqij4txxnze++ELjC50ZxgRYzRkjRNmzrIxguZYlpZp1L8GjgkQkTALSYA/RGD3E+/5N121wrvsP10vtB2WIpqEFwOwSKhIsriUCuOKKmyBg9tJgDdIZ2EVITD4QlvoVHC01GD8+3/AbfHP0H6370b5Tep+jOknOmOJz+RVTp7vkDNa/ISPn/3957R1mSXOedv4h0z7965W13VXs7psd7YAACBAgjkhBFSpQokVoIK7crs1zt6sic1dldrVZH65dnJVGEKFECSIAEQBAeAwxmMLane9r7ri5vX5nn00XsH/mquqvrVbWZnukezvvO6VMzaSIjMvNl3Lj3u999mx/O/GvKwdKGq9e02c621CP0Jw409Jp9kNER28acO0LeG224X6N4bf5LtNp9dMa21QmFN48V/sa5wkucLbzEpdIbt9XPhXKVK/OLvDkygV+f9HtzGWYLZX77p4cZassRKNUwhc4NA0o1l6VKlflSmRMTM3RnU4wvFnhjeIyR+SV2dbWzo7OVY+PTfOPYGQ6PjLOnu4OaH1CsusyXKxRqLlIIJpeLnJ+Z58TENEvlKoPtOQ5t7cUxb+7bJ6Wg4vlMLhQo1jxAMLlYZGlilnOT8/XxwfhCgZH5JS5Oz9cNncbtVX2fQrXGYrnK7HKJ46PT9ObSvHFpjDMTc5RqLgOtLTy2Y4COdJLlchU/CMkmYnz8vl1888gZRueW6MqmGOxsvaEXYLy8xPcnzzGYamNLqgWtYaS8wI8nLzJbLfKpLfvoS0aiXqXA5cuXjzKYaqM3mcGWJgtume+Mn2GyvMzndz/BwVzPTd23ewHvq9mtIzbEtrDIWPnYhspiihB0GDnpbtHY98IKQhgYwmDBG2e6dpGLxddImjnanS30xHezNfkgcSPznpO0VlaKgXLxtYerqhvq00PdbRksgRCYwsIQ1j3MpdCEOiTUPn5Yww3L6E1ccaEOqARL2DKOJZ26rPS7PxEKBLaM0xkb4v6WT3Jy+fv4DWLDipBFd5KTS98nXxtlX8uH6XS2ETczG5a1XZlYlrwppqrnmaicZrxygkqwtGGKXEym6I3v4YHcJ+/hZ3v30BXbzkz1Ipd5Y0O3fcGf4/X8V9ideZqtyQdImjdTn0BHRZCq5zlX+CmT1TMsepOE15CbI8NMrz7XzZCOOWzvbCWXjBMqRS4ZxzIMhtpybOtoJWnbBCqkLZlASkHXNez5IyOT+KHiC889RnsqybdOnOPjB3ZyX183W1tbqPo+3Zk0KcfmkcF+dnS2YUlJJh5Da83enk68IMCxTATQkojTlopzsK+LIIwIe9YGefxP7R5EaU06ZtOeTvLgYB8xyyQIFVXPx7GjNjPxGG4QMNTZSqgU7akklmmQS8bZ2d2KUpq2dBLbXP8Ov3h6mN29nTy1Z4iYafK94+f5S88e4uk9gzw42EuoNB2ZJFIKHt0xwIEtXXVtgIh8+alDewlChWVKUjGHoY5W0nGHuGUSt01+9ZkH1ywXhRBoDV++fIRq6GMIiSUlSdPmv3/gY+zKdJAw7PozFljS4A+vHKMSRM/elAYxw+C/ue95HmofIGu/f8Tv3lfGQNzI0B3bwZ7ss5xaeoFqWLijxUBWDQkNUKUaFgDIu2MseZMsepMU/Dm2px4lZ/divIOa8YHyqITLlPw8IQGhClAEKB2gdEioo/8OdYAirG/zV/9NVs80FMCB6BNUC4ucWPoejpHEECaGMJHX/F2p1Lh2u4HEwJAWKbPtHQmyzLsjeKqKUgEhV8cS1scX/X+Iwl/dFuqAQHlMVM7g641j7qUgz6mlH5IwWzCltVpjwhAmEhNDXh1fNDYDKSwMYWAIG0cmaXX6bmtcUhjEjDR7Ms8w7w4zVxumptar3Pm6xpI3RTUooghptftJW+0kjBasuhETfXgUgfYJlEspWGDRm2DeHSHvjrHsT2/YD0NY9Cb2si31CG3Oltsay592pMxW2pwBWp3+jVNCtcd45RRKhxT8WdqdwUhxUiYxhIUQEo0iVD6+rlELS1TDAuVgganqeUbLxykHC/VwZASBpD9xgFD7df7I5mRC2zSwjBjpmBPVU60L/eiYpi2VqAvXiNX0wGz8quvZMgxMQ0YCODWX1mSclGPTlkrQllrLH+mwTNpTiTVCRK119/a1Xod0zIGbkBfoyl41SlKxq14V1aBNpaOxXDu+hGPRrtduux6WaSClIAhDqipqI25btKbWc2M6MkngapEuxzLZ0t6C0hp05LXIJa9OzhYGO7ra1rSRthwO5HrwVEDRryGRJC2brniap7q2kbFjqwsPSxrsz3XjhgEFv4bSmphh0RlP8Vz3DtpiydUsg/cD3lfGgCFMMlYnD+Q+yVztChOV05tOGncKioAlf5plf4bxymmUDtib/RAps+22U8g8VWWmepEr5SP1tMkqgXbxlUugPXxVI1A1/LonINQeYT198sbQVMMCby58dc1WAwvLiGEJB1PamMLGkjFMEcOSDqZ0sOrbBpOH2JK877Zy5rXWDJfeouDP4obl1XF5qkqwOj4XX7v49W3qFoy6gj/LkcVvrP6/QCKFiSVsTBmNZXWM9f+OJmCbmJGm1e6/bWMAIoNgIHmAXe5TKB0yW7uE30DpTqOpqSJnCz9BIEmaOVqsHhJmC3EjgyFNAuXhqQq1sMhM7TK1sHhDA1cgydm97Eo/xbb0I3c8jfFPCywZo83ZwmDyQRbcCRSNwzrVcJlLpdcZr5ykK76T3vhuMlYXtoxjCAOlFa6qUA2XWfKmWPQmyLvjDVNMATJWB/uzH6EaFnDDEnlv7IZ9FSJSrrvRtuuxvaOVy/MLXJidxxCSRwb7aElsHKe+vk0pBHdaMrNRmze77XocHOjizOQc56fmcEyTJ3dvxWrgQbhxf27u2Kwd57HOrTzWufWGxzqGycPtW3i4/U+HMf6+MQZWSCwCgSOT7Mt+mGVvmmV/dsMf+R3vAxpXlXhp7ndxVYW9mefojG2/rZBBLSwxXjnF0cVvvgs9bYwQnzD0qVHc9DiBQAqL7vjO2y6He3r5R+TdsQ3lYO8kNKpuLHnQYJV+LRyZpDe+l/tzn3jH132o9TM4Msnp5R9xpXyUzeJSGkUpyG8qGnRzEMSNNM93fZ6u+E7ixvsjh/luodMZwsiaXC4dZsGbYLNn5Koyo+W3GS2/fdvXM4TFM51/ma3JB8i7o8zWLt2UMXC76M6m6c6meXL7jSev9yO2duTY2nHnSks3sTHeN8ZAMZhnpPw2l4pvcKV8lEC79Zj53WH5H85/DU/VONDyEXrje+5KH5q4+9iX/TBdsR2cKfyY1+Z/n3fzfbRknHZnC5/s/Xu0WN0bpjg2cRVSmLQ6/Xym/7/jK6P/iEpwa2l/N30dDNJWO893fYGh1CEMYeGZ7fTEd3Gm8OM7fr0mmrjTuMe/JhqtFWcKL3Fm+ccseONUg0K9Glijj64gYbZgSwdD2Ddk/0f0HlWPZftUgwKBdm+K9KMIGCkfxZZxumI7r8leaOKDA4EUBi12D/e1/CwtVg9vLXyNJW/6joavBAY98V0Mph5kR+oJslYXUpjN9+0mEAnDmOTsPj7W87c4uvhNpqsXN3Tx3w6SZo6BxEH2ZT9Mf+JAnWsQfYs6nKE7dp0mmng3cc8aAxqNG5YZq5zg1PIPmaycwW1gBCSMHFm7k5zdS8psJ25kMKWFIcyb0GjXqxKiSod4qoqvqtTCEpVwiSVvmnl3ZMMYbtGfX9Wr72z+6D+wsKRDxupkKPUQipDp6gXy7iiL3uRtV64TRNr5bbEtdMd20h3bSVd8Ox3OtnoqbNMQuFkIITGx6U8cIFAeObuPmeoF5tyRd1BmWBCTyVWOQW98D72JfcSuUYC0pEPSbCFltlK6BXni24XWmiW/ggAcaRE3I02BYlDDCwNihkXCdNBaM+8WiRsWMcPGkBI39Al0iNIQNywc4/bJ0bcLLwx4ZXaYuVqZh9r6WXArnFqaxhSSgVSO+1t7CZTixemLFH2XtBVjKN3Kg239a9oJtaLku1xYnmOqWqDgu9SCaHxxw6Y7nmYo3UZ/soW4ufE452tlhot5pirLLHlVqqGPLU3ipkWLHac7nmFXtoO4YTX8PS66FcbKS4yVFllwK1RCD0saxAyLjOXQk8iyt6WLmGHdE9UN71ljwFdV8u4Yxxe/w3DpLa43AgSStNVBd2wnfYm99CcO0O4MYorGD+bmr+tSDhZY9CaZrJxZlZkNGtRECLRLwZ9lonLqlo0BU1ikrXa6Yjtuu6/vFgSCtNX+jtzQrfYAhrBugfT43sCWcXJ27x1vVwpJymrj/twn6IntYrJ6lrHKCeZqV/BUhaBOAF3JnLhqYIo6R8PAEBamsDClgy3jJM0cQ6mH2Zl+gpTZdkdFhVJmG+3OFswGqY6tdv8tayNkrE46nMGGqXlZu2vDlMpGEAhydg+1sNjwd5c0c7f8bgohiBkpdmWeos0ZYMzuwy6+yZI/WSfrRs8nWhio1ecTJanJekaKiSlsTGljyzgtdg97Ms8ykDhIympbVxdlJfMk4g+MrfE4rmTu3CkoramELmOVPI60aLWTOIZJNfSZqCzgK0VHLE3MsPFUwGR1ESkkPfEW0maM8coCQghKgUtvvIUOmX5P0nWvhadCvj9xjrfzE4Q7FcOFPD+YPI9G80BbH7Y0qAQ+X71yjMlKgRY7zqMdW9mR6SBl2qvffV+FXCrO88ejp7hYnGfRrVAJPDwVYkuDoXQbT3YN8WTnIHtbulezGVagtaYceLw1P8Yrs8NcWJ5lwa1Q8l1sw1zNGDiQ62Eg1ULMsNY8+ajCqMexhUnenBvh+MIk87UyRd+NBJoMi5wdZ19LN1tTORxp3nES5+1A6HtQWk9rzVjlBKeXf8Txpe80PCZuZPhI919jIHGQ9LtUv11rxUT1DN+b+r/qP+b1HoKEkWVr8kE+1febzdVaE2sQ6oAlb4rRynHmapdZ9mYoBXnKwTK+rqLRGJiYwiFmpmmxumhzBmh3ttLubKUztg2B8Z6+V9HnQAEhkUDpO79+1KYGAiJa97sX4lh/LYlooMEQFTcLGKucZKZ2gXl3hGVvmkq4TFWV8MNaVMdemJjSIWW2krY6yNm9tDkDdMa20xXb/q6M4Xbghj5HF0e4UpqnK55lKNlBVyzD8aUxLhSnGUi2sj3VSWcsS94tMVbO8+PZszzWtp2d6S6+NvYWuzLdHF0c4Yn2HTzUNkTc2NwgjLRAVv4Z8A5DpSXf5X8+9n2+P3meA7lultwqPYkMpxencVXI7mwH+VqZbZl2ZqpFxstL+Erxzx/5NE93DWHW0/jmayX+3zMv8wfDb9OfbKE3kaUjlsKRJofzY4yWFogZFo93buVfPPJZYsba9zFQIa/NjfCbb3yDSuDRm8hyINdN1o4zVytzuZhnrlYkbcX48od/jawdX7Oy11rz6uwV/qdj32e0tEh7LMmhtgFanDiLboXR8hKTlWX8MOTbH/9rtNjxhoJS7zXuSc9AwZ/jUvF1Ti5/v+F+Q9h8qu836UvswxLvptyjoC++j33Z5zlXeImZ2sV1R1TDIuOVU+9iH5p4v0Ji0Gr3k7P7WPFsbcZHWVlZRn/vjmGpwyuo2guExX+JjH0EM/OPwOh6Z42qRbR/BH/pHyDtAxjpf4iwdt6ZDq+Di/Zex1/8rxDWXoz4L2Ak/mzDI6Uw2Zq8ny3J+1kR7t1sZXQvPJ+NYEuTB3JbGSvlGSnNETcsWuwE5wpTfKznIK12lPNeClzOFaaYqi6x6JbxVRCNWcCHuvfi65C4aVMJvBsbA+4LhLVvotyXMRK/hpH6PPDOvVdFr8ZwcYE/+uhvkDRtfuvMT/nGyAnemh/nFwbv5+8e+BALbpmvj57ki+ff4M25ER7v3IpZ97S02HE+v+dJfmXbQ/QkMmtW7r4K+cdHvsWrs1e4sDzPyzOX+WjvrjXX91TI718+StGv8Ru7HuezWw8ykMzVi+VFb0nRd1n2arQ0EBXSwFevHGO2WuQTA3v59V2PsT3dseb8WhgwUy2uKYh0t3FPGgPD5beYrl1oqLDnyBT9if30xvdiCeddXTWttN0b38t09XxDY0Cj8FUVX1exiCHugGst787zrak/Ytad5le3/lW6Yu8fScsmrmLl/Xl/ldVe8QwE6E0ULm+v3QCN9a67RCPvgA86hA34Ple/G++vp7MRAq0YryyghGbJq1D0q8QMi/taBvj25DFa7Dh7Mr10xDLMVAsE9Uysa40fibzmbtzYYazR0T3WPhvd59tBwrQZSrWRthwkgp5Ehp5ElgWvwmOdW3EMk654hnYniUKRdytr5IwNIWl3kuTsBJY0VlftWmscw+SxjkEmKgUmK8tMVJYa9iGsK6AaQmIKiXnNyl0DWTtGytx4/gm1QhPdU1MY685PCsmWZAvAPeNRvieNgdnaRZa9xuprMSPJluT92DL+nt3EtNW+hhh0PRQKN6xgms4d+bAE2mfOnWGyNoav7q2YexNN3DJEHGHuwEj/bYTsRci2G5/TxC1BCkGLlWBfpo/BZDsdTgZLGmxJtuGpANswabGTJAybAy19hFoxkGhjMNlOxorzePsOBLAr003CsEkYd6/olSUNck48Mk2EIG5YJMyIZNcTTyNFJANs1wXRamHAtcbLShni6ye3lfmiM54ibTmEWlEJ/HpVwquQQrA728krM1d4c34USxo83L6FvS1dJEyrLtwk2azK865sJ0fmxzmzNM3XR07wVNc29ue6iZtWVK5ZiNX+3yu4p3qjtSbUPkveFOUNWNi2TLznef0xI4l5A/KWr2o3lZLYxLsDT3lUghKmtEgYyfec/HQzcMMa1bCKKU2SRuqeWRG82xAyDnII02xm3LxbMISkzUmRtaL488r7n3OSPOJsWy0ZLIRgX7YPhV6dbAEOtQ4CsC3VebeGsApDCBLm1RCFKaOVuUCQspw64Vas1iBYWYWvYIUGt+hWokyC0MdTIaFWKK25Ulqg6LugIVTrPRqmkDzSsZWfTF/iUmGeBbfCTLXIkldlINVCm5MkbTk4G0zmAniobYC35sc4vzzLn4ydZsGtsuzV6EtmaY8lyVixTTMZ7gbuKWMAIt35aljckIVuyRgdsfdWbUsK84YV4ZqGwN3Fopfn5PJRcnY7B7MP3pPGwIw7zcXSWVqsVh5oeegmUl+baOLmIYTAum6CWuU5XGN4RtoL964hKoVY41a/NthmyfU9v/bbGxV0g2rg85PpyxzOjzJSXGTeLVHyXaqhTy0M8FVIV6yxeqcpDZ7oHOTXdz3O10dO8PbCBF8ePsqXLh/hya4hfrZ/Lw+29bM1lWtYe0AIwWOdW6mGPt8eP8NL05f4w5FjfPXK2zzUPsCHenbwZOcQO7MdmOLe0ae5B42BBcINSsNKYdR159/bSlDVoICvNheRsUTsT0n08f2JeXeW1xde5tHWpzateHg3MVUd5/DCKxzKPd40HZtoYhPc7pdUoRkvL/EPD/8Jh+dH6Yln2N3SxUPte+iKp0lbMYaLeV6cvshsdXPp8k/07+X5nl2Mlhd4afoSXxs5wSszw7w8fYld2U4+vfUAn9/95IbnP9e9nSc7B5mrlXhh8gJfufI2xxYmeGNuhJ5Ehl8YvJ+/te/Ze2bWuOeMAT+sbvgxjwrSvLepVgBFf45auLGevyFMElb2jtaTv0eMxfcFykGJRT/PgvdOdf/fPZSCIot+nkV/kbsloX3LEAY6nEW5L6G819DBJVBlMNoQ5h6k/ShG7MNwXUaP1gp0ET//F0GvrawprH2Y6b+DMLfd8PI6GEG5P0X5h9HBMKgiG907I/P3kfYzcK3+gJBoXUV5pwirX0H7x0FXgBjC6ETYhzASv4oQcXgXPElaLaGDS4TVb6D9k6BLsMFCR1h7MbP/A4hcVLFQLaDcVwhL/w9G4i8h459BBxdQ3hso73UIp0BYCNmFtB9Gxj+BMAauXlu76HAO7b6E8t9EhxP1uh1hxOEwtiFjH0Za9yPMgYZ90jqEYJiw+vso/yyoRRASYQxhJD5Xfx6beLe0BqqEtR+gvCPo4CKoeRAOwhhCOo8i7EeQ5p3LLBktLfKjqQscW5jgYK6Xv7nvWXZlO4gZJoaQq16HtxcmbmgMCCFwDIPBVCtdQ2l+tn8vR/LjfGX4GOeWZ/jD4WM81rGVAy09DVMDRZ3b0BVP89mtB3m+dycnF6f59vhpXpsd4fcvH+XZ7u3sbekmdg/wB+5+D66DEHJDs1DpkEBFwi3yPcy/nqldpBjMN9xnCpuU2RpJH99CdwIVMOfOMFK5zExtEl/7JM00vbF+HOlgbFKffrQyzGR1nLw3TzUsI5Hk7Da2JIbodLpJW5lIOCMs8dbia0gkO9N7OLF0lGJQoC++hd74AEkjyZGlNyj4S3TFetia2E5fYmDVwxHqkNnaNJO1Maark1TCMqa0yFhZupweBpPb18S+l/1FLpcuMlK5xEe7fo6p6jhTtQkWvDyB9kkaKfoSW+iJ9dHudK7xpCitqIQVrpQvMl4ZoRpWUKhIuEXG6Yx1sT21i5SRxpQWoQ65UDzDWHWERS/PRHUMV9V4e+kws+4MxkqZUWGzI7WH/dn7Vssx18IqZ4snGa+M8mT7c4BgsjrGaGWYclDGkiatdgd98QG2p66mHS15i8x7s0xXJ1nyF6iF1eiDIWO0WDl2Z/bTZneshig85XGpdI6xyhUW/Dyj5WEqQYm3lw4z786u3reYTLAttZMDmfvXZKMorRguX2S0MkwhWMZXHoYwabXb2Z3aR85uxTHWTsShDlnyFhmvjjBZHaMSRgp7pjSJywTtTge70/uIGYlN3zHQoMuElf+IDmdALUQTrZDo4FK0LZwAtYCR/AsNzjeR9n1otQy6hA7n0cEpkC1o7W36U9FagZomLP8uKjgJqowwOsDoRoeT6GAU9DJgI51nwOhEyC4QFmuMBV1De28R+ifR/lkQDiDRKo9WU4hwMoo/x38BYbRv0qNbhw7zKO9VVO1bKP880uwD2tG6WL93wwAI+1GE0Yswd9f7t9oAqCI6GEaH4yj3hyjvLXRwAfBAOGhVQId5kB1I7OuuP4VyX0VVv4JWCwhhAhZgotUyOnwVrSbRzjQy9ty6CVnrGjocJyz/Ftp7G00YTeIk0eEoYfmLIFvRaq7x+HUYvT/l30F5hyMjAB9EAnQN5b+NDscR/jmIfxZhPXhH7vuyV2O0tIgbBjzUPsD2TBvd8fSauaLou1SDmyNmrxD9bMMka8expEG+ViHUivPLs1wozLG3pXtDk2iFzJixDTJ2DMewqIYebhjw4vQlLhbm2ZFpbxoDjWDJGHKDFXaUxlejEiyRMt99RrLWmlpYZKp6noLf+KW3ZIwWu4cVJbkbtln/WE3XJjhbPMXpwnEW/TyGMEgaKRa9eVJmumEWgdKKmdoUx5beYqI6yrK/hK/9eiVHh7w3z87Ubnam9hI3EtTCGseXjuApl1CHnC+dYbI6xlRtgiFvjna7k7OFE8x5s+QqrYQ6JGe3kqhXKpyojnKueJrLpQssePOEOkAISUzG6Ih1UQurHMg+iCOjFJtyUOJS+RyvzL/IlsQQI5VhJqtjFIMCgfLRaKZrE+zJHMCWDlmrZXVsBX+J0coVjiy+Qd6bxVf+alljQxjkqm10xXqIG0lMoglj3ptjrHKFeW+WRW8BpRV5bx5XuavPImbEaHM6IlJR/fFEk/R53lp8jb74FhQhF4pnmayNUwnKaDS98T4MYawxBmbdKc4VT3GlfJlqWMFXHhpdvydxXOXyeNszxI0EUkiUDsl7c4xWrzDvRn0MdUjem8dTLisdSplpcnbrGlazp1xma9O8tfgak9UxqqpKqEMkkfFRCyvsSR+gO9a7xiCYd2e5XL7AmcIJFrw8vvIiowqBKS1arFYGk9txZPwGxquGcBoVjCCMHoS1H2H0Ax7KP4P2T6G8N9G6iIx/CkR61ZARCLRwkLHPRIaAmkf75wiDm9XjUCjvCKH7XcBEWvuiSV/m6t6Cl9D+GcBF2o8g7IcQ5hBCWOhrFQvVAlpX0EikuQth7gICdDgdGTT+McJqgLAfA5lGiDvHoNfBhcij4r6MMHcinY+AbAVdRPkXUbUqqGmkuQ9hP4KwdiIaaqYodDgSGUG6hJCtCHMQRAodzkE4gTD6QFyX7aT9uifCQ1i7kbIbZIbIGMijvVfQ3lEUBkJm4frVuVpEuS+jqn+CMHqQ1n4wtiBkOjLsvLcgnEXrDWo8aBflHyWs/iGgEUY/wtyGkN3RO+GfQwUX0OEkQjgY1h7uhHZDoMN6dgF0xlLY8uqicUVZ8GJhjnl3Y6+A1pqCXyNVT2281pDojKfZmsrREUtxdmmGglfjet2+6HyXhGmt4wS0x5IMJHP0JrKApuBVUfeI7t89ZwwkzBxyE+lST1WZrp5nR/oJWJcUcuewUrNgonqWqeo5yhtoizsySXdsV8N9jRuGkJBX8j/h5PLbVMMy+7P3sTWxjWJQYLQyzInlo+ukSrXW+MrnhdnvcGz5MG12B1sSQ3THenFVjWNLb/FG/iXGK1eIGwl2pvaunjtVG+fNxVc4mD2EAIbLlxirjNAb72MgPkiL3cqJ5aOcL56mJ9bHjtQePOXywux3uFg6h9aK3en9DCQGKQclhssXOVs4ydnCKdqdLvrjWzDrq26tNTVV5TvTX8eSNq12OztTe5DC4O2lNzmx/DbL/hKWcHi49fHVPl4uX+Ancz/kSuUST7Y9R7vdCQgqYZl5d5rhymWikj31CUdI+mIDZK0WFr08l0rneHvpMHvTBziQvR9TRu+QxKDd6UA2WAVXgjLHlw/ja5+iX2R3ah8pK81cbQbHiBE31nJTpmtTDJcv4SuP3en9ZK0WamGVydo454qn+ObUV9mR2r06QRvCpC+2hYzVwoI3z5nCCS4Wz7IvfZCDLVdJjqawaLPbVw0YpRULXp4XZr/D4cXX2JPez1ByJykzTU1VOLL4Bt+e/hqVsMwjuSfpT1wl1L69dJijS2+y4M3zSO5JOp1uAu1TDJaZd2cZr45h3lSRI40OziGc5zETfwHpPF1feYMOpwmK/wpV+zbaP4sKLiCt+2HFiBcCgYlwHq0fv4AWWcLK79zgmisICStfBbWIjP88RvxzSPuBq3tlJ0p8PVopCwdp7Ytc/dePIBwHYwgj9hxm+u8DNgiBVkWUf5Rg8e+gg3Oo4DyG0QnGndPzUO5LaO8IiBRm6m8gnOdWlRBlOEmIR1j5zyBshNmPNDeSJQ9R7usIox0j+RvI2PMI2Xp1jGoZEAi5ttS4MAeQ8lNIczvCfhIh7NVQiNYhqvKfCEq/FU3K8tU1wkxaK3QwQlj+j4CPTPwZZOznVvuotUbVvktY/v8gGF9niESenQXC0r+FcAoZ/0WM+KeRzuOsvCM6uExQ/iKq+keE5f+MTHwOzTu//wnDXhXymawWVmsYCAGh0pxZmuG12StMlJdpcxqXZw+14vjCJLuzXfVUwKsLvVArRsuLzFSLmNKg3Uk2rCtwcnGSwVQbaduJDIJrftuTlWXGykuRHkIsuerFvNu4t4wBAS1WF3EzvaGufSnIc2TxT9iRfgzuYIz+eoTaZ8Gb4HtT/yflYGnD41JWK7syG5NIGuFs4SQXS2cxpcHTuef5TO/VH2Lem+PU8nG+PP5FYvLqB66mqoyUL/FK/sc83vYMT7Y9x7bkVSPk6fbn+er473GmeIJvT32dnTuvGgNps4U96QN8vOvTTGcP8Xujv82it0Cb3cGf6ftlXOUyWZ2gECwzXZtie2o3r+Z/UjcO+nm09SkebX1qtb1yWOLk8tt8aezf85Xx/8Dnt/3XZGT2ulEKPtXzC+xK71s1bB5ve5ovXvktpmtTnCgcXWMMLHjzzLhT7Env5xf7/8I675BGr/G8SCSDyUgOdtadphQUkcKgJ9bH/syD2PLG2vqBDrhSvsyzHR/l6fYPN9TpvxZPtD3L423PYF133KKf5+TyMb409jtcLJ8jZaZxjBimMFf7OFUbZ642wxV5iZ54Hwezhzb0gC16ec4VTvHm4it8qONjfKzr06TMFeaz5pn2j/K/XfhnHF18E61ZYwxM1caphRUOZh/gc/3r3ffX38fNITFTfx1h7WXNp0J2Ie1HIZxC+W+Dfxqs/URu6DsBhQ5OgPaR5k6EdXDNXmHtRoT7wH0dgjORS73hkCTSfgwj9dfhWje6SCHN7cjYz6CqfwDBZbS5B3EnjYFwOApFyF6E8wxrvlUiFW2rfAkdDkOY3+TWadB5jNQ/QdqPgbiu9oPIbHCeE63Cne4G+yQy8SuIyh+ggzPocOq6zk+hg7NRKEP2YcR/CeTVdoQQyNjHUf4RtJoHVbiuywV0cA7t/RSMLRjJX0aYB1nzkIyhyMAMZ1HuC4SVPwanUbjp1jCQyvF01zb+w8U3+b1Lhxkp5dmSbMU2TMbLS/xg4jzP9mwnUJqi35gUXgl8/u7rX0MIwUBdzjjnJNBac3ZphsvFPBrN/lwPH+3bjXndZK7Q/IM3/xg3DOiMp9mSytEeS6G15mJhjiulBUq+y7Z0Ox/t231Dpcf3CveWMQAgBD2xXZT8PPPuyLrdnqoyWT3NT+f+E4daP0PczNxRFr9Gk3dHuVI6wlsL36AcLG5YtTBn99MX30fLLRa+uVw+TyUsMxDfyn3ZB9es0nJ2G4PJbaTNDME1ZKNyUOJk4RgazcHsITpjPWvOSxhJumI9jFaGmXYnqYSl1X7HjTjtdlS/IWu1YgqLlJmiMxb9wB3p4EiHQAd4KirhfKpwDE95DCV3cDD7wLpr9cT62JPex+nCcRa8uXWx64dyj9PudK1yO7TWJIwUObudeXeOZW9xzfEZq4Wc1crF0nlemf8x+7P3kzGzmNJaM4E1TJW6ph1Rzz2+WT7JAy2PsD2166YKXK14P64/Lm4k6Yn3AVG4w6+7qm/U3kb7p2uTXCqfRyJ5uPUJYkb8GldnFFboi2/hUukC894MlaBEwoxWZ212B+OVUc4Xz/BK/sfc3/Iw8To/oNF93KR3YO4C2Q7E1jBahRAImQEZuTojd7TeYEK+DWjqRLtI9nf9bTJYIa5pXWNDQqbRF5HjRHZd/zUWwuiKrqGrwB0W99Iq+tdQrz+qzxD130PTmFR4FYnIIJKt69ra6B3a7N2LdkXhAS1sojoUK7UoRBRGCUcBE+k8GHEFxPVtCKQ5iDYG0Gpt+EeHsyg/2iatBxAiU+/P9e9QGxi9gILgHDjri1LdKpKmzYFcD//qsZ/nixde5+JynlMLM5hS0p3I8Jv3Pc+He3by+8NHeWHyfMM2bMPgV3c8zOuzI9fUQAjrEs9xnu/dycPtAzzROUSsQXVHgeCXtx3i8PwYE5VljsyP46mAuGGRtWI81DbAg219PN+7i7hhN7MJGmHlA9WfOMCCN97QGACNr1zOFn5CoH22JO+j0xkiZd0+h0DpEDcsM+sOM1k5w5x7hbw7SsGf3dAQMITJQGI/Q6mHV4lpN4slfwlf+cSMOC1265p9hjCwpUPazLLoX2XHB9pn3p0F4MW57/PW4mvrrjtVG2fZX0QgV4ljEKVk2nVFsRUXsSlNYteoOEoho+86CtDkvTksaZE0U8SMtfrZUkgcI0ab3UGgA5b9JdrstWIl7U4HjoxdleStu9psYSOFJLjO67MlMcRjrU/zav4lfpr/MeeKp+mMddMd62VLYogup+emJ/hbQavdRspM31TbNVVj0cszVrnCnDsT8Qa0j688ykEUgwx1uC6GeKsohyWW/UVCHfLtqa/V7+Pa1cdIeZhKUMZXPlVVJUFkDOzP3I9AcHL5GC/Nv8D54hl64n30xPrpjffT4dxsnYHoYy02NJJMrn4+7nAqpxAIoz+K66s5dDgdxcXriCarqTqzvZ+NPIRCZqJVeEM3rODqcjx65+/oEGQ7WmbR1FDBMMLYfvU+ajdi1qMRshMhGue7r/bTaEOIWMOCS5tBaxcdXEb7J9FqNsom0LU6r8JHBRehQVVIrStoVYyuLduJDK8G74BII0Ry/Z3TFbSKvl3aP0ZQ+Ocg1yu4arUQkVABrQs4UvK5oQd4pns7vYmrHo+DuV5a7ASfHNhH2zVu+YfbB/hnD/0cbVaSsUvztGSTpDMxkjGbQ239xITJ7HIBbQjiMYuk5bAtHREKP7PlAA+1D9CXyK4bmSUNPjmwj4fbByj5Hp4KUFpjCEnctGhzknTEUnTGUw1DBAL42f69PNQ+QNF3ccOAQIeYwiBmmLTYcdpjKbqcFKXlKsl0DMO4+ybBPWUMrKDdGaTDGWLCPEMpaJQuplnwxjlfeJlysMhSfJKc3UfCzGLLOKZwkMJEConEQKPqYhQKpRUhQb1sqYuvariqTNGfZ9a9xHj5NIVgDl9VN+1jZ2w7/YkDdMRuXVUtWn0rTGHiNFA2lEKs2x5qRa3ep2V/kUpQotEPtM3uJG4m6vv0ansrzPGrevlyPZtc1BdlGmphDUOYGPX7eD0MYeDUY+puGBEUr0XcSK6upNdcYmWFe90npM1uZ2/mIDVV40LpDHPuDIt+nqnqOPPubJ2zsBVL2HfUKHCMGNZNhBQqQZmRymWGyxeZqI6itELV36tAB6vP5k5wgXzl49Y9NAtevuEqPmEkSRhJ2pwOrn0P+uIDkeElJJdL55msjbPg5ZmuTTJbDwENJrchblhhTkTM73cxFLcxJNJ+hDCciDgJ7stI+yHAidj4/pEo1VBkohXzhsa4g9g09PPufYCltRsdXESHl1C17yOdKKVPax8djqK8N0HEkdZuhLGZ6p+ox+Rvpa86SisMzqPcV9HBabR2WTV6VqoN6jINjSAd1OsNAMQ3uba1yiNZiwB05ILXagHtn2CzFERh9CFkC6a0eLBtvbHam8zSm7w+DAlbUq0MJHOEgeLI5UtUiy5hdwu9/TlanQSPtG5hWZSJxW2yLWsXNHtautjT0tgwNoRkZ6aDnZnbq4YrhGB7pp3tmShDxfdDlhbLtLWnESLar5TCdQMKhSrxpLOptPH1qFU9ymWX1rbU6vXuBO5JYyBttdEb38uCO8HZwosoGhdMWfKnWFqe4lzhJ6StdvoT+8laPSTNXCQhLBxMYaMIV2vJB8qjpkoU/TlKwQJFf5Ylb3rD1MFGiMkk97X8LAOJg8SNzaz6xpBcnRAbzR1RqZi1q63IyRZ9mB/OPUGL3bphzNmUJkkzVTcYrm3h5mEIuVrXXWu97oWLSsBGz0U2UNGyhHVLRZssadMZ6+ZjXZ/iwZZHOF04wcXSWcarI5wtnmK4fIHP9f9F2u2OW/bEbAZDGBvex2sxXZvk5fkfcb54ila7gyfan6XT6SZuJOpZDLN8sXzpjvSp7hxHCoOn25/HlhYbPb8WK0fiGs+NY8QYSu5gS2KIvDvH0aU3OV88xcXSOU4XjnOhdJZfG/wCcSNxE6GCu0VsMpDxz6K811HeYXQ4gQ6ugOyIJlj/CFotI6zdGM7TwEbGXOT2vhsQzpPIcIawcj4i0qkFkB2gliOPh/c6wtyOsB9b4/VojFsbh9ZelI1R+V209yYY/UjrPoQ5gJDtkXEhYgSl/xuCK416f831buD1uYHxK8xBhLkbcX22w3XXE2Yv1+tVrLuU1tFCpeahVfRNMgyJZRuYpsHE+GIUZu5rAaBcrFFYrqyGKbXW1KqRkWOaEiklQRCilEbIaCUkpcC0DNyaj9YayzIxTIlhrP0thKFaPVcpjWUZq8esnGvbJkIKSsUqb785zBPP7sZ2TAxDokJFqRD1r6Mrg9aSMFR4XgAabNuMvrFq5Z+K2hOCudkCly/M8PATO7BtA8MwkPKdv+f3pDEAMJA8SNzIsOCNM11rHNtZQaA9Fr1JFr3Jd7lXAlvG+Ej3f8mO9BM4xu2Vn8yYOUxhUVM1lvwlkubaH0qoA5a8tdkLEeM8slS3JAYZSu4kYSYbftBX3NQVyuv23QyEELTbXYxWh6kEFQIdrCHNaa3xlEfejdItc3YbjoxRZmNhps1wrVtdCEFnrJvOWDfPdnyEOXeWF+e+y4/nvs/O5be4L3uI7tj1HI218ch3A28tvsZ4dYSuWB+/suWv0BvrXzWAikGBYrBBitVt9DFhpshaOSarY2xP7aTL6VnNjrgWK/ft2tSpFRjCoDPWzce7P81Huz7JpdJ5Xsv/hNcXXubtpcPclz1E2tqIfHZ3IYQB1v2Ymd+M8tTdlwmDC9FO2Yq0DiITfx4j/mkQjRnhdxvC2IaR+BzCyBEU/kfC8r+v70ggjG6M+M8jU38TIXObeDZuD9o/hnJfQHuHEUY/Ztt/Qsj2NV4SrTWi/LsNlyNCJkCmAV3XB9igeqUuo3Vl/XaRWC1GJcx9mKkvIMwtvNPfqNZQLFT5oy+/TmG5QiLhsGWwnZ/5ufvJtaWYn7+6+Am8kO9+821ybSl27e2lrT3N8nKFP/zSa6hQs3tvL509Wc6emmR6cpG29hRhqGhtS/Hgo9v4468cplSs8cDDg+w50E9X91rPxNzMMqePjzM3WyA/X+Dhx3fQv6WNIFB875tvUyrWePr5veRySaYnl/jmHx1mdqbArr09bBlsJwwVL3z3BD19rQxu68QnYHQkz4s/OEUYKJ7+0B7KZZf8fJH8XJH8fJEnntlNKh3j/JlJXv7RGSbHF9l3sJ9tu7pIp9+5Ku89awxIDNqcLXy677/lO1P/B9PV8/h6c0ngdxNxI0N3fBePtv4ifYm9GOL2GaDbU7s4VzrJTHWS40uH6Yv3r+5b9PKMlCORGUdetZRTZpr7s4d4Nf8ir+ZfwpQ2u9P71rUd6pBQB1jvoH8CwQMtDzPjTjFcvsDx5bd4KHeV+V8Jy0zVxjlTPMlAYittdnvDcMfNIgrfRB+la0MLAkHGzHAg8yAvzv2Akl/EU+tjnLZ0iMkYSqtIb+BOx7ChHhJQWNIia7as2bfo5Tm29Nam58dkHEc6q6JAehPCXW+sn12pPZxcPsoLs9/lM71/lozMrjP8VgimlrCuCYNFJMFrQ0ASSafTxe70fl5beImCv7yGnHqvQWsNepGw9G/RuoqR/A2MxF+G1fQ4s+6evnuV9W4IXUK5PyUsfxnhPIeV+adXwy5CErnYV8J5d/jSwUQkBiSSCOfD9VTEq78rrXUUq9cFGhEnhexGGpH+hvaP1nkF69O4dTgKanz9+UYXwr4PyqD9w2i9jLgDaeBhEHL+zCSD2zowzIjjZFqN/eumbfD4M7s4c3KCStllebnCD751jI98/CC5thTHj4zwwndPIKTguef3MTqSZ2G+iNaa1rYUH/u5+7lwborFhTKXz0+vMwZqNZ9iscae/X1sGbqfH33vJLWqz+NP7+Jjn3qAc6cnGB2ew7ZMhnZ0smWog8987mGcmIVpSFw34LGndnL08BXCUHH54gzDl2Zp70jT0Z3lh985QTobp7M7y94D/Qzt7OIr//EVnvvofga3dzI1ucRnPvcwtmOt81rcLu5ZYyAqpmGSsTp4pvMvcrl4mNHKMeZqw/jafc/6ETcybEncR19iP93xnbQ7WzHeYdx6KLWDvvgWhssXObr0JlIYdMV6cMMak7UxRsqXyVo53GvqIThGjIHEEPdlDzFeGeHHs99juHSBzlg3lrSpBOW6qE1Azm7lybYP3Xb/BIL92fs5VzzNrDvNS3M/ZNHL0+F0UVM1JqqjjJQvYwjJcx0fj1zO7+B+uKrGieW3OVc4RW+8n6yVwzEcAhWw6C9wvngGKQz6ElvImOtXswkjSdZqIWEmOV04Tlesh4zVgkSiiciMPbG+d1S8qCfWx0j5Mnl3jh/NfZdd6X0Eymfem2WmNknem9/U7Z40U2StFmxpc6pwnJ54HykzA0QV2NrsqI9CCFqsHIPJHezP3s+ZwgkMYdAb6ydjZTGESSUoM+tOEzNi9Me3sD/7AADVsMLL8z9iyVukI9ZF1mrBkQ6ucpmuTXK5dAFTWGxNDhEzNnfJ3l0otH+xrhPQBUYXwshxfVbDvQwdTqDDYbReRBoPg8whRBJukQR4W5AJhIhFqoFqAa4VztEKqETKkmoD+W6jA8xBkJ3ocBLlvoR0nlpNvdRao/2jaP8EOpxmHR9ApBHGEMK6Dx1cQlW/DTpAWgfXcgy0QusSOriCMHdEokub/EalIenf2saf/NFbpDNxBra2s23H+rj/SljAtlcyTnTkli/WSGfiJFMxNJpazSeZdIjH7VUXfGG5yqljYyzkS9i2CWh8f71nROsozBuLW6TScYJAMTW5yKnjYywtlkkkbJYWyyilMAyJaUicmIVlRdlVUgos20SpKPTh1nzcmk86Eycet7j/oa0sL1UxTYNsS4JMJo7nRym0Zr29WNxGSvGnmzOwAiEEUpj0JfZjCoekmaPF7iHvjlHwZ6ldkz53p2AIE1smSRhZ0lYb7c4g/Yn9dMa2kbW6bikO3hACslYLezMHUVozXh3h+NIRWu12hIhesISZpFf2M1y+eE2/DNJWhkdan+TNhVcpBQXOFU8xXh3FFCaB9nHDGnEzeU1O+u2j1W7n/paHOFM8wUxtilPLx0hZUbpj0S8Amvuyh9iT3t/QhX0r0ETa/ePVURb9PDEjjiVsNBpX1Vj2l7kve4j++AAJc71b2JIW7U4X+9L3cb50mpPLR1fldm3pcDD7YIPQwq1hS2IbM7UpLpcvcLZ4koIfhQU85WJKk8HEdqZq4xv+MG1p0x3rY1d6P5dK5zi+fARHxjGEJGEk2Ze5j55YLyCwDYfOWBcP5x6va0CMseDNE5NxpDAIlE9VlemJ9a8ZlwYWvQVGKpeZdadwjBh23RtRDcvUVJX7Wg7RE+t7R56jG0HrIFKnw43IaGoBHY7Vd1bQwWUUAjBAmAjZUmemXzOpCAtEHFQJ7Z8irPzRNROJUd+fRBhbori02IDxfpcghFmPgZvoYBxV/TqR3PBKuMiMJm2jL1J4lDnuVP+F7AEZTZKREuKLV9vXlchQ8Y9H/Wkg1iREHGEMIJ0nUdWvo2rfi9QPjSGEsNG6jHJfRqtCdP51GQlC2GB0ImOfICz/Nsp7DbQXyQ/LXDR27aJ1uf5ujGIkP48WzqZ3QAhBKh1jemqZWs2ntS2FBipll6mJReZnCximJD9fJBazmBhbID9XIJGwyWTjdPW2MHx5lmTSwXMDcq0pAj9ESIE0oknV80LmZgtMjC3Q2ZUh8DeeX8JAMTWxiO8rLMtAKcXszDLTk0t092RRYV1B1ZCkM3EunJ2isytDLG5Tq3pMjC2wmC8xP1fAtAxS6RiFpQpOzKStPY3rBlF9A9uMOAGRBYJlm8TiFufOTNLdkyWdiWNZ73wqv6eNAbiabtgd30l7bCvb/Ue5UjrC5fKbzNdG8VQVRYjSIUoHKB2uxsGiv3pdaytM6uhvlHEghYEhLOJGhha7h05nGwPJg/TF993R4kgr43mw5RFyVivHlt6K9OurV2iz29md3s+WxCDj1RGW/SVMefURGcLgUO4x0laW88XTjFQuM12bJNQBSTNNm93OQGIr21O768dL0laWQPlrRHgyZpZA+2tWhxkri6984vWcdoHg4dYnaHc6OVs8yfniaUYrwzjSocPpYii5k0O5R8lZV3OfDRERF1vt9ihvv8FPO2mkaLHWaupbIpoot6d2MVGXS1Za4UiHtJVlZ2oPT7Y9R4fTteZ+rN5TIeiO9fB858dBaCaqY8y7c0hhkLNz9bDB1fdACknCjJj4tozqo1/PW7geg8lthNonbiQ4XzrNhdIZYkaC/vgWdqf3sSO1h6naGCkz1ZDgKIRgS3IQx3CQQjJRHcULp+uGTOdqaEMphe8GGMrhUPZxEmaaY0tvMl2bYsHLo7UiZabpifcxlNxJT+xqkRlHxhhMbifQPrO1aRb8eZTWxI0EOauVwcyOiHxqrc9Xv6PQLsp7JVp56kpEmqsbA1rlUe4PEf6paLKUcaT1IMLcG7nNV3gQRjsYnWj/DLp6GVX73jUXMOux93ak8xGMxC+jZesNMgfeI+j6N0dkEbIDRALtvUHgH7nmIAE4YLRh2E8hY8+D9eA6FcHbhTC3Ia29aLcTHV4hLP1WVMRICLRaguAywtqLFDtQweXGbRi9yPgvobw3Ud4rEJyLxiOzoObQ4TzSfhAhEyjv2PrzRQYj8edQ3tvo4CRh7RsI9weRroBIgFqODAFVAGFjJP8yN2Ijaq1ZWihjmdHKuFSsceXSLFsHOxgbmSefL2FaBjOTS7S2pxi5PMfSQplEwqGjK8P9hwb5/reOYVoG3T0t7Nrbw+iVeUzTIBazSaYclNIkEjZhqCiXXQxTEos3fq8CP+TSuWmuXJplcHsnyZRDteJF55ZcHMfCdkxsx2LLUAdHDw9z4P4BWttSVCseI5dnKRVrTE0s0jfQimEYvPbyOSzbrE/wBpZlYprRAjSdjWPZBo5j09ndwpE3LvPQY9twYvYdMQaEfqdJ0XcRBW+OBW+cRW+CRW+KvDdK0Z/HV1UC7eGH7irPYGXCN6WNYySIyQxJM0fSbCFltpFz+uhytpG1u7HkigXfxAcBum5xB0G4GoNccTXeDt7JueVCjTNvDbOcL/HUJ+4nlmwckrqZ9hsRCzfbBqCCy6jaDwmL/wLhfBQr+0/r4jxroWovEFb/GOX+ADP115CJX4+IZ4AOp/Dzv4gOZ7mxBoGBTPwaZuqvIozueoGbImHxfyGs/RAhsvXVcxsRq15H+fNqDu1fBL2AkfwCRuIXEeZ2tK6h3NcIlv4mwtyHkfgcRuKX1t+/cI6w8vuEpf8dI/EryPhn6+mL7xBao6miyr9H6P4A7Z9HWrtB9q56FbVWEfHOP4pWC0j7CWT8z2AkfmFN/5T7A4Llf4yw9mLlfjsq1nSz3VBLUT2I8r9D+8fQuggijpC9yNgzGMkvEFa/jqp+AwTYbf+Z67MWoqqFZwhK/wblH6+TCSXS2oNM/BcIswflvkhY+tcYyb+CkfoCa+o7aA34hLXvo91XUP4xdDhCVLAohzC6o+qJzhPI2GciY24TI7VSdvndf/Njfv6XHyfXmuTcqUlefekcn//bP3Ozj6b+V6+m+DU+ThMGCsPcOP12+NIsJ46OcOiRbXT1ZjHNq+Jqjc7VWhMECnOTNlevHWoMY3P3/822dyt4XxsDWqsoPa/+V6HqP8aVIW00NLG6+o3W6lFetojKUtTfx6Yx8EFBreIxM5bnxa8f4bGf2c/W3b3EEja1isvUSJ5MLkFLexq35lMp1qhWXFSo6Nnaju8GVCsufs3HtC3aujJIQ1JYLJOfXiLXkSbbmsbzAnwvoFZxyU8X2L4/SidbmC2wnC/R0pGmvU5SOvXmZWbHFzj07G6ybY0FkZYXSlRLNXwvciX2DnaglGZqZJ7AD8m0pmjrivgVM+MLeDUfJ27R2ReJXE2P5vG9gFjCpqM3t9puVD48AO0SrcCdhqExrQPAj8IAwgbsawwNFXkEbiqEJ+rufxshJDoYJSz/O8LKlzDTfw/hPIMwtqyfJHQNHVzBz38OYW7HzPwTpPNU3cgJiVQFJQgrcluv67+q99+rX9+6ZVGfRtBaExb/Jar2JwhzCCPx5xH2Uw36r9FqlmDxC2i1iHSexWr5X6/r38pzkNFEfgshyqv3wWOtqJKsZy/YROOPCIQrhtxaY0DXz/VAh9e0YdSfuaj30Vv3DqyMEagrLIb1NlbeiXq4ZIUQihldeZOJLQhCLpyb5thbwyilae9Is21nNzt2NZJcbnRPVv8r6sEmxsAKNjpmZmqJK5fn2Laza41+wEbn3kyb78Zxt4J7PkywGaIJnPeGlNPEn1oopfBcn8JiGSElQsDklXmmR+epVTzeenGOZz99iNEL04ycmyIMFS1tKUpLVRZml/G9gEwuyZVzU3zslx5jaiRPYakMWnPytUs8+9lDlAtVxi/NMjO2wODuHoSASsmlWnZZmF3m8I9P8+f+xs8gjatFTTYz01/97nGKS1VMU5JpTVIu1OjobaFW9ZgcnmNxrsBnf/05hk9PMDWaRxqSzt4cHb05Lp+aYGpkHssx6ahvW0E04dj1j/vGiEriNi7bHZUh3yyvfGNoXUL5R4EQYR9CmFtpWIQIuy7WY0eu5pVJTdTj8Zuq+q2M04E7WKlwBco/FukgyO6ooqJcn4Ks0Qijux5zr6sDruvfjZ/DRrh6Hzb7xG/eftSGAWxW4dLY+B6uCpxFxtY7XV8ZhmTrYDvZbPQ+2LZJMnXzRNirc+bmHbmZybWlNcmeuE08Ya/xMtyOPPS7edyt4H1tDDTRxJ2AaRnYMRvDMmhpT2OYBguzy4xemMGJWRQWy7hVj9JSheV6XNLpb2V5ocTc5BJ2zKJ3qBO34lEq1JganUdrTd9QJ6MXZljOl3FrHl4tKuPc2pVBSMFyvsjC7DL5mQKj56dvKbSwnC9RKbqkshF5KD+zjJCwlC8zO7HI7HikU2E5FrWKh1vzkIZk2/4+LMekWnYpLlcwTINt+24kevNeIqznrq+sbBulYGqiYjjnARWlzr2LhMhbhq4AAZHKXyPvSMQr0P6Z6FgRr5cXbmIzCCFIJB0SybufUuo4Fo5zD3BU7iCaxkATH3jYjkUyHSOWcIglbIQUeK5PuVglDELSLckoFcgxSWbixJMO/ds6mRqdJwwVlm2S60yTaU3iuT5u1SeedKIQQXuaWsVFK00qGyeRjtG3LZKfzc8UmJ1YZGmuiFuLVraBH9Tb8KiUamQ3kByNJRwM06CtM0vP1nYmR+aZHluguFShsFDGd6P22rqz2GdNFmaWCYNoYmrvaeHyqQkW54s34mzdBdgI2YXmEso7jsQGo5urn6owYqIHl1C17wC6XsSn5e51+XoYnRBcQYfTKP8IkoOsrYMQoHUFVfsGWi3VMyJuoQx6E028C2gaA000UUcYhPz0W8d46hP30TvYgVcLuHBsjFjKJpVN4MQshKSu/b/yD+YmF/nJ148weWWe5z77ENsP9DN2cYbv/8GbFBZKPP6R/eRnl6lVXOQ1BUmW8yUqhRoavSonOje5xOjFGYbPTOB5IR/7pcdw4va6UKqQV6+/snNxtkAYRqtOUW9vamQet+ZTq/nUalEK2OSVObw6fyEM77xA0zuBMDowEp8j8N6IYu/WdoSxLZLyBdALUfGdcAJUAWH0YiT/EsIcursdvwZm/BcJggm09ypBcAJhPVQvqFRP7VPzaP80qBmQncjYhyM1xSaauIt4XxMIm2jiTiFiAYeEgcJyLAQRlyAMFKKuV67rOuFCRJOx1vDTbx3D9wKe+dQDQJQDHImcREIn0hAYplHPOKsXjqorhoWhQoUq0lmvXwMNobq6bUVd7HrPQBiEdU03gZD1FOT6ddFX29Na47uR2qBhGhimjOSkawFCRNs2UnG7G9B1BrryXkNVv4b2z6HDGdD1wlwyHVX6M4eQ9oPI+GdBtNC4VPB7jxXSnQ5OodxXUe7LEFyMUvqgTsjMIMx+hPMURuzj9TK+8Xui/018cNE0Bppooo6VFEMaMIOhMWnn0slxVKjYcd/AmuOurxuw6TVv4ribRaP2lNIRd1tet22T9Kq7ieg5lNBqPsoK0B5X9fHrTHYRQ4hUXQNfbspCvxvQqgK6iFbL9cyGFflnyQrpTshspEyIxWbKe0008V6gaQw00cQ7QGGhhNasxvabaKKJJt6PaBoDTTTRRBNNNPEBR9M31UQTTTTRRBMfcDSNgSaaaKKJJpr4gKNpDDTRRBNNNNHEBxxNY6CJJppoookmPuBoGgNNNNFEE0008QFH0xhoookmmmiiiQ84msZAE0000UQTTXzA0TQGmmiiiSaaaOIDjqYx0EQTTTTRRBMfcDSNgSaaaKKJJpr4gKNpDDTRRBNNNNHEBxz/P671td0DezA8AAAAAElFTkSuQmCC",
      "text/plain": [
       "<Figure size 640x480 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pred_word_cloud(gt_trancripts.tolist(),stop_words=stop_words+get_keywords(gt_trancripts=gt_trancripts_))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9fb82c90",
   "metadata": {},
   "source": [
    "## LIME test ??"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 349,
   "id": "ed3d00eb",
   "metadata": {},
   "outputs": [],
   "source": [
    "for el in input_df.transcription:\n",
    "    if type(el) != str:\n",
    "        type(el)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 337,
   "id": "6261f6c5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Int64Index: 371 entries, 3425 to 2554\n",
      "Data columns (total 2 columns):\n",
      " #   Column             Non-Null Count  Dtype \n",
      "---  ------             --------------  ----- \n",
      " 0   transcription      371 non-null    object\n",
      " 1   medical_specialty  371 non-null    object\n",
      "dtypes: object(2)\n",
      "memory usage: 16.8+ KB\n"
     ]
    }
   ],
   "source": [
    "input_df.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 362,
   "id": "6d2088ec",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>transcription</th>\n",
       "      <th>medical_specialty</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>3425</th>\n",
       "      <td>REASON FOR THE CONSULT:,  Nonhealing right ank...</td>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2855</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Low back pain.,POSTO...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          transcription medical_specialty\n",
       "3425  REASON FOR THE CONSULT:,  Nonhealing right ank...  General Medicine\n",
       "2855  PREOPERATIVE DIAGNOSIS: , Low back pain.,POSTO...         Neurology"
      ]
     },
     "execution_count": 362,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "wrapped_data.df[:2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 364,
   "id": "87c72e11",
   "metadata": {},
   "outputs": [],
   "source": [
    "wrapped_data.df.reset_index(drop=True, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 365,
   "id": "e0d621c9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>transcription</th>\n",
       "      <th>medical_specialty</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>REASON FOR THE CONSULT:,  Nonhealing right ank...</td>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Low back pain.,POSTO...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS:  ,Carcinoma of the pro...</td>\n",
       "      <td>Urology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>CURRENT HISTORY:,  A 94-year-old female from t...</td>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Adenoid hypertrophy....</td>\n",
       "      <td>ENT - Otolaryngology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>366</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Intrauterine pregnan...</td>\n",
       "      <td>Obstetrics / Gynecology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>FINAL DIAGNOSES:,1.  Herniated nucleuses pulpo...</td>\n",
       "      <td>Neurosurgery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>368</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS:,  Desire for sterility...</td>\n",
       "      <td>Urology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>369</th>\n",
       "      <td>SUBJECTIVE:,  The patient is here for a follow...</td>\n",
       "      <td>SOAP / Chart / Progress Notes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>370</th>\n",
       "      <td>CHIEF COMPLAINT - REASON FOR VISIT: ,Pelvic Pa...</td>\n",
       "      <td>Obstetrics / Gynecology</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>371 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                         transcription  \\\n",
       "0    REASON FOR THE CONSULT:,  Nonhealing right ank...   \n",
       "1    PREOPERATIVE DIAGNOSIS: , Low back pain.,POSTO...   \n",
       "2    PREOPERATIVE DIAGNOSIS:  ,Carcinoma of the pro...   \n",
       "3    CURRENT HISTORY:,  A 94-year-old female from t...   \n",
       "4    PREOPERATIVE DIAGNOSIS: , Adenoid hypertrophy....   \n",
       "..                                                 ...   \n",
       "366  PREOPERATIVE DIAGNOSIS: , Intrauterine pregnan...   \n",
       "367  FINAL DIAGNOSES:,1.  Herniated nucleuses pulpo...   \n",
       "368  PREOPERATIVE DIAGNOSIS:,  Desire for sterility...   \n",
       "369  SUBJECTIVE:,  The patient is here for a follow...   \n",
       "370  CHIEF COMPLAINT - REASON FOR VISIT: ,Pelvic Pa...   \n",
       "\n",
       "                 medical_specialty  \n",
       "0                 General Medicine  \n",
       "1                        Neurology  \n",
       "2                          Urology  \n",
       "3                 General Medicine  \n",
       "4             ENT - Otolaryngology  \n",
       "..                             ...  \n",
       "366        Obstetrics / Gynecology  \n",
       "367                   Neurosurgery  \n",
       "368                        Urology  \n",
       "369  SOAP / Chart / Progress Notes  \n",
       "370        Obstetrics / Gynecology  \n",
       "\n",
       "[371 rows x 2 columns]"
      ]
     },
     "execution_count": 365,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "wrapped_data.df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 368,
   "id": "94f7db26",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "--- Logging error ---\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/models/model_explanation.py\", line 71, in explain_text\n",
      "    shap_values = text_explainer(pd.Series([text_document]))\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/explainers/_partition.py\", line 128, in __call__\n",
      "    return super().__call__(\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/explainers/_explainer.py\", line 264, in __call__\n",
      "    row_result = self.explain_row(\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/explainers/_partition.py\", line 153, in explain_row\n",
      "    self._curr_base_value = fm(m00.reshape(1, -1), zero_index=0)[0] # the zero index param tells the masked model what the baseline is\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/utils/_masked_model.py\", line 69, in __call__\n",
      "    return self._full_masking_call(masks, batch_size=batch_size)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/utils/_masked_model.py\", line 146, in _full_masking_call\n",
      "    outputs = self.model(*joined_masked_inputs)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/models/_model.py\", line 28, in __call__\n",
      "    out = self.inner_model(*args)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/models/model_explanation.py\", line 136, in text_predict\n",
      "    return prediction_function(df_with_text_documents)\n",
      "  File \"pydantic/decorator.py\", line 40, in pydantic.decorator.validate_arguments.validate.wrapper_function\n",
      "  File \"pydantic/decorator.py\", line 134, in pydantic.decorator.ValidatedFunction.call\n",
      "  File \"pydantic/decorator.py\", line 206, in pydantic.decorator.ValidatedFunction.execute\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/models/base/wrapper.py\", line 129, in predict_df\n",
      "    output = self.model_predict(batch)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/models/function.py\", line 40, in model_predict\n",
      "    return self.model(df)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/sklearn/utils/metaestimators.py\", line 113, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)  # noqa\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/sklearn/pipeline.py\", line 535, in predict_proba\n",
      "    Xt = transform.transform(Xt)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/sklearn/preprocessing/_function_transformer.py\", line 182, in transform\n",
      "    return self._transform(X, func=self.func, kw_args=self.kw_args)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/sklearn/preprocessing/_function_transformer.py\", line 205, in _transform\n",
      "    return func(X, **(kw_args if kw_args else {}))\n",
      "  File \"/var/folders/0k/jpy013m11jb32ny3n1lkthzw0000gn/T/ipykernel_6545/757042665.py\", line 8, in preprocess_text\n",
      "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(lambda x: x.lower())\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/pandas/core/series.py\", line 4771, in apply\n",
      "    return SeriesApply(self, func, convert_dtype, args, kwargs).apply()\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/pandas/core/apply.py\", line 1123, in apply\n",
      "    return self.apply_standard()\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/pandas/core/apply.py\", line 1174, in apply_standard\n",
      "    mapped = lib.map_infer(\n",
      "  File \"pandas/_libs/lib.pyx\", line 2924, in pandas._libs.lib.map_infer\n",
      "  File \"/var/folders/0k/jpy013m11jb32ny3n1lkthzw0000gn/T/ipykernel_6545/757042665.py\", line 8, in <lambda>\n",
      "    df[TEXT_COLUMN_NAME] = df[TEXT_COLUMN_NAME].apply(lambda x: x.lower())\n",
      "AttributeError: 'float' object has no attribute 'lower'\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/logging/__init__.py\", line 1100, in emit\n",
      "    msg = self.format(record)\n",
      "  File \"/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/logging/__init__.py\", line 943, in format\n",
      "    return fmt.format(record)\n",
      "  File \"/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/logging/__init__.py\", line 678, in format\n",
      "    record.message = record.getMessage()\n",
      "  File \"/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/logging/__init__.py\", line 368, in getMessage\n",
      "    msg = msg % self.args\n",
      "TypeError: %X format: an integer is required, not AttributeError\n",
      "Call stack:\n",
      "  File \"/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py\", line 196, in _run_module_as_main\n",
      "    return _run_code(code, main_globals, None,\n",
      "  File \"/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py\", line 86, in _run_code\n",
      "    exec(code, run_globals)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/ipykernel_launcher.py\", line 17, in <module>\n",
      "    app.launch_new_instance()\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/traitlets/config/application.py\", line 1043, in launch_instance\n",
      "    app.start()\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/ipykernel/kernelapp.py\", line 728, in start\n",
      "    self.io_loop.start()\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/tornado/platform/asyncio.py\", line 195, in start\n",
      "    self.asyncio_loop.run_forever()\n",
      "  File \"/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n",
      "    self._run_once()\n",
      "  File \"/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n",
      "    handle._run()\n",
      "  File \"/opt/homebrew/Cellar/python@3.10/3.10.11/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/events.py\", line 80, in _run\n",
      "    self._context.run(self._callback, *self._args)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/ipykernel/kernelbase.py\", line 516, in dispatch_queue\n",
      "    await self.process_one()\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/ipykernel/kernelbase.py\", line 505, in process_one\n",
      "    await dispatch(*args)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/ipykernel/kernelbase.py\", line 412, in dispatch_shell\n",
      "    await result\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/ipykernel/kernelbase.py\", line 740, in execute_request\n",
      "    reply_content = await reply_content\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/ipykernel/ipkernel.py\", line 422, in do_execute\n",
      "    res = shell.run_cell(\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/ipykernel/zmqshell.py\", line 540, in run_cell\n",
      "    return super().run_cell(*args, **kwargs)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/IPython/core/interactiveshell.py\", line 3006, in run_cell\n",
      "    result = self._run_cell(\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/IPython/core/interactiveshell.py\", line 3061, in _run_cell\n",
      "    result = runner(coro)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/IPython/core/async_helpers.py\", line 129, in _pseudo_sync_runner\n",
      "    coro.send(None)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/IPython/core/interactiveshell.py\", line 3266, in run_cell_async\n",
      "    has_raised = await self.run_ast_nodes(code_ast.body, cell_name,\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/IPython/core/interactiveshell.py\", line 3445, in run_ast_nodes\n",
      "    if await self.run_code(code, result, async_=asy):\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/IPython/core/interactiveshell.py\", line 3505, in run_code\n",
      "    exec(code_obj, self.user_global_ns, self.user_ns)\n",
      "  File \"/var/folders/0k/jpy013m11jb32ny3n1lkthzw0000gn/T/ipykernel_6545/1713860283.py\", line 4, in <module>\n",
      "    explain_text(\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/ml_worker/utils/logging.py\", line 76, in wrap\n",
      "    result = fn(*args, **kw)\n",
      "  File \"/Users/mathieuroques/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/models/model_explanation.py\", line 79, in explain_text\n",
      "    logger.exception(f\"Failed to explain text: {text_document}\", e)\n",
      "Message: 'Failed to explain text: PREOPERATIVE DIAGNOSIS: , Low back pain.,POSTOPERATIVE DIAGNOSIS: , Low back pain.,PROCEDURE PERFORMED:,1.  Lumbar discogram L2-3.,2.  Lumbar discogram L3-4.,3.  Lumbar discogram L4-5.,4.  Lumbar discogram L5-S1.,ANESTHESIA:  ,IV sedation.,PROCEDURE IN DETAIL:  ,The patient was brought to the Radiology Suite and placed prone onto a radiolucent table.  The C-arm was brought into the operative field and AP, left right oblique and lateral fluoroscopic images of the L1-2 through L5-S1 levels were obtained.  We then proceeded to prepare the low back with a Betadine solution and draped sterile.  Using an oblique approach to the spine, the L5-S1 level was addressed using an oblique projection angled C-arm in order to allow for perpendicular penetration of the disc space.  A metallic marker was then placed laterally and a needle entrance point was determined.  A skin wheal was raised with 1% Xylocaine and an #18-gauge needle was advanced up to the level of the disc space using AP, oblique and lateral fluoroscopic projections.  A second needle, #22-gauge 6-inch needle was then introduced into the disc space and with AP and lateral fluoroscopic projections, was placed into the center of the nucleus.  We then proceeded to perform a similar placement of needles at the L4-5, L3-4 and L2-3 levels.,A solution of Isovue 300 with 1 gm of Ancef was then drawn into a 10 cc syringe and without informing the patient of our injecting, we then proceeded to inject the disc spaces sequentially.'\n",
      "Arguments: (AttributeError(\"'float' object has no attribute 'lower'\"),)\n"
     ]
    },
    {
     "ename": "Exception",
     "evalue": "Failed to create text explanation",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/models/model_explanation.py:71\u001b[0m, in \u001b[0;36mexplain_text\u001b[0;34m(model, input_df, text_column, text_document)\u001b[0m\n\u001b[1;32m     66\u001b[0m text_explainer \u001b[39m=\u001b[39m shap\u001b[39m.\u001b[39mExplainer(\n\u001b[1;32m     67\u001b[0m     text_explanation_prediction_wrapper(model\u001b[39m.\u001b[39mpredict_df, input_df, text_column),\n\u001b[1;32m     68\u001b[0m     shap\u001b[39m.\u001b[39mmaskers\u001b[39m.\u001b[39mText(tokenizer\u001b[39m=\u001b[39m\u001b[39mr\u001b[39m\u001b[39m\"\u001b[39m\u001b[39m\\\u001b[39m\u001b[39mW+\u001b[39m\u001b[39m\"\u001b[39m),\n\u001b[1;32m     69\u001b[0m )\n\u001b[0;32m---> 71\u001b[0m shap_values \u001b[39m=\u001b[39m text_explainer(pd\u001b[39m.\u001b[39;49mSeries([text_document]))\n\u001b[1;32m     73\u001b[0m \u001b[39mreturn\u001b[39;00m (\n\u001b[1;32m     74\u001b[0m     (shap_values[\u001b[39m0\u001b[39m]\u001b[39m.\u001b[39mdata, [shap_values[\u001b[39m0\u001b[39m]\u001b[39m.\u001b[39mvalues[:, i] \u001b[39mfor\u001b[39;00m i \u001b[39min\u001b[39;00m \u001b[39mrange\u001b[39m(shap_values[\u001b[39m0\u001b[39m]\u001b[39m.\u001b[39mvalues\u001b[39m.\u001b[39mshape[\u001b[39m1\u001b[39m])])\n\u001b[1;32m     75\u001b[0m     \u001b[39mif\u001b[39;00m model\u001b[39m.\u001b[39mis_classification\n\u001b[1;32m     76\u001b[0m     \u001b[39melse\u001b[39;00m (shap_values[\u001b[39m0\u001b[39m]\u001b[39m.\u001b[39mdata, shap_values[\u001b[39m0\u001b[39m]\u001b[39m.\u001b[39mvalues)\n\u001b[1;32m     77\u001b[0m )\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/explainers/_partition.py:128\u001b[0m, in \u001b[0;36mPartition.__call__\u001b[0;34m(self, max_evals, fixed_context, main_effects, error_bounds, batch_size, outputs, silent, *args)\u001b[0m\n\u001b[1;32m    126\u001b[0m \u001b[39m\u001b[39m\u001b[39m\"\"\" Explain the output of the model on the given arguments.\u001b[39;00m\n\u001b[1;32m    127\u001b[0m \u001b[39m\"\"\"\u001b[39;00m\n\u001b[0;32m--> 128\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39msuper\u001b[39;49m()\u001b[39m.\u001b[39;49m\u001b[39m__call__\u001b[39;49m(\n\u001b[1;32m    129\u001b[0m     \u001b[39m*\u001b[39;49margs, max_evals\u001b[39m=\u001b[39;49mmax_evals, fixed_context\u001b[39m=\u001b[39;49mfixed_context, main_effects\u001b[39m=\u001b[39;49mmain_effects, error_bounds\u001b[39m=\u001b[39;49merror_bounds, batch_size\u001b[39m=\u001b[39;49mbatch_size,\n\u001b[1;32m    130\u001b[0m     outputs\u001b[39m=\u001b[39;49moutputs, silent\u001b[39m=\u001b[39;49msilent\n\u001b[1;32m    131\u001b[0m )\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/explainers/_explainer.py:264\u001b[0m, in \u001b[0;36mExplainer.__call__\u001b[0;34m(self, max_evals, main_effects, error_bounds, batch_size, outputs, silent, *args, **kwargs)\u001b[0m\n\u001b[1;32m    263\u001b[0m \u001b[39mfor\u001b[39;00m row_args \u001b[39min\u001b[39;00m show_progress(\u001b[39mzip\u001b[39m(\u001b[39m*\u001b[39margs), num_rows, \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m\u001b[39m__class__\u001b[39m\u001b[39m.\u001b[39m\u001b[39m__name__\u001b[39m\u001b[39m+\u001b[39m\u001b[39m\"\u001b[39m\u001b[39m explainer\u001b[39m\u001b[39m\"\u001b[39m, silent):\n\u001b[0;32m--> 264\u001b[0m     row_result \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mexplain_row(\n\u001b[1;32m    265\u001b[0m         \u001b[39m*\u001b[39;49mrow_args, max_evals\u001b[39m=\u001b[39;49mmax_evals, main_effects\u001b[39m=\u001b[39;49mmain_effects, error_bounds\u001b[39m=\u001b[39;49merror_bounds,\n\u001b[1;32m    266\u001b[0m         batch_size\u001b[39m=\u001b[39;49mbatch_size, outputs\u001b[39m=\u001b[39;49moutputs, silent\u001b[39m=\u001b[39;49msilent, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs\n\u001b[1;32m    267\u001b[0m     )\n\u001b[1;32m    268\u001b[0m     values\u001b[39m.\u001b[39mappend(row_result\u001b[39m.\u001b[39mget(\u001b[39m\"\u001b[39m\u001b[39mvalues\u001b[39m\u001b[39m\"\u001b[39m, \u001b[39mNone\u001b[39;00m))\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/explainers/_partition.py:153\u001b[0m, in \u001b[0;36mPartition.explain_row\u001b[0;34m(self, max_evals, main_effects, error_bounds, batch_size, outputs, silent, fixed_context, *row_args)\u001b[0m\n\u001b[1;32m    152\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_curr_base_value \u001b[39mis\u001b[39;00m \u001b[39mNone\u001b[39;00m \u001b[39mor\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mgetattr\u001b[39m(\u001b[39mself\u001b[39m\u001b[39m.\u001b[39mmasker, \u001b[39m\"\u001b[39m\u001b[39mfixed_background\u001b[39m\u001b[39m\"\u001b[39m, \u001b[39mFalse\u001b[39;00m):\n\u001b[0;32m--> 153\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_curr_base_value \u001b[39m=\u001b[39m fm(m00\u001b[39m.\u001b[39;49mreshape(\u001b[39m1\u001b[39;49m, \u001b[39m-\u001b[39;49m\u001b[39m1\u001b[39;49m), zero_index\u001b[39m=\u001b[39;49m\u001b[39m0\u001b[39;49m)[\u001b[39m0\u001b[39m] \u001b[39m# the zero index param tells the masked model what the baseline is\u001b[39;00m\n\u001b[1;32m    154\u001b[0m f11 \u001b[39m=\u001b[39m fm(\u001b[39m~\u001b[39mm00\u001b[39m.\u001b[39mreshape(\u001b[39m1\u001b[39m, \u001b[39m-\u001b[39m\u001b[39m1\u001b[39m))[\u001b[39m0\u001b[39m]\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/utils/_masked_model.py:69\u001b[0m, in \u001b[0;36mMaskedModel.__call__\u001b[0;34m(self, masks, zero_index, batch_size)\u001b[0m\n\u001b[1;32m     68\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m---> 69\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_full_masking_call(masks, batch_size\u001b[39m=\u001b[39;49mbatch_size)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/utils/_masked_model.py:146\u001b[0m, in \u001b[0;36mMaskedModel._full_masking_call\u001b[0;34m(self, masks, zero_index, batch_size)\u001b[0m\n\u001b[1;32m    145\u001b[0m joined_masked_inputs \u001b[39m=\u001b[39m \u001b[39mtuple\u001b[39m([np\u001b[39m.\u001b[39mconcatenate(v) \u001b[39mfor\u001b[39;00m v \u001b[39min\u001b[39;00m all_masked_inputs])\n\u001b[0;32m--> 146\u001b[0m outputs \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mmodel(\u001b[39m*\u001b[39;49mjoined_masked_inputs)\n\u001b[1;32m    147\u001b[0m _assert_output_input_match(joined_masked_inputs, outputs)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/shap/models/_model.py:28\u001b[0m, in \u001b[0;36mModel.__call__\u001b[0;34m(self, *args)\u001b[0m\n\u001b[1;32m     27\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39m__call__\u001b[39m(\u001b[39mself\u001b[39m, \u001b[39m*\u001b[39margs):\n\u001b[0;32m---> 28\u001b[0m     out \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49minner_model(\u001b[39m*\u001b[39;49margs)\n\u001b[1;32m     29\u001b[0m     is_tensor \u001b[39m=\u001b[39m safe_isinstance(out, \u001b[39m\"\u001b[39m\u001b[39mtorch.Tensor\u001b[39m\u001b[39m\"\u001b[39m)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/models/model_explanation.py:136\u001b[0m, in \u001b[0;36mtext_explanation_prediction_wrapper.<locals>.text_predict\u001b[0;34m(text_documents)\u001b[0m\n\u001b[1;32m    135\u001b[0m df_with_text_documents[text_column] \u001b[39m=\u001b[39m pd\u001b[39m.\u001b[39mDataFrame(text_documents)\n\u001b[0;32m--> 136\u001b[0m \u001b[39mreturn\u001b[39;00m prediction_function(df_with_text_documents)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/pydantic/decorator.py:40\u001b[0m, in \u001b[0;36mpydantic.decorator.validate_arguments.validate.wrapper_function\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/pydantic/decorator.py:134\u001b[0m, in \u001b[0;36mpydantic.decorator.ValidatedFunction.call\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/pydantic/decorator.py:206\u001b[0m, in \u001b[0;36mpydantic.decorator.ValidatedFunction.execute\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/models/base/wrapper.py:129\u001b[0m, in \u001b[0;36mWrapperModel.predict_df\u001b[0;34m(self, df)\u001b[0m\n\u001b[1;32m    128\u001b[0m \u001b[39mfor\u001b[39;00m batch \u001b[39min\u001b[39;00m \u001b[39mmap\u001b[39m(\u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_preprocess, dfs):\n\u001b[0;32m--> 129\u001b[0m     output \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mmodel_predict(batch)\n\u001b[1;32m    130\u001b[0m     output \u001b[39m=\u001b[39m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_postprocess(output)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/models/function.py:40\u001b[0m, in \u001b[0;36mPredictionFunctionModel.model_predict\u001b[0;34m(self, df)\u001b[0m\n\u001b[1;32m     39\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39mmodel_predict\u001b[39m(\u001b[39mself\u001b[39m, df: pd\u001b[39m.\u001b[39mDataFrame) \u001b[39m-\u001b[39m\u001b[39m>\u001b[39m numpy\u001b[39m.\u001b[39mndarray:\n\u001b[0;32m---> 40\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mmodel(df)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/sklearn/utils/metaestimators.py:113\u001b[0m, in \u001b[0;36m_AvailableIfDescriptor.__get__.<locals>.<lambda>\u001b[0;34m(*args, **kwargs)\u001b[0m\n\u001b[1;32m    112\u001b[0m     \u001b[39m# lambda, but not partial, allows help() to work with update_wrapper\u001b[39;00m\n\u001b[0;32m--> 113\u001b[0m     out \u001b[39m=\u001b[39m \u001b[39mlambda\u001b[39;00m \u001b[39m*\u001b[39margs, \u001b[39m*\u001b[39m\u001b[39m*\u001b[39mkwargs: \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mfn(obj, \u001b[39m*\u001b[39;49margs, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs)  \u001b[39m# noqa\u001b[39;00m\n\u001b[1;32m    114\u001b[0m \u001b[39melse\u001b[39;00m:\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/sklearn/pipeline.py:535\u001b[0m, in \u001b[0;36mPipeline.predict_proba\u001b[0;34m(self, X, **predict_proba_params)\u001b[0m\n\u001b[1;32m    534\u001b[0m \u001b[39mfor\u001b[39;00m _, name, transform \u001b[39min\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_iter(with_final\u001b[39m=\u001b[39m\u001b[39mFalse\u001b[39;00m):\n\u001b[0;32m--> 535\u001b[0m     Xt \u001b[39m=\u001b[39m transform\u001b[39m.\u001b[39;49mtransform(Xt)\n\u001b[1;32m    536\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39msteps[\u001b[39m-\u001b[39m\u001b[39m1\u001b[39m][\u001b[39m1\u001b[39m]\u001b[39m.\u001b[39mpredict_proba(Xt, \u001b[39m*\u001b[39m\u001b[39m*\u001b[39mpredict_proba_params)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/sklearn/preprocessing/_function_transformer.py:182\u001b[0m, in \u001b[0;36mFunctionTransformer.transform\u001b[0;34m(self, X)\u001b[0m\n\u001b[1;32m    181\u001b[0m X \u001b[39m=\u001b[39m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_check_input(X, reset\u001b[39m=\u001b[39m\u001b[39mFalse\u001b[39;00m)\n\u001b[0;32m--> 182\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_transform(X, func\u001b[39m=\u001b[39;49m\u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mfunc, kw_args\u001b[39m=\u001b[39;49m\u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mkw_args)\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/sklearn/preprocessing/_function_transformer.py:205\u001b[0m, in \u001b[0;36mFunctionTransformer._transform\u001b[0;34m(self, X, func, kw_args)\u001b[0m\n\u001b[1;32m    203\u001b[0m     func \u001b[39m=\u001b[39m _identity\n\u001b[0;32m--> 205\u001b[0m \u001b[39mreturn\u001b[39;00m func(X, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49m(kw_args \u001b[39mif\u001b[39;49;00m kw_args \u001b[39melse\u001b[39;49;00m {}))\n",
      "Cell \u001b[0;32mIn[216], line 8\u001b[0m, in \u001b[0;36mpreprocess_text\u001b[0;34m(df)\u001b[0m\n\u001b[1;32m      7\u001b[0m \u001b[39m# Lower.\u001b[39;00m\n\u001b[0;32m----> 8\u001b[0m df[TEXT_COLUMN_NAME] \u001b[39m=\u001b[39m df[TEXT_COLUMN_NAME]\u001b[39m.\u001b[39;49mapply(\u001b[39mlambda\u001b[39;49;00m x: x\u001b[39m.\u001b[39;49mlower())\n\u001b[1;32m     10\u001b[0m \u001b[39m# Remove punctuation.\u001b[39;00m\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/pandas/core/series.py:4771\u001b[0m, in \u001b[0;36mSeries.apply\u001b[0;34m(self, func, convert_dtype, args, **kwargs)\u001b[0m\n\u001b[1;32m   4668\u001b[0m \u001b[39m\u001b[39m\u001b[39m\"\"\"\u001b[39;00m\n\u001b[1;32m   4669\u001b[0m \u001b[39mInvoke function on values of Series.\u001b[39;00m\n\u001b[1;32m   4670\u001b[0m \n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m   4769\u001b[0m \u001b[39mdtype: float64\u001b[39;00m\n\u001b[1;32m   4770\u001b[0m \u001b[39m\"\"\"\u001b[39;00m\n\u001b[0;32m-> 4771\u001b[0m \u001b[39mreturn\u001b[39;00m SeriesApply(\u001b[39mself\u001b[39;49m, func, convert_dtype, args, kwargs)\u001b[39m.\u001b[39;49mapply()\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/pandas/core/apply.py:1123\u001b[0m, in \u001b[0;36mSeriesApply.apply\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1122\u001b[0m \u001b[39m# self.f is Callable\u001b[39;00m\n\u001b[0;32m-> 1123\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mapply_standard()\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/pandas/core/apply.py:1174\u001b[0m, in \u001b[0;36mSeriesApply.apply_standard\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1173\u001b[0m         values \u001b[39m=\u001b[39m obj\u001b[39m.\u001b[39mastype(\u001b[39mobject\u001b[39m)\u001b[39m.\u001b[39m_values\n\u001b[0;32m-> 1174\u001b[0m         mapped \u001b[39m=\u001b[39m lib\u001b[39m.\u001b[39;49mmap_infer(\n\u001b[1;32m   1175\u001b[0m             values,\n\u001b[1;32m   1176\u001b[0m             f,\n\u001b[1;32m   1177\u001b[0m             convert\u001b[39m=\u001b[39;49m\u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mconvert_dtype,\n\u001b[1;32m   1178\u001b[0m         )\n\u001b[1;32m   1180\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39mlen\u001b[39m(mapped) \u001b[39mand\u001b[39;00m \u001b[39misinstance\u001b[39m(mapped[\u001b[39m0\u001b[39m], ABCSeries):\n\u001b[1;32m   1181\u001b[0m     \u001b[39m# GH#43986 Need to do list(mapped) in order to get treated as nested\u001b[39;00m\n\u001b[1;32m   1182\u001b[0m     \u001b[39m#  See also GH#25959 regarding EA support\u001b[39;00m\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/pandas/_libs/lib.pyx:2924\u001b[0m, in \u001b[0;36mpandas._libs.lib.map_infer\u001b[0;34m()\u001b[0m\n",
      "Cell \u001b[0;32mIn[216], line 8\u001b[0m, in \u001b[0;36mpreprocess_text.<locals>.<lambda>\u001b[0;34m(x)\u001b[0m\n\u001b[1;32m      7\u001b[0m \u001b[39m# Lower.\u001b[39;00m\n\u001b[0;32m----> 8\u001b[0m df[TEXT_COLUMN_NAME] \u001b[39m=\u001b[39m df[TEXT_COLUMN_NAME]\u001b[39m.\u001b[39mapply(\u001b[39mlambda\u001b[39;00m x: x\u001b[39m.\u001b[39;49mlower())\n\u001b[1;32m     10\u001b[0m \u001b[39m# Remove punctuation.\u001b[39;00m\n",
      "\u001b[0;31mAttributeError\u001b[0m: 'float' object has no attribute 'lower'",
      "\nThe above exception was the direct cause of the following exception:\n",
      "\u001b[0;31mException\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[368], line 4\u001b[0m\n\u001b[1;32m      2\u001b[0m input_df \u001b[39m=\u001b[39m wrapped_data\u001b[39m.\u001b[39mdf\u001b[39m.\u001b[39mdropna()\n\u001b[1;32m      3\u001b[0m input_df[\u001b[39m\"\u001b[39m\u001b[39mtranscription\u001b[39m\u001b[39m\"\u001b[39m]\u001b[39m.\u001b[39mastype(\u001b[39mstr\u001b[39m)\n\u001b[0;32m----> 4\u001b[0m explain_text(\n\u001b[1;32m      5\u001b[0m         model\u001b[39m=\u001b[39;49mwrapped_model, input_df\u001b[39m=\u001b[39;49minput_df, text_column\u001b[39m=\u001b[39;49m\u001b[39m\"\u001b[39;49m\u001b[39mtranscription\u001b[39;49m\u001b[39m\"\u001b[39;49m, text_document\u001b[39m=\u001b[39;49mwrapped_data\u001b[39m.\u001b[39;49mdf[\u001b[39m\"\u001b[39;49m\u001b[39mtranscription\u001b[39;49m\u001b[39m\"\u001b[39;49m]\u001b[39m.\u001b[39;49miloc[\u001b[39m1\u001b[39;49m]\n\u001b[1;32m      6\u001b[0m     )\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/ml_worker/utils/logging.py:76\u001b[0m, in \u001b[0;36mtimer.<locals>.timing_decorator.<locals>.wrap\u001b[0;34m(*args, **kw)\u001b[0m\n\u001b[1;32m     73\u001b[0m \u001b[39m@wraps\u001b[39m(timing_decorator)\n\u001b[1;32m     74\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39mwrap\u001b[39m(\u001b[39m*\u001b[39margs, \u001b[39m*\u001b[39m\u001b[39m*\u001b[39mkw):\n\u001b[1;32m     75\u001b[0m     \u001b[39mwith\u001b[39;00m Timer(message \u001b[39mif\u001b[39;00m message \u001b[39melse\u001b[39;00m \u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39m{\u001b[39;00mfn\u001b[39m.\u001b[39m\u001b[39m__module__\u001b[39m\u001b[39m}\u001b[39;00m\u001b[39m.\u001b[39m\u001b[39m{\u001b[39;00mfn\u001b[39m.\u001b[39m\u001b[39m__qualname__\u001b[39m\u001b[39m}\u001b[39;00m\u001b[39m\"\u001b[39m):\n\u001b[0;32m---> 76\u001b[0m         result \u001b[39m=\u001b[39m fn(\u001b[39m*\u001b[39;49margs, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkw)\n\u001b[1;32m     77\u001b[0m     \u001b[39mreturn\u001b[39;00m result\n",
      "File \u001b[0;32m~/PycharmProjects/giskard/python-client/.venv/lib/python3.10/site-packages/giskard/models/model_explanation.py:80\u001b[0m, in \u001b[0;36mexplain_text\u001b[0;34m(model, input_df, text_column, text_document)\u001b[0m\n\u001b[1;32m     78\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mException\u001b[39;00m \u001b[39mas\u001b[39;00m e:\n\u001b[1;32m     79\u001b[0m     logger\u001b[39m.\u001b[39mexception(\u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mFailed to explain text: \u001b[39m\u001b[39m{\u001b[39;00mtext_document\u001b[39m}\u001b[39;00m\u001b[39m\"\u001b[39m, e)\n\u001b[0;32m---> 80\u001b[0m     \u001b[39mraise\u001b[39;00m \u001b[39mException\u001b[39;00m(\u001b[39m\"\u001b[39m\u001b[39mFailed to create text explanation\u001b[39m\u001b[39m\"\u001b[39m) \u001b[39mfrom\u001b[39;00m \u001b[39me\u001b[39;00m\n",
      "\u001b[0;31mException\u001b[0m: Failed to create text explanation"
     ]
    }
   ],
   "source": [
    "from giskard.models.model_explanation import explain_text\n",
    "input_df = wrapped_data.df.dropna()\n",
    "input_df[\"transcription\"].astype(str)\n",
    "explain_text(\n",
    "        model=wrapped_model, input_df=input_df, text_column=\"transcription\", text_document=wrapped_data.df[\"transcription\"].iloc[1]\n",
    "    )\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 321,
   "id": "4563b8f9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>transcription</th>\n",
       "      <th>medical_specialty</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>3425</th>\n",
       "      <td>REASON FOR THE CONSULT:,  Nonhealing right ank...</td>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2855</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Low back pain.,POSTO...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS:  ,Carcinoma of the pro...</td>\n",
       "      <td>Urology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3419</th>\n",
       "      <td>CURRENT HISTORY:,  A 94-year-old female from t...</td>\n",
       "      <td>General Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3781</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Adenoid hypertrophy....</td>\n",
       "      <td>ENT - Otolaryngology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2562</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Intrauterine pregnan...</td>\n",
       "      <td>Obstetrics / Gynecology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2733</th>\n",
       "      <td>FINAL DIAGNOSES:,1.  Herniated nucleuses pulpo...</td>\n",
       "      <td>Neurosurgery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS:,  Desire for sterility...</td>\n",
       "      <td>Urology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1313</th>\n",
       "      <td>SUBJECTIVE:,  The patient is here for a follow...</td>\n",
       "      <td>SOAP / Chart / Progress Notes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2554</th>\n",
       "      <td>CHIEF COMPLAINT - REASON FOR VISIT: ,Pelvic Pa...</td>\n",
       "      <td>Obstetrics / Gynecology</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>371 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          transcription  \\\n",
       "3425  REASON FOR THE CONSULT:,  Nonhealing right ank...   \n",
       "2855  PREOPERATIVE DIAGNOSIS: , Low back pain.,POSTO...   \n",
       "155   PREOPERATIVE DIAGNOSIS:  ,Carcinoma of the pro...   \n",
       "3419  CURRENT HISTORY:,  A 94-year-old female from t...   \n",
       "3781  PREOPERATIVE DIAGNOSIS: , Adenoid hypertrophy....   \n",
       "...                                                 ...   \n",
       "2562  PREOPERATIVE DIAGNOSIS: , Intrauterine pregnan...   \n",
       "2733  FINAL DIAGNOSES:,1.  Herniated nucleuses pulpo...   \n",
       "29    PREOPERATIVE DIAGNOSIS:,  Desire for sterility...   \n",
       "1313  SUBJECTIVE:,  The patient is here for a follow...   \n",
       "2554  CHIEF COMPLAINT - REASON FOR VISIT: ,Pelvic Pa...   \n",
       "\n",
       "                  medical_specialty  \n",
       "3425               General Medicine  \n",
       "2855                      Neurology  \n",
       "155                         Urology  \n",
       "3419               General Medicine  \n",
       "3781           ENT - Otolaryngology  \n",
       "...                             ...  \n",
       "2562        Obstetrics / Gynecology  \n",
       "2733                   Neurosurgery  \n",
       "29                          Urology  \n",
       "1313  SOAP / Chart / Progress Notes  \n",
       "2554        Obstetrics / Gynecology  \n",
       "\n",
       "[371 rows x 2 columns]"
      ]
     },
     "execution_count": 321,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "wrapped_data.df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 322,
   "id": "69f2416d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>transcription</th>\n",
       "      <th>medical_specialty</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>2855</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Low back pain.,POSTO...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2794</th>\n",
       "      <td>REASON FOR CONSULT:,  Organic brain syndrome i...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2770</th>\n",
       "      <td>SUBJECTIVE: , The patient is a 55-year-old Afr...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2882</th>\n",
       "      <td>PREOPERATIVE DIAGNOSES:,1.  Epidural hematoma,...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2861</th>\n",
       "      <td>CC: ,RLE weakness.,HX: ,This 42y/o RHM was fou...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2765</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS:,  Rule out temporal ar...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2797</th>\n",
       "      <td>REASON FOR NEUROLOGICAL CONSULTATION:,  Muscle...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2900</th>\n",
       "      <td>IMPRESSION:,  Abnormal electroencephalogram re...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2865</th>\n",
       "      <td>CHIEF COMPLAINT:,  Intractable epilepsy, here ...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2764</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS:,  Headaches, question ...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2781</th>\n",
       "      <td>DIAGNOSIS:,  Status post brain tumor removal.,...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2852</th>\n",
       "      <td>CC: ,Delayed motor development.,HX:, This 21 m...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2842</th>\n",
       "      <td>CC: ,Progressive left visual field loss.,HX:, ...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2901</th>\n",
       "      <td>HISTORY:,  The patient is a 52-year-old female...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2843</th>\n",
       "      <td>CC: ,Progressive loss of color vision OD,HX:, ...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2786</th>\n",
       "      <td>EXAM: , Cervical, lumbosacral, thoracic spine ...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2925</th>\n",
       "      <td>EXAM:,  CT head.,REASON FOR EXAM:,  Seizure di...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2949</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Right chronic subdur...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2905</th>\n",
       "      <td>CHIEF COMPLAINT:,  Recurrent dizziness x1 mont...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2836</th>\n",
       "      <td>CC:, Lethargy.,HX:, This 28y/o RHM was admitte...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2849</th>\n",
       "      <td>HX: ,This 46y/o RHM with HTN was well until 2 ...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2757</th>\n",
       "      <td>TIME SEEN: , 0734 hours and 1034 hours.,TOTAL ...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2962</th>\n",
       "      <td>CC:, Fluctuating level of consciousness.,HX:, ...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2955</th>\n",
       "      <td>As you know, the patient is a 50-year-old righ...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2897</th>\n",
       "      <td>REPORT:,  The electroencephalogram shows backg...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2937</th>\n",
       "      <td>CC:, Progressive memory and cognitive decline....</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2848</th>\n",
       "      <td>CC:, Progressive visual loss.,HX:, 76 y/o male...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2766</th>\n",
       "      <td>REASON FOR CONSULTATION: , This 92-year-old fe...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2928</th>\n",
       "      <td>CC:, Horizontal diplopia.,HX: , This 67 y/oRHM...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2789</th>\n",
       "      <td>REASON FOR REFERRAL:,  The patient is a 76-yea...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2886</th>\n",
       "      <td>NERVE CONDUCTION STUDIES:,  Bilateral ulnar se...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2823</th>\n",
       "      <td>CC:, Left hemibody numbness.,HX:, This 44y/o R...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2826</th>\n",
       "      <td>FINDINGS:,Normal foramen magnum.,Normal brains...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2844</th>\n",
       "      <td>FINDINGS:,There is moderate to severe generali...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2880</th>\n",
       "      <td>CC:, Fall with questionable associated loss of...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2817</th>\n",
       "      <td>CC:, Progressive lower extremity weakness.,HX:...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2941</th>\n",
       "      <td>PREOPERATIVE DIAGNOSIS: , Brain tumors, multip...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2853</th>\n",
       "      <td>PROCEDURE: , Lumbar puncture with moderate sed...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2891</th>\n",
       "      <td>HISTORY:, This is a digital EEG performed on a...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2907</th>\n",
       "      <td>HISTORY OF PRESENT ILLNESS: , The patient is a...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2825</th>\n",
       "      <td>CC:, Right shoulder pain.,HX: ,This 46 y/o RHF...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2784</th>\n",
       "      <td>PROCEDURES PERFORMED:,  Phenol neurolysis left...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2967</th>\n",
       "      <td>CC:, Headache.,HX:, 63 y/o RHF first seen by N...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2814</th>\n",
       "      <td>EXAM:  ,Thoracic Spine.,REASON FOR EXAM: , Inj...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2911</th>\n",
       "      <td>DISCHARGE DIAGNOSES:,1.  Bilateral lower extre...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2816</th>\n",
       "      <td>INTERPRETATION: , MRI of the cervical spine wi...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2850</th>\n",
       "      <td>HISTORY OF PRESENT ILLNESS: , The patient is a...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2934</th>\n",
       "      <td>CC:, Left-sided weakness.,HX:, This 28y/o RHM ...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2841</th>\n",
       "      <td>FINDINGS:,There is a large intrasellar mass le...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2920</th>\n",
       "      <td>REASON FOR CT SCAN: , The patient is a 79-year...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2818</th>\n",
       "      <td>CC:, Progressive lower extremity weakness.,HX:...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2795</th>\n",
       "      <td>REASON FOR EVALUATION: , The patient is a 37-y...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2910</th>\n",
       "      <td>DIAGNOSES ON ADMISSION,1.  Cerebrovascular acc...</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          transcription medical_specialty\n",
       "2855  PREOPERATIVE DIAGNOSIS: , Low back pain.,POSTO...         Neurology\n",
       "2794  REASON FOR CONSULT:,  Organic brain syndrome i...         Neurology\n",
       "2770  SUBJECTIVE: , The patient is a 55-year-old Afr...         Neurology\n",
       "2882  PREOPERATIVE DIAGNOSES:,1.  Epidural hematoma,...         Neurology\n",
       "2861  CC: ,RLE weakness.,HX: ,This 42y/o RHM was fou...         Neurology\n",
       "2765  PREOPERATIVE DIAGNOSIS:,  Rule out temporal ar...         Neurology\n",
       "2797  REASON FOR NEUROLOGICAL CONSULTATION:,  Muscle...         Neurology\n",
       "2900  IMPRESSION:,  Abnormal electroencephalogram re...         Neurology\n",
       "2865  CHIEF COMPLAINT:,  Intractable epilepsy, here ...         Neurology\n",
       "2764  PREOPERATIVE DIAGNOSIS:,  Headaches, question ...         Neurology\n",
       "2781  DIAGNOSIS:,  Status post brain tumor removal.,...         Neurology\n",
       "2852  CC: ,Delayed motor development.,HX:, This 21 m...         Neurology\n",
       "2842  CC: ,Progressive left visual field loss.,HX:, ...         Neurology\n",
       "2901  HISTORY:,  The patient is a 52-year-old female...         Neurology\n",
       "2843  CC: ,Progressive loss of color vision OD,HX:, ...         Neurology\n",
       "2786  EXAM: , Cervical, lumbosacral, thoracic spine ...         Neurology\n",
       "2925  EXAM:,  CT head.,REASON FOR EXAM:,  Seizure di...         Neurology\n",
       "2949  PREOPERATIVE DIAGNOSIS: , Right chronic subdur...         Neurology\n",
       "2905  CHIEF COMPLAINT:,  Recurrent dizziness x1 mont...         Neurology\n",
       "2836  CC:, Lethargy.,HX:, This 28y/o RHM was admitte...         Neurology\n",
       "2849  HX: ,This 46y/o RHM with HTN was well until 2 ...         Neurology\n",
       "2757  TIME SEEN: , 0734 hours and 1034 hours.,TOTAL ...         Neurology\n",
       "2962  CC:, Fluctuating level of consciousness.,HX:, ...         Neurology\n",
       "2955  As you know, the patient is a 50-year-old righ...         Neurology\n",
       "2897  REPORT:,  The electroencephalogram shows backg...         Neurology\n",
       "2937  CC:, Progressive memory and cognitive decline....         Neurology\n",
       "2848  CC:, Progressive visual loss.,HX:, 76 y/o male...         Neurology\n",
       "2766  REASON FOR CONSULTATION: , This 92-year-old fe...         Neurology\n",
       "2928  CC:, Horizontal diplopia.,HX: , This 67 y/oRHM...         Neurology\n",
       "2789  REASON FOR REFERRAL:,  The patient is a 76-yea...         Neurology\n",
       "2886  NERVE CONDUCTION STUDIES:,  Bilateral ulnar se...         Neurology\n",
       "2823  CC:, Left hemibody numbness.,HX:, This 44y/o R...         Neurology\n",
       "2826  FINDINGS:,Normal foramen magnum.,Normal brains...         Neurology\n",
       "2844  FINDINGS:,There is moderate to severe generali...         Neurology\n",
       "2880  CC:, Fall with questionable associated loss of...         Neurology\n",
       "2817  CC:, Progressive lower extremity weakness.,HX:...         Neurology\n",
       "2941  PREOPERATIVE DIAGNOSIS: , Brain tumors, multip...         Neurology\n",
       "2853  PROCEDURE: , Lumbar puncture with moderate sed...         Neurology\n",
       "2891  HISTORY:, This is a digital EEG performed on a...         Neurology\n",
       "2907  HISTORY OF PRESENT ILLNESS: , The patient is a...         Neurology\n",
       "2825  CC:, Right shoulder pain.,HX: ,This 46 y/o RHF...         Neurology\n",
       "2784  PROCEDURES PERFORMED:,  Phenol neurolysis left...         Neurology\n",
       "2967  CC:, Headache.,HX:, 63 y/o RHF first seen by N...         Neurology\n",
       "2814  EXAM:  ,Thoracic Spine.,REASON FOR EXAM: , Inj...         Neurology\n",
       "2911  DISCHARGE DIAGNOSES:,1.  Bilateral lower extre...         Neurology\n",
       "2816  INTERPRETATION: , MRI of the cervical spine wi...         Neurology\n",
       "2850  HISTORY OF PRESENT ILLNESS: , The patient is a...         Neurology\n",
       "2934  CC:, Left-sided weakness.,HX:, This 28y/o RHM ...         Neurology\n",
       "2841  FINDINGS:,There is a large intrasellar mass le...         Neurology\n",
       "2920  REASON FOR CT SCAN: , The patient is a 79-year...         Neurology\n",
       "2818  CC:, Progressive lower extremity weakness.,HX:...         Neurology\n",
       "2795  REASON FOR EVALUATION: , The patient is a 37-y...         Neurology\n",
       "2910  DIAGNOSES ON ADMISSION,1.  Cerebrovascular acc...         Neurology"
      ]
     },
     "execution_count": 322,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "wrapped_data.df[wrapped_data.df.medical_specialty==\"Neurology\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b585fac1",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "for-vscode",
   "language": "python",
   "name": "for-vscode"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.11"
  },
  "papermill": {
   "default_parameters": {},
   "duration": 1295.42731,
   "end_time": "2022-09-14T15:45:13.308486",
   "environment_variables": {},
   "exception": null,
   "input_path": "__notebook__.ipynb",
   "output_path": "__notebook__.ipynb",
   "parameters": {},
   "start_time": "2022-09-14T15:23:37.881176",
   "version": "2.3.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
